{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# NER Spacy model,  with ready data, for few categories\n",
    "\n",
    "Plan:\n",
    "    - load the fully ready sentences\n",
    "    - count how many are for each category (cell, protein, gene)\n",
    "    - make training / test set with entities labelled for spacy \n",
    "    - \n",
    "    - ...\n",
    "    - train bert on the same set and compare \n",
    "    "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " https://medium.com/@nikkisharma536/how-to-create-custom-ner-in-spacy-cfcd531f8773"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. [Define entities of interest.](#fyrsta)\n",
    "2. [Load the newest tagged set.](#annað)\n",
    "3. [Select fully labelled sentences.](#þriðja)\n",
    "4. [Transform into Spacy NER format.](#fjórða)\n",
    "5. [Split into training / test sets.](#fimmta)\n",
    "6. [Load scispacy model. Train (for a few shuffled iterations).](#sjötta)\n",
    "7. [Test!](#sjöunda)\n",
    "8. [Try scispacy's NER models](#áttunda)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "341\n"
     ]
    }
   ],
   "source": [
    "# Load wikilabelled and use it to tag the newet set.\n",
    "\n",
    "import json \n",
    "\n",
    "with open(os.path.dirname(os.getcwd()) + '/wiki_labelled.json') as f:\n",
    "    wikilab = json.load(f)\n",
    "    print(len(wikilab))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "1. Define classes and update the newest tagged set <a name=\"fyrsta\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "# define entities of interest\n",
    "\n",
    "interesting = ['cell_line', 'science_word', 'reagent', 'gene']\n",
    "#interesting = ['number', 'unit', 'table_cell']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 62,
   "metadata": {},
   "outputs": [],
   "source": [
    "# mark unambiguous terms from  from wikilabelled\n",
    "\n",
    "catz = []\n",
    "n_wikicats = []\n",
    "ambiguous = []\n",
    "#new_file = open('SPACY NER/wiki_unambiguous', 'w') \n",
    "\n",
    "\n",
    "with open('SPACY NER/imbalanced3') as cells:\n",
    "\n",
    "    # label everything else with an 'O'\n",
    "    \n",
    "    for l in cells.readlines():\n",
    "        items = l.split(',')\n",
    "        if len(items) == 5:\n",
    "            # skip strange rows \n",
    "            word = items[2]\n",
    "            cat = items[4].strip()\n",
    "            catz.append(cat)\n",
    "                \n",
    "            if word in [x[0] for x in wikilab]:\n",
    "                # been wikilabelled\n",
    "                if cat == 'X':\n",
    "                    # was unlabelled\n",
    "                    its_cat = [x[1] for x in wikilab if x[0] == word]\n",
    "                    if len(its_cat) == 1:\n",
    "                        # unambiguously labelled \n",
    "                        items[4] = its_cat[0]\n",
    "                        \n",
    "                        l = ','.join(items) + '\\n'\n",
    "                    else:\n",
    "                        ambiguous.append((word,its_cat))\n",
    "#            new_file.write(l)\n",
    "\n",
    "            "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "826"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "unlabelled = count_unknowns('SPACY NER/wiki_ambiguous')\n",
    "len(unlabelled)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'/Users/valdimareggertsson/Documents/Valdi/Vetrarönn 2019/NER/Spacy NER'"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import os\n",
    "os.getcwd()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "857"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "def count_unknowns(path_to_csv):\n",
    "\n",
    "    unknowns = []\n",
    "\n",
    "    with open(path_to_csv) as labelled_data:\n",
    "\n",
    "\n",
    "        for l in labelled_data.readlines():\n",
    "            items = l.split(',')\n",
    "\n",
    "            if len(items) == 5:\n",
    "                # skip strange rows \n",
    "\n",
    "                new_sentence_nr = items[1]\n",
    "                word = items[2]\n",
    "                category = items[4].strip()\n",
    "\n",
    "                if category == 'X':\n",
    "                    unknowns.append(word)\n",
    "\n",
    "\n",
    "    unknowns = sorted(list(set(unknowns)))\n",
    "\n",
    "    return unknowns\n",
    "\n",
    "\n",
    "len(count_unknowns('SPACY NER/wiki_unambiguous'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. Take the marked sentences from the categories of interest <a name=\"annað\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "\n",
    "unlabelled = count_unknowns('SPACY NER/imbalanced')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "### Get unlabelled and labelled\n",
    "\n",
    "sentences_with_unlabelled = []\n",
    "sentence_ids = [] \n",
    "with open('SPACY NER/imbalanced2') as cells:\n",
    "    \n",
    "    # skip the header \n",
    "    for l in cells.readlines()[1:]:\n",
    "        items = l.split(',')\n",
    "        \n",
    "        for i in unlabelled:\n",
    "            if i in l:\n",
    "                sentences_with_unlabelled.append(items[1])\n",
    "#                print(items[1])\n",
    "\n",
    "        if len(items) == 5:\n",
    "            sentence_ids.append(items[1])\n",
    "                "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[0,\n",
       " 1,\n",
       " 2,\n",
       " 3,\n",
       " 4,\n",
       " 5,\n",
       " 6,\n",
       " 7,\n",
       " 8,\n",
       " 9,\n",
       " 10,\n",
       " 11,\n",
       " 12,\n",
       " 13,\n",
       " 14,\n",
       " 15,\n",
       " 16,\n",
       " 17,\n",
       " 18,\n",
       " 19,\n",
       " 20,\n",
       " 21,\n",
       " 22,\n",
       " 23,\n",
       " 24,\n",
       " 25,\n",
       " 26,\n",
       " 27,\n",
       " 28,\n",
       " 29,\n",
       " 30,\n",
       " 31,\n",
       " 32,\n",
       " 33,\n",
       " 34,\n",
       " 35,\n",
       " 36,\n",
       " 37,\n",
       " 38,\n",
       " 39,\n",
       " 40,\n",
       " 41,\n",
       " 42,\n",
       " 43,\n",
       " 44,\n",
       " 45,\n",
       " 46,\n",
       " 47,\n",
       " 48,\n",
       " 49,\n",
       " 50,\n",
       " 51,\n",
       " 52,\n",
       " 53,\n",
       " 54,\n",
       " 55,\n",
       " 56,\n",
       " 57,\n",
       " 58,\n",
       " 59,\n",
       " 60,\n",
       " 61,\n",
       " 62,\n",
       " 63,\n",
       " 64,\n",
       " 65,\n",
       " 66,\n",
       " 67,\n",
       " 68,\n",
       " 69,\n",
       " 70,\n",
       " 71,\n",
       " 72,\n",
       " 73,\n",
       " 74,\n",
       " 75,\n",
       " 76,\n",
       " 77,\n",
       " 78,\n",
       " 79,\n",
       " 80,\n",
       " 81,\n",
       " 82,\n",
       " 83,\n",
       " 84,\n",
       " 85,\n",
       " 86,\n",
       " 87,\n",
       " 88,\n",
       " 89,\n",
       " 90,\n",
       " 91,\n",
       " 92,\n",
       " 93,\n",
       " 94,\n",
       " 95,\n",
       " 96,\n",
       " 97,\n",
       " 98,\n",
       " 99,\n",
       " 100,\n",
       " 101,\n",
       " 102,\n",
       " 103,\n",
       " 104,\n",
       " 105,\n",
       " 106,\n",
       " 107,\n",
       " 108,\n",
       " 109,\n",
       " 110,\n",
       " 111,\n",
       " 112,\n",
       " 113,\n",
       " 114,\n",
       " 115,\n",
       " 116,\n",
       " 117,\n",
       " 118,\n",
       " 119,\n",
       " 120,\n",
       " 121,\n",
       " 122,\n",
       " 123,\n",
       " 124,\n",
       " 125,\n",
       " 126,\n",
       " 127,\n",
       " 128,\n",
       " 129,\n",
       " 130,\n",
       " 131,\n",
       " 132,\n",
       " 133,\n",
       " 134,\n",
       " 135,\n",
       " 136,\n",
       " 137,\n",
       " 138,\n",
       " 139,\n",
       " 140,\n",
       " 141,\n",
       " 142,\n",
       " 143,\n",
       " 144,\n",
       " 145,\n",
       " 146,\n",
       " 147,\n",
       " 148,\n",
       " 149,\n",
       " 150,\n",
       " 151,\n",
       " 152,\n",
       " 153,\n",
       " 154,\n",
       " 155,\n",
       " 156,\n",
       " 157,\n",
       " 158,\n",
       " 159,\n",
       " 160,\n",
       " 161,\n",
       " 162,\n",
       " 163,\n",
       " 164,\n",
       " 165,\n",
       " 166,\n",
       " 167,\n",
       " 168,\n",
       " 169,\n",
       " 170,\n",
       " 171,\n",
       " 172,\n",
       " 173,\n",
       " 174,\n",
       " 175,\n",
       " 176,\n",
       " 177,\n",
       " 178,\n",
       " 179,\n",
       " 180,\n",
       " 181,\n",
       " 182,\n",
       " 183,\n",
       " 184,\n",
       " 185,\n",
       " 186,\n",
       " 187,\n",
       " 188,\n",
       " 189,\n",
       " 190,\n",
       " 191,\n",
       " 192,\n",
       " 193,\n",
       " 194,\n",
       " 195,\n",
       " 196,\n",
       " 197,\n",
       " 198,\n",
       " 199,\n",
       " 200,\n",
       " 201,\n",
       " 202,\n",
       " 203,\n",
       " 204,\n",
       " 205,\n",
       " 206,\n",
       " 207,\n",
       " 208,\n",
       " 209,\n",
       " 210,\n",
       " 211,\n",
       " 212,\n",
       " 213,\n",
       " 214,\n",
       " 215,\n",
       " 216,\n",
       " 217,\n",
       " 218,\n",
       " 219,\n",
       " 220,\n",
       " 221,\n",
       " 222,\n",
       " 223,\n",
       " 224,\n",
       " 225,\n",
       " 226,\n",
       " 227,\n",
       " 228,\n",
       " 229,\n",
       " 230,\n",
       " 231,\n",
       " 232,\n",
       " 233,\n",
       " 234,\n",
       " 235,\n",
       " 236,\n",
       " 237,\n",
       " 238,\n",
       " 239,\n",
       " 240,\n",
       " 241,\n",
       " 242,\n",
       " 243,\n",
       " 244,\n",
       " 245,\n",
       " 246,\n",
       " 247,\n",
       " 248,\n",
       " 249,\n",
       " 250,\n",
       " 251,\n",
       " 252,\n",
       " 253,\n",
       " 254,\n",
       " 255,\n",
       " 256,\n",
       " 257,\n",
       " 258,\n",
       " 259,\n",
       " 260,\n",
       " 261,\n",
       " 262,\n",
       " 263,\n",
       " 264,\n",
       " 265,\n",
       " 266,\n",
       " 267,\n",
       " 268,\n",
       " 269,\n",
       " 270,\n",
       " 271,\n",
       " 272,\n",
       " 273,\n",
       " 274,\n",
       " 275,\n",
       " 276,\n",
       " 277,\n",
       " 278,\n",
       " 279,\n",
       " 280,\n",
       " 281,\n",
       " 282,\n",
       " 283,\n",
       " 284,\n",
       " 285,\n",
       " 286,\n",
       " 287,\n",
       " 288,\n",
       " 289,\n",
       " 290,\n",
       " 291,\n",
       " 292,\n",
       " 293,\n",
       " 294,\n",
       " 295,\n",
       " 296,\n",
       " 297,\n",
       " 298,\n",
       " 299,\n",
       " 300,\n",
       " 301,\n",
       " 302,\n",
       " 303,\n",
       " 304,\n",
       " 305,\n",
       " 306,\n",
       " 307,\n",
       " 308,\n",
       " 309,\n",
       " 310,\n",
       " 311,\n",
       " 312,\n",
       " 313,\n",
       " 314,\n",
       " 315,\n",
       " 316,\n",
       " 317,\n",
       " 318,\n",
       " 319,\n",
       " 320,\n",
       " 321,\n",
       " 322,\n",
       " 323,\n",
       " 324,\n",
       " 325,\n",
       " 326,\n",
       " 327,\n",
       " 328,\n",
       " 329,\n",
       " 330,\n",
       " 331,\n",
       " 332,\n",
       " 333,\n",
       " 334,\n",
       " 335,\n",
       " 336,\n",
       " 337,\n",
       " 338,\n",
       " 339,\n",
       " 340,\n",
       " 341,\n",
       " 342,\n",
       " 343,\n",
       " 344,\n",
       " 345,\n",
       " 346,\n",
       " 347,\n",
       " 348,\n",
       " 349,\n",
       " 350,\n",
       " 351,\n",
       " 352,\n",
       " 353,\n",
       " 354,\n",
       " 355,\n",
       " 356,\n",
       " 357,\n",
       " 358,\n",
       " 359,\n",
       " 360,\n",
       " 361,\n",
       " 362,\n",
       " 363,\n",
       " 364,\n",
       " 365,\n",
       " 366,\n",
       " 367,\n",
       " 368,\n",
       " 369,\n",
       " 370,\n",
       " 371,\n",
       " 372,\n",
       " 373,\n",
       " 374,\n",
       " 375,\n",
       " 376,\n",
       " 377,\n",
       " 378,\n",
       " 379,\n",
       " 380,\n",
       " 381,\n",
       " 382,\n",
       " 383,\n",
       " 384,\n",
       " 385,\n",
       " 386,\n",
       " 387,\n",
       " 388,\n",
       " 389,\n",
       " 390,\n",
       " 391,\n",
       " 392,\n",
       " 393,\n",
       " 394,\n",
       " 395,\n",
       " 396,\n",
       " 397,\n",
       " 398,\n",
       " 399,\n",
       " 400,\n",
       " 401,\n",
       " 402,\n",
       " 403,\n",
       " 404,\n",
       " 405,\n",
       " 406,\n",
       " 407,\n",
       " 408,\n",
       " 409,\n",
       " 410,\n",
       " 411,\n",
       " 412,\n",
       " 413,\n",
       " 414,\n",
       " 415,\n",
       " 416,\n",
       " 417,\n",
       " 418,\n",
       " 419,\n",
       " 420,\n",
       " 421,\n",
       " 422,\n",
       " 423,\n",
       " 424,\n",
       " 425,\n",
       " 426,\n",
       " 427,\n",
       " 428,\n",
       " 429,\n",
       " 430,\n",
       " 431,\n",
       " 432,\n",
       " 433,\n",
       " 434,\n",
       " 435,\n",
       " 436,\n",
       " 437,\n",
       " 438,\n",
       " 439,\n",
       " 440,\n",
       " 441,\n",
       " 442,\n",
       " 443,\n",
       " 444,\n",
       " 445,\n",
       " 446,\n",
       " 447,\n",
       " 448,\n",
       " 449,\n",
       " 450,\n",
       " 451,\n",
       " 452,\n",
       " 453,\n",
       " 454,\n",
       " 455,\n",
       " 456,\n",
       " 457,\n",
       " 458,\n",
       " 459,\n",
       " 460,\n",
       " 461,\n",
       " 462,\n",
       " 463,\n",
       " 464,\n",
       " 465,\n",
       " 466,\n",
       " 467,\n",
       " 468,\n",
       " 469,\n",
       " 470,\n",
       " 471,\n",
       " 472,\n",
       " 473,\n",
       " 474,\n",
       " 475,\n",
       " 476,\n",
       " 477,\n",
       " 478,\n",
       " 479,\n",
       " 480,\n",
       " 481,\n",
       " 482,\n",
       " 483,\n",
       " 484,\n",
       " 485,\n",
       " 486,\n",
       " 487,\n",
       " 488,\n",
       " 489,\n",
       " 490,\n",
       " 491,\n",
       " 492,\n",
       " 493,\n",
       " 494,\n",
       " 495,\n",
       " 496,\n",
       " 497,\n",
       " 498,\n",
       " 499,\n",
       " 500,\n",
       " 501,\n",
       " 502,\n",
       " 503,\n",
       " 504,\n",
       " 505,\n",
       " 506,\n",
       " 507,\n",
       " 508,\n",
       " 509,\n",
       " 510,\n",
       " 511,\n",
       " 512,\n",
       " 513,\n",
       " 514,\n",
       " 515,\n",
       " 516,\n",
       " 517,\n",
       " 518,\n",
       " 519,\n",
       " 520,\n",
       " 521,\n",
       " 522,\n",
       " 523,\n",
       " 524,\n",
       " 525,\n",
       " 526,\n",
       " 527,\n",
       " 528,\n",
       " 529,\n",
       " 530,\n",
       " 531,\n",
       " 532,\n",
       " 533,\n",
       " 534,\n",
       " 535,\n",
       " 536,\n",
       " 537,\n",
       " 538,\n",
       " 539,\n",
       " 540,\n",
       " 541,\n",
       " 542,\n",
       " 543,\n",
       " 544,\n",
       " 545,\n",
       " 546,\n",
       " 547,\n",
       " 548,\n",
       " 549,\n",
       " 550,\n",
       " 551,\n",
       " 552,\n",
       " 553,\n",
       " 554,\n",
       " 555,\n",
       " 556,\n",
       " 557,\n",
       " 558,\n",
       " 559,\n",
       " 560,\n",
       " 561,\n",
       " 562,\n",
       " 563,\n",
       " 564,\n",
       " 565,\n",
       " 566,\n",
       " 567,\n",
       " 568,\n",
       " 569,\n",
       " 570,\n",
       " 571,\n",
       " 572,\n",
       " 573,\n",
       " 574,\n",
       " 575,\n",
       " 576,\n",
       " 577,\n",
       " 578,\n",
       " 579,\n",
       " 580,\n",
       " 581,\n",
       " 582,\n",
       " 583,\n",
       " 584,\n",
       " 585,\n",
       " 586,\n",
       " 587,\n",
       " 588,\n",
       " 589,\n",
       " 590,\n",
       " 591,\n",
       " 592,\n",
       " 593,\n",
       " 594,\n",
       " 595,\n",
       " 596,\n",
       " 597,\n",
       " 598,\n",
       " 599,\n",
       " 600,\n",
       " 601,\n",
       " 602,\n",
       " 603,\n",
       " 604,\n",
       " 605,\n",
       " 606,\n",
       " 607,\n",
       " 608,\n",
       " 609,\n",
       " 610,\n",
       " 611,\n",
       " 612,\n",
       " 613,\n",
       " 614,\n",
       " 615,\n",
       " 616,\n",
       " 617,\n",
       " 618,\n",
       " 619,\n",
       " 620,\n",
       " 621,\n",
       " 622,\n",
       " 623,\n",
       " 624,\n",
       " 625,\n",
       " 626,\n",
       " 627,\n",
       " 628,\n",
       " 629,\n",
       " 630,\n",
       " 631,\n",
       " 632,\n",
       " 633,\n",
       " 634,\n",
       " 635,\n",
       " 636,\n",
       " 637,\n",
       " 638,\n",
       " 639,\n",
       " 640,\n",
       " 641,\n",
       " 642,\n",
       " 643,\n",
       " 644,\n",
       " 645,\n",
       " 646,\n",
       " 647,\n",
       " 648,\n",
       " 649,\n",
       " 650,\n",
       " 651,\n",
       " 652,\n",
       " 653,\n",
       " 654,\n",
       " 655,\n",
       " 656,\n",
       " 657,\n",
       " 658,\n",
       " 659,\n",
       " 660,\n",
       " 661,\n",
       " 662,\n",
       " 663,\n",
       " 664,\n",
       " 665,\n",
       " 666,\n",
       " 667,\n",
       " 668,\n",
       " 669,\n",
       " 670,\n",
       " 671,\n",
       " 672,\n",
       " 673,\n",
       " 674,\n",
       " 675,\n",
       " 676,\n",
       " 677,\n",
       " 678,\n",
       " 679,\n",
       " 680,\n",
       " 681,\n",
       " 682,\n",
       " 683,\n",
       " 684,\n",
       " 685,\n",
       " 686,\n",
       " 687,\n",
       " 688,\n",
       " 689,\n",
       " 690,\n",
       " 691,\n",
       " 692,\n",
       " 693,\n",
       " 694,\n",
       " 695,\n",
       " 696,\n",
       " 697,\n",
       " 698,\n",
       " 699,\n",
       " 700,\n",
       " 701,\n",
       " 702,\n",
       " 703,\n",
       " 704,\n",
       " 705,\n",
       " 706,\n",
       " 707,\n",
       " 708,\n",
       " 709,\n",
       " 710,\n",
       " 711,\n",
       " 712,\n",
       " 713,\n",
       " 714,\n",
       " 715,\n",
       " 716,\n",
       " 717,\n",
       " 718,\n",
       " 719,\n",
       " 720,\n",
       " 721,\n",
       " 722,\n",
       " 723,\n",
       " 724,\n",
       " 725,\n",
       " 726,\n",
       " 727,\n",
       " 728,\n",
       " 729,\n",
       " 730,\n",
       " 731,\n",
       " 732,\n",
       " 733,\n",
       " 734,\n",
       " 735,\n",
       " 736,\n",
       " 737,\n",
       " 738,\n",
       " 739,\n",
       " 740,\n",
       " 741,\n",
       " 742,\n",
       " 743,\n",
       " 744,\n",
       " 745,\n",
       " 746,\n",
       " 747,\n",
       " 748,\n",
       " 749,\n",
       " 750,\n",
       " 751,\n",
       " 752,\n",
       " 753,\n",
       " 754,\n",
       " 755,\n",
       " 756,\n",
       " 757,\n",
       " 758,\n",
       " 759,\n",
       " 760,\n",
       " 761,\n",
       " 762,\n",
       " 763,\n",
       " 764,\n",
       " 765,\n",
       " 766,\n",
       " 767,\n",
       " 768,\n",
       " 769,\n",
       " 770,\n",
       " 771,\n",
       " 772,\n",
       " 773,\n",
       " 774,\n",
       " 775,\n",
       " 776,\n",
       " 777,\n",
       " 778,\n",
       " 779,\n",
       " 780,\n",
       " 781,\n",
       " 782,\n",
       " 783,\n",
       " 784,\n",
       " 785,\n",
       " 786,\n",
       " 787,\n",
       " 788,\n",
       " 789,\n",
       " 790,\n",
       " 791,\n",
       " 792,\n",
       " 793,\n",
       " 794,\n",
       " 795,\n",
       " 796,\n",
       " 797,\n",
       " 798,\n",
       " 799,\n",
       " 800,\n",
       " 801,\n",
       " 802,\n",
       " 803,\n",
       " 804,\n",
       " 805,\n",
       " 806,\n",
       " 807,\n",
       " 808,\n",
       " 809,\n",
       " 810,\n",
       " 811,\n",
       " 812,\n",
       " 813,\n",
       " 814,\n",
       " 815,\n",
       " 816,\n",
       " 817,\n",
       " 818,\n",
       " 819,\n",
       " 820,\n",
       " 821,\n",
       " 822,\n",
       " 823,\n",
       " 824,\n",
       " 825,\n",
       " 826,\n",
       " 827,\n",
       " 828,\n",
       " 829,\n",
       " 830,\n",
       " 831,\n",
       " 832,\n",
       " 833,\n",
       " 834,\n",
       " 835,\n",
       " 836,\n",
       " 837,\n",
       " 838,\n",
       " 839,\n",
       " 840,\n",
       " 841,\n",
       " 842,\n",
       " 843,\n",
       " 844,\n",
       " 845,\n",
       " 846,\n",
       " 847,\n",
       " 848,\n",
       " 849,\n",
       " 850,\n",
       " 851,\n",
       " 852,\n",
       " 853,\n",
       " 854,\n",
       " 855,\n",
       " 856,\n",
       " 857,\n",
       " 858,\n",
       " 859,\n",
       " 860,\n",
       " 861,\n",
       " 862,\n",
       " 863,\n",
       " 864,\n",
       " 865,\n",
       " 866,\n",
       " 867,\n",
       " 868,\n",
       " 869,\n",
       " 870,\n",
       " 871,\n",
       " 872,\n",
       " 873,\n",
       " 874,\n",
       " 875,\n",
       " 876,\n",
       " 877,\n",
       " 878,\n",
       " 879,\n",
       " 880,\n",
       " 881,\n",
       " 882,\n",
       " 883,\n",
       " 884,\n",
       " 885,\n",
       " 886,\n",
       " 887,\n",
       " 888,\n",
       " 889,\n",
       " 890,\n",
       " 891,\n",
       " 892,\n",
       " 893,\n",
       " 894,\n",
       " 895,\n",
       " 896,\n",
       " 897,\n",
       " 898,\n",
       " 899,\n",
       " 900,\n",
       " 901,\n",
       " 902,\n",
       " 903,\n",
       " 904,\n",
       " 905,\n",
       " 906,\n",
       " 907,\n",
       " 908,\n",
       " 909,\n",
       " 910,\n",
       " 911,\n",
       " 912,\n",
       " 913,\n",
       " 914,\n",
       " 915,\n",
       " 916,\n",
       " 917,\n",
       " 918,\n",
       " 919,\n",
       " 920,\n",
       " 921,\n",
       " 922,\n",
       " 923,\n",
       " 924,\n",
       " 925,\n",
       " 926,\n",
       " 927,\n",
       " 928,\n",
       " 929,\n",
       " 930,\n",
       " 931,\n",
       " 932,\n",
       " 933,\n",
       " 934,\n",
       " 935,\n",
       " 936,\n",
       " 937,\n",
       " 938,\n",
       " 939,\n",
       " 940,\n",
       " 941,\n",
       " 942,\n",
       " 943,\n",
       " 944,\n",
       " 945,\n",
       " 946,\n",
       " 947,\n",
       " 948,\n",
       " 949,\n",
       " 950,\n",
       " 951,\n",
       " 952,\n",
       " 953,\n",
       " 954,\n",
       " 955,\n",
       " 956,\n",
       " 957,\n",
       " 958,\n",
       " 959,\n",
       " 960,\n",
       " 961,\n",
       " 962,\n",
       " 963,\n",
       " 964,\n",
       " 965,\n",
       " 966,\n",
       " 967,\n",
       " 968,\n",
       " 969,\n",
       " 970,\n",
       " 971,\n",
       " 972,\n",
       " 973,\n",
       " 974,\n",
       " 975,\n",
       " 976,\n",
       " 977,\n",
       " 978,\n",
       " 979,\n",
       " 980,\n",
       " 981,\n",
       " 982,\n",
       " 983,\n",
       " 984,\n",
       " 985,\n",
       " 986,\n",
       " 987,\n",
       " 988,\n",
       " 989,\n",
       " 990,\n",
       " 991,\n",
       " 992,\n",
       " 993,\n",
       " 994,\n",
       " 995,\n",
       " 996,\n",
       " 997,\n",
       " 998,\n",
       " 999,\n",
       " ...]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ready_ids = sorted(list(set([int(x) for x in sentence_ids if x not in sentences_with_unlabelled])))\n",
    "ready_ids"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "17705"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(ready_ids)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "ready_words = []\n",
    "with open('SPACY NER/imbalanced2') as f:\n",
    "    \n",
    "    for l in f.readlines()[1:]:\n",
    "        items = l.split(',')\n",
    "        \n",
    "        if len(items) == 5:\n",
    "            if int(items[1]) in ready_ids:\n",
    "                ready_words.append(items)\n",
    "                "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3. Filter for fully labelled sentences <a name=\"þriðja\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "# filter for fully labelled sentences containing the entities \n",
    "\n",
    "current_sentence = '0'\n",
    "ready_s = [[]]\n",
    "ready_labels = [[]] \n",
    "\n",
    "for w in ready_words:\n",
    "    if w[1] == current_sentence:\n",
    "        ready_s[-1].append(w[2])\n",
    "        ready_labels[-1].append(w[4].strip())\n",
    "\n",
    "    else:\n",
    "        current_sentence = w[1]\n",
    "        ready_s.append([])\n",
    "        ready_labels.append([])\n",
    "        \n",
    "        "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now we have ready sentences and their corresponding labels."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['A1', '66056', '.', 'A2', '5.11'],\n",
       " ['2',\n",
       "  '.',\n",
       "  'Cm',\n",
       "  'R',\n",
       "  '=',\n",
       "  '2.5',\n",
       "  '.',\n",
       "  'Nice',\n",
       "  'calcium',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  '.',\n",
       "  'Caffeine',\n",
       "  'was',\n",
       "  'beautiful',\n",
       "  '.',\n",
       "  'Isoprenaline',\n",
       "  'was',\n",
       "  'also',\n",
       "  'done',\n",
       "  'but',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'important',\n",
       "  'to',\n",
       "  'note',\n",
       "  'that',\n",
       "  'the',\n",
       "  'isoprenaline',\n",
       "  'stock',\n",
       "  'was',\n",
       "  'made',\n",
       "  'from',\n",
       "  'the',\n",
       "  'old',\n",
       "  'stock',\n",
       "  'which',\n",
       "  'might',\n",
       "  'reflect',\n",
       "  'in',\n",
       "  'the',\n",
       "  'reduced',\n",
       "  'activation',\n",
       "  'of',\n",
       "  'beta',\n",
       "  'adrenergic',\n",
       "  'receptors',\n",
       "  '.',\n",
       "  'Leak',\n",
       "  'was',\n",
       "  'also',\n",
       "  'attempted',\n",
       "  'nice',\n",
       "  'effective',\n",
       "  'tetracaine',\n",
       "  'was',\n",
       "  'seen',\n",
       "  'but',\n",
       "  'it',\n",
       "  'seems',\n",
       "  'the',\n",
       "  'is',\n",
       "  'not',\n",
       "  'properly',\n",
       "  'working',\n",
       "  'as',\n",
       "  'expected',\n",
       "  'probably',\n",
       "  'due',\n",
       "  'to',\n",
       "  'a',\n",
       "  'low',\n",
       "  'perfusion',\n",
       "  'pressure',\n",
       "  '.',\n",
       "  'However',\n",
       "  'a',\n",
       "  'nice',\n",
       "  'leak',\n",
       "  'experiment',\n",
       "  'was',\n",
       "  'done',\n",
       "  '.'],\n",
       " ['temperature',\n",
       "  'for',\n",
       "  'amplification',\n",
       "  'was',\n",
       "  'achieved',\n",
       "  'in',\n",
       "  'D2',\n",
       "  '',\n",
       "  'E2',\n",
       "  '',\n",
       "  'F2',\n",
       "  '',\n",
       "  'G2',\n",
       "  '',\n",
       "  'and',\n",
       "  'H2',\n",
       "  '.'],\n",
       " ['µl', '2', 'N', 'NaOH', 'into', 'dilution', '.'],\n",
       " ['is',\n",
       "  'added',\n",
       "  'to',\n",
       "  'the',\n",
       "  'plate',\n",
       "  'and',\n",
       "  'put',\n",
       "  'on',\n",
       "  'a',\n",
       "  'plate',\n",
       "  'shaker',\n",
       "  'at',\n",
       "  '450',\n",
       "  'rpm',\n",
       "  '.'],\n",
       " ['gel',\n",
       "  'with',\n",
       "  'first',\n",
       "  'lane',\n",
       "  '10',\n",
       "  'µl',\n",
       "  'precision',\n",
       "  'plus',\n",
       "  'protein',\n",
       "  'unstained',\n",
       "  'standard',\n",
       "  'from',\n",
       "  'BioRad',\n",
       "  'catalog',\n",
       "  'number',\n",
       "  '161',\n",
       "  '-',\n",
       "  '036310',\n",
       "  '.'],\n",
       " ['µl', 'to', 'lane', '1', '.'],\n",
       " ['A1',\n",
       "  'to',\n",
       "  'A9',\n",
       "  'is',\n",
       "  'my',\n",
       "  'serial',\n",
       "  'dilution',\n",
       "  'of',\n",
       "  'standard',\n",
       "  'with',\n",
       "  'known',\n",
       "  'concentration',\n",
       "  '.',\n",
       "  'From',\n",
       "  'B1',\n",
       "  'to',\n",
       "  'B',\n",
       "  '9',\n",
       "  '.'],\n",
       " ['low', 'amount', 'of', 'RNA', 'in', 'sample', '44', 'and', '40', '.'],\n",
       " ['decided',\n",
       "  'to',\n",
       "  'put',\n",
       "  '2000000',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'cells',\n",
       "  'into',\n",
       "  '10µl',\n",
       "  'of',\n",
       "  'RPMI',\n",
       "  'media',\n",
       "  'and',\n",
       "  'they',\n",
       "  'will',\n",
       "  'be',\n",
       "  'cultured',\n",
       "  'for',\n",
       "  'the',\n",
       "  'next',\n",
       "  '2',\n",
       "  'to',\n",
       "  '3',\n",
       "  'days',\n",
       "  '.',\n",
       "  'I',\n",
       "  'will',\n",
       "  'monitor',\n",
       "  'the',\n",
       "  'growth',\n",
       "  'probably',\n",
       "  'on',\n",
       "  'Sunday',\n",
       "  'quickly',\n",
       "  'and',\n",
       "  'see',\n",
       "  'if',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'exchange',\n",
       "  'the',\n",
       "  'media',\n",
       "  '.',\n",
       "  'Jokerit',\n",
       "  'cells',\n",
       "  'are',\n",
       "  'highly',\n",
       "  'productive',\n",
       "  'cells',\n",
       "  'that',\n",
       "  'consume',\n",
       "  'a',\n",
       "  'lot',\n",
       "  'of',\n",
       "  'media',\n",
       "  'so',\n",
       "  'it',\n",
       "  'would',\n",
       "  'be',\n",
       "  'advisable',\n",
       "  'to',\n",
       "  'see',\n",
       "  'that',\n",
       "  'they',\n",
       "  'have',\n",
       "  'enough',\n",
       "  'food',\n",
       "  '.'],\n",
       " ['mix',\n",
       "  'lucifer',\n",
       "  'yellow',\n",
       "  'added',\n",
       "  'to',\n",
       "  '10',\n",
       "  '%',\n",
       "  'FBS',\n",
       "  'plus',\n",
       "  '90',\n",
       "  '%',\n",
       "  'PBS',\n",
       "  '.'],\n",
       " ['dehydrogenase',\n",
       "  '5',\n",
       "  'mg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  'was',\n",
       "  'diluted',\n",
       "  'in',\n",
       "  'PBS',\n",
       "  'to',\n",
       "  '3',\n",
       "  'mg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '.',\n",
       "  'Therefore',\n",
       "  '',\n",
       "  '60',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'Alcohol',\n",
       "  'dehydrogenase',\n",
       "  'was',\n",
       "  'added',\n",
       "  'to',\n",
       "  '40',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'PBS',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['decreased',\n",
       "  'to',\n",
       "  '65',\n",
       "  '%',\n",
       "  'but',\n",
       "  'Placed',\n",
       "  'the',\n",
       "  'speakers',\n",
       "  'in',\n",
       "  'a',\n",
       "  'different',\n",
       "  'place',\n",
       "  'so',\n",
       "  'they',\n",
       "  'face',\n",
       "  'the',\n",
       "  'mouse',\n",
       "  '.',\n",
       "  'Now',\n",
       "  'I',\n",
       "  'have',\n",
       "  '85',\n",
       "  'decibels',\n",
       "  '.'],\n",
       " ['for', '10', 'min', '.', 'Timer', 'for', '10', 'min', '.', 'Blueberry', '.'],\n",
       " ['started',\n",
       "  'with',\n",
       "  '12',\n",
       "  'samples',\n",
       "  '.',\n",
       "  'With',\n",
       "  '3',\n",
       "  'different',\n",
       "  'concentrations',\n",
       "  'for',\n",
       "  'the',\n",
       "  'dilution',\n",
       "  'profile',\n",
       "  'till',\n",
       "  'and',\n",
       "  'total',\n",
       "  'volume',\n",
       "  'is',\n",
       "  '10',\n",
       "  'µl',\n",
       "  'with',\n",
       "  '0.5',\n",
       "  '.',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'and',\n",
       "  '0.5',\n",
       "  '.',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'primers',\n",
       "  'is',\n",
       "  'done',\n",
       "  'for',\n",
       "  'the',\n",
       "  '4',\n",
       "  'primers',\n",
       "  'CTGF',\n",
       "  '',\n",
       "  'collagen',\n",
       "  '1',\n",
       "  'alpha',\n",
       "  '1',\n",
       "  '',\n",
       "  'periostin',\n",
       "  'and',\n",
       "  'PBGD',\n",
       "  '.'],\n",
       " ['out', 'what', \"'s\", 'new', 'in', 'Labtwin', '2.0', '.'],\n",
       " ['it',\n",
       "  'mini',\n",
       "  '-',\n",
       "  'protean',\n",
       "  'TGX',\n",
       "  'stainfree',\n",
       "  'gel',\n",
       "  '.',\n",
       "  '4',\n",
       "  'to',\n",
       "  '15',\n",
       "  '%',\n",
       "  '12',\n",
       "  'well',\n",
       "  'comb',\n",
       "  '',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'per',\n",
       "  'well',\n",
       "  'from',\n",
       "  'BioRad',\n",
       "  '.',\n",
       "  'Catalog',\n",
       "  'number',\n",
       "  '456',\n",
       "  '-',\n",
       "  '8085',\n",
       "  '.',\n",
       "  'And',\n",
       "  'use',\n",
       "  'precision',\n",
       "  'plus',\n",
       "  'protein',\n",
       "  'standard',\n",
       "  '.'],\n",
       " ['cell',\n",
       "  'viability',\n",
       "  'is',\n",
       "  '65.3',\n",
       "  '%',\n",
       "  'and',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  '4.4',\n",
       "  '*',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '5th',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'next',\n",
       "  'time',\n",
       "  'we',\n",
       "  'will',\n",
       "  'be',\n",
       "  'doing',\n",
       "  'this',\n",
       "  'experiment',\n",
       "  'is',\n",
       "  'very',\n",
       "  'important',\n",
       "  'for',\n",
       "  'me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'enough',\n",
       "  'note',\n",
       "  'before',\n",
       "  'hand',\n",
       "  'so',\n",
       "  'that',\n",
       "  'I',\n",
       "  'do',\n",
       "  'not',\n",
       "  'make',\n",
       "  'the',\n",
       "  'same',\n",
       "  'mistakes',\n",
       "  'again',\n",
       "  'and',\n",
       "  'again',\n",
       "  '.',\n",
       "  'I',\n",
       "  'just',\n",
       "  'told',\n",
       "  'Larry',\n",
       "  'so',\n",
       "  'that',\n",
       "  'he',\n",
       "  'would',\n",
       "  'remind',\n",
       "  'me',\n",
       "  'to',\n",
       "  'place',\n",
       "  'order',\n",
       "  'for',\n",
       "  '6',\n",
       "  'l',\n",
       "  'of',\n",
       "  'alcohol',\n",
       "  'from',\n",
       "  'Sigma',\n",
       "  'Corporation',\n",
       "  '.',\n",
       "  'Shaziya',\n",
       "  'is',\n",
       "  'going',\n",
       "  'to',\n",
       "  'go',\n",
       "  'through',\n",
       "  'all',\n",
       "  'my',\n",
       "  'slides',\n",
       "  'for',\n",
       "  'the',\n",
       "  'evaluation',\n",
       "  'of',\n",
       "  'the',\n",
       "  'data',\n",
       "  '.'],\n",
       " ['took', 'place', 'at', '10000'],\n",
       " ['and',\n",
       "  'washing',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'with',\n",
       "  'PBS',\n",
       "  '',\n",
       "  'centrifugation',\n",
       "  '300',\n",
       "  'G',\n",
       "  '10',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['me', 'to', 'pipet', 'a', 'plate', 'tomorrow', 'at', '1:00', 'PM', '.'],\n",
       " ['I',\n",
       "  \"'m\",\n",
       "  'preparing',\n",
       "  'the',\n",
       "  'needles',\n",
       "  'for',\n",
       "  'microinjection',\n",
       "  '.',\n",
       "  'Parameters',\n",
       "  'are',\n",
       "  'as',\n",
       "  'follows',\n",
       "  ':',\n",
       "  'heat',\n",
       "  '540',\n",
       "  '',\n",
       "  'pull',\n",
       "  '80',\n",
       "  '',\n",
       "  'velocity',\n",
       "  '30',\n",
       "  '',\n",
       "  'time',\n",
       "  '250',\n",
       "  '',\n",
       "  'pressure',\n",
       "  '250',\n",
       "  '',\n",
       "  'ram',\n",
       "  '525',\n",
       "  '.',\n",
       "  'Instrument',\n",
       "  'model',\n",
       "  'P',\n",
       "  '1000',\n",
       "  '.',\n",
       "  'Source',\n",
       "  'Sutter',\n",
       "  'instruments',\n",
       "  'Flaming',\n",
       "  '/',\n",
       "  'Brown',\n",
       "  'micropipette',\n",
       "  'puller',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'check',\n",
       "  'sample',\n",
       "  'number',\n",
       "  '4',\n",
       "  'on',\n",
       "  '1st',\n",
       "  'day',\n",
       "  'at',\n",
       "  '10:00',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['me', 'to', 'call', 'Sarah', 'at', '2', 'PM', 'tomorrow', '.'],\n",
       " ['µl', 'E', '1', 'to', 'tube', '3', '.'],\n",
       " ['culture',\n",
       "  '.',\n",
       "  'Saw',\n",
       "  'MB',\n",
       "  'cells',\n",
       "  'MMB',\n",
       "  '1843',\n",
       "  'and',\n",
       "  'MMB',\n",
       "  '1845',\n",
       "  '.'],\n",
       " ['1',\n",
       "  'aliquot',\n",
       "  'of',\n",
       "  'wash',\n",
       "  'membrane',\n",
       "  'preparations',\n",
       "  'from',\n",
       "  'the',\n",
       "  '3rd',\n",
       "  'of',\n",
       "  'August',\n",
       "  '2018',\n",
       "  '.',\n",
       "  'In',\n",
       "  'aliquots',\n",
       "  'are',\n",
       "  '200',\n",
       "  'µl',\n",
       "  'with',\n",
       "  'a',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '321',\n",
       "  'µM',\n",
       "  'made',\n",
       "  'by',\n",
       "  '.'],\n",
       " ['up',\n",
       "  'antibody',\n",
       "  'dilutions',\n",
       "  'for',\n",
       "  'flow',\n",
       "  'experiments',\n",
       "  'day',\n",
       "  '54',\n",
       "  'I',\n",
       "  'think',\n",
       "  '.',\n",
       "  'Dragon',\n",
       "  '.'],\n",
       " ['µl', 'diluted', 'DNA', 'into', 'denaturation', '.'],\n",
       " ['added', '3', 'µl', 'of', 'PBS', 'to', 'my', 'cells', '.'],\n",
       " ['LH',\n",
       "  '77',\n",
       "  '',\n",
       "  'both',\n",
       "  'mice',\n",
       "  'look',\n",
       "  'slightly',\n",
       "  'scruffy',\n",
       "  '',\n",
       "  'but',\n",
       "  'overall',\n",
       "  'good',\n",
       "  'and',\n",
       "  'active',\n",
       "  '.'],\n",
       " ['temperature', 'to', '25', '°', '.'],\n",
       " ['is', '4.0', 'ng/µl', '.'],\n",
       " ['I',\n",
       "  'leave',\n",
       "  'the',\n",
       "  'queue',\n",
       "  'of',\n",
       "  'bikers',\n",
       "  'along',\n",
       "  'my',\n",
       "  'path',\n",
       "  'upon',\n",
       "  'and',\n",
       "  'leave',\n",
       "  'the',\n",
       "  'zone',\n",
       "  'parking',\n",
       "  'once',\n",
       "  'behind',\n",
       "  'me',\n",
       "  '',\n",
       "  'I',\n",
       "  'realize',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'going',\n",
       "  'to',\n",
       "  'be',\n",
       "  'profound',\n",
       "  'disturbing',\n",
       "  'normal',\n",
       "  'to',\n",
       "  'return',\n",
       "  'to',\n",
       "  'the',\n",
       "  'city',\n",
       "  'in',\n",
       "  '3',\n",
       "  'weeks',\n",
       "  'time',\n",
       "  '.',\n",
       "  'As',\n",
       "  'I',\n",
       "  'will',\n",
       "  'hopefully',\n",
       "  'have',\n",
       "  'gotten',\n",
       "  'this',\n",
       "  'week',\n",
       "  'where',\n",
       "  '?'],\n",
       " ['μl',\n",
       "  '2',\n",
       "  'years',\n",
       "  'old',\n",
       "  'frighteners',\n",
       "  '.',\n",
       "  'Pipette',\n",
       "  '5',\n",
       "  'µl',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'primers'],\n",
       " ['7', '', '1.494', 'E', '5'],\n",
       " ['.', 'Purification', '.'],\n",
       " ['67',\n",
       "  'the',\n",
       "  'second',\n",
       "  'Nanodrop',\n",
       "  'measurement',\n",
       "  'looks',\n",
       "  'more',\n",
       "  'promising',\n",
       "  '',\n",
       "  'better',\n",
       "  'spectrum',\n",
       "  'as',\n",
       "  'a',\n",
       "  'absorbance',\n",
       "  'of',\n",
       "  '0.11',\n",
       "  '.'],\n",
       " ['length', 'is', '352', 'mm', '.'],\n",
       " ['streptavidin',\n",
       "  'Strep',\n",
       "  '-',\n",
       "  'Tactin',\n",
       "  'buffers',\n",
       "  'for',\n",
       "  'GSK',\n",
       "  'compound',\n",
       "  'AZ0161',\n",
       "  '.',\n",
       "  'Use',\n",
       "  'strep',\n",
       "  '-',\n",
       "  'A',\n",
       "  'buffer',\n",
       "  'BE002397',\n",
       "  'strep',\n",
       "  '60',\n",
       "  '.',\n",
       "  'Take',\n",
       "  '10',\n",
       "  'ml',\n",
       "  'and',\n",
       "  'add',\n",
       "  '7',\n",
       "  'mg',\n",
       "  'D',\n",
       "  'Biotin',\n",
       "  '.'],\n",
       " ['diluted', 'my', 'cells', 'into', '100', 'ml', '.'],\n",
       " ['is',\n",
       "  'experiment',\n",
       "  'FEK',\n",
       "  '109',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'taking',\n",
       "  'samples',\n",
       "  'at',\n",
       "  'time',\n",
       "  '.',\n",
       "  'Points',\n",
       "  '0',\n",
       "  '',\n",
       "  '1',\n",
       "  '',\n",
       "  '2',\n",
       "  '',\n",
       "  '5',\n",
       "  '',\n",
       "  '10',\n",
       "  '',\n",
       "  '15',\n",
       "  '',\n",
       "  '20',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['50',\n",
       "  'sorry',\n",
       "  '',\n",
       "  'analyzed',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'flowthrough',\n",
       "  'fraction',\n",
       "  'from',\n",
       "  'experiment',\n",
       "  '67',\n",
       "  'on',\n",
       "  'FSEC',\n",
       "  '.'],\n",
       " ['Biacore', '8K.', 'This', 'is', 'a', 'high', 'throughput', 'screen', '.'],\n",
       " ['cells', 'with', '3', 'µl', 'of', 'TCB', '.'],\n",
       " ['thawed', 'my', 'competent', 'cells', 'for', '20', 'seconds', '.'],\n",
       " ['11', '1.926', 'E5', '.'],\n",
       " ['%',\n",
       "  'leads',\n",
       "  'to',\n",
       "  '77',\n",
       "  'sound',\n",
       "  'pressure',\n",
       "  'level',\n",
       "  '.',\n",
       "  'And',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'already',\n",
       "  'pretty',\n",
       "  'low',\n",
       "  '.'],\n",
       " ['µg', 'pcr', 'successful', '.'],\n",
       " ['white',\n",
       "  'suspension',\n",
       "  'was',\n",
       "  'filtered',\n",
       "  'off',\n",
       "  'and',\n",
       "  'dried',\n",
       "  '.',\n",
       "  'Dried',\n",
       "  'in',\n",
       "  'the',\n",
       "  'vacuum',\n",
       "  'oven',\n",
       "  'at',\n",
       "  '40',\n",
       "  '°'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'check',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'confluence',\n",
       "  'on',\n",
       "  'Saturday',\n",
       "  '',\n",
       "  'August',\n",
       "  '10th',\n",
       "  '.'],\n",
       " ['for', '10', 'min', '.', '[', 'unsupported', 'language', ']'],\n",
       " ['me', 'to', 'order', '20', 'Falcon', 'tubes', '.'],\n",
       " ['loading',\n",
       "  'must',\n",
       "  'have',\n",
       "  'been',\n",
       "  'lower',\n",
       "  'than',\n",
       "  '5',\n",
       "  'mililiters',\n",
       "  'per',\n",
       "  'min',\n",
       "  'from',\n",
       "  'sample',\n",
       "  'pump',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'pump',\n",
       "  'must',\n",
       "  'have',\n",
       "  'been',\n",
       "  '.',\n",
       "  'There',\n",
       "  'must',\n",
       "  'be',\n",
       "  'something',\n",
       "  'wrong',\n",
       "  'with',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'pump',\n",
       "  '.',\n",
       "  'Increase',\n",
       "  'now',\n",
       "  'to',\n",
       "  '10',\n",
       "  'ml',\n",
       "  '/',\n",
       "  'min',\n",
       "  '.',\n",
       "  'Downloading',\n",
       "  'the',\n",
       "  'remainder',\n",
       "  '80',\n",
       "  'ml',\n",
       "  'on',\n",
       "  'the',\n",
       "  'column',\n",
       "  '.',\n",
       "  'Flowthrough',\n",
       "  'is',\n",
       "  'collected',\n",
       "  'in',\n",
       "  'waste',\n",
       "  '1',\n",
       "  'which',\n",
       "  'is',\n",
       "  'put',\n",
       "  'into',\n",
       "  'a',\n",
       "  'bottle',\n",
       "  '.'],\n",
       " ['µl', 'RSB', 'in', 'the', 'lanes', '1', 'and', '2', '.'],\n",
       " ['20', 'µl', 'of', 'the', 'master', 'mix', '.'],\n",
       " ['make',\n",
       "  'high',\n",
       "  'virus',\n",
       "  'I',\n",
       "  'need',\n",
       "  'also',\n",
       "  'fresh',\n",
       "  'cells',\n",
       "  'for',\n",
       "  'today',\n",
       "  '.',\n",
       "  'So',\n",
       "  'Nico',\n",
       "  'already',\n",
       "  'prepared',\n",
       "  '133',\n",
       "  'milliliers',\n",
       "  'of',\n",
       "  'one',\n",
       "  'million',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'milliliter',\n",
       "  'yesterday',\n",
       "  '.',\n",
       "  'So',\n",
       "  'we',\n",
       "  'also',\n",
       "  'have',\n",
       "  'checked',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'density',\n",
       "  'and',\n",
       "  'healthyness',\n",
       "  'in',\n",
       "  'seconds',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'checking',\n",
       "  'adequate',\n",
       "  '100',\n",
       "  'microliter',\n",
       "  'to',\n",
       "  'cell',\n",
       "  'density',\n",
       "  'and',\n",
       "  'viability',\n",
       "  '.'],\n",
       " ['am',\n",
       "  'running',\n",
       "  'a',\n",
       "  'Her2',\n",
       "  'assay',\n",
       "  '.',\n",
       "  'Her2YVMA',\n",
       "  'assay',\n",
       "  'for',\n",
       "  'kinetic',\n",
       "  'characterization',\n",
       "  'of',\n",
       "  '.',\n",
       "  'Repeat',\n",
       "  '',\n",
       "  'I',\n",
       "  'am',\n",
       "  'repeating',\n",
       "  '.',\n",
       "  'I',\n",
       "  'am',\n",
       "  'running',\n",
       "  'Her2',\n",
       "  'assay',\n",
       "  'for',\n",
       "  'the',\n",
       "  'kinetic',\n",
       "  'characterization',\n",
       "  'of',\n",
       "  'compounds',\n",
       "  'to',\n",
       "  'determine',\n",
       "  'k',\n",
       "  '-',\n",
       "  'elect',\n",
       "  'by',\n",
       "  'ki',\n",
       "  '-',\n",
       "  'values',\n",
       "  '.'],\n",
       " ['50', 'to', 'column', 'A', 'position', '1', '.'],\n",
       " ['again',\n",
       "  '5',\n",
       "  'min',\n",
       "  'now',\n",
       "  'at',\n",
       "  '10000',\n",
       "  'rpm',\n",
       "  '.',\n",
       "  'After',\n",
       "  'adding',\n",
       "  'more',\n",
       "  'of',\n",
       "  'C6',\n",
       "  'to',\n",
       "  'the',\n",
       "  'solution',\n",
       "  '.',\n",
       "  'Start',\n",
       "  'another',\n",
       "  'size',\n",
       "  'exclusion',\n",
       "  'with',\n",
       "  'about',\n",
       "  '0\\\\.3',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'AZ',\n",
       "  '2250',\n",
       "  '.'],\n",
       " ['with',\n",
       "  'a',\n",
       "  'relatively',\n",
       "  'high',\n",
       "  'aspirate',\n",
       "  'offset',\n",
       "  'and',\n",
       "  'then',\n",
       "  'will',\n",
       "  'reduce',\n",
       "  'gradually',\n",
       "  '.',\n",
       "  'Starting',\n",
       "  'from',\n",
       "  '55',\n",
       "  'original',\n",
       "  'values',\n",
       "  '.',\n",
       "  'Checking',\n",
       "  'for',\n",
       "  'plate',\n",
       "  'patterns',\n",
       "  'in',\n",
       "  'terms',\n",
       "  'of',\n",
       "  'cell',\n",
       "  'loss',\n",
       "  'and',\n",
       "  'then',\n",
       "  'will',\n",
       "  'reduce',\n",
       "  'gradually',\n",
       "  '.'],\n",
       " ['also',\n",
       "  'inoculate',\n",
       "  'AX',\n",
       "  '50',\n",
       "  'and',\n",
       "  'pLenti',\n",
       "  'EphA2',\n",
       "  'sgmut',\n",
       "  'delta',\n",
       "  'plus',\n",
       "  'ATG',\n",
       "  '.'],\n",
       " ['is', '66056', '.', 'A2', 'is', '15'],\n",
       " ['cells',\n",
       "  'are',\n",
       "  'assay',\n",
       "  'ready',\n",
       "  'so',\n",
       "  'you',\n",
       "  'need',\n",
       "  'to',\n",
       "  'culture',\n",
       "  'them',\n",
       "  'in',\n",
       "  'incubators',\n",
       "  'with',\n",
       "  '37',\n",
       "  '°',\n",
       "  'We',\n",
       "  'have',\n",
       "  '4',\n",
       "  'incubators',\n",
       "  'to',\n",
       "  'choose',\n",
       "  'from',\n",
       "  '.',\n",
       "  'I',\n",
       "  'do',\n",
       "  \"n't\",\n",
       "  'write',\n",
       "  'in',\n",
       "  'a',\n",
       "  'notebook',\n",
       "  '.',\n",
       "  'I',\n",
       "  'tend',\n",
       "  'to',\n",
       "  'use',\n",
       "  'in',\n",
       "  'network',\n",
       "  'PC',\n",
       "  'if',\n",
       "  'I',\n",
       "  'can',\n",
       "  '.'],\n",
       " ['15', '1.388', 'E5', '.'],\n",
       " [\"'s\",\n",
       "  'test',\n",
       "  'it',\n",
       "  'with',\n",
       "  'a',\n",
       "  'replacement',\n",
       "  'of',\n",
       "  'CT2',\n",
       "  'to',\n",
       "  'CT02',\n",
       "  'and',\n",
       "  'check',\n",
       "  'if',\n",
       "  'it',\n",
       "  'works',\n",
       "  '',\n",
       "  'or',\n",
       "  'not',\n",
       "  '.'],\n",
       " ['spheroids', 'left', 'of', 'the', 'assay', '.', '38', 'spheroids'],\n",
       " ['a',\n",
       "  'miniprotean',\n",
       "  'TXTGX',\n",
       "  'stainfree',\n",
       "  'gel',\n",
       "  'from',\n",
       "  'bio',\n",
       "  'rad',\n",
       "  '.',\n",
       "  'Batch',\n",
       "  '64267254',\n",
       "  '.'],\n",
       " ['3', 'µl', 'of', 'magnesium', 'chloride', '.'],\n",
       " ['temperature', 'in', 'cell', 'HepG2', 'to', '28', '°', 'Fahrenheit', '.'],\n",
       " ['colon', '-', 'no', 'abnormalities', 'detected'],\n",
       " ['clear',\n",
       "  'CDT',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'put',\n",
       "  'in',\n",
       "  'a',\n",
       "  'syringe',\n",
       "  'an',\n",
       "  'added',\n",
       "  'in',\n",
       "  'one',\n",
       "  'shot',\n",
       "  'to',\n",
       "  'the',\n",
       "  '0',\n",
       "  '°',\n",
       "  '-',\n",
       "  'solution',\n",
       "  '.'],\n",
       " ['with',\n",
       "  'Olga',\n",
       "  'with',\n",
       "  '2',\n",
       "  'different',\n",
       "  'compounds',\n",
       "  'is',\n",
       "  'possible',\n",
       "  '.',\n",
       "  'You',\n",
       "  'need',\n",
       "  'to',\n",
       "  'add',\n",
       "  'both',\n",
       "  'compounds',\n",
       "  'on',\n",
       "  'one',\n",
       "  'plate',\n",
       "  'and',\n",
       "  'then',\n",
       "  'add',\n",
       "  'the',\n",
       "  'right',\n",
       "  'compound',\n",
       "  'column',\n",
       "  'to',\n",
       "  'the',\n",
       "  'right',\n",
       "  'assay',\n",
       "  'plate',\n",
       "  'column'],\n",
       " ['me', 'to', 'warm', 'cells', 'at', '5:00', 'PM', '.'],\n",
       " ['g',\n",
       "  'Lavender',\n",
       "  'oil',\n",
       "  '.',\n",
       "  '2.19',\n",
       "  'g',\n",
       "  'Pine',\n",
       "  'oil',\n",
       "  '.',\n",
       "  '162.01',\n",
       "  'g',\n",
       "  'alcohol',\n",
       "  '.',\n",
       "  '70',\n",
       "  '.'],\n",
       " ['need', '3', 'cases', 'of'],\n",
       " ['re', 'activate', 'protocol', 'step', '5', '.'],\n",
       " ['a',\n",
       "  'look',\n",
       "  'at',\n",
       "  'the',\n",
       "  'reaction',\n",
       "  '1',\n",
       "  'in',\n",
       "  'fume',\n",
       "  'hood',\n",
       "  'left',\n",
       "  '.'],\n",
       " ['weight', 'is', '22', 'ng', '.'],\n",
       " ['assay',\n",
       "  'BTRL12',\n",
       "  'optimal',\n",
       "  'temperature',\n",
       "  'was',\n",
       "  'achieved',\n",
       "  'in',\n",
       "  'all',\n",
       "  'wells',\n",
       "  '.'],\n",
       " ['',\n",
       "  'so',\n",
       "  'FACS',\n",
       "  'is',\n",
       "  'done',\n",
       "  '',\n",
       "  'we',\n",
       "  'have',\n",
       "  'very',\n",
       "  'low',\n",
       "  'numbers',\n",
       "  'so',\n",
       "  'we',\n",
       "  'will',\n",
       "  'have',\n",
       "  'to',\n",
       "  'do',\n",
       "  'only',\n",
       "  'in',\n",
       "  '40',\n",
       "  'μl',\n",
       "  '.',\n",
       "  'Count',\n",
       "  'once',\n",
       "  'and',\n",
       "  'load',\n",
       "  'up',\n",
       "  '.',\n",
       "  'Hopefully',\n",
       "  'we',\n",
       "  'will',\n",
       "  'have',\n",
       "  'enough',\n",
       "  'for',\n",
       "  '10000',\n",
       "  '.'],\n",
       " ['hydroxide', 'dilution', '180', 'µl', 'PW1', '.'],\n",
       " ['B3',\n",
       "  '',\n",
       "  'B1',\n",
       "  '',\n",
       "  'C2',\n",
       "  '',\n",
       "  'C4',\n",
       "  '',\n",
       "  'C6',\n",
       "  '',\n",
       "  'C8',\n",
       "  '',\n",
       "  'D3',\n",
       "  'and',\n",
       "  'E4',\n",
       "  '.'],\n",
       " ['seconds', 'injection', '5', 'µl', '/', 'min', '.'],\n",
       " ['32', '', '1.130', 'E', '5'],\n",
       " ['39', '6', '.', 'Dam', 'it', '.', '3.650', 'E5', '.'],\n",
       " ['',\n",
       "  'I',\n",
       "  \"'ve\",\n",
       "  'been',\n",
       "  'using',\n",
       "  'it',\n",
       "  '.',\n",
       "  '.',\n",
       "  'Set',\n",
       "  'the',\n",
       "  'timer',\n",
       "  'for',\n",
       "  '2',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['set',\n",
       "  'up',\n",
       "  'the',\n",
       "  'baculovirus',\n",
       "  'in',\n",
       "  'the',\n",
       "  'shaker',\n",
       "  'flask',\n",
       "  'for',\n",
       "  'Experiment',\n",
       "  'A-12',\n",
       "  '.'],\n",
       " ['round', '5D5', 'full', 'length', '26', '.'],\n",
       " ['put', 'a', 'timer', 'for', '10', 'min', '.'],\n",
       " ['100', 'ml', 'of', 'acid', 'into', 'a', 'bowl', '.'],\n",
       " ['cycles',\n",
       "  'PV',\n",
       "  '1210',\n",
       "  'first',\n",
       "  'part',\n",
       "  '.',\n",
       "  '200',\n",
       "  'h',\n",
       "  'constant',\n",
       "  'humidity',\n",
       "  'CH',\n",
       "  'first',\n",
       "  'part',\n",
       "  '.',\n",
       "  'Stone',\n",
       "  'chip',\n",
       "  'resistance',\n",
       "  'panels',\n",
       "  'to',\n",
       "  'be',\n",
       "  'sent',\n",
       "  '.',\n",
       "  'Chemicals',\n",
       "  'resistance',\n",
       "  'panels',\n",
       "  'to',\n",
       "  'be',\n",
       "  'sent',\n",
       "  '.'],\n",
       " ['concentration', 'in', 'sample', 'P1', 'is', '5', 'ng/µl', '.'],\n",
       " ['sample', 'was', 'repeated', '3', 'times', '.'],\n",
       " ['migs', 'of', 'lucifer', 'yellow', '.'],\n",
       " ['plasmid',\n",
       "  'number',\n",
       "  '5',\n",
       "  '.',\n",
       "  'Concentration',\n",
       "  '220.7',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Fraction',\n",
       "  'number',\n",
       "  '1',\n",
       "  '1.86',\n",
       "  '',\n",
       "  'fraction',\n",
       "  '2',\n",
       "  '2.35',\n",
       "  '.',\n",
       "  'Stop',\n",
       "  '.'],\n",
       " ['cell',\n",
       "  'viability',\n",
       "  'is',\n",
       "  '65.3',\n",
       "  '%',\n",
       "  'and',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  '4.4',\n",
       "  '*',\n",
       "  '10',\n",
       "  '^',\n",
       "  '5',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['I',\n",
       "  \"'m\",\n",
       "  'preparing',\n",
       "  'samples',\n",
       "  'for',\n",
       "  'our',\n",
       "  'needs',\n",
       "  'it',\n",
       "  '.',\n",
       "  'And',\n",
       "  'first',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'prepare',\n",
       "  'the',\n",
       "  'dilutions',\n",
       "  'for',\n",
       "  'the',\n",
       "  'RNA',\n",
       "  '.',\n",
       "  'So',\n",
       "  'the',\n",
       "  'RNA',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'stand',\n",
       "  'too',\n",
       "  'long',\n",
       "  'on',\n",
       "  'ice',\n",
       "  'to',\n",
       "  'prevent',\n",
       "  'degradation',\n",
       "  '.',\n",
       "  'And',\n",
       "  'um',\n",
       "  '.',\n",
       "  'Then',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  '.',\n",
       "  'So',\n",
       "  'first',\n",
       "  'prepare',\n",
       "  'the',\n",
       "  'water',\n",
       "  '',\n",
       "  'then',\n",
       "  'prepare',\n",
       "  'the',\n",
       "  'master',\n",
       "  'mixes',\n",
       "  'so',\n",
       "  'with',\n",
       "  'oligo',\n",
       "  'DTs',\n",
       "  '',\n",
       "  'Dntps',\n",
       "  'and',\n",
       "  'RNase',\n",
       "  'Inhibitors',\n",
       "  'and',\n",
       "  'the',\n",
       "  'second',\n",
       "  'mastermix',\n",
       "  'that',\n",
       "  'contains',\n",
       "  'RT',\n",
       "  'reaction',\n",
       "  '.',\n",
       "  'Without',\n",
       "  'the',\n",
       "  'reverse',\n",
       "  'transcriptase',\n",
       "  'that',\n",
       "  'will',\n",
       "  'come',\n",
       "  'the',\n",
       "  'last',\n",
       "  '.',\n",
       "  'Once',\n",
       "  'I',\n",
       "  'have',\n",
       "  'prepared',\n",
       "  'that',\n",
       "  '',\n",
       "  'then',\n",
       "  'I',\n",
       "  'pick',\n",
       "  'up',\n",
       "  '30',\n",
       "  'samples',\n",
       "  'make',\n",
       "  'dilutions',\n",
       "  '.',\n",
       "  'And',\n",
       "  'start',\n",
       "  'typing',\n",
       "  'basically',\n",
       "  'to',\n",
       "  'do',\n",
       "  'the',\n",
       "  'binding',\n",
       "  'with',\n",
       "  'oligos',\n",
       "  '.',\n",
       "  'Factors',\n",
       "  'that',\n",
       "  'Add',\n",
       "  'RT',\n",
       "  'reaction',\n",
       "  'and',\n",
       "  'run',\n",
       "  'the',\n",
       "  'pcr',\n",
       "  '.'],\n",
       " ['solution', '1', 'in', 'fridge', 'RNase', 'added', '.'],\n",
       " ['-',\n",
       "  'PROTEAN',\n",
       "  'TGX',\n",
       "  'stain',\n",
       "  'free',\n",
       "  'pre',\n",
       "  'cast',\n",
       "  'gel',\n",
       "  '4',\n",
       "  '-',\n",
       "  '15',\n",
       "  '%',\n",
       "  '',\n",
       "  'twelve',\n",
       "  'well',\n",
       "  '',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'per',\n",
       "  'well',\n",
       "  'BioRAD',\n",
       "  'and',\n",
       "  '5',\n",
       "  'µl',\n",
       "  'precision',\n",
       "  'plus',\n",
       "  'protein',\n",
       "  'unstained',\n",
       "  'standard',\n",
       "  'BioRAD',\n",
       "  'Catalog',\n",
       "  '161',\n",
       "  '-',\n",
       "  '0363',\n",
       "  '.'],\n",
       " ['1',\n",
       "  'chip',\n",
       "  'fluid',\n",
       "  'flow',\n",
       "  'going',\n",
       "  'from',\n",
       "  'bottom',\n",
       "  'to',\n",
       "  'top',\n",
       "  'of',\n",
       "  'image',\n",
       "  '.'],\n",
       " ['immediate',\n",
       "  'team',\n",
       "  'is',\n",
       "  'about',\n",
       "  '15',\n",
       "  'to',\n",
       "  '17',\n",
       "  'people',\n",
       "  '.',\n",
       "  'We',\n",
       "  'are',\n",
       "  'separated',\n",
       "  'into',\n",
       "  'therapeutic',\n",
       "  'areas',\n",
       "  'within',\n",
       "  'CPSS',\n",
       "  '.',\n",
       "  'This',\n",
       "  'means',\n",
       "  'there',\n",
       "  'it',\n",
       "  'could',\n",
       "  'be',\n",
       "  'more',\n",
       "  'than',\n",
       "  '100',\n",
       "  'people',\n",
       "  'in',\n",
       "  'CPSS',\n",
       "  'alone',\n",
       "  '.'],\n",
       " ['bio',\n",
       "  'analyzer',\n",
       "  '.',\n",
       "  '490',\n",
       "  'concentration',\n",
       "  '1',\n",
       "  '.',\n",
       "  '491',\n",
       "  'oncentration',\n",
       "  '2',\n",
       "  '.',\n",
       "  '492',\n",
       "  'concentration',\n",
       "  '0.4',\n",
       "  '.'],\n",
       " ['50', 'position', 'G3', '3', 'phase', 'separation', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'turn',\n",
       "  'on',\n",
       "  'the',\n",
       "  'LSR2',\n",
       "  'at',\n",
       "  '2:00',\n",
       "  'PM',\n",
       "  '.',\n",
       "  'Turn',\n",
       "  'on',\n",
       "  'the',\n",
       "  'LSR2',\n",
       "  '.'],\n",
       " ['my', 'reaction', 'at', '45', '°', '.'],\n",
       " ['I',\n",
       "  'did',\n",
       "  'another',\n",
       "  'experiment',\n",
       "  'at',\n",
       "  'using',\n",
       "  'the',\n",
       "  'same',\n",
       "  'conditions',\n",
       "  'and',\n",
       "  'that',\n",
       "  'looks',\n",
       "  'a',\n",
       "  'lot',\n",
       "  'better',\n",
       "  '.',\n",
       "  'Y02',\n",
       "  'and',\n",
       "  'N02',\n",
       "  'hold',\n",
       "  'up',\n",
       "  'really',\n",
       "  'well',\n",
       "  'for',\n",
       "  'the',\n",
       "  '10',\n",
       "  'min',\n",
       "  'runtime',\n",
       "  'I',\n",
       "  'had',\n",
       "  '.',\n",
       "  'So',\n",
       "  'for',\n",
       "  'next',\n",
       "  'experiment',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'trying',\n",
       "  'out',\n",
       "  'for',\n",
       "  'how',\n",
       "  'long',\n",
       "  'we',\n",
       "  'retain',\n",
       "  'any',\n",
       "  'activity',\n",
       "  '.'],\n",
       " ['110', 'µl', 'of', 'EPX', '3', 'to', 'X', 'AMP', '.'],\n",
       " ['it',\n",
       "  \"'s\",\n",
       "  'important',\n",
       "  'to',\n",
       "  'also',\n",
       "  'note',\n",
       "  'in',\n",
       "  'experiment',\n",
       "  '4',\n",
       "  'shortening',\n",
       "  'is',\n",
       "  'also',\n",
       "  'included',\n",
       "  '.'],\n",
       " ['methods',\n",
       "  '181219',\n",
       "  'sepharose',\n",
       "  '6',\n",
       "  'position',\n",
       "  '6',\n",
       "  '.',\n",
       "  'According',\n",
       "  'to',\n",
       "  'standard',\n",
       "  'with',\n",
       "  '8.8',\n",
       "  'mililiter',\n",
       "  'per',\n",
       "  'min',\n",
       "  'flow',\n",
       "  'rate',\n",
       "  '.'],\n",
       " ['21', ':', 'Made', 'new', 'buffer', '.'],\n",
       " ['seeded',\n",
       "  'back',\n",
       "  '500000',\n",
       "  'cells',\n",
       "  'for',\n",
       "  '983',\n",
       "  'and',\n",
       "  '987',\n",
       "  '',\n",
       "  'but',\n",
       "  'I',\n",
       "  'seeded',\n",
       "  '1000000',\n",
       "  'cells',\n",
       "  'for',\n",
       "  '988',\n",
       "  '.',\n",
       "  'I',\n",
       "  'hope',\n",
       "  'I',\n",
       "  'can',\n",
       "  'keep',\n",
       "  'the',\n",
       "  'same',\n",
       "  'number',\n",
       "  'of',\n",
       "  'cells',\n",
       "  'for',\n",
       "  'the',\n",
       "  'next',\n",
       "  'time',\n",
       "  '.'],\n",
       " ['1', 'is', 'my', 'donor', '.', 'A'],\n",
       " ['homogenised',\n",
       "  'sample',\n",
       "  'number',\n",
       "  '2',\n",
       "  'is',\n",
       "  'very',\n",
       "  'yellow',\n",
       "  'compared',\n",
       "  'to',\n",
       "  'the',\n",
       "  'others',\n",
       "  'that',\n",
       "  'are',\n",
       "  'dark',\n",
       "  'red',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'tomorrow',\n",
       "  'to',\n",
       "  'take',\n",
       "  'out',\n",
       "  'experiment',\n",
       "  '1',\n",
       "  'from',\n",
       "  'fume',\n",
       "  'hood',\n",
       "  '2',\n",
       "  '.'],\n",
       " ['some',\n",
       "  'reason',\n",
       "  '',\n",
       "  'it',\n",
       "  'looks',\n",
       "  'like',\n",
       "  'the',\n",
       "  'HL',\n",
       "  'ADR',\n",
       "  'is',\n",
       "  'showing',\n",
       "  'up',\n",
       "  'very',\n",
       "  'oddly',\n",
       "  'in',\n",
       "  'the',\n",
       "  'CD',\n",
       "  'ADP',\n",
       "  'S',\n",
       "  '7',\n",
       "  'channel',\n",
       "  'and',\n",
       "  'I',\n",
       "  'do',\n",
       "  \"n't\",\n",
       "  'think',\n",
       "  'there',\n",
       "  \"'s\",\n",
       "  'anything',\n",
       "  'I',\n",
       "  'can',\n",
       "  'do',\n",
       "  'about',\n",
       "  'it',\n",
       "  '.'],\n",
       " ['a', '130', 'µl', 'into', 'lane', '1', '.'],\n",
       " ['finished', 'at', '11:35', '.'],\n",
       " ['improved', 'background', 'on', 'pancreas', 'in', 'protocol', '695', '.'],\n",
       " ['ID', '30', 'also', 'added', 'to', 'C9', '.'],\n",
       " ['3', 'into', 'cell', 'A2', '.'],\n",
       " ['14', '2.0.6', 'E5', '.'],\n",
       " ['at',\n",
       "  '96',\n",
       "  'well',\n",
       "  'plate',\n",
       "  '.',\n",
       "  'Third',\n",
       "  '3-',\n",
       "  'GFP',\n",
       "  'UOC',\n",
       "  'meg',\n",
       "  'N',\n",
       "  'going',\n",
       "  'from',\n",
       "  'the',\n",
       "  'plate',\n",
       "  'left',\n",
       "  'to',\n",
       "  'right',\n",
       "  '.',\n",
       "  'Starting',\n",
       "  'at',\n",
       "  'A1',\n",
       "  '',\n",
       "  'A2',\n",
       "  '.'],\n",
       " ['3849',\n",
       "  '-',\n",
       "  '3',\n",
       "  '',\n",
       "  'human',\n",
       "  'CD',\n",
       "  '45',\n",
       "  'at',\n",
       "  '1.38',\n",
       "  '',\n",
       "  'CD',\n",
       "  '19',\n",
       "  'at',\n",
       "  '0.76',\n",
       "  '',\n",
       "  'CD',\n",
       "  '10',\n",
       "  'at',\n",
       "  '1.07',\n",
       "  '',\n",
       "  'CD',\n",
       "  '34',\n",
       "  'at',\n",
       "  '0.65',\n",
       "  'and',\n",
       "  'CD',\n",
       "  '22',\n",
       "  'at',\n",
       "  '0.23',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'out',\n",
       "  'of',\n",
       "  'the',\n",
       "  'incubator',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '11:00',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['502', 'column', 'A', 'position', '1', '.'],\n",
       " ['cell', 'A1', 'put', 'the', 'number', '1', '.'],\n",
       " ['100', 'μl', 'betaine'],\n",
       " ['value', 'is', '1524', 'parts', 'per', 'billion', '.'],\n",
       " ['tube', '4', 'on', 'the', 'rotation', 'for', '20', 'min', '.'],\n",
       " ['membrane',\n",
       "  'pellet',\n",
       "  'is',\n",
       "  'resuspended',\n",
       "  'in',\n",
       "  '1',\n",
       "  '%',\n",
       "  '',\n",
       "  'DDM',\n",
       "  'Lysis',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Incubated',\n",
       "  'for',\n",
       "  'at',\n",
       "  'least',\n",
       "  '5',\n",
       "  'min',\n",
       "  '.',\n",
       "  'After',\n",
       "  'centrifugation',\n",
       "  'of',\n",
       "  '10',\n",
       "  'min',\n",
       "  '.',\n",
       "  'Every',\n",
       "  'thing',\n",
       "  'is',\n",
       "  'recovered',\n",
       "  'in',\n",
       "  'the',\n",
       "  'soluble',\n",
       "  'fraction',\n",
       "  '.'],\n",
       " ['me', 'to', 'talk', 'to', 'Mimi', 'tomorrow', 'at', '9:00', '.'],\n",
       " ['me', 'to', 'order', '10', 'cm', 'dishes', '.'],\n",
       " ['me', 'to', 'pipet', 'the', 'plate', 'at', '1:00', 'PM', 'tomorrow', '.'],\n",
       " ['pipetted',\n",
       "  '5',\n",
       "  'µl',\n",
       "  'into',\n",
       "  'a',\n",
       "  'Falcon',\n",
       "  'tube',\n",
       "  '.',\n",
       "  'I',\n",
       "  'put',\n",
       "  'the',\n",
       "  'Falcon',\n",
       "  'tubes',\n",
       "  'into',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'for',\n",
       "  '10',\n",
       "  'min',\n",
       "  '.',\n",
       "  'All',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'were',\n",
       "  'immersed',\n",
       "  'into',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'except',\n",
       "  'the',\n",
       "  'last',\n",
       "  'one',\n",
       "  'as',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'was',\n",
       "  'too',\n",
       "  'small',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'out',\n",
       "  'the',\n",
       "  'extraction',\n",
       "  'samples',\n",
       "  '.',\n",
       "  'In',\n",
       "  '2',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['cell', 'culture', 'medium', 'DMEM', '10', 'bottles', '.'],\n",
       " ['probe', 'in', 'new', 'spot', '.', 'Temperature', '36.9', '.'],\n",
       " ['lab',\n",
       "  'chart',\n",
       "  ':',\n",
       "  'Comments',\n",
       "  '2',\n",
       "  'fingers',\n",
       "  'remember',\n",
       "  '.',\n",
       "  'Lab',\n",
       "  'chart',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'tomorrow',\n",
       "  'to',\n",
       "  'take',\n",
       "  'my',\n",
       "  'samples',\n",
       "  'out',\n",
       "  'of',\n",
       "  'the',\n",
       "  'fridge',\n",
       "  'at',\n",
       "  '7:35',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['20',\n",
       "  '%',\n",
       "  'ethanol',\n",
       "  'solution',\n",
       "  'for',\n",
       "  'cleaning',\n",
       "  'MX',\n",
       "  'system',\n",
       "  '.',\n",
       "  'Using',\n",
       "  '1.42l70',\n",
       "  '%',\n",
       "  'ethanol',\n",
       "  '.',\n",
       "  'Filling',\n",
       "  'up',\n",
       "  'to',\n",
       "  '5',\n",
       "  'l',\n",
       "  'with',\n",
       "  'milliQ',\n",
       "  'water',\n",
       "  '.'],\n",
       " ['',\n",
       "  'I',\n",
       "  'needed',\n",
       "  'thawed',\n",
       "  'out',\n",
       "  'Miller',\n",
       "  \"'s\",\n",
       "  'cryo',\n",
       "  'vial',\n",
       "  'of',\n",
       "  'el2B',\n",
       "  'glycerol',\n",
       "  'stock',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'standard',\n",
       "  'curve',\n",
       "  'so',\n",
       "  'row',\n",
       "  '4',\n",
       "  'and',\n",
       "  'row',\n",
       "  '6',\n",
       "  'are',\n",
       "  'exchanged',\n",
       "  '.'],\n",
       " ['my',\n",
       "  'experiment',\n",
       "  'DNA',\n",
       "  'Research',\n",
       "  'G2',\n",
       "  '237',\n",
       "  '.',\n",
       "  'Sorry',\n",
       "  'I',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'understand',\n",
       "  '.'],\n",
       " ['are',\n",
       "  'sacking',\n",
       "  'a',\n",
       "  'lot',\n",
       "  'of',\n",
       "  'mice',\n",
       "  'today',\n",
       "  '',\n",
       "  'including',\n",
       "  'if',\n",
       "  'I',\n",
       "  'remember',\n",
       "  'correctly',\n",
       "  '828',\n",
       "  '',\n",
       "  '859',\n",
       "  'no',\n",
       "  '849',\n",
       "  '2',\n",
       "  'and',\n",
       "  'one',\n",
       "  '839',\n",
       "  '',\n",
       "  '840',\n",
       "  '',\n",
       "  '845',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'with',\n",
       "  'headphones',\n",
       "  'holding',\n",
       "  'microphone',\n",
       "  'close',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'steered',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['have', 'to', 'order', 'new', 'filter', 'systems', 'oh', '0.45', 'µm', '.'],\n",
       " ['in', 'both', 'lanes', '.', '5', 'µl', '.'],\n",
       " ['#', '3', 'is', '18'],\n",
       " ['4',\n",
       "  '1.141',\n",
       "  'E5',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'was',\n",
       "  'repeated',\n",
       "  'once',\n",
       "  '.',\n",
       "  'Dragonfly',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'number',\n",
       "  '0',\n",
       "  '',\n",
       "  'no',\n",
       "  'noise',\n",
       "  'in',\n",
       "  'the',\n",
       "  'office',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['for', '30', 'min', '.'],\n",
       " ['a',\n",
       "  'method',\n",
       "  '.',\n",
       "  'The',\n",
       "  'method',\n",
       "  'is',\n",
       "  'called',\n",
       "  'tube',\n",
       "  'preparation',\n",
       "  '6',\n",
       "  'samples',\n",
       "  '.'],\n",
       " ['at', '37', '°', 'overnight', '.'],\n",
       " ['-', '17', '-', 'light', '86', '.', 'Eyes', '.', '2', 'sections', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'put',\n",
       "  'the',\n",
       "  'CHO',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'the',\n",
       "  '37',\n",
       "  '°',\n",
       "  'incubator',\n",
       "  'at',\n",
       "  '9:30',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['seems',\n",
       "  'to',\n",
       "  'be',\n",
       "  'having',\n",
       "  'a',\n",
       "  'hard',\n",
       "  'time',\n",
       "  'differentiating',\n",
       "  'between',\n",
       "  '15',\n",
       "  'and',\n",
       "  '50',\n",
       "  '.'],\n",
       " ['µl', 'EFP12', 'added', 'to', 'denaturation', '.'],\n",
       " ['storage',\n",
       "  'room',\n",
       "  'is',\n",
       "  'open',\n",
       "  'between',\n",
       "  '8:30',\n",
       "  'AM',\n",
       "  'to',\n",
       "  '9:30',\n",
       "  'AM',\n",
       "  'and',\n",
       "  'also',\n",
       "  'between',\n",
       "  '1:00',\n",
       "  'PM',\n",
       "  'to',\n",
       "  '2:00',\n",
       "  'PM',\n",
       "  '.'],\n",
       " ['SDS',\n",
       "  'samples',\n",
       "  'pipetting',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'with',\n",
       "  '7.7',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'nupage',\n",
       "  'LDS',\n",
       "  'sample',\n",
       "  'buffer',\n",
       "  '4',\n",
       "  'times',\n",
       "  'and',\n",
       "  '3.1',\n",
       "  'µl',\n",
       "  'new',\n",
       "  'paste',\n",
       "  'sample',\n",
       "  'reducing',\n",
       "  'agents',\n",
       "  '.',\n",
       "  'Samples',\n",
       "  'are',\n",
       "  'pipetted',\n",
       "  'in',\n",
       "  'order',\n",
       "  '13',\n",
       "  '',\n",
       "  '15',\n",
       "  '',\n",
       "  '19',\n",
       "  '',\n",
       "  '20',\n",
       "  '',\n",
       "  '21',\n",
       "  '',\n",
       "  'and',\n",
       "  'GFP',\n",
       "  '.',\n",
       "  'Respective',\n",
       "  '.'],\n",
       " ['select',\n",
       "  'experiment',\n",
       "  'DNA',\n",
       "  'research',\n",
       "  'D',\n",
       "  '237',\n",
       "  '.',\n",
       "  'DNA',\n",
       "  'research',\n",
       "  '.'],\n",
       " ['robot',\n",
       "  'test',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'water',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '2',\n",
       "  'methanol',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '3',\n",
       "  'Ethanol',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '5',\n",
       "  'DMSO',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '7',\n",
       "  'acetone',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '8',\n",
       "  'ethyl',\n",
       "  'acetate',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '9',\n",
       "  '.',\n",
       "  'Water',\n",
       "  'at',\n",
       "  'pH',\n",
       "  '10',\n",
       "  '.',\n",
       "  'Stock',\n",
       "  'solution',\n",
       "  'number',\n",
       "  '10',\n",
       "  'water',\n",
       "  'at',\n",
       "  'pH',\n",
       "  '2',\n",
       "  '.'],\n",
       " ['a', 'timer', 'for', '2', 'min', '.'],\n",
       " ['neural',\n",
       "  'D',\n",
       "  '1',\n",
       "  'protocol',\n",
       "  '330',\n",
       "  '.',\n",
       "  'Medi',\n",
       "  'small',\n",
       "  'cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  'screen',\n",
       "  '.'],\n",
       " ['.', 'next', '.', '4', 'boxes', 'P-100', 'tips', '.', 'Next'],\n",
       " ['8', '', '1.384', 'E', '5'],\n",
       " ['500', 'µl', 'of', '2X', 'media', 'to', 'each', 'condition', '.'],\n",
       " ['A', 'column', '1', 'is', '500', '.'],\n",
       " ['of', 'the', 'bacteria', '1.560'],\n",
       " ['round',\n",
       "  '5',\n",
       "  'D',\n",
       "  '5',\n",
       "  '.',\n",
       "  '35',\n",
       "  'mosquitoes',\n",
       "  '.',\n",
       "  '[',\n",
       "  'In',\n",
       "  'German',\n",
       "  ':',\n",
       "  '\"',\n",
       "  'Yes',\n",
       "  '.'],\n",
       " ['most',\n",
       "  'important',\n",
       "  'is',\n",
       "  'to',\n",
       "  'order',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'and',\n",
       "  'the',\n",
       "  'collection',\n",
       "  'tube',\n",
       "  'in',\n",
       "  'the',\n",
       "  'same',\n",
       "  'way',\n",
       "  'as',\n",
       "  'described',\n",
       "  'on',\n",
       "  'the',\n",
       "  'manual',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  'volume',\n",
       "  'for',\n",
       "  'the',\n",
       "  'isolation',\n",
       "  'of',\n",
       "  'the',\n",
       "  'RNA',\n",
       "  'should',\n",
       "  'be',\n",
       "  'depending',\n",
       "  'upon',\n",
       "  'the',\n",
       "  'number',\n",
       "  'of',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'that',\n",
       "  'are',\n",
       "  'being',\n",
       "  'lysed',\n",
       "  'in',\n",
       "  'the',\n",
       "  'RLT',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'The',\n",
       "  'RNA',\n",
       "  'samples',\n",
       "  'were',\n",
       "  'collected',\n",
       "  'on',\n",
       "  '25th',\n",
       "  'of',\n",
       "  'March',\n",
       "  'and',\n",
       "  'stored',\n",
       "  'at',\n",
       "  '-80',\n",
       "  'horizontal',\n",
       "  '.'],\n",
       " ['-', '17', '-', 'slide', '86', '.', 'Eyes', '.', '2', 'sections', '.'],\n",
       " ['me', 'to', 'check', 'on', 'sample', '#', '5', '.'],\n",
       " ['sample',\n",
       "  'is',\n",
       "  'high',\n",
       "  'affinity',\n",
       "  'peptide',\n",
       "  '.',\n",
       "  'So',\n",
       "  'far',\n",
       "  'measurement',\n",
       "  'looks',\n",
       "  'good',\n",
       "  'incubation',\n",
       "  'was',\n",
       "  '2',\n",
       "  'min',\n",
       "  'in',\n",
       "  'the',\n",
       "  'chamber',\n",
       "  '.'],\n",
       " ['order', '2', 'units', '', 'Stary', 'Zart', '.'],\n",
       " ['9',\n",
       "  'with',\n",
       "  'very',\n",
       "  'loud',\n",
       "  'lab',\n",
       "  'technician',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lab',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  'the',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'the',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'was',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['7', '', '3.174', 'E5', '.'],\n",
       " ['', 'starting', 'volume', 'of', 'the', 'bacteria', '24', 'ml', '.'],\n",
       " ['is', '1.529', 'E', '5', '.'],\n",
       " ['have', 'an', 'appointment', 'at', '14', 'PM', 'today', '.'],\n",
       " ['me', 'to', 'turn', 'on', 'the', 'machine', 'at', '10', 'AM', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'drink',\n",
       "  'a',\n",
       "  'glass',\n",
       "  'of',\n",
       "  'water',\n",
       "  'at',\n",
       "  '7:00',\n",
       "  'PM',\n",
       "  'today',\n",
       "  '.'],\n",
       " ['BHSD13',\n",
       "  'belongs',\n",
       "  'to',\n",
       "  'a',\n",
       "  'family',\n",
       "  'of',\n",
       "  'poorly',\n",
       "  'characterised',\n",
       "  'Enzymes',\n",
       "  'and',\n",
       "  'neither',\n",
       "  'is',\n",
       "  'physiological',\n",
       "  'function',\n",
       "  '',\n",
       "  'nor',\n",
       "  'pathological',\n",
       "  'role',\n",
       "  'is',\n",
       "  'developed',\n",
       "  'in',\n",
       "  'development',\n",
       "  'of',\n",
       "  'NASH',\n",
       "  'or',\n",
       "  'understood',\n",
       "  '.'],\n",
       " ['will',\n",
       "  'expand',\n",
       "  'RS',\n",
       "  'from',\n",
       "  '10',\n",
       "  'flasks',\n",
       "  'to',\n",
       "  '40',\n",
       "  'flasks',\n",
       "  '.',\n",
       "  'an',\n",
       "  '1',\n",
       "  'to',\n",
       "  '4',\n",
       "  'split',\n",
       "  '.'],\n",
       " ['made',\n",
       "  'the',\n",
       "  'Cas9',\n",
       "  'KRAB',\n",
       "  'teary',\n",
       "  'virus',\n",
       "  'stain',\n",
       "  'there',\n",
       "  'are',\n",
       "  '10',\n",
       "  'aliquots',\n",
       "  '.',\n",
       "  'The',\n",
       "  'aliquots',\n",
       "  'with',\n",
       "  'the',\n",
       "  'smaller',\n",
       "  'volume',\n",
       "  'that',\n",
       "  \"'s\",\n",
       "  'approximately',\n",
       "  '250',\n",
       "  'µl',\n",
       "  'should',\n",
       "  'be',\n",
       "  'split',\n",
       "  'into',\n",
       "  'thirds',\n",
       "  'while',\n",
       "  'the',\n",
       "  'aliquot',\n",
       "  'that',\n",
       "  'is',\n",
       "  'one',\n",
       "  'mL',\n",
       "  'should',\n",
       "  'be',\n",
       "  'split',\n",
       "  'in',\n",
       "  'half',\n",
       "  '.',\n",
       "  'I',\n",
       "  'put',\n",
       "  'them',\n",
       "  'in',\n",
       "  'the',\n",
       "  'virus',\n",
       "  '2',\n",
       "  'box',\n",
       "  'in',\n",
       "  '-80',\n",
       "  '.'],\n",
       " ['40', '2.444', 'E5', '.'],\n",
       " ['words',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.'],\n",
       " ['novel',\n",
       "  'D',\n",
       "  '1',\n",
       "  'protocol',\n",
       "  '330',\n",
       "  '.',\n",
       "  'Medi',\n",
       "  'small',\n",
       "  'cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  'screen',\n",
       "  '.'],\n",
       " ['neural',\n",
       "  'D',\n",
       "  '1',\n",
       "  'protocol',\n",
       "  '330',\n",
       "  '.',\n",
       "  'Medi',\n",
       "  'small',\n",
       "  'cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  'screen',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'said',\n",
       "  'about',\n",
       "  'the',\n",
       "  'Neuralinks',\n",
       "  'file',\n",
       "  'was',\n",
       "  'all',\n",
       "  'wrong',\n",
       "  'because',\n",
       "  'now',\n",
       "  'suddenly',\n",
       "  'I',\n",
       "  'do',\n",
       "  'not',\n",
       "  'get',\n",
       "  'error',\n",
       "  'messages',\n",
       "  '?',\n",
       "  'I',\n",
       "  'have',\n",
       "  'the',\n",
       "  'following',\n",
       "  'thng',\n",
       "  ':',\n",
       "  'the',\n",
       "  'first',\n",
       "  'one',\n",
       "  '',\n",
       "  'is',\n",
       "  'optostimulus',\n",
       "  'than',\n",
       "  '0',\n",
       "  '',\n",
       "  '1',\n",
       "  'is',\n",
       "  'the',\n",
       "  'clock',\n",
       "  'behavior',\n",
       "  '10',\n",
       "  'milliseconds',\n",
       "  '',\n",
       "  '2',\n",
       "  'is',\n",
       "  'force',\n",
       "  'sensor',\n",
       "  '',\n",
       "  '3',\n",
       "  'and',\n",
       "  '4',\n",
       "  'empty',\n",
       "  '.',\n",
       "  'Yeah',\n",
       "  '.'],\n",
       " ['images', 'end', 'on', '094024', 'Chip', 'D2-', '3', '.'],\n",
       " ['500', 'µl', 'of', '2X', 'media', 'to', 'each', 'condition', '.'],\n",
       " ['Imaging', 'for', 'experiment', '190801', '.'],\n",
       " ['timer', 'in', '10', 'min', '.'],\n",
       " ['of',\n",
       "  'the',\n",
       "  'most',\n",
       "  'annoying',\n",
       "  'things',\n",
       "  'is',\n",
       "  'the',\n",
       "  'fact',\n",
       "  'that',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'used',\n",
       "  'to',\n",
       "  'being',\n",
       "  'able',\n",
       "  'to',\n",
       "  'call',\n",
       "  'on',\n",
       "  'Siri',\n",
       "  'and',\n",
       "  'she',\n",
       "  'responds',\n",
       "  'a',\n",
       "  'lot',\n",
       "  'faster',\n",
       "  'than'],\n",
       " ['mastermix',\n",
       "  'for',\n",
       "  '5',\n",
       "  'replicates',\n",
       "  'was',\n",
       "  'made',\n",
       "  'up',\n",
       "  'for',\n",
       "  'the',\n",
       "  'cDNA',\n",
       "  '.'],\n",
       " ['CH', 'protein', 'stored', 'after', 'Ni2', '+', 'purification', '.'],\n",
       " ['round',\n",
       "  'of',\n",
       "  'calibration',\n",
       "  '.',\n",
       "  'CAL5',\n",
       "  'number',\n",
       "  '25',\n",
       "  '',\n",
       "  'and',\n",
       "  'PIN',\n",
       "  'number',\n",
       "  '25',\n",
       "  '.',\n",
       "  'Amplitudes',\n",
       "  'used',\n",
       "  '',\n",
       "  '1.5',\n",
       "  '',\n",
       "  '2.5',\n",
       "  '',\n",
       "  '3.5',\n",
       "  '',\n",
       "  '4.5',\n",
       "  '',\n",
       "  '5.5',\n",
       "  '',\n",
       "  '6.5',\n",
       "  '',\n",
       "  '7.5',\n",
       "  '',\n",
       "  '8.5',\n",
       "  '',\n",
       "  '9.5',\n",
       "  'volts',\n",
       "  '.',\n",
       "  'Maximum',\n",
       "  'redoubt',\n",
       "  'for',\n",
       "  'the',\n",
       "  'first',\n",
       "  'sensor',\n",
       "  'is',\n",
       "  'only',\n",
       "  'one',\n",
       "  'volt',\n",
       "  '.',\n",
       "  'Voltage',\n",
       "  'output',\n",
       "  'is',\n",
       "  'around',\n",
       "  '10',\n",
       "  'volts',\n",
       "  '.',\n",
       "  'Specially',\n",
       "  '9.5',\n",
       "  \"'s\",\n",
       "  'correct',\n",
       "  '.'],\n",
       " ['PAGE',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'sticky',\n",
       "  'and',\n",
       "  'thick',\n",
       "  '.',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'dilute',\n",
       "  'them',\n",
       "  '.',\n",
       "  'I',\n",
       "  'will',\n",
       "  'dilute',\n",
       "  'them',\n",
       "  'with',\n",
       "  'SDS',\n",
       "  'PAGE',\n",
       "  'buffer',\n",
       "  '',\n",
       "  'uh',\n",
       "  '',\n",
       "  'without',\n",
       "  'reducing',\n",
       "  'agents',\n",
       "  '.',\n",
       "  'I',\n",
       "  'take',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'SDS',\n",
       "  'PAGE',\n",
       "  '.'],\n",
       " ['a', 'temperature', 'change', 'from', '24', 'to', '22', '°', '.'],\n",
       " ['cells',\n",
       "  'looked',\n",
       "  'contaminated',\n",
       "  'this',\n",
       "  'morning',\n",
       "  '',\n",
       "  'so',\n",
       "  'I',\n",
       "  'treated',\n",
       "  'them',\n",
       "  'with',\n",
       "  '2',\n",
       "  'rounds',\n",
       "  'of',\n",
       "  '5',\n",
       "  'X',\n",
       "  'pen',\n",
       "  'strep',\n",
       "  '.',\n",
       "  'Wash',\n",
       "  'for',\n",
       "  '15',\n",
       "  'total',\n",
       "  'min',\n",
       "  '',\n",
       "  '10',\n",
       "  'min',\n",
       "  'for',\n",
       "  'one',\n",
       "  'wash',\n",
       "  'in',\n",
       "  '5',\n",
       "  'min',\n",
       "  'for',\n",
       "  'the',\n",
       "  'second',\n",
       "  '.'],\n",
       " ['BioELN',\n",
       "  'good',\n",
       "  'reference',\n",
       "  'for',\n",
       "  'for',\n",
       "  'archive',\n",
       "  'data',\n",
       "  '',\n",
       "  'but',\n",
       "  'I',\n",
       "  'tend',\n",
       "  'to',\n",
       "  'use',\n",
       "  'the',\n",
       "  'network',\n",
       "  'drive',\n",
       "  '1st',\n",
       "  '.'],\n",
       " ['on', 'ice', '10', 'min', '.'],\n",
       " ['concentration', 'for', 'L1', 'PSAD', 'is', '73.6', 'ng/µl', '.'],\n",
       " ['added', 'Coomassie', 'at', '1', 'in', '200', 'dilution', '.'],\n",
       " ['dilution', 'of', 'standards', 'from', 'wells', 'A1', 'to', 'A9', '.'],\n",
       " ['remind',\n",
       "  'me',\n",
       "  'to',\n",
       "  'do',\n",
       "  'writeups',\n",
       "  'today',\n",
       "  'at',\n",
       "  '4:00',\n",
       "  'PM',\n",
       "  'and',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '3:00',\n",
       "  'PM',\n",
       "  '.'],\n",
       " ['switchavidin',\n",
       "  '4',\n",
       "  'µl',\n",
       "  'MPR',\n",
       "  '3',\n",
       "  'bio',\n",
       "  'human',\n",
       "  '1139',\n",
       "  '2.5',\n",
       "  'µl',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  '.'],\n",
       " ['after',\n",
       "  'centrifugation',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'possible',\n",
       "  'to',\n",
       "  'pipette',\n",
       "  'sample',\n",
       "  '12',\n",
       "  '.',\n",
       "  'A',\n",
       "  'big',\n",
       "  'pellet',\n",
       "  'has',\n",
       "  'formed',\n",
       "  'in',\n",
       "  'the',\n",
       "  'bottom',\n",
       "  'of',\n",
       "  'the',\n",
       "  'tube',\n",
       "  '.'],\n",
       " ['me', 'to', 'clean', 'the', 'dishes', 'in', '5', 'min', '.'],\n",
       " ['set',\n",
       "  'up',\n",
       "  'the',\n",
       "  'baculovirus',\n",
       "  'in',\n",
       "  'the',\n",
       "  'Shaker',\n",
       "  'cell',\n",
       "  '.',\n",
       "  'For',\n",
       "  'experiment',\n",
       "  'A',\n",
       "  '12345',\n",
       "  '.'],\n",
       " ['NanoTemper',\n",
       "  'Tycho',\n",
       "  'NT6',\n",
       "  'with',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'in',\n",
       "  'PDB',\n",
       "  'number',\n",
       "  'order',\n",
       "  '',\n",
       "  'so',\n",
       "  'starting',\n",
       "  'from',\n",
       "  'PB19',\n",
       "  '-',\n",
       "  '1088',\n",
       "  'then',\n",
       "  '1089',\n",
       "  '',\n",
       "  'then',\n",
       "  '1090',\n",
       "  '',\n",
       "  'then',\n",
       "  '1091',\n",
       "  '',\n",
       "  'then',\n",
       "  '1092',\n",
       "  '',\n",
       "  'which',\n",
       "  'is',\n",
       "  'GFP',\n",
       "  '',\n",
       "  'and',\n",
       "  'finally',\n",
       "  '1094',\n",
       "  '.'],\n",
       " ['1', 'µl', 'of', 'fluid', 'into', 'the', 'tube', '.'],\n",
       " ['23rd',\n",
       "  'of',\n",
       "  'November',\n",
       "  '2018',\n",
       "  '',\n",
       "  'boiling',\n",
       "  'experiment',\n",
       "  'antifoam',\n",
       "  'in',\n",
       "  'water',\n",
       "  'distilled',\n",
       "  '.',\n",
       "  'Condition',\n",
       "  'no',\n",
       "  'foam',\n",
       "  '20',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  '72',\n",
       "  '°',\n",
       "  '.',\n",
       "  'With',\n",
       "  'foam',\n",
       "  '72.5',\n",
       "  '°',\n",
       "  '.',\n",
       "  '[',\n",
       "  'in',\n",
       "  'German',\n",
       "  ':',\n",
       "  '\"',\n",
       "  '.',\n",
       "  'Wrong',\n",
       "  'language',\n",
       "  ']'],\n",
       " ['12', '1.0.6', 'E5', '.'],\n",
       " ['more', 'MPR', '3', '.'],\n",
       " ['added',\n",
       "  'the',\n",
       "  'baculovirus',\n",
       "  'to',\n",
       "  'the',\n",
       "  'Shaker',\n",
       "  'flask',\n",
       "  '.',\n",
       "  'For',\n",
       "  'the',\n",
       "  'A',\n",
       "  '12345',\n",
       "  'experiment',\n",
       "  '.'],\n",
       " ['37', '4.797', 'E4', '.'],\n",
       " ['for',\n",
       "  'top',\n",
       "  'Agar',\n",
       "  'in',\n",
       "  '100',\n",
       "  'ml',\n",
       "  'of',\n",
       "  '7.5',\n",
       "  'g',\n",
       "  'of',\n",
       "  'Agar',\n",
       "  'pre',\n",
       "  'liter',\n",
       "  'added',\n",
       "  '25',\n",
       "  'ml',\n",
       "  'to',\n",
       "  'a',\n",
       "  'total',\n",
       "  'of',\n",
       "  '125',\n",
       "  'ml',\n",
       "  'to',\n",
       "  'get',\n",
       "  '6',\n",
       "  'g',\n",
       "  '/',\n",
       "  'l',\n",
       "  '.',\n",
       "  'To',\n",
       "  'that',\n",
       "  'add',\n",
       "  'magnesium',\n",
       "  'chloride',\n",
       "  'or',\n",
       "  'magnesium',\n",
       "  'sulfate',\n",
       "  'and',\n",
       "  'calcium',\n",
       "  'chloride',\n",
       "  '.'],\n",
       " ['is', '.249', 'at', '8:20', '.'],\n",
       " ['add',\n",
       "  '7.7',\n",
       "  'µl',\n",
       "  'LDS',\n",
       "  'to',\n",
       "  'each',\n",
       "  'tube',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'buffer',\n",
       "  'from',\n",
       "  'new',\n",
       "  '-',\n",
       "  'page',\n",
       "  'lot',\n",
       "  'number',\n",
       "  '1904557',\n",
       "  '.'],\n",
       " ['concentration',\n",
       "  'of',\n",
       "  'the',\n",
       "  'second',\n",
       "  'psPax',\n",
       "  '2',\n",
       "  'is',\n",
       "  '921.4',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Ah',\n",
       "  'the',\n",
       "  'fraction',\n",
       "  'of',\n",
       "  '260',\n",
       "  '/',\n",
       "  '280',\n",
       "  'is',\n",
       "  '2.13',\n",
       "  '.',\n",
       "  'The',\n",
       "  'fraction',\n",
       "  'of',\n",
       "  'the',\n",
       "  '60',\n",
       "  '/',\n",
       "  '230',\n",
       "  'is',\n",
       "  '2.4',\n",
       "  '.'],\n",
       " ['in',\n",
       "  'cold',\n",
       "  'methanol',\n",
       "  'at',\n",
       "  '500',\n",
       "  'rcf',\n",
       "  '',\n",
       "  'at',\n",
       "  '800',\n",
       "  'rcf',\n",
       "  'for',\n",
       "  '5',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['2',\n",
       "  'with',\n",
       "  'radio',\n",
       "  'and',\n",
       "  'second',\n",
       "  'person',\n",
       "  'but',\n",
       "  'no',\n",
       "  'talking',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lab',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'to',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['210', 'µl', 'of', 'EPX1', 'to', 'X', 'AMP', '.'],\n",
       " ['334',\n",
       "  '.',\n",
       "  'I',\n",
       "  'took',\n",
       "  'the',\n",
       "  'white',\n",
       "  'sample',\n",
       "  'or',\n",
       "  'yellow',\n",
       "  'from',\n",
       "  'the',\n",
       "  'mixed',\n",
       "  'culture',\n",
       "  '.'],\n",
       " ['terms',\n",
       "  'of',\n",
       "  'calculations',\n",
       "  '.',\n",
       "  'Are',\n",
       "  'you',\n",
       "  'recording',\n",
       "  '?',\n",
       "  'Okay',\n",
       "  'perfect',\n",
       "  '.',\n",
       "  'In',\n",
       "  'terms',\n",
       "  'of',\n",
       "  'calculation',\n",
       "  'for',\n",
       "  'my',\n",
       "  'first',\n",
       "  'set',\n",
       "  'of',\n",
       "  '30',\n",
       "  'slides',\n",
       "  'for',\n",
       "  'the',\n",
       "  'phospho',\n",
       "  '-',\n",
       "  'RAD50',\n",
       "  '',\n",
       "  'I',\n",
       "  'have',\n",
       "  'calculated',\n",
       "  'the',\n",
       "  'required',\n",
       "  'total',\n",
       "  'volume',\n",
       "  'for',\n",
       "  'this',\n",
       "  'run',\n",
       "  'will',\n",
       "  'be',\n",
       "  '5000',\n",
       "  'by',\n",
       "  'doing',\n",
       "  '30',\n",
       "  '*',\n",
       "  '150',\n",
       "  'at',\n",
       "  '350',\n",
       "  'and',\n",
       "  'rising',\n",
       "  'up',\n",
       "  'to',\n",
       "  '5000',\n",
       "  '.',\n",
       "  'To',\n",
       "  'calculate',\n",
       "  'my',\n",
       "  'antibody',\n",
       "  'volume',\n",
       "  'I',\n",
       "  'use',\n",
       "  'the',\n",
       "  'C1V1',\n",
       "  'equals',\n",
       "  'C2V2',\n",
       "  'calculation',\n",
       "  'and',\n",
       "  'calculated',\n",
       "  'that',\n",
       "  'of',\n",
       "  'a',\n",
       "  'stock',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '70',\n",
       "  'µg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '',\n",
       "  'in',\n",
       "  'which',\n",
       "  'I',\n",
       "  'require',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '1',\n",
       "  'ug',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '',\n",
       "  'the',\n",
       "  'total',\n",
       "  'antibody',\n",
       "  'volume',\n",
       "  'I',\n",
       "  'will',\n",
       "  'need',\n",
       "  'for',\n",
       "  'this',\n",
       "  'one',\n",
       "  'is',\n",
       "  '71.43',\n",
       "  'µl',\n",
       "  '.',\n",
       "  'Therefore',\n",
       "  '',\n",
       "  'the',\n",
       "  'total',\n",
       "  'diluent',\n",
       "  'volume',\n",
       "  'I',\n",
       "  'will',\n",
       "  'require',\n",
       "  'is',\n",
       "  '5000',\n",
       "  '-',\n",
       "  '71.43',\n",
       "  'which',\n",
       "  'will',\n",
       "  'equal',\n",
       "  '4928.57',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'make',\n",
       "  'a',\n",
       "  'test',\n",
       "  'tomorrow',\n",
       "  'morning',\n",
       "  'at',\n",
       "  '9:00',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['µl', 'pool', '23', 'into', 'lane', '4', '.'],\n",
       " ['me', 'about', 'the', 'meeting', 'at', '3:00', 'PM', '.', 'Meeting', '.'],\n",
       " ['in',\n",
       "  'deteriate',\n",
       "  'cells',\n",
       "  'from',\n",
       "  '5th',\n",
       "  'of',\n",
       "  'November',\n",
       "  '',\n",
       "  'wild',\n",
       "  'type',\n",
       "  '.',\n",
       "  'Looking',\n",
       "  'good',\n",
       "  '.',\n",
       "  'Culturing',\n",
       "  'with',\n",
       "  'a',\n",
       "  'endothelia',\n",
       "  'cell',\n",
       "  'growth',\n",
       "  'factor',\n",
       "  '.',\n",
       "  'Comparing',\n",
       "  'with',\n",
       "  'list',\n",
       "  '.',\n",
       "  'Looking',\n",
       "  'good',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'out',\n",
       "  'of',\n",
       "  'the',\n",
       "  'incubator',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '2:00',\n",
       "  'PM',\n",
       "  '.'],\n",
       " ['means',\n",
       "  'you',\n",
       "  'can',\n",
       "  'annotate',\n",
       "  'what',\n",
       "  'viable',\n",
       "  'cells',\n",
       "  'on',\n",
       "  'the',\n",
       "  'viability',\n",
       "  'counts',\n",
       "  'is',\n",
       "  '.',\n",
       "  'If',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'lower',\n",
       "  'than',\n",
       "  '85',\n",
       "  '%',\n",
       "  'then',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'a',\n",
       "  'worry',\n",
       "  '.',\n",
       "  'She',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'start',\n",
       "  'the',\n",
       "  'assay',\n",
       "  'but',\n",
       "  'needs',\n",
       "  'to',\n",
       "  'be',\n",
       "  'in',\n",
       "  'the',\n",
       "  'right',\n",
       "  'conditions',\n",
       "  'for',\n",
       "  'it',\n",
       "  '.'],\n",
       " ['timer', 'for', '30', 'min', '', 'thank', 'you', '.'],\n",
       " ['330',\n",
       "  'neuro',\n",
       "  '-',\n",
       "  'D1',\n",
       "  'Medi',\n",
       "  'small',\n",
       "  'cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  '04',\n",
       "  'model',\n",
       "  '.',\n",
       "  'High',\n",
       "  'nuclear',\n",
       "  'expression',\n",
       "  'across',\n",
       "  'tissue',\n",
       "  'medium',\n",
       "  'intensity',\n",
       "  'level',\n",
       "  '.',\n",
       "  'Few',\n",
       "  'areas',\n",
       "  'that',\n",
       "  'are',\n",
       "  'not',\n",
       "  'stained',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'add',\n",
       "  'an',\n",
       "  'appointment',\n",
       "  'for',\n",
       "  'TEM',\n",
       "  'on',\n",
       "  'Wednesday',\n",
       "  'at',\n",
       "  '9',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'heavy',\n",
       "  'and',\n",
       "  'light',\n",
       "  'standard',\n",
       "  'mix',\n",
       "  'into',\n",
       "  'column',\n",
       "  '20',\n",
       "  '.'],\n",
       " ['1',\n",
       "  'spr',\n",
       "  'wells',\n",
       "  'B11',\n",
       "  '',\n",
       "  'B12',\n",
       "  '',\n",
       "  'G12',\n",
       "  '',\n",
       "  'J11',\n",
       "  '',\n",
       "  'J12',\n",
       "  '',\n",
       "  'K12',\n",
       "  '',\n",
       "  'O12'],\n",
       " ['is',\n",
       "  'in',\n",
       "  'start',\n",
       "  'of',\n",
       "  'experiment',\n",
       "  '.',\n",
       "  'Isoflurane',\n",
       "  'on',\n",
       "  'one',\n",
       "  'point',\n",
       "  '.',\n",
       "  '7',\n",
       "  'months',\n",
       "  'was',\n",
       "  'not',\n",
       "  'really',\n",
       "  'Liking',\n",
       "  'the',\n",
       "  'isoflurane',\n",
       "  '',\n",
       "  'but',\n",
       "  'now',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'calm',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'induced',\n",
       "  'IFLK',\n",
       "  'BAC',\n",
       "  '16',\n",
       "  'quadruple',\n",
       "  '15',\n",
       "  'times',\n",
       "  'T175',\n",
       "  'flasks',\n",
       "  '.'],\n",
       " ['when', 'ozone', '1', 'mice', 'are', 'being', 'culled', '.'],\n",
       " ['trying',\n",
       "  'free',\n",
       "  'protocol',\n",
       "  'using',\n",
       "  'best',\n",
       "  'sensitivity',\n",
       "  '.',\n",
       "  'So',\n",
       "  '5',\n",
       "  'min',\n",
       "  'activation',\n",
       "  'and',\n",
       "  'exposure',\n",
       "  'for',\n",
       "  'faint',\n",
       "  'bands',\n",
       "  '.',\n",
       "  'Running',\n",
       "  'the',\n",
       "  'protocol',\n",
       "  '.'],\n",
       " ['to',\n",
       "  'A9',\n",
       "  'very',\n",
       "  'complicated',\n",
       "  'chemical',\n",
       "  'poured',\n",
       "  'into',\n",
       "  'another',\n",
       "  'one',\n",
       "  '.'],\n",
       " ['21', '1.631', 'E5', '.'],\n",
       " ['µl',\n",
       "  'apply',\n",
       "  'pluro',\n",
       "  'pluronic',\n",
       "  'was',\n",
       "  'added',\n",
       "  'to',\n",
       "  'kinase',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['is', '200', '.', 'Next', '.', 'A2', 'is', '10.5', '.'],\n",
       " ['exotherm',\n",
       "  'reaction',\n",
       "  '',\n",
       "  'however',\n",
       "  '',\n",
       "  'the',\n",
       "  'added',\n",
       "  'CDT',\n",
       "  'solution',\n",
       "  'has',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'so',\n",
       "  'already',\n",
       "  'big',\n",
       "  'temperature',\n",
       "  'difference',\n",
       "  '.',\n",
       "  'Dark',\n",
       "  'brown',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dense',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  '20',\n",
       "  '°',\n",
       "  'overnight',\n",
       "  '.'],\n",
       " ['the', 'timer', '30', 'min', '.'],\n",
       " ['more', 'test', 'to', 'identify', 'the', 'latency', '.'],\n",
       " ['measurement', 'is', '0.7', '.', 'It', \"'s\", 'number', '16', 'already', '.'],\n",
       " ['decided',\n",
       "  'to',\n",
       "  'split',\n",
       "  'the',\n",
       "  'Jurkat',\n",
       "  'cells',\n",
       "  'one',\n",
       "  'more',\n",
       "  'time',\n",
       "  '.',\n",
       "  'I',\n",
       "  'did',\n",
       "  'a',\n",
       "  'full',\n",
       "  'split',\n",
       "  'of',\n",
       "  'the',\n",
       "  'culture',\n",
       "  '',\n",
       "  'so',\n",
       "  'not',\n",
       "  'wasting',\n",
       "  'any',\n",
       "  'cell',\n",
       "  'at',\n",
       "  'all',\n",
       "  '.',\n",
       "  'And',\n",
       "  'split',\n",
       "  'them',\n",
       "  'into',\n",
       "  '2',\n",
       "  '',\n",
       "  'so',\n",
       "  'I',\n",
       "  'have',\n",
       "  'a',\n",
       "  'backup',\n",
       "  'flask',\n",
       "  'in',\n",
       "  'case',\n",
       "  'something',\n",
       "  'goes',\n",
       "  'wrong',\n",
       "  'today',\n",
       "  'during',\n",
       "  'the',\n",
       "  'set',\n",
       "  'up',\n",
       "  'of',\n",
       "  'the',\n",
       "  'drug',\n",
       "  'dilution',\n",
       "  '',\n",
       "  'so',\n",
       "  'kill',\n",
       "  'curve',\n",
       "  'test',\n",
       "  '.'],\n",
       " ['μl',\n",
       "  'of',\n",
       "  'Fusion',\n",
       "  'master',\n",
       "  'makes',\n",
       "  '1.2',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'DMSO',\n",
       "  '0.4',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'forward',\n",
       "  'primer',\n",
       "  'and',\n",
       "  'reverse',\n",
       "  'primer',\n",
       "  'and',\n",
       "  '17.6',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'water',\n",
       "  '.',\n",
       "  'To',\n",
       "  'the',\n",
       "  'pcr',\n",
       "  'reaction',\n",
       "  'for',\n",
       "  'chop',\n",
       "  'and',\n",
       "  'LISA',\n",
       "  'stuff',\n",
       "  '.'],\n",
       " ['same', 'with', '22', 'and', 'neon', '.'],\n",
       " ['are',\n",
       "  '18',\n",
       "  'mil',\n",
       "  'of',\n",
       "  'Jurkat',\n",
       "  'cells',\n",
       "  'and',\n",
       "  '23',\n",
       "  'mil',\n",
       "  'of',\n",
       "  'Raji',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ['timer', '5', 'min', '.'],\n",
       " ['the',\n",
       "  'gel',\n",
       "  'also',\n",
       "  'with',\n",
       "  'instant',\n",
       "  'blue',\n",
       "  'use',\n",
       "  '3',\n",
       "  'times',\n",
       "  '.',\n",
       "  'Leave',\n",
       "  'it',\n",
       "  'over',\n",
       "  'the',\n",
       "  'weekend',\n",
       "  'with',\n",
       "  'a',\n",
       "  'lock',\n",
       "  '.',\n",
       "  'Mark',\n",
       "  'the',\n",
       "  'box',\n",
       "  'with',\n",
       "  'tape',\n",
       "  'and',\n",
       "  'describing',\n",
       "  'dates',\n",
       "  'and',\n",
       "  'contents',\n",
       "  '.'],\n",
       " ['here',\n",
       "  'the',\n",
       "  'IMAC',\n",
       "  'medium',\n",
       "  'for',\n",
       "  'the',\n",
       "  'dissections',\n",
       "  '.',\n",
       "  'It',\n",
       "  'carries',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  '6',\n",
       "  'times',\n",
       "  '',\n",
       "  '6',\n",
       "  'parts',\n",
       "  'of',\n",
       "  'Schneider',\n",
       "  \"'s\",\n",
       "  'medium',\n",
       "  '.',\n",
       "  '3',\n",
       "  'parts',\n",
       "  'of',\n",
       "  'citrate',\n",
       "  'buffer',\n",
       "  'and',\n",
       "  'one',\n",
       "  'part',\n",
       "  'of',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'FBS',\n",
       "  '.',\n",
       "  'So',\n",
       "  'basically',\n",
       "  '6',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'Schneider',\n",
       "  \"'s\",\n",
       "  '',\n",
       "  '3',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'citrate',\n",
       "  'and',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'FBS',\n",
       "  '.',\n",
       "  'Later',\n",
       "  'everything',\n",
       "  'is',\n",
       "  'mixed',\n",
       "  'and',\n",
       "  'filtered',\n",
       "  'through',\n",
       "  'the',\n",
       "  'filter',\n",
       "  '22',\n",
       "  'µm',\n",
       "  'filter',\n",
       "  '',\n",
       "  'so',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'sterile',\n",
       "  '.',\n",
       "  'So',\n",
       "  'I',\n",
       "  'made',\n",
       "  'that',\n",
       "  'and',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'filling',\n",
       "  'the',\n",
       "  'tubes',\n",
       "  'that',\n",
       "  'I',\n",
       "  'prepare',\n",
       "  'yesterday',\n",
       "  'with',\n",
       "  '30',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'this',\n",
       "  'IMAC',\n",
       "  'medium',\n",
       "  '.',\n",
       "  'So',\n",
       "  'once',\n",
       "  'I',\n",
       "  'dissected',\n",
       "  'tissues',\n",
       "  'and',\n",
       "  'cells',\n",
       "  'they',\n",
       "  'have',\n",
       "  'something',\n",
       "  'to',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'stay',\n",
       "  'and',\n",
       "  'stay',\n",
       "  'alive',\n",
       "  '.'],\n",
       " ['surgery', 'at', '10:00', \"o'clock\", '.'],\n",
       " ['strain',\n",
       "  '168',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'a',\n",
       "  'tryptophan',\n",
       "  'auxotroph',\n",
       "  'so',\n",
       "  'I',\n",
       "  'have',\n",
       "  'to',\n",
       "  'add',\n",
       "  'tryptophan',\n",
       "  'to',\n",
       "  'the',\n",
       "  'media',\n",
       "  'used',\n",
       "  'in',\n",
       "  'PY',\n",
       "  '79',\n",
       "  '',\n",
       "  'the',\n",
       "  'amount',\n",
       "  'of',\n",
       "  'auxoph',\n",
       "  'the',\n",
       "  'amount',\n",
       "  'of',\n",
       "  'tryptophan',\n",
       "  'of',\n",
       "  'an',\n",
       "  'old',\n",
       "  'recipe',\n",
       "  'for',\n",
       "  'transformation',\n",
       "  '',\n",
       "  'they',\n",
       "  'one',\n",
       "  'from',\n",
       "  'S',\n",
       "  'SM',\n",
       "  'one',\n",
       "  'SM',\n",
       "  '2',\n",
       "  'media',\n",
       "  '.'],\n",
       " ['20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'and',\n",
       "  '6.7',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'LDS',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Samples',\n",
       "  'are',\n",
       "  'prepared',\n",
       "  'without',\n",
       "  'heating',\n",
       "  'and',\n",
       "  'run',\n",
       "  'on',\n",
       "  'a',\n",
       "  'BioRad',\n",
       "  'gel',\n",
       "  '.'],\n",
       " ['20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'and',\n",
       "  '6.7',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'LDS',\n",
       "  'buffer',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'prepared',\n",
       "  'without',\n",
       "  'heating',\n",
       "  'and',\n",
       "  'run',\n",
       "  'on',\n",
       "  'a',\n",
       "  'BioRad',\n",
       "  'gel'],\n",
       " ['1', 'sample', 'and', 'diluted', 'by', '5ML', '.'],\n",
       " ['for', '10', 'min', '.'],\n",
       " ['group', '3', 'first', '.'],\n",
       " ['20', 'ml', 'to', 'vial', '.'],\n",
       " ['me', 'to', 'check', 'stat', '3', '.'],\n",
       " ['me', 'to', 'call', 'my', 'sister', 'at', '2', 'PM', 'tomorrow', '.'],\n",
       " ['cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  '08',\n",
       "  'model',\n",
       "  'study',\n",
       "  '19',\n",
       "  '091',\n",
       "  '.',\n",
       "  'Medium',\n",
       "  'intensity',\n",
       "  'nuclear',\n",
       "  'staining',\n",
       "  'throughout',\n",
       "  'entire',\n",
       "  'tumor',\n",
       "  'tissue',\n",
       "  '.',\n",
       "  'Some',\n",
       "  'areas',\n",
       "  'of',\n",
       "  'no',\n",
       "  'staining',\n",
       "  '.'],\n",
       " ['sample',\n",
       "  'tubes',\n",
       "  'for',\n",
       "  'the',\n",
       "  'dissection',\n",
       "  'tomorrow',\n",
       "  '.',\n",
       "  'Abbreviations',\n",
       "  'that',\n",
       "  'are',\n",
       "  'written',\n",
       "  'on',\n",
       "  'the',\n",
       "  'tubes',\n",
       "  'are',\n",
       "  'consisting',\n",
       "  'of',\n",
       "  'the',\n",
       "  'number',\n",
       "  'and',\n",
       "  'letters',\n",
       "  '.',\n",
       "  'So',\n",
       "  'the',\n",
       "  'number',\n",
       "  '3',\n",
       "  'represents',\n",
       "  'third',\n",
       "  'replicate',\n",
       "  '.',\n",
       "  'First',\n",
       "  'G',\n",
       "  '',\n",
       "  'G1',\n",
       "  'is',\n",
       "  'foregut',\n",
       "  '.',\n",
       "  'G2',\n",
       "  'is',\n",
       "  'hindgut',\n",
       "  '',\n",
       "  'G',\n",
       "  'alone',\n",
       "  'is',\n",
       "  'whole',\n",
       "  'gut',\n",
       "  '.',\n",
       "  'Then',\n",
       "  '',\n",
       "  'the',\n",
       "  'second',\n",
       "  '-',\n",
       "  'um',\n",
       "  '',\n",
       "  'no',\n",
       "  '',\n",
       "  'there',\n",
       "  'is',\n",
       "  'one',\n",
       "  'letters',\n",
       "  '-',\n",
       "  'F',\n",
       "  'stand',\n",
       "  'for',\n",
       "  'fat',\n",
       "  'body',\n",
       "  '',\n",
       "  'H',\n",
       "  'for',\n",
       "  'hemocytes',\n",
       "  '.',\n",
       "  'Second',\n",
       "  'letter',\n",
       "  'is',\n",
       "  '-',\n",
       "  'um',\n",
       "  '-',\n",
       "  'labeling',\n",
       "  'the',\n",
       "  'strain',\n",
       "  'of',\n",
       "  'the',\n",
       "  'mosquitoes',\n",
       "  '.',\n",
       "  'So',\n",
       "  'G',\n",
       "  'stands',\n",
       "  'for',\n",
       "  'G3',\n",
       "  'and',\n",
       "  'R',\n",
       "  'for',\n",
       "  'RKG',\n",
       "  '.',\n",
       "  'SF',\n",
       "  'below',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'labels',\n",
       "  'means',\n",
       "  'sugar',\n",
       "  'fat',\n",
       "  '',\n",
       "  'BF',\n",
       "  'blood',\n",
       "  'fat',\n",
       "  'and',\n",
       "  'PF',\n",
       "  'plasmodium',\n",
       "  'falciparum',\n",
       "  'infected',\n",
       "  '.'],\n",
       " ['am', 'starting', 'with', 'Channel', '2', '.'],\n",
       " ['LB', 'medium', '500', 'g', '10', 'boxes', '.'],\n",
       " ['test',\n",
       "  '2',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'to',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '?',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['phase',\n",
       "  'of',\n",
       "  'experiments',\n",
       "  'will',\n",
       "  'be',\n",
       "  'attempted',\n",
       "  'today',\n",
       "  '.',\n",
       "  'With',\n",
       "  'the',\n",
       "  'first',\n",
       "  'perfusion',\n",
       "  'solution',\n",
       "  'arranged',\n",
       "  'in',\n",
       "  'this',\n",
       "  'order',\n",
       "  '.',\n",
       "  'Pyrot',\n",
       "  '4',\n",
       "  'AP',\n",
       "  'caffeine',\n",
       "  '5',\n",
       "  'mM',\n",
       "  '480',\n",
       "  '.',\n",
       "  '4',\n",
       "  'AP',\n",
       "  'with',\n",
       "  'Interleukin',\n",
       "  '1',\n",
       "  'beta',\n",
       "  '.',\n",
       "  '4',\n",
       "  'AP',\n",
       "  'beta',\n",
       "  'leukin',\n",
       "  '1',\n",
       "  '5',\n",
       "  'mM.'],\n",
       " ['sample',\n",
       "  '2',\n",
       "  'and',\n",
       "  '3',\n",
       "  'are',\n",
       "  'very',\n",
       "  'grainy',\n",
       "  'and',\n",
       "  'basically',\n",
       "  'impossible',\n",
       "  'to',\n",
       "  'pipette',\n",
       "  '.'],\n",
       " ['357',\n",
       "  'ml',\n",
       "  'of',\n",
       "  '0.7',\n",
       "  'molar',\n",
       "  'TCEP',\n",
       "  'to',\n",
       "  'read',\n",
       "  '1',\n",
       "  'mM',\n",
       "  'TCEP',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lysis',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['labeled', '1.5', 'ml', 'cups', 'for', 'the', 'compliment', 'pool', '.'],\n",
       " ['test',\n",
       "  'number',\n",
       "  '2',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'roto',\n",
       "  'bath',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['1',\n",
       "  'spr',\n",
       "  'buffer',\n",
       "  '20',\n",
       "  'mM',\n",
       "  'HEPES',\n",
       "  '',\n",
       "  '150',\n",
       "  'mM',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  '',\n",
       "  '1',\n",
       "  'mM',\n",
       "  'TCEP',\n",
       "  '0.05',\n",
       "  '%',\n",
       "  'Tween20',\n",
       "  'pH',\n",
       "  '7.5',\n",
       "  '.'],\n",
       " ['timer', 'for', '2', 'min', '.'],\n",
       " ['to', 'check', 'the', 'cells', 'at', '11:50', 'PM', '.'],\n",
       " ['for',\n",
       "  'gore',\n",
       "  'tex',\n",
       "  'membran',\n",
       "  '',\n",
       "  'dermalon',\n",
       "  '',\n",
       "  'monofilament',\n",
       "  '4',\n",
       "  '-',\n",
       "  '0',\n",
       "  '.'],\n",
       " ['me', 'to', 'turn', 'on', 'the', 'alarm', 'at', '1:30', '.'],\n",
       " ['9', '1.848', 'E5', '.'],\n",
       " ['am',\n",
       "  'using',\n",
       "  'the',\n",
       "  'PAL',\n",
       "  'pipette',\n",
       "  'and',\n",
       "  'a',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'stripette',\n",
       "  'to',\n",
       "  'do',\n",
       "  '4',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'my',\n",
       "  'volume',\n",
       "  'and',\n",
       "  'to',\n",
       "  'do',\n",
       "  'the',\n",
       "  '845',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'using',\n",
       "  'the',\n",
       "  'P1000',\n",
       "  '6',\n",
       "  'purple',\n",
       "  'pipette',\n",
       "  '',\n",
       "  'serial',\n",
       "  'number',\n",
       "  'T6052HE',\n",
       "  '',\n",
       "  'calculate',\n",
       "  'number',\n",
       "  'DOB9UBB6KS',\n",
       "  '.'],\n",
       " ['started',\n",
       "  '4',\n",
       "  'equivalents',\n",
       "  'of',\n",
       "  'the',\n",
       "  'acid',\n",
       "  'and',\n",
       "  'power',\n",
       "  'formaldehyde',\n",
       "  'was',\n",
       "  'added',\n",
       "  '',\n",
       "  'and',\n",
       "  'the',\n",
       "  'reaction',\n",
       "  'runs',\n",
       "  'at',\n",
       "  'at',\n",
       "  '50',\n",
       "  '°',\n",
       "  '.'],\n",
       " ['#',\n",
       "  '3',\n",
       "  '.',\n",
       "  'Top',\n",
       "  '.',\n",
       "  'Three',\n",
       "  'sterile',\n",
       "  'filtration',\n",
       "  'flask',\n",
       "  'bottoms',\n",
       "  'about',\n",
       "  '1',\n",
       "  'l',\n",
       "  'each',\n",
       "  '.',\n",
       "  'A',\n",
       "  'set',\n",
       "  'of',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'HPLC',\n",
       "  'columns',\n",
       "  'or',\n",
       "  'micro',\n",
       "  'LC',\n",
       "  'columns',\n",
       "  '',\n",
       "  'not',\n",
       "  'used',\n",
       "  '',\n",
       "  'a',\n",
       "  'series',\n",
       "  'of',\n",
       "  'last',\n",
       "  'pipets',\n",
       "  'not',\n",
       "  'in',\n",
       "  'use',\n",
       "  '.',\n",
       "  'About',\n",
       "  '15',\n",
       "  'plate',\n",
       "  'seals',\n",
       "  '',\n",
       "  'microplate',\n",
       "  'seals',\n",
       "  'aluminum',\n",
       "  '.',\n",
       "  'And',\n",
       "  'a',\n",
       "  'case',\n",
       "  'of',\n",
       "  'plates',\n",
       "  'not',\n",
       "  'used',\n",
       "  '.',\n",
       "  'Next',\n",
       "  '.',\n",
       "  'Waters',\n",
       "  'acuity',\n",
       "  'bottles',\n",
       "  '',\n",
       "  'not',\n",
       "  'used',\n",
       "  '.',\n",
       "  'A',\n",
       "  'lab',\n",
       "  'jack',\n",
       "  'stand',\n",
       "  '.',\n",
       "  'Lab',\n",
       "  'bag',\n",
       "  'role',\n",
       "  'of',\n",
       "  'absorbent',\n",
       "  'liner',\n",
       "  '.',\n",
       "  'And',\n",
       "  '3',\n",
       "  'more',\n",
       "  'filtration',\n",
       "  'flask',\n",
       "  'that',\n",
       "  'are',\n",
       "  'PVC',\n",
       "  'coated',\n",
       "  '.',\n",
       "  'Next',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  '.',\n",
       "  'About'],\n",
       " ['purification',\n",
       "  'loading',\n",
       "  'the',\n",
       "  'diluted',\n",
       "  'sample',\n",
       "  'via',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'pump',\n",
       "  'on',\n",
       "  'the',\n",
       "  'resource',\n",
       "  'Q',\n",
       "  'Column',\n",
       "  'in',\n",
       "  'position',\n",
       "  '2',\n",
       "  'with',\n",
       "  'fractionation',\n",
       "  'in',\n",
       "  '50',\n",
       "  'ml',\n",
       "  'fractions',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  '.',\n",
       "  'Right',\n",
       "  '.'],\n",
       " ['counts',\n",
       "  'are',\n",
       "  'finished',\n",
       "  'resuspending',\n",
       "  'samples',\n",
       "  'to',\n",
       "  '400',\n",
       "  'micro',\n",
       "  '.'],\n",
       " ['timer', 'for', '45', 'min', '.'],\n",
       " ['want',\n",
       "  'to',\n",
       "  'add',\n",
       "  '20',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'amylase',\n",
       "  'to',\n",
       "  'the',\n",
       "  'experiment',\n",
       "  '.'],\n",
       " ['4', 'well', 'plate', 'read', 'at', '3:15', 'PM', '.'],\n",
       " ['3', '9.565', 'E4', '.'],\n",
       " ['110044394', '.', 'Cultured', 'in', '4.2', 'ml'],\n",
       " ['dose', 'of', 'ABT', '199', 'occured', 'today', 'as', 'well', '.'],\n",
       " ['5', 'ml', 'ammonium', 'formate', '.'],\n",
       " ['ending',\n",
       "  'in',\n",
       "  '8543',\n",
       "  '?',\n",
       "  'Well',\n",
       "  '',\n",
       "  'I',\n",
       "  'was',\n",
       "  'precipitated',\n",
       "  'in',\n",
       "  'K10k11K12',\n",
       "  '.',\n",
       "  'Oh',\n",
       "  'trials',\n",
       "  '.'],\n",
       " ['one',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'MACS',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Centrifugation',\n",
       "  '300',\n",
       "  'g',\n",
       "  '5',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['run', 'human', 'run', '.', 'OK', '.'],\n",
       " ['TOC', 'value', 'is', '286', 'parts', 'per', 'billion', '.'],\n",
       " ['µl', 'RSB', 'in', 'the', 'lane', '2', '.'],\n",
       " ['37', '', '2.670', 'E', '5'],\n",
       " ['23', '.', 'Combination', '1', 'media', 'added', '.'],\n",
       " ['the',\n",
       "  'prep',\n",
       "  'of',\n",
       "  'the',\n",
       "  'instruments',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'immobilized',\n",
       "  'my',\n",
       "  'protein',\n",
       "  '.',\n",
       "  'This',\n",
       "  'can',\n",
       "  'take',\n",
       "  '30',\n",
       "  'min',\n",
       "  'to',\n",
       "  'a',\n",
       "  'few',\n",
       "  'h.'],\n",
       " ['one',\n",
       "  'construct',\n",
       "  'we',\n",
       "  'continue',\n",
       "  'to',\n",
       "  'work',\n",
       "  'with',\n",
       "  'will',\n",
       "  'be',\n",
       "  'the',\n",
       "  'clone',\n",
       "  '5',\n",
       "  '.',\n",
       "  'Which',\n",
       "  'shows',\n",
       "  'Newman',\n",
       "  'Newman',\n",
       "  'no',\n",
       "  'mutations',\n",
       "  'and',\n",
       "  'Um',\n",
       "  'perfectly',\n",
       "  'alliance',\n",
       "  'with',\n",
       "  'the',\n",
       "  'target',\n",
       "  'sequence',\n",
       "  '.'],\n",
       " ['',\n",
       "  'uh',\n",
       "  '',\n",
       "  'apply',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'in',\n",
       "  'each',\n",
       "  'lane',\n",
       "  '.'],\n",
       " ['72',\n",
       "  'h',\n",
       "  'time',\n",
       "  'point',\n",
       "  'was',\n",
       "  'washed',\n",
       "  '1',\n",
       "  'x',\n",
       "  'with',\n",
       "  'PBS',\n",
       "  '.',\n",
       "  'The',\n",
       "  'PBS',\n",
       "  'was',\n",
       "  'then',\n",
       "  'removed',\n",
       "  'and',\n",
       "  'the',\n",
       "  'plate',\n",
       "  'was',\n",
       "  'placed',\n",
       "  'at',\n",
       "  '-',\n",
       "  '80',\n",
       "  '°',\n",
       "  '.'],\n",
       " ['cells',\n",
       "  '',\n",
       "  '24',\n",
       "  'hours',\n",
       "  'post',\n",
       "  'infection',\n",
       "  ':',\n",
       "  'supernatant',\n",
       "  'and',\n",
       "  'dilutions',\n",
       "  'of',\n",
       "  'case',\n",
       "  'HV',\n",
       "  'wild',\n",
       "  'type',\n",
       "  'stock',\n",
       "  'and',\n",
       "  'Delta',\n",
       "  'K',\n",
       "  '8.1',\n",
       "  '.'],\n",
       " ['sample',\n",
       "  'across',\n",
       "  'full',\n",
       "  'protein',\n",
       "  'coding',\n",
       "  'genes',\n",
       "  'in',\n",
       "  'the',\n",
       "  'genome',\n",
       "  'and',\n",
       "  'cross',\n",
       "  'references',\n",
       "  'with',\n",
       "  'around',\n",
       "  'phenotypic',\n",
       "  'end',\n",
       "  'points',\n",
       "  'that',\n",
       "  'they',\n",
       "  'recorded',\n",
       "  'which',\n",
       "  'comprise',\n",
       "  'very',\n",
       "  'binary',\n",
       "  'and',\n",
       "  'quantitative',\n",
       "  'phenotypes',\n",
       "  '.',\n",
       "  'And',\n",
       "  'we',\n",
       "  \"'ve\",\n",
       "  'also',\n",
       "  'done',\n",
       "  'this',\n",
       "  'in',\n",
       "  '11',\n",
       "  'different',\n",
       "  'ways',\n",
       "  'that',\n",
       "  'before',\n",
       "  'genetic',\n",
       "  'models',\n",
       "  '',\n",
       "  'are',\n",
       "  'just',\n",
       "  'different',\n",
       "  'ways',\n",
       "  'of',\n",
       "  'categorizing',\n",
       "  'the',\n",
       "  'mutations',\n",
       "  'or',\n",
       "  'variants',\n",
       "  '.',\n",
       "  'So',\n",
       "  'some',\n",
       "  'focus',\n",
       "  'for',\n",
       "  'example',\n",
       "  '',\n",
       "  'on',\n",
       "  'protein',\n",
       "  'estate',\n",
       "  'invariant',\n",
       "  'or',\n",
       "  'loss',\n",
       "  'of',\n",
       "  'function',\n",
       "  'variants',\n",
       "  'and',\n",
       "  'others',\n",
       "  'also',\n",
       "  'include',\n",
       "  'rare',\n",
       "  'missense',\n",
       "  'variant',\n",
       "  '.',\n",
       "  'So',\n",
       "  '',\n",
       "  'in',\n",
       "  'to'],\n",
       " ['105', 'µl', 'X', 'AMP', 'into', 'lane', '3', '.'],\n",
       " ['MCF-7',\n",
       "  'cells',\n",
       "  'appear',\n",
       "  'to',\n",
       "  'be',\n",
       "  'coming',\n",
       "  'off',\n",
       "  'the',\n",
       "  'plate',\n",
       "  '.'],\n",
       " ['12',\n",
       "  '',\n",
       "  '13',\n",
       "  '',\n",
       "  '14',\n",
       "  'contain',\n",
       "  'the',\n",
       "  'protein',\n",
       "  '',\n",
       "  'but',\n",
       "  'also',\n",
       "  'contaminations',\n",
       "  '.'],\n",
       " ['have', 'to', 'buy', '15', 'mL', 'centrifuge', 'tubes', '.'],\n",
       " ['did',\n",
       "  'the',\n",
       "  'cloning',\n",
       "  'for',\n",
       "  'M',\n",
       "  'scarlet',\n",
       "  'tubulin',\n",
       "  'and',\n",
       "  'M',\n",
       "  'scarlet',\n",
       "  'life',\n",
       "  'act',\n",
       "  '.',\n",
       "  'I',\n",
       "  'used',\n",
       "  '50',\n",
       "  'ng',\n",
       "  'of',\n",
       "  'the',\n",
       "  'vector',\n",
       "  'and',\n",
       "  '100',\n",
       "  'ng',\n",
       "  'of',\n",
       "  'the',\n",
       "  'insert',\n",
       "  '.',\n",
       "  'At',\n",
       "  'first',\n",
       "  'I',\n",
       "  'did',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'ethanol',\n",
       "  'precipitation',\n",
       "  'and',\n",
       "  'than',\n",
       "  'I',\n",
       "  'assembled',\n",
       "  'with',\n",
       "  'Gibson',\n",
       "  'master',\n",
       "  'mix',\n",
       "  '.',\n",
       "  '[',\n",
       "  'I',\n",
       "  'transfused',\n",
       "  ']',\n",
       "  'I',\n",
       "  'transformed',\n",
       "  'in',\n",
       "  'Stbl',\n",
       "  '3',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ['cells', 'for', '30', 'min', 'at', '37', '℃', '.'],\n",
       " ['photo',\n",
       "  'shows',\n",
       "  'the',\n",
       "  'actual',\n",
       "  'information',\n",
       "  'on',\n",
       "  'the',\n",
       "  'four',\n",
       "  'plates',\n",
       "  'and',\n",
       "  'numbered',\n",
       "  'plate',\n",
       "  '1',\n",
       "  '',\n",
       "  'plate',\n",
       "  '2',\n",
       "  '',\n",
       "  'plate',\n",
       "  '4',\n",
       "  '',\n",
       "  'and',\n",
       "  'P',\n",
       "  'plate',\n",
       "  '.',\n",
       "  'Cancel',\n",
       "  '.'],\n",
       " ['timers',\n",
       "  'have',\n",
       "  'gone',\n",
       "  'off',\n",
       "  '.',\n",
       "  'Begin',\n",
       "  'dispensing',\n",
       "  '.',\n",
       "  '45',\n",
       "  'µl',\n",
       "  'per',\n",
       "  'well',\n",
       "  'to',\n",
       "  'a',\n",
       "  'new',\n",
       "  'clear',\n",
       "  'plate',\n",
       "  'for',\n",
       "  'the',\n",
       "  'QUANTI',\n",
       "  '-',\n",
       "  'Blue',\n",
       "  'reagent',\n",
       "  '',\n",
       "  'dispensing',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['a', '3', '.', 'timer', 'for', '16', 'min', '.'],\n",
       " ['test',\n",
       "  'with',\n",
       "  'jabra',\n",
       "  'difficult',\n",
       "  'words',\n",
       "  'and',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['images', 'for', 'D2-', '2', 'starting', 'on', '09365', '.'],\n",
       " ['CAS',\n",
       "  '9',\n",
       "  'LCMS',\n",
       "  'experiment',\n",
       "  'to',\n",
       "  'identify',\n",
       "  'peptides',\n",
       "  'for',\n",
       "  'a',\n",
       "  'quantitative',\n",
       "  'assay',\n",
       "  '.'],\n",
       " ['',\n",
       "  'OK',\n",
       "  '',\n",
       "  'so',\n",
       "  'at',\n",
       "  'least',\n",
       "  'one',\n",
       "  'dish',\n",
       "  'from',\n",
       "  'day',\n",
       "  '22',\n",
       "  'extra',\n",
       "  'and',\n",
       "  'one',\n",
       "  'dish',\n",
       "  'from',\n",
       "  'day',\n",
       "  '29',\n",
       "  'extra',\n",
       "  '.',\n",
       "  'That',\n",
       "  \"'s\",\n",
       "  'why',\n",
       "  'I',\n",
       "  'decided',\n",
       "  'to',\n",
       "  'first',\n",
       "  'do',\n",
       "  'the',\n",
       "  'staining',\n",
       "  'on',\n",
       "  'those',\n",
       "  'two',\n",
       "  'dishes',\n",
       "  '.'],\n",
       " ['we',\n",
       "  'demonstrate',\n",
       "  'the',\n",
       "  'weighting',\n",
       "  'process',\n",
       "  'and',\n",
       "  'the',\n",
       "  'automatic',\n",
       "  'data',\n",
       "  'transfer',\n",
       "  'from',\n",
       "  'the',\n",
       "  'scale',\n",
       "  'into',\n",
       "  'the',\n",
       "  'ELN',\n",
       "  'during',\n",
       "  'the',\n",
       "  'making',\n",
       "  'of',\n",
       "  'the',\n",
       "  'new',\n",
       "  'formulation',\n",
       "  '.',\n",
       "  'Each',\n",
       "  'of',\n",
       "  'the',\n",
       "  'components',\n",
       "  'is',\n",
       "  'being',\n",
       "  'weighed',\n",
       "  'individually',\n",
       "  'and',\n",
       "  'the',\n",
       "  'balance',\n",
       "  'will',\n",
       "  'transfer',\n",
       "  'the',\n",
       "  'weight',\n",
       "  'data',\n",
       "  '.',\n",
       "  'In',\n",
       "  'this',\n",
       "  'case',\n",
       "  '.',\n",
       "  'We',\n",
       "  'chose',\n",
       "  'calibration',\n",
       "  'weights',\n",
       "  'of',\n",
       "  '55',\n",
       "  'and',\n",
       "  '20',\n",
       "  'g',\n",
       "  '',\n",
       "  'respectively',\n",
       "  '',\n",
       "  'for',\n",
       "  'demonstration',\n",
       "  'purposes',\n",
       "  'to',\n",
       "  'showcase',\n",
       "  'the',\n",
       "  'functionality',\n",
       "  '.',\n",
       "  'At',\n",
       "  'the',\n",
       "  'end',\n",
       "  'of',\n",
       "  'the',\n",
       "  'weighting',\n",
       "  'process',\n",
       "  'except',\n",
       "  'the',\n",
       "  'data',\n",
       "  'input',\n",
       "  '.'],\n",
       " ['.',\n",
       "  'It',\n",
       "  'depends',\n",
       "  'on',\n",
       "  'the',\n",
       "  'data',\n",
       "  'that',\n",
       "  'we',\n",
       "  \"'re\",\n",
       "  'getting',\n",
       "  '.',\n",
       "  'Usually',\n",
       "  'this',\n",
       "  'influences',\n",
       "  'the',\n",
       "  'investment',\n",
       "  'decision',\n",
       "  '.',\n",
       "  'New',\n",
       "  'projects',\n",
       "  'have',\n",
       "  'a',\n",
       "  'target',\n",
       "  'that',\n",
       "  'we',\n",
       "  'generate',\n",
       "  '.'],\n",
       " ['has',\n",
       "  'a',\n",
       "  'live',\n",
       "  'cell',\n",
       "  'count',\n",
       "  'of',\n",
       "  '7',\n",
       "  '*',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '5th',\n",
       "  '.'],\n",
       " ['.',\n",
       "  'The',\n",
       "  'cycling',\n",
       "  'program',\n",
       "  'will',\n",
       "  'expire',\n",
       "  'in',\n",
       "  '5',\n",
       "  'min',\n",
       "  '.',\n",
       "  'And',\n",
       "  'then',\n",
       "  'we',\n",
       "  'are',\n",
       "  'setting',\n",
       "  'up',\n",
       "  'a',\n",
       "  'ligation',\n",
       "  'reaction',\n",
       "  'using'],\n",
       " ['molecular', 'weight', '213.25', '.'],\n",
       " ['is', 'Daniel', 'Sutton', '.'],\n",
       " ['all',\n",
       "  'the',\n",
       "  'tubes',\n",
       "  '',\n",
       "  'thus',\n",
       "  'includes',\n",
       "  'all',\n",
       "  '6',\n",
       "  'inlet',\n",
       "  'tubes',\n",
       "  'A1',\n",
       "  'tube',\n",
       "  '',\n",
       "  'B1',\n",
       "  '',\n",
       "  'B4',\n",
       "  'and',\n",
       "  'B6',\n",
       "  '.'],\n",
       " ['transduction',\n",
       "  'test',\n",
       "  '.',\n",
       "  'I',\n",
       "  'did',\n",
       "  'yesterday',\n",
       "  'a',\n",
       "  'test',\n",
       "  'with',\n",
       "  'different',\n",
       "  'conditions',\n",
       "  '.',\n",
       "  'First',\n",
       "  'condition',\n",
       "  'transduction',\n",
       "  'without',\n",
       "  'spinfection',\n",
       "  '.',\n",
       "  'The',\n",
       "  'second',\n",
       "  '',\n",
       "  'is',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'HFF',\n",
       "  'pellet',\n",
       "  'on',\n",
       "  'ice',\n",
       "  'for',\n",
       "  '6',\n",
       "  'h',\n",
       "  'with',\n",
       "  'polybrene',\n",
       "  '',\n",
       "  'without',\n",
       "  'polybrene',\n",
       "  'and',\n",
       "  'the',\n",
       "  'control',\n",
       "  '.',\n",
       "  'The',\n",
       "  'third',\n",
       "  'is',\n",
       "  'spinfection',\n",
       "  'in',\n",
       "  'plate',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h',\n",
       "  \"'s\",\n",
       "  '.',\n",
       "  'The',\n",
       "  'fourth',\n",
       "  'is',\n",
       "  'spinfection',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h',\n",
       "  '',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'as',\n",
       "  'in',\n",
       "  'pellet',\n",
       "  'and',\n",
       "  'the',\n",
       "  'fifth',\n",
       "  'is',\n",
       "  'spinfection',\n",
       "  'for',\n",
       "  '6',\n",
       "  'h',\n",
       "  '',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'as',\n",
       "  'in',\n",
       "  'pellet',\n",
       "  '.',\n",
       "  'At',\n",
       "  'first',\n",
       "  'I',\n",
       "  'will',\n",
       "  'check',\n",
       "  'the',\n",
       "  'transduction',\n",
       "  'in',\n",
       "  'plate',\n",
       "  '.'],\n",
       " ['bioELN',\n",
       "  'good',\n",
       "  'reference',\n",
       "  'for',\n",
       "  'for',\n",
       "  'archive',\n",
       "  'data',\n",
       "  '',\n",
       "  'but',\n",
       "  'I',\n",
       "  'tend',\n",
       "  'to',\n",
       "  'use',\n",
       "  'the',\n",
       "  'network',\n",
       "  'drive',\n",
       "  '1st',\n",
       "  '.'],\n",
       " ['34', 'best', 'in', 'class', '3', 'media', 'added', '.'],\n",
       " ['SEC',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Filter',\n",
       "  'through',\n",
       "  '0.22',\n",
       "  'µm',\n",
       "  'filter',\n",
       "  '.',\n",
       "  'Then',\n",
       "  'distribute',\n",
       "  '100',\n",
       "  'ml',\n",
       "  'in',\n",
       "  '3',\n",
       "  'bottles',\n",
       "  '.',\n",
       "  'Add',\n",
       "  'to',\n",
       "  'AZ2250',\n",
       "  '10',\n",
       "  'µl',\n",
       "  '10',\n",
       "  'mM',\n",
       "  'compound',\n",
       "  'to',\n",
       "  'give',\n",
       "  'a',\n",
       "  'final',\n",
       "  '1',\n",
       "  'mM',\n",
       "  'solution',\n",
       "  '.',\n",
       "  'For',\n",
       "  'the',\n",
       "  'AZ14180161',\n",
       "  'add',\n",
       "  '2',\n",
       "  'µl',\n",
       "  'to',\n",
       "  'get',\n",
       "  'a',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '0.2',\n",
       "  'µM.'],\n",
       " ['?',\n",
       "  ']',\n",
       "  'is',\n",
       "  'the',\n",
       "  'pool',\n",
       "  'of',\n",
       "  '[',\n",
       "  '?',\n",
       "  ']',\n",
       "  'in',\n",
       "  'the',\n",
       "  'MXBI',\n",
       "  '5',\n",
       "  'for',\n",
       "  'Doreen',\n",
       "  '.',\n",
       "  'Maybe',\n",
       "  '5',\n",
       "  'ml'],\n",
       " ['465', 'µl', 'of', 'denaturation', 'into', 'library', '2', '.'],\n",
       " ['is', 'unstimulated', '.'],\n",
       " ['the',\n",
       "  'first',\n",
       "  'well',\n",
       "  'with',\n",
       "  'polybrene',\n",
       "  'of',\n",
       "  'the',\n",
       "  'spinfection',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h',\n",
       "  'pellet',\n",
       "  '',\n",
       "  'I',\n",
       "  'can',\n",
       "  'see',\n",
       "  'green',\n",
       "  'cells',\n",
       "  '',\n",
       "  'but',\n",
       "  'not',\n",
       "  'a',\n",
       "  'lot',\n",
       "  '.',\n",
       "  'I',\n",
       "  'would',\n",
       "  'say',\n",
       "  'maybe',\n",
       "  '1',\n",
       "  '%',\n",
       "  '.',\n",
       "  'Um',\n",
       "  '.',\n",
       "  'Without',\n",
       "  'polybrene',\n",
       "  'I',\n",
       "  'ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  '.',\n",
       "  'I',\n",
       "  'ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  'green',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ['just',\n",
       "  'set',\n",
       "  'up',\n",
       "  'the',\n",
       "  'sorter',\n",
       "  'on',\n",
       "  '150',\n",
       "  'µm',\n",
       "  'nozzle',\n",
       "  'for',\n",
       "  'single',\n",
       "  'cell',\n",
       "  'sorting',\n",
       "  'from',\n",
       "  'days',\n",
       "  '.',\n",
       "  'Efficiency',\n",
       "  'of',\n",
       "  'the',\n",
       "  'drop',\n",
       "  'delay',\n",
       "  'is',\n",
       "  'little',\n",
       "  'bit',\n",
       "  'flexible',\n",
       "  'around',\n",
       "  '0.1',\n",
       "  '%',\n",
       "  'to',\n",
       "  '0.5',\n",
       "  '%',\n",
       "  'terms',\n",
       "  'of',\n",
       "  'non',\n",
       "  'sorted',\n",
       "  'reads',\n",
       "  '.',\n",
       "  'This',\n",
       "  'should',\n",
       "  'be',\n",
       "  'OK',\n",
       "  'for',\n",
       "  'single',\n",
       "  'cell',\n",
       "  'sorting',\n",
       "  'for',\n",
       "  'her',\n",
       "  'purposes',\n",
       "  '.'],\n",
       " ['5',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'AZ',\n",
       "  '225021',\n",
       "  'of',\n",
       "  'the',\n",
       "  'solubilization',\n",
       "  'buffers',\n",
       "  '.',\n",
       "  'Into',\n",
       "  'the',\n",
       "  'second',\n",
       "  'add',\n",
       "  'AZ0161',\n",
       "  'GSK',\n",
       "  'compound',\n",
       "  '.',\n",
       "  'Final',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  'both',\n",
       "  'is',\n",
       "  '1',\n",
       "  'µM'],\n",
       " ['HU', '1645', '.'],\n",
       " ['33', '140000'],\n",
       " ['29065',\n",
       "  '-',\n",
       "  '013',\n",
       "  '.',\n",
       "  'E9',\n",
       "  'was',\n",
       "  'dissolved',\n",
       "  'in',\n",
       "  'DMF',\n",
       "  '2',\n",
       "  'ml',\n",
       "  'and',\n",
       "  'cool',\n",
       "  'down',\n",
       "  'to',\n",
       "  '0',\n",
       "  '°',\n",
       "  '.',\n",
       "  'Stirred',\n",
       "  'until',\n",
       "  'temperature',\n",
       "  'was',\n",
       "  'stable',\n",
       "  '.',\n",
       "  'In',\n",
       "  'the',\n",
       "  'mean',\n",
       "  'time',\n",
       "  'the',\n",
       "  'CDT',\n",
       "  'was',\n",
       "  'dissolved',\n",
       "  'in',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'DMF',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  '.'],\n",
       " ['50',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'each',\n",
       "  'solubilise',\n",
       "  'solubilisate',\n",
       "  'and',\n",
       "  'FSEC',\n",
       "  'analysis',\n",
       "  '.',\n",
       "  'Add',\n",
       "  'the',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'washed',\n",
       "  'strep',\n",
       "  'tactin',\n",
       "  'to',\n",
       "  'the',\n",
       "  'remainder',\n",
       "  'of',\n",
       "  'the',\n",
       "  'solubilisate',\n",
       "  '.'],\n",
       " ['condition',\n",
       "  'will',\n",
       "  'have',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'total',\n",
       "  'volume',\n",
       "  '.',\n",
       "  'Resuspend',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'in',\n",
       "  '1.5',\n",
       "  'ml',\n",
       "  'and',\n",
       "  'aliquot',\n",
       "  '500',\n",
       "  'µl',\n",
       "  '/',\n",
       "  'condition',\n",
       "  '.'],\n",
       " ['condition',\n",
       "  'will',\n",
       "  'have',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'total',\n",
       "  'volume',\n",
       "  '.',\n",
       "  'Resuspend',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'in',\n",
       "  '1.5',\n",
       "  'ml',\n",
       "  'and',\n",
       "  'aliquot',\n",
       "  '500',\n",
       "  'µl',\n",
       "  '/',\n",
       "  'condition',\n",
       "  '.'],\n",
       " ['bands',\n",
       "  'Project',\n",
       "  'PMS',\n",
       "  '6',\n",
       "  '.',\n",
       "  '1',\n",
       "  '.',\n",
       "  'No',\n",
       "  'bands',\n",
       "  'PMS6',\n",
       "  '2',\n",
       "  '',\n",
       "  'no',\n",
       "  'bands',\n",
       "  'PMS6',\n",
       "  '3',\n",
       "  '.'],\n",
       " ['my', 'experiment', 'plan', '001', '.'],\n",
       " ['chabra',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lab',\n",
       "  '',\n",
       "  'radio',\n",
       "  'in',\n",
       "  'the',\n",
       "  'background',\n",
       "  '',\n",
       "  'no',\n",
       "  'special',\n",
       "  'noise',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['block',\n",
       "  'for',\n",
       "  'today',\n",
       "  '.',\n",
       "  'Started',\n",
       "  'working',\n",
       "  'on',\n",
       "  'the',\n",
       "  'speed',\n",
       "  'slide',\n",
       "  'from',\n",
       "  '8:00',\n",
       "  'to',\n",
       "  '9:30',\n",
       "  '.',\n",
       "  '10:30',\n",
       "  'to',\n",
       "  '11:30',\n",
       "  '',\n",
       "  'synchronization',\n",
       "  '.',\n",
       "  'Then',\n",
       "  'lunch',\n",
       "  'break',\n",
       "  '',\n",
       "  'now',\n",
       "  'start',\n",
       "  'again',\n",
       "  '.'],\n",
       " ['means',\n",
       "  'you',\n",
       "  'can',\n",
       "  'annotate',\n",
       "  'what',\n",
       "  'viable',\n",
       "  'cells',\n",
       "  'on',\n",
       "  'the',\n",
       "  'viability',\n",
       "  'counts',\n",
       "  'is',\n",
       "  'if',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'lower',\n",
       "  'than',\n",
       "  '85',\n",
       "  '%',\n",
       "  'then',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'a',\n",
       "  'worry',\n",
       "  '.',\n",
       "  'She',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'start',\n",
       "  'the',\n",
       "  'assay',\n",
       "  '',\n",
       "  'but',\n",
       "  'needs',\n",
       "  'to',\n",
       "  'be',\n",
       "  'in',\n",
       "  'the',\n",
       "  'right',\n",
       "  'conditions',\n",
       "  'for',\n",
       "  'it',\n",
       "  '.'],\n",
       " ['x', 'PBS', 'into', 'well', 'A12', 'of', 'third', 'plate', '.'],\n",
       " ['weighed', 'out', '32.7', 'mg', 'of', 'standard', 'for', 'MK', '1234', '.'],\n",
       " ['mentioned',\n",
       "  'he',\n",
       "  'kept',\n",
       "  'the',\n",
       "  'pump',\n",
       "  'speed',\n",
       "  'at',\n",
       "  '60',\n",
       "  '',\n",
       "  'but',\n",
       "  'the',\n",
       "  'pump',\n",
       "  'speed',\n",
       "  'is',\n",
       "  'set',\n",
       "  'to',\n",
       "  '120',\n",
       "  '.'],\n",
       " ['light', 'stream', 'mounts', 'one', 'stimulation', 'MA', '.'],\n",
       " ['29', '1.634', 'E5', '.'],\n",
       " ['Pi',\n",
       "  'PCR',\n",
       "  'cycle',\n",
       "  'test',\n",
       "  ':',\n",
       "  'protein',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  \"n't\",\n",
       "  'limitating',\n",
       "  'step',\n",
       "  'for',\n",
       "  'melting',\n",
       "  'curves',\n",
       "  '.',\n",
       "  'High',\n",
       "  'amount',\n",
       "  'of',\n",
       "  'Body',\n",
       "  'Pi',\n",
       "  'agent',\n",
       "  'needed',\n",
       "  'to',\n",
       "  'see',\n",
       "  'good',\n",
       "  'curves',\n",
       "  '.',\n",
       "  'More',\n",
       "  'than',\n",
       "  '10',\n",
       "  'µg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  'to',\n",
       "  'get',\n",
       "  'good',\n",
       "  'results',\n",
       "  'and',\n",
       "  'great',\n",
       "  'curves',\n",
       "  '.'],\n",
       " ['buffer',\n",
       "  'for',\n",
       "  'FSEC',\n",
       "  '40',\n",
       "  'mM',\n",
       "  'HEPES',\n",
       "  'pH',\n",
       "  '7.4',\n",
       "  '',\n",
       "  '300',\n",
       "  'mM',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  '',\n",
       "  '1',\n",
       "  'mM',\n",
       "  'TCEP',\n",
       "  '',\n",
       "  '10',\n",
       "  '%',\n",
       "  'Glycerol',\n",
       "  '.',\n",
       "  'Changing',\n",
       "  'FSEC',\n",
       "  'from',\n",
       "  'water',\n",
       "  'to',\n",
       "  'buffer',\n",
       "  'and',\n",
       "  'let',\n",
       "  'it',\n",
       "  'run',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['in', 'the', 'back', '12', '%', 'gel', 'in', 'the', 'front', '6', '%', '.'],\n",
       " ['the',\n",
       "  'buffer',\n",
       "  'of',\n",
       "  '20',\n",
       "  'mM',\n",
       "  'Tris',\n",
       "  'pH',\n",
       "  '7.5',\n",
       "  '',\n",
       "  '120',\n",
       "  'mM',\n",
       "  'sodium',\n",
       "  'chlorid',\n",
       "  '',\n",
       "  '0.5',\n",
       "  'mM',\n",
       "  'TCEP',\n",
       "  'at',\n",
       "  'the',\n",
       "  'flow',\n",
       "  'rate',\n",
       "  'of',\n",
       "  '1',\n",
       "  'ml',\n",
       "  '/',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['fade',\n",
       "  'prepared',\n",
       "  'from',\n",
       "  'aliquotes',\n",
       "  'in',\n",
       "  'assay',\n",
       "  'diluent',\n",
       "  'is',\n",
       "  'added',\n",
       "  'to',\n",
       "  'column',\n",
       "  '3',\n",
       "  'and',\n",
       "  'column',\n",
       "  '4',\n",
       "  '.'],\n",
       " ['today',\n",
       "  'we',\n",
       "  'are',\n",
       "  'going',\n",
       "  'to',\n",
       "  'isolate',\n",
       "  'RNA',\n",
       "  'for',\n",
       "  'the',\n",
       "  'complete',\n",
       "  'set',\n",
       "  'of',\n",
       "  'human',\n",
       "  'adult',\n",
       "  'fibroblasts',\n",
       "  '.',\n",
       "  'We',\n",
       "  'have',\n",
       "  'two',\n",
       "  'sets',\n",
       "  '.',\n",
       "  'Set',\n",
       "  'one',\n",
       "  '',\n",
       "  'set',\n",
       "  'two',\n",
       "  '.',\n",
       "  'The',\n",
       "  'protocol',\n",
       "  'is',\n",
       "  'step',\n",
       "  '1',\n",
       "  ':',\n",
       "  'wash',\n",
       "  'all',\n",
       "  '6-well',\n",
       "  'plates',\n",
       "  'with',\n",
       "  'ice',\n",
       "  'cold',\n",
       "  'PBS',\n",
       "  'three',\n",
       "  'times',\n",
       "  'for',\n",
       "  'TRIzol',\n",
       "  'reagent',\n",
       "  'in',\n",
       "  '6-well',\n",
       "  'plate',\n",
       "  'for',\n",
       "  '10',\n",
       "  'minutes',\n",
       "  '',\n",
       "  '600',\n",
       "  'microliters',\n",
       "  'each',\n",
       "  '.',\n",
       "  'Pipette',\n",
       "  'the',\n",
       "  'lysate',\n",
       "  'into',\n",
       "  'ice',\n",
       "  'cold',\n",
       "  'eppys',\n",
       "  '..'],\n",
       " ['me', 'to', 'select', 'the', 'HFFs', '48', 'hours', '.', 'Later'],\n",
       " ['try',\n",
       "  'of',\n",
       "  'experiment',\n",
       "  'for',\n",
       "  'calibration',\n",
       "  '.',\n",
       "  'File',\n",
       "  'name',\n",
       "  'is',\n",
       "  'Cal25',\n",
       "  'for',\n",
       "  'film',\n",
       "  'and',\n",
       "  'for',\n",
       "  'behavior',\n",
       "  '.',\n",
       "  'Same',\n",
       "  'conditions',\n",
       "  'as',\n",
       "  'before',\n",
       "  '.'],\n",
       " ['is', 'test', 'number', '2', '.'],\n",
       " ['were', 'started', 'at', '7:07', '.', 'Sorry', '', '3', 'min', '.'],\n",
       " ['project',\n",
       "  'PMS',\n",
       "  '6',\n",
       "  '-',\n",
       "  '1-UC',\n",
       "  ':',\n",
       "  'Digest',\n",
       "  'with',\n",
       "  'DPN1',\n",
       "  'restriction',\n",
       "  'enzyme',\n",
       "  '.'],\n",
       " ['me', 'to', 'check', 'the', 'solvent', 'tomorrow', 'at', '8:00', 'AM', '.'],\n",
       " ['C9', 'to', 'C12', 'precipitated', '.'],\n",
       " ['29', 'Carboplatin', 'media', 'added', '.'],\n",
       " ['cell', 'number', 'is', '3', 'to', 'the', 'power', 'of', '6', '.'],\n",
       " ['me',\n",
       "  'in',\n",
       "  '10',\n",
       "  'min',\n",
       "  'about',\n",
       "  'blood',\n",
       "  'sample',\n",
       "  '.',\n",
       "  'Avocado',\n",
       "  '.',\n",
       "  'Blood',\n",
       "  'sample',\n",
       "  '.'],\n",
       " ['a', 'timer', 'for', '5', 'min', '.'],\n",
       " ['2',\n",
       "  'ml',\n",
       "  'slurry',\n",
       "  'strep',\n",
       "  'Tactin',\n",
       "  'Sepharose',\n",
       "  'High',\n",
       "  'performance',\n",
       "  'from',\n",
       "  'GE',\n",
       "  'Healthcare',\n",
       "  'catalog',\n",
       "  'number',\n",
       "  '28',\n",
       "  '-93255',\n",
       "  '-',\n",
       "  '99',\n",
       "  '.',\n",
       "  'LOT',\n",
       "  '1028253',\n",
       "  'expiry',\n",
       "  'date',\n",
       "  '22',\n",
       "  'August',\n",
       "  '2022',\n",
       "  'August',\n",
       "  '.',\n",
       "  'Take',\n",
       "  '2',\n",
       "  'ml',\n",
       "  'slurry',\n",
       "  'and',\n",
       "  'wash',\n",
       "  'with',\n",
       "  'water',\n",
       "  '.'],\n",
       " ['5',\n",
       "  'cycle',\n",
       "  '.',\n",
       "  'No',\n",
       "  'surface',\n",
       "  'changes',\n",
       "  'including',\n",
       "  'cut',\n",
       "  'edges',\n",
       "  'and',\n",
       "  'wells',\n",
       "  '.',\n",
       "  '°',\n",
       "  'of',\n",
       "  'Ristine',\n",
       "  'DINNISO',\n",
       "  '4628',\n",
       "  'dash',\n",
       "  '3',\n",
       "  '.',\n",
       "  'After',\n",
       "  '30',\n",
       "  'cycles',\n",
       "  'no',\n",
       "  'surface',\n",
       "  'changes',\n",
       "  'including',\n",
       "  'cut',\n",
       "  'edges',\n",
       "  'and',\n",
       "  'wells',\n",
       "  'loss',\n",
       "  'of',\n",
       "  'addition',\n",
       "  '.',\n",
       "  'D',\n",
       "  'less',\n",
       "  'than',\n",
       "  '1.5',\n",
       "  'mm',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'said',\n",
       "  'about',\n",
       "  'the',\n",
       "  'Neuralinks',\n",
       "  'file',\n",
       "  'was',\n",
       "  'all',\n",
       "  'wrong',\n",
       "  'because',\n",
       "  'now',\n",
       "  'suddenly',\n",
       "  'I',\n",
       "  'do',\n",
       "  'not',\n",
       "  'get',\n",
       "  'error',\n",
       "  'messages',\n",
       "  '?',\n",
       "  'I',\n",
       "  'have',\n",
       "  'the',\n",
       "  'following',\n",
       "  'thng',\n",
       "  ':',\n",
       "  'the',\n",
       "  'first',\n",
       "  'one',\n",
       "  '',\n",
       "  'is',\n",
       "  'optostimulus',\n",
       "  'than',\n",
       "  '0',\n",
       "  '',\n",
       "  '1',\n",
       "  'is',\n",
       "  'the',\n",
       "  'clock',\n",
       "  'behavior',\n",
       "  '10',\n",
       "  'milliseconds',\n",
       "  '',\n",
       "  '2',\n",
       "  'is',\n",
       "  'force',\n",
       "  'sensor',\n",
       "  '',\n",
       "  '3',\n",
       "  'and',\n",
       "  '4',\n",
       "  'empty',\n",
       "  '.',\n",
       "  'Yeah',\n",
       "  '.'],\n",
       " ['number',\n",
       "  '3',\n",
       "  'is',\n",
       "  'very',\n",
       "  'fatty',\n",
       "  'which',\n",
       "  'will',\n",
       "  'make',\n",
       "  'it',\n",
       "  'even',\n",
       "  'easier',\n",
       "  'even',\n",
       "  'more',\n",
       "  'difficult',\n",
       "  'to',\n",
       "  'dounce',\n",
       "  '.',\n",
       "  'You',\n",
       "  'can',\n",
       "  'see',\n",
       "  'visibly',\n",
       "  'the',\n",
       "  'fat',\n",
       "  'the',\n",
       "  'position',\n",
       "  'and',\n",
       "  'the',\n",
       "  'yellow',\n",
       "  'color',\n",
       "  'so',\n",
       "  'we',\n",
       "  'can',\n",
       "  'expect',\n",
       "  'a',\n",
       "  'lot',\n",
       "  'of',\n",
       "  'adipocytes',\n",
       "  'in',\n",
       "  'the',\n",
       "  'analysis',\n",
       "  '.',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']'],\n",
       " ['cassette', 'is', '3.9', '.'],\n",
       " ['ml', 'of', 'water', 'were', 'added', '.'],\n",
       " ['cell',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  '260000',\n",
       "  '',\n",
       "  'uh',\n",
       "  '',\n",
       "  'in',\n",
       "  'a',\n",
       "  'total',\n",
       "  'volume',\n",
       "  'of',\n",
       "  '15',\n",
       "  'ml',\n",
       "  '.',\n",
       "  'Uhm',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'add',\n",
       "  '297',\n",
       "  'ml',\n",
       "  'to',\n",
       "  'a',\n",
       "  'final',\n",
       "  'volume',\n",
       "  'of',\n",
       "  '312',\n",
       "  '',\n",
       "  'Uh',\n",
       "  '',\n",
       "  'to',\n",
       "  'see',\n",
       "  '.'],\n",
       " ['a', 'second', 'timer', 'for', '43', 'min', '.'],\n",
       " ['started', 'at', '3:07', 'PM', '.'],\n",
       " ['2-',\n",
       "  '2',\n",
       "  'chip',\n",
       "  'has',\n",
       "  'a',\n",
       "  'entry',\n",
       "  'channel',\n",
       "  'minor',\n",
       "  'debris',\n",
       "  '.',\n",
       "  'There',\n",
       "  'is',\n",
       "  'a',\n",
       "  'fiber',\n",
       "  'inside',\n",
       "  'one',\n",
       "  'of',\n",
       "  'the',\n",
       "  'channels',\n",
       "  '.',\n",
       "  'And',\n",
       "  'there',\n",
       "  'is',\n",
       "  'another',\n",
       "  'debris',\n",
       "  'at',\n",
       "  'the',\n",
       "  'entry',\n",
       "  'to',\n",
       "  'post',\n",
       "  'interface',\n",
       "  '.',\n",
       "  'There',\n",
       "  'is',\n",
       "  'minor',\n",
       "  'cell',\n",
       "  'wrapping',\n",
       "  'in',\n",
       "  'the',\n",
       "  'post',\n",
       "  'arrays',\n",
       "  'getting',\n",
       "  'progressively',\n",
       "  'worse',\n",
       "  'as',\n",
       "  'you',\n",
       "  'go',\n",
       "  'through',\n",
       "  'the',\n",
       "  'chip',\n",
       "  '.',\n",
       "  'Um',\n",
       "  'at',\n",
       "  'the',\n",
       "  'exit',\n",
       "  'post',\n",
       "  'entry',\n",
       "  'interface',\n",
       "  'there',\n",
       "  'is',\n",
       "  'minor',\n",
       "  'cell',\n",
       "  'debris',\n",
       "  'and',\n",
       "  'spider',\n",
       "  'like',\n",
       "  'spider',\n",
       "  'web',\n",
       "  'like',\n",
       "  'cell',\n",
       "  'debris',\n",
       "  '.',\n",
       "  'Um',\n",
       "  'chip',\n",
       "  'flow',\n",
       "  'is',\n",
       "  'going',\n",
       "  'from',\n",
       "  'bottom',\n",
       "  'to',\n",
       "  'top',\n",
       "  'of',\n",
       "  'image',\n",
       "  '.',\n",
       "  'There',\n",
       "  'is',\n",
       "  'a',\n",
       "  'larger',\n",
       "  'dimple',\n",
       "  'in',\n",
       "  'the',\n",
       "  'chip',\n",
       "  'that',\n",
       "  'might',\n",
       "  'not',\n",
       "  'have',\n",
       "  'been',\n",
       "  'caught',\n",
       "  'during',\n",
       "  'an',\n",
       "  'earlier',\n",
       "  'inspection',\n",
       "  '.'],\n",
       " ['we',\n",
       "  'are',\n",
       "  'going',\n",
       "  'to',\n",
       "  'seed',\n",
       "  '20000',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'milliliter',\n",
       "  '',\n",
       "  'and',\n",
       "  'we',\n",
       "  'have',\n",
       "  'three',\n",
       "  'control',\n",
       "  'cell',\n",
       "  'lines',\n",
       "  '',\n",
       "  'and',\n",
       "  'five',\n",
       "  'patient',\n",
       "  'cell',\n",
       "  'lines',\n",
       "  '.',\n",
       "  'The',\n",
       "  'cells',\n",
       "  'will',\n",
       "  'be',\n",
       "  'seeded',\n",
       "  'in',\n",
       "  'normal',\n",
       "  'DMEM',\n",
       "  'medium',\n",
       "  '',\n",
       "  'and',\n",
       "  'the',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'lysis',\n",
       "  'would',\n",
       "  'be',\n",
       "  'prepared',\n",
       "  'exactly',\n",
       "  'after',\n",
       "  '24',\n",
       "  'hours',\n",
       "  'of',\n",
       "  'incubation',\n",
       "  'at',\n",
       "  '37',\n",
       "  'degrees',\n",
       "  '.'],\n",
       " ['day', '14', 'harvests', '.'],\n",
       " ['.',\n",
       "  '1.8',\n",
       "  'million',\n",
       "  '.',\n",
       "  '1732',\n",
       "  '.',\n",
       "  '2.2',\n",
       "  'million',\n",
       "  '.',\n",
       "  'Image',\n",
       "  '2',\n",
       "  '',\n",
       "  'one',\n",
       "  'point',\n",
       "  '5000000',\n",
       "  '.'],\n",
       " ['still', 'have', '10', 'samples', '.'],\n",
       " ['pipetted', '5', 'µl', '.'],\n",
       " ['solution',\n",
       "  'addition',\n",
       "  'completed',\n",
       "  'plate',\n",
       "  'is',\n",
       "  'incubated',\n",
       "  'at',\n",
       "  '450',\n",
       "  'rpm',\n",
       "  '.'],\n",
       " ['was',\n",
       "  'not',\n",
       "  'very',\n",
       "  'much',\n",
       "  'VES',\n",
       "  'and',\n",
       "  'other',\n",
       "  'arrhythmias',\n",
       "  'during',\n",
       "  'the',\n",
       "  'critical',\n",
       "  'time',\n",
       "  'point',\n",
       "  'of',\n",
       "  'the',\n",
       "  'occlusion',\n",
       "  '.',\n",
       "  'Possible',\n",
       "  'reasons',\n",
       "  'for',\n",
       "  'that',\n",
       "  'could',\n",
       "  'be',\n",
       "  'that',\n",
       "  'the',\n",
       "  'temperature',\n",
       "  'of',\n",
       "  'the',\n",
       "  'pig',\n",
       "  'is',\n",
       "  'quite',\n",
       "  'low',\n",
       "  '.',\n",
       "  '35.8',\n",
       "  '°',\n",
       "  '',\n",
       "  'and',\n",
       "  'or',\n",
       "  'maybe',\n",
       "  'that',\n",
       "  'there',\n",
       "  'is',\n",
       "  'a',\n",
       "  'leakage',\n",
       "  'along',\n",
       "  'the',\n",
       "  'balloon',\n",
       "  '',\n",
       "  'so',\n",
       "  'that',\n",
       "  'there',\n",
       "  'is',\n",
       "  'some',\n",
       "  'kind',\n",
       "  'of',\n",
       "  'blood',\n",
       "  'supply',\n",
       "  'maybe',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  '3rd',\n",
       "  'might',\n",
       "  'be',\n",
       "  'that',\n",
       "  'she',\n",
       "  'is',\n",
       "  'in',\n",
       "  'Bronst',\n",
       "  '.'],\n",
       " ['weight', 'is', '3.1', 'pg', '.'],\n",
       " ['8', 'dilutions', 'starting', 'at', '150', '.', 'Going', 'to', '0', '.'],\n",
       " ['5', '9.754', 'E4', '.'],\n",
       " ['me',\n",
       "  'introduce',\n",
       "  'this',\n",
       "  'person',\n",
       "  '.',\n",
       "  'He',\n",
       "  \"'s\",\n",
       "  'from',\n",
       "  'Discovery',\n",
       "  'Sciences',\n",
       "  'Department',\n",
       "  'with',\n",
       "  'Discovery',\n",
       "  'Biology',\n",
       "  'team',\n",
       "  '.',\n",
       "  'He',\n",
       "  'is',\n",
       "  'an',\n",
       "  'associate',\n",
       "  'principal',\n",
       "  'scientist',\n",
       "  'working',\n",
       "  'in',\n",
       "  'SC3',\n",
       "  '',\n",
       "  'SC4',\n",
       "  '',\n",
       "  'SC1',\n",
       "  'Lab',\n",
       "  '.'],\n",
       " ['power',\n",
       "  'reading',\n",
       "  '3.7',\n",
       "  '...',\n",
       "  'no',\n",
       "  'sorry',\n",
       "  '',\n",
       "  'maximum',\n",
       "  'power',\n",
       "  'reading',\n",
       "  'for',\n",
       "  'light',\n",
       "  'measurement',\n",
       "  '3.06',\n",
       "  'mW.',\n",
       "  'Diameter',\n",
       "  '200',\n",
       "  'micrometers',\n",
       "  '.',\n",
       "  'Producer',\n",
       "  ':',\n",
       "  'Thorlabs',\n",
       "  '.',\n",
       "  'Name',\n",
       "  ':',\n",
       "  'FG200UEA',\n",
       "  '-',\n",
       "  'Custom',\n",
       "  '-',\n",
       "  'Muke',\n",
       "  '?',\n",
       "  '.',\n",
       "  ':',\n",
       "  '.22',\n",
       "  ''],\n",
       " ['me', 'that', 'tonight', 'is', 'a', 'lab', 'meeting', 'at', '5', 'PM', '.'],\n",
       " ['set', 'the', 'water', 'bath', 'at', '212', '°', 'Fahrenheit', '.'],\n",
       " ['21.5',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'cold',\n",
       "  'water',\n",
       "  'to',\n",
       "  'the',\n",
       "  'solubilisation',\n",
       "  'mixture',\n",
       "  '.',\n",
       "  'Then',\n",
       "  'add',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'of',\n",
       "  '10',\n",
       "  'percent',\n",
       "  'filtered',\n",
       "  'LMNG',\n",
       "  '.',\n",
       "  'And',\n",
       "  'finally',\n",
       "  'mix',\n",
       "  'in',\n",
       "  'the',\n",
       "  'membrane',\n",
       "  'suspension',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'with',\n",
       "  'headphones',\n",
       "  'next',\n",
       "  'to',\n",
       "  'the',\n",
       "  'radio',\n",
       "  'holding',\n",
       "  'microphone',\n",
       "  'close',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'steered',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  '.'],\n",
       " ['taking',\n",
       "  'off',\n",
       "  'the',\n",
       "  'PFA',\n",
       "  'from',\n",
       "  'my',\n",
       "  'calbindin',\n",
       "  'DCA',\n",
       "  '10305',\n",
       "  'slices',\n",
       "  '.'],\n",
       " ['2',\n",
       "  '.',\n",
       "  'Cm',\n",
       "  '.',\n",
       "  '170',\n",
       "  '.',\n",
       "  'R',\n",
       "  '=',\n",
       "  '3.4',\n",
       "  '.',\n",
       "  'A',\n",
       "  'nice',\n",
       "  'recording',\n",
       "  'with',\n",
       "  'good',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  '.',\n",
       "  'NCX',\n",
       "  'and',\n",
       "  'caffeine',\n",
       "  'was',\n",
       "  'very',\n",
       "  'beautiful',\n",
       "  'and',\n",
       "  'stable',\n",
       "  '.',\n",
       "  'Also',\n",
       "  '',\n",
       "  'isoprenaline',\n",
       "  'was',\n",
       "  'attempted',\n",
       "  'with',\n",
       "  'nice',\n",
       "  'current',\n",
       "  '',\n",
       "  'although',\n",
       "  'the',\n",
       "  'series',\n",
       "  'resistance',\n",
       "  'was',\n",
       "  'increasing',\n",
       "  'probably',\n",
       "  'due',\n",
       "  'to',\n",
       "  'the',\n",
       "  'suction',\n",
       "  'being',\n",
       "  'too',\n",
       "  'high',\n",
       "  'and',\n",
       "  'the',\n",
       "  'volume',\n",
       "  'of',\n",
       "  'the',\n",
       "  'liquid',\n",
       "  'in',\n",
       "  'the',\n",
       "  'bathmaking',\n",
       "  'the',\n",
       "  'seal',\n",
       "  'unstable',\n",
       "  '.',\n",
       "  'However',\n",
       "  '',\n",
       "  'isoprenaline',\n",
       "  'and',\n",
       "  'Isoprenaline',\n",
       "  'load',\n",
       "  'was',\n",
       "  'also',\n",
       "  'seen',\n",
       "  'or',\n",
       "  'done',\n",
       "  'in',\n",
       "  'this',\n",
       "  'experiment',\n",
       "  '.'],\n",
       " ['so',\n",
       "  'that',\n",
       "  'totally',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'fix',\n",
       "  'APCH',\n",
       "  '7',\n",
       "  'comma',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'going',\n",
       "  'to',\n",
       "  'have',\n",
       "  'to',\n",
       "  'manually',\n",
       "  'do',\n",
       "  'that',\n",
       "  '1',\n",
       "  '.'],\n",
       " ['addition',\n",
       "  'to',\n",
       "  'the',\n",
       "  'creatinine',\n",
       "  'analysis',\n",
       "  'of',\n",
       "  'urine',\n",
       "  'samples',\n",
       "  'yesterday',\n",
       "  '',\n",
       "  'I',\n",
       "  'have',\n",
       "  'now',\n",
       "  'also',\n",
       "  'analyzed',\n",
       "  'plasma',\n",
       "  'creatinine',\n",
       "  'in',\n",
       "  'the',\n",
       "  'same',\n",
       "  'study',\n",
       "  '',\n",
       "  'meaning',\n",
       "  '27',\n",
       "  'samples',\n",
       "  '.',\n",
       "  'I',\n",
       "  'used',\n",
       "  '15',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'plasma',\n",
       "  'provided',\n",
       "  'by',\n",
       "  'Gun',\n",
       "  '-',\n",
       "  'Britt',\n",
       "  'Forsberg',\n",
       "  'and',\n",
       "  'used',\n",
       "  'the',\n",
       "  'same',\n",
       "  'method',\n",
       "  '',\n",
       "  'namely',\n",
       "  '15',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'plasma',\n",
       "  'and',\n",
       "  'then',\n",
       "  '200',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'internal',\n",
       "  'standard',\n",
       "  '.',\n",
       "  'And',\n",
       "  'I',\n",
       "  'added',\n",
       "  'these',\n",
       "  'set',\n",
       "  'of',\n",
       "  'samples',\n",
       "  'to',\n",
       "  'the',\n",
       "  'previous',\n",
       "  'sample',\n",
       "  'list',\n",
       "  'in',\n",
       "  'the',\n",
       "  'mass',\n",
       "  'spectrometer',\n",
       "  'using',\n",
       "  'the',\n",
       "  'previously',\n",
       "  'prepared',\n",
       "  'calibration',\n",
       "  'and',\n",
       "  'QC',\n",
       "  'samples',\n",
       "  'for',\n",
       "  'quantification',\n",
       "  '.'],\n",
       " ['.', '190', 'µl', 'PW', '1', 'to', 'dilution', '.'],\n",
       " ['just',\n",
       "  'have',\n",
       "  'to',\n",
       "  'design',\n",
       "  'a',\n",
       "  'sequence',\n",
       "  'with',\n",
       "  'T7',\n",
       "  '7',\n",
       "  'promoter',\n",
       "  'followed',\n",
       "  'by',\n",
       "  'DNA',\n",
       "  'sequence',\n",
       "  '.'],\n",
       " ['are',\n",
       "  'seeing',\n",
       "  '5',\n",
       "  'T2',\n",
       "  '25',\n",
       "  'layer',\n",
       "  'flask',\n",
       "  '.',\n",
       "  'With',\n",
       "  'about',\n",
       "  '10000000',\n",
       "  'cells',\n",
       "  'each',\n",
       "  'and',\n",
       "  '45',\n",
       "  'mLs',\n",
       "  'of',\n",
       "  'media',\n",
       "  '.'],\n",
       " ['is',\n",
       "  'where',\n",
       "  'tests',\n",
       "  'are',\n",
       "  'requested',\n",
       "  'by',\n",
       "  'chemists',\n",
       "  '.',\n",
       "  'This',\n",
       "  'point',\n",
       "  'integrates',\n",
       "  'the',\n",
       "  'tested',\n",
       "  'and',\n",
       "  'untested',\n",
       "  'assays',\n",
       "  'into',\n",
       "  '1',\n",
       "  'area',\n",
       "  '.'],\n",
       " ['be',\n",
       "  'the',\n",
       "  'NP40',\n",
       "  'buffer',\n",
       "  'the',\n",
       "  'problem',\n",
       "  '?',\n",
       "  'I',\n",
       "  'have',\n",
       "  'to',\n",
       "  'repeat',\n",
       "  'it',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'with',\n",
       "  'the',\n",
       "  'same',\n",
       "  'amount',\n",
       "  '60',\n",
       "  'µg',\n",
       "  'of',\n",
       "  'protein',\n",
       "  '.',\n",
       "  'And',\n",
       "  'I',\n",
       "  'try',\n",
       "  'to',\n",
       "  'devoid',\n",
       "  'the',\n",
       "  'smear',\n",
       "  '.'],\n",
       " ['the', 'detection', 'plates', 'for', '2', 'to', '3', 'min'],\n",
       " ['inspection',\n",
       "  '',\n",
       "  'prepare',\n",
       "  'the',\n",
       "  'DNA',\n",
       "  'by',\n",
       "  'mixing',\n",
       "  '0.1',\n",
       "  'µg',\n",
       "  'of',\n",
       "  'super',\n",
       "  'coiled',\n",
       "  'plasma',\n",
       "  'DNA',\n",
       "  'into',\n",
       "  'the',\n",
       "  'prepared',\n",
       "  'PBS',\n",
       "  'solution',\n",
       "  '.'],\n",
       " ['equal',\n",
       "  '0.1',\n",
       "  '.',\n",
       "  '5.2',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.1',\n",
       "  '.',\n",
       "  '5.3',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.1',\n",
       "  '.',\n",
       "  '5.4',\n",
       "  'equal',\n",
       "  '.',\n",
       "  '0.11',\n",
       "  '.',\n",
       "  '5.5',\n",
       "  'equal',\n",
       "  '.',\n",
       "  '0.096',\n",
       "  '.'],\n",
       " ['for', 'f', '*', '*', '*', 'sound', '2', \"o'clock\", '.'],\n",
       " ['ordered',\n",
       "  'some',\n",
       "  'lithium',\n",
       "  'chloride',\n",
       "  'and',\n",
       "  'some',\n",
       "  '2',\n",
       "  '3',\n",
       "  'benzene',\n",
       "  'chloride',\n",
       "  '.',\n",
       "  'That',\n",
       "  \"'s\",\n",
       "  'not',\n",
       "  'bad',\n",
       "  '.'],\n",
       " ['15', '-', '3', 'starts', 'on', '09442', '.'],\n",
       " ['08095',\n",
       "  '.',\n",
       "  'At',\n",
       "  'least',\n",
       "  '10',\n",
       "  '%',\n",
       "  'conversion',\n",
       "  'in',\n",
       "  '1',\n",
       "  'h.',\n",
       "  'Continue',\n",
       "  'stirring',\n",
       "  '.'],\n",
       " ['26',\n",
       "  '%',\n",
       "  'B',\n",
       "  '',\n",
       "  'changed',\n",
       "  'gradient',\n",
       "  'to',\n",
       "  '100',\n",
       "  '%',\n",
       "  'B',\n",
       "  'in',\n",
       "  '3',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['reducing',\n",
       "  'aspirate',\n",
       "  'to',\n",
       "  '20',\n",
       "  'eliminates',\n",
       "  'all',\n",
       "  'plate',\n",
       "  'patterns',\n",
       "  '.',\n",
       "  'We',\n",
       "  'just',\n",
       "  'need',\n",
       "  'to',\n",
       "  'check',\n",
       "  'whether',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'compatible',\n",
       "  'with',\n",
       "  'cells',\n",
       "  'and',\n",
       "  'whether',\n",
       "  'it',\n",
       "  'washes',\n",
       "  'them',\n",
       "  'off',\n",
       "  '.'],\n",
       " ['502', 'row', '1', 'column', 'A'],\n",
       " ['temperature',\n",
       "  'for',\n",
       "  'amplification',\n",
       "  'was',\n",
       "  'achieved',\n",
       "  'in',\n",
       "  'C1',\n",
       "  '',\n",
       "  'D1',\n",
       "  '',\n",
       "  'E1',\n",
       "  '',\n",
       "  'F1',\n",
       "  '',\n",
       "  'G1',\n",
       "  'and',\n",
       "  'H1',\n",
       "  '.'],\n",
       " ['in', 'batch', '3', 'are', 'unlikely', 'to', 'be', 'contaminated', '.'],\n",
       " ['plate',\n",
       "  'is',\n",
       "  'on',\n",
       "  'Orbital',\n",
       "  'Shaker',\n",
       "  'at',\n",
       "  '600',\n",
       "  'RPM',\n",
       "  '.',\n",
       "  'Approximately',\n",
       "  'will',\n",
       "  'be',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h',\n",
       "  'from',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['ml', 'of', 'solution', 'have', 'been', 'solved', '.'],\n",
       " ['fourth', 'TOC', 'value', 'is', '571', 'parts', 'per', 'billion', '.'],\n",
       " ['the',\n",
       "  'following',\n",
       "  'notes',\n",
       "  'to',\n",
       "  'experiment',\n",
       "  'BE0002668',\n",
       "  '-',\n",
       "  '33',\n",
       "  '.'],\n",
       " ['to', 'a', 'seminar', 'at', '5', 'pm', 'on', 'October', '29th', '.'],\n",
       " ['upload',\n",
       "  'my',\n",
       "  'results',\n",
       "  'to',\n",
       "  'the',\n",
       "  'D360',\n",
       "  '.',\n",
       "  'Here',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'difficult',\n",
       "  'to',\n",
       "  'find',\n",
       "  'the',\n",
       "  'information',\n",
       "  'from',\n",
       "  'the',\n",
       "  'legacy',\n",
       "  'experiments',\n",
       "  '.',\n",
       "  'I',\n",
       "  'rely',\n",
       "  'on',\n",
       "  'links',\n",
       "  'within',\n",
       "  'the',\n",
       "  'write',\n",
       "  'up',\n",
       "  'for',\n",
       "  'this',\n",
       "  '.'],\n",
       " ['ISF',\n",
       "  'cells',\n",
       "  'passage',\n",
       "  '13',\n",
       "  'measured',\n",
       "  'at',\n",
       "  '11',\n",
       "  'AM',\n",
       "  '.',\n",
       "  'Cells',\n",
       "  'in',\n",
       "  'total',\n",
       "  '3.8',\n",
       "  '*',\n",
       "  '10',\n",
       "  '^6',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '',\n",
       "  'viable',\n",
       "  'cells',\n",
       "  '3.77',\n",
       "  '.',\n",
       "  'Resulting',\n",
       "  'in',\n",
       "  'a',\n",
       "  'viability',\n",
       "  'of',\n",
       "  '99',\n",
       "  '%',\n",
       "  '',\n",
       "  'the',\n",
       "  'new',\n",
       "  'cell',\n",
       "  'concentration',\n",
       "  'will',\n",
       "  'set',\n",
       "  'to',\n",
       "  '1000000',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  'with',\n",
       "  '30',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'cell',\n",
       "  'suspension',\n",
       "  'and',\n",
       "  '37',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'medium',\n",
       "  '.'],\n",
       " ['nonreducing',\n",
       "  'samples',\n",
       "  'by',\n",
       "  'pipetting',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'and',\n",
       "  '6.7',\n",
       "  'µl',\n",
       "  '4',\n",
       "  'times',\n",
       "  'LDS',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['formate', 'buffer', 'pH', 'is', '7.12', '.'],\n",
       " ['C5',\n",
       "  'to',\n",
       "  'the',\n",
       "  'tube',\n",
       "  'concentration',\n",
       "  'tube',\n",
       "  'and',\n",
       "  'concentrate',\n",
       "  'again',\n",
       "  'for',\n",
       "  '5',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['a', '100', '.', 'Bye', '.'],\n",
       " ['the', 'experiment', 'DNA', 'research', 'G237', '.', 'DNA', 'research', '.'],\n",
       " ['temperature',\n",
       "  '920',\n",
       "  '37.3',\n",
       "  '.',\n",
       "  'It',\n",
       "  \"'s\",\n",
       "  'before',\n",
       "  'Blood',\n",
       "  'Gas',\n",
       "  '.'],\n",
       " ['%',\n",
       "  'of',\n",
       "  'my',\n",
       "  'work',\n",
       "  'is',\n",
       "  'bespoke',\n",
       "  'talk',\n",
       "  'studies',\n",
       "  '.',\n",
       "  '20',\n",
       "  '%',\n",
       "  'will',\n",
       "  'be',\n",
       "  'what',\n",
       "  'we',\n",
       "  'have',\n",
       "  'observed',\n",
       "  'today',\n",
       "  '.'],\n",
       " ['did', \"n't\", 'patent', '29', '.'],\n",
       " ['size', 'exclusion', 'at', '10', 'to', '4', '.'],\n",
       " [\"'s\",\n",
       "  'an',\n",
       "  'RKG',\n",
       "  'hemolin',\n",
       "  'perfusion',\n",
       "  '.',\n",
       "  'Replicate',\n",
       "  '3',\n",
       "  'blood',\n",
       "  'fat',\n",
       "  'mosquitoes',\n",
       "  '.',\n",
       "  'I',\n",
       "  'perfused',\n",
       "  '12',\n",
       "  'mosquitoes',\n",
       "  'in',\n",
       "  'total',\n",
       "  'and',\n",
       "  'dissected',\n",
       "  '5',\n",
       "  'guts',\n",
       "  '',\n",
       "  'the',\n",
       "  'foregut',\n",
       "  'and',\n",
       "  'the',\n",
       "  'hindgut',\n",
       "  '',\n",
       "  'hindgut',\n",
       "  '.'],\n",
       " ['a',\n",
       "  'list',\n",
       "  '.',\n",
       "  'Remind',\n",
       "  'me',\n",
       "  'that',\n",
       "  'I',\n",
       "  'have',\n",
       "  '3476',\n",
       "  'million',\n",
       "  'cells',\n",
       "  'for',\n",
       "  'sample',\n",
       "  '3',\n",
       "  '.'],\n",
       " ['we',\n",
       "  'go',\n",
       "  '.',\n",
       "  'Day',\n",
       "  '0',\n",
       "  'samples',\n",
       "  'labeled',\n",
       "  '79',\n",
       "  'and',\n",
       "  '84',\n",
       "  '.',\n",
       "  'PHH',\n",
       "  'followed',\n",
       "  'by',\n",
       "  'NPC',\n",
       "  'Day',\n",
       "  '7',\n",
       "  '81',\n",
       "  '',\n",
       "  '82',\n",
       "  'PHH',\n",
       "  'followed',\n",
       "  'by',\n",
       "  'NPC',\n",
       "  '.'],\n",
       " ['MPR', '3', 'immobilization', '5.5', '.'],\n",
       " ['with',\n",
       "  'difficult',\n",
       "  'words',\n",
       "  'writing',\n",
       "  'zone',\n",
       "  '',\n",
       "  'no',\n",
       "  'noise',\n",
       "  'microphone',\n",
       "  'close',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'have',\n",
       "  '12',\n",
       "  'IMR',\n",
       "  '32',\n",
       "  'vials',\n",
       "  '',\n",
       "  'Cryovials',\n",
       "  'and',\n",
       "  'I',\n",
       "  'put',\n",
       "  'them',\n",
       "  'in',\n",
       "  'my',\n",
       "  'box',\n",
       "  'for',\n",
       "  'the',\n",
       "  'liquid',\n",
       "  'nitrogen',\n",
       "  '.',\n",
       "  'In',\n",
       "  'row',\n",
       "  '6',\n",
       "  '61',\n",
       "  'through',\n",
       "  '9',\n",
       "  'and',\n",
       "  'then',\n",
       "  'row',\n",
       "  '7',\n",
       "  '133',\n",
       "  '.'],\n",
       " ['sample', '2', '203000', '.'],\n",
       " ['viability',\n",
       "  '',\n",
       "  '1.92',\n",
       "  'number',\n",
       "  'of',\n",
       "  'cells',\n",
       "  '1732',\n",
       "  '.',\n",
       "  '95',\n",
       "  '%',\n",
       "  'viability',\n",
       "  '',\n",
       "  '1.98',\n",
       "  'cells',\n",
       "  'images',\n",
       "  '2',\n",
       "  '.'],\n",
       " ['stop',\n",
       "  'equals',\n",
       "  '16',\n",
       "  'µl',\n",
       "  'per',\n",
       "  '6',\n",
       "  'ml',\n",
       "  'for',\n",
       "  'a',\n",
       "  '4',\n",
       "  'x',\n",
       "  'stock',\n",
       "  'concentration',\n",
       "  '.'],\n",
       " ['primary', 'antibody', 'to', 'Western', 'blot', 'after', '2', 'h.'],\n",
       " ['',\n",
       "  'the',\n",
       "  'last',\n",
       "  'results',\n",
       "  'are',\n",
       "  'more',\n",
       "  'and',\n",
       "  'less',\n",
       "  'comfortable',\n",
       "  '',\n",
       "  'but',\n",
       "  'ionic',\n",
       "  'stimulation',\n",
       "  'to',\n",
       "  'be',\n",
       "  'changed',\n",
       "  'our',\n",
       "  'race',\n",
       "  'from',\n",
       "  '200',\n",
       "  'to',\n",
       "  '400',\n",
       "  'in',\n",
       "  'a',\n",
       "  'seconds',\n",
       "  'and',\n",
       "  'here',\n",
       "  'I',\n",
       "  'guess',\n",
       "  'we',\n",
       "  'also',\n",
       "  'have',\n",
       "  'to',\n",
       "  'extend',\n",
       "  'the',\n",
       "  'surface',\n",
       "  '.',\n",
       "  'Maybe',\n",
       "  'just',\n",
       "  'check',\n",
       "  'the',\n",
       "  'surface',\n",
       "  'shape',\n",
       "  'in',\n",
       "  'comparison',\n",
       "  'to',\n",
       "  'the',\n",
       "  'surface',\n",
       "  'of',\n",
       "  'this',\n",
       "  'special',\n",
       "  'amino',\n",
       "  'acid',\n",
       "  '.',\n",
       "  'And',\n",
       "  'also',\n",
       "  'just',\n",
       "  'divide',\n",
       "  'this',\n",
       "  'protein',\n",
       "  'into',\n",
       "  '2',\n",
       "  'halfs',\n",
       "  'and',\n",
       "  'then',\n",
       "  'simulate',\n",
       "  'per',\n",
       "  'each',\n",
       "  'half',\n",
       "  '.',\n",
       "  'This',\n",
       "  'would',\n",
       "  'be',\n",
       "  'a',\n",
       "  'great',\n",
       "  'idea',\n",
       "  '.'],\n",
       " ['596', 'µl', 'of', 'hyb', 'buffer', '.'],\n",
       " ['20', 'mililiter'],\n",
       " ['that',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'give',\n",
       "  'Sue',\n",
       "  'the',\n",
       "  'DCAS9',\n",
       "  'krab',\n",
       "  'MCherry',\n",
       "  'Vector',\n",
       "  '.'],\n",
       " ['will',\n",
       "  'harvest',\n",
       "  'all',\n",
       "  'HFFAX',\n",
       "  '172',\n",
       "  'cells',\n",
       "  'with',\n",
       "  'all',\n",
       "  'dilution',\n",
       "  '.',\n",
       "  'But',\n",
       "  'for',\n",
       "  'the',\n",
       "  'HFFAX',\n",
       "  '173',\n",
       "  '',\n",
       "  '174',\n",
       "  'and',\n",
       "  '175',\n",
       "  'just',\n",
       "  'the',\n",
       "  'dilutions',\n",
       "  '1',\n",
       "  'in',\n",
       "  '25',\n",
       "  'and',\n",
       "  '1',\n",
       "  'in',\n",
       "  '50',\n",
       "  '.'],\n",
       " ['44',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'Light',\n",
       "  'Glutathione',\n",
       "  '.',\n",
       "  '200',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'the',\n",
       "  'the',\n",
       "  '1X',\n",
       "  'of',\n",
       "  'the',\n",
       "  'heavy',\n",
       "  'internal',\n",
       "  'standard',\n",
       "  'mix',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'tell',\n",
       "  'the',\n",
       "  'following',\n",
       "  '.',\n",
       "  '1',\n",
       "  'the',\n",
       "  'tasks',\n",
       "  'if',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'a',\n",
       "  'reminder',\n",
       "  'do',\n",
       "  \"n't\",\n",
       "  'work',\n",
       "  '.',\n",
       "  '2',\n",
       "  '.',\n",
       "  'Need',\n",
       "  'to',\n",
       "  'add',\n",
       "  'notebook',\n",
       "  'pages',\n",
       "  '.',\n",
       "  '3',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'number',\n",
       "  '7',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'New',\n",
       "  'paragraph',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'assay',\n",
       "  'we',\n",
       "  'optimized',\n",
       "  'the',\n",
       "  'set',\n",
       "  'up',\n",
       "  '.',\n",
       "  'Everything',\n",
       "  'that',\n",
       "  'is',\n",
       "  'written',\n",
       "  'in',\n",
       "  'the',\n",
       "  'protocol',\n",
       "  'will',\n",
       "  'be',\n",
       "  'fine',\n",
       "  '.',\n",
       "  'But',\n",
       "  'today',\n",
       "  '',\n",
       "  'we',\n",
       "  'want',\n",
       "  'to',\n",
       "  'measure',\n",
       "  'cytokines',\n",
       "  'from',\n",
       "  'the',\n",
       "  'same',\n",
       "  'wells',\n",
       "  'that',\n",
       "  'were',\n",
       "  'measuring',\n",
       "  'of',\n",
       "  'the',\n",
       "  'viability',\n",
       "  'assay',\n",
       "  '.',\n",
       "  'So',\n",
       "  'what',\n",
       "  'we',\n",
       "  'did',\n",
       "  'was',\n",
       "  'we',\n",
       "  'took',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'culture',\n",
       "  'and',\n",
       "  'transferred',\n",
       "  'this',\n",
       "  'over',\n",
       "  'into',\n",
       "  'a',\n",
       "  '96',\n",
       "  'well',\n",
       "  'V',\n",
       "  '-',\n",
       "  'bottom',\n",
       "  'plate',\n",
       "  'and',\n",
       "  'we',\n",
       "  'added',\n",
       "  'the',\n",
       "  'correct',\n",
       "  'amount',\n",
       "  'of',\n",
       "  'membrane',\n",
       "  'permeability',\n",
       "  'as',\n",
       "  'well',\n",
       "  'as',\n",
       "  'caspase',\n",
       "  'activity',\n",
       "  '',\n",
       "  'dye',\n",
       "  'into',\n",
       "  'this',\n",
       "  'well',\n",
       "  'in',\n",
       "  'a',\n",
       "  'formula',\n",
       "  'of',\n",
       "  'one',\n",
       "  'to',\n",
       "  'one',\n",
       "  'dilution',\n",
       "  '.',\n",
       "  'So',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'culture',\n",
       "  '+2',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'um',\n",
       "  '',\n",
       "  'reagent',\n",
       "  'that',\n",
       "  'we',\n",
       "  'use',\n",
       "  'in',\n",
       "  'this',\n",
       "  'case',\n",
       "  '.',\n",
       "  'The',\n",
       "  'reagent',\n",
       "  'mass',\n",
       "  'was',\n",
       "  'set',\n",
       "  'up',\n",
       "  'in',\n",
       "  'a',\n",
       "  'way',\n",
       "  'that',\n",
       "  'we',\n",
       "  'use',\n",
       "  '2470',\n",
       "  'µl',\n",
       "  'media',\n",
       "  'we',\n",
       "  'added',\n",
       "  '25',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'cell',\n",
       "  'permeability',\n",
       "  'dye',\n",
       "  'and',\n",
       "  '5',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'Caspase',\n",
       "  'activity',\n",
       "  'stain',\n",
       "  '.',\n",
       "  'This',\n",
       "  'was',\n",
       "  'incubated',\n",
       "  'in',\n",
       "  'the',\n",
       "  'incubator',\n",
       "  'for',\n",
       "  'one',\n",
       "  'h',\n",
       "  'at',\n",
       "  '37',\n",
       "  '°',\n",
       "  'and',\n",
       "  'in',\n",
       "  'order',\n",
       "  'to',\n",
       "  'stretch',\n",
       "  'the',\n",
       "  'time',\n",
       "  'between',\n",
       "  'reading',\n",
       "  'and',\n",
       "  'measuring',\n",
       "  'with',\n",
       "  'the',\n",
       "  'attendees',\n",
       "  '',\n",
       "  'we',\n",
       "  'stored',\n",
       "  'them',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'in',\n",
       "  'the',\n",
       "  'dark',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'out',\n",
       "  'the',\n",
       "  'plate',\n",
       "  'samples',\n",
       "  'at',\n",
       "  '8:00',\n",
       "  'AM',\n",
       "  'tomorrow',\n",
       "  '.'],\n",
       " ['cells',\n",
       "  'in',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'phosphorus',\n",
       "  'safe',\n",
       "  'lysis',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['this',\n",
       "  'new',\n",
       "  'components',\n",
       "  'are',\n",
       "  'being',\n",
       "  'added',\n",
       "  'to',\n",
       "  'the',\n",
       "  'formulation',\n",
       "  '.',\n",
       "  'In',\n",
       "  'this',\n",
       "  'case',\n",
       "  '',\n",
       "  'we',\n",
       "  'want',\n",
       "  'to',\n",
       "  'use',\n",
       "  'three',\n",
       "  'components',\n",
       "  'in',\n",
       "  'our',\n",
       "  'new',\n",
       "  'formulation',\n",
       "  '.',\n",
       "  'We',\n",
       "  'then',\n",
       "  'define',\n",
       "  'the',\n",
       "  'type',\n",
       "  'of',\n",
       "  'component',\n",
       "  'from',\n",
       "  'the',\n",
       "  'drop',\n",
       "  'down',\n",
       "  'menu',\n",
       "  '.',\n",
       "  'In',\n",
       "  'this',\n",
       "  'case',\n",
       "  '',\n",
       "  'we',\n",
       "  \"'re\",\n",
       "  'using',\n",
       "  'an',\n",
       "  'additive',\n",
       "  'a',\n",
       "  'solvent',\n",
       "  'and',\n",
       "  'a',\n",
       "  'particle',\n",
       "  '.',\n",
       "  'Then',\n",
       "  'we',\n",
       "  'identify',\n",
       "  'the',\n",
       "  'specific',\n",
       "  'components',\n",
       "  'to',\n",
       "  'be',\n",
       "  'used',\n",
       "  'from',\n",
       "  'the',\n",
       "  'inventory',\n",
       "  'management',\n",
       "  'module',\n",
       "  '.',\n",
       "  'Here',\n",
       "  'we',\n",
       "  \"'re\",\n",
       "  'using',\n",
       "  'cyclohexanol',\n",
       "  'MEP0035',\n",
       "  'and',\n",
       "  'lower',\n",
       "  '11',\n",
       "  'K',\n",
       "  'and',\n",
       "  'we',\n",
       "  'defined',\n",
       "  'the',\n",
       "  'formulation',\n",
       "  'specific',\n",
       "  'weight',\n",
       "  'percentage',\n",
       "  '.',\n",
       "  'The',\n",
       "  'total',\n",
       "  'percentage',\n",
       "  'is',\n",
       "  'being',\n",
       "  'calculated',\n",
       "  'and',\n",
       "  'needs',\n",
       "  'to',\n",
       "  'be',\n",
       "  '100',\n",
       "  '%',\n",
       "  '.'],\n",
       " ['20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'and',\n",
       "  '6.7',\n",
       "  '.',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'LDS',\n",
       "  'buffer',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'prepared',\n",
       "  'without',\n",
       "  'heating',\n",
       "  'and',\n",
       "  'run',\n",
       "  'on',\n",
       "  'a',\n",
       "  'buy',\n",
       "  'Orajel',\n",
       "  '.'],\n",
       " ['ID',\n",
       "  '18S006290',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'value',\n",
       "  '3.705',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'sample',\n",
       "  'ID',\n",
       "  '19S00460',\n",
       "  'sample',\n",
       "  'value',\n",
       "  '2.00',\n",
       "  'ng/µl',\n",
       "  'sample',\n",
       "  'ID',\n",
       "  '18S006286',\n",
       "  'sample',\n",
       "  'value',\n",
       "  '16.80',\n",
       "  'nG/µl',\n",
       "  '.'],\n",
       " ['ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  'green',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'the',\n",
       "  'normal',\n",
       "  'transduction',\n",
       "  'without',\n",
       "  'spinfection',\n",
       "  '.',\n",
       "  'And',\n",
       "  'I',\n",
       "  'ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  'green',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'the',\n",
       "  'wells',\n",
       "  '',\n",
       "  'which',\n",
       "  'were',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'on',\n",
       "  'ice',\n",
       "  'before',\n",
       "  '.',\n",
       "  'But',\n",
       "  'in',\n",
       "  'the',\n",
       "  'normal',\n",
       "  'protocol',\n",
       "  'for',\n",
       "  'spinfection',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h',\n",
       "  '',\n",
       "  'I',\n",
       "  'can',\n",
       "  'see',\n",
       "  'some',\n",
       "  'a',\n",
       "  'lot',\n",
       "  'of',\n",
       "  'green',\n",
       "  'cells',\n",
       "  'after',\n",
       "  '26',\n",
       "  'h',\n",
       "  'in',\n",
       "  'the',\n",
       "  'well',\n",
       "  'with',\n",
       "  'polybrene',\n",
       "  '.',\n",
       "  'I',\n",
       "  'would',\n",
       "  'say',\n",
       "  'I',\n",
       "  'can',\n",
       "  'see',\n",
       "  'round',\n",
       "  'about',\n",
       "  '10',\n",
       "  '%',\n",
       "  'green',\n",
       "  'cells',\n",
       "  '',\n",
       "  '5',\n",
       "  '',\n",
       "  'between',\n",
       "  '5',\n",
       "  'and',\n",
       "  '10',\n",
       "  '%',\n",
       "  'in',\n",
       "  'the',\n",
       "  'well',\n",
       "  'without',\n",
       "  'polybrene',\n",
       "  '.',\n",
       "  'I',\n",
       "  'can',\n",
       "  '',\n",
       "  'I',\n",
       "  'can',\n",
       "  'see',\n",
       "  'some',\n",
       "  'green',\n",
       "  'cells',\n",
       "  'but',\n",
       "  'more',\n",
       "  'and',\n",
       "  'less',\n",
       "  'as',\n",
       "  'in',\n",
       "  'the',\n",
       "  'well',\n",
       "  'with',\n",
       "  'polybrene',\n",
       "  '.',\n",
       "  'I',\n",
       "  'can',\n",
       "  'see',\n",
       "  'just',\n",
       "  'one',\n",
       "  'or',\n",
       "  '2',\n",
       "  'green',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'The',\n",
       "  'control',\n",
       "  'well',\n",
       "  'is',\n",
       "  'ok',\n",
       "  '',\n",
       "  'I',\n",
       "  'ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  'green',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'Now',\n",
       "  'I',\n",
       "  'will',\n",
       "  'check',\n",
       "  'the',\n",
       "  'spinfection',\n",
       "  'as',\n",
       "  'in',\n",
       "  'pellet',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h.',\n",
       "  'And',\n",
       "  'I',\n",
       "  'can',\n",
       "  'see',\n",
       "  'green',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ['', 'the', 'experiment', 'is', 'BS', '17', '.'],\n",
       " ['thinks',\n",
       "  'that',\n",
       "  'the',\n",
       "  '500',\n",
       "  'µl',\n",
       "  'is',\n",
       "  'too',\n",
       "  'dilute',\n",
       "  'he',\n",
       "  'could',\n",
       "  'not',\n",
       "  'see',\n",
       "  'a',\n",
       "  'film',\n",
       "  'optically',\n",
       "  'with',\n",
       "  'his',\n",
       "  'eye',\n",
       "  '.',\n",
       "  'He',\n",
       "  'believes',\n",
       "  'the',\n",
       "  'threshold',\n",
       "  'should',\n",
       "  'be',\n",
       "  'approximately',\n",
       "  'so',\n",
       "  'we',\n",
       "  'I',\n",
       "  'have',\n",
       "  'prepared',\n",
       "  'new',\n",
       "  'stocks',\n",
       "  'with',\n",
       "  'POBC',\n",
       "  '.'],\n",
       " ['volume', 'is', '1.0035', 'nl', '.'],\n",
       " ['me', 'to', 'pay', 'rent', 'at', '5:00', 'PM', 'tomorrow', '.'],\n",
       " ['group', '3', 'first', '.'],\n",
       " ['650', 'out', 'of', 'PE', 'looks', 'weird', '.'],\n",
       " ['up',\n",
       "  'the',\n",
       "  'Cas9',\n",
       "  'protein',\n",
       "  'from',\n",
       "  'the',\n",
       "  '-80',\n",
       "  'freezer',\n",
       "  'in',\n",
       "  'Tray',\n",
       "  'C2',\n",
       "  '.'],\n",
       " ['cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  '08',\n",
       "  'model',\n",
       "  'study',\n",
       "  '19',\n",
       "  '091',\n",
       "  '.',\n",
       "  'Medium',\n",
       "  'intensity',\n",
       "  'nuclear',\n",
       "  'staining',\n",
       "  'throughout',\n",
       "  'an',\n",
       "  'entire',\n",
       "  'tumor',\n",
       "  'tissue',\n",
       "  '.',\n",
       "  'Some',\n",
       "  'areas',\n",
       "  'of',\n",
       "  'no',\n",
       "  'staining',\n",
       "  '.'],\n",
       " ['denaturation', '50', 'µl', '200', 'mM', 'Tris', 'to', 'denaturation', '.'],\n",
       " ['44', '8.106', 'E', 'to', 'the', '4', '.'],\n",
       " ['A2', '1.11', 'e5', '.'],\n",
       " ['7', '432000', '.'],\n",
       " ['1.25',\n",
       "  'ml',\n",
       "  'imidazole',\n",
       "  'from',\n",
       "  '1',\n",
       "  'molar',\n",
       "  'stock',\n",
       "  'pH',\n",
       "  '7.5',\n",
       "  'to',\n",
       "  'the',\n",
       "  'lysis',\n",
       "  'buffer',\n",
       "  'to',\n",
       "  'read',\n",
       "  '5',\n",
       "  'mM',\n",
       "  'imidazole',\n",
       "  '.'],\n",
       " ['me', 'to', 'check', 'on', 'sample', '#', '3', '.'],\n",
       " ['concentrations',\n",
       "  '.',\n",
       "  'I',\n",
       "  'have',\n",
       "  '12',\n",
       "  'samples',\n",
       "  '.',\n",
       "  'Always',\n",
       "  'agree',\n",
       "  '',\n",
       "  'Elisa',\n",
       "  '.',\n",
       "  'Elisa',\n",
       "  '',\n",
       "  'every',\n",
       "  'morning',\n",
       "  'I',\n",
       "  'have',\n",
       "  'bring',\n",
       "  'the',\n",
       "  'ice',\n",
       "  'packet',\n",
       "  'full',\n",
       "  'ice',\n",
       "  'to',\n",
       "  'you',\n",
       "  '',\n",
       "  'every',\n",
       "  'morning',\n",
       "  '.',\n",
       "  'Saving',\n",
       "  'now',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'testing',\n",
       "  'it',\n",
       "  '.',\n",
       "  'How',\n",
       "  'are',\n",
       "  'you',\n",
       "  '?',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'trying',\n",
       "  '.',\n",
       "  'How',\n",
       "  'are',\n",
       "  'you',\n",
       "  'trying',\n",
       "  '?'],\n",
       " ['export',\n",
       "  'things',\n",
       "  'like',\n",
       "  'the',\n",
       "  'kinetic',\n",
       "  'affinity',\n",
       "  'and',\n",
       "  'the',\n",
       "  'binding',\n",
       "  'data',\n",
       "  '.',\n",
       "  'This',\n",
       "  'table',\n",
       "  'goes',\n",
       "  'into',\n",
       "  'the',\n",
       "  'D360',\n",
       "  'and',\n",
       "  'my',\n",
       "  'write',\n",
       "  'up',\n",
       "  '.'],\n",
       " ['ID',\n",
       "  '18s00068',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'value',\n",
       "  '3',\n",
       "  'dot',\n",
       "  '705',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'ID',\n",
       "  '18s0089',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'value',\n",
       "  '3.705',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'ID',\n",
       "  '19s0098',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'value',\n",
       "  '3.50',\n",
       "  'ng/µl',\n",
       "  '.'],\n",
       " ['10',\n",
       "  '%',\n",
       "  'TFA',\n",
       "  '1',\n",
       "  'µl',\n",
       "  'is',\n",
       "  'added',\n",
       "  'to',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'to',\n",
       "  'give',\n",
       "  'a',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '0.5',\n",
       "  '%',\n",
       "  'TFA',\n",
       "  'in',\n",
       "  'samples',\n",
       "  'to',\n",
       "  'stop',\n",
       "  'trypsination',\n",
       "  '.'],\n",
       " ['note', 'a', 'change', 'in', 'temperature', 'from', '27', '.'],\n",
       " ['30', '1.613x10', 'to', 'the', '5'],\n",
       " ['started', 'at', '4:10', '.'],\n",
       " ['cells',\n",
       "  'are',\n",
       "  'at',\n",
       "  '85.6',\n",
       "  'percent',\n",
       "  'viability',\n",
       "  'and',\n",
       "  'their',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  '1.71',\n",
       "  '.',\n",
       "  'And',\n",
       "  'before',\n",
       "  'cells',\n",
       "  'are',\n",
       "  'at',\n",
       "  '93.3',\n",
       "  'percent',\n",
       "  'viability',\n",
       "  'their',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  '2.10',\n",
       "  '.',\n",
       "  'OCI',\n",
       "  'cells',\n",
       "  'are',\n",
       "  'at',\n",
       "  '97.8',\n",
       "  'percent',\n",
       "  'viability',\n",
       "  'in',\n",
       "  'their',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  '2.43',\n",
       "  '.'],\n",
       " ['µl', 'RSB', 'all', '8', 'lanes', '.'],\n",
       " ['plasmid', '.', '7', '.'],\n",
       " ['incubated', 'at', '42', '°', 'for', '5', 'min', '.'],\n",
       " ['reduction',\n",
       "  'of',\n",
       "  'the',\n",
       "  'aspirate',\n",
       "  'to',\n",
       "  '25',\n",
       "  'immensely',\n",
       "  'improves',\n",
       "  'plate',\n",
       "  'pattern',\n",
       "  '.',\n",
       "  'The',\n",
       "  'only',\n",
       "  'wells',\n",
       "  'left',\n",
       "  'are',\n",
       "  'now',\n",
       "  'B',\n",
       "  '23',\n",
       "  'and',\n",
       "  '24',\n",
       "  'and',\n",
       "  'C',\n",
       "  '23',\n",
       "  'and',\n",
       "  '24',\n",
       "  '.'],\n",
       " ['4',\n",
       "  'number',\n",
       "  'sample',\n",
       "  'number',\n",
       "  '13',\n",
       "  '',\n",
       "  'E2',\n",
       "  '',\n",
       "  'E3',\n",
       "  '',\n",
       "  'E4',\n",
       "  '',\n",
       "  'E5',\n",
       "  '',\n",
       "  'and',\n",
       "  'E6',\n",
       "  '.'],\n",
       " ['step',\n",
       "  'is',\n",
       "  'to',\n",
       "  'add',\n",
       "  '200',\n",
       "  'µl',\n",
       "  'of',\n",
       "  '0.1',\n",
       "  '%',\n",
       "  'no',\n",
       "  '',\n",
       "  '0.5',\n",
       "  '%',\n",
       "  'TFA',\n",
       "  'in',\n",
       "  '5',\n",
       "  '%',\n",
       "  'acetonitrile',\n",
       "  'and',\n",
       "  'centrifuge',\n",
       "  'at',\n",
       "  '1500',\n",
       "  'RCF',\n",
       "  'for',\n",
       "  '1',\n",
       "  'min',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  '.',\n",
       "  'This',\n",
       "  'step',\n",
       "  'is',\n",
       "  'repeated',\n",
       "  'twice',\n",
       "  '.'],\n",
       " ['change', 'in', 'CO2', '.', 'Let', \"'s\", 'start', 'occlusion'],\n",
       " ['me', '', 'I', 'have', 'a', 'meeting', 'with', 'Dennis', 'at', '1330', '.'],\n",
       " ['D2', '-', '3', 'images', 'end', '09402', '.'],\n",
       " ['started', 'incubation', 'for', 'plate', '1', 'now', '.'],\n",
       " ['Prop',\n",
       "  '2',\n",
       "  'turn',\n",
       "  'out',\n",
       "  'to',\n",
       "  'be',\n",
       "  'the',\n",
       "  'best',\n",
       "  'probability',\n",
       "  'of',\n",
       "  'the',\n",
       "  'events',\n",
       "  '.',\n",
       "  'In',\n",
       "  'control',\n",
       "  'Def',\n",
       "  'Prop',\n",
       "  '3',\n",
       "  '',\n",
       "  'the',\n",
       "  'events',\n",
       "  'probability',\n",
       "  'of',\n",
       "  'infected',\n",
       "  'so',\n",
       "  'I',\n",
       "  'need',\n",
       "  'now',\n",
       "  'to',\n",
       "  'control',\n",
       "  'this',\n",
       "  '2',\n",
       "  'compared',\n",
       "  'with',\n",
       "  '2',\n",
       "  'different',\n",
       "  'health',\n",
       "  'functions',\n",
       "  '.',\n",
       "  'The',\n",
       "  'one',\n",
       "  'that',\n",
       "  'I',\n",
       "  'already',\n",
       "  'have',\n",
       "  '.',\n",
       "  'And',\n",
       "  'another',\n",
       "  'one',\n",
       "  'that',\n",
       "  'includes',\n",
       "  'a',\n",
       "  'longer',\n",
       "  'lifespan',\n",
       "  'and',\n",
       "  'with',\n",
       "  'the',\n",
       "  'Orange',\n",
       "  'curve',\n",
       "  'that',\n",
       "  'I',\n",
       "  'sealed',\n",
       "  'in',\n",
       "  'a',\n",
       "  'where',\n",
       "  'I',\n",
       "  'got',\n",
       "  'it',\n",
       "  'from',\n",
       "  '.',\n",
       "  'But',\n",
       "  'it',\n",
       "  'gives',\n",
       "  'me',\n",
       "  'a',\n",
       "  'much',\n",
       "  'shorter',\n",
       "  'lifespan',\n",
       "  '.',\n",
       "  'I',\n",
       "  'expect',\n",
       "  'that',\n",
       "  'in',\n",
       "  'the',\n",
       "  'Orange',\n",
       "  '.',\n",
       "  'There',\n",
       "  'should',\n",
       "  'not',\n",
       "  'be',\n",
       "  'an',\n",
       "  'infection',\n",
       "  'and',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'important',\n",
       "  'now',\n",
       "  'to',\n",
       "  'compare',\n",
       "  'the',\n",
       "  'event',\n",
       "  'probability',\n",
       "  'of',\n",
       "  'infected',\n",
       "  'versus',\n",
       "  'controlled',\n",
       "  'because',\n",
       "  'they',\n",
       "  \"'re\",\n",
       "  '.',\n",
       "  'There',\n",
       "  'might',\n",
       "  'be',\n",
       "  'a',\n",
       "  'little',\n",
       "  'bit',\n",
       "  'of',\n",
       "  '.',\n",
       "  'Differences',\n",
       "  'um',\n",
       "  '.',\n",
       "  'Maybe',\n",
       "  'I',\n",
       "  'said',\n",
       "  '',\n",
       "  'also',\n",
       "  'the',\n",
       "  'red',\n",
       "  'curve',\n",
       "  '',\n",
       "  'um',\n",
       "  'just',\n",
       "  'as',\n",
       "  'an',\n",
       "  'extra',\n",
       "  'test',\n",
       "  '',\n",
       "  'but',\n",
       "  'I',\n",
       "  'do',\n",
       "  \"n't\",\n",
       "  'think',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'very',\n",
       "  'well',\n",
       "  '.',\n",
       "  'Really',\n",
       "  '',\n",
       "  'necessary',\n",
       "  '',\n",
       "  'so',\n",
       "  'set',\n",
       "  'up',\n",
       "  'a',\n",
       "  'list',\n",
       "  'of',\n",
       "  'events',\n",
       "  'probability',\n",
       "  'infected',\n",
       "  'control',\n",
       "  'to',\n",
       "  'heal',\n",
       "  'functions',\n",
       "  'the',\n",
       "  'one',\n",
       "  'that',\n",
       "  'I',\n",
       "  'have',\n",
       "  'the',\n",
       "  'longer',\n",
       "  'one',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  'Orange',\n",
       "  'curve',\n",
       "  'and',\n",
       "  'for',\n",
       "  'each',\n",
       "  'of',\n",
       "  'these',\n",
       "  'different',\n",
       "  'death',\n",
       "  'rates',\n",
       "  '',\n",
       "  'then',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'make',\n",
       "  '5',\n",
       "  'to',\n",
       "  '10',\n",
       "  'simulations',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'in',\n",
       "  '30',\n",
       "  'min',\n",
       "  'to',\n",
       "  'bring',\n",
       "  'out',\n",
       "  'the',\n",
       "  'week',\n",
       "  'planning',\n",
       "  'protocol',\n",
       "  '.',\n",
       "  'Print',\n",
       "  'out',\n",
       "  'the',\n",
       "  'week',\n",
       "  'planning',\n",
       "  'protocol',\n",
       "  '.'],\n",
       " ['15',\n",
       "  'ml',\n",
       "  '5',\n",
       "  'molar',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  'to',\n",
       "  'lysis',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['remembered',\n",
       "  'that',\n",
       "  'the',\n",
       "  'cider',\n",
       "  'green',\n",
       "  'mix',\n",
       "  'was',\n",
       "  'not',\n",
       "  'supposed',\n",
       "  '',\n",
       "  'there',\n",
       "  'was',\n",
       "  'no',\n",
       "  'miscalculation',\n",
       "  'on',\n",
       "  'the',\n",
       "  '5',\n",
       "  \"o'clock\",\n",
       "  'mix',\n",
       "  '?',\n",
       "  'Bumblebee',\n",
       "  '.'],\n",
       " ['reminder',\n",
       "  'in',\n",
       "  '20',\n",
       "  'min',\n",
       "  '.',\n",
       "  'Give',\n",
       "  'peptides',\n",
       "  'to',\n",
       "  '[',\n",
       "  '?',\n",
       "  ']',\n",
       "  '.'],\n",
       " ['12',\n",
       "  '',\n",
       "  '13',\n",
       "  'and',\n",
       "  '14',\n",
       "  'will',\n",
       "  'be',\n",
       "  'pooled',\n",
       "  'and',\n",
       "  'fractions',\n",
       "  '15',\n",
       "  '',\n",
       "  '16',\n",
       "  'and',\n",
       "  '17',\n",
       "  'will',\n",
       "  'be',\n",
       "  'pooled',\n",
       "  '.'],\n",
       " ['have',\n",
       "  'pipetted',\n",
       "  '5µl',\n",
       "  'into',\n",
       "  'Falcon',\n",
       "  'tube',\n",
       "  '.',\n",
       "  'I',\n",
       "  'want',\n",
       "  'to',\n",
       "  '.'],\n",
       " ['jabra',\n",
       "  'radio',\n",
       "  'German',\n",
       "  'in',\n",
       "  'the',\n",
       "  'background',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  'The',\n",
       "  'BI',\n",
       "  'phasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'still',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transfer',\n",
       "  'it',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'follow',\n",
       "  '.',\n",
       "  'Face',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'organic',\n",
       "  'lawyer',\n",
       "  'was',\n",
       "  'separated',\n",
       "  'and',\n",
       "  'washed',\n",
       "  'once',\n",
       "  'with',\n",
       "  '2',\n",
       "  'ml',\n",
       "  'bicarb',\n",
       "  'solution',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'road',\n",
       "  'about',\n",
       "  'rotavap',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['µl',\n",
       "  'of',\n",
       "  'protein',\n",
       "  'at',\n",
       "  'different',\n",
       "  'concentrations',\n",
       "  'was',\n",
       "  'added',\n",
       "  'to',\n",
       "  'plate',\n",
       "  '.'],\n",
       " ['4', 'is', 'being', 'verified', 'using', '2.5', 'µl', '6', 'shots', '.'],\n",
       " ['20', 'mix', 'to', 'vial', '.', 'Is', 'it', 'working', '?'],\n",
       " ['close',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '12',\n",
       "  'bar',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['1st',\n",
       "  'lane',\n",
       "  'of',\n",
       "  'the',\n",
       "  'gel',\n",
       "  'is',\n",
       "  'filled',\n",
       "  'with',\n",
       "  'the',\n",
       "  'selection',\n",
       "  'plate',\n",
       "  'and',\n",
       "  'the',\n",
       "  '2nd',\n",
       "  'lane',\n",
       "  'is',\n",
       "  'filled',\n",
       "  'with',\n",
       "  'L1A',\n",
       "  '',\n",
       "  'L1B',\n",
       "  '',\n",
       "  'L1C',\n",
       "  '',\n",
       "  'RFP',\n",
       "  'and',\n",
       "  'after',\n",
       "  'that',\n",
       "  'comes',\n",
       "  'the',\n",
       "  'selection',\n",
       "  'plate',\n",
       "  'again',\n",
       "  '.'],\n",
       " ['blue', 'plates', '', '1', 'min', '', '1000', 'rpm'],\n",
       " ['18', '1.915'],\n",
       " ['I',\n",
       "  \"'m\",\n",
       "  'adding',\n",
       "  '30',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'reagent',\n",
       "  'A',\n",
       "  'into',\n",
       "  'well',\n",
       "  'B6',\n",
       "  '.'],\n",
       " ['.',\n",
       "  '0.29',\n",
       "  '.',\n",
       "  '4.2',\n",
       "  '.',\n",
       "  '0.30',\n",
       "  '.',\n",
       "  '4.3',\n",
       "  '.',\n",
       "  '0.27',\n",
       "  '.',\n",
       "  '4.4',\n",
       "  '.',\n",
       "  '0.28',\n",
       "  '.',\n",
       "  '4.5',\n",
       "  '.',\n",
       "  '0.3',\n",
       "  '.'],\n",
       " ['1', '[', 'ladder', '?', ']', '.', 'Second', '.', 'Point', '.'],\n",
       " ['23', '2.0752', 'E5', '.'],\n",
       " ['timers',\n",
       "  'have',\n",
       "  'gone',\n",
       "  'off',\n",
       "  '.',\n",
       "  'Begin',\n",
       "  'dispensing',\n",
       "  '.',\n",
       "  '45',\n",
       "  'µl',\n",
       "  'per',\n",
       "  'well',\n",
       "  'to',\n",
       "  'a',\n",
       "  'new',\n",
       "  'clear',\n",
       "  'plate',\n",
       "  'for',\n",
       "  'the',\n",
       "  'QUANTI',\n",
       "  '-',\n",
       "  'Blue',\n",
       "  'reagent',\n",
       "  '',\n",
       "  'dispensing',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['timers',\n",
       "  'have',\n",
       "  'gone',\n",
       "  'off',\n",
       "  '.',\n",
       "  'Begin',\n",
       "  'dispensing',\n",
       "  '.',\n",
       "  '45',\n",
       "  'µl',\n",
       "  'per',\n",
       "  'well',\n",
       "  'to',\n",
       "  'a',\n",
       "  'new',\n",
       "  'clear',\n",
       "  'plate',\n",
       "  'for',\n",
       "  'the',\n",
       "  'contact',\n",
       "  'blue',\n",
       "  'reagent',\n",
       "  '',\n",
       "  'dispensing',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['46', 'µl', 'leaders', 'of', 'Phi', 'X', 'into', 'dilution', '.'],\n",
       " ['cell', 'line', 'CHODG', '44', '.'],\n",
       " ['formate',\n",
       "  'is',\n",
       "  'normally',\n",
       "  '9.45',\n",
       "  'g',\n",
       "  '/',\n",
       "  'l',\n",
       "  'times',\n",
       "  '1.5',\n",
       "  '.',\n",
       "  'It',\n",
       "  'is',\n",
       "  '.'],\n",
       " ['seems',\n",
       "  'to',\n",
       "  'be',\n",
       "  'the',\n",
       "  'optimal',\n",
       "  'setting',\n",
       "  '.',\n",
       "  'We',\n",
       "  \"'re\",\n",
       "  'not',\n",
       "  'losing',\n",
       "  'any',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'Strikes',\n",
       "  'a',\n",
       "  'balance',\n",
       "  'between',\n",
       "  'good',\n",
       "  'wash',\n",
       "  'no',\n",
       "  'plate',\n",
       "  'effects',\n",
       "  'in',\n",
       "  'terms',\n",
       "  'of',\n",
       "  'media',\n",
       "  '.',\n",
       "  'And',\n",
       "  'cells',\n",
       "  'wash',\n",
       "  'ideally',\n",
       "  'I',\n",
       "  'wanted',\n",
       "  'to',\n",
       "  'be',\n",
       "  'lower',\n",
       "  '.'],\n",
       " ['330',\n",
       "  '',\n",
       "  'neural',\n",
       "  '-',\n",
       "  'D1',\n",
       "  'Medi',\n",
       "  'small',\n",
       "  'cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  'model',\n",
       "  '5',\n",
       "  '.',\n",
       "  'Pockets',\n",
       "  'of',\n",
       "  'nuclear',\n",
       "  'expression',\n",
       "  'throughout',\n",
       "  'the',\n",
       "  'tumor',\n",
       "  'tissue',\n",
       "  '',\n",
       "  'but',\n",
       "  'to',\n",
       "  'a',\n",
       "  'much',\n",
       "  'lesser',\n",
       "  'degree',\n",
       "  'than',\n",
       "  'the',\n",
       "  'last',\n",
       "  'slide',\n",
       "  'that',\n",
       "  'I',\n",
       "  'reviewed',\n",
       "  'which',\n",
       "  'was',\n",
       "  'small',\n",
       "  'cell',\n",
       "  'lung',\n",
       "  'cancer',\n",
       "  '02',\n",
       "  'model',\n",
       "  '.',\n",
       "  'Some',\n",
       "  'cytoplasmic',\n",
       "  'staining',\n",
       "  'also',\n",
       "  '.'],\n",
       " ['130', 'µl', 'into', 'lane', '2', '.'],\n",
       " ['the',\n",
       "  'maximum',\n",
       "  'and',\n",
       "  'minimum',\n",
       "  'of',\n",
       "  'experiment',\n",
       "  '5',\n",
       "  '',\n",
       "  'there',\n",
       "  'are',\n",
       "  '2',\n",
       "  'maximums',\n",
       "  'because',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'was',\n",
       "  \"n't\",\n",
       "  'completely',\n",
       "  'induced',\n",
       "  'after',\n",
       "  'killing',\n",
       "  'the',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'But',\n",
       "  'you',\n",
       "  'should',\n",
       "  'try',\n",
       "  'analyzing',\n",
       "  'the',\n",
       "  '2',\n",
       "  'peaks',\n",
       "  'to',\n",
       "  'see',\n",
       "  'whether',\n",
       "  'fits',\n",
       "  'with',\n",
       "  'them',\n",
       "  'similar',\n",
       "  'experiments',\n",
       "  'because',\n",
       "  '.',\n",
       "  'The',\n",
       "  'pipette',\n",
       "  'might',\n",
       "  'contribute',\n",
       "  'to',\n",
       "  'the',\n",
       "  'fluorescence',\n",
       "  '.'],\n",
       " ['ID', '76', 'also', 'added', 'to', 'T9', '.'],\n",
       " ['serial',\n",
       "  'dilution',\n",
       "  'was',\n",
       "  'done',\n",
       "  'with',\n",
       "  '200',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'each',\n",
       "  'concentration',\n",
       "  '.'],\n",
       " ['samples',\n",
       "  'EGDE',\n",
       "  '1485',\n",
       "  '',\n",
       "  '1020',\n",
       "  '',\n",
       "  '1042',\n",
       "  '',\n",
       "  '1019',\n",
       "  '',\n",
       "  '1033',\n",
       "  '',\n",
       "  '1018',\n",
       "  '',\n",
       "  '3009',\n",
       "  '',\n",
       "  '2011',\n",
       "  '',\n",
       "  '2012',\n",
       "  '',\n",
       "  '1034',\n",
       "  'of',\n",
       "  'listeria',\n",
       "  'for',\n",
       "  'listeria',\n",
       "  'monocytogenes',\n",
       "  'detection',\n",
       "  'system',\n",
       "  'tomorrow',\n",
       "  '.'],\n",
       " ['me', 'to', 'feed', 'the', 'cells', 'Wednesday', 'at', '11', 'AM', '.'],\n",
       " ['protein',\n",
       "  'PMS',\n",
       "  '2',\n",
       "  '-',\n",
       "  '12',\n",
       "  '.',\n",
       "  'Evaluation',\n",
       "  '.',\n",
       "  'Elution',\n",
       "  'program',\n",
       "  '135',\n",
       "  'to',\n",
       "  '210',\n",
       "  '3',\n",
       "  'ml',\n",
       "  'each',\n",
       "  '.',\n",
       "  'Pooling',\n",
       "  'fraction',\n",
       "  '16',\n",
       "  'to',\n",
       "  'fraction',\n",
       "  '40',\n",
       "  '',\n",
       "  'pooling',\n",
       "  'and',\n",
       "  'loading',\n",
       "  '2',\n",
       "  'Q',\n",
       "  'Sepharose',\n",
       "  'Column',\n",
       "  '.'],\n",
       " ['11:50', 'restart', 'elution', '.'],\n",
       " ['19', '2.660', 'E5', '.'],\n",
       " ['next', 'measurement', 'is', '[', 'Tukat', '?', ']', '25', '.'],\n",
       " ['the',\n",
       "  'SEC',\n",
       "  'run',\n",
       "  'the',\n",
       "  'fractions',\n",
       "  'of',\n",
       "  'B3',\n",
       "  '',\n",
       "  'B1',\n",
       "  '',\n",
       "  'C2',\n",
       "  '',\n",
       "  'C4',\n",
       "  '',\n",
       "  'C6',\n",
       "  '',\n",
       "  'C8',\n",
       "  '',\n",
       "  'D3',\n",
       "  'and',\n",
       "  'E4',\n",
       "  'were',\n",
       "  'tested',\n",
       "  'on',\n",
       "  'SDS',\n",
       "  'PAGE',\n",
       "  '.'],\n",
       " ['32', '', '1.963', 'E5', '.'],\n",
       " ['the',\n",
       "  '3rd',\n",
       "  'plate',\n",
       "  'as',\n",
       "  'the',\n",
       "  'first',\n",
       "  'plate',\n",
       "  'to',\n",
       "  'dispense',\n",
       "  'with',\n",
       "  'the',\n",
       "  'multidrop',\n",
       "  '',\n",
       "  'to'],\n",
       " ['volume', '70', 'µl', '.'],\n",
       " ['would', 'make', '3', 'l', 'of', 'buffer', '.'],\n",
       " ['695',\n",
       "  'neuro',\n",
       "  'D1',\n",
       "  'with',\n",
       "  'human',\n",
       "  'ovary',\n",
       "  'TBST',\n",
       "  '1',\n",
       "  'to',\n",
       "  '50',\n",
       "  '.'],\n",
       " ['preparing',\n",
       "  'the',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  'buffer',\n",
       "  'weigh',\n",
       "  '29.2',\n",
       "  'g',\n",
       "  'of',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  'from',\n",
       "  'a',\n",
       "  'new',\n",
       "  'bottle',\n",
       "  '.',\n",
       "  'Merck',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  '.'],\n",
       " ['a', 'temperature', 'change', 'from', '22', '°', 'to', '25', '°', '.'],\n",
       " ['it',\n",
       "  \"'s\",\n",
       "  'working',\n",
       "  '.',\n",
       "  'Now',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'starting',\n",
       "  'up',\n",
       "  'to',\n",
       "  '5',\n",
       "  'min',\n",
       "  '',\n",
       "  '2',\n",
       "  'ml',\n",
       "  'on',\n",
       "  'the',\n",
       "  'first',\n",
       "  'condition',\n",
       "  'for',\n",
       "  'the',\n",
       "  'first',\n",
       "  'condition',\n",
       "  'and',\n",
       "  'OK',\n",
       "  '',\n",
       "  'then',\n",
       "  'when',\n",
       "  'it',\n",
       "  'starts',\n",
       "  'with',\n",
       "  'a',\n",
       "  'wake',\n",
       "  'word',\n",
       "  '.'],\n",
       " ['of',\n",
       "  'membrane',\n",
       "  'prep',\n",
       "  '3',\n",
       "  'correct',\n",
       "  'it',\n",
       "  'membrane',\n",
       "  'prep',\n",
       "  '8',\n",
       "  '.'],\n",
       " ['50',\n",
       "  'ml',\n",
       "  'solubilization',\n",
       "  'buffer',\n",
       "  'by',\n",
       "  'taking',\n",
       "  '27.5',\n",
       "  'ml',\n",
       "  '2',\n",
       "  'times',\n",
       "  'solubilization',\n",
       "  'buffer',\n",
       "  '27.5',\n",
       "  'is',\n",
       "  'to',\n",
       "  'compensate',\n",
       "  'for',\n",
       "  'the',\n",
       "  'slight',\n",
       "  'extra',\n",
       "  'dilution',\n",
       "  '',\n",
       "  '1',\n",
       "  'tablet',\n",
       "  'of',\n",
       "  'protease',\n",
       "  'inhibitor',\n",
       "  '',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'of',\n",
       "  '10',\n",
       "  '%',\n",
       "  'LMNG',\n",
       "  '',\n",
       "  'and',\n",
       "  'fill',\n",
       "  'up',\n",
       "  'with',\n",
       "  'Milli',\n",
       "  'Q',\n",
       "  'Water',\n",
       "  'until',\n",
       "  '50',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['D', '', '3', 'and', '3', 'µl', '.'],\n",
       " ['project', 'PMS', '6', '1', 'agarose', 'gel', '.'],\n",
       " ['codes',\n",
       "  'ending',\n",
       "  'on',\n",
       "  '0112',\n",
       "  'and',\n",
       "  'barcode',\n",
       "  'ending',\n",
       "  'on',\n",
       "  '027',\n",
       "  'are',\n",
       "  'outlier',\n",
       "  'plates',\n",
       "  '.',\n",
       "  'Do',\n",
       "  'not',\n",
       "  'include',\n",
       "  'those',\n",
       "  'in',\n",
       "  'the',\n",
       "  'analysis',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'cell',\n",
       "  'wash',\n",
       "  'travel',\n",
       "  'rate',\n",
       "  'to',\n",
       "  '1',\n",
       "  'does',\n",
       "  'not',\n",
       "  'appear',\n",
       "  'to',\n",
       "  'change',\n",
       "  'the',\n",
       "  'pattern',\n",
       "  'of',\n",
       "  'the',\n",
       "  'washer',\n",
       "  '.'],\n",
       " ['a',\n",
       "  'quality',\n",
       "  'control',\n",
       "  'sample',\n",
       "  'for',\n",
       "  'acoustic',\n",
       "  'mass',\n",
       "  'spectrometry',\n",
       "  '.',\n",
       "  'Mass',\n",
       "  'spectrometer',\n",
       "  'from',\n",
       "  'HepG2',\n",
       "  'C3A',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  'doing',\n",
       "  'a',\n",
       "  'wash',\n",
       "  'with',\n",
       "  '150',\n",
       "  'mM',\n",
       "  'ammonium',\n",
       "  'formate',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Wash',\n",
       "  'number',\n",
       "  '1',\n",
       "  '.'],\n",
       " ['µl', 'TLRG', 'into', 'lane', '1', '.'],\n",
       " ['number',\n",
       "  '15',\n",
       "  '.',\n",
       "  'Concentration',\n",
       "  '170',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Fraction',\n",
       "  'A',\n",
       "  '1.87',\n",
       "  '.',\n",
       "  'Fraction',\n",
       "  'B',\n",
       "  '2.39',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'on',\n",
       "  'Monday',\n",
       "  'at',\n",
       "  '9:00',\n",
       "  \"o'clock\",\n",
       "  'to',\n",
       "  'run',\n",
       "  'a',\n",
       "  'SDS',\n",
       "  '-',\n",
       "  'PAGE',\n",
       "  'of',\n",
       "  'the',\n",
       "  'pooled',\n",
       "  'samples',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'go',\n",
       "  'to',\n",
       "  'my',\n",
       "  'colonoscopy',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '8:30',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['upload',\n",
       "  'my',\n",
       "  'results',\n",
       "  'to',\n",
       "  'the',\n",
       "  'D360',\n",
       "  '.',\n",
       "  'Here',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'difficult',\n",
       "  'to',\n",
       "  'find',\n",
       "  'the',\n",
       "  'information',\n",
       "  'from',\n",
       "  'legacy',\n",
       "  'experiments',\n",
       "  '.',\n",
       "  'I',\n",
       "  'rely',\n",
       "  'on',\n",
       "  'links',\n",
       "  'within',\n",
       "  'the',\n",
       "  'write',\n",
       "  'up',\n",
       "  'for',\n",
       "  'this',\n",
       "  '.'],\n",
       " ['me', 'after', '5', 'min', '.'],\n",
       " ['plate', '1', 'was', 'plated', 'at', '3:10', 'AM', '.', 'Yeah', '.'],\n",
       " ['number', '8', '.', 'Concentration', '209', '.'],\n",
       " ['me', 'in', '30', 'min', 'incubation', 'is', 'finished', '.'],\n",
       " ['PB17', '0692', 'with', 'date', '2017', '0530', '.'],\n",
       " ['of',\n",
       "  'things',\n",
       "  'in',\n",
       "  'the',\n",
       "  'gel',\n",
       "  '.',\n",
       "  'Point',\n",
       "  '1',\n",
       "  '.',\n",
       "  'jill',\n",
       "  '.',\n",
       "  'Jill'],\n",
       " ['Five',\n",
       "  'cells',\n",
       "  'passage',\n",
       "  '13',\n",
       "  'measured',\n",
       "  'at',\n",
       "  '11:30',\n",
       "  '',\n",
       "  'at',\n",
       "  '11:30',\n",
       "  '.'],\n",
       " ['were',\n",
       "  'measuring',\n",
       "  'the',\n",
       "  'optical',\n",
       "  'density',\n",
       "  'of',\n",
       "  'cyanobacteria',\n",
       "  'and',\n",
       "  'their',\n",
       "  'adsorption',\n",
       "  'was',\n",
       "  'at',\n",
       "  '0.23',\n",
       "  '.'],\n",
       " ['BJAB', 'cells', 'X', '127', '.'],\n",
       " ['the',\n",
       "  'colony',\n",
       "  'PCR',\n",
       "  'here',\n",
       "  'we',\n",
       "  'are',\n",
       "  'choosing',\n",
       "  '10',\n",
       "  'new',\n",
       "  'colonies',\n",
       "  'of',\n",
       "  'our',\n",
       "  'spectrum',\n",
       "  'esteem',\n",
       "  'plates',\n",
       "  '.'],\n",
       " ['100', 'nM'],\n",
       " ['analysis', 'of', 'Cas9', 'in', 'vivo', 'samples', 'storage', '.'],\n",
       " ['49', '', 'position', 'A7', '', 'maybe', 'a', 'small', 'crystal', '.'],\n",
       " ['of',\n",
       "  'more',\n",
       "  'UPF2',\n",
       "  'to',\n",
       "  'the',\n",
       "  'reaction',\n",
       "  '.',\n",
       "  'would',\n",
       "  'it',\n",
       "  'effect',\n",
       "  '?'],\n",
       " ['for', '10', 'min', '.'],\n",
       " ['for',\n",
       "  'the',\n",
       "  'LPS',\n",
       "  'Assay',\n",
       "  'the',\n",
       "  'number',\n",
       "  'of',\n",
       "  'reactions',\n",
       "  'for',\n",
       "  'the',\n",
       "  'master',\n",
       "  'mix',\n",
       "  'has',\n",
       "  'to',\n",
       "  'be',\n",
       "  'exactly',\n",
       "  'that',\n",
       "  'of',\n",
       "  'the',\n",
       "  'one',\n",
       "  'is',\n",
       "  'stated',\n",
       "  'on',\n",
       "  'the',\n",
       "  'protocol',\n",
       "  'because',\n",
       "  'that',\n",
       "  'one',\n",
       "  'already',\n",
       "  'has',\n",
       "  'extra',\n",
       "  'reactions',\n",
       "  'for',\n",
       "  'pipetting',\n",
       "  'error',\n",
       "  '.'],\n",
       " ['culture',\n",
       "  '25',\n",
       "  'ml',\n",
       "  'pipette',\n",
       "  '6',\n",
       "  'bags',\n",
       "  '',\n",
       "  'Cell',\n",
       "  'culture',\n",
       "  '10',\n",
       "  'ml',\n",
       "  'pipettes',\n",
       "  '15',\n",
       "  'bags',\n",
       "  '',\n",
       "  'cell',\n",
       "  'culture',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'pipette',\n",
       "  '6',\n",
       "  'bags',\n",
       "  '',\n",
       "  'General',\n",
       "  'pipettes',\n",
       "  '5',\n",
       "  'ml',\n",
       "  '20',\n",
       "  'bags',\n",
       "  '',\n",
       "  'General',\n",
       "  'pipettes',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '12',\n",
       "  'bags',\n",
       "  '',\n",
       "  'General',\n",
       "  'pipettes',\n",
       "  '10',\n",
       "  'ml',\n",
       "  '30',\n",
       "  'bags',\n",
       "  '.'],\n",
       " ['pipette',\n",
       "  'my',\n",
       "  'volume',\n",
       "  '',\n",
       "  'I',\n",
       "  'use',\n",
       "  'the',\n",
       "  'PAL',\n",
       "  'pipette',\n",
       "  'pro',\n",
       "  'and',\n",
       "  'a',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'stripette',\n",
       "  'to',\n",
       "  'pipette',\n",
       "  'my',\n",
       "  '5000',\n",
       "  'volume',\n",
       "  '.',\n",
       "  'To',\n",
       "  'pipette',\n",
       "  'the',\n",
       "  'I',\n",
       "  \"'ve\",\n",
       "  'pipetted',\n",
       "  '4000',\n",
       "  'µl',\n",
       "  'of',\n",
       "  '4',\n",
       "  'milliters',\n",
       "  'and',\n",
       "  'to',\n",
       "  'pipette',\n",
       "  'the',\n",
       "  'remaining',\n",
       "  'amount',\n",
       "  'I',\n",
       "  'did',\n",
       "  'the',\n",
       "  'P1000C',\n",
       "  'blue',\n",
       "  '',\n",
       "  'serial',\n",
       "  'number',\n",
       "  'R59658C',\n",
       "  'calculation',\n",
       "  'number',\n",
       "  'DOB9URYCEZ',\n",
       "  '.'],\n",
       " ['105', 'µl', 'X', 'AMP', 'into', 'lane', '4', '.'],\n",
       " ['37', '', '5.965', 'E', '4', '.'],\n",
       " ['phospholipidosis',\n",
       "  'assay',\n",
       "  'take',\n",
       "  '10',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'the',\n",
       "  'stock',\n",
       "  'cell',\n",
       "  'suspension',\n",
       "  'and',\n",
       "  'add',\n",
       "  '148.57',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'media',\n",
       "  '.',\n",
       "  'For',\n",
       "  'the',\n",
       "  'acoustic',\n",
       "  'mass',\n",
       "  'spec',\n",
       "  'assay',\n",
       "  'take',\n",
       "  '6',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'suspension',\n",
       "  'and',\n",
       "  'add',\n",
       "  '120.86',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'media',\n",
       "  '.'],\n",
       " ['to',\n",
       "  'self',\n",
       "  'figure',\n",
       "  'out',\n",
       "  'what',\n",
       "  'to',\n",
       "  'do',\n",
       "  'about',\n",
       "  'like',\n",
       "  'yielding',\n",
       "  'TIA1',\n",
       "  '',\n",
       "  'TIA1',\n",
       "  'exogenous',\n",
       "  'overexpression'],\n",
       " ['buffer', '20', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'do',\n",
       "  'the',\n",
       "  'treatment',\n",
       "  'at',\n",
       "  '9',\n",
       "  'and',\n",
       "  '10',\n",
       "  \"o'clock\",\n",
       "  '.',\n",
       "  'And',\n",
       "  'also',\n",
       "  'to',\n",
       "  'prepare',\n",
       "  'Fred',\n",
       "  'buffer',\n",
       "  'and',\n",
       "  'H2O2',\n",
       "  'dilutions',\n",
       "  '.'],\n",
       " ['stop',\n",
       "  'equals',\n",
       "  '16',\n",
       "  'µl',\n",
       "  'per',\n",
       "  '6',\n",
       "  'ml',\n",
       "  'for',\n",
       "  'a',\n",
       "  '4',\n",
       "  'x',\n",
       "  'stock',\n",
       "  'concentration',\n",
       "  '.'],\n",
       " ['µl', 'pool', '20', 'into', 'lane', '1', '.'],\n",
       " ['worst',\n",
       "  'thing',\n",
       "  'is',\n",
       "  'the',\n",
       "  'slowness',\n",
       "  'of',\n",
       "  'the',\n",
       "  'app',\n",
       "  'itself',\n",
       "  '.',\n",
       "  'The',\n",
       "  'completing',\n",
       "  'and',\n",
       "  'closing',\n",
       "  'and',\n",
       "  'converting',\n",
       "  'it',\n",
       "  'to',\n",
       "  'a',\n",
       "  'PDF',\n",
       "  'takes',\n",
       "  'around',\n",
       "  '10',\n",
       "  'min',\n",
       "  'alone',\n",
       "  '.'],\n",
       " ['of',\n",
       "  'the',\n",
       "  'aspirate',\n",
       "  'Z',\n",
       "  'offset',\n",
       "  'eliminated',\n",
       "  'plate',\n",
       "  'pattern',\n",
       "  'completely',\n",
       "  '.',\n",
       "  'However',\n",
       "  '',\n",
       "  'may',\n",
       "  'be',\n",
       "  'dangerous',\n",
       "  'territory',\n",
       "  'for',\n",
       "  'getting',\n",
       "  'a',\n",
       "  'good',\n",
       "  'cell',\n",
       "  'wash',\n",
       "  '.',\n",
       "  'We',\n",
       "  'need',\n",
       "  'to',\n",
       "  'investigate',\n",
       "  'how',\n",
       "  'much',\n",
       "  'further',\n",
       "  'we',\n",
       "  'can',\n",
       "  'increase',\n",
       "  'it',\n",
       "  'so',\n",
       "  'as',\n",
       "  'not',\n",
       "  'to',\n",
       "  'have',\n",
       "  'a',\n",
       "  'plate',\n",
       "  'pattern',\n",
       "  '',\n",
       "  'but',\n",
       "  'not',\n",
       "  'to',\n",
       "  'wash',\n",
       "  'out',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'either',\n",
       "  '.',\n",
       "  'Going',\n",
       "  'back',\n",
       "  'to',\n",
       "  '30',\n",
       "  '.'],\n",
       " ['added',\n",
       "  '2',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'TFA',\n",
       "  'to',\n",
       "  'my',\n",
       "  'protein',\n",
       "  'prep',\n",
       "  '.',\n",
       "  'And',\n",
       "  'my',\n",
       "  'protein',\n",
       "  'precipitated',\n",
       "  '.',\n",
       "  'And',\n",
       "  'I',\n",
       "  'spun',\n",
       "  'it',\n",
       "  'down',\n",
       "  'and',\n",
       "  'examine',\n",
       "  'the',\n",
       "  'supernatant',\n",
       "  '.',\n",
       "  'The',\n",
       "  'supernatant',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'have',\n",
       "  'any',\n",
       "  'protein',\n",
       "  '.',\n",
       "  'I',\n",
       "  'was',\n",
       "  'very',\n",
       "  'angry',\n",
       "  'and',\n",
       "  'went',\n",
       "  'home',\n",
       "  '.'],\n",
       " ['will',\n",
       "  'seed',\n",
       "  '11',\n",
       "  'flasks',\n",
       "  'of',\n",
       "  'fibroblasts',\n",
       "  'today',\n",
       "  '.',\n",
       "  'Probably',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'density',\n",
       "  'of',\n",
       "  'around',\n",
       "  '4.5',\n",
       "  'million',\n",
       "  'cells',\n",
       "  'per',\n",
       "  '225',\n",
       "  '.'],\n",
       " ['marker',\n",
       "  '1',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'Marker',\n",
       "  '6',\n",
       "  'µl',\n",
       "  'buffer',\n",
       "  'protein',\n",
       "  'marker',\n",
       "  '2',\n",
       "  '9',\n",
       "  'µl',\n",
       "  'marker',\n",
       "  '3',\n",
       "  'µl',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['new', 'notebook', 'entry', '001', '.'],\n",
       " ['220',\n",
       "  'µl',\n",
       "  'to',\n",
       "  'tubes',\n",
       "  '1',\n",
       "  'through',\n",
       "  '12',\n",
       "  '.',\n",
       "  'Good',\n",
       "  'morning',\n",
       "  '',\n",
       "  'Nami',\n",
       "  '.'],\n",
       " ['for', '3', 'min', '.'],\n",
       " ['10', 'µl', 'of', 'LysoPI'],\n",
       " ['test',\n",
       "  'experiment',\n",
       "  '',\n",
       "  'CAL',\n",
       "  '25',\n",
       "  '',\n",
       "  'with',\n",
       "  'the',\n",
       "  'same',\n",
       "  'amplitudes',\n",
       "  'as',\n",
       "  'before',\n",
       "  '.'],\n",
       " ['residual',\n",
       "  'softener',\n",
       "  'itself',\n",
       "  'smells',\n",
       "  'as',\n",
       "  'well',\n",
       "  '.',\n",
       "  'The',\n",
       "  'whole',\n",
       "  'plastic',\n",
       "  'part',\n",
       "  'from',\n",
       "  'the',\n",
       "  'outside',\n",
       "  'is',\n",
       "  'cheap',\n",
       "  '',\n",
       "  'really',\n",
       "  'cheap',\n",
       "  'probably',\n",
       "  'sounds',\n",
       "  'polyethylene',\n",
       "  'or',\n",
       "  'maybe',\n",
       "  'polycarbonate',\n",
       "  'but',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'even',\n",
       "  'look',\n",
       "  'glass',\n",
       "  'fiber',\n",
       "  'reinforced',\n",
       "  'and',\n",
       "  'makes',\n",
       "  'a',\n",
       "  'quite',\n",
       "  'soft',\n",
       "  'impression',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  'smell',\n",
       "  'is',\n",
       "  'also',\n",
       "  'very',\n",
       "  'prominent',\n",
       "  '.',\n",
       "  'The',\n",
       "  'display',\n",
       "  'is',\n",
       "  'somehow',\n",
       "  'movable',\n",
       "  'but',\n",
       "  'when',\n",
       "  'you',\n",
       "  'turn',\n",
       "  'the',\n",
       "  'display',\n",
       "  'you',\n",
       "  'move',\n",
       "  'the',\n",
       "  'part',\n",
       "  'inside',\n",
       "  'the',\n",
       "  'tank',\n",
       "  'as',\n",
       "  'well',\n",
       "  'the',\n",
       "  'white',\n",
       "  'filter',\n",
       "  'is',\n",
       "  'what',\n",
       "  'I',\n",
       "  'mean',\n",
       "  'by',\n",
       "  'part',\n",
       "  '.',\n",
       "  'The',\n",
       "  'Box',\n",
       "  'is',\n",
       "  'made',\n",
       "  'out',\n",
       "  'of',\n",
       "  'a',\n",
       "  'gray',\n",
       "  'material',\n",
       "  'and',\n",
       "  'the',\n",
       "  'material',\n",
       "  'is',\n",
       "  'the',\n",
       "  'same',\n",
       "  'inside',\n",
       "  'the',\n",
       "  'box',\n",
       "  '.',\n",
       "  'So',\n",
       "  'the',\n",
       "  'tank',\n",
       "  'is',\n",
       "  'made',\n",
       "  'out',\n",
       "  'of',\n",
       "  'Polycarbonate',\n",
       "  'or',\n",
       "  'something',\n",
       "  'like',\n",
       "  'that',\n",
       "  '.',\n",
       "  'On',\n",
       "  'the',\n",
       "  'backside',\n",
       "  'they',\n",
       "  'just',\n",
       "  'drilled',\n",
       "  'a',\n",
       "  'hole',\n",
       "  'in',\n",
       "  'the',\n",
       "  'tank',\n",
       "  'and',\n",
       "  'pushed',\n",
       "  'a',\n",
       "  'tube',\n",
       "  'through',\n",
       "  '.',\n",
       "  'I',\n",
       "  'ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  'if',\n",
       "  'there',\n",
       "  \"'s\",\n",
       "  'anything',\n",
       "  'like',\n",
       "  'a',\n",
       "  'seal',\n",
       "  'or',\n",
       "  'something',\n",
       "  'like',\n",
       "  'that',\n",
       "  '',\n",
       "  'and',\n",
       "  'it',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'even',\n",
       "  'look',\n",
       "  'properly',\n",
       "  'drilled',\n",
       "  '.',\n",
       "  'And',\n",
       "  'that',\n",
       "  'is',\n",
       "  'the',\n",
       "  'transparent',\n",
       "  'tube',\n",
       "  'that',\n",
       "  'goes',\n",
       "  'into',\n",
       "  'the',\n",
       "  'top',\n",
       "  'unit',\n",
       "  '.',\n",
       "  'Well',\n",
       "  '',\n",
       "  'Yeah',\n",
       "  '.',\n",
       "  'Um',\n",
       "  '.',\n",
       "  'The',\n",
       "  'power',\n",
       "  'unit',\n",
       "  'is',\n",
       "  'stated',\n",
       "  'with',\n",
       "  '102',\n",
       "  '.'],\n",
       " ['cells',\n",
       "  'in',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'phosphorus',\n",
       "  'safe',\n",
       "  'lysis',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['-',\n",
       "  '3',\n",
       "  'has',\n",
       "  'minor',\n",
       "  'cell',\n",
       "  'wrapping',\n",
       "  'around',\n",
       "  'the',\n",
       "  'entry',\n",
       "  'channels',\n",
       "  'dibbles',\n",
       "  'are',\n",
       "  'present',\n",
       "  'in',\n",
       "  'the',\n",
       "  'chip',\n",
       "  '.',\n",
       "  'Fluid',\n",
       "  'flow',\n",
       "  'goes',\n",
       "  'from',\n",
       "  'bottom',\n",
       "  'to',\n",
       "  'top',\n",
       "  'of',\n",
       "  'image',\n",
       "  '.',\n",
       "  'One',\n",
       "  'piece',\n",
       "  'of',\n",
       "  'debris',\n",
       "  'in',\n",
       "  'the',\n",
       "  'entry',\n",
       "  'channel',\n",
       "  'post',\n",
       "  'interface',\n",
       "  'starting',\n",
       "  'at',\n",
       "  'the',\n",
       "  '2nd',\n",
       "  'row',\n",
       "  '.',\n",
       "  'There',\n",
       "  'is',\n",
       "  'spider',\n",
       "  'web',\n",
       "  'like',\n",
       "  'cell',\n",
       "  'debris',\n",
       "  '.',\n",
       "  'An',\n",
       "  'increasingly',\n",
       "  'gets',\n",
       "  'worse',\n",
       "  'throughout',\n",
       "  'the',\n",
       "  'chip',\n",
       "  'at',\n",
       "  'the',\n",
       "  'exit',\n",
       "  'posts',\n",
       "  'channel',\n",
       "  'interface',\n",
       "  'uh',\n",
       "  'There',\n",
       "  'is',\n",
       "  'minor',\n",
       "  'sell',\n",
       "  'wrapping',\n",
       "  'and',\n",
       "  'spider',\n",
       "  'web',\n",
       "  'like',\n",
       "  'debris',\n",
       "  '.'],\n",
       " ['38', '1.813', 'E5', '.'],\n",
       " ['10',\n",
       "  'kilodalton',\n",
       "  'centrifugal',\n",
       "  'filter',\n",
       "  'to',\n",
       "  'concentrate',\n",
       "  'the',\n",
       "  'fractions',\n",
       "  'of',\n",
       "  'the',\n",
       "  'His',\n",
       "  'purification',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'samples',\n",
       "  'from',\n",
       "  'the',\n",
       "  'HPLC',\n",
       "  '',\n",
       "  'which',\n",
       "  'have',\n",
       "  '10',\n",
       "  'µl',\n",
       "  'left',\n",
       "  'for',\n",
       "  'Mass',\n",
       "  '-',\n",
       "  'spec',\n",
       "  'dilute',\n",
       "  'with',\n",
       "  '90',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'LCMS',\n",
       "  'buffer',\n",
       "  '',\n",
       "  'which',\n",
       "  'contains',\n",
       "  '3',\n",
       "  '%',\n",
       "  'acetonitrile',\n",
       "  'and',\n",
       "  '0.1',\n",
       "  '%',\n",
       "  'formic',\n",
       "  'acid',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'enter',\n",
       "  'the',\n",
       "  'imaging',\n",
       "  'data',\n",
       "  'at',\n",
       "  '4:00',\n",
       "  'PM',\n",
       "  '.',\n",
       "  'OK',\n",
       "  '',\n",
       "  'there',\n",
       "  'it',\n",
       "  'went',\n",
       "  '.',\n",
       "  'It',\n",
       "  'worked',\n",
       "  '',\n",
       "  'yeah',\n",
       "  '.'],\n",
       " ['timer', 'for', '2', 'min', '.'],\n",
       " ['4', 'best', 'in', 'class', '2', 'media', 'added', '.'],\n",
       " ['Apo',\n",
       "  'elution',\n",
       "  'from',\n",
       "  'experiment',\n",
       "  '66',\n",
       "  'is',\n",
       "  'slightly',\n",
       "  'less',\n",
       "  'than',\n",
       "  '20',\n",
       "  'µl',\n",
       "  '.',\n",
       "  'Add',\n",
       "  'to',\n",
       "  'the',\n",
       "  'concentrated',\n",
       "  'samples',\n",
       "  'recovered',\n",
       "  'from',\n",
       "  'the',\n",
       "  'Nanodrop',\n",
       "  '.',\n",
       "  'Add',\n",
       "  '19',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'water',\n",
       "  'and',\n",
       "  'then',\n",
       "  'prepare',\n",
       "  'samples',\n",
       "  'as',\n",
       "  'for',\n",
       "  'the',\n",
       "  'others',\n",
       "  '.'],\n",
       " ['.', 'I', 'will', 'aim', 'at', '70', 'decibels', 'noise', '.'],\n",
       " [\"'m\",\n",
       "  'going',\n",
       "  'to',\n",
       "  'inactivate',\n",
       "  'bacteria',\n",
       "  'in',\n",
       "  'different',\n",
       "  'ways',\n",
       "  ':',\n",
       "  'one',\n",
       "  'way',\n",
       "  'is',\n",
       "  'by',\n",
       "  'fixation',\n",
       "  'in',\n",
       "  'paraformaldehyde',\n",
       "  '',\n",
       "  'secondly',\n",
       "  'is',\n",
       "  'fixation',\n",
       "  'by',\n",
       "  'ethanol',\n",
       "  '.',\n",
       "  'way',\n",
       "  'is',\n",
       "  'inactivation',\n",
       "  'by',\n",
       "  'it',\n",
       "  '.',\n",
       "  'Forth',\n",
       "  'way',\n",
       "  'is',\n",
       "  'by',\n",
       "  'treatments',\n",
       "  'with',\n",
       "  'antibiotics',\n",
       "  'and',\n",
       "  'third',\n",
       "  'way',\n",
       "  'is',\n",
       "  'UV',\n",
       "  'treated',\n",
       "  '.',\n",
       "  'So',\n",
       "  '.',\n",
       "  'first',\n",
       "  'wash',\n",
       "  'the',\n",
       "  'bacteria',\n",
       "  'two',\n",
       "  'times',\n",
       "  'in',\n",
       "  '0.1',\n",
       "  '%',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  '.'],\n",
       " ['Assay', 'R271', 'no', 'optimal', 'temperature', 'was', 'achieved', '.'],\n",
       " ['air', 'filter', '.'],\n",
       " ['.',\n",
       "  'Equal',\n",
       "  '0.16',\n",
       "  '.',\n",
       "  '4.2',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.17',\n",
       "  '.',\n",
       "  '4.3',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.17',\n",
       "  '.',\n",
       "  '4.4',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.17',\n",
       "  '.',\n",
       "  '4.5',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.17',\n",
       "  '.',\n",
       "  '5.1',\n",
       "  'equal',\n",
       "  '.',\n",
       "  '0.063',\n",
       "  '.',\n",
       "  '5.2',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.063',\n",
       "  '.',\n",
       "  '5.3',\n",
       "  '.',\n",
       "  'Equal',\n",
       "  '.',\n",
       "  '0.065',\n",
       "  '.',\n",
       "  '5.4',\n",
       "  'equal',\n",
       "  '.',\n",
       "  '0.066',\n",
       "  '.',\n",
       "  '5.5',\n",
       "  'equal',\n",
       "  '.',\n",
       "  '0.046',\n",
       "  '.'],\n",
       " ['5', 'ml', 'of', 'DNA', 'substance', '.'],\n",
       " ['Five',\n",
       "  'cells',\n",
       "  'passage',\n",
       "  '13',\n",
       "  '',\n",
       "  'measured',\n",
       "  'at',\n",
       "  '11:30',\n",
       "  '',\n",
       "  'at',\n",
       "  '11:30',\n",
       "  '.'],\n",
       " ['',\n",
       "  'need',\n",
       "  'to',\n",
       "  'dilute',\n",
       "  '231.5',\n",
       "  'total',\n",
       "  'ml',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '',\n",
       "  'will',\n",
       "  'add',\n",
       "  '6.5',\n",
       "  'ml',\n",
       "  'to',\n",
       "  'bring',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'solution',\n",
       "  'to',\n",
       "  '800000',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml'],\n",
       " ['µl', 'transferred', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'check',\n",
       "  'if',\n",
       "  'we',\n",
       "  'have',\n",
       "  'enough',\n",
       "  'filter',\n",
       "  'tips',\n",
       "  '1000',\n",
       "  'µl',\n",
       "  '.'],\n",
       " ['case',\n",
       "  'you',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'catch',\n",
       "  'that',\n",
       "  'that',\n",
       "  'was',\n",
       "  'after',\n",
       "  '48',\n",
       "  'h.'],\n",
       " ['in', 'batch', '3', 'are', 'unlikely', 'to', 'be', 'contaminated', '.'],\n",
       " ['transduced',\n",
       "  '48',\n",
       "  'h',\n",
       "  'before',\n",
       "  'BJab',\n",
       "  'cells',\n",
       "  'with',\n",
       "  'different',\n",
       "  'lentiviruses',\n",
       "  '.',\n",
       "  'I',\n",
       "  'will',\n",
       "  'harvest',\n",
       "  'I',\n",
       "  'will',\n",
       "  'select',\n",
       "  'them',\n",
       "  'after',\n",
       "  '48',\n",
       "  'h',\n",
       "  'normally',\n",
       "  'I',\n",
       "  'select',\n",
       "  'after',\n",
       "  '24',\n",
       "  'h.',\n",
       "  'But',\n",
       "  'the',\n",
       "  'control',\n",
       "  'with',\n",
       "  'GSP',\n",
       "  'looked',\n",
       "  'after',\n",
       "  '24',\n",
       "  'h',\n",
       "  'not',\n",
       "  'so',\n",
       "  'good',\n",
       "  'so',\n",
       "  'after',\n",
       "  '48',\n",
       "  'h',\n",
       "  'all',\n",
       "  'cells',\n",
       "  'are',\n",
       "  'green',\n",
       "  'means',\n",
       "  'they',\n",
       "  'are',\n",
       "  'transduce',\n",
       "  '.'],\n",
       " ['aim',\n",
       "  'of',\n",
       "  'today',\n",
       "  \"'s\",\n",
       "  'assay',\n",
       "  'is',\n",
       "  'to',\n",
       "  'determine',\n",
       "  'kinetic',\n",
       "  'values',\n",
       "  'for',\n",
       "  'Her2',\n",
       "  'protein',\n",
       "  '.'],\n",
       " ['me', 'to', 'buy', 'a', 'new', 'book', 'tomorrow', 'at', '4:00', 'PM', '.'],\n",
       " ['response',\n",
       "  'does',\n",
       "  'seem',\n",
       "  'to',\n",
       "  'have',\n",
       "  'worked',\n",
       "  '.',\n",
       "  'I',\n",
       "  'was',\n",
       "  'too',\n",
       "  'hasty',\n",
       "  'in',\n",
       "  'my',\n",
       "  'assessment',\n",
       "  'assay',\n",
       "  'seems',\n",
       "  'to',\n",
       "  'be',\n",
       "  'best',\n",
       "  '0\\\\.5',\n",
       "  '%',\n",
       "  'DMSO',\n",
       "  '',\n",
       "  'but',\n",
       "  'seems',\n",
       "  'to',\n",
       "  'be',\n",
       "  'tolerant',\n",
       "  'up',\n",
       "  'to',\n",
       "  '5',\n",
       "  '%',\n",
       "  'DMSO',\n",
       "  'screening',\n",
       "  'at',\n",
       "  '2',\n",
       "  '%',\n",
       "  'DMSO',\n",
       "  'should',\n",
       "  'be',\n",
       "  'fine',\n",
       "  'for',\n",
       "  'the',\n",
       "  'fragment',\n",
       "  'set',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'jabra',\n",
       "  'with',\n",
       "  'English',\n",
       "  'radio',\n",
       "  'in',\n",
       "  'the',\n",
       "  'background',\n",
       "  'extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stared',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  'the',\n",
       "  'face',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  'the',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotarvap',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbars',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end'],\n",
       " ['5',\n",
       "  'cells',\n",
       "  'passage',\n",
       "  '13',\n",
       "  'measured',\n",
       "  'at',\n",
       "  '11:30',\n",
       "  'at',\n",
       "  '11:30',\n",
       "  '.'],\n",
       " ['µl',\n",
       "  'of',\n",
       "  'magnesium',\n",
       "  'chloride',\n",
       "  'was',\n",
       "  'added',\n",
       "  'to',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'water',\n",
       "  'to',\n",
       "  'create',\n",
       "  'a',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '10',\n",
       "  'mM'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'out',\n",
       "  'of',\n",
       "  'the',\n",
       "  'incubator',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '8:00',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['cells',\n",
       "  'wild',\n",
       "  'type',\n",
       "  'cells',\n",
       "  'needs',\n",
       "  'to',\n",
       "  'be',\n",
       "  'a',\n",
       "  'split',\n",
       "  'into',\n",
       "  'a',\n",
       "  'T75',\n",
       "  'and',\n",
       "  'the',\n",
       "  'label',\n",
       "  'with',\n",
       "  'heavy',\n",
       "  'SILAC',\n",
       "  'media',\n",
       "  '.'],\n",
       " ['me', 'on', 'Friday', '25th', 'to', 'email', 'San', '.'],\n",
       " ['need', 'to', 'buy', '20', 'ml', 'of', '.'],\n",
       " ['10', 'ml', 'of', 'water', 'add', '.'],\n",
       " ['23', '4.484', 'E4', '.'],\n",
       " ['will',\n",
       "  'prepare',\n",
       "  'a',\n",
       "  'cell',\n",
       "  'culture',\n",
       "  'experiment',\n",
       "  'with',\n",
       "  'DDB1',\n",
       "  '',\n",
       "  'DDA1',\n",
       "  'CAT2',\n",
       "  'viruses',\n",
       "  'in',\n",
       "  'different',\n",
       "  'concentrations',\n",
       "  'and',\n",
       "  'he',\n",
       "  'will',\n",
       "  'titrate',\n",
       "  'them',\n",
       "  'against',\n",
       "  'each',\n",
       "  'other',\n",
       "  'and',\n",
       "  'try',\n",
       "  'to',\n",
       "  'harvest',\n",
       "  'the',\n",
       "  'protein',\n",
       "  'which',\n",
       "  'is',\n",
       "  'expressed',\n",
       "  'after',\n",
       "  '48',\n",
       "  'and',\n",
       "  '60h',\n",
       "  'and',\n",
       "  'compare',\n",
       "  'protein',\n",
       "  'production',\n",
       "  'in',\n",
       "  'a',\n",
       "  'SDS',\n",
       "  'gel',\n",
       "  '.'],\n",
       " ['filter',\n",
       "  'the',\n",
       "  'lentiviruses',\n",
       "  'and',\n",
       "  'retroviruses',\n",
       "  'with',\n",
       "  'small',\n",
       "  'filter',\n",
       "  '0.45',\n",
       "  'µm',\n",
       "  'and',\n",
       "  'freeze',\n",
       "  'it',\n",
       "  'at',\n",
       "  '-80',\n",
       "  '°',\n",
       "  '.'],\n",
       " ['9', '', '9.216', 'E', '4'],\n",
       " [\"'re\",\n",
       "  'gon',\n",
       "  'na',\n",
       "  'add',\n",
       "  'Pen',\n",
       "  '-',\n",
       "  'strep',\n",
       "  'and',\n",
       "  'gentamicin',\n",
       "  '.',\n",
       "  'I',\n",
       "  'got',\n",
       "  'ta',\n",
       "  'add',\n",
       "  '5',\n",
       "  'mL',\n",
       "  'of',\n",
       "  'Pen',\n",
       "  '-',\n",
       "  'strep',\n",
       "  'from',\n",
       "  'the',\n",
       "  'freezer',\n",
       "  '.',\n",
       "  'Am',\n",
       "  'I',\n",
       "  'gon',\n",
       "  'na',\n",
       "  'add',\n",
       "  '.',\n",
       "  'gentamicin',\n",
       "  'as',\n",
       "  'well',\n",
       "  '',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'going',\n",
       "  'to',\n",
       "  'have',\n",
       "  'both',\n",
       "  'roles',\n",
       "  'directly',\n",
       "  'to',\n",
       "  'the',\n",
       "  'Abbey',\n",
       "  'Cell',\n",
       "  'D',\n",
       "  'man',\n",
       "  'mix',\n",
       "  '.',\n",
       "  'Gentamicin',\n",
       "  'is',\n",
       "  '10',\n",
       "  'mg',\n",
       "  'per',\n",
       "  'mL',\n",
       "  'concentration',\n",
       "  '.',\n",
       "  'Also',\n",
       "  'I',\n",
       "  'should',\n",
       "  'end',\n",
       "  'up',\n",
       "  'with',\n",
       "  'a',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  'slightly',\n",
       "  'more',\n",
       "  'than',\n",
       "  '10',\n",
       "  'µg',\n",
       "  'per',\n",
       "  'mL.',\n",
       "  'The',\n",
       "  'pen',\n",
       "  '-',\n",
       "  'strep',\n",
       "  'concentration',\n",
       "  'is',\n",
       "  'the',\n",
       "  'same',\n",
       "  'one',\n",
       "  'that',\n",
       "  'we',\n",
       "  'use',\n",
       "  'all',\n",
       "  'the',\n",
       "  'time',\n",
       "  '.'],\n",
       " ['tubes',\n",
       "  'are',\n",
       "  'labeled',\n",
       "  '5B3',\n",
       "  'and',\n",
       "  '3A3',\n",
       "  'and',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'the',\n",
       "  '2nd',\n",
       "  'A3',\n",
       "  '.',\n",
       "  'Pour',\n",
       "  'these',\n",
       "  'in',\n",
       "  'water',\n",
       "  'and',\n",
       "  'add',\n",
       "  'Tris',\n",
       "  'buffer',\n",
       "  'to',\n",
       "  'get',\n",
       "  'pH',\n",
       "  '8.3',\n",
       "  '.'],\n",
       " ['10', 'µl', 'of', 'all', 'samples', '.'],\n",
       " ['all',\n",
       "  'the',\n",
       "  'tubes',\n",
       "  'in',\n",
       "  'water',\n",
       "  '.',\n",
       "  'This',\n",
       "  'include',\n",
       "  '6',\n",
       "  'sample',\n",
       "  'inlet',\n",
       "  'A1'],\n",
       " ['heavy', 'labelled', 'carnitine', 'a', '100', 'µl', '.'],\n",
       " ['14', '', '2.271', 'E', '5', '.'],\n",
       " ['viability', '83.7', '%', '.', 'Live'],\n",
       " ['1',\n",
       "  'to',\n",
       "  '8',\n",
       "  'are',\n",
       "  'coated',\n",
       "  'with',\n",
       "  '52717',\n",
       "  'and',\n",
       "  'columns',\n",
       "  '9',\n",
       "  'to',\n",
       "  '12',\n",
       "  'are',\n",
       "  'coated',\n",
       "  'with',\n",
       "  'M232',\n",
       "  '.'],\n",
       " ['48', '2.329', 'E5', '.'],\n",
       " ['SDS',\n",
       "  'page',\n",
       "  'samples',\n",
       "  'starting',\n",
       "  'with',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'peak',\n",
       "  '1',\n",
       "  '',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'peak',\n",
       "  '2',\n",
       "  '',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'peak',\n",
       "  '3',\n",
       "  '',\n",
       "  'which',\n",
       "  'is',\n",
       "  'the',\n",
       "  'most',\n",
       "  'interesting',\n",
       "  '1',\n",
       "  'then',\n",
       "  'peak',\n",
       "  '4',\n",
       "  'and',\n",
       "  'finally',\n",
       "  'from',\n",
       "  'the',\n",
       "  'last',\n",
       "  'purification',\n",
       "  'flowthrough',\n",
       "  'fraction',\n",
       "  'B3',\n",
       "  '.'],\n",
       " ['substrates',\n",
       "  'for',\n",
       "  'large',\n",
       "  'batch',\n",
       "  '.',\n",
       "  'Final',\n",
       "  'concentration',\n",
       "  '.5',\n",
       "  'mg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['have', 'to', 'order', 'new', 'T25', 'flasks', '.'],\n",
       " ['me', 'of', 'a', 'meeting', 'at', '8', 'PM', 'today', '.'],\n",
       " ['45', '1.783', 'E5', '.'],\n",
       " ['.',\n",
       "  'I',\n",
       "  'centrifuged',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'the',\n",
       "  'ultra',\n",
       "  'centrifuge',\n",
       "  '.',\n",
       "  'Avocado',\n",
       "  '.'],\n",
       " ['heat',\n",
       "  'stock',\n",
       "  'that',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'going',\n",
       "  'to',\n",
       "  'use',\n",
       "  'today',\n",
       "  'is',\n",
       "  'the',\n",
       "  'one',\n",
       "  'I',\n",
       "  'made',\n",
       "  'myself',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  'heat',\n",
       "  'stock',\n",
       "  'is',\n",
       "  '500',\n",
       "  'million',\n",
       "  'molar',\n",
       "  '.'],\n",
       " ['timer', 'for', '2', 'min', '.'],\n",
       " ['number', 'is', 'HU', '17', '28', '.'],\n",
       " ['one',\n",
       "  '.',\n",
       "  'extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'No',\n",
       "  'special',\n",
       "  'noises',\n",
       "  '',\n",
       "  'alone',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lab',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['a',\n",
       "  'list',\n",
       "  'of',\n",
       "  'the',\n",
       "  'following',\n",
       "  '3',\n",
       "  'donors',\n",
       "  ':',\n",
       "  'RBC',\n",
       "  '5619',\n",
       "  '',\n",
       "  '110044394',\n",
       "  '',\n",
       "  'KP59331',\n",
       "  '.'],\n",
       " ['with',\n",
       "  'difficult',\n",
       "  'words',\n",
       "  'and',\n",
       "  'PLT',\n",
       "  'legend',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '',\n",
       "  '17',\n",
       "  'µl',\n",
       "  '',\n",
       "  'diluted',\n",
       "  '',\n",
       "  'test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['MEM',\n",
       "  'nonessential',\n",
       "  'amino',\n",
       "  'acid',\n",
       "  'solution',\n",
       "  '.',\n",
       "  '5',\n",
       "  'ml',\n",
       "  'to',\n",
       "  '500',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'media',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'am',\n",
       "  'harvesting',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'NP16',\n",
       "  'with',\n",
       "  'and',\n",
       "  'without',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']'],\n",
       " ['well',\n",
       "  'plates',\n",
       "  '2',\n",
       "  '.',\n",
       "  '1.5',\n",
       "  'ml',\n",
       "  'cell',\n",
       "  'suspension',\n",
       "  '.',\n",
       "  '180',\n",
       "  'µl',\n",
       "  'per',\n",
       "  'well',\n",
       "  '.',\n",
       "  'Last',\n",
       "  'row',\n",
       "  'no',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ['volume',\n",
       "  '220',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'the',\n",
       "  '2',\n",
       "  'times',\n",
       "  'solubilization',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['in', 'batch', '3', 'are', 'unlikely', 'to', 'be', 'contaminated', '.'],\n",
       " ['washing',\n",
       "  'the',\n",
       "  'FSEC',\n",
       "  'on',\n",
       "  'SC3',\n",
       "  'with',\n",
       "  'Milli',\n",
       "  'Q',\n",
       "  'Water',\n",
       "  'at',\n",
       "  '0.35',\n",
       "  'ml',\n",
       "  '/',\n",
       "  'min',\n",
       "  '.',\n",
       "  'The',\n",
       "  'column',\n",
       "  'is',\n",
       "  'Sepac',\n",
       "  'SN2A13625',\n",
       "  '5',\n",
       "  'µm',\n",
       "  '',\n",
       "  '500',\n",
       "  'angstroms',\n",
       "  '',\n",
       "  '4.6',\n",
       "  '*',\n",
       "  '300',\n",
       "  'mm',\n",
       "  '.'],\n",
       " ['number',\n",
       "  '7',\n",
       "  'or',\n",
       "  'plasmid',\n",
       "  'number',\n",
       "  '7',\n",
       "  '.',\n",
       "  'Concentration',\n",
       "  '197.2',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Fraction',\n",
       "  '1',\n",
       "  '1.86',\n",
       "  '',\n",
       "  'fraction',\n",
       "  '2',\n",
       "  '2.35',\n",
       "  '.',\n",
       "  'Stop',\n",
       "  '.'],\n",
       " ['puri',\n",
       "  'for',\n",
       "  'the',\n",
       "  'purification',\n",
       "  'for',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'that',\n",
       "  'I',\n",
       "  'made',\n",
       "  'a',\n",
       "  'mistake',\n",
       "  'previously',\n",
       "  'in',\n",
       "  'during',\n",
       "  'the',\n",
       "  '.',\n",
       "  'Um',\n",
       "  'preamplifications',\n",
       "  'step',\n",
       "  'that',\n",
       "  'I',\n",
       "  'have',\n",
       "  'now',\n",
       "  'increase',\n",
       "  'the',\n",
       "  'volume',\n",
       "  'of',\n",
       "  'the',\n",
       "  'beads',\n",
       "  'I',\n",
       "  'start',\n",
       "  'at',\n",
       "  '25',\n",
       "  'I',\n",
       "  'put',\n",
       "  '30',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'just',\n",
       "  'in',\n",
       "  'case',\n",
       "  'because',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'much',\n",
       "  'higher',\n",
       "  'volume',\n",
       "  'that',\n",
       "  'I',\n",
       "  'have',\n",
       "  'not',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'needed',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mistake',\n",
       "  '.'],\n",
       " ['the',\n",
       "  'Q',\n",
       "  '-',\n",
       "  'box',\n",
       "  'to',\n",
       "  'add',\n",
       "  '2',\n",
       "  '*',\n",
       "  '.5',\n",
       "  'l',\n",
       "  'of',\n",
       "  'water',\n",
       "  'to',\n",
       "  'the',\n",
       "  'Tris',\n",
       "  'powder',\n",
       "  '.',\n",
       "  'Note',\n",
       "  'that',\n",
       "  'the',\n",
       "  'variation',\n",
       "  'on',\n",
       "  'the',\n",
       "  'Q',\n",
       "  '-',\n",
       "  'pot',\n",
       "  'is',\n",
       "  'I',\n",
       "  'would',\n",
       "  'say',\n",
       "  'approximately',\n",
       "  '10',\n",
       "  'to',\n",
       "  '15',\n",
       "  'ml',\n",
       "  'extra',\n",
       "  'if',\n",
       "  'you',\n",
       "  'compare',\n",
       "  'it',\n",
       "  'to',\n",
       "  'a',\n",
       "  '500',\n",
       "  'ml',\n",
       "  'cylinder',\n",
       "  '.'],\n",
       " ['4',\n",
       "  'experiment',\n",
       "  '',\n",
       "  '5',\n",
       "  'inactivated',\n",
       "  'bacteria',\n",
       "  'KU',\n",
       "  'parent',\n",
       "  'wild',\n",
       "  'type',\n",
       "  '2.171',\n",
       "  '.'],\n",
       " ['through',\n",
       "  'checked',\n",
       "  '.',\n",
       "  'Expression',\n",
       "  'of',\n",
       "  'aBA',\n",
       "  'First',\n",
       "  'ID',\n",
       "  '1A',\n",
       "  'Ecolize',\n",
       "  'train',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  '',\n",
       "  'chaperone',\n",
       "  'plasmid',\n",
       "  'PJKJE8',\n",
       "  '',\n",
       "  'ID1B',\n",
       "  '',\n",
       "  'BL21',\n",
       "  'BE',\n",
       "  'free',\n",
       "  '',\n",
       "  'PGKJE8',\n",
       "  '',\n",
       "  'ID',\n",
       "  '4A',\n",
       "  '',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  '',\n",
       "  'chaperone',\n",
       "  'plasmid',\n",
       "  'PGTF2',\n",
       "  '',\n",
       "  'ID4B',\n",
       "  '',\n",
       "  'equalize',\n",
       "  'train',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  'chaperone',\n",
       "  'plasmid',\n",
       "  'PGTF2',\n",
       "  '',\n",
       "  '',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  'and',\n",
       "  'chaperone',\n",
       "  'plasmid',\n",
       "  'G',\n",
       "  'PGTF2',\n",
       "  '78',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  'P',\n",
       "  'Grow',\n",
       "  '7',\n",
       "  'ID7B',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  'P',\n",
       "  'Grow',\n",
       "  '7',\n",
       "  'ID',\n",
       "  '10A',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  'and',\n",
       "  'chaperone',\n",
       "  'plasmid',\n",
       "  'PTF16',\n",
       "  'ID',\n",
       "  '10B',\n",
       "  'strain',\n",
       "  'BL21',\n",
       "  'D',\n",
       "  'free',\n",
       "  'PTFC',\n",
       "  '16',\n",
       "  '.'],\n",
       " ['timer', 'for', '2', 'min', '.', 'Now', '.'],\n",
       " ['me', 'to', 'order', 'AP2', 'for', 'iPSCs', '.'],\n",
       " ['of',\n",
       "  'the',\n",
       "  'IC',\n",
       "  '50',\n",
       "  'values',\n",
       "  'for',\n",
       "  'NVP',\n",
       "  '2',\n",
       "  'tend',\n",
       "  'to',\n",
       "  'be',\n",
       "  'clustered',\n",
       "  'around',\n",
       "  '25',\n",
       "  'to',\n",
       "  '35',\n",
       "  'nmol',\n",
       "  '.',\n",
       "  '?',\n",
       "  '16',\n",
       "  '',\n",
       "  '13',\n",
       "  '',\n",
       "  '32s',\n",
       "  '.'],\n",
       " ['in', 'batch', '3', 'are', 'unlikely', 'to', 'be', 'uncontaminated', '.'],\n",
       " ['sodium',\n",
       "  'hydroxide',\n",
       "  'dilution',\n",
       "  '.',\n",
       "  '180',\n",
       "  'µl',\n",
       "  'PWR',\n",
       "  'to',\n",
       "  'dilution',\n",
       "  '.'],\n",
       " ['.', '123', '.', 'Bullets', 'hello', '.', '.', 'Bullet', '.', '123', '.'],\n",
       " ['filtering',\n",
       "  '',\n",
       "  'the',\n",
       "  'Tyrode',\n",
       "  'with',\n",
       "  'ascorbic',\n",
       "  'acid',\n",
       "  'with',\n",
       "  'glucose',\n",
       "  'now',\n",
       "  'is',\n",
       "  'like',\n",
       "  '0.2',\n",
       "  'µm',\n",
       "  'filter',\n",
       "  '.'],\n",
       " ['HEPES',\n",
       "  'buffer',\n",
       "  'I',\n",
       "  'used',\n",
       "  'was',\n",
       "  'half',\n",
       "  'the',\n",
       "  'concentration',\n",
       "  'than',\n",
       "  'the',\n",
       "  'standard',\n",
       "  'concentration',\n",
       "  '',\n",
       "  'so',\n",
       "  'I',\n",
       "  'had',\n",
       "  'to',\n",
       "  'take',\n",
       "  'out',\n",
       "  '717',\n",
       "  'µl',\n",
       "  'from',\n",
       "  'initial',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Bumblebee',\n",
       "  '.'],\n",
       " ['Mass',\n",
       "  'spectrometry',\n",
       "  '.',\n",
       "  'Check',\n",
       "  'metabolite',\n",
       "  'abundance',\n",
       "  'between',\n",
       "  '5',\n",
       "  'biological',\n",
       "  'replicates',\n",
       "  'of',\n",
       "  '2',\n",
       "  'mutants',\n",
       "  '.',\n",
       "  'Blueberry',\n",
       "  '.'],\n",
       " ['4',\n",
       "  '.',\n",
       "  'Cm',\n",
       "  '=',\n",
       "  '96',\n",
       "  '.',\n",
       "  'R',\n",
       "  '=',\n",
       "  '2.9',\n",
       "  '.',\n",
       "  'Very',\n",
       "  'good',\n",
       "  'recording',\n",
       "  'the',\n",
       "  'L',\n",
       "  'type',\n",
       "  'calcium',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  'very',\n",
       "  'stable',\n",
       "  'initially',\n",
       "  '.',\n",
       "  'Caffeine',\n",
       "  'and',\n",
       "  'NCX',\n",
       "  'was',\n",
       "  'done',\n",
       "  'with',\n",
       "  'very',\n",
       "  'nice',\n",
       "  'representative',\n",
       "  'traces',\n",
       "  '.',\n",
       "  'The',\n",
       "  'effect',\n",
       "  'of',\n",
       "  'Isoprenaline',\n",
       "  'was',\n",
       "  'also',\n",
       "  'done',\n",
       "  'with',\n",
       "  'a',\n",
       "  'nice',\n",
       "  'increase',\n",
       "  'in',\n",
       "  'the',\n",
       "  'current',\n",
       "  '.',\n",
       "  'Also',\n",
       "  '',\n",
       "  'isoprenaline',\n",
       "  '',\n",
       "  'caffeine',\n",
       "  '/',\n",
       "  'ncx',\n",
       "  'and',\n",
       "  'caffeine',\n",
       "  'induced',\n",
       "  'calcium',\n",
       "  'transient',\n",
       "  'was',\n",
       "  'also',\n",
       "  'very',\n",
       "  'good',\n",
       "  'and',\n",
       "  'representative',\n",
       "  '.',\n",
       "  'After',\n",
       "  'the',\n",
       "  'ISO',\n",
       "  'experiment',\n",
       "  'leak',\n",
       "  'was',\n",
       "  'also',\n",
       "  'done',\n",
       "  '.',\n",
       "  'A',\n",
       "  'nice',\n",
       "  'representative',\n",
       "  'Leak',\n",
       "  'Trace',\n",
       "  'was',\n",
       "  'seen',\n",
       "  '.',\n",
       "  'Still',\n",
       "  'observed',\n",
       "  'the',\n",
       "  'continuous',\n",
       "  'increasing',\n",
       "  'series',\n",
       "  'resistance',\n",
       "  '.',\n",
       "  'However',\n",
       "  '',\n",
       "  'this',\n",
       "  'only',\n",
       "  'happened',\n",
       "  'after',\n",
       "  'I',\n",
       "  'applied',\n",
       "  'Caffeine',\n",
       "  '',\n",
       "  'which',\n",
       "  'disrupted',\n",
       "  'the',\n",
       "  'position',\n",
       "  'of',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'and',\n",
       "  'could',\n",
       "  'influence',\n",
       "  'the',\n",
       "  'stability',\n",
       "  'of',\n",
       "  'the',\n",
       "  'seal',\n",
       "  '.'],\n",
       " ['making',\n",
       "  'buffer',\n",
       "  'I',\n",
       "  'have',\n",
       "  'used',\n",
       "  'my',\n",
       "  'own',\n",
       "  'stock',\n",
       "  'of',\n",
       "  '500',\n",
       "  'mM',\n",
       "  'HEPES',\n",
       "  'and',\n",
       "  'since',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'half',\n",
       "  'concentration',\n",
       "  'than',\n",
       "  'the',\n",
       "  'usual',\n",
       "  'concentration',\n",
       "  'I',\n",
       "  'had',\n",
       "  'to',\n",
       "  'add',\n",
       "  'double',\n",
       "  'the',\n",
       "  'volume',\n",
       "  '.'],\n",
       " ['in',\n",
       "  'batch',\n",
       "  '3',\n",
       "  'are',\n",
       "  'unlikely',\n",
       "  'to',\n",
       "  'be',\n",
       "  'contaminated',\n",
       "  '.',\n",
       "  'Add',\n",
       "  '5000',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'water',\n",
       "  'and',\n",
       "  'acetone',\n",
       "  'and',\n",
       "  'mix',\n",
       "  'for',\n",
       "  '10',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'distil',\n",
       "  '.'],\n",
       " ['moment', 'potential', 'around', '-', '60', '.'],\n",
       " ['D2', '-', '1', '09211', '.'],\n",
       " [\"'m\",\n",
       "  'adding',\n",
       "  '6',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'penicillin',\n",
       "  'and',\n",
       "  'streptomycin',\n",
       "  'to',\n",
       "  'the',\n",
       "  'new',\n",
       "  'hFF',\n",
       "  'medium',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['number',\n",
       "  '3',\n",
       "  'looks',\n",
       "  'super',\n",
       "  'weird',\n",
       "  '.',\n",
       "  'All',\n",
       "  'the',\n",
       "  'fat',\n",
       "  'created',\n",
       "  'a',\n",
       "  'layer',\n",
       "  'on',\n",
       "  'the',\n",
       "  'top',\n",
       "  'of',\n",
       "  'the',\n",
       "  'tube',\n",
       "  'after',\n",
       "  'the',\n",
       "  'centrifugation',\n",
       "  '.',\n",
       "  'And',\n",
       "  'there',\n",
       "  'is',\n",
       "  'a',\n",
       "  'tiny',\n",
       "  '',\n",
       "  'tiny',\n",
       "  'pellet',\n",
       "  '.',\n",
       "  'On',\n",
       "  'the',\n",
       "  'bottom',\n",
       "  '.',\n",
       "  'Let',\n",
       "  \"'s\",\n",
       "  'collect',\n",
       "  'everything',\n",
       "  '.'],\n",
       " ['really',\n",
       "  'difficult',\n",
       "  'to',\n",
       "  'track',\n",
       "  'what',\n",
       "  'almost',\n",
       "  'up',\n",
       "  'to',\n",
       "  'date',\n",
       "  'version',\n",
       "  'is',\n",
       "  'as',\n",
       "  'well',\n",
       "  '.',\n",
       "  'For',\n",
       "  'some',\n",
       "  'studies',\n",
       "  'for',\n",
       "  'example',\n",
       "  'in',\n",
       "  'the',\n",
       "  'necropsy',\n",
       "  'it',\n",
       "  'can',\n",
       "  'take',\n",
       "  '2',\n",
       "  'days',\n",
       "  '.',\n",
       "  '1',\n",
       "  'study',\n",
       "  'can',\n",
       "  'take',\n",
       "  'almost',\n",
       "  'a',\n",
       "  'month',\n",
       "  '.',\n",
       "  'As',\n",
       "  'the',\n",
       "  'animals',\n",
       "  'need',\n",
       "  'to',\n",
       "  'be',\n",
       "  'here',\n",
       "  'for',\n",
       "  'at',\n",
       "  'least',\n",
       "  'a',\n",
       "  'week',\n",
       "  'before',\n",
       "  'we',\n",
       "  'can',\n",
       "  'do',\n",
       "  'start',\n",
       "  'on',\n",
       "  'anything',\n",
       "  '.'],\n",
       " ['got',\n",
       "  'colonies',\n",
       "  'of',\n",
       "  'MTSC',\n",
       "  'with',\n",
       "  'his',\n",
       "  '-',\n",
       "  'tag',\n",
       "  'and',\n",
       "  'PSTP',\n",
       "  '.',\n",
       "  'And',\n",
       "  'we',\n",
       "  'do',\n",
       "  'colony',\n",
       "  'PCR',\n",
       "  'with',\n",
       "  'the',\n",
       "  'primer',\n",
       "  'pair',\n",
       "  '6',\n",
       "  'and',\n",
       "  '4',\n",
       "  'to',\n",
       "  'detect',\n",
       "  'the',\n",
       "  'insert',\n",
       "  '.',\n",
       "  'And',\n",
       "  'we',\n",
       "  'are',\n",
       "  'expecting',\n",
       "  'a',\n",
       "  'product',\n",
       "  'of',\n",
       "  '2000',\n",
       "  'base',\n",
       "  'pair',\n",
       "  '.',\n",
       "  'And',\n",
       "  'for',\n",
       "  'a',\n",
       "  'positive',\n",
       "  'control',\n",
       "  'we',\n",
       "  'use',\n",
       "  'primer',\n",
       "  'pair',\n",
       "  '5',\n",
       "  'and',\n",
       "  '4',\n",
       "  'to',\n",
       "  'detect',\n",
       "  'YFP',\n",
       "  '.',\n",
       "  'And',\n",
       "  'if',\n",
       "  'YFP',\n",
       "  'is',\n",
       "  'still',\n",
       "  'inside',\n",
       "  'the',\n",
       "  'vector',\n",
       "  'we',\n",
       "  'should',\n",
       "  'achieve',\n",
       "  'a',\n",
       "  'product',\n",
       "  'of',\n",
       "  '800',\n",
       "  'base',\n",
       "  'pair',\n",
       "  '.',\n",
       "  'The',\n",
       "  'annealing',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  'the',\n",
       "  'base',\n",
       "  'pairs',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'primer',\n",
       "  'pairs',\n",
       "  '6',\n",
       "  'and',\n",
       "  '4',\n",
       "  'is',\n",
       "  '46',\n",
       "  '°',\n",
       "  'and',\n",
       "  'for',\n",
       "  'the',\n",
       "  'primer',\n",
       "  'pair',\n",
       "  '5',\n",
       "  'and',\n",
       "  '4',\n",
       "  'is',\n",
       "  '53',\n",
       "  '°',\n",
       "  'C',\n",
       "  '.'],\n",
       " ['again',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'labels',\n",
       "  '.',\n",
       "  'So',\n",
       "  'I',\n",
       "  'have',\n",
       "  'sample',\n",
       "  'labels',\n",
       "  'that',\n",
       "  'that',\n",
       "  'consist',\n",
       "  'of',\n",
       "  'a',\n",
       "  'number',\n",
       "  'and',\n",
       "  'letters',\n",
       "  '.',\n",
       "  'Number',\n",
       "  '3',\n",
       "  'in',\n",
       "  'this',\n",
       "  'case',\n",
       "  'represents',\n",
       "  'the',\n",
       "  'third',\n",
       "  'replicate',\n",
       "  '.',\n",
       "  'First',\n",
       "  'G',\n",
       "  'in',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'label',\n",
       "  'is',\n",
       "  'referring',\n",
       "  'to',\n",
       "  'foregut',\n",
       "  '',\n",
       "  'so',\n",
       "  'G1',\n",
       "  '.',\n",
       "  'G2',\n",
       "  'is',\n",
       "  'hindgut',\n",
       "  'and',\n",
       "  'G',\n",
       "  'alone',\n",
       "  'is',\n",
       "  'the',\n",
       "  'whole',\n",
       "  'gut',\n",
       "  '.',\n",
       "  'Then',\n",
       "  '',\n",
       "  'F',\n",
       "  'stands',\n",
       "  'for',\n",
       "  'fat',\n",
       "  'body',\n",
       "  '',\n",
       "  'H',\n",
       "  'for',\n",
       "  'hemocytes',\n",
       "  '.',\n",
       "  'Apart',\n",
       "  'from',\n",
       "  'that',\n",
       "  'we',\n",
       "  'have',\n",
       "  'a',\n",
       "  'second',\n",
       "  'letter',\n",
       "  'that',\n",
       "  'labels',\n",
       "  'the',\n",
       "  'strain',\n",
       "  'of',\n",
       "  'the',\n",
       "  'mosquitos',\n",
       "  '.',\n",
       "  'G',\n",
       "  'is',\n",
       "  'for',\n",
       "  'G3',\n",
       "  '.',\n",
       "  'RKG',\n",
       "  'is',\n",
       "  'labeled',\n",
       "  'with',\n",
       "  'Besides',\n",
       "  'that',\n",
       "  'we',\n",
       "  'have',\n",
       "  'written',\n",
       "  'below',\n",
       "  ':',\n",
       "  'SF',\n",
       "  'that',\n",
       "  'stands',\n",
       "  'for',\n",
       "  'sugar',\n",
       "  'fat',\n",
       "  '',\n",
       "  'BF',\n",
       "  'for',\n",
       "  'blood',\n",
       "  'fat',\n",
       "  'and',\n",
       "  'PF',\n",
       "  'for',\n",
       "  'Plasmodium',\n",
       "  'infected',\n",
       "  'mosquitos',\n",
       "  '.'],\n",
       " ['cells',\n",
       "  'show',\n",
       "  'a',\n",
       "  'confluence',\n",
       "  'of',\n",
       "  '70',\n",
       "  '%',\n",
       "  '.',\n",
       "  'And',\n",
       "  'can',\n",
       "  'be',\n",
       "  'used',\n",
       "  'for',\n",
       "  'further',\n",
       "  'experiments',\n",
       "  '.'],\n",
       " ['timer', 'for', '2', 'min', '.'],\n",
       " ['lot',\n",
       "  'number',\n",
       "  '9112011',\n",
       "  '.',\n",
       "  'Pre',\n",
       "  '-',\n",
       "  'coated',\n",
       "  'PAMPA',\n",
       "  'plates',\n",
       "  '.'],\n",
       " ['sample',\n",
       "  'has',\n",
       "  'now',\n",
       "  'been',\n",
       "  'concentrated',\n",
       "  'to',\n",
       "  'approximately',\n",
       "  '10',\n",
       "  'ml',\n",
       "  'and',\n",
       "  'this',\n",
       "  'is',\n",
       "  'the',\n",
       "  'Relaxin',\n",
       "  '',\n",
       "  'which',\n",
       "  'is',\n",
       "  'coexpressed',\n",
       "  'so',\n",
       "  'it',\n",
       "  'is',\n",
       "  'processed',\n",
       "  '.',\n",
       "  'Pull',\n",
       "  'from',\n",
       "  'the',\n",
       "  'two',\n",
       "  'concentration',\n",
       "  'tubes',\n",
       "  'for',\n",
       "  'subsequent',\n",
       "  'filtering',\n",
       "  'and',\n",
       "  'application',\n",
       "  'to',\n",
       "  'the',\n",
       "  'size',\n",
       "  'exclusion',\n",
       "  'column',\n",
       "  'when',\n",
       "  'you',\n",
       "  'make',\n",
       "  'the',\n",
       "  'next',\n",
       "  'IMAC',\n",
       "  'is',\n",
       "  'finished',\n",
       "  '.'],\n",
       " ['AMP', 'mastermix', '126', 'µl', 'dpx', '1'],\n",
       " ['1',\n",
       "  'was',\n",
       "  'run',\n",
       "  'with',\n",
       "  'a',\n",
       "  'latter',\n",
       "  'of',\n",
       "  '100',\n",
       "  'base',\n",
       "  'pairs',\n",
       "  '',\n",
       "  'gel',\n",
       "  '2',\n",
       "  'was',\n",
       "  'run',\n",
       "  'with',\n",
       "  'a',\n",
       "  'latter',\n",
       "  'with',\n",
       "  '50',\n",
       "  'base',\n",
       "  'pairs',\n",
       "  '.'],\n",
       " ['two',\n",
       "  'eluted',\n",
       "  'peptides',\n",
       "  'from',\n",
       "  'the',\n",
       "  'column',\n",
       "  '30',\n",
       "  'µl',\n",
       "  'of',\n",
       "  '70',\n",
       "  '%',\n",
       "  'acetonitrile',\n",
       "  'is',\n",
       "  'added',\n",
       "  'to',\n",
       "  'the',\n",
       "  'spin',\n",
       "  'filters',\n",
       "  'and',\n",
       "  'centrifuged',\n",
       "  'at',\n",
       "  '1500',\n",
       "  'RCF',\n",
       "  'for',\n",
       "  '1',\n",
       "  'min',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'and',\n",
       "  'this',\n",
       "  'step',\n",
       "  'is',\n",
       "  'repeated',\n",
       "  'twice',\n",
       "  '.'],\n",
       " ['000', '.', 'Leaky', 'seal', '.', 'Cell', 'dying', 'at', '4', 'Hertz', '.'],\n",
       " ['temperature', '37', '.'],\n",
       " ['the', 'volume', 'is', '500', 'µl', '.', '500', 'µl', '.'],\n",
       " ['total', '', '1434', 'µl', 'of', 'HEPES', 'buffer', 'was', 'added', '.'],\n",
       " ['reaction', 'mass', 'into', '2', 'portions', '.'],\n",
       " ['cabinet',\n",
       "  'on',\n",
       "  'right',\n",
       "  'hand',\n",
       "  'side',\n",
       "  '.',\n",
       "  'Top',\n",
       "  'shelf',\n",
       "  '',\n",
       "  'sex',\n",
       "  'of',\n",
       "  'Finn',\n",
       "  '',\n",
       "  'sets',\n",
       "  'of',\n",
       "  'FinnTip',\n",
       "  '1096',\n",
       "  'well',\n",
       "  'racks',\n",
       "  '.',\n",
       "  'Set',\n",
       "  'of',\n",
       "  '250',\n",
       "  'µl',\n",
       "  'universal',\n",
       "  'refill',\n",
       "  'kits',\n",
       "  '20',\n",
       "  '*',\n",
       "  '96',\n",
       "  'pieces',\n",
       "  'per',\n",
       "  'real',\n",
       "  'refill',\n",
       "  '4',\n",
       "  'boxes',\n",
       "  '.',\n",
       "  'Second',\n",
       "  'shelf',\n",
       "  '',\n",
       "  'more',\n",
       "  'disposable',\n",
       "  'tips',\n",
       "  '',\n",
       "  'not',\n",
       "  'in',\n",
       "  'use',\n",
       "  '.',\n",
       "  'Bottom',\n",
       "  '',\n",
       "  'set',\n",
       "  'up',\n",
       "  'disposable',\n",
       "  'tips',\n",
       "  'also',\n",
       "  'not',\n",
       "  'use',\n",
       "  'and'],\n",
       " ['eluting',\n",
       "  'from',\n",
       "  'Strep',\n",
       "  '-',\n",
       "  'Tactin',\n",
       "  'column',\n",
       "  'for',\n",
       "  'experiment',\n",
       "  '67',\n",
       "  'with',\n",
       "  '3',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'Apo',\n",
       "  'elution',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Allow',\n",
       "  'it',\n",
       "  'to',\n",
       "  'do',\n",
       "  'that',\n",
       "  'by',\n",
       "  'gravity',\n",
       "  'flow',\n",
       "  '.'],\n",
       " ['7',\n",
       "  '4',\n",
       "  'µl',\n",
       "  'Albamine',\n",
       "  'And',\n",
       "  'lane',\n",
       "  '9',\n",
       "  '.',\n",
       "  '4',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'cynomologous',\n",
       "  'Serum',\n",
       "  'albumine',\n",
       "  '.'],\n",
       " ['66',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'ATP',\n",
       "  'and',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'deoxycholic',\n",
       "  'acid',\n",
       "  '.'],\n",
       " ['2',\n",
       "  'µg',\n",
       "  'of',\n",
       "  'Cas9',\n",
       "  'protein',\n",
       "  'and',\n",
       "  '2',\n",
       "  'µg',\n",
       "  'of',\n",
       "  'modified',\n",
       "  'Cas9',\n",
       "  'protein',\n",
       "  'will',\n",
       "  'be',\n",
       "  'digested',\n",
       "  'with',\n",
       "  'trypsin',\n",
       "  'overnight',\n",
       "  '.',\n",
       "  'The',\n",
       "  'proteins',\n",
       "  'will',\n",
       "  'not',\n",
       "  'be',\n",
       "  'reduced',\n",
       "  'and',\n",
       "  'activated',\n",
       "  'prior',\n",
       "  'to',\n",
       "  'trypsinization',\n",
       "  '.'],\n",
       " ['for',\n",
       "  'experiment',\n",
       "  '67',\n",
       "  '.',\n",
       "  'Another',\n",
       "  'and',\n",
       "  'centrifugation',\n",
       "  '10',\n",
       "  'min',\n",
       "  'at',\n",
       "  '400',\n",
       "  '4000',\n",
       "  'RPM',\n",
       "  'in',\n",
       "  'Milton',\n",
       "  'fuels',\n",
       "  '3',\n",
       "  'both',\n",
       "  'arrays',\n",
       "  '.'],\n",
       " ['pipetted', '5', 'µl', 'into', 'well', 'a1', '.'],\n",
       " ['cells',\n",
       "  'viability',\n",
       "  '62.2',\n",
       "  '%',\n",
       "  '.',\n",
       "  'Live',\n",
       "  'cells',\n",
       "  '8.31',\n",
       "  '*',\n",
       "  '10',\n",
       "  '^',\n",
       "  '5',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['did',\n",
       "  'a',\n",
       "  'colony',\n",
       "  'PCR',\n",
       "  'of',\n",
       "  'the',\n",
       "  'second',\n",
       "  'recombination',\n",
       "  'of',\n",
       "  'BAC',\n",
       "  '16',\n",
       "  'gB',\n",
       "  'Delta',\n",
       "  'RGD',\n",
       "  'to',\n",
       "  'test',\n",
       "  'if',\n",
       "  'the',\n",
       "  'Kana',\n",
       "  'cassette',\n",
       "  'is',\n",
       "  'removed',\n",
       "  '.'],\n",
       " ['round', '5D5', 'full', 'length', '25', '.'],\n",
       " ['incubator', 'for', 'project', '15', '.'],\n",
       " ['180',\n",
       "  'µl',\n",
       "  'Evergreen',\n",
       "  'Supermix',\n",
       "  '',\n",
       "  '3.6',\n",
       "  'µl',\n",
       "  'forward',\n",
       "  'primer',\n",
       "  '',\n",
       "  '3.6',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'reverse',\n",
       "  'primer',\n",
       "  '',\n",
       "  '82.8',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'ultrapure',\n",
       "  'water',\n",
       "  '.'],\n",
       " ['am',\n",
       "  'starting',\n",
       "  'RNA',\n",
       "  'extraction',\n",
       "  'now',\n",
       "  '.',\n",
       "  'Yesterday',\n",
       "  'I',\n",
       "  'have',\n",
       "  'homogenized',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'with',\n",
       "  'TRizol',\n",
       "  '.',\n",
       "  'I',\n",
       "  'used',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'TRizol',\n",
       "  'for',\n",
       "  '3',\n",
       "  'wells',\n",
       "  'in',\n",
       "  'a',\n",
       "  '24',\n",
       "  'well',\n",
       "  'plate',\n",
       "  '.'],\n",
       " ['46', '3.259', 'E5', '.'],\n",
       " ['difficult',\n",
       "  'words',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lab',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['',\n",
       "  'process',\n",
       "  'midi',\n",
       "  'for',\n",
       "  'S1',\n",
       "  'construct',\n",
       "  'number',\n",
       "  '6',\n",
       "  '.',\n",
       "  'Second',\n",
       "  '',\n",
       "  'prepare',\n",
       "  'RNA',\n",
       "  'and',\n",
       "  'protein',\n",
       "  'LiSET',\n",
       "  \"'s\",\n",
       "  'for',\n",
       "  'human',\n",
       "  'fibroblasts',\n",
       "  'seated',\n",
       "  'on',\n",
       "  'Monday',\n",
       "  '',\n",
       "  '20th',\n",
       "  'of',\n",
       "  'August',\n",
       "  '.',\n",
       "  'Third',\n",
       "  '',\n",
       "  'analyze',\n",
       "  'adipocyte',\n",
       "  'differentiation',\n",
       "  'that',\n",
       "  'are',\n",
       "  'being',\n",
       "  'performed',\n",
       "  'on',\n",
       "  'mouse',\n",
       "  'and',\n",
       "  'browning',\n",
       "  'pepper',\n",
       "  'blasts',\n",
       "  '.',\n",
       "  'Process',\n",
       "  'LN9SV',\n",
       "  'in',\n",
       "  'modulised',\n",
       "  'fibroblasts',\n",
       "  'saline',\n",
       "  'for',\n",
       "  'transfection',\n",
       "  'experiments',\n",
       "  '.',\n",
       "  'Set',\n",
       "  'up',\n",
       "  'PCR',\n",
       "  'for',\n",
       "  'new',\n",
       "  'patients',\n",
       "  '.'],\n",
       " ['number', '3', '.'],\n",
       " ['have',\n",
       "  'decided',\n",
       "  'to',\n",
       "  'use',\n",
       "  'channel',\n",
       "  '3',\n",
       "  'for',\n",
       "  'substrate',\n",
       "  '.',\n",
       "  'Channel',\n",
       "  '4',\n",
       "  '.',\n",
       "  'Sorry',\n",
       "  '.',\n",
       "  'Repeat',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'No',\n",
       "  '',\n",
       "  'no',\n",
       "  '',\n",
       "  'I',\n",
       "  \"'ve\",\n",
       "  'decided',\n",
       "  'to',\n",
       "  'use',\n",
       "  'channel',\n",
       "  '3',\n",
       "  'for',\n",
       "  'ATP',\n",
       "  '',\n",
       "  'Channel',\n",
       "  '4',\n",
       "  'for',\n",
       "  'substrate',\n",
       "  'and',\n",
       "  'again',\n",
       "  'channel',\n",
       "  '3',\n",
       "  'for',\n",
       "  'both',\n",
       "  'enzymes',\n",
       "  '.'],\n",
       " ['wells', 'total', 'used', '10', 'ml', 'of', 'cells', 'to', 'populate', '.'],\n",
       " ['ml',\n",
       "  'to',\n",
       "  'well',\n",
       "  '3',\n",
       "  'a',\n",
       "  'big',\n",
       "  'well',\n",
       "  '-',\n",
       "  'It',\n",
       "  \"'s\",\n",
       "  'a',\n",
       "  'big',\n",
       "  'well',\n",
       "  '-',\n",
       "  'Oh',\n",
       "  '',\n",
       "  'it',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'go',\n",
       "  '.',\n",
       "  'Wait',\n",
       "  '.',\n",
       "  'Does',\n",
       "  \"n't\",\n",
       "  'wanna',\n",
       "  'order',\n",
       "  'me',\n",
       "  'anything',\n",
       "  'let',\n",
       "  \"'s\",\n",
       "  'say',\n",
       "  '-',\n",
       "  'Avocado',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'mention',\n",
       "  'we',\n",
       "  'have',\n",
       "  'to',\n",
       "  'change',\n",
       "  'the',\n",
       "  'flasks',\n",
       "  'for',\n",
       "  'higher',\n",
       "  'volumes',\n",
       "  '',\n",
       "  'so',\n",
       "  'we',\n",
       "  'changed',\n",
       "  'from',\n",
       "  '125',\n",
       "  'milliliter',\n",
       "  'flasks',\n",
       "  'to',\n",
       "  '150',\n",
       "  'milliliter',\n",
       "  'flasks',\n",
       "  '.'],\n",
       " ['KP59331',\n",
       "  'viability',\n",
       "  '67.1',\n",
       "  '%',\n",
       "  'concentration',\n",
       "  '1.02',\n",
       "  '',\n",
       "  '*',\n",
       "  '10',\n",
       "  '^',\n",
       "  '6',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['antibody',\n",
       "  'is',\n",
       "  'buffer',\n",
       "  'exchange',\n",
       "  'using',\n",
       "  '50',\n",
       "  'kDa',\n",
       "  'molecular',\n",
       "  'weight',\n",
       "  'cut',\n",
       "  'off',\n",
       "  'filters',\n",
       "  'by',\n",
       "  'washing',\n",
       "  'the',\n",
       "  'solution',\n",
       "  '3',\n",
       "  'times',\n",
       "  'with',\n",
       "  'PBS',\n",
       "  '.',\n",
       "  'After',\n",
       "  'that',\n",
       "  '',\n",
       "  'the',\n",
       "  'antibody',\n",
       "  'solution',\n",
       "  '100',\n",
       "  'µl',\n",
       "  'is',\n",
       "  'mixed',\n",
       "  'with',\n",
       "  '2.67',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sulfur',\n",
       "  'tag',\n",
       "  'and',\n",
       "  'leave',\n",
       "  'to',\n",
       "  'react',\n",
       "  'for',\n",
       "  '2',\n",
       "  'h',\n",
       "  'in',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'protected',\n",
       "  'from',\n",
       "  'light',\n",
       "  '.'],\n",
       " ['2', 'days', 'do', 'mini', 'preps', '.'],\n",
       " ['up',\n",
       "  'include',\n",
       "  'diluted',\n",
       "  'into',\n",
       "  'new',\n",
       "  'Aliquotes',\n",
       "  'per',\n",
       "  'for',\n",
       "  '15',\n",
       "  'µl',\n",
       "  'kept',\n",
       "  'on',\n",
       "  'ice',\n",
       "  'as',\n",
       "  'long',\n",
       "  'as',\n",
       "  'possible',\n",
       "  'further',\n",
       "  'activity',\n",
       "  'has',\n",
       "  'to',\n",
       "  'be',\n",
       "  'proved',\n",
       "  '.'],\n",
       " ['11:20',\n",
       "  'started',\n",
       "  'incubation',\n",
       "  'of',\n",
       "  'all',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'with',\n",
       "  '2',\n",
       "  'ml',\n",
       "  'slurry',\n",
       "  'of',\n",
       "  'the',\n",
       "  'Amgen',\n",
       "  'nickel',\n",
       "  'loaded',\n",
       "  'magnetic',\n",
       "  'beads',\n",
       "  'which',\n",
       "  'had',\n",
       "  'been',\n",
       "  'washed',\n",
       "  'twice',\n",
       "  '.',\n",
       "  'The',\n",
       "  'beads',\n",
       "  'had',\n",
       "  'previously',\n",
       "  'been',\n",
       "  'used',\n",
       "  '4',\n",
       "  'times',\n",
       "  'were',\n",
       "  'stored',\n",
       "  'in',\n",
       "  '20',\n",
       "  '%',\n",
       "  'ethanol',\n",
       "  '.',\n",
       "  'Rotation',\n",
       "  '.'],\n",
       " ['110044394',\n",
       "  '.',\n",
       "  'NK',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'Make',\n",
       "  'media',\n",
       "  'much',\n",
       "  'more',\n",
       "  'yellow',\n",
       "  '.'],\n",
       " ['volume', 'is', '3.2', 'ml', '.'],\n",
       " ['spheroids',\n",
       "  'in',\n",
       "  'Corning',\n",
       "  '3830',\n",
       "  'ultra',\n",
       "  'low',\n",
       "  'attachment',\n",
       "  'plates',\n",
       "  '384',\n",
       "  'well',\n",
       "  '.'],\n",
       " ['a',\n",
       "  'pinch',\n",
       "  'of',\n",
       "  'lysozyme',\n",
       "  'from',\n",
       "  'Roche',\n",
       "  '837059',\n",
       "  '',\n",
       "  'Lot',\n",
       "  '93141822',\n",
       "  '',\n",
       "  'July',\n",
       "  '06',\n",
       "  '.'],\n",
       " ['blotting', 'finished', 'at', '6:20', '.'],\n",
       " ['should',\n",
       "  'be',\n",
       "  'an',\n",
       "  'option',\n",
       "  'for',\n",
       "  'previous',\n",
       "  'builds',\n",
       "  'you',\n",
       "  'see',\n",
       "  'that',\n",
       "  '',\n",
       "  'yeah',\n",
       "  'true',\n",
       "  'but',\n",
       "  'then',\n",
       "  'I',\n",
       "  'have',\n",
       "  '3',\n",
       "  '3'],\n",
       " ['found',\n",
       "  '2',\n",
       "  'batches',\n",
       "  'of',\n",
       "  'protein',\n",
       "  'in',\n",
       "  '-80',\n",
       "  'and',\n",
       "  'I',\n",
       "  'am',\n",
       "  'using',\n",
       "  'the',\n",
       "  'previous',\n",
       "  'Batch',\n",
       "  '.',\n",
       "  'I',\n",
       "  'have',\n",
       "  \"n't\",\n",
       "  'tested',\n",
       "  'yet',\n",
       "  'the',\n",
       "  'new',\n",
       "  'batch',\n",
       "  'of',\n",
       "  'protein',\n",
       "  '.',\n",
       "  'So',\n",
       "  'for',\n",
       "  'this',\n",
       "  'assay',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'the',\n",
       "  'old',\n",
       "  'batch',\n",
       "  'that',\n",
       "  'I',\n",
       "  'am',\n",
       "  'using'],\n",
       " ['columns',\n",
       "  'strep',\n",
       "  'tactin',\n",
       "  'columns',\n",
       "  'are',\n",
       "  'moved',\n",
       "  'to',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'and',\n",
       "  'they',\n",
       "  'will',\n",
       "  'be',\n",
       "  'regenerated',\n",
       "  '.',\n",
       "  'Note',\n",
       "  'that',\n",
       "  'on',\n",
       "  'the',\n",
       "  '66',\n",
       "  'experiment',\n",
       "  'with',\n",
       "  'Apo',\n",
       "  'it',\n",
       "  'looks',\n",
       "  'like',\n",
       "  'there',\n",
       "  'may',\n",
       "  'be',\n",
       "  'some',\n",
       "  'aggregation',\n",
       "  'at',\n",
       "  'the',\n",
       "  'bottom',\n",
       "  'of',\n",
       "  'the',\n",
       "  'column',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'ll\",\n",
       "  'have',\n",
       "  'to',\n",
       "  'carefully',\n",
       "  'remove',\n",
       "  'that',\n",
       "  '.'],\n",
       " ['standard',\n",
       "  'curve',\n",
       "  'is',\n",
       "  'prepared',\n",
       "  'in',\n",
       "  'a',\n",
       "  'in',\n",
       "  'a',\n",
       "  'pure',\n",
       "  'assay',\n",
       "  'buffer',\n",
       "  'and',\n",
       "  'the',\n",
       "  'Q',\n",
       "  'Standard',\n",
       "  'curve',\n",
       "  'is',\n",
       "  'prepared',\n",
       "  'in',\n",
       "  '10',\n",
       "  '%',\n",
       "  'of',\n",
       "  'cell',\n",
       "  'lysate',\n",
       "  'mixed',\n",
       "  'with',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'extraction',\n",
       "  'buffer',\n",
       "  '.'],\n",
       " ['-', '1', 'images', 'ending', 'an', '09225', '.'],\n",
       " ['Nephric', '?', ']', 'samples', 'from', '245322458', '.'],\n",
       " ['µl', '100', 'mM', 'Tris', 'HCL', 'to', 'both', 'lanes', '.'],\n",
       " ['.',\n",
       "  'Set',\n",
       "  'up',\n",
       "  'a',\n",
       "  'reminder',\n",
       "  'for',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '2:00',\n",
       "  'PM',\n",
       "  'to',\n",
       "  'organize',\n",
       "  'test',\n",
       "  'for',\n",
       "  'labtwin',\n",
       "  '.'],\n",
       " ['Sf9',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'order',\n",
       "  'to',\n",
       "  'start',\n",
       "  'expression',\n",
       "  'of',\n",
       "  'test',\n",
       "  'expressions',\n",
       "  'of',\n",
       "  'MC4',\n",
       "  'construct',\n",
       "  '2',\n",
       "  '.',\n",
       "  'Therefore',\n",
       "  '200',\n",
       "  'mL',\n",
       "  'cells',\n",
       "  'are',\n",
       "  'stored',\n",
       "  'in',\n",
       "  '4',\n",
       "  'flasks',\n",
       "  'as',\n",
       "  '4',\n",
       "  'mL',\n",
       "  'and',\n",
       "  'they',\n",
       "  'will',\n",
       "  'now',\n",
       "  'be',\n",
       "  'collected',\n",
       "  'in',\n",
       "  'one',\n",
       "  '500',\n",
       "  'mL',\n",
       "  'flasks',\n",
       "  '.',\n",
       "  'Then',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'are',\n",
       "  'counted',\n",
       "  'and',\n",
       "  'further',\n",
       "  'to',\n",
       "  '8',\n",
       "  'flasks',\n",
       "  'at',\n",
       "  '1',\n",
       "  'million',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'flasks',\n",
       "  '.'],\n",
       " ['did',\n",
       "  'yesterday',\n",
       "  'the',\n",
       "  'first',\n",
       "  'recombination',\n",
       "  'of',\n",
       "  'this',\n",
       "  'mScarlet',\n",
       "  'Strep',\n",
       "  '-',\n",
       "  'cassette',\n",
       "  '.',\n",
       "  'In',\n",
       "  'the',\n",
       "  'BAC',\n",
       "  '',\n",
       "  'um',\n",
       "  '',\n",
       "  'in',\n",
       "  'the',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'Bacmids',\n",
       "  'Gao',\n",
       "  'RRV',\n",
       "  'mScarlet',\n",
       "  'orf',\n",
       "  '65',\n",
       "  'and',\n",
       "  'GaoRRV',\n",
       "  'mScarlet',\n",
       "  'orf',\n",
       "  '65',\n",
       "  'orf',\n",
       "  '22',\n",
       "  '[',\n",
       "  '.',\n",
       "  'I',\n",
       "  'ca',\n",
       "  \"n't\",\n",
       "  'see',\n",
       "  'single',\n",
       "  'colonies',\n",
       "  '.',\n",
       "  'Could',\n",
       "  'be',\n",
       "  'the',\n",
       "  'problem',\n",
       "  'the',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  'the',\n",
       "  'kanamycin',\n",
       "  'on',\n",
       "  'LB-',\n",
       "  'plates',\n",
       "  '?',\n",
       "  'Or',\n",
       "  'is',\n",
       "  'the',\n",
       "  'efficiency',\n",
       "  'so',\n",
       "  'good',\n",
       "  '.',\n",
       "  'So',\n",
       "  'I',\n",
       "  'have',\n",
       "  'to',\n",
       "  'plate',\n",
       "  'bacterias',\n",
       "  'again',\n",
       "  'on',\n",
       "  'fresh',\n",
       "  'plates',\n",
       "  'with',\n",
       "  'kanamycin',\n",
       "  '15',\n",
       "  'µg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['pipette',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'using',\n",
       "  'is',\n",
       "  'P100',\n",
       "  'at',\n",
       "  'a',\n",
       "  '0.73',\n",
       "  'µg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  'dilution',\n",
       "  '.'],\n",
       " ['me', 'to', 'check', 'on', 'sample', '3', '.'],\n",
       " ['tubes', 'A5', 'to', '12', 'for'],\n",
       " ['2', '465', 'ng', '/', 'ml', '.'],\n",
       " ['PBS', 'plus', 'DMSO', 'plus', 'FBS', 'to', 'column', '5', 'and', '10', '.'],\n",
       " ['biological',\n",
       "  'replicate',\n",
       "  'should',\n",
       "  'be',\n",
       "  'run',\n",
       "  'with',\n",
       "  '3',\n",
       "  'technical',\n",
       "  'replicates',\n",
       "  'and',\n",
       "  '3',\n",
       "  'different',\n",
       "  'stage',\n",
       "  '.',\n",
       "  'Plus',\n",
       "  '',\n",
       "  'the',\n",
       "  'two',\n",
       "  'housekeeping',\n",
       "  'genes'],\n",
       " ['µl',\n",
       "  'of',\n",
       "  'protein',\n",
       "  'at',\n",
       "  'different',\n",
       "  'concentrations',\n",
       "  'was',\n",
       "  'added',\n",
       "  'to',\n",
       "  'plate',\n",
       "  '.'],\n",
       " ['me', 'to', 'give', 'a', 'talk', 'at', '9', 'AM', 'today', '.'],\n",
       " ['40', 'mg', '/', 'ml', '.'],\n",
       " ['3', '', '1.608', 'E5', '.'],\n",
       " ['thawed', 'my', 'competent', 'cells', 'for', '20', 'seconds', '.'],\n",
       " ['.',\n",
       "  'Slide',\n",
       "  '97',\n",
       "  '.',\n",
       "  'Ileum',\n",
       "  '.',\n",
       "  'No',\n",
       "  'abnormalities',\n",
       "  'detected',\n",
       "  '.',\n",
       "  'Cecum',\n",
       "  '.',\n",
       "  'No',\n",
       "  'abnormalities',\n",
       "  'detected',\n",
       "  '.',\n",
       "  'Peyer',\n",
       "  \"'s\",\n",
       "  'Patch',\n",
       "  '.',\n",
       "  'No',\n",
       "  'abnormalities',\n",
       "  'detected',\n",
       "  '.'],\n",
       " ['ID',\n",
       "  '18',\n",
       "  'S00629',\n",
       "  'sample',\n",
       "  'value',\n",
       "  '3.705',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  'ID',\n",
       "  '19',\n",
       "  'S00460',\n",
       "  'sample',\n",
       "  'value',\n",
       "  '2.00',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'Bullet',\n",
       "  'Sample',\n",
       "  'ID',\n",
       "  '18',\n",
       "  'S006286',\n",
       "  'sample',\n",
       "  'value',\n",
       "  '16.80',\n",
       "  'ng/µl',\n",
       "  '.'],\n",
       " ['500', 'to', 'cell', 'A', '1', '.'],\n",
       " ['forgot',\n",
       "  'her',\n",
       "  'sheets',\n",
       "  'in',\n",
       "  'the',\n",
       "  'other',\n",
       "  'lab',\n",
       "  'when',\n",
       "  'she',\n",
       "  'went',\n",
       "  'towards',\n",
       "  'GL51',\n",
       "  '.'],\n",
       " ['19928001', '.'],\n",
       " ['aliquot',\n",
       "  'the',\n",
       "  'TCV',\n",
       "  'catalog',\n",
       "  'number',\n",
       "  '72254',\n",
       "  '',\n",
       "  'it',\n",
       "  'comes',\n",
       "  'as',\n",
       "  '5',\n",
       "  'mg',\n",
       "  '',\n",
       "  'which',\n",
       "  'gets',\n",
       "  'aliquotted',\n",
       "  'into',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'aliquots',\n",
       "  '.',\n",
       "  'Resuspend',\n",
       "  'the',\n",
       "  '5',\n",
       "  'mg',\n",
       "  'in',\n",
       "  '1.6',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['3',\n",
       "  '.',\n",
       "  'Cm',\n",
       "  '=',\n",
       "  '54',\n",
       "  '.',\n",
       "  'R',\n",
       "  '=',\n",
       "  '2.9',\n",
       "  '.',\n",
       "  'A',\n",
       "  'nice',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  '.',\n",
       "  'Also',\n",
       "  'the',\n",
       "  'second',\n",
       "  'caffeine',\n",
       "  'was',\n",
       "  'also',\n",
       "  'very',\n",
       "  'good',\n",
       "  '.',\n",
       "  'However',\n",
       "  '',\n",
       "  'the',\n",
       "  'same',\n",
       "  'problem',\n",
       "  'as',\n",
       "  'experienced',\n",
       "  'with',\n",
       "  'increasing',\n",
       "  'series',\n",
       "  'resistance',\n",
       "  '',\n",
       "  'which',\n",
       "  'may',\n",
       "  'not',\n",
       "  'necessarily',\n",
       "  'be',\n",
       "  'as',\n",
       "  'a',\n",
       "  'result',\n",
       "  'of',\n",
       "  'the',\n",
       "  'volume',\n",
       "  'of',\n",
       "  'the',\n",
       "  'fluid',\n",
       "  'in',\n",
       "  'the',\n",
       "  'bath',\n",
       "  'or',\n",
       "  'because',\n",
       "  'of',\n",
       "  'drifting',\n",
       "  'of',\n",
       "  'the',\n",
       "  'pipette',\n",
       "  '',\n",
       "  'thereby',\n",
       "  'preventing',\n",
       "  'me',\n",
       "  'from',\n",
       "  'doing',\n",
       "  'isoprenaline',\n",
       "  'experiments',\n",
       "  '.',\n",
       "  'The',\n",
       "  'maximum',\n",
       "  'and',\n",
       "  'minimum',\n",
       "  '.',\n",
       "  'Was',\n",
       "  'a',\n",
       "  'little',\n",
       "  'tricky',\n",
       "  'be',\n",
       "  'cause',\n",
       "  'the',\n",
       "  'pipette',\n",
       "  'was',\n",
       "  'broken',\n",
       "  'at',\n",
       "  'the',\n",
       "  'minimum',\n",
       "  'stage',\n",
       "  '.',\n",
       "  'However',\n",
       "  '',\n",
       "  'after',\n",
       "  'prolonged',\n",
       "  'perfusion',\n",
       "  '',\n",
       "  'The',\n",
       "  'minimum',\n",
       "  'was',\n",
       "  'restored',\n",
       "  'back',\n",
       "  'to',\n",
       "  'basal',\n",
       "  'level',\n",
       "  '.'],\n",
       " ['count',\n",
       "  'is',\n",
       "  '1.5',\n",
       "  'E5',\n",
       "  '',\n",
       "  'which',\n",
       "  'is',\n",
       "  'in',\n",
       "  '75',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'media',\n",
       "  '',\n",
       "  'which',\n",
       "  'translates',\n",
       "  'to',\n",
       "  'roughly',\n",
       "  'double',\n",
       "  'the',\n",
       "  'expected',\n",
       "  'number',\n",
       "  'of',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'vial',\n",
       "  '.',\n",
       "  'We',\n",
       "  'think',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'the',\n",
       "  'wrong',\n",
       "  'cell',\n",
       "  'count',\n",
       "  'so',\n",
       "  'we',\n",
       "  'continuing',\n",
       "  'as',\n",
       "  'normal',\n",
       "  '',\n",
       "  'adding',\n",
       "  '250',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'media',\n",
       "  '.'],\n",
       " ['number', 'HU', '1642', '.'],\n",
       " ['count',\n",
       "  'for',\n",
       "  'HepG2',\n",
       "  'C3A',\n",
       "  'cells',\n",
       "  'its',\n",
       "  '2.09',\n",
       "  '*',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '6th',\n",
       "  '.',\n",
       "  'Viable',\n",
       "  'cells',\n",
       "  '1.85',\n",
       "  '*',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '6th',\n",
       "  '.',\n",
       "  'Viability',\n",
       "  'is',\n",
       "  '88.6',\n",
       "  '%',\n",
       "  '.'],\n",
       " ['weighed',\n",
       "  'into',\n",
       "  'g',\n",
       "  'of',\n",
       "  'Mgcl',\n",
       "  '2',\n",
       "  '.',\n",
       "  'In',\n",
       "  '200',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'water',\n",
       "  '.',\n",
       "  'I',\n",
       "  'said',\n",
       "  '100',\n",
       "  'ml',\n",
       "  'photo',\n",
       "  '.'],\n",
       " ['', 'order', '20', 'Falcon', 'tubes', '.'],\n",
       " ['11', 'combination', '3', 'media', 'added', '.'],\n",
       " ['pellet', 'size', ':', '288', 'mg', '.'],\n",
       " ['lowering',\n",
       "  'the',\n",
       "  'aspirate',\n",
       "  'to',\n",
       "  'forge',\n",
       "  'it',\n",
       "  'does',\n",
       "  'not',\n",
       "  'improve',\n",
       "  'the',\n",
       "  'pattern',\n",
       "  '',\n",
       "  'reversing',\n",
       "  'back',\n",
       "  'to',\n",
       "  '55',\n",
       "  'original',\n",
       "  'settings',\n",
       "  '.',\n",
       "  'OK',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'one',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'steered',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transfered',\n",
       "  'to',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['that',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'incubated',\n",
       "  'at',\n",
       "  '600',\n",
       "  'RPM',\n",
       "  'during',\n",
       "  'all',\n",
       "  'the',\n",
       "  'incubation',\n",
       "  \"'s\",\n",
       "  'in',\n",
       "  'the',\n",
       "  'protocol',\n",
       "  '.'],\n",
       " ['17', '', '1.75', 'extend', 'til', '5', '.'],\n",
       " ['about', '150ML', 'increased', 'to', '2', '%'],\n",
       " ['11',\n",
       "  'with',\n",
       "  'headphones',\n",
       "  'talking',\n",
       "  'in',\n",
       "  'the',\n",
       "  'background',\n",
       "  'not',\n",
       "  'too',\n",
       "  'loud',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '10',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'to',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotavap',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'New',\n",
       "  'paragraph',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['.',\n",
       "  'Create',\n",
       "  'a',\n",
       "  'reminder',\n",
       "  'for',\n",
       "  'tomorrow',\n",
       "  '9:00',\n",
       "  'AM',\n",
       "  'to',\n",
       "  'call',\n",
       "  'people',\n",
       "  'from',\n",
       "  'the',\n",
       "  'team',\n",
       "  'in',\n",
       "  'Valenciennes'],\n",
       " ['number',\n",
       "  'one',\n",
       "  '.',\n",
       "  'Step',\n",
       "  'one',\n",
       "  'preparation',\n",
       "  'of',\n",
       "  'reagents',\n",
       "  '.',\n",
       "  'Um',\n",
       "  '',\n",
       "  'preparation',\n",
       "  'of',\n",
       "  'lysis',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Um',\n",
       "  '',\n",
       "  'step',\n",
       "  'one',\n",
       "  'pipet',\n",
       "  '65',\n",
       "  'μl',\n",
       "  'nuclease',\n",
       "  'free',\n",
       "  'water',\n",
       "  '.'],\n",
       " ['902.97', 'g', 'water', 'and', '234', 'g', 'poloxamer', '407', '.'],\n",
       " ['an',\n",
       "  'appointment',\n",
       "  'on',\n",
       "  'Wednesday',\n",
       "  'morning',\n",
       "  'at',\n",
       "  '9',\n",
       "  'with',\n",
       "  'Mobies',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'check',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'confluence',\n",
       "  'on',\n",
       "  'Saturday',\n",
       "  'at',\n",
       "  '9:00',\n",
       "  '.'],\n",
       " ['4', 'µl', 'of', 'light', 'citrate', '.'],\n",
       " ['HCL', 'pH', '8', 'Medicago', '12', '-', '9', '.'],\n",
       " ['10',\n",
       "  'ml',\n",
       "  '1',\n",
       "  'molar',\n",
       "  'imidazole',\n",
       "  'to',\n",
       "  'the',\n",
       "  'wash',\n",
       "  'buffer',\n",
       "  'to',\n",
       "  'read',\n",
       "  '5',\n",
       "  'mM.',\n",
       "  'And',\n",
       "  'add'],\n",
       " ['inaudible', ']', '16', 'A', '?'],\n",
       " ['1',\n",
       "  'viscosity',\n",
       "  '0.2',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  '2',\n",
       "  'viscosity',\n",
       "  '2.5',\n",
       "  '.',\n",
       "  'Sample',\n",
       "  '3',\n",
       "  'viscosity',\n",
       "  '5.6',\n",
       "  '.'],\n",
       " ['me', 'to', 'call', 'Sarah', 'tomorrow', 'at', '2', 'PM', '.'],\n",
       " ['jabra',\n",
       "  'English',\n",
       "  'radio',\n",
       "  'in',\n",
       "  'the',\n",
       "  'background',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'steered',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  'the',\n",
       "  'face',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'rotarvap',\n",
       "  'up',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbars',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end'],\n",
       " ['gel',\n",
       "  'activation',\n",
       "  'for',\n",
       "  '2.5',\n",
       "  'min',\n",
       "  'on',\n",
       "  'the',\n",
       "  'BioRAD',\n",
       "  'easy',\n",
       "  'imager',\n",
       "  '.'],\n",
       " ['homogenised',\n",
       "  'rat',\n",
       "  'liver',\n",
       "  'is',\n",
       "  'sonicated',\n",
       "  'in',\n",
       "  'the',\n",
       "  'sonication',\n",
       "  'bath',\n",
       "  'in',\n",
       "  'AC1',\n",
       "  'west',\n",
       "  'using',\n",
       "  'these',\n",
       "  'sweep',\n",
       "  'function',\n",
       "  'for',\n",
       "  '3',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['delivered',\n",
       "  'IgfR',\n",
       "  '1Rtide',\n",
       "  '.',\n",
       "  'Catalog',\n",
       "  'number',\n",
       "  '.',\n",
       "  'BML',\n",
       "  '-',\n",
       "  'P257',\n",
       "  '-',\n",
       "  '001',\n",
       "  '.',\n",
       "  'Store',\n",
       "  'at',\n",
       "  '-20',\n",
       "  '°',\n",
       "  'in',\n",
       "  'freezer',\n",
       "  '.'],\n",
       " ['liver', '', 'no'],\n",
       " ['160', 'µl', 'of', 'cholic', 'acid', 'internal', 'standard', '.'],\n",
       " ['me', 'to', 'put', 'it', 'in', 'order', 'for', 'CD19', 'antibody', '.'],\n",
       " ['in', '10', 'minute', 'countdown', '.'],\n",
       " ['ml',\n",
       "  'were',\n",
       "  'added',\n",
       "  'to',\n",
       "  '5',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'water',\n",
       "  'and',\n",
       "  'afterwards',\n",
       "  'the',\n",
       "  'DNA',\n",
       "  'was',\n",
       "  'about',\n",
       "  '2',\n",
       "  'mM.'],\n",
       " ['primers',\n",
       "  'for',\n",
       "  'tubulin',\n",
       "  'they',\n",
       "  'have',\n",
       "  'a',\n",
       "  'melting',\n",
       "  'temperature',\n",
       "  'or',\n",
       "  '72',\n",
       "  '°',\n",
       "  'or',\n",
       "  'higher',\n",
       "  '',\n",
       "  'so',\n",
       "  'I',\n",
       "  'need',\n",
       "  'to',\n",
       "  'perform',\n",
       "  'a',\n",
       "  '2',\n",
       "  'Step',\n",
       "  'PCR',\n",
       "  'reaction',\n",
       "  '.',\n",
       "  'And',\n",
       "  'therefore',\n",
       "  'I',\n",
       "  'will',\n",
       "  'do',\n",
       "  'the',\n",
       "  'reaction',\n",
       "  'for',\n",
       "  'this',\n",
       "  'primers',\n",
       "  'separately',\n",
       "  'from',\n",
       "  'the',\n",
       "  'rest',\n",
       "  '.'],\n",
       " ['delivered',\n",
       "  'polyglycine',\n",
       "  'tyrosine',\n",
       "  'sodium',\n",
       "  'salt',\n",
       "  '100',\n",
       "  'mg',\n",
       "  '.',\n",
       "  'P0275-',\n",
       "  '100',\n",
       "  'mg',\n",
       "  'stored',\n",
       "  'at',\n",
       "  '-',\n",
       "  '20',\n",
       "  '°',\n",
       "  'C'],\n",
       " ['rigid', '61', 'µl', '.'],\n",
       " ['this', 'command', 'to', 'experiment', 'number', '02055', '.'],\n",
       " ['',\n",
       "  'check',\n",
       "  'fractions',\n",
       "  'E12',\n",
       "  'to',\n",
       "  'F9',\n",
       "  'and',\n",
       "  'G5',\n",
       "  '',\n",
       "  'sorry',\n",
       "  '',\n",
       "  'G3',\n",
       "  'to',\n",
       "  'G9',\n",
       "  '.'],\n",
       " ['media', 'with', 'FBS', 'lot', '160316', '.', 'Dragonfly', '.'],\n",
       " ['to',\n",
       "  'pre',\n",
       "  'pulse',\n",
       "  'inhibition',\n",
       "  'article',\n",
       "  'in',\n",
       "  'Wikipedia',\n",
       "  '.',\n",
       "  'Background',\n",
       "  'white',\n",
       "  'noise',\n",
       "  'is',\n",
       "  '30',\n",
       "  'to',\n",
       "  '40',\n",
       "  'decibels',\n",
       "  'in',\n",
       "  'rodent',\n",
       "  'experiment',\n",
       "  '.',\n",
       "  'And',\n",
       "  'the',\n",
       "  'starting',\n",
       "  'noise',\n",
       "  'is',\n",
       "  '3',\n",
       "  'to',\n",
       "  '12',\n",
       "  'decibels',\n",
       "  'louder',\n",
       "  'than',\n",
       "  'the',\n",
       "  'background',\n",
       "  'noise',\n",
       "  '.',\n",
       "  'I',\n",
       "  'want',\n",
       "  'a',\n",
       "  'loudness',\n",
       "  '.',\n",
       "  'That',\n",
       "  'is',\n",
       "  'uh',\n",
       "  'below',\n",
       "  'starter',\n",
       "  'noise',\n",
       "  '.'],\n",
       " ['use', '2', 'µl', 'of', 'buffer', 'solution'],\n",
       " ['export',\n",
       "  'two',\n",
       "  'types',\n",
       "  'of',\n",
       "  'graphs',\n",
       "  '',\n",
       "  '1',\n",
       "  'is',\n",
       "  'steady',\n",
       "  'state',\n",
       "  'affinity',\n",
       "  'this',\n",
       "  'is',\n",
       "  'the',\n",
       "  'concentration',\n",
       "  'against',\n",
       "  'the',\n",
       "  'binding',\n",
       "  'signal',\n",
       "  'and',\n",
       "  '2',\n",
       "  'the',\n",
       "  'connect',\n",
       "  'of',\n",
       "  'the',\n",
       "  'binding',\n",
       "  'and',\n",
       "  'how',\n",
       "  'quickly'],\n",
       " ['TOC',\n",
       "  'value',\n",
       "  'is',\n",
       "  '1673',\n",
       "  'and',\n",
       "  '1603',\n",
       "  'that',\n",
       "  'was',\n",
       "  'the',\n",
       "  'value',\n",
       "  'before',\n",
       "  '.'],\n",
       " ['pellet', '245', 'mg', '.'],\n",
       " ['of',\n",
       "  '125',\n",
       "  'µl',\n",
       "  'are',\n",
       "  'made',\n",
       "  'from',\n",
       "  'the',\n",
       "  'urinary',\n",
       "  'biomarkers',\n",
       "  '.'],\n",
       " ['reminders', 'every', '4', 'h', 'for', '24', 'h.'],\n",
       " ['infusion',\n",
       "  'Rehydrex',\n",
       "  'at',\n",
       "  '10:00',\n",
       "  \"o'clock\",\n",
       "  '.',\n",
       "  'Start',\n",
       "  'infusion',\n",
       "  '125',\n",
       "  'ml',\n",
       "  '/',\n",
       "  'h',\n",
       "  '11:00',\n",
       "  \"o'clock\",\n",
       "  '.'],\n",
       " ['density', 'measured', 'was', '0.97', '.'],\n",
       " ['image',\n",
       "  'from',\n",
       "  'gel',\n",
       "  'using',\n",
       "  'the',\n",
       "  'GelDoc',\n",
       "  'EZ',\n",
       "  'imager',\n",
       "  'stain',\n",
       "  '-',\n",
       "  'free',\n",
       "  'with',\n",
       "  'activation',\n",
       "  'of',\n",
       "  '1.5',\n",
       "  'min',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  '.',\n",
       "  'To',\n",
       "  'highlight',\n",
       "  'intense',\n",
       "  'bands',\n",
       "  '',\n",
       "  'Uh',\n",
       "  'image',\n",
       "  'transformed',\n",
       "  'using',\n",
       "  'high',\n",
       "  '912',\n",
       "  'low',\n",
       "  '0',\n",
       "  'and',\n",
       "  'gamma',\n",
       "  '0.56',\n",
       "  '.',\n",
       "  'And',\n",
       "  'saved',\n",
       "  '.'],\n",
       " ['could',\n",
       "  'do',\n",
       "  'that',\n",
       "  'will',\n",
       "  'be',\n",
       "  'nozzle',\n",
       "  '.',\n",
       "  '5',\n",
       "  'mg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['for',\n",
       "  'small',\n",
       "  'patches',\n",
       "  '.',\n",
       "  '3',\n",
       "  'experimental',\n",
       "  'groups',\n",
       "  'medium',\n",
       "  '.9',\n",
       "  '%',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  'and',\n",
       "  '[',\n",
       "  '?',\n",
       "  ']',\n",
       "  'acetate'],\n",
       " ['experiment',\n",
       "  '.',\n",
       "  'Choose',\n",
       "  'day',\n",
       "  '5th',\n",
       "  'of',\n",
       "  'November',\n",
       "  '.',\n",
       "  'I',\n",
       "  'will',\n",
       "  'today',\n",
       "  'take',\n",
       "  'all',\n",
       "  'the',\n",
       "  '48',\n",
       "  'well',\n",
       "  'plates',\n",
       "  'with',\n",
       "  'a',\n",
       "  'ULK1',\n",
       "  'knockout',\n",
       "  '.',\n",
       "  'Divide',\n",
       "  'in',\n",
       "  'half',\n",
       "  'of',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'new',\n",
       "  '48',\n",
       "  'well',\n",
       "  'plate',\n",
       "  'and',\n",
       "  'the',\n",
       "  'rest',\n",
       "  'will',\n",
       "  'continue',\n",
       "  'to',\n",
       "  'make',\n",
       "  'genomic',\n",
       "  'DNA',\n",
       "  'on',\n",
       "  'to',\n",
       "  'see',\n",
       "  'if',\n",
       "  'knock',\n",
       "  'out',\n",
       "  'still',\n",
       "  'in',\n",
       "  'the',\n",
       "  'pool',\n",
       "  '.'],\n",
       " ['counts',\n",
       "  '.',\n",
       "  'G3',\n",
       "  'has',\n",
       "  '25',\n",
       "  'oocytes',\n",
       "  '.',\n",
       "  'Following',\n",
       "  'line',\n",
       "  '36',\n",
       "  'oocytes',\n",
       "  '.',\n",
       "  'Another',\n",
       "  'line',\n",
       "  '27',\n",
       "  'oocytes',\n",
       "  '.',\n",
       "  'Another',\n",
       "  'line',\n",
       "  '154',\n",
       "  'oocytes',\n",
       "  '.'],\n",
       " ['47', '1.222', 'E5', '.'],\n",
       " ['maybe',\n",
       "  'spend',\n",
       "  '20',\n",
       "  '%',\n",
       "  'of',\n",
       "  'my',\n",
       "  'time',\n",
       "  'on',\n",
       "  'this',\n",
       "  'depending',\n",
       "  'on',\n",
       "  'the',\n",
       "  'assay',\n",
       "  '.'],\n",
       " ['1200',\n",
       "  'µl',\n",
       "  'of',\n",
       "  '20',\n",
       "  'mM',\n",
       "  'BTP',\n",
       "  'pH',\n",
       "  '7.5',\n",
       "  'with',\n",
       "  '1.25',\n",
       "  '%',\n",
       "  'of',\n",
       "  'Koji',\n",
       "  'to',\n",
       "  'the',\n",
       "  '200',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'washed',\n",
       "  'membranes',\n",
       "  '.'],\n",
       " ['have',\n",
       "  '3',\n",
       "  'good',\n",
       "  'organoids',\n",
       "  'and',\n",
       "  '20',\n",
       "  'plates',\n",
       "  'of',\n",
       "  'human',\n",
       "  'podocytes',\n",
       "  '.'],\n",
       " ['filled',\n",
       "  'up',\n",
       "  'the',\n",
       "  'water',\n",
       "  'jackets',\n",
       "  'of',\n",
       "  'the',\n",
       "  'Organ',\n",
       "  'Bath',\n",
       "  'and',\n",
       "  'they',\n",
       "  'are',\n",
       "  'all',\n",
       "  'now',\n",
       "  'at',\n",
       "  '37',\n",
       "  '℃',\n",
       "  '.'],\n",
       " ['V4RF5', 'membranes', '.', '150819', 'SK', 'membrane', 'prep', '5'],\n",
       " ['of',\n",
       "  'HEPES',\n",
       "  'buffer',\n",
       "  'was',\n",
       "  'added',\n",
       "  'to',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'water',\n",
       "  'to',\n",
       "  'create',\n",
       "  'a',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '50',\n",
       "  'mM.'],\n",
       " ['',\n",
       "  'dilute',\n",
       "  'the',\n",
       "  'processed',\n",
       "  'pool',\n",
       "  '20',\n",
       "  'fold',\n",
       "  'in',\n",
       "  'small',\n",
       "  'scale',\n",
       "  '1st',\n",
       "  'to',\n",
       "  'test',\n",
       "  'if',\n",
       "  'the',\n",
       "  'protein',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'precipitate',\n",
       "  'at',\n",
       "  'low',\n",
       "  'salt',\n",
       "  'concentration',\n",
       "  '.'],\n",
       " ['26', '338000'],\n",
       " ['me', 'to', 'buy', 'new', 'book', 'tomorrow', 'at', '6:00', 'PM', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'clean',\n",
       "  'up',\n",
       "  'room',\n",
       "  '10',\n",
       "  'for',\n",
       "  'our',\n",
       "  'colleagues',\n",
       "  'and',\n",
       "  'do',\n",
       "  'the',\n",
       "  'photos',\n",
       "  'for',\n",
       "  'the',\n",
       "  'research',\n",
       "  'wiki',\n",
       "  '.'],\n",
       " ['the', 'size', 'exclusion', 'pull', 'fractions', 'B11', 'to', 'C6', '.'],\n",
       " ['ml', 'water', 'were', 'added', 'to', 'the', 'white', 'suspension', '.'],\n",
       " ['me', 'to', 'check', 'on', 'sample', '#', '3', '.'],\n",
       " ['ID', '18S006290', '.', 'Value', '3.705', 'ng/µl', '.', 'Pineapple', '.'],\n",
       " ['reason',\n",
       "  'why',\n",
       "  'the',\n",
       "  'caff',\n",
       "  'in',\n",
       "  'the',\n",
       "  'leak',\n",
       "  'experiment',\n",
       "  'did',\n",
       "  'not',\n",
       "  'really',\n",
       "  'work',\n",
       "  '',\n",
       "  'is',\n",
       "  'because',\n",
       "  'the',\n",
       "  'tube',\n",
       "  'was',\n",
       "  'disconnected',\n",
       "  'from',\n",
       "  'the',\n",
       "  'profusion',\n",
       "  'itself',\n",
       "  '.',\n",
       "  'so',\n",
       "  'there',\n",
       "  'was',\n",
       "  'no',\n",
       "  'flow',\n",
       "  'thereby',\n",
       "  'not',\n",
       "  'allowing',\n",
       "  'the',\n",
       "  'caffeine',\n",
       "  'to',\n",
       "  'have',\n",
       "  'it',\n",
       "  'effect',\n",
       "  'on',\n",
       "  'itself',\n",
       "  '.',\n",
       "  'So',\n",
       "  'therefore',\n",
       "  'you',\n",
       "  'have',\n",
       "  'to',\n",
       "  'use',\n",
       "  'the',\n",
       "  'caffeine',\n",
       "  'from',\n",
       "  'previous',\n",
       "  'experiments',\n",
       "  'to',\n",
       "  'determine',\n",
       "  'the',\n",
       "  'SR',\n",
       "  'calcium',\n",
       "  'leak',\n",
       "  '.'],\n",
       " ['initiating', 'Cas9', 'experiment', '.'],\n",
       " ['me', 'after', '5', 'min', '.'],\n",
       " ['me', 'to', 'take', 'the', 'bacillus', 'out', 'today', 'at', '3', 'PM', '.'],\n",
       " ['current',\n",
       "  'plate',\n",
       "  'map',\n",
       "  'CDK1',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'in',\n",
       "  'row',\n",
       "  'B',\n",
       "  'and',\n",
       "  'CDK2',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'in',\n",
       "  'row',\n",
       "  'Followed',\n",
       "  'by',\n",
       "  '3',\n",
       "  'wells',\n",
       "  'of',\n",
       "  'substrate',\n",
       "  'controls',\n",
       "  '.',\n",
       "  'Bumblebee',\n",
       "  '.'],\n",
       " ['pericardium', '.', '10:46', '.'],\n",
       " ['up', 'syringes', 'with', '20', '%', 'ethanol', '.'],\n",
       " ['cell',\n",
       "  'plates',\n",
       "  'out',\n",
       "  'of',\n",
       "  'incubator',\n",
       "  'now',\n",
       "  '.',\n",
       "  'Set',\n",
       "  'timer',\n",
       "  'for',\n",
       "  '1',\n",
       "  'min',\n",
       "  '.'],\n",
       " ['46', '1.550', 'E5', '.'],\n",
       " ['ID', '19', 'S', '000460', 'sample', 'value', '2.200', 'ng/µl', '.'],\n",
       " ['of',\n",
       "  '19',\n",
       "  'and',\n",
       "  '20',\n",
       "  'tubes',\n",
       "  'was',\n",
       "  'more',\n",
       "  'difficult',\n",
       "  'than',\n",
       "  'the',\n",
       "  'other',\n",
       "  'tubes',\n",
       "  '.',\n",
       "  'Might',\n",
       "  'indicate',\n",
       "  'aggregation',\n",
       "  '.'],\n",
       " ['PBS', 'plus', 'FBS', 'to', 'column', '3', 'and', '9', '.'],\n",
       " ['with',\n",
       "  'concentration',\n",
       "  'with',\n",
       "  'fraction',\n",
       "  'C3',\n",
       "  '',\n",
       "  'it',\n",
       "  'starts',\n",
       "  'to',\n",
       "  'go',\n",
       "  'slower',\n",
       "  'doing',\n",
       "  'runs',\n",
       "  'of',\n",
       "  'ten',\n",
       "  'min',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['28', '', '2019', '.'],\n",
       " ['F3',\n",
       "  'until',\n",
       "  'F11',\n",
       "  '',\n",
       "  'Uh',\n",
       "  '',\n",
       "  'for',\n",
       "  'the',\n",
       "  'co',\n",
       "  '-',\n",
       "  'expressed',\n",
       "  'purification',\n",
       "  'SEC',\n",
       "  'to',\n",
       "  'get',\n",
       "  'the',\n",
       "  'unprocessed',\n",
       "  'form',\n",
       "  '.',\n",
       "  'Repeat',\n",
       "  'F3',\n",
       "  'to',\n",
       "  'F11',\n",
       "  '.'],\n",
       " ['I',\n",
       "  'started',\n",
       "  'looking',\n",
       "  'at',\n",
       "  'the',\n",
       "  'left',\n",
       "  'side',\n",
       "  '',\n",
       "  'then',\n",
       "  'I',\n",
       "  'skipped',\n",
       "  'right',\n",
       "  'one',\n",
       "  '',\n",
       "  'and',\n",
       "  'right',\n",
       "  '2',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'now',\n",
       "  'taking',\n",
       "  'an',\n",
       "  'image',\n",
       "  'of',\n",
       "  'our',\n",
       "  '3',\n",
       "  '.'],\n",
       " ['to',\n",
       "  'the',\n",
       "  'previous',\n",
       "  'note',\n",
       "  '.',\n",
       "  'Add',\n",
       "  'only',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  '1',\n",
       "  'molar',\n",
       "  'TCEP',\n",
       "  'so',\n",
       "  'that',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'will',\n",
       "  'be',\n",
       "  '100',\n",
       "  'mM',\n",
       "  'TCEP',\n",
       "  '.'],\n",
       " ['equals', '1.948E5', '.'],\n",
       " ['5', 'ml', 'of', 'DMA', 'substance', '.'],\n",
       " ['3',\n",
       "  '.',\n",
       "  'Cm',\n",
       "  '=',\n",
       "  '81',\n",
       "  '.',\n",
       "  'Very',\n",
       "  'nice',\n",
       "  'recording',\n",
       "  'with',\n",
       "  'nice',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  '.',\n",
       "  'Isoprenaline',\n",
       "  'was',\n",
       "  'attempted',\n",
       "  '',\n",
       "  'although',\n",
       "  'the',\n",
       "  'effect',\n",
       "  'of',\n",
       "  'CAFF',\n",
       "  '/',\n",
       "  'ISO',\n",
       "  'was',\n",
       "  'delayed',\n",
       "  'because',\n",
       "  'of',\n",
       "  'a',\n",
       "  'bubble',\n",
       "  'that',\n",
       "  'stopped',\n",
       "  'the',\n",
       "  'perfusion',\n",
       "  'but',\n",
       "  'after',\n",
       "  'multiple',\n",
       "  'attempts',\n",
       "  'CAFF',\n",
       "  '/',\n",
       "  'ISO',\n",
       "  'was',\n",
       "  'done',\n",
       "  '.',\n",
       "  'However',\n",
       "  '',\n",
       "  'because',\n",
       "  'of',\n",
       "  'a',\n",
       "  'bubble',\n",
       "  '',\n",
       "  'the',\n",
       "  'response',\n",
       "  'was',\n",
       "  'very',\n",
       "  'slow',\n",
       "  'so',\n",
       "  'I',\n",
       "  'had',\n",
       "  'to',\n",
       "  'repeat',\n",
       "  'it',\n",
       "  'with',\n",
       "  'caffeine',\n",
       "  'without',\n",
       "  'Isoprenaline',\n",
       "  '.'],\n",
       " ['cells',\n",
       "  'will',\n",
       "  'stimulated',\n",
       "  'for',\n",
       "  '3',\n",
       "  'h',\n",
       "  'with',\n",
       "  '0.5',\n",
       "  'percent',\n",
       "  'FCS',\n",
       "  'under',\n",
       "  'flow',\n",
       "  'conditions',\n",
       "  'for',\n",
       "  '6',\n",
       "  'h.'],\n",
       " ['millliter', 'Marcain', '.'],\n",
       " ['viability',\n",
       "  '83.7',\n",
       "  '%',\n",
       "  '.',\n",
       "  'Live',\n",
       "  'cells',\n",
       "  '1.02',\n",
       "  '*',\n",
       "  '10',\n",
       "  '^',\n",
       "  '6',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['add',\n",
       "  '3',\n",
       "  'µl',\n",
       "  '3.1',\n",
       "  'µl',\n",
       "  'reducing',\n",
       "  'agents',\n",
       "  'from',\n",
       "  'new',\n",
       "  '-',\n",
       "  'page',\n",
       "  'lot',\n",
       "  'number',\n",
       "  '1941681',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'ll\",\n",
       "  'heat',\n",
       "  'all',\n",
       "  'the',\n",
       "  'SDS',\n",
       "  'PAGE',\n",
       "  'samples',\n",
       "  '5',\n",
       "  'min',\n",
       "  'at',\n",
       "  '65',\n",
       "  '°',\n",
       "  'in',\n",
       "  'the',\n",
       "  'heating',\n",
       "  'block',\n",
       "  '.'],\n",
       " ['.', 'I', 'have', 'to', 'make', 'sure', 'that'],\n",
       " ['culture',\n",
       "  '.',\n",
       "  'Yesterday',\n",
       "  'I',\n",
       "  'expanded',\n",
       "  'LEC',\n",
       "  'cells',\n",
       "  'AX',\n",
       "  '205',\n",
       "  'and',\n",
       "  'AX',\n",
       "  '218',\n",
       "  'to',\n",
       "  'T25',\n",
       "  'flasks',\n",
       "  '.'],\n",
       " ['test',\n",
       "  'number',\n",
       "  '2',\n",
       "  'with',\n",
       "  'having',\n",
       "  'the',\n",
       "  'microphone',\n",
       "  'closer',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'to',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['32', '1.709', 'E5', '.'],\n",
       " ['appointment',\n",
       "  'for',\n",
       "  'Thursday',\n",
       "  'at',\n",
       "  '12',\n",
       "  'PM',\n",
       "  '.',\n",
       "  'I',\n",
       "  'mean',\n",
       "  'am',\n",
       "  'f',\n",
       "  '*',\n",
       "  '*',\n",
       "  '*'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'take',\n",
       "  'the',\n",
       "  'acinetobacter',\n",
       "  'out',\n",
       "  'today',\n",
       "  'at',\n",
       "  '12:30',\n",
       "  '.'],\n",
       " ['up', 'protocol', '3', 'for', 'organoids', '.'],\n",
       " ['buffer',\n",
       "  'was',\n",
       "  'made',\n",
       "  'with',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'HEPES',\n",
       "  '',\n",
       "  '10',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'pluronic',\n",
       "  '',\n",
       "  '100',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'magnesium',\n",
       "  'chloride',\n",
       "  'and',\n",
       "  '20',\n",
       "  'per',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'TCEP',\n",
       "  '.'],\n",
       " ['remind',\n",
       "  'me',\n",
       "  'to',\n",
       "  'clean',\n",
       "  'the',\n",
       "  '[',\n",
       "  '?',\n",
       "  ']',\n",
       "  'tomorrow',\n",
       "  'at',\n",
       "  '10:00',\n",
       "  'AM',\n",
       "  '.'],\n",
       " ['next',\n",
       "  'week',\n",
       "  '',\n",
       "  'meet',\n",
       "  'Evgeni',\n",
       "  'at',\n",
       "  '12:15',\n",
       "  'in',\n",
       "  'front',\n",
       "  'of',\n",
       "  'Amrite',\n",
       "  'Indian',\n",
       "  'Restaurant',\n",
       "  'in',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'to',\n",
       "  'talk',\n",
       "  'about',\n",
       "  'open',\n",
       "  'science',\n",
       "  '.'],\n",
       " ['1', '3.527x10', 'to', 'the', '5', '.'],\n",
       " ['is', 'at', '1', 'mg', '/', 'ml', '.', 'I', 'will', 'use', '.'],\n",
       " [\"'s\",\n",
       "  'try',\n",
       "  'another',\n",
       "  'note',\n",
       "  '.',\n",
       "  'What',\n",
       "  'happens',\n",
       "  'if',\n",
       "  'I',\n",
       "  'tell',\n",
       "  'you',\n",
       "  'I',\n",
       "  'used',\n",
       "  '100',\n",
       "  'mm',\n",
       "  'NAOH',\n",
       "  '?',\n",
       "  'Also',\n",
       "  '',\n",
       "  'can',\n",
       "  'you',\n",
       "  'do',\n",
       "  'citric',\n",
       "  'acid',\n",
       "  '?'],\n",
       " ['polylysine', '6', 'well', 'plates', '.'],\n",
       " ['1', 'to', '10', '.'],\n",
       " [\"'m\",\n",
       "  'filling',\n",
       "  'up',\n",
       "  'the',\n",
       "  'organ',\n",
       "  'baths',\n",
       "  'now',\n",
       "  'with',\n",
       "  '25',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'Tyrode',\n",
       "  '.'],\n",
       " ['19', '', '1.727x10', 'to', 'the', '5'],\n",
       " ['with',\n",
       "  'difficult',\n",
       "  'words',\n",
       "  'in',\n",
       "  'the',\n",
       "  'writing',\n",
       "  'zone',\n",
       "  '',\n",
       "  'no',\n",
       "  'noise',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['µl', 'NaOH', '2', 'nanomol', '.'],\n",
       " ['aspirate', 'options', 'from', 'Z', 'from', '55', 'to', '50', '.'],\n",
       " ['description',\n",
       "  '.',\n",
       "  'Homologation',\n",
       "  'for',\n",
       "  'the',\n",
       "  'customer',\n",
       "  '.',\n",
       "  'Please',\n",
       "  'perform',\n",
       "  'test',\n",
       "  'according',\n",
       "  'TL260',\n",
       "  'on',\n",
       "  'parts',\n",
       "  'processed',\n",
       "  'on',\n",
       "  'the',\n",
       "  'customer',\n",
       "  '.'],\n",
       " ['buffer',\n",
       "  '20',\n",
       "  'mM',\n",
       "  'Tris',\n",
       "  'pH',\n",
       "  '7.5',\n",
       "  '',\n",
       "  '200',\n",
       "  'mM',\n",
       "  'sodium',\n",
       "  'chloride',\n",
       "  '',\n",
       "  '0.5',\n",
       "  'mM',\n",
       "  'TCEP',\n",
       "  '',\n",
       "  '10',\n",
       "  '%',\n",
       "  'Glycerol',\n",
       "  'and',\n",
       "  'protease',\n",
       "  'inhibitor',\n",
       "  '.'],\n",
       " ['forgot',\n",
       "  'to',\n",
       "  'pipette',\n",
       "  'the',\n",
       "  'volume',\n",
       "  '5',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'sample',\n",
       "  'one',\n",
       "  '.'],\n",
       " ['something',\n",
       "  '.',\n",
       "  'Hello',\n",
       "  'my',\n",
       "  'name',\n",
       "  'is',\n",
       "  'Julie',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  '28',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'French',\n",
       "  '.',\n",
       "  'See',\n",
       "  'it',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'understand',\n",
       "  'me',\n",
       "  '.',\n",
       "  'Is',\n",
       "  'just',\n",
       "  'be',\n",
       "  'tre',\n",
       "  'cute',\n",
       "  '?',\n",
       "  'But',\n",
       "  'he',\n",
       "  \"'s\",\n",
       "  'not',\n",
       "  '.',\n",
       "  'Sometimes',\n",
       "  'they',\n",
       "  'are',\n",
       "  'mean',\n",
       "  'with',\n",
       "  'me',\n",
       "  '.'],\n",
       " ['it',\n",
       "  'works',\n",
       "  '.',\n",
       "  'Plan',\n",
       "  'for',\n",
       "  'today',\n",
       "  'evening',\n",
       "  '.',\n",
       "  'No',\n",
       "  '',\n",
       "  'plan',\n",
       "  'for',\n",
       "  'today',\n",
       "  'evening',\n",
       "  '.',\n",
       "  'First',\n",
       "  '',\n",
       "  'do',\n",
       "  'the',\n",
       "  'calibration',\n",
       "  'experiment',\n",
       "  '',\n",
       "  'then',\n",
       "  'do',\n",
       "  'another',\n",
       "  'test',\n",
       "  'experiment',\n",
       "  'with',\n",
       "  'a',\n",
       "  'raised',\n",
       "  'sampling',\n",
       "  'rate',\n",
       "  'up',\n",
       "  'to',\n",
       "  'Hertz',\n",
       "  '.'],\n",
       " ['28', '2', '.', '0308', 'E5', '.'],\n",
       " ['for',\n",
       "  'example',\n",
       "  '',\n",
       "  'the',\n",
       "  'time',\n",
       "  'points',\n",
       "  'will',\n",
       "  'be',\n",
       "  '0',\n",
       "  'h',\n",
       "  '',\n",
       "  '6',\n",
       "  'h',\n",
       "  '',\n",
       "  '15',\n",
       "  'h',\n",
       "  '',\n",
       "  '24',\n",
       "  'h',\n",
       "  '',\n",
       "  '36',\n",
       "  'and',\n",
       "  '75',\n",
       "  'h.'],\n",
       " ['and',\n",
       "  'shake',\n",
       "  'samples',\n",
       "  'to',\n",
       "  'mix',\n",
       "  '',\n",
       "  'leave',\n",
       "  'on',\n",
       "  'the',\n",
       "  'bench',\n",
       "  'for',\n",
       "  '3',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'set',\n",
       "  'up',\n",
       "  'a',\n",
       "  'gel',\n",
       "  '.'],\n",
       " ['the',\n",
       "  \"'\",\n",
       "  'aim',\n",
       "  'for',\n",
       "  \"'\",\n",
       "  'immobilization',\n",
       "  'method',\n",
       "  ':',\n",
       "  'Flow',\n",
       "  'cell',\n",
       "  '2',\n",
       "  ':',\n",
       "  '4000',\n",
       "  'RU',\n",
       "  'Flow',\n",
       "  'cell',\n",
       "  '3',\n",
       "  ':',\n",
       "  '8000',\n",
       "  'RU',\n",
       "  'Flow',\n",
       "  'Cell',\n",
       "  '4',\n",
       "  ':',\n",
       "  '16000',\n",
       "  'RU'],\n",
       " ['amount',\n",
       "  'in',\n",
       "  'samples',\n",
       "  'in',\n",
       "  '79',\n",
       "  '',\n",
       "  '80',\n",
       "  '',\n",
       "  '78',\n",
       "  'and',\n",
       "  '77',\n",
       "  'and',\n",
       "  '76',\n",
       "  '.'],\n",
       " ['if',\n",
       "  'I',\n",
       "  'have',\n",
       "  'any',\n",
       "  'CHLA',\n",
       "  '79',\n",
       "  'in',\n",
       "  'the',\n",
       "  'liquid',\n",
       "  'nitrogen',\n",
       "  '.'],\n",
       " ['27', '', '2.337', 'E5', '.'],\n",
       " ['the', 'BMP', '4', 'box', 'and', 'the', 'Activin', 'A', 'box', '.'],\n",
       " ['me',\n",
       "  'on',\n",
       "  'the',\n",
       "  'centrifuge',\n",
       "  'in',\n",
       "  '10',\n",
       "  'min',\n",
       "  '.',\n",
       "  'Avocado',\n",
       "  '.',\n",
       "  'The',\n",
       "  'centrifuge',\n",
       "  '.'],\n",
       " ['concentration', 'of', 'L1a', 'is', '36.6', 'ng/µl', '.'],\n",
       " ['concentration', 'of', 'L1a', 'is', '36.6', 'ng/µl', '.'],\n",
       " ['spheroids', 'left', 'of', 'fatty', 'acid', 'treated', '.'],\n",
       " ['urine',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'now',\n",
       "  'transferred',\n",
       "  'from',\n",
       "  'the',\n",
       "  'HBS',\n",
       "  'freezer',\n",
       "  'to',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'and',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'will',\n",
       "  'be',\n",
       "  'thawed',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  'about',\n",
       "  '30',\n",
       "  'min',\n",
       "  '.',\n",
       "  'After',\n",
       "  'that',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'water',\n",
       "  'will',\n",
       "  'be',\n",
       "  'added',\n",
       "  'to',\n",
       "  'each',\n",
       "  'sample',\n",
       "  'containing',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'and',\n",
       "  'each',\n",
       "  'sample',\n",
       "  'will',\n",
       "  'be',\n",
       "  'mixed',\n",
       "  '.',\n",
       "  'Meanwhile',\n",
       "  '',\n",
       "  'the',\n",
       "  'samples',\n",
       "  'are',\n",
       "  'thawing',\n",
       "  'I',\n",
       "  'will',\n",
       "  're',\n",
       "  '-',\n",
       "  'arrange',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'list',\n",
       "  'and',\n",
       "  'make',\n",
       "  'sure',\n",
       "  'that',\n",
       "  'the',\n",
       "  'identity',\n",
       "  'of',\n",
       "  'our',\n",
       "  'tubes',\n",
       "  'are',\n",
       "  'matched',\n",
       "  'with',\n",
       "  'the',\n",
       "  'sample',\n",
       "  'lists',\n",
       "  'and',\n",
       "  'unique',\n",
       "  'lab',\n",
       "  'ID',\n",
       "  'number',\n",
       "  '.'],\n",
       " ['632',\n",
       "  'because',\n",
       "  'the',\n",
       "  'de',\n",
       "  '-',\n",
       "  'lamination',\n",
       "  'and',\n",
       "  'he',\n",
       "  'and',\n",
       "  'adhesion',\n",
       "  'of',\n",
       "  'superior',\n",
       "  'at',\n",
       "  'specification',\n",
       "  'requirements',\n",
       "  '.',\n",
       "  'After',\n",
       "  '5',\n",
       "  'cycles',\n",
       "  'no',\n",
       "  'surface',\n",
       "  'changes',\n",
       "  '',\n",
       "  'including',\n",
       "  'cut',\n",
       "  'edges',\n",
       "  'in',\n",
       "  'wells',\n",
       "  '.',\n",
       "  '°',\n",
       "  'of',\n",
       "  'rusting',\n",
       "  'DIN',\n",
       "  'dash',\n",
       "  'EN',\n",
       "  'dash',\n",
       "  'O60',\n",
       "  'dash',\n",
       "  '4628',\n",
       "  'dash',\n",
       "  '3',\n",
       "  'dash',\n",
       "  'RI0',\n",
       "  '.',\n",
       "  'After',\n",
       "  '30',\n",
       "  'cycles',\n",
       "  '.'],\n",
       " ['B12',\n",
       "  'and',\n",
       "  'B11',\n",
       "  'do',\n",
       "  \"n't\",\n",
       "  'fit',\n",
       "  'on',\n",
       "  'the',\n",
       "  'gel',\n",
       "  '',\n",
       "  'but',\n",
       "  'that',\n",
       "  \"'s\",\n",
       "  'okay',\n",
       "  '.',\n",
       "  'Run',\n",
       "  'the',\n",
       "  'gel',\n",
       "  '30',\n",
       "  'min',\n",
       "  'at',\n",
       "  '200',\n",
       "  'V',\n",
       "  'in',\n",
       "  'MES',\n",
       "  'buffer',\n",
       "  'using',\n",
       "  'standard',\n",
       "  'conditions',\n",
       "  '.'],\n",
       " ['2',\n",
       "  'G',\n",
       "  'has',\n",
       "  'a',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '187.1',\n",
       "  'ng/µl',\n",
       "  '.',\n",
       "  'The',\n",
       "  'fraction',\n",
       "  'of',\n",
       "  '260',\n",
       "  'by',\n",
       "  '280',\n",
       "  'is',\n",
       "  '1.98',\n",
       "  '.',\n",
       "  'The',\n",
       "  'fraction',\n",
       "  'of',\n",
       "  '260',\n",
       "  'by',\n",
       "  '230',\n",
       "  'is',\n",
       "  '2.33',\n",
       "  '.'],\n",
       " ['22', '89000', '.'],\n",
       " ['test',\n",
       "  'difficult',\n",
       "  'words',\n",
       "  'in',\n",
       "  'writing',\n",
       "  'zone',\n",
       "  '',\n",
       "  'no',\n",
       "  'noise',\n",
       "  '',\n",
       "  'microphone',\n",
       "  'close',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '25',\n",
       "  'µl',\n",
       "  '50',\n",
       "  'bar',\n",
       "  '50',\n",
       "  'mg',\n",
       "  '15',\n",
       "  'mg',\n",
       "  '12',\n",
       "  'mbar',\n",
       "  '70',\n",
       "  'mbar',\n",
       "  '17',\n",
       "  'mg',\n",
       "  '17',\n",
       "  'µl',\n",
       "  'diluted',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['tomorrow',\n",
       "  '.',\n",
       "  'Go',\n",
       "  'there',\n",
       "  'early',\n",
       "  '.',\n",
       "  'Redo',\n",
       "  'the',\n",
       "  'calibration',\n",
       "  '',\n",
       "  'and',\n",
       "  'the',\n",
       "  'test',\n",
       "  'experiment',\n",
       "  'with',\n",
       "  'the',\n",
       "  'high',\n",
       "  '[',\n",
       "  'frequency',\n",
       "  '00:00:13',\n",
       "  ']',\n",
       "  'test',\n",
       "  'patch',\n",
       "  'cables',\n",
       "  '.',\n",
       "  'Clean',\n",
       "  'up',\n",
       "  'a',\n",
       "  'little',\n",
       "  'bit',\n",
       "  '.',\n",
       "  'Prepare',\n",
       "  'for',\n",
       "  'come',\n",
       "  'out',\n",
       "  'to',\n",
       "  'me',\n",
       "  '',\n",
       "  'and',\n",
       "  'then',\n",
       "  'recording',\n",
       "  '.',\n",
       "  'Take',\n",
       "  'a',\n",
       "  'break',\n",
       "  '.',\n",
       "  'No',\n",
       "  '',\n",
       "  'I',\n",
       "  'can',\n",
       "  'not',\n",
       "  'do',\n",
       "  'recording',\n",
       "  'because',\n",
       "  'I',\n",
       "  'have',\n",
       "  'the',\n",
       "  'course',\n",
       "  '',\n",
       "  'and',\n",
       "  'sports',\n",
       "  '.',\n",
       "  'So',\n",
       "  '',\n",
       "  'just',\n",
       "  'come',\n",
       "  'out',\n",
       "  'to',\n",
       "  'me',\n",
       "  '',\n",
       "  'and',\n",
       "  'then',\n",
       "  'the',\n",
       "  'recording',\n",
       "  'either',\n",
       "  'in',\n",
       "  'the',\n",
       "  'night',\n",
       "  '',\n",
       "  'or',\n",
       "  'in',\n",
       "  'Wednesday',\n",
       "  '',\n",
       "  'Wednesday',\n",
       "  'morning',\n",
       "  'is',\n",
       "  'meeting',\n",
       "  '.',\n",
       "  'Also',\n",
       "  '',\n",
       "  'I',\n",
       "  'have',\n",
       "  'to',\n",
       "  'prepare',\n",
       "  'the',\n",
       "  'slides',\n",
       "  '.'],\n",
       " ['is',\n",
       "  'wrong',\n",
       "  'with',\n",
       "  'the',\n",
       "  'dilutions',\n",
       "  'of',\n",
       "  'the',\n",
       "  'Q',\n",
       "  'standards',\n",
       "  '.',\n",
       "  'The',\n",
       "  'sample',\n",
       "  'volume',\n",
       "  'in',\n",
       "  'Q2',\n",
       "  'do',\n",
       "  \"n't\",\n",
       "  'add',\n",
       "  'up',\n",
       "  'to',\n",
       "  'what',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'supposed',\n",
       "  'to',\n",
       "  'be',\n",
       "  '.'],\n",
       " ['19', '208000', '.'],\n",
       " ['µl', 'of', 'Metamizol', 'to', '1', 'ml', 'of', 'NaCl', '.'],\n",
       " ['sure',\n",
       "  'plate',\n",
       "  '11',\n",
       "  'is',\n",
       "  'a',\n",
       "  'Max',\n",
       "  'plate',\n",
       "  '.',\n",
       "  'Use',\n",
       "  'only',\n",
       "  'plates',\n",
       "  '13',\n",
       "  '',\n",
       "  '14',\n",
       "  '',\n",
       "  '15',\n",
       "  'for',\n",
       "  'um',\n",
       "  'the',\n",
       "  'assay',\n",
       "  'treatment',\n",
       "  'for',\n",
       "  'the',\n",
       "  'drug',\n",
       "  'treatment',\n",
       "  'with',\n",
       "  'compounds',\n",
       "  '.'],\n",
       " ['50',\n",
       "  'ml',\n",
       "  'TBME',\n",
       "  'were',\n",
       "  'added',\n",
       "  'to',\n",
       "  'the',\n",
       "  'white',\n",
       "  'suspension',\n",
       "  'and',\n",
       "  'then',\n",
       "  'filtered',\n",
       "  'off',\n",
       "  '.'],\n",
       " ['tube',\n",
       "  '3A3',\n",
       "  'into',\n",
       "  'equal',\n",
       "  'portions',\n",
       "  'of',\n",
       "  '27',\n",
       "  'ml',\n",
       "  '.',\n",
       "  'To',\n",
       "  '1',\n",
       "  'portion',\n",
       "  'add',\n",
       "  'Tris',\n",
       "  'pH',\n",
       "  '8',\n",
       "  'to',\n",
       "  '20',\n",
       "  'mM.'],\n",
       " ['pipettes', 'and', '3', 'centrifuges', 'and', '1', 'FACS', 'set', '.'],\n",
       " ['109537345',\n",
       "  'has',\n",
       "  'been',\n",
       "  'dropped',\n",
       "  'discard',\n",
       "  'that',\n",
       "  'from',\n",
       "  'the',\n",
       "  'analysis',\n",
       "  '.'],\n",
       " ['with',\n",
       "  'Chabra',\n",
       "  '.',\n",
       "  'English',\n",
       "  'background',\n",
       "  'talking',\n",
       "  'very',\n",
       "  'loud',\n",
       "  '.',\n",
       "  'New',\n",
       "  'extraction',\n",
       "  '.',\n",
       "  'Extraction',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temp',\n",
       "  'for',\n",
       "  '20',\n",
       "  'min',\n",
       "  'and',\n",
       "  'then',\n",
       "  'transferred',\n",
       "  'into',\n",
       "  'a',\n",
       "  'separation',\n",
       "  'funnel',\n",
       "  '.',\n",
       "  'Phase',\n",
       "  'cut',\n",
       "  'is',\n",
       "  'clear',\n",
       "  'and',\n",
       "  'fast',\n",
       "  '.',\n",
       "  'The',\n",
       "  'clear',\n",
       "  'yellow',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'dried',\n",
       "  'on',\n",
       "  'the',\n",
       "  'water',\n",
       "  'bath',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  '10',\n",
       "  'mbar',\n",
       "  'to',\n",
       "  'give',\n",
       "  '.',\n",
       "  'Test',\n",
       "  'end',\n",
       "  '.'],\n",
       " ['count',\n",
       "  'from',\n",
       "  '3',\n",
       "  'flasks',\n",
       "  'is',\n",
       "  '8.8',\n",
       "  'times',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '6',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '',\n",
       "  'giving',\n",
       "  '88',\n",
       "  'times',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '6',\n",
       "  'total',\n",
       "  '.'],\n",
       " ['is', 'Thursday', '', 'September', '6', '.'],\n",
       " ['have',\n",
       "  'policies',\n",
       "  'in',\n",
       "  'place',\n",
       "  'where',\n",
       "  'we',\n",
       "  'encourage',\n",
       "  'people',\n",
       "  'to',\n",
       "  'start',\n",
       "  'that',\n",
       "  'write',\n",
       "  'up',\n",
       "  'within',\n",
       "  '1',\n",
       "  'week',\n",
       "  'of',\n",
       "  'the',\n",
       "  'start',\n",
       "  'of',\n",
       "  'their',\n",
       "  'experiment',\n",
       "  '.',\n",
       "  'As',\n",
       "  'soon',\n",
       "  'as',\n",
       "  'the',\n",
       "  'analysis',\n",
       "  'is',\n",
       "  'complete',\n",
       "  '',\n",
       "  'they',\n",
       "  'need',\n",
       "  'to',\n",
       "  'have',\n",
       "  'the',\n",
       "  'write',\n",
       "  'up',\n",
       "  'done',\n",
       "  'within',\n",
       "  '2',\n",
       "  'weeks',\n",
       "  '.'],\n",
       " ['2',\n",
       "  '.',\n",
       "  'Cm',\n",
       "  '=',\n",
       "  'R=3.4',\n",
       "  '.',\n",
       "  'L',\n",
       "  '-',\n",
       "  'type',\n",
       "  'Ca',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  'as',\n",
       "  'well',\n",
       "  'as',\n",
       "  'SR',\n",
       "  'calcium',\n",
       "  'content',\n",
       "  'was',\n",
       "  'recorded',\n",
       "  '.',\n",
       "  'The',\n",
       "  'current',\n",
       "  'and',\n",
       "  'transient',\n",
       "  'were',\n",
       "  'very',\n",
       "  'good',\n",
       "  '',\n",
       "  'but',\n",
       "  'the',\n",
       "  'cause',\n",
       "  'of',\n",
       "  'a',\n",
       "  'bubble',\n",
       "  'isoprenaline',\n",
       "  'effect',\n",
       "  'was',\n",
       "  'not',\n",
       "  'done',\n",
       "  '.'],\n",
       " ['make',\n",
       "  'the',\n",
       "  '1',\n",
       "  'mg',\n",
       "  '/',\n",
       "  'ml',\n",
       "  'stock',\n",
       "  'of',\n",
       "  'the',\n",
       "  'IGF',\n",
       "  '.',\n",
       "  'Need',\n",
       "  'to',\n",
       "  'add',\n",
       "  '2',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  '5',\n",
       "  'M'],\n",
       " ['just',\n",
       "  'took',\n",
       "  '10',\n",
       "  'samples',\n",
       "  'of',\n",
       "  'HFF',\n",
       "  'cells',\n",
       "  'passage',\n",
       "  'number',\n",
       "  '20',\n",
       "  '+',\n",
       "  '1',\n",
       "  'from',\n",
       "  'box',\n",
       "  '3',\n",
       "  'of',\n",
       "  'the',\n",
       "  '-152',\n",
       "  '°',\n",
       "  'freezer',\n",
       "  '.'],\n",
       " ['culture',\n",
       "  '.',\n",
       "  'Yesterday',\n",
       "  'I',\n",
       "  'A549',\n",
       "  'transduced',\n",
       "  'with',\n",
       "  'IFITM',\n",
       "  '.',\n",
       "  'I',\n",
       "  'split',\n",
       "  'one',\n",
       "  'in',\n",
       "  'ten',\n",
       "  'and',\n",
       "  'harvest',\n",
       "  'a',\n",
       "  'cell',\n",
       "  'pellet',\n",
       "  'for',\n",
       "  'western',\n",
       "  'blot',\n",
       "  '.',\n",
       "  'Yesterday',\n",
       "  'I',\n",
       "  'SLKs',\n",
       "  'transduced',\n",
       "  '',\n",
       "  'SLK',\n",
       "  'EphA2',\n",
       "  'knock',\n",
       "  '-',\n",
       "  'out',\n",
       "  'cells',\n",
       "  'with',\n",
       "  'AX',\n",
       "  '245',\n",
       "  '.',\n",
       "  'I',\n",
       "  'to',\n",
       "  'one',\n",
       "  'in',\n",
       "  'ten',\n",
       "  'and',\n",
       "  'harvested',\n",
       "  'a',\n",
       "  'cell',\n",
       "  'pellet',\n",
       "  'for',\n",
       "  'western',\n",
       "  'blot',\n",
       "  '.'],\n",
       " ['[',\n",
       "  '?',\n",
       "  ']',\n",
       "  'stock',\n",
       "  'bottle',\n",
       "  '',\n",
       "  'measured',\n",
       "  'at',\n",
       "  '11',\n",
       "  'A.M.',\n",
       "  'Cells',\n",
       "  '4.49',\n",
       "  'in',\n",
       "  'total',\n",
       "  '.',\n",
       "  '4.47',\n",
       "  'viable',\n",
       "  '.',\n",
       "  'Viability',\n",
       "  '100',\n",
       "  '%',\n",
       "  '.',\n",
       "  'Split',\n",
       "  'in',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'with',\n",
       "  '11',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'cells',\n",
       "  'and',\n",
       "  '39',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'medium',\n",
       "  '.'],\n",
       " ['QPCR', 'Primers', 'for', 'F1', '.', 'Check', 'integrase'],\n",
       " ['am',\n",
       "  'adding',\n",
       "  'NBS',\n",
       "  '.',\n",
       "  'To',\n",
       "  'solution',\n",
       "  'Of',\n",
       "  'imidazole',\n",
       "  '.',\n",
       "  'The',\n",
       "  'reaction',\n",
       "  'is',\n",
       "  'stirred',\n",
       "  'at',\n",
       "  '0',\n",
       "  '°',\n",
       "  '.'],\n",
       " ['TOC', 'value', 'is', '324', 'parts', 'per', 'billion', '.'],\n",
       " ['2', 'molar', 'NaOH', '.'],\n",
       " ['330',\n",
       "  '',\n",
       "  'neural',\n",
       "  '-',\n",
       "  'D1',\n",
       "  '.',\n",
       "  'Two',\n",
       "  'more',\n",
       "  'pancreas',\n",
       "  '.',\n",
       "  'IgG',\n",
       "  'slide',\n",
       "  'section',\n",
       "  'is',\n",
       "  'completely',\n",
       "  'clean',\n",
       "  '',\n",
       "  'no',\n",
       "  'staining',\n",
       "  'whatsoever',\n",
       "  '.'],\n",
       " ['splitting',\n",
       "  'of',\n",
       "  'HepG2',\n",
       "  'C3A',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'We',\n",
       "  \"'ve\",\n",
       "  'got',\n",
       "  '2.8',\n",
       "  'million',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'mill',\n",
       "  'in',\n",
       "  'a',\n",
       "  'volume',\n",
       "  'of',\n",
       "  '25',\n",
       "  'ml',\n",
       "  '.',\n",
       "  'Viability',\n",
       "  'is',\n",
       "  '96.1',\n",
       "  '%',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'ll\",\n",
       "  'be',\n",
       "  'seeding',\n",
       "  'two',\n",
       "  'flasks',\n",
       "  '',\n",
       "  'one',\n",
       "  'with',\n",
       "  '5',\n",
       "  'million',\n",
       "  'and',\n",
       "  'one',\n",
       "  'with',\n",
       "  '15',\n",
       "  'million',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ['add',\n",
       "  '10',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'template',\n",
       "  'to',\n",
       "  'the',\n",
       "  'appropriate',\n",
       "  'wells',\n",
       "  'of',\n",
       "  'the',\n",
       "  'reaction',\n",
       "  'plate',\n",
       "  '.',\n",
       "  'Store',\n",
       "  'plate',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'or',\n",
       "  'on',\n",
       "  'ice',\n",
       "  '.'],\n",
       " ['did',\n",
       "  \"n't\",\n",
       "  'understand',\n",
       "  'that',\n",
       "  '.',\n",
       "  'EP0023',\n",
       "  'Started',\n",
       "  'DNA',\n",
       "  'measurement',\n",
       "  '.'],\n",
       " ['weight', 'is', '352'],\n",
       " ['10',\n",
       "  'ml',\n",
       "  'of',\n",
       "  'media',\n",
       "  'for',\n",
       "  'the',\n",
       "  'acoustic',\n",
       "  'mass',\n",
       "  'spec',\n",
       "  'of',\n",
       "  'stock',\n",
       "  'solution',\n",
       "  '.',\n",
       "  'Sorry',\n",
       "  '.',\n",
       "  'Correction',\n",
       "  '.'],\n",
       " ['cas9',\n",
       "  'standard',\n",
       "  'prepared',\n",
       "  'in',\n",
       "  'assay',\n",
       "  'diluent',\n",
       "  'is',\n",
       "  'added',\n",
       "  'to',\n",
       "  'row',\n",
       "  '1',\n",
       "  'and',\n",
       "  'row',\n",
       "  '2',\n",
       "  '.'],\n",
       " ['321', 'is', 'recorded', 'at', '23', '°', 'at', '09:58', 'AM', '.'],\n",
       " ['wash',\n",
       "  'superose',\n",
       "  'columns',\n",
       "  'with',\n",
       "  'a',\n",
       "  'program',\n",
       "  'with',\n",
       "  'water',\n",
       "  'subsequently',\n",
       "  'with',\n",
       "  '.'],\n",
       " ['lidocaine', 'start', '14:15', '.'],\n",
       " ['me', 'at', '8', 'PM', 'to', 'check', 'antibodies', 'for', 'tomorrow', '.'],\n",
       " ['new',\n",
       "  'strep',\n",
       "  'avidin',\n",
       "  'bio',\n",
       "  'core',\n",
       "  'chip',\n",
       "  'onto',\n",
       "  'the',\n",
       "  'bio',\n",
       "  'core',\n",
       "  '8',\n",
       "  'K',\n",
       "  '.'],\n",
       " ['whether',\n",
       "  'for',\n",
       "  'the',\n",
       "  'lowering',\n",
       "  'down',\n",
       "  'the',\n",
       "  'aspirate',\n",
       "  'to',\n",
       "  '40',\n",
       "  'would',\n",
       "  'improve',\n",
       "  'the',\n",
       "  'pattern',\n",
       "  '.'],\n",
       " ['is', '12.3', 'ng/µl', '.'],\n",
       " ['40', '1.145', 'E5', '.'],\n",
       " ['start', '', '15:35', '.'],\n",
       " ['96',\n",
       "  'h',\n",
       "  'time',\n",
       "  'point',\n",
       "  'was',\n",
       "  'washed',\n",
       "  '1',\n",
       "  'x',\n",
       "  'with',\n",
       "  'PBS',\n",
       "  '.',\n",
       "  'The',\n",
       "  'PBS',\n",
       "  'was',\n",
       "  'removed',\n",
       "  'and',\n",
       "  'the',\n",
       "  'plate',\n",
       "  'was',\n",
       "  'placed',\n",
       "  'at',\n",
       "  '-80',\n",
       "  '°',\n",
       "  '.'],\n",
       " ['that',\n",
       "  'I',\n",
       "  'have',\n",
       "  'book',\n",
       "  'a',\n",
       "  'hood',\n",
       "  'for',\n",
       "  'tomorrow',\n",
       "  '14th',\n",
       "  'at',\n",
       "  '10:30',\n",
       "  '.'],\n",
       " ['opens', 'at', '8:00', 'AM', 'in', 'the', 'congress', '.'],\n",
       " ['bioRAD',\n",
       "  'TGX',\n",
       "  'stainfree',\n",
       "  'gel',\n",
       "  '.',\n",
       "  '350',\n",
       "  'folds',\n",
       "  'for',\n",
       "  '11',\n",
       "  'min',\n",
       "  'in',\n",
       "  '1',\n",
       "  'time',\n",
       "  'tris',\n",
       "  'glycine',\n",
       "  'SDS',\n",
       "  'buffer',\n",
       "  '.',\n",
       "  'Forgot',\n",
       "  'to',\n",
       "  'remove',\n",
       "  'the',\n",
       "  'strip',\n",
       "  'at',\n",
       "  'the',\n",
       "  'bottom',\n",
       "  '.',\n",
       "  'Uh',\n",
       "  'Sample',\n",
       "  '.',\n",
       "  'It',\n",
       "  \"'s\",\n",
       "  'marker',\n",
       "  'in',\n",
       "  'lane',\n",
       "  '1',\n",
       "  '',\n",
       "  'sample',\n",
       "  '2',\n",
       "  'is',\n",
       "  'input',\n",
       "  '',\n",
       "  'sample',\n",
       "  '3',\n",
       "  'is',\n",
       "  'the',\n",
       "  'flow',\n",
       "  'through',\n",
       "  '',\n",
       "  'and',\n",
       "  'then',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'the',\n",
       "  'fractions',\n",
       "  'from',\n",
       "  'the',\n",
       "  'gradient',\n",
       "  '.',\n",
       "  'The',\n",
       "  'last',\n",
       "  '1',\n",
       "  'did',\n",
       "  \"n't\",\n",
       "  'fit',\n",
       "  'on',\n",
       "  'the',\n",
       "  'gel',\n",
       "  '',\n",
       "  'so',\n",
       "  'F2',\n",
       "  'is',\n",
       "  'not',\n",
       "  'included',\n",
       "  'in',\n",
       "  'this',\n",
       "  'run',\n",
       "  '.'],\n",
       " ['10:15',\n",
       "  'material',\n",
       "  'was',\n",
       "  'loaded',\n",
       "  '.',\n",
       "  'Switch',\n",
       "  'to',\n",
       "  'washing',\n",
       "  '.',\n",
       "  'Washing',\n",
       "  'at',\n",
       "  '5ML',\n",
       "  '/',\n",
       "  'min',\n",
       "  '',\n",
       "  'initially',\n",
       "  'at',\n",
       "  '0',\n",
       "  '%'],\n",
       " ['reaction',\n",
       "  'mixture',\n",
       "  'to',\n",
       "  'Lynn',\n",
       "  'Slash',\n",
       "  'Water',\n",
       "  '121',\n",
       "  'was',\n",
       "  'directly',\n",
       "  'used',\n",
       "  'in',\n",
       "  'knew',\n",
       "  'experiment',\n",
       "  '.',\n",
       "  'The',\n",
       "  'biphasic',\n",
       "  'system',\n",
       "  'was',\n",
       "  'cooled',\n",
       "  'down',\n",
       "  'to',\n",
       "  '50',\n",
       "  '℃',\n",
       "  'and',\n",
       "  'sodium',\n",
       "  '32',\n",
       "  '%',\n",
       "  'was',\n",
       "  'added',\n",
       "  'in',\n",
       "  'one',\n",
       "  'shot',\n",
       "  '.',\n",
       "  'Big',\n",
       "  'orange',\n",
       "  'junk',\n",
       "  'crashed',\n",
       "  'out',\n",
       "  '.',\n",
       "  'To',\n",
       "  'dissolve',\n",
       "  'the',\n",
       "  'chunk',\n",
       "  'additional',\n",
       "  'methanol',\n",
       "  'was',\n",
       "  'added',\n",
       "  '',\n",
       "  'and',\n",
       "  'stirred',\n",
       "  'the',\n",
       "  'mixture',\n",
       "  'at',\n",
       "  '6',\n",
       "  'h',\n",
       "  'for',\n",
       "  'fifth',\n",
       "  'at',\n",
       "  '50',\n",
       "  '℃',\n",
       "  '.'],\n",
       " ['',\n",
       "  'a',\n",
       "  '50',\n",
       "  '%',\n",
       "  'leads',\n",
       "  'to',\n",
       "  '79',\n",
       "  'decibel',\n",
       "  'sound',\n",
       "  'pressure',\n",
       "  'level',\n",
       "  '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'sign',\n",
       "  'up',\n",
       "  'for',\n",
       "  'the',\n",
       "  'flow',\n",
       "  'cytometer',\n",
       "  '.',\n",
       "  'Sunday',\n",
       "  '24th',\n",
       "  'of',\n",
       "  'March',\n",
       "  'of',\n",
       "  '2019',\n",
       "  '.'],\n",
       " ['1', 'control', 'media', 'added', '.'],\n",
       " ['weight', 'is', '9.2', 'ng', '.'],\n",
       " ['assay', 'started', 'at', '15:21', 'PM', '.'],\n",
       " ['cells',\n",
       "  'for',\n",
       "  'the',\n",
       "  'IT',\n",
       "  '25834',\n",
       "  'HepG2',\n",
       "  'C3a',\n",
       "  'spheroid',\n",
       "  'assay',\n",
       "  'following',\n",
       "  'a',\n",
       "  'single',\n",
       "  'vial',\n",
       "  'and',\n",
       "  '.'],\n",
       " ['.',\n",
       "  'Slide',\n",
       "  '98',\n",
       "  '.',\n",
       "  'Heart',\n",
       "  '.',\n",
       "  'Good',\n",
       "  'section',\n",
       "  'of',\n",
       "  '.',\n",
       "  'Is',\n",
       "  'sending',\n",
       "  'a',\n",
       "  'order',\n",
       "  '?',\n",
       "  'Within',\n",
       "  'normal',\n",
       "  'limits',\n",
       "  '.'],\n",
       " ['gel',\n",
       "  'is',\n",
       "  '4',\n",
       "  'to',\n",
       "  '15',\n",
       "  '%',\n",
       "  'a',\n",
       "  '12',\n",
       "  'well',\n",
       "  'comb',\n",
       "  'with',\n",
       "  '20',\n",
       "  'µl',\n",
       "  '/',\n",
       "  'gel',\n",
       "  '.',\n",
       "  'Category',\n",
       "  'of',\n",
       "  'a',\n",
       "  'catalog',\n",
       "  'number',\n",
       "  '456',\n",
       "  '-',\n",
       "  '8085',\n",
       "  '.'],\n",
       " ['Culture',\n",
       "  '.',\n",
       "  'Yesterday',\n",
       "  'I',\n",
       "  'expanded',\n",
       "  'rhesus',\n",
       "  'fibroblasts',\n",
       "  'to',\n",
       "  '40',\n",
       "  'times',\n",
       "  'T175',\n",
       "  'flasks',\n",
       "  'plus',\n",
       "  'one',\n",
       "  'T75',\n",
       "  'flask',\n",
       "  '.'],\n",
       " ['is',\n",
       "  'a',\n",
       "  'correction',\n",
       "  'of',\n",
       "  'the',\n",
       "  'last',\n",
       "  'sentence',\n",
       "  '.',\n",
       "  'The',\n",
       "  'water',\n",
       "  'was',\n",
       "  'stored',\n",
       "  'in',\n",
       "  'the',\n",
       "  'freezer',\n",
       "  'for',\n",
       "  '10',\n",
       "  'min',\n",
       "  'to',\n",
       "  'make',\n",
       "  'sure',\n",
       "  'samples',\n",
       "  'mixed',\n",
       "  'with',\n",
       "  'water',\n",
       "  'were',\n",
       "  'still',\n",
       "  'ice',\n",
       "  'cold',\n",
       "  'to',\n",
       "  'avoid',\n",
       "  'elevated',\n",
       "  'temperatures',\n",
       "  'during',\n",
       "  'sample',\n",
       "  'handling',\n",
       "  '.'],\n",
       " ['in', 'batch', '3', 'are', 'unlikely', '.'],\n",
       " ['round', '55', 'full', 'length', '37', '', 'mosquitoes', '.'],\n",
       " ['is', 'set', 'in', 'place', '.', 'A', 'quarter', 'past', '9', '.'],\n",
       " ['need',\n",
       "  'to',\n",
       "  'buy',\n",
       "  'more',\n",
       "  '24-well',\n",
       "  'plate',\n",
       "  'for',\n",
       "  'cell',\n",
       "  'culture',\n",
       "  '.'],\n",
       " ['a', 'temperature', 'change', 'from', '21', 'to', '22', '℃', '.'],\n",
       " ['24-well', 'table', '.'],\n",
       " ['have',\n",
       "  'to',\n",
       "  'suck',\n",
       "  '3',\n",
       "  'mice',\n",
       "  'and',\n",
       "  'process',\n",
       "  '4',\n",
       "  'mice',\n",
       "  'today',\n",
       "  '.',\n",
       "  'Let',\n",
       "  'me',\n",
       "  'step',\n",
       "  'out',\n",
       "  '.',\n",
       "  'Sometimes',\n",
       "  '',\n",
       "  'it',\n",
       "  'may',\n",
       "  'help',\n",
       "  'if',\n",
       "  'you',\n",
       "  '',\n",
       "  'if',\n",
       "  'you',\n",
       "  'wait',\n",
       "  'like',\n",
       "  '',\n",
       "  'if',\n",
       "  'you',\n",
       "  'say',\n",
       "  'Avocado',\n",
       "  'than',\n",
       "  'wait',\n",
       "  'for',\n",
       "  'two',\n",
       "  'seconds',\n",
       "  'and',\n",
       "  'than',\n",
       "  'say',\n",
       "  'Order'],\n",
       " [':',\n",
       "  'Petri',\n",
       "  'dishes',\n",
       "  '',\n",
       "  'LB',\n",
       "  'medium',\n",
       "  '2',\n",
       "  'boxes',\n",
       "  '',\n",
       "  '20',\n",
       "  'mL',\n",
       "  'syringes',\n",
       "  '3',\n",
       "  'boxes',\n",
       "  '',\n",
       "  '20',\n",
       "  'mL',\n",
       "  'syringes',\n",
       "  '10',\n",
       "  'boxes',\n",
       "  '.'],\n",
       " ['me', 'to', 'call', 'Sarah', 'at', '5', 'PM', 'tomorrow', '.'],\n",
       " ['TOP',\n",
       "  'agar',\n",
       "  'add',\n",
       "  'a',\n",
       "  'final',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '5',\n",
       "  'mM',\n",
       "  'of',\n",
       "  'calcium',\n",
       "  'chloride',\n",
       "  'and',\n",
       "  '1',\n",
       "  'mM',\n",
       "  'of',\n",
       "  'magnesium',\n",
       "  'sulfate',\n",
       "  '.'],\n",
       " ['30000',\n",
       "  'G',\n",
       "  'for',\n",
       "  '30',\n",
       "  'min',\n",
       "  '.',\n",
       "  'The',\n",
       "  'supernatant',\n",
       "  'is',\n",
       "  'incubated',\n",
       "  'with',\n",
       "  '1',\n",
       "  'ml',\n",
       "  'amylose',\n",
       "  'resin',\n",
       "  'at',\n",
       "  '4',\n",
       "  '°',\n",
       "  'overnight',\n",
       "  '.'],\n",
       " ['have',\n",
       "  'to',\n",
       "  'order',\n",
       "  'new',\n",
       "  'single',\n",
       "  'use',\n",
       "  'filter',\n",
       "  'units',\n",
       "  '0.45',\n",
       "  'µm',\n",
       "  '.'],\n",
       " ['',\n",
       "  'the',\n",
       "  'pin',\n",
       "  'connection',\n",
       "  'is',\n",
       "  'as',\n",
       "  'follows',\n",
       "  '.',\n",
       "  'Printer',\n",
       "  'TXD',\n",
       "  '2',\n",
       "  'is',\n",
       "  'connected',\n",
       "  'to',\n",
       "  'pin',\n",
       "  '3',\n",
       "  '',\n",
       "  'printer',\n",
       "  'RXD',\n",
       "  '2',\n",
       "  '.',\n",
       "  'Is',\n",
       "  'connected',\n",
       "  'to',\n",
       "  '?',\n",
       "  'Pin',\n",
       "  'number',\n",
       "  '8',\n",
       "  '.'],\n",
       " ['54',\n",
       "  '',\n",
       "  'position',\n",
       "  'H12',\n",
       "  '.',\n",
       "  'There',\n",
       "  \"'s\",\n",
       "  'a',\n",
       "  'small',\n",
       "  'crystal',\n",
       "  '.'],\n",
       " ['is', '8.783', 'e4', '.'],\n",
       " ['1', 'spr', 'assay', 'using'],\n",
       " ['make',\n",
       "  'my',\n",
       "  'virus',\n",
       "  'dilutions',\n",
       "  '',\n",
       "  'I',\n",
       "  'will',\n",
       "  'take',\n",
       "  '280',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'virus',\n",
       "  'stock',\n",
       "  '.',\n",
       "  'And',\n",
       "  'add',\n",
       "  'this',\n",
       "  'to',\n",
       "  '.',\n",
       "  'μl',\n",
       "  '.',\n",
       "  'Of',\n",
       "  'PBS',\n",
       "  '.',\n",
       "  'That',\n",
       "  'gives',\n",
       "  'me',\n",
       "  '-',\n",
       "  '2',\n",
       "  'dilution',\n",
       "  '.',\n",
       "  'I',\n",
       "  'will',\n",
       "  'then',\n",
       "  'take',\n",
       "  '280',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'that',\n",
       "  'and',\n",
       "  'add',\n",
       "  'it',\n",
       "  'to',\n",
       "  'μl',\n",
       "  'of',\n",
       "  'PBS',\n",
       "  'to',\n",
       "  'give',\n",
       "  'them',\n",
       "  '-4',\n",
       "  'and',\n",
       "  'then',\n",
       "  'the',\n",
       "  'same',\n",
       "  'together',\n",
       "  '-6',\n",
       "  'dilution',\n",
       "  'full',\n",
       "  'stop',\n",
       "  '.',\n",
       "  'That',\n",
       "  'will',\n",
       "  'give',\n",
       "  'me',\n",
       "  'my',\n",
       "  '-2',\n",
       "  '-4',\n",
       "  '-6',\n",
       "  'donation',\n",
       "  'and',\n",
       "  'I',\n",
       "  'live',\n",
       "  'there',\n",
       "  'and',\n",
       "  'remove',\n",
       "  '.',\n",
       "  '?',\n",
       "  'cell',\n",
       "  'and',\n",
       "  'add',\n",
       "  '200',\n",
       "  'μl',\n",
       "  'to',\n",
       "  'each',\n",
       "  'well',\n",
       "  '.',\n",
       "  'In',\n",
       "  'the',\n",
       "  'corresponding',\n",
       "  '-2',\n",
       "  '-4',\n",
       "  '-6',\n",
       "  'wells',\n",
       "  '.',\n",
       "  'So',\n",
       "  'should',\n",
       "  'be',\n",
       "  'adding',\n",
       "  'virus',\n",
       "  'to',\n",
       "  'all',\n",
       "  'of',\n",
       "  'them',\n",
       "  'and',\n",
       "  'then',\n",
       "  'I',\n",
       "  'will',\n",
       "  'put',\n",
       "  'the',\n",
       "  '?',\n",
       "  'cell',\n",
       "  'back',\n",
       "  'on',\n",
       "  'top',\n",
       "  '.',\n",
       "  'I',\n",
       "  'want',\n",
       "  'to',\n",
       "  'make',\n",
       "  'sure',\n",
       "  'that',\n",
       "  'I',\n",
       "  'add',\n",
       "  'extra',\n",
       "  'pen',\n",
       "  '-',\n",
       "  'strep',\n",
       "  'and',\n",
       "  'gentamicin',\n",
       "  '.',\n",
       "  'To',\n",
       "  'the',\n",
       "  '?',\n",
       "  'cell',\n",
       "  'before',\n",
       "  'add',\n",
       "  'it',\n",
       "  'back',\n",
       "  'on',\n",
       "  'to',\n",
       "  'make',\n",
       "  'sure',\n",
       "  'it',\n",
       "  \"'s\",\n",
       "  'killing',\n",
       "  'all',\n",
       "  'the',\n",
       "  'bacteria',\n",
       "  'that',\n",
       "  'are',\n",
       "  'in',\n",
       "  'there',\n",
       "  '.',\n",
       "  'OK',\n",
       "  '.'],\n",
       " ['3.96', 'µl', 'of', 'denatured', 'DNA', 'to', 'hyb', 'buffer', '.'],\n",
       " ['one',\n",
       "  '.',\n",
       "  'Value',\n",
       "  'it',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'like',\n",
       "  'it',\n",
       "  '94.4',\n",
       "  '',\n",
       "  'but',\n",
       "  'it',\n",
       "  'does',\n",
       "  \"n't\",\n",
       "  'now',\n",
       "  'do',\n",
       "  'something',\n",
       "  '.'],\n",
       " ['ml',\n",
       "  'of',\n",
       "  'water',\n",
       "  'were',\n",
       "  'added',\n",
       "  'to',\n",
       "  'the',\n",
       "  'right',\n",
       "  'suspension',\n",
       "  '.'],\n",
       " ['pipetted', '60', 'µl', 'into', 'the', 'Eppendorf', 'tube', '.'],\n",
       " ['new',\n",
       "  'plate',\n",
       "  'type',\n",
       "  'has',\n",
       "  'shown',\n",
       "  'promising',\n",
       "  'results',\n",
       "  '.',\n",
       "  'Both',\n",
       "  'CDK1',\n",
       "  'and',\n",
       "  'CDK2',\n",
       "  'shows',\n",
       "  'viability',\n",
       "  'for',\n",
       "  'longer',\n",
       "  'up',\n",
       "  'to',\n",
       "  '8000',\n",
       "  'seconds',\n",
       "  '.'],\n",
       " ['.', 'Column', '1', '.'],\n",
       " ['me',\n",
       "  'to',\n",
       "  'check',\n",
       "  'the',\n",
       "  'cells',\n",
       "  '',\n",
       "  'tomorrow',\n",
       "  'morning',\n",
       "  'at',\n",
       "  '9:00',\n",
       "  '.'],\n",
       " ['HFFs',\n",
       "  'are',\n",
       "  'in',\n",
       "  'the',\n",
       "  'growing',\n",
       "  'medium',\n",
       "  'with',\n",
       "  'includes',\n",
       "  '.',\n",
       "  'I',\n",
       "  'have',\n",
       "  'to',\n",
       "  'change',\n",
       "  'it',\n",
       "  'too',\n",
       "  '.'],\n",
       " ['measurement',\n",
       "  'is',\n",
       "  'buffer',\n",
       "  'only',\n",
       "  '.',\n",
       "  '160',\n",
       "  '150',\n",
       "  'seconds',\n",
       "  '.',\n",
       "  'After',\n",
       "  '10',\n",
       "  'seconds',\n",
       "  'flash',\n",
       "  '.',\n",
       "  'After',\n",
       "  '10',\n",
       "  '%',\n",
       "  '.'],\n",
       " ['ml', 'of', 'PBS', '.', '2', 'µl', 'of', 'anti', 'rabbit', '.'],\n",
       " [\"'s\", 'experiment', 'HU', '1650', '.'],\n",
       " ['ADP',\n",
       "  '-',\n",
       "  'Glo',\n",
       "  'reagent',\n",
       "  'was',\n",
       "  'added',\n",
       "  'using',\n",
       "  'Certus',\n",
       "  'dispenser',\n",
       "  'and',\n",
       "  'read',\n",
       "  'on',\n",
       "  'a',\n",
       "  'FlexStation',\n",
       "  '.',\n",
       "  '10',\n",
       "  'mM',\n",
       "  'magnesium',\n",
       "  'chloride',\n",
       "  'solution',\n",
       "  'was',\n",
       "  'used',\n",
       "  '.'],\n",
       " [':',\n",
       "  '20000',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.',\n",
       "  'Three',\n",
       "  'controls',\n",
       "  '.',\n",
       "  'Five',\n",
       "  'patients',\n",
       "  '.',\n",
       "  'Prepare',\n",
       "  'RNA',\n",
       "  'post',\n",
       "  '24',\n",
       "  'hours',\n",
       "  'incubation',\n",
       "  '.'],\n",
       " ['count',\n",
       "  'for',\n",
       "  'MCF',\n",
       "  '7',\n",
       "  'cells',\n",
       "  'is',\n",
       "  '3.5',\n",
       "  'million',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['speed', 'to', '300', 'rpm', '.'],\n",
       " ['D',\n",
       "  '360',\n",
       "  'is',\n",
       "  'for',\n",
       "  'published',\n",
       "  'data',\n",
       "  'so',\n",
       "  'this',\n",
       "  'is',\n",
       "  'new',\n",
       "  'and',\n",
       "  'also',\n",
       "  'retested',\n",
       "  'assays',\n",
       "  '.'],\n",
       " ['have',\n",
       "  'split',\n",
       "  'neuroblastoma',\n",
       "  'cells',\n",
       "  '.',\n",
       "  'I',\n",
       "  'got',\n",
       "  'around',\n",
       "  '10000000',\n",
       "  'cells',\n",
       "  'in',\n",
       "  'total',\n",
       "  '.',\n",
       "  'I',\n",
       "  'prepared',\n",
       "  '2',\n",
       "  '24',\n",
       "  'well',\n",
       "  'plates',\n",
       "  'with',\n",
       "  '100000',\n",
       "  'cells',\n",
       "  'per',\n",
       "  'well',\n",
       "  '.',\n",
       "  'Tomorrow',\n",
       "  '',\n",
       "  'I',\n",
       "  'will',\n",
       "  'start',\n",
       "  'differentiation',\n",
       "  'so',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'going',\n",
       "  'add',\n",
       "  'retinoic',\n",
       "  'acid',\n",
       "  'at',\n",
       "  '10',\n",
       "  'µM.'],\n",
       " ['me', 'to', 'order', '5', 'Falcon', 'tubes', '.'],\n",
       " ['ending',\n",
       "  'this',\n",
       "  'small',\n",
       "  'scale',\n",
       "  'centrifusion',\n",
       "  '',\n",
       "  'pipette',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'the',\n",
       "  'supernatant',\n",
       "  'into',\n",
       "  'a',\n",
       "  'vial',\n",
       "  'for',\n",
       "  'FSEC',\n",
       "  'analysis',\n",
       "  '.',\n",
       "  'And',\n",
       "  'run',\n",
       "  'FSEC',\n",
       "  'to',\n",
       "  'understand',\n",
       "  'the',\n",
       "  'solubilisation'],\n",
       " ['massspec',\n",
       "  'samples',\n",
       "  'for',\n",
       "  'the',\n",
       "  'massspec',\n",
       "  'samples',\n",
       "  'for',\n",
       "  'the',\n",
       "  'non',\n",
       "  'processed',\n",
       "  'form',\n",
       "  'or',\n",
       "  'the',\n",
       "  '1',\n",
       "  'that',\n",
       "  'is',\n",
       "  'not',\n",
       "  'co',\n",
       "  '-',\n",
       "  'expressed',\n",
       "  'using',\n",
       "  '70',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'buffer',\n",
       "  'A',\n",
       "  '',\n",
       "  'then',\n",
       "  '50',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'IMAC',\n",
       "  'eluate',\n",
       "  '',\n",
       "  'and',\n",
       "  'subsequently',\n",
       "  'for',\n",
       "  'oxidized',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'water',\n",
       "  'and',\n",
       "  'for',\n",
       "  'reduced',\n",
       "  '20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  '1',\n",
       "  'molar',\n",
       "  'TCEP',\n",
       "  '.'],\n",
       " ['20',\n",
       "  'µl',\n",
       "  'of',\n",
       "  'Cas9',\n",
       "  'solution',\n",
       "  'at',\n",
       "  '0.1',\n",
       "  'ug/µl',\n",
       "  'I',\n",
       "  'will',\n",
       "  'add',\n",
       "  '1',\n",
       "  'micolitre',\n",
       "  'of',\n",
       "  'the',\n",
       "  'trypsin',\n",
       "  'solution',\n",
       "  'to',\n",
       "  'give',\n",
       "  'a',\n",
       "  '1',\n",
       "  'to',\n",
       "  '20',\n",
       "  'ratio',\n",
       "  '.'],\n",
       " ['100', '%', 'melting', 'point', '193', '°', 'centigrade', '.'],\n",
       " ['aliquotation',\n",
       "  'are',\n",
       "  'the',\n",
       "  'tubes',\n",
       "  'flash',\n",
       "  'frozen',\n",
       "  'in',\n",
       "  'liquid',\n",
       "  'nitrogen',\n",
       "  '.',\n",
       "  'There',\n",
       "  'is',\n",
       "  'a',\n",
       "  '24th',\n",
       "  'tube',\n",
       "  'with',\n",
       "  'about',\n",
       "  'half',\n",
       "  'of',\n",
       "  'the',\n",
       "  'volume',\n",
       "  'available',\n",
       "  'that',\n",
       "  'will',\n",
       "  'be',\n",
       "  'used',\n",
       "  'next',\n",
       "  'week',\n",
       "  'for',\n",
       "  'testing',\n",
       "  'freeze',\n",
       "  'point',\n",
       "  'and',\n",
       "  'will',\n",
       "  'not',\n",
       "  'be',\n",
       "  'registered',\n",
       "  'in',\n",
       "  'lab',\n",
       "  'guru',\n",
       "  '.'],\n",
       " ['exp', '.', 'day', '-2'],\n",
       " ['volume', 'is', '23.2'],\n",
       " ['502', 'row', 'A', 'column', '1', '.'],\n",
       " ['the', 'channel', 'integration', 'number', '2', '.'],\n",
       " ['130', 'µl', 'into', 'lane', '3', '.'],\n",
       " ['occlusion',\n",
       "  '10:57',\n",
       "  '.',\n",
       "  'Alone',\n",
       "  'placed',\n",
       "  'distal',\n",
       "  'to',\n",
       "  'the',\n",
       "  'first',\n",
       "  'diagonal',\n",
       "  '.'],\n",
       " ['if',\n",
       "  'noise',\n",
       "  'in',\n",
       "  'the',\n",
       "  'lab',\n",
       "  'is',\n",
       "  'too',\n",
       "  'loud',\n",
       "  'to',\n",
       "  'detect',\n",
       "  'my',\n",
       "  'voice',\n",
       "  '.',\n",
       "  'Dishwasher',\n",
       "  '',\n",
       "  'Cryo',\n",
       "  'other',\n",
       "  'noise',\n",
       "  'sources',\n",
       "  'like',\n",
       "  'hoods',\n",
       "  'are',\n",
       "  'running',\n",
       "  'right',\n",
       "  'now',\n",
       "  '.',\n",
       "  'The',\n",
       "  'phone',\n",
       "  'is',\n",
       "  'being',\n",
       "  'hold',\n",
       "  'close',\n",
       "  'to',\n",
       "  'the',\n",
       "  'mouth',\n",
       "  '.',\n",
       "  'Test',\n",
       "  '3',\n",
       "  '.',\n",
       "  'Put',\n",
       "  'the',\n",
       "  'phone',\n",
       "  'in',\n",
       "  'my',\n",
       "  'lab',\n",
       "  'coat',\n",
       "  '.',\n",
       "  'See',\n",
       "  'if',\n",
       "  'detection',\n",
       "  'is',\n",
       "  'still',\n",
       "  'good',\n",
       "  'enough',\n",
       "  '.',\n",
       "  'Test',\n",
       "  '4',\n",
       "  '.',\n",
       "  'Test',\n",
       "  '5',\n",
       "  '.',\n",
       "  'pH',\n",
       "  'is',\n",
       "  'still',\n",
       "  'calibrated',\n",
       "  'And',\n",
       "  'stopped',\n",
       "  'manually',\n",
       "  'now',\n",
       "  '.'],\n",
       " ['the', 'concentration', 'of', 'row', '2', 'and', '4', '.'],\n",
       " ['further', 'to', '35', 'the', 'aspirate', 'Z', 'offset', '.'],\n",
       " ['this',\n",
       "  'is',\n",
       "  'a',\n",
       "  'test',\n",
       "  'to',\n",
       "  'see',\n",
       "  'if',\n",
       "  'LabTwin',\n",
       "  'can',\n",
       "  'say',\n",
       "  'Leukotriene',\n",
       "  'E4',\n",
       "  '',\n",
       "  'or',\n",
       "  'LTE4',\n",
       "  '',\n",
       "  'or',\n",
       "  'oxylipids',\n",
       "  '.'],\n",
       " ['down', '2', 'day', '24', 'h', 'samples', 'for', 'AZ1768', '.'],\n",
       " ['first', 'round', '.', 'Single', 'chain', '5D5', '25', 'mosquitoes', '.'],\n",
       " ['27', '258000'],\n",
       " ['',\n",
       "  'it',\n",
       "  'takes',\n",
       "  '20',\n",
       "  'min',\n",
       "  'for',\n",
       "  'instrument',\n",
       "  'linked',\n",
       "  'PCs',\n",
       "  'to',\n",
       "  'warm',\n",
       "  'up',\n",
       "  '.',\n",
       "  'Having',\n",
       "  'a',\n",
       "  'facial',\n",
       "  'recognition',\n",
       "  'as',\n",
       "  'a',\n",
       "  'login',\n",
       "  'would',\n",
       "  'be',\n",
       "  'really',\n",
       "  'useful',\n",
       "  'and',\n",
       "  'would',\n",
       "  'avoid',\n",
       "  'contamination',\n",
       "  '.'],\n",
       " ['assays',\n",
       "  'that',\n",
       "  'we',\n",
       "  'do',\n",
       "  'each',\n",
       "  'week',\n",
       "  'vary',\n",
       "  'depending',\n",
       "  'on',\n",
       "  'a',\n",
       "  'few',\n",
       "  'different',\n",
       "  'factors',\n",
       "  '.',\n",
       "  'Number',\n",
       "  '1',\n",
       "  'is',\n",
       "  'the',\n",
       "  'number',\n",
       "  'of',\n",
       "  'the',\n",
       "  'cells',\n",
       "  'of',\n",
       "  'the',\n",
       "  'compound',\n",
       "  'submitted',\n",
       "  'and',\n",
       "  'assays',\n",
       "  '.'],\n",
       " ['10', 'µl', 'water', 'to', 'microplate', '.'],\n",
       " ['needs',\n",
       "  'to',\n",
       "  'prepare',\n",
       "  'needles',\n",
       "  'for',\n",
       "  'injection',\n",
       "  '.',\n",
       "  'Program',\n",
       "  '2',\n",
       "  '.',\n",
       "  'um',\n",
       "  '',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']',\n",
       "  'it',\n",
       "  'is',\n",
       "  '[',\n",
       "  'inaudible',\n",
       "  ']'],\n",
       " ['.',\n",
       "  'Slide',\n",
       "  '99',\n",
       "  '.',\n",
       "  'Sciatic',\n",
       "  'nerve',\n",
       "  '.',\n",
       "  'Within',\n",
       "  'normal',\n",
       "  'limits',\n",
       "  '.'],\n",
       " ['number', '1', 'is', '17.7'],\n",
       " ['microchip', 'runs', 'faster', 'than', 'the', 'other', 'ones', '.'],\n",
       " ['the',\n",
       "  'Santa',\n",
       "  'Cruz',\n",
       "  'antibody',\n",
       "  'at',\n",
       "  'a',\n",
       "  'concentration',\n",
       "  'of',\n",
       "  '1',\n",
       "  'to',\n",
       "  '1000',\n",
       "  'mg'],\n",
       " ['buffer', 'is', 'made', 'up', 'of', '47', 'mL', 'of', 'water', '.'],\n",
       " ['assay',\n",
       "  'has',\n",
       "  'a',\n",
       "  'folder',\n",
       "  'and',\n",
       "  'experiment',\n",
       "  'write',\n",
       "  'up',\n",
       "  'notes',\n",
       "  '.',\n",
       "  'These',\n",
       "  'are',\n",
       "  'dated',\n",
       "  'and',\n",
       "  '1',\n",
       "  'can',\n",
       "  'change',\n",
       "  'manually',\n",
       "  'depending',\n",
       "  'on',\n",
       "  'the',\n",
       "  'Assay',\n",
       "  '.',\n",
       "  'They',\n",
       "  'have',\n",
       "  'an',\n",
       "  'Excel',\n",
       "  'that',\n",
       "  \"'s\",\n",
       "  'embedded',\n",
       "  'with',\n",
       "  'the',\n",
       "  'cell',\n",
       "  'count',\n",
       "  'on',\n",
       "  'volumes',\n",
       "  'as',\n",
       "  'well',\n",
       "  '.'],\n",
       " [':',\n",
       "  'Cell',\n",
       "  'count',\n",
       "  'is',\n",
       "  '4.4',\n",
       "  'times',\n",
       "  '10',\n",
       "  'to',\n",
       "  'the',\n",
       "  '6',\n",
       "  'per',\n",
       "  'ml',\n",
       "  '.'],\n",
       " ['of',\n",
       "  'edited',\n",
       "  'cells',\n",
       "  'and',\n",
       "  'immunogenicity',\n",
       "  'of',\n",
       "  'CRISPR',\n",
       "  '-',\n",
       "  'Cas9',\n",
       "  '.'],\n",
       " ['did',\n",
       "  'the',\n",
       "  'cloning',\n",
       "  'for',\n",
       "  'the',\n",
       "  'sample',\n",
       "  '.',\n",
       "  'I',\n",
       "  \"'m\",\n",
       "  'scarred',\n",
       "  'to',\n",
       "  'leave',\n",
       "  'and',\n",
       "  'any',\n",
       "  'starlet',\n",
       "  'life',\n",
       "  'act',\n",
       "  '.',\n",
       "  'I',\n",
       "  'used',\n",
       "  '100',\n",
       "  'ng',\n",
       "  'vector',\n",
       "  'and',\n",
       "  '37.59',\n",
       "  'ng',\n",
       "  'insert',\n",
       "  '.',\n",
       "  'Fixed',\n",
       "  'libation',\n",
       "  'mix',\n",
       "  '.',\n",
       "  'I',\n",
       "  'kept',\n",
       "  'at',\n",
       "  'room',\n",
       "  'temperature',\n",
       "  'for',\n",
       "  '15',\n",
       "  'minutes',\n",
       "  'and',\n",
       "  'transformed',\n",
       "  'in',\n",
       "  'stable',\n",
       "  '3',\n",
       "  'cells',\n",
       "  '.'],\n",
       " ...]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ready_s"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "4030 3999 0 2169 3069\n"
     ]
    }
   ],
   "source": [
    "len(ready_s)\n",
    "\n",
    "n_genes,n_cell_lines, n_proteins, n_reagents,n_science = 0,0,0,0,0\n",
    "for i,s in enumerate(ready_s):\n",
    "    for j,w in enumerate(s):\n",
    "        if ready_labels[i][j] == 'gene':\n",
    "            n_genes += 1 \n",
    "            break\n",
    "        \n",
    "        if ready_labels[i][j] == 'cell_line':\n",
    "            n_cell_lines += 1 \n",
    "            break\n",
    "        if ready_labels[i][j] == 'science_word':\n",
    "            n_science += 1 \n",
    "            break\n",
    "        \n",
    "        if ready_labels[i][j] == 'protein':\n",
    "            n_proteins += 1 \n",
    "            break\n",
    "            \n",
    "        if ready_labels[i][j] == 'reagent':\n",
    "            n_reagents += 1 \n",
    "            break\n",
    "        \n",
    "\n",
    "print(n_genes,n_cell_lines, n_proteins, n_reagents, n_science)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Ok there are a decent amount of sentences for the interesting categories...\n",
    "\n",
    "### 4. Make spacy training data <a name=\"fjórða\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "17705\n"
     ]
    }
   ],
   "source": [
    "# transform sentences with entities of interest into the spacy format\n",
    "\n",
    "sentences = [[]]\n",
    "entities = []\n",
    "tagger = {\"entities\": []}\n",
    "old_sentence_nr = '0'\n",
    "\n",
    "# skip the header \n",
    "for i,sent in enumerate(ready_s):\n",
    "\n",
    "    for j,w in enumerate(sent):\n",
    "      #  print(w)\n",
    "        category = ready_labels[i][j]\n",
    "        \n",
    " \n",
    "\n",
    "        start_index = sum([len(x) +1 for x in sent[:j]])\n",
    "        if category in interesting:\n",
    "            tagger[\"entities\"].append((start_index, start_index + len(w), category))\n",
    "   # print('')\n",
    "    # loop over the sentence and mark the entities:\n",
    "    #entities.append(tagger)\n",
    "\n",
    "    entities.append((' '.join(sent),tagger))\n",
    "    tagger = {\"entities\": []}\n",
    "\n",
    "\n",
    "\n",
    "            \n",
    "print(len(entities))\n",
    "        \n",
    "# sentences are the sentences pieced together again as lists \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('A1 66056 . A2 5.11', {'entities': []}) \n",
      "\n",
      "(\"2 . Cm R = 2.5 . Nice calcium current and transient . Caffeine was beautiful . Isoprenaline was also done but it 's important to note that the isoprenaline stock was made from the old stock which might reflect in the reduced activation of beta adrenergic receptors . Leak was also attempted nice effective tetracaine was seen but it seems the is not properly working as expected probably due to a low perfusion pressure . However a nice leak experiment was done .\", {'entities': []}) \n",
      "\n",
      "('temperature for amplification was achieved in D2  E2  F2  G2  and H2 .', {'entities': []}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('is added to the plate and put on a plate shaker at 450 rpm .', {'entities': []}) \n",
      "\n",
      "('gel with first lane 10 µl precision plus protein unstained standard from BioRad catalog number 161 - 036310 .', {'entities': []}) \n",
      "\n",
      "('µl to lane 1 .', {'entities': []}) \n",
      "\n",
      "('A1 to A9 is my serial dilution of standard with known concentration . From B1 to B 9 .', {'entities': []}) \n",
      "\n",
      "('low amount of RNA in sample 44 and 40 .', {'entities': []}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('decreased to 65 % but Placed the speakers in a different place so they face the mouse . Now I have 85 decibels .', {'entities': []}) \n",
      "\n",
      "('for 10 min . Timer for 10 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "(\"out what 's new in Labtwin 2.0 .\", {'entities': []}) \n",
      "\n",
      "('it mini - protean TGX stainfree gel . 4 to 15 % 12 well comb  20 µl per well from BioRad . Catalog number 456 - 8085 . And use precision plus protein standard .', {'entities': []}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('the next time we will be doing this experiment is very important for me to take enough note before hand so that I do not make the same mistakes again and again . I just told Larry so that he would remind me to place order for 6 l of alcohol from Sigma Corporation . Shaziya is going to go through all my slides for the evaluation of the data .', {'entities': []}) \n",
      "\n",
      "('took place at 10000', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('me to pipet a plate tomorrow at 1:00 PM .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing the needles for microinjection . Parameters are as follows : heat 540  pull 80  velocity 30  time 250  pressure 250  ram 525 . Instrument model P 1000 . Source Sutter instruments Flaming / Brown micropipette puller .\", {'entities': []}) \n",
      "\n",
      "('me to check sample number 4 on 1st day at 10:00 AM .', {'entities': []}) \n",
      "\n",
      "('me to call Sarah at 2 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('µl E 1 to tube 3 .', {'entities': []}) \n",
      "\n",
      "('culture . Saw MB cells MMB 1843 and MMB 1845 .', {'entities': []}) \n",
      "\n",
      "('1 aliquot of wash membrane preparations from the 3rd of August 2018 . In aliquots are 200 µl with a concentration of 321 µM made by .', {'entities': []}) \n",
      "\n",
      "('up antibody dilutions for flow experiments day 54 I think . Dragon .', {'entities': []}) \n",
      "\n",
      "('µl diluted DNA into denaturation .', {'entities': []}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('LH 77  both mice look slightly scruffy  but overall good and active .', {'entities': []}) \n",
      "\n",
      "('temperature to 25 ° .', {'entities': []}) \n",
      "\n",
      "('is 4.0 ng/µl .', {'entities': []}) \n",
      "\n",
      "(\"I leave the queue of bikers along my path upon and leave the zone parking once behind me  I realize it 's going to be profound disturbing normal to return to the city in 3 weeks time . As I will hopefully have gotten this week where ?\", {'entities': []}) \n",
      "\n",
      "('μl 2 years old frighteners . Pipette 5 µl [ inaudible ] primers', {'entities': []}) \n",
      "\n",
      "('7  1.494 E 5', {'entities': []}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('length is 352 mm .', {'entities': []}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "('diluted my cells into 100 ml .', {'entities': []}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('50 sorry  analyzed 20 µl of the flowthrough fraction from experiment 67 on FSEC .', {'entities': []}) \n",
      "\n",
      "('Biacore 8K. This is a high throughput screen .', {'entities': []}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('thawed my competent cells for 20 seconds .', {'entities': []}) \n",
      "\n",
      "('11 1.926 E5 .', {'entities': []}) \n",
      "\n",
      "(\"% leads to 77 sound pressure level . And it 's already pretty low .\", {'entities': []}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('white suspension was filtered off and dried . Dried in the vacuum oven at 40 °', {'entities': []}) \n",
      "\n",
      "('me to check the cells confluence on Saturday  August 10th .', {'entities': []}) \n",
      "\n",
      "('for 10 min . [ unsupported language ]', {'entities': []}) \n",
      "\n",
      "('me to order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('loading must have been lower than 5 mililiters per min from sample pump . Sample pump must have been . There must be something wrong with the sample pump . Increase now to 10 ml / min . Downloading the remainder 80 ml on the column . Flowthrough is collected in waste 1 which is put into a bottle .', {'entities': []}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('20 µl of the master mix .', {'entities': []}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "('am running a Her2 assay . Her2YVMA assay for kinetic characterization of . Repeat  I am repeating . I am running Her2 assay for the kinetic characterization of compounds to determine k - elect by ki - values .', {'entities': []}) \n",
      "\n",
      "('50 to column A position 1 .', {'entities': []}) \n",
      "\n",
      "('again 5 min now at 10000 rpm . After adding more of C6 to the solution . Start another size exclusion with about 0\\\\.3 ml of AZ 2250 .', {'entities': []}) \n",
      "\n",
      "('with a relatively high aspirate offset and then will reduce gradually . Starting from 55 original values . Checking for plate patterns in terms of cell loss and then will reduce gradually .', {'entities': []}) \n",
      "\n",
      "('also inoculate AX 50 and pLenti EphA2 sgmut delta plus ATG .', {'entities': []}) \n",
      "\n",
      "('is 66056 . A2 is 15', {'entities': []}) \n",
      "\n",
      "(\"cells are assay ready so you need to culture them in incubators with 37 ° We have 4 incubators to choose from . I do n't write in a notebook . I tend to use in network PC if I can .\", {'entities': []}) \n",
      "\n",
      "('15 1.388 E5 .', {'entities': []}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('a miniprotean TXTGX stainfree gel from bio rad . Batch 64267254 .', {'entities': []}) \n",
      "\n",
      "('3 µl of magnesium chloride .', {'entities': []}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('colon - no abnormalities detected', {'entities': []}) \n",
      "\n",
      "('clear CDT solution was put in a syringe an added in one shot to the 0 ° - solution .', {'entities': []}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "('me to warm cells at 5:00 PM .', {'entities': []}) \n",
      "\n",
      "('g Lavender oil . 2.19 g Pine oil . 162.01 g alcohol . 70 .', {'entities': []}) \n",
      "\n",
      "('need 3 cases of', {'entities': []}) \n",
      "\n",
      "('re activate protocol step 5 .', {'entities': []}) \n",
      "\n",
      "('a look at the reaction 1 in fume hood left .', {'entities': []}) \n",
      "\n",
      "('weight is 22 ng .', {'entities': []}) \n",
      "\n",
      "('assay BTRL12 optimal temperature was achieved in all wells .', {'entities': []}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('B3  B1  C2  C4  C6  C8  D3 and E4 .', {'entities': []}) \n",
      "\n",
      "('seconds injection 5 µl / min .', {'entities': []}) \n",
      "\n",
      "('32  1.130 E 5', {'entities': []}) \n",
      "\n",
      "('39 6 . Dam it . 3.650 E5 .', {'entities': []}) \n",
      "\n",
      "(\" I 've been using it . . Set the timer for 2 min .\", {'entities': []}) \n",
      "\n",
      "('set up the baculovirus in the shaker flask for Experiment A-12 .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('put a timer for 10 min .', {'entities': []}) \n",
      "\n",
      "('100 ml of acid into a bowl .', {'entities': []}) \n",
      "\n",
      "('cycles PV 1210 first part . 200 h constant humidity CH first part . Stone chip resistance panels to be sent . Chemicals resistance panels to be sent .', {'entities': []}) \n",
      "\n",
      "('concentration in sample P1 is 5 ng/µl .', {'entities': []}) \n",
      "\n",
      "('sample was repeated 3 times .', {'entities': []}) \n",
      "\n",
      "('migs of lucifer yellow .', {'entities': []}) \n",
      "\n",
      "('plasmid number 5 . Concentration 220.7 ng/µl . Fraction number 1 1.86  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('solution 1 in fridge RNase added .', {'entities': []}) \n",
      "\n",
      "('- PROTEAN TGX stain free pre cast gel 4 - 15 %  twelve well  20 µl per well BioRAD and 5 µl precision plus protein unstained standard BioRAD Catalog 161 - 0363 .', {'entities': []}) \n",
      "\n",
      "('1 chip fluid flow going from bottom to top of image .', {'entities': []}) \n",
      "\n",
      "('immediate team is about 15 to 17 people . We are separated into therapeutic areas within CPSS . This means there it could be more than 100 people in CPSS alone .', {'entities': []}) \n",
      "\n",
      "('bio analyzer . 490 concentration 1 . 491 oncentration 2 . 492 concentration 0.4 .', {'entities': []}) \n",
      "\n",
      "('50 position G3 3 phase separation .', {'entities': []}) \n",
      "\n",
      "('me to turn on the LSR2 at 2:00 PM . Turn on the LSR2 .', {'entities': []}) \n",
      "\n",
      "('my reaction at 45 ° .', {'entities': []}) \n",
      "\n",
      "(\"I did another experiment at using the same conditions and that looks a lot better . Y02 and N02 hold up really well for the 10 min runtime I had . So for next experiment I 'm trying out for how long we retain any activity .\", {'entities': []}) \n",
      "\n",
      "('110 µl of EPX 3 to X AMP .', {'entities': []}) \n",
      "\n",
      "(\"it 's important to also note in experiment 4 shortening is also included .\", {'entities': []}) \n",
      "\n",
      "('methods 181219 sepharose 6 position 6 . According to standard with 8.8 mililiter per min flow rate .', {'entities': []}) \n",
      "\n",
      "('21 : Made new buffer .', {'entities': []}) \n",
      "\n",
      "('seeded back 500000 cells for 983 and 987  but I seeded 1000000 cells for 988 . I hope I can keep the same number of cells for the next time .', {'entities': []}) \n",
      "\n",
      "('1 is my donor . A', {'entities': []}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "('me tomorrow to take out experiment 1 from fume hood 2 .', {'entities': []}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('a 130 µl into lane 1 .', {'entities': []}) \n",
      "\n",
      "('finished at 11:35 .', {'entities': []}) \n",
      "\n",
      "('improved background on pancreas in protocol 695 .', {'entities': []}) \n",
      "\n",
      "('ID 30 also added to C9 .', {'entities': []}) \n",
      "\n",
      "('3 into cell A2 .', {'entities': []}) \n",
      "\n",
      "('14 2.0.6 E5 .', {'entities': []}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('3849 - 3  human CD 45 at 1.38  CD 19 at 0.76  CD 10 at 1.07  CD 34 at 0.65 and CD 22 at 0.23 .', {'entities': []}) \n",
      "\n",
      "('me to take the cells out of the incubator tomorrow at 11:00 AM .', {'entities': []}) \n",
      "\n",
      "('502 column A position 1 .', {'entities': []}) \n",
      "\n",
      "('cell A1 put the number 1 .', {'entities': []}) \n",
      "\n",
      "('100 μl betaine', {'entities': []}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('tube 4 on the rotation for 20 min .', {'entities': []}) \n",
      "\n",
      "('membrane pellet is resuspended in 1 %  DDM Lysis buffer . Incubated for at least 5 min . After centrifugation of 10 min . Every thing is recovered in the soluble fraction .', {'entities': []}) \n",
      "\n",
      "('me to talk to Mimi tomorrow at 9:00 .', {'entities': []}) \n",
      "\n",
      "('me to order 10 cm dishes .', {'entities': []}) \n",
      "\n",
      "('me to pipet the plate at 1:00 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('me to take out the extraction samples . In 2 min .', {'entities': []}) \n",
      "\n",
      "('cell culture medium DMEM 10 bottles .', {'entities': []}) \n",
      "\n",
      "('probe in new spot . Temperature 36.9 .', {'entities': []}) \n",
      "\n",
      "('lab chart : Comments 2 fingers remember . Lab chart .', {'entities': []}) \n",
      "\n",
      "('me tomorrow to take my samples out of the fridge at 7:35 AM .', {'entities': []}) \n",
      "\n",
      "('20 % ethanol solution for cleaning MX system . Using 1.42l70 % ethanol . Filling up to 5 l with milliQ water .', {'entities': []}) \n",
      "\n",
      "(\" I needed thawed out Miller 's cryo vial of el2B glycerol stock .\", {'entities': []}) \n",
      "\n",
      "('the standard curve so row 4 and row 6 are exchanged .', {'entities': []}) \n",
      "\n",
      "(\"my experiment DNA Research G2 237 . Sorry I did n't understand .\", {'entities': []}) \n",
      "\n",
      "('are sacking a lot of mice today  including if I remember correctly 828  859 no 849 2 and one 839  840  845 .', {'entities': []}) \n",
      "\n",
      "('test with headphones holding microphone close to the mouth . Extraction . The biphasic solution was steered at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('have to order new filter systems oh 0.45 µm .', {'entities': []}) \n",
      "\n",
      "('in both lanes . 5 µl .', {'entities': []}) \n",
      "\n",
      "('# 3 is 18', {'entities': []}) \n",
      "\n",
      "('4 1.141 E5 . Sample was repeated once . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('test number 0  no noise in the office . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('for 30 min .', {'entities': []}) \n",
      "\n",
      "('a method . The method is called tube preparation 6 samples .', {'entities': []}) \n",
      "\n",
      "('at 37 ° overnight .', {'entities': []}) \n",
      "\n",
      "('- 17 - light 86 . Eyes . 2 sections .', {'entities': []}) \n",
      "\n",
      "('me to put the CHO cells in the 37 ° incubator at 9:30 AM .', {'entities': []}) \n",
      "\n",
      "('seems to be having a hard time differentiating between 15 and 50 .', {'entities': []}) \n",
      "\n",
      "('µl EFP12 added to denaturation .', {'entities': []}) \n",
      "\n",
      "('storage room is open between 8:30 AM to 9:30 AM and also between 1:00 PM to 2:00 PM .', {'entities': []}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('select experiment DNA research D 237 . DNA research .', {'entities': []}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('a timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('neural D 1 protocol 330 . Medi small cell lung cancer screen .', {'entities': []}) \n",
      "\n",
      "('. next . 4 boxes P-100 tips . Next', {'entities': []}) \n",
      "\n",
      "('8  1.384 E 5', {'entities': []}) \n",
      "\n",
      "('500 µl of 2X media to each condition .', {'entities': []}) \n",
      "\n",
      "('A column 1 is 500 .', {'entities': []}) \n",
      "\n",
      "('of the bacteria 1.560', {'entities': []}) \n",
      "\n",
      "('round 5 D 5 . 35 mosquitoes . [ In German : \" Yes .', {'entities': []}) \n",
      "\n",
      "('most important is to order the samples and the collection tube in the same way as described on the manual . And the volume for the isolation of the RNA should be depending upon the number of the cells that are being lysed in the RLT buffer . The RNA samples were collected on 25th of March and stored at -80 horizontal .', {'entities': []}) \n",
      "\n",
      "('- 17 - slide 86 . Eyes . 2 sections .', {'entities': []}) \n",
      "\n",
      "('me to check on sample # 5 .', {'entities': []}) \n",
      "\n",
      "('sample is high affinity peptide . So far measurement looks good incubation was 2 min in the chamber .', {'entities': []}) \n",
      "\n",
      "('order 2 units  Stary Zart .', {'entities': []}) \n",
      "\n",
      "('9 with very loud lab technician in the lab . Extraction the biphasic solution was stirred at the room temp for 20 min and transferred into a separation funnel . Phase cut was clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('7  3.174 E5 .', {'entities': []}) \n",
      "\n",
      "(' starting volume of the bacteria 24 ml .', {'entities': []}) \n",
      "\n",
      "('is 1.529 E 5 .', {'entities': []}) \n",
      "\n",
      "('have an appointment at 14 PM today .', {'entities': []}) \n",
      "\n",
      "('me to turn on the machine at 10 AM .', {'entities': []}) \n",
      "\n",
      "('me to drink a glass of water at 7:00 PM today .', {'entities': []}) \n",
      "\n",
      "('BHSD13 belongs to a family of poorly characterised Enzymes and neither is physiological function  nor pathological role is developed in development of NASH or understood .', {'entities': []}) \n",
      "\n",
      "('will expand RS from 10 flasks to 40 flasks . an 1 to 4 split .', {'entities': []}) \n",
      "\n",
      "(\"made the Cas9 KRAB teary virus stain there are 10 aliquots . The aliquots with the smaller volume that 's approximately 250 µl should be split into thirds while the aliquot that is one mL should be split in half . I put them in the virus 2 box in -80 .\", {'entities': []}) \n",
      "\n",
      "('40 2.444 E5 .', {'entities': []}) \n",
      "\n",
      "('words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('novel D 1 protocol 330 . Medi small cell lung cancer screen .', {'entities': []}) \n",
      "\n",
      "('neural D 1 protocol 330 . Medi small cell lung cancer screen .', {'entities': []}) \n",
      "\n",
      "('I said about the Neuralinks file was all wrong because now suddenly I do not get error messages ? I have the following thng : the first one  is optostimulus than 0  1 is the clock behavior 10 milliseconds  2 is force sensor  3 and 4 empty . Yeah .', {'entities': []}) \n",
      "\n",
      "('images end on 094024 Chip D2- 3 .', {'entities': []}) \n",
      "\n",
      "('500 µl of 2X media to each condition .', {'entities': []}) \n",
      "\n",
      "('Imaging for experiment 190801 .', {'entities': []}) \n",
      "\n",
      "('timer in 10 min .', {'entities': []}) \n",
      "\n",
      "(\"of the most annoying things is the fact that I 'm used to being able to call on Siri and she responds a lot faster than\", {'entities': []}) \n",
      "\n",
      "('mastermix for 5 replicates was made up for the cDNA .', {'entities': []}) \n",
      "\n",
      "('CH protein stored after Ni2 + purification .', {'entities': []}) \n",
      "\n",
      "(\"round of calibration . CAL5 number 25  and PIN number 25 . Amplitudes used  1.5  2.5  3.5  4.5  5.5  6.5  7.5  8.5  9.5 volts . Maximum redoubt for the first sensor is only one volt . Voltage output is around 10 volts . Specially 9.5 's correct .\", {'entities': []}) \n",
      "\n",
      "('PAGE samples are sticky and thick . I need to dilute them . I will dilute them with SDS PAGE buffer  uh  without reducing agents . I take 50 µl SDS PAGE .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 24 to 22 ° .', {'entities': []}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('BioELN good reference for for archive data  but I tend to use the network drive 1st .', {'entities': []}) \n",
      "\n",
      "('on ice 10 min .', {'entities': []}) \n",
      "\n",
      "('concentration for L1 PSAD is 73.6 ng/µl .', {'entities': []}) \n",
      "\n",
      "('added Coomassie at 1 in 200 dilution .', {'entities': []}) \n",
      "\n",
      "('dilution of standards from wells A1 to A9 .', {'entities': []}) \n",
      "\n",
      "('remind me to do writeups today at 4:00 PM and tomorrow at 3:00 PM .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "(\"after centrifugation it 's possible to pipette sample 12 . A big pellet has formed in the bottom of the tube .\", {'entities': []}) \n",
      "\n",
      "('me to clean the dishes in 5 min .', {'entities': []}) \n",
      "\n",
      "('set up the baculovirus in the Shaker cell . For experiment A 12345 .', {'entities': []}) \n",
      "\n",
      "('NanoTemper Tycho NT6 with the samples in PDB number order  so starting from PB19 - 1088 then 1089  then 1090  then 1091  then 1092  which is GFP  and finally 1094 .', {'entities': []}) \n",
      "\n",
      "('1 µl of fluid into the tube .', {'entities': []}) \n",
      "\n",
      "('23rd of November 2018  boiling experiment antifoam in water distilled . Condition no foam 20  um  72 ° . With foam 72.5 ° . [ in German : \" . Wrong language ]', {'entities': []}) \n",
      "\n",
      "('12 1.0.6 E5 .', {'entities': []}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('added the baculovirus to the Shaker flask . For the A 12345 experiment .', {'entities': []}) \n",
      "\n",
      "('37 4.797 E4 .', {'entities': []}) \n",
      "\n",
      "('for top Agar in 100 ml of 7.5 g of Agar pre liter added 25 ml to a total of 125 ml to get 6 g / l . To that add magnesium chloride or magnesium sulfate and calcium chloride .', {'entities': []}) \n",
      "\n",
      "('is .249 at 8:20 .', {'entities': []}) \n",
      "\n",
      "('add 7.7 µl LDS to each tube . Sample buffer from new - page lot number 1904557 .', {'entities': []}) \n",
      "\n",
      "('concentration of the second psPax 2 is 921.4 ng/µl . Ah the fraction of 260 / 280 is 2.13 . The fraction of the 60 / 230 is 2.4 .', {'entities': []}) \n",
      "\n",
      "('in cold methanol at 500 rcf  at 800 rcf for 5 min .', {'entities': []}) \n",
      "\n",
      "('2 with radio and second person but no talking in the lab . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('210 µl of EPX1 to X AMP .', {'entities': []}) \n",
      "\n",
      "('334 . I took the white sample or yellow from the mixed culture .', {'entities': []}) \n",
      "\n",
      "('terms of calculations . Are you recording ? Okay perfect . In terms of calculation for my first set of 30 slides for the phospho - RAD50  I have calculated the required total volume for this run will be 5000 by doing 30 * 150 at 350 and rising up to 5000 . To calculate my antibody volume I use the C1V1 equals C2V2 calculation and calculated that of a stock concentration of 70 µg / ml  in which I require concentration of 1 ug / ml  the total antibody volume I will need for this one is 71.43 µl . Therefore  the total diluent volume I will require is 5000 - 71.43 which will equal 4928.57 .', {'entities': []}) \n",
      "\n",
      "('me to make a test tomorrow morning at 9:00 AM .', {'entities': []}) \n",
      "\n",
      "('µl pool 23 into lane 4 .', {'entities': []}) \n",
      "\n",
      "('me about the meeting at 3:00 PM . Meeting .', {'entities': []}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('me to take the cells out of the incubator tomorrow at 2:00 PM .', {'entities': []}) \n",
      "\n",
      "(\"means you can annotate what viable cells on the viability counts is . If it 's lower than 85 % then it 's a worry . She does n't start the assay but needs to be in the right conditions for it .\", {'entities': []}) \n",
      "\n",
      "('timer for 30 min  thank you .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('the heavy and light standard mix into column 20 .', {'entities': []}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"is in start of experiment . Isoflurane on one point . 7 months was not really Liking the isoflurane  but now it 's calm .\", {'entities': []}) \n",
      "\n",
      "('I induced IFLK BAC 16 quadruple 15 times T175 flasks .', {'entities': []}) \n",
      "\n",
      "('when ozone 1 mice are being culled .', {'entities': []}) \n",
      "\n",
      "('trying free protocol using best sensitivity . So 5 min activation and exposure for faint bands . Running the protocol .', {'entities': []}) \n",
      "\n",
      "('to A9 very complicated chemical poured into another one .', {'entities': []}) \n",
      "\n",
      "('21 1.631 E5 .', {'entities': []}) \n",
      "\n",
      "('µl apply pluro pluronic was added to kinase buffer .', {'entities': []}) \n",
      "\n",
      "('is 200 . Next . A2 is 10.5 .', {'entities': []}) \n",
      "\n",
      "('exotherm reaction  however  the added CDT solution has room temp so already big temperature difference . Dark brown solution was dense stirred at 20 ° overnight .', {'entities': []}) \n",
      "\n",
      "('the timer 30 min .', {'entities': []}) \n",
      "\n",
      "('more test to identify the latency .', {'entities': []}) \n",
      "\n",
      "(\"measurement is 0.7 . It 's number 16 already .\", {'entities': []}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('same with 22 and neon .', {'entities': []}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('timer 5 min .', {'entities': []}) \n",
      "\n",
      "('the gel also with instant blue use 3 times . Leave it over the weekend with a lock . Mark the box with tape and describing dates and contents .', {'entities': []}) \n",
      "\n",
      "(\"here the IMAC medium for the dissections . It carries  um  6 times  6 parts of Schneider 's medium . 3 parts of citrate buffer and one part of  um  FBS . So basically 6 ml of Schneider 's  3 ml of citrate and 1 ml of FBS . Later everything is mixed and filtered through the filter 22 µm filter  so it 's sterile . So I made that and I 'm filling the tubes that I prepare yesterday with 30 μl of this IMAC medium . So once I dissected tissues and cells they have something to  um  stay and stay alive .\", {'entities': []}) \n",
      "\n",
      "(\"surgery at 10:00 o'clock .\", {'entities': []}) \n",
      "\n",
      "(\"strain 168 it 's a tryptophan auxotroph so I have to add tryptophan to the media used in PY 79  the amount of auxoph the amount of tryptophan of an old recipe for transformation  they one from S SM one SM 2 media .\", {'entities': []}) \n",
      "\n",
      "('20 µl of sample and 6.7 µl of LDS buffer . Samples are prepared without heating and run on a BioRad gel .', {'entities': []}) \n",
      "\n",
      "('20 µl of sample and 6.7 µl of LDS buffer samples are prepared without heating and run on a BioRad gel', {'entities': []}) \n",
      "\n",
      "('1 sample and diluted by 5ML .', {'entities': []}) \n",
      "\n",
      "('for 10 min .', {'entities': []}) \n",
      "\n",
      "('group 3 first .', {'entities': []}) \n",
      "\n",
      "('20 ml to vial .', {'entities': []}) \n",
      "\n",
      "('me to check stat 3 .', {'entities': []}) \n",
      "\n",
      "('me to call my sister at 2 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('cell lung cancer 08 model study 19 091 . Medium intensity nuclear staining throughout entire tumor tissue . Some areas of no staining .', {'entities': []}) \n",
      "\n",
      "('sample tubes for the dissection tomorrow . Abbreviations that are written on the tubes are consisting of the number and letters . So the number 3 represents third replicate . First G  G1 is foregut . G2 is hindgut  G alone is whole gut . Then  the second - um  no  there is one letters - F stand for fat body  H for hemocytes . Second letter is - um - labeling the strain of the mosquitoes . So G stands for G3 and R for RKG . SF below the sample labels means sugar fat  BF blood fat and PF plasmodium falciparum infected .', {'entities': []}) \n",
      "\n",
      "('am starting with Channel 2 .', {'entities': []}) \n",
      "\n",
      "('LB medium 500 g 10 boxes .', {'entities': []}) \n",
      "\n",
      "('test 2 Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give ? Test end .', {'entities': []}) \n",
      "\n",
      "('phase of experiments will be attempted today . With the first perfusion solution arranged in this order . Pyrot 4 AP caffeine 5 mM 480 . 4 AP with Interleukin 1 beta . 4 AP beta leukin 1 5 mM.', {'entities': []}) \n",
      "\n",
      "('sample 2 and 3 are very grainy and basically impossible to pipette .', {'entities': []}) \n",
      "\n",
      "('357 ml of 0.7 molar TCEP to read 1 mM TCEP in the lysis buffer .', {'entities': []}) \n",
      "\n",
      "('labeled 1.5 ml cups for the compliment pool .', {'entities': []}) \n",
      "\n",
      "('test number 2 . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the roto bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('to check the cells at 11:50 PM .', {'entities': []}) \n",
      "\n",
      "('for gore tex membran  dermalon  monofilament 4 - 0 .', {'entities': []}) \n",
      "\n",
      "('me to turn on the alarm at 1:30 .', {'entities': []}) \n",
      "\n",
      "('9 1.848 E5 .', {'entities': []}) \n",
      "\n",
      "(\"am using the PAL pipette and a 5 ml stripette to do 4 ml of my volume and to do the 845 I 'm using the P1000 6 purple pipette  serial number T6052HE  calculate number DOB9UBB6KS .\", {'entities': []}) \n",
      "\n",
      "('started 4 equivalents of the acid and power formaldehyde was added  and the reaction runs at at 50 ° .', {'entities': []}) \n",
      "\n",
      "('# 3 . Top . Three sterile filtration flask bottoms about 1 l each . A set of  um  HPLC columns or micro LC columns  not used  a series of last pipets not in use . About 15 plate seals  microplate seals aluminum . And a case of plates not used . Next . Waters acuity bottles  not used . A lab jack stand . Lab bag role of absorbent liner . And 3 more filtration flask that are PVC coated . Next . Uh . About', {'entities': []}) \n",
      "\n",
      "('purification loading the diluted sample via the sample pump on the resource Q Column in position 2 with fractionation in 50 ml fractions . Uh . Right .', {'entities': []}) \n",
      "\n",
      "('counts are finished resuspending samples to 400 micro .', {'entities': []}) \n",
      "\n",
      "('timer for 45 min .', {'entities': []}) \n",
      "\n",
      "('want to add 20 ml of amylase to the experiment .', {'entities': []}) \n",
      "\n",
      "('4 well plate read at 3:15 PM .', {'entities': []}) \n",
      "\n",
      "('3 9.565 E4 .', {'entities': []}) \n",
      "\n",
      "('110044394 . Cultured in 4.2 ml', {'entities': []}) \n",
      "\n",
      "('dose of ABT 199 occured today as well .', {'entities': []}) \n",
      "\n",
      "('5 ml ammonium formate .', {'entities': []}) \n",
      "\n",
      "('ending in 8543 ? Well  I was precipitated in K10k11K12 . Oh trials .', {'entities': []}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('run human run . OK .', {'entities': []}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('37  2.670 E 5', {'entities': []}) \n",
      "\n",
      "('23 . Combination 1 media added .', {'entities': []}) \n",
      "\n",
      "('the prep of the instruments I need to immobilized my protein . This can take 30 min to a few h.', {'entities': []}) \n",
      "\n",
      "('one construct we continue to work with will be the clone 5 . Which shows Newman Newman no mutations and Um perfectly alliance with the target sequence .', {'entities': []}) \n",
      "\n",
      "(' uh  apply 20 µl of sample in each lane .', {'entities': []}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells  24 hours post infection : supernatant and dilutions of case HV wild type stock and Delta K 8.1 .', {'entities': []}) \n",
      "\n",
      "(\"sample across full protein coding genes in the genome and cross references with around phenotypic end points that they recorded which comprise very binary and quantitative phenotypes . And we 've also done this in 11 different ways that before genetic models  are just different ways of categorizing the mutations or variants . So some focus for example  on protein estate invariant or loss of function variants and others also include rare missense variant . So  in to\", {'entities': []}) \n",
      "\n",
      "('105 µl X AMP into lane 3 .', {'entities': []}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('12  13  14 contain the protein  but also contaminations .', {'entities': []}) \n",
      "\n",
      "('have to buy 15 mL centrifuge tubes .', {'entities': []}) \n",
      "\n",
      "('did the cloning for M scarlet tubulin and M scarlet life act . I used 50 ng of the vector and 100 ng of the insert . At first I did  um  ethanol precipitation and than I assembled with Gibson master mix . [ I transfused ] I transformed in Stbl 3 cells .', {'entities': []}) \n",
      "\n",
      "('cells for 30 min at 37 ℃ .', {'entities': []}) \n",
      "\n",
      "('photo shows the actual information on the four plates and numbered plate 1  plate 2  plate 4  and P plate . Cancel .', {'entities': []}) \n",
      "\n",
      "('timers have gone off . Begin dispensing . 45 µl per well to a new clear plate for the QUANTI - Blue reagent  dispensing now .', {'entities': []}) \n",
      "\n",
      "('a 3 . timer for 16 min .', {'entities': []}) \n",
      "\n",
      "('test with jabra difficult words and 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('images for D2- 2 starting on 09365 .', {'entities': []}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "(\" OK  so at least one dish from day 22 extra and one dish from day 29 extra . That 's why I decided to first do the staining on those two dishes .\", {'entities': []}) \n",
      "\n",
      "('we demonstrate the weighting process and the automatic data transfer from the scale into the ELN during the making of the new formulation . Each of the components is being weighed individually and the balance will transfer the weight data . In this case . We chose calibration weights of 55 and 20 g  respectively  for demonstration purposes to showcase the functionality . At the end of the weighting process except the data input .', {'entities': []}) \n",
      "\n",
      "(\". It depends on the data that we 're getting . Usually this influences the investment decision . New projects have a target that we generate .\", {'entities': []}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('. The cycling program will expire in 5 min . And then we are setting up a ligation reaction using', {'entities': []}) \n",
      "\n",
      "('molecular weight 213.25 .', {'entities': []}) \n",
      "\n",
      "('is Daniel Sutton .', {'entities': []}) \n",
      "\n",
      "('all the tubes  thus includes all 6 inlet tubes A1 tube  B1  B4 and B6 .', {'entities': []}) \n",
      "\n",
      "(\"transduction test . I did yesterday a test with different conditions . First condition transduction without spinfection . The second  is  um  HFF pellet on ice for 6 h with polybrene  without polybrene and the control . The third is spinfection in plate for 2 h 's . The fourth is spinfection for 2 h  the cells as in pellet and the fifth is spinfection for 6 h  the cells as in pellet . At first I will check the transduction in plate .\", {'entities': []}) \n",
      "\n",
      "('bioELN good reference for for archive data  but I tend to use the network drive 1st .', {'entities': []}) \n",
      "\n",
      "('34 best in class 3 media added .', {'entities': []}) \n",
      "\n",
      "('SEC buffer . Filter through 0.22 µm filter . Then distribute 100 ml in 3 bottles . Add to AZ2250 10 µl 10 mM compound to give a final 1 mM solution . For the AZ14180161 add 2 µl to get a final concentration of 0.2 µM.', {'entities': []}) \n",
      "\n",
      "('? ] is the pool of [ ? ] in the MXBI 5 for Doreen . Maybe 5 ml', {'entities': []}) \n",
      "\n",
      "('465 µl of denaturation into library 2 .', {'entities': []}) \n",
      "\n",
      "('is unstimulated .', {'entities': []}) \n",
      "\n",
      "(\"the first well with polybrene of the spinfection for 2 h pellet  I can see green cells  but not a lot . I would say maybe 1 % . Um . Without polybrene I ca n't see . I ca n't see green cells .\", {'entities': []}) \n",
      "\n",
      "('just set up the sorter on 150 µm nozzle for single cell sorting from days . Efficiency of the drop delay is little bit flexible around 0.1 % to 0.5 % terms of non sorted reads . This should be OK for single cell sorting for her purposes .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('HU 1645 .', {'entities': []}) \n",
      "\n",
      "('33 140000', {'entities': []}) \n",
      "\n",
      "('29065 - 013 . E9 was dissolved in DMF 2 ml and cool down to 0 ° . Stirred until temperature was stable . In the mean time the CDT was dissolved in 5 ml DMF at room temp .', {'entities': []}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('condition will have 1 ml of total volume . Resuspend the cells in 1.5 ml and aliquot 500 µl / condition .', {'entities': []}) \n",
      "\n",
      "('condition will have 1 ml of total volume . Resuspend the cells in 1.5 ml and aliquot 500 µl / condition .', {'entities': []}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('my experiment plan 001 .', {'entities': []}) \n",
      "\n",
      "('chabra in the lab  radio in the background  no special noise . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('block for today . Started working on the speed slide from 8:00 to 9:30 . 10:30 to 11:30  synchronization . Then lunch break  now start again .', {'entities': []}) \n",
      "\n",
      "(\"means you can annotate what viable cells on the viability counts is if it 's lower than 85 % then it 's a worry . She does n't start the assay  but needs to be in the right conditions for it .\", {'entities': []}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('weighed out 32.7 mg of standard for MK 1234 .', {'entities': []}) \n",
      "\n",
      "('mentioned he kept the pump speed at 60  but the pump speed is set to 120 .', {'entities': []}) \n",
      "\n",
      "('light stream mounts one stimulation MA .', {'entities': []}) \n",
      "\n",
      "('29 1.634 E5 .', {'entities': []}) \n",
      "\n",
      "(\"Pi PCR cycle test : protein concentration is n't limitating step for melting curves . High amount of Body Pi agent needed to see good curves . More than 10 µg / ml to get good results and great curves .\", {'entities': []}) \n",
      "\n",
      "('buffer for FSEC 40 mM HEPES pH 7.4  300 mM sodium chloride  1 mM TCEP  10 % Glycerol . Changing FSEC from water to buffer and let it run for 20 min .', {'entities': []}) \n",
      "\n",
      "('in the back 12 % gel in the front 6 % .', {'entities': []}) \n",
      "\n",
      "('the buffer of 20 mM Tris pH 7.5  120 mM sodium chlorid  0.5 mM TCEP at the flow rate of 1 ml / min .', {'entities': []}) \n",
      "\n",
      "('fade prepared from aliquotes in assay diluent is added to column 3 and column 4 .', {'entities': []}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('me to select the HFFs 48 hours . Later', {'entities': []}) \n",
      "\n",
      "('try of experiment for calibration . File name is Cal25 for film and for behavior . Same conditions as before .', {'entities': []}) \n",
      "\n",
      "('is test number 2 .', {'entities': []}) \n",
      "\n",
      "('were started at 7:07 . Sorry  3 min .', {'entities': []}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('me to check the solvent tomorrow at 8:00 AM .', {'entities': []}) \n",
      "\n",
      "('C9 to C12 precipitated .', {'entities': []}) \n",
      "\n",
      "('29 Carboplatin media added .', {'entities': []}) \n",
      "\n",
      "('cell number is 3 to the power of 6 .', {'entities': []}) \n",
      "\n",
      "('me in 10 min about blood sample . Avocado . Blood sample .', {'entities': []}) \n",
      "\n",
      "('a timer for 5 min .', {'entities': []}) \n",
      "\n",
      "('2 ml slurry strep Tactin Sepharose High performance from GE Healthcare catalog number 28 -93255 - 99 . LOT 1028253 expiry date 22 August 2022 August . Take 2 ml slurry and wash with water .', {'entities': []}) \n",
      "\n",
      "('5 cycle . No surface changes including cut edges and wells . ° of Ristine DINNISO 4628 dash 3 . After 30 cycles no surface changes including cut edges and wells loss of addition . D less than 1.5 mm .', {'entities': []}) \n",
      "\n",
      "('I said about the Neuralinks file was all wrong because now suddenly I do not get error messages ? I have the following thng : the first one  is optostimulus than 0  1 is the clock behavior 10 milliseconds  2 is force sensor  3 and 4 empty . Yeah .', {'entities': []}) \n",
      "\n",
      "('number 3 is very fatty which will make it even easier even more difficult to dounce . You can see visibly the fat the position and the yellow color so we can expect a lot of adipocytes in the analysis . [ inaudible ]', {'entities': []}) \n",
      "\n",
      "('cassette is 3.9 .', {'entities': []}) \n",
      "\n",
      "('ml of water were added .', {'entities': []}) \n",
      "\n",
      "('cell concentration is 260000  uh  in a total volume of 15 ml . Uhm I need to add 297 ml to a final volume of 312  Uh  to see .', {'entities': []}) \n",
      "\n",
      "('a second timer for 43 min .', {'entities': []}) \n",
      "\n",
      "('started at 3:07 PM .', {'entities': []}) \n",
      "\n",
      "('2- 2 chip has a entry channel minor debris . There is a fiber inside one of the channels . And there is another debris at the entry to post interface . There is minor cell wrapping in the post arrays getting progressively worse as you go through the chip . Um at the exit post entry interface there is minor cell debris and spider like spider web like cell debris . Um chip flow is going from bottom to top of image . There is a larger dimple in the chip that might not have been caught during an earlier inspection .', {'entities': []}) \n",
      "\n",
      "('we are going to seed 20000 cells per milliliter  and we have three control cell lines  and five patient cell lines . The cells will be seeded in normal DMEM medium  and the [ inaudible ] lysis would be prepared exactly after 24 hours of incubation at 37 degrees .', {'entities': []}) \n",
      "\n",
      "('day 14 harvests .', {'entities': []}) \n",
      "\n",
      "('. 1.8 million . 1732 . 2.2 million . Image 2  one point 5000000 .', {'entities': []}) \n",
      "\n",
      "('still have 10 samples .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('solution addition completed plate is incubated at 450 rpm .', {'entities': []}) \n",
      "\n",
      "('was not very much VES and other arrhythmias during the critical time point of the occlusion . Possible reasons for that could be that the temperature of the pig is quite low . 35.8 °  and or maybe that there is a leakage along the balloon  so that there is some kind of blood supply maybe . And the 3rd might be that she is in Bronst .', {'entities': []}) \n",
      "\n",
      "('weight is 3.1 pg .', {'entities': []}) \n",
      "\n",
      "('8 dilutions starting at 150 . Going to 0 .', {'entities': []}) \n",
      "\n",
      "('5 9.754 E4 .', {'entities': []}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('power reading 3.7 ... no sorry  maximum power reading for light measurement 3.06 mW. Diameter 200 micrometers . Producer : Thorlabs . Name : FG200UEA - Custom - Muke ? . : .22 ', {'entities': []}) \n",
      "\n",
      "('me that tonight is a lab meeting at 5 PM .', {'entities': []}) \n",
      "\n",
      "('set the water bath at 212 ° Fahrenheit .', {'entities': []}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('test with headphones next to the radio holding microphone close to the mouth . Extraction . The biphasic solution was steered at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test .', {'entities': []}) \n",
      "\n",
      "('taking off the PFA from my calbindin DCA 10305 slices .', {'entities': []}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "(\"so that totally did n't fix APCH 7 comma . I 'm going to have to manually do that 1 .\", {'entities': []}) \n",
      "\n",
      "('addition to the creatinine analysis of urine samples yesterday  I have now also analyzed plasma creatinine in the same study  meaning 27 samples . I used 15 µl of plasma provided by Gun - Britt Forsberg and used the same method  namely 15 µl of plasma and then 200 µl of internal standard . And I added these set of samples to the previous sample list in the mass spectrometer using the previously prepared calibration and QC samples for quantification .', {'entities': []}) \n",
      "\n",
      "('. 190 µl PW 1 to dilution .', {'entities': []}) \n",
      "\n",
      "('just have to design a sequence with T7 7 promoter followed by DNA sequence .', {'entities': []}) \n",
      "\n",
      "('are seeing 5 T2 25 layer flask . With about 10000000 cells each and 45 mLs of media .', {'entities': []}) \n",
      "\n",
      "('is where tests are requested by chemists . This point integrates the tested and untested assays into 1 area .', {'entities': []}) \n",
      "\n",
      "('be the NP40 buffer the problem ? I have to repeat it  um  with the same amount 60 µg of protein . And I try to devoid the smear .', {'entities': []}) \n",
      "\n",
      "('the detection plates for 2 to 3 min', {'entities': []}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('equal 0.1 . 5.2 . Equal . 0.1 . 5.3 . Equal . 0.1 . 5.4 equal . 0.11 . 5.5 equal . 0.096 .', {'entities': []}) \n",
      "\n",
      "(\"for f * * * sound 2 o'clock .\", {'entities': []}) \n",
      "\n",
      "(\"ordered some lithium chloride and some 2 3 benzene chloride . That 's not bad .\", {'entities': []}) \n",
      "\n",
      "('15 - 3 starts on 09442 .', {'entities': []}) \n",
      "\n",
      "('08095 . At least 10 % conversion in 1 h. Continue stirring .', {'entities': []}) \n",
      "\n",
      "('26 % B  changed gradient to 100 % B in 3 min .', {'entities': []}) \n",
      "\n",
      "(\"reducing aspirate to 20 eliminates all plate patterns . We just need to check whether it 's compatible with cells and whether it washes them off .\", {'entities': []}) \n",
      "\n",
      "('502 row 1 column A', {'entities': []}) \n",
      "\n",
      "('temperature for amplification was achieved in C1  D1  E1  F1  G1 and H1 .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('plate is on Orbital Shaker at 600 RPM . Approximately will be for 2 h from now .', {'entities': []}) \n",
      "\n",
      "('ml of solution have been solved .', {'entities': []}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('the following notes to experiment BE0002668 - 33 .', {'entities': []}) \n",
      "\n",
      "('to a seminar at 5 pm on October 29th .', {'entities': []}) \n",
      "\n",
      "(\"upload my results to the D360 . Here it 's difficult to find the information from the legacy experiments . I rely on links within the write up for this .\", {'entities': []}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('formate buffer pH is 7.12 .', {'entities': []}) \n",
      "\n",
      "('C5 to the tube concentration tube and concentrate again for 5 min .', {'entities': []}) \n",
      "\n",
      "('a 100 . Bye .', {'entities': []}) \n",
      "\n",
      "('the experiment DNA research G237 . DNA research .', {'entities': []}) \n",
      "\n",
      "(\"temperature 920 37.3 . It 's before Blood Gas .\", {'entities': []}) \n",
      "\n",
      "('% of my work is bespoke talk studies . 20 % will be what we have observed today .', {'entities': []}) \n",
      "\n",
      "(\"did n't patent 29 .\", {'entities': []}) \n",
      "\n",
      "('size exclusion at 10 to 4 .', {'entities': []}) \n",
      "\n",
      "(\"'s an RKG hemolin perfusion . Replicate 3 blood fat mosquitoes . I perfused 12 mosquitoes in total and dissected 5 guts  the foregut and the hindgut  hindgut .\", {'entities': []}) \n",
      "\n",
      "('a list . Remind me that I have 3476 million cells for sample 3 .', {'entities': []}) \n",
      "\n",
      "('we go . Day 0 samples labeled 79 and 84 . PHH followed by NPC Day 7 81  82 PHH followed by NPC .', {'entities': []}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('with difficult words writing zone  no noise microphone close to the mouth . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('I have 12 IMR 32 vials  Cryovials and I put them in my box for the liquid nitrogen . In row 6 61 through 9 and then row 7 133 .', {'entities': []}) \n",
      "\n",
      "('sample 2 203000 .', {'entities': []}) \n",
      "\n",
      "('viability  1.92 number of cells 1732 . 95 % viability  1.98 cells images 2 .', {'entities': []}) \n",
      "\n",
      "('stop equals 16 µl per 6 ml for a 4 x stock concentration .', {'entities': []}) \n",
      "\n",
      "('primary antibody to Western blot after 2 h.', {'entities': []}) \n",
      "\n",
      "(' the last results are more and less comfortable  but ionic stimulation to be changed our race from 200 to 400 in a seconds and here I guess we also have to extend the surface . Maybe just check the surface shape in comparison to the surface of this special amino acid . And also just divide this protein into 2 halfs and then simulate per each half . This would be a great idea .', {'entities': []}) \n",
      "\n",
      "('596 µl of hyb buffer .', {'entities': []}) \n",
      "\n",
      "('20 mililiter', {'entities': []}) \n",
      "\n",
      "('that I need to give Sue the DCAS9 krab MCherry Vector .', {'entities': []}) \n",
      "\n",
      "('will harvest all HFFAX 172 cells with all dilution . But for the HFFAX 173  174 and 175 just the dilutions 1 in 25 and 1 in 50 .', {'entities': []}) \n",
      "\n",
      "('44 µl of Light Glutathione . 200 ml of the the 1X of the heavy internal standard mix .', {'entities': []}) \n",
      "\n",
      "(\"me to tell the following . 1 the tasks if it 's a reminder do n't work . 2 . Need to add notebook pages . 3 .\", {'entities': []}) \n",
      "\n",
      "('test number 7 . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . New paragraph . Test end .', {'entities': []}) \n",
      "\n",
      "('the assay we optimized the set up . Everything that is written in the protocol will be fine . But today  we want to measure cytokines from the same wells that were measuring of the viability assay . So what we did was we took 20 µl of the culture and transferred this over into a 96 well V - bottom plate and we added the correct amount of membrane permeability as well as caspase activity  dye into this well in a formula of one to one dilution . So 20 µl of culture +2 µl of the um  reagent that we use in this case . The reagent mass was set up in a way that we use 2470 µl media we added 25 µl of cell permeability dye and 5 µl of Caspase activity stain . This was incubated in the incubator for one h at 37 ° and in order to stretch the time between reading and measuring with the attendees  we stored them at room temperature in the dark .', {'entities': []}) \n",
      "\n",
      "('me to take out the plate samples at 8:00 AM tomorrow .', {'entities': []}) \n",
      "\n",
      "('cells in 20 µl of phosphorus safe lysis buffer .', {'entities': []}) \n",
      "\n",
      "(\"this new components are being added to the formulation . In this case  we want to use three components in our new formulation . We then define the type of component from the drop down menu . In this case  we 're using an additive a solvent and a particle . Then we identify the specific components to be used from the inventory management module . Here we 're using cyclohexanol MEP0035 and lower 11 K and we defined the formulation specific weight percentage . The total percentage is being calculated and needs to be 100 % .\", {'entities': []}) \n",
      "\n",
      "('20 µl of sample and 6.7 . µl of LDS buffer samples are prepared without heating and run on a buy Orajel .', {'entities': []}) \n",
      "\n",
      "('ID 18S006290 . Sample value 3.705 ng/µl . sample ID 19S00460 sample value 2.00 ng/µl sample ID 18S006286 sample value 16.80 nG/µl .', {'entities': []}) \n",
      "\n",
      "(\"ca n't see green cells in the normal transduction without spinfection . And I ca n't see green cells in the wells  which were the cells on ice before . But in the normal protocol for spinfection for 2 h  I can see some a lot of green cells after 26 h in the well with polybrene . I would say I can see round about 10 % green cells  5  between 5 and 10 % in the well without polybrene . I can  I can see some green cells but more and less as in the well with polybrene . I can see just one or 2 green cells . The control well is ok  I ca n't see green cells . Now I will check the spinfection as in pellet for 2 h. And I can see green cells .\", {'entities': []}) \n",
      "\n",
      "(' the experiment is BS 17 .', {'entities': []}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('volume is 1.0035 nl .', {'entities': []}) \n",
      "\n",
      "('me to pay rent at 5:00 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('group 3 first .', {'entities': []}) \n",
      "\n",
      "('650 out of PE looks weird .', {'entities': []}) \n",
      "\n",
      "('up the Cas9 protein from the -80 freezer in Tray C2 .', {'entities': []}) \n",
      "\n",
      "('cell lung cancer 08 model study 19 091 . Medium intensity nuclear staining throughout an entire tumor tissue . Some areas of no staining .', {'entities': []}) \n",
      "\n",
      "('denaturation 50 µl 200 mM Tris to denaturation .', {'entities': []}) \n",
      "\n",
      "('44 8.106 E to the 4 .', {'entities': []}) \n",
      "\n",
      "('A2 1.11 e5 .', {'entities': []}) \n",
      "\n",
      "('7 432000 .', {'entities': []}) \n",
      "\n",
      "('1.25 ml imidazole from 1 molar stock pH 7.5 to the lysis buffer to read 5 mM imidazole .', {'entities': []}) \n",
      "\n",
      "('me to check on sample # 3 .', {'entities': []}) \n",
      "\n",
      "(\"concentrations . I have 12 samples . Always agree  Elisa . Elisa  every morning I have bring the ice packet full ice to you  every morning . Saving now . I 'm testing it . How are you ? I 'm trying . How are you trying ?\", {'entities': []}) \n",
      "\n",
      "('export things like the kinetic affinity and the binding data . This table goes into the D360 and my write up .', {'entities': []}) \n",
      "\n",
      "('ID 18s00068 . Sample value 3 dot 705 ng/µl . Sample ID 18s0089 . Sample value 3.705 ng/µl . Sample ID 19s0098 . Sample value 3.50 ng/µl .', {'entities': []}) \n",
      "\n",
      "('10 % TFA 1 µl is added to the sample to give a final concentration of 0.5 % TFA in samples to stop trypsination .', {'entities': []}) \n",
      "\n",
      "('note a change in temperature from 27 .', {'entities': []}) \n",
      "\n",
      "('30 1.613x10 to the 5', {'entities': []}) \n",
      "\n",
      "('started at 4:10 .', {'entities': []}) \n",
      "\n",
      "('cells are at 85.6 percent viability and their concentration is 1.71 . And before cells are at 93.3 percent viability their concentration is 2.10 . OCI cells are at 97.8 percent viability in their concentration is 2.43 .', {'entities': []}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('plasmid . 7 .', {'entities': []}) \n",
      "\n",
      "('incubated at 42 ° for 5 min .', {'entities': []}) \n",
      "\n",
      "('reduction of the aspirate to 25 immensely improves plate pattern . The only wells left are now B 23 and 24 and C 23 and 24 .', {'entities': []}) \n",
      "\n",
      "('4 number sample number 13  E2  E3  E4  E5  and E6 .', {'entities': []}) \n",
      "\n",
      "('step is to add 200 µl of 0.1 % no  0.5 % TFA in 5 % acetonitrile and centrifuge at 1500 RCF for 1 min at room temperature . This step is repeated twice .', {'entities': []}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('me  I have a meeting with Dennis at 1330 .', {'entities': []}) \n",
      "\n",
      "('D2 - 3 images end 09402 .', {'entities': []}) \n",
      "\n",
      "('started incubation for plate 1 now .', {'entities': []}) \n",
      "\n",
      "(\"Prop 2 turn out to be the best probability of the events . In control Def Prop 3  the events probability of infected so I need now to control this 2 compared with 2 different health functions . The one that I already have . And another one that includes a longer lifespan and with the Orange curve that I sealed in a where I got it from . But it gives me a much shorter lifespan . I expect that in the Orange . There should not be an infection and it 's important now to compare the event probability of infected versus controlled because they 're . There might be a little bit of . Differences um . Maybe I said  also the red curve  um just as an extra test  but I do n't think it 's very well . Really  necessary  so set up a list of events probability infected control to heal functions the one that I have the longer one . And the Orange curve and for each of these different death rates  then I need to make 5 to 10 simulations .\", {'entities': []}) \n",
      "\n",
      "('me in 30 min to bring out the week planning protocol . Print out the week planning protocol .', {'entities': []}) \n",
      "\n",
      "('15 ml 5 molar sodium chloride to lysis buffer .', {'entities': []}) \n",
      "\n",
      "(\"remembered that the cider green mix was not supposed  there was no miscalculation on the 5 o'clock mix ? Bumblebee .\", {'entities': []}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('12  13 and 14 will be pooled and fractions 15  16 and 17 will be pooled .', {'entities': []}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('jabra radio German in the background . Extraction The BI phasic solution was still at room temp for 20 min and then transfer it into a separation follow . Face cut is clear and fast . The organic lawyer was separated and washed once with 2 ml bicarb solution . The clear yellow solution was dried on the road about rotavap at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('µl of protein at different concentrations was added to plate .', {'entities': []}) \n",
      "\n",
      "('4 is being verified using 2.5 µl 6 shots .', {'entities': []}) \n",
      "\n",
      "('20 mix to vial . Is it working ?', {'entities': []}) \n",
      "\n",
      "('close to the mouth 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 bar 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('blue plates  1 min  1000 rpm', {'entities': []}) \n",
      "\n",
      "('18 1.915', {'entities': []}) \n",
      "\n",
      "(\"I 'm adding 30 µl of reagent A into well B6 .\", {'entities': []}) \n",
      "\n",
      "('. 0.29 . 4.2 . 0.30 . 4.3 . 0.27 . 4.4 . 0.28 . 4.5 . 0.3 .', {'entities': []}) \n",
      "\n",
      "('1 [ ladder ? ] . Second . Point .', {'entities': []}) \n",
      "\n",
      "('23 2.0752 E5 .', {'entities': []}) \n",
      "\n",
      "('timers have gone off . Begin dispensing . 45 µl per well to a new clear plate for the QUANTI - Blue reagent  dispensing now .', {'entities': []}) \n",
      "\n",
      "('timers have gone off . Begin dispensing . 45 µl per well to a new clear plate for the contact blue reagent  dispensing now .', {'entities': []}) \n",
      "\n",
      "('46 µl leaders of Phi X into dilution .', {'entities': []}) \n",
      "\n",
      "('cell line CHODG 44 .', {'entities': []}) \n",
      "\n",
      "('formate is normally 9.45 g / l times 1.5 . It is .', {'entities': []}) \n",
      "\n",
      "(\"seems to be the optimal setting . We 're not losing any cells . Strikes a balance between good wash no plate effects in terms of media . And cells wash ideally I wanted to be lower .\", {'entities': []}) \n",
      "\n",
      "('330  neural - D1 Medi small cell lung cancer model 5 . Pockets of nuclear expression throughout the tumor tissue  but to a much lesser degree than the last slide that I reviewed which was small cell lung cancer 02 model . Some cytoplasmic staining also .', {'entities': []}) \n",
      "\n",
      "('130 µl into lane 2 .', {'entities': []}) \n",
      "\n",
      "(\"the maximum and minimum of experiment 5  there are 2 maximums because the cell was n't completely induced after killing the cells . But you should try analyzing the 2 peaks to see whether fits with them similar experiments because . The pipette might contribute to the fluorescence .\", {'entities': []}) \n",
      "\n",
      "('ID 76 also added to T9 .', {'entities': []}) \n",
      "\n",
      "('serial dilution was done with 200 µl of each concentration .', {'entities': []}) \n",
      "\n",
      "('samples EGDE 1485  1020  1042  1019  1033  1018  3009  2011  2012  1034 of listeria for listeria monocytogenes detection system tomorrow .', {'entities': []}) \n",
      "\n",
      "('me to feed the cells Wednesday at 11 AM .', {'entities': []}) \n",
      "\n",
      "('protein PMS 2 - 12 . Evaluation . Elution program 135 to 210 3 ml each . Pooling fraction 16 to fraction 40  pooling and loading 2 Q Sepharose Column .', {'entities': []}) \n",
      "\n",
      "('11:50 restart elution .', {'entities': []}) \n",
      "\n",
      "('19 2.660 E5 .', {'entities': []}) \n",
      "\n",
      "('next measurement is [ Tukat ? ] 25 .', {'entities': []}) \n",
      "\n",
      "('the SEC run the fractions of B3  B1  C2  C4  C6  C8  D3 and E4 were tested on SDS PAGE .', {'entities': []}) \n",
      "\n",
      "('32  1.963 E5 .', {'entities': []}) \n",
      "\n",
      "('the 3rd plate as the first plate to dispense with the multidrop  to', {'entities': []}) \n",
      "\n",
      "('volume 70 µl .', {'entities': []}) \n",
      "\n",
      "('would make 3 l of buffer .', {'entities': []}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('preparing the sodium chloride buffer weigh 29.2 g of sodium chloride from a new bottle . Merck . Uh .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 22 ° to 25 ° .', {'entities': []}) \n",
      "\n",
      "(\"it 's working . Now I 'm starting up to 5 min  2 ml on the first condition for the first condition and OK  then when it starts with a wake word .\", {'entities': []}) \n",
      "\n",
      "('of membrane prep 3 correct it membrane prep 8 .', {'entities': []}) \n",
      "\n",
      "('50 ml solubilization buffer by taking 27.5 ml 2 times solubilization buffer 27.5 is to compensate for the slight extra dilution  1 tablet of protease inhibitor  5 ml of 10 % LMNG  and fill up with Milli Q Water until 50 ml .', {'entities': []}) \n",
      "\n",
      "('D  3 and 3 µl .', {'entities': []}) \n",
      "\n",
      "('project PMS 6 1 agarose gel .', {'entities': []}) \n",
      "\n",
      "('codes ending on 0112 and barcode ending on 027 are outlier plates . Do not include those in the analysis .', {'entities': []}) \n",
      "\n",
      "('the cell wash travel rate to 1 does not appear to change the pattern of the washer .', {'entities': []}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('µl TLRG into lane 1 .', {'entities': []}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "(\"me on Monday at 9:00 o'clock to run a SDS - PAGE of the pooled samples .\", {'entities': []}) \n",
      "\n",
      "('me to go to my colonoscopy tomorrow at 8:30 AM .', {'entities': []}) \n",
      "\n",
      "(\"upload my results to the D360 . Here it 's difficult to find the information from legacy experiments . I rely on links within the write up for this .\", {'entities': []}) \n",
      "\n",
      "('me after 5 min .', {'entities': []}) \n",
      "\n",
      "('plate 1 was plated at 3:10 AM . Yeah .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('me in 30 min incubation is finished .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of things in the gel . Point 1 . jill . Jill', {'entities': []}) \n",
      "\n",
      "('Five cells passage 13 measured at 11:30  at 11:30 .', {'entities': []}) \n",
      "\n",
      "('were measuring the optical density of cyanobacteria and their adsorption was at 0.23 .', {'entities': []}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('the colony PCR here we are choosing 10 new colonies of our spectrum esteem plates .', {'entities': []}) \n",
      "\n",
      "('100 nM', {'entities': []}) \n",
      "\n",
      "('analysis of Cas9 in vivo samples storage .', {'entities': []}) \n",
      "\n",
      "('49  position A7  maybe a small crystal .', {'entities': []}) \n",
      "\n",
      "('of more UPF2 to the reaction . would it effect ?', {'entities': []}) \n",
      "\n",
      "('for 10 min .', {'entities': []}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "('culture 25 ml pipette 6 bags  Cell culture 10 ml pipettes 15 bags  cell culture 5 ml pipette 6 bags  General pipettes 5 ml 20 bags  General pipettes 25 ml 12 bags  General pipettes 10 ml 30 bags .', {'entities': []}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('105 µl X AMP into lane 4 .', {'entities': []}) \n",
      "\n",
      "('37  5.965 E 4 .', {'entities': []}) \n",
      "\n",
      "('phospholipidosis assay take 10 ml of the stock cell suspension and add 148.57 ml of media . For the acoustic mass spec assay take 6 ml of the cell suspension and add 120.86 ml of media .', {'entities': []}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('buffer 20 .', {'entities': []}) \n",
      "\n",
      "(\"me to do the treatment at 9 and 10 o'clock . And also to prepare Fred buffer and H2O2 dilutions .\", {'entities': []}) \n",
      "\n",
      "('stop equals 16 µl per 6 ml for a 4 x stock concentration .', {'entities': []}) \n",
      "\n",
      "('µl pool 20 into lane 1 .', {'entities': []}) \n",
      "\n",
      "('worst thing is the slowness of the app itself . The completing and closing and converting it to a PDF takes around 10 min alone .', {'entities': []}) \n",
      "\n",
      "('of the aspirate Z offset eliminated plate pattern completely . However  may be dangerous territory for getting a good cell wash . We need to investigate how much further we can increase it so as not to have a plate pattern  but not to wash out the cells either . Going back to 30 .', {'entities': []}) \n",
      "\n",
      "(\"added 2 µl of TFA to my protein prep . And my protein precipitated . And I spun it down and examine the supernatant . The supernatant did n't have any protein . I was very angry and went home .\", {'entities': []}) \n",
      "\n",
      "('will seed 11 flasks of fibroblasts today . Probably the cell density of around 4.5 million cells per 225 .', {'entities': []}) \n",
      "\n",
      "('marker 1 20 µl of the Marker 6 µl buffer protein marker 2 9 µl marker 3 µl buffer .', {'entities': []}) \n",
      "\n",
      "('new notebook entry 001 .', {'entities': []}) \n",
      "\n",
      "('220 µl to tubes 1 through 12 . Good morning  Nami .', {'entities': []}) \n",
      "\n",
      "('for 3 min .', {'entities': []}) \n",
      "\n",
      "('10 µl of LysoPI', {'entities': []}) \n",
      "\n",
      "('test experiment  CAL 25  with the same amplitudes as before .', {'entities': []}) \n",
      "\n",
      "(\"residual softener itself smells as well . The whole plastic part from the outside is cheap  really cheap probably sounds polyethylene or maybe polycarbonate but does n't even look glass fiber reinforced and makes a quite soft impression . And the smell is also very prominent . The display is somehow movable but when you turn the display you move the part inside the tank as well the white filter is what I mean by part . The Box is made out of a gray material and the material is the same inside the box . So the tank is made out of Polycarbonate or something like that . On the backside they just drilled a hole in the tank and pushed a tube through . I ca n't see if there 's anything like a seal or something like that  and it does n't even look properly drilled . And that is the transparent tube that goes into the top unit . Well  Yeah . Um . The power unit is stated with 102 .\", {'entities': []}) \n",
      "\n",
      "('cells in 20 µl of phosphorus safe lysis buffer .', {'entities': []}) \n",
      "\n",
      "('- 3 has minor cell wrapping around the entry channels dibbles are present in the chip . Fluid flow goes from bottom to top of image . One piece of debris in the entry channel post interface starting at the 2nd row . There is spider web like cell debris . An increasingly gets worse throughout the chip at the exit posts channel interface uh There is minor sell wrapping and spider web like debris .', {'entities': []}) \n",
      "\n",
      "('38 1.813 E5 .', {'entities': []}) \n",
      "\n",
      "('10 kilodalton centrifugal filter to concentrate the fractions of the His purification .', {'entities': []}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('me to enter the imaging data at 4:00 PM . OK  there it went . It worked  yeah .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('4 best in class 2 media added .', {'entities': []}) \n",
      "\n",
      "('Apo elution from experiment 66 is slightly less than 20 µl . Add to the concentrated samples recovered from the Nanodrop . Add 19 µl of water and then prepare samples as for the others .', {'entities': []}) \n",
      "\n",
      "('. I will aim at 70 decibels noise .', {'entities': []}) \n",
      "\n",
      "(\"'m going to inactivate bacteria in different ways : one way is by fixation in paraformaldehyde  secondly is fixation by ethanol . way is inactivation by it . Forth way is by treatments with antibiotics and third way is UV treated . So . first wash the bacteria two times in 0.1 % sodium chloride .\", {'entities': []}) \n",
      "\n",
      "('Assay R271 no optimal temperature was achieved .', {'entities': []}) \n",
      "\n",
      "('air filter .', {'entities': []}) \n",
      "\n",
      "('. Equal 0.16 . 4.2 . Equal . 0.17 . 4.3 . Equal . 0.17 . 4.4 Equal . 0.17 . 4.5 . Equal . 0.17 . 5.1 equal . 0.063 . 5.2 . Equal . 0.063 . 5.3 . Equal . 0.065 . 5.4 equal . 0.066 . 5.5 equal . 0.046 .', {'entities': []}) \n",
      "\n",
      "('5 ml of DNA substance .', {'entities': []}) \n",
      "\n",
      "('Five cells passage 13  measured at 11:30  at 11:30 .', {'entities': []}) \n",
      "\n",
      "(' need to dilute 231.5 total ml / ml  will add 6.5 ml to bring  um  solution to 800000 cells per ml', {'entities': []}) \n",
      "\n",
      "('µl transferred .', {'entities': []}) \n",
      "\n",
      "('me to check if we have enough filter tips 1000 µl .', {'entities': []}) \n",
      "\n",
      "(\"case you did n't catch that that was after 48 h.\", {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "(\"aim of today 's assay is to determine kinetic values for Her2 protein .\", {'entities': []}) \n",
      "\n",
      "('me to buy a new book tomorrow at 4:00 PM .', {'entities': []}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('test jabra with English radio in the background extraction . The biphasic solution was stared at room temp for 20 min and then transferred into separation funnel the face cut is clear and fast the clear yellow solution was dried on the rotarvap at 50 ℃ and 10 mbars to give . Test end', {'entities': []}) \n",
      "\n",
      "('5 cells passage 13 measured at 11:30 at 11:30 .', {'entities': []}) \n",
      "\n",
      "('µl of magnesium chloride was added to 1 ml of water to create a final concentration of 10 mM', {'entities': []}) \n",
      "\n",
      "('me to take the cells out of the incubator tomorrow at 8:00 AM .', {'entities': []}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "('me on Friday 25th to email San .', {'entities': []}) \n",
      "\n",
      "('need to buy 20 ml of .', {'entities': []}) \n",
      "\n",
      "('10 ml of water add .', {'entities': []}) \n",
      "\n",
      "('23 4.484 E4 .', {'entities': []}) \n",
      "\n",
      "('will prepare a cell culture experiment with DDB1  DDA1 CAT2 viruses in different concentrations and he will titrate them against each other and try to harvest the protein which is expressed after 48 and 60h and compare protein production in a SDS gel .', {'entities': []}) \n",
      "\n",
      "('filter the lentiviruses and retroviruses with small filter 0.45 µm and freeze it at -80 ° .', {'entities': []}) \n",
      "\n",
      "('9  9.216 E 4', {'entities': []}) \n",
      "\n",
      "(\"'re gon na add Pen - strep and gentamicin . I got ta add 5 mL of Pen - strep from the freezer . Am I gon na add . gentamicin as well  I 'm going to have both roles directly to the Abbey Cell D man mix . Gentamicin is 10 mg per mL concentration . Also I should end up with a concentration of slightly more than 10 µg per mL. The pen - strep concentration is the same one that we use all the time .\", {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('10 µl of all samples .', {'entities': []}) \n",
      "\n",
      "('all the tubes in water . This include 6 sample inlet A1', {'entities': []}) \n",
      "\n",
      "('heavy labelled carnitine a 100 µl .', {'entities': []}) \n",
      "\n",
      "('14  2.271 E 5 .', {'entities': []}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('1 to 8 are coated with 52717 and columns 9 to 12 are coated with M232 .', {'entities': []}) \n",
      "\n",
      "('48 2.329 E5 .', {'entities': []}) \n",
      "\n",
      "('SDS page samples starting with 50 µl peak 1  50 µl peak 2  50 µl peak 3  which is the most interesting 1 then peak 4 and finally from the last purification flowthrough fraction B3 .', {'entities': []}) \n",
      "\n",
      "('substrates for large batch . Final concentration .5 mg / ml .', {'entities': []}) \n",
      "\n",
      "('have to order new T25 flasks .', {'entities': []}) \n",
      "\n",
      "('me of a meeting at 8 PM today .', {'entities': []}) \n",
      "\n",
      "('45 1.783 E5 .', {'entities': []}) \n",
      "\n",
      "('. I centrifuged the cells in the ultra centrifuge . Avocado .', {'entities': []}) \n",
      "\n",
      "(\"heat stock that I 'm going to use today is the one I made myself . And the concentration of heat stock is 500 million molar .\", {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('number is HU 17 28 .', {'entities': []}) \n",
      "\n",
      "('one . extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . No special noises  alone in the lab . Test end .', {'entities': []}) \n",
      "\n",
      "('a list of the following 3 donors : RBC 5619  110044394  KP59331 .', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('MEM nonessential amino acid solution . 5 ml to 500 ml of media .', {'entities': []}) \n",
      "\n",
      "('I am harvesting [ inaudible ] NP16 with and without [ inaudible ]', {'entities': []}) \n",
      "\n",
      "('well plates 2 . 1.5 ml cell suspension . 180 µl per well . Last row no cells .', {'entities': []}) \n",
      "\n",
      "('volume 220 ml of the 2 times solubilization buffer .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "('number 7 or plasmid number 7 . Concentration 197.2 ng/µl . Fraction 1 1.86  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('the Q - box to add 2 * .5 l of water to the Tris powder . Note that the variation on the Q - pot is I would say approximately 10 to 15 ml extra if you compare it to a 500 ml cylinder .', {'entities': []}) \n",
      "\n",
      "('4 experiment  5 inactivated bacteria KU parent wild type 2.171 .', {'entities': []}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('timer for 2 min . Now .', {'entities': []}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('of the IC 50 values for NVP 2 tend to be clustered around 25 to 35 nmol . ? 16  13  32s .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be uncontaminated .', {'entities': []}) \n",
      "\n",
      "('sodium hydroxide dilution . 180 µl PWR to dilution .', {'entities': []}) \n",
      "\n",
      "('. 123 . Bullets hello . . Bullet . 123 .', {'entities': []}) \n",
      "\n",
      "('filtering  the Tyrode with ascorbic acid with glucose now is like 0.2 µm filter .', {'entities': []}) \n",
      "\n",
      "('HEPES buffer I used was half the concentration than the standard concentration  so I had to take out 717 µl from initial buffer . Bumblebee .', {'entities': []}) \n",
      "\n",
      "('Mass spectrometry . Check metabolite abundance between 5 biological replicates of 2 mutants . Blueberry .', {'entities': []}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "(\"making buffer I have used my own stock of 500 mM HEPES and since it 's half concentration than the usual concentration I had to add double the volume .\", {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated . Add 5000 ml of water and acetone and mix for 10 min and then distil .', {'entities': []}) \n",
      "\n",
      "('moment potential around - 60 .', {'entities': []}) \n",
      "\n",
      "('D2 - 1 09211 .', {'entities': []}) \n",
      "\n",
      "(\"'m adding 6 ml of penicillin and streptomycin to the new hFF medium now .\", {'entities': []}) \n",
      "\n",
      "(\"number 3 looks super weird . All the fat created a layer on the top of the tube after the centrifugation . And there is a tiny  tiny pellet . On the bottom . Let 's collect everything .\", {'entities': []}) \n",
      "\n",
      "('really difficult to track what almost up to date version is as well . For some studies for example in the necropsy it can take 2 days . 1 study can take almost a month . As the animals need to be here for at least a week before we can do start on anything .', {'entities': []}) \n",
      "\n",
      "('got colonies of MTSC with his - tag and PSTP . And we do colony PCR with the primer pair 6 and 4 to detect the insert . And we are expecting a product of 2000 base pair . And for a positive control we use primer pair 5 and 4 to detect YFP . And if YFP is still inside the vector we should achieve a product of 800 base pair . The annealing temperature for the base pairs  um  primer pairs 6 and 4 is 46 ° and for the primer pair 5 and 4 is 53 ° C .', {'entities': []}) \n",
      "\n",
      "('again the sample labels . So I have sample labels that that consist of a number and letters . Number 3 in this case represents the third replicate . First G in the sample label is referring to foregut  so G1 . G2 is hindgut and G alone is the whole gut . Then  F stands for fat body  H for hemocytes . Apart from that we have a second letter that labels the strain of the mosquitos . G is for G3 . RKG is labeled with Besides that we have written below : SF that stands for sugar fat  BF for blood fat and PF for Plasmodium infected mosquitos .', {'entities': []}) \n",
      "\n",
      "('cells show a confluence of 70 % . And can be used for further experiments .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('lot number 9112011 . Pre - coated PAMPA plates .', {'entities': []}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('AMP mastermix 126 µl dpx 1', {'entities': []}) \n",
      "\n",
      "('1 was run with a latter of 100 base pairs  gel 2 was run with a latter with 50 base pairs .', {'entities': []}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('000 . Leaky seal . Cell dying at 4 Hertz .', {'entities': []}) \n",
      "\n",
      "('temperature 37 .', {'entities': []}) \n",
      "\n",
      "('the volume is 500 µl . 500 µl .', {'entities': []}) \n",
      "\n",
      "('total  1434 µl of HEPES buffer was added .', {'entities': []}) \n",
      "\n",
      "('reaction mass into 2 portions .', {'entities': []}) \n",
      "\n",
      "('cabinet on right hand side . Top shelf  sex of Finn  sets of FinnTip 1096 well racks . Set of 250 µl universal refill kits 20 * 96 pieces per real refill 4 boxes . Second shelf  more disposable tips  not in use . Bottom  set up disposable tips also not use and', {'entities': []}) \n",
      "\n",
      "('eluting from Strep - Tactin column for experiment 67 with 3 ml of Apo elution buffer . Allow it to do that by gravity flow .', {'entities': []}) \n",
      "\n",
      "('7 4 µl Albamine And lane 9 . 4 µl of the cynomologous Serum albumine .', {'entities': []}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('2 µg of Cas9 protein and 2 µg of modified Cas9 protein will be digested with trypsin overnight . The proteins will not be reduced and activated prior to trypsinization .', {'entities': []}) \n",
      "\n",
      "('for experiment 67 . Another and centrifugation 10 min at 400 4000 RPM in Milton fuels 3 both arrays .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('did a colony PCR of the second recombination of BAC 16 gB Delta RGD to test if the Kana cassette is removed .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('incubator for project 15 .', {'entities': []}) \n",
      "\n",
      "('180 µl Evergreen Supermix  3.6 µl forward primer  3.6 µl of reverse primer  82.8 µl of ultrapure water .', {'entities': []}) \n",
      "\n",
      "('am starting RNA extraction now . Yesterday I have homogenized the samples with TRizol . I used 1 ml of TRizol for 3 wells in a 24 well plate .', {'entities': []}) \n",
      "\n",
      "('46 3.259 E5 .', {'entities': []}) \n",
      "\n",
      "('difficult words in the lab . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "('number 3 .', {'entities': []}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('wells total used 10 ml of cells to populate .', {'entities': []}) \n",
      "\n",
      "(\"ml to well 3 a big well - It 's a big well - Oh  it did n't go . Wait . Does n't wanna order me anything let 's say - Avocado .\", {'entities': []}) \n",
      "\n",
      "(\"I did n't mention we have to change the flasks for higher volumes  so we changed from 125 milliliter flasks to 150 milliliter flasks .\", {'entities': []}) \n",
      "\n",
      "('KP59331 viability 67.1 % concentration 1.02  * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('2 days do mini preps .', {'entities': []}) \n",
      "\n",
      "('up include diluted into new Aliquotes per for 15 µl kept on ice as long as possible further activity has to be proved .', {'entities': []}) \n",
      "\n",
      "('11:20 started incubation of all the samples with 2 ml slurry of the Amgen nickel loaded magnetic beads which had been washed twice . The beads had previously been used 4 times were stored in 20 % ethanol . Rotation .', {'entities': []}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('volume is 3.2 ml .', {'entities': []}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a pinch of lysozyme from Roche 837059  Lot 93141822  July 06 .', {'entities': []}) \n",
      "\n",
      "('blotting finished at 6:20 .', {'entities': []}) \n",
      "\n",
      "('should be an option for previous builds you see that  yeah true but then I have 3 3', {'entities': []}) \n",
      "\n",
      "(\"found 2 batches of protein in -80 and I am using the previous Batch . I have n't tested yet the new batch of protein . So for this assay it 's the old batch that I am using\", {'entities': []}) \n",
      "\n",
      "(\"columns strep tactin columns are moved to room temperature and they will be regenerated . Note that on the 66 experiment with Apo it looks like there may be some aggregation at the bottom of the column . I 'll have to carefully remove that .\", {'entities': []}) \n",
      "\n",
      "('standard curve is prepared in a in a pure assay buffer and the Q Standard curve is prepared in 10 % of cell lysate mixed with the cell extraction buffer .', {'entities': []}) \n",
      "\n",
      "('- 1 images ending an 09225 .', {'entities': []}) \n",
      "\n",
      "('Nephric ? ] samples from 245322458 .', {'entities': []}) \n",
      "\n",
      "('µl 100 mM Tris HCL to both lanes .', {'entities': []}) \n",
      "\n",
      "('. Set up a reminder for tomorrow at 2:00 PM to organize test for labtwin .', {'entities': []}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"did yesterday the first recombination of this mScarlet Strep - cassette . In the BAC  um  in the [ inaudible ] Bacmids Gao RRV mScarlet orf 65 and GaoRRV mScarlet orf 65 orf 22 [ . I ca n't see single colonies . Could be the problem the concentration of the kanamycin on LB- plates ? Or is the efficiency so good . So I have to plate bacterias again on fresh plates with kanamycin 15 µg / ml .\", {'entities': []}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('me to check on sample 3 .', {'entities': []}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('2 465 ng / ml .', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('biological replicate should be run with 3 technical replicates and 3 different stage . Plus  the two housekeeping genes', {'entities': []}) \n",
      "\n",
      "('µl of protein at different concentrations was added to plate .', {'entities': []}) \n",
      "\n",
      "('me to give a talk at 9 AM today .', {'entities': []}) \n",
      "\n",
      "('40 mg / ml .', {'entities': []}) \n",
      "\n",
      "('3  1.608 E5 .', {'entities': []}) \n",
      "\n",
      "('thawed my competent cells for 20 seconds .', {'entities': []}) \n",
      "\n",
      "(\". Slide 97 . Ileum . No abnormalities detected . Cecum . No abnormalities detected . Peyer 's Patch . No abnormalities detected .\", {'entities': []}) \n",
      "\n",
      "('ID 18 S00629 sample value 3.705 ng/µl . Sample ID 19 S00460 sample value 2.00 ng/µl . Bullet Sample ID 18 S006286 sample value 16.80 ng/µl .', {'entities': []}) \n",
      "\n",
      "('500 to cell A 1 .', {'entities': []}) \n",
      "\n",
      "('forgot her sheets in the other lab when she went towards GL51 .', {'entities': []}) \n",
      "\n",
      "('19928001 .', {'entities': []}) \n",
      "\n",
      "('aliquot the TCV catalog number 72254  it comes as 5 mg  which gets aliquotted into 50 µl aliquots . Resuspend the 5 mg in 1.6 ml .', {'entities': []}) \n",
      "\n",
      "('3 . Cm = 54 . R = 2.9 . A nice current and transient . Also the second caffeine was also very good . However  the same problem as experienced with increasing series resistance  which may not necessarily be as a result of the volume of the fluid in the bath or because of drifting of the pipette  thereby preventing me from doing isoprenaline experiments . The maximum and minimum . Was a little tricky be cause the pipette was broken at the minimum stage . However  after prolonged perfusion  The minimum was restored back to basal level .', {'entities': []}) \n",
      "\n",
      "(\"count is 1.5 E5  which is in 75 ml of media  which translates to roughly double the expected number of cells per vial . We think it 's the wrong cell count so we continuing as normal  adding 250 ml of media .\", {'entities': []}) \n",
      "\n",
      "('number HU 1642 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('weighed into g of Mgcl 2 . In 200 ml of water . I said 100 ml photo .', {'entities': []}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('11 combination 3 media added .', {'entities': []}) \n",
      "\n",
      "('pellet size : 288 mg .', {'entities': []}) \n",
      "\n",
      "('lowering the aspirate to forge it does not improve the pattern  reversing back to 55 original settings . OK .', {'entities': []}) \n",
      "\n",
      "('test one . Extraction . The biphasic solution was steered at room temperature for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "(\"that samples are incubated at 600 RPM during all the incubation 's in the protocol .\", {'entities': []}) \n",
      "\n",
      "('17  1.75 extend til 5 .', {'entities': []}) \n",
      "\n",
      "('about 150ML increased to 2 %', {'entities': []}) \n",
      "\n",
      "('11 with headphones talking in the background not too loud . Extraction . The biphasic solution was stirred at room temperature for 10 for 20 min and then transferred to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . New paragraph . Test end .', {'entities': []}) \n",
      "\n",
      "('. Create a reminder for tomorrow 9:00 AM to call people from the team in Valenciennes', {'entities': []}) \n",
      "\n",
      "('number one . Step one preparation of reagents . Um  preparation of lysis buffer . Um  step one pipet 65 μl nuclease free water .', {'entities': []}) \n",
      "\n",
      "('902.97 g water and 234 g poloxamer 407 .', {'entities': []}) \n",
      "\n",
      "('an appointment on Wednesday morning at 9 with Mobies .', {'entities': []}) \n",
      "\n",
      "('me to check the cells confluence on Saturday at 9:00 .', {'entities': []}) \n",
      "\n",
      "('4 µl of light citrate .', {'entities': []}) \n",
      "\n",
      "('HCL pH 8 Medicago 12 - 9 .', {'entities': []}) \n",
      "\n",
      "('10 ml 1 molar imidazole to the wash buffer to read 5 mM. And add', {'entities': []}) \n",
      "\n",
      "('inaudible ] 16 A ?', {'entities': []}) \n",
      "\n",
      "('1 viscosity 0.2 . Sample 2 viscosity 2.5 . Sample 3 viscosity 5.6 .', {'entities': []}) \n",
      "\n",
      "('me to call Sarah tomorrow at 2 PM .', {'entities': []}) \n",
      "\n",
      "('jabra English radio in the background . Extraction . The biphasic solution was steered at room temp for 20 min and then transferred into separation funnel the face cut is clear and fast . The clear yellow solution was dried on the rotarvap up at 50 ℃ and 10 mbars to give . Test end', {'entities': []}) \n",
      "\n",
      "('gel activation for 2.5 min on the BioRAD easy imager .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('liver  no', {'entities': []}) \n",
      "\n",
      "('160 µl of cholic acid internal standard .', {'entities': []}) \n",
      "\n",
      "('me to put it in order for CD19 antibody .', {'entities': []}) \n",
      "\n",
      "('in 10 minute countdown .', {'entities': []}) \n",
      "\n",
      "('ml were added to 5 µl of water and afterwards the DNA was about 2 mM.', {'entities': []}) \n",
      "\n",
      "('primers for tubulin they have a melting temperature or 72 ° or higher  so I need to perform a 2 Step PCR reaction . And therefore I will do the reaction for this primers separately from the rest .', {'entities': []}) \n",
      "\n",
      "('delivered polyglycine tyrosine sodium salt 100 mg . P0275- 100 mg stored at - 20 ° C', {'entities': []}) \n",
      "\n",
      "('rigid 61 µl .', {'entities': []}) \n",
      "\n",
      "('this command to experiment number 02055 .', {'entities': []}) \n",
      "\n",
      "(' check fractions E12 to F9 and G5  sorry  G3 to G9 .', {'entities': []}) \n",
      "\n",
      "('media with FBS lot 160316 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('to pre pulse inhibition article in Wikipedia . Background white noise is 30 to 40 decibels in rodent experiment . And the starting noise is 3 to 12 decibels louder than the background noise . I want a loudness . That is uh below starter noise .', {'entities': []}) \n",
      "\n",
      "('use 2 µl of buffer solution', {'entities': []}) \n",
      "\n",
      "('export two types of graphs  1 is steady state affinity this is the concentration against the binding signal and 2 the connect of the binding and how quickly', {'entities': []}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('pellet 245 mg .', {'entities': []}) \n",
      "\n",
      "('of 125 µl are made from the urinary biomarkers .', {'entities': []}) \n",
      "\n",
      "('reminders every 4 h for 24 h.', {'entities': []}) \n",
      "\n",
      "(\"infusion Rehydrex at 10:00 o'clock . Start infusion 125 ml / h 11:00 o'clock .\", {'entities': []}) \n",
      "\n",
      "('density measured was 0.97 .', {'entities': []}) \n",
      "\n",
      "('image from gel using the GelDoc EZ imager stain - free with activation of 1.5 min . Uh . To highlight intense bands  Uh image transformed using high 912 low 0 and gamma 0.56 . And saved .', {'entities': []}) \n",
      "\n",
      "('could do that will be nozzle . 5 mg / ml .', {'entities': []}) \n",
      "\n",
      "('for small patches . 3 experimental groups medium .9 % sodium chloride and [ ? ] acetate', {'entities': []}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('counts . G3 has 25 oocytes . Following line 36 oocytes . Another line 27 oocytes . Another line 154 oocytes .', {'entities': []}) \n",
      "\n",
      "('47 1.222 E5 .', {'entities': []}) \n",
      "\n",
      "('maybe spend 20 % of my time on this depending on the assay .', {'entities': []}) \n",
      "\n",
      "('1200 µl of 20 mM BTP pH 7.5 with 1.25 % of Koji to the 200 µl of washed membranes .', {'entities': []}) \n",
      "\n",
      "('have 3 good organoids and 20 plates of human podocytes .', {'entities': []}) \n",
      "\n",
      "('filled up the water jackets of the Organ Bath and they are all now at 37 ℃ .', {'entities': []}) \n",
      "\n",
      "('V4RF5 membranes . 150819 SK membrane prep 5', {'entities': []}) \n",
      "\n",
      "('of HEPES buffer was added to 1 ml of water to create a final concentration of 50 mM.', {'entities': []}) \n",
      "\n",
      "(\" dilute the processed pool 20 fold in small scale 1st to test if the protein does n't precipitate at low salt concentration .\", {'entities': []}) \n",
      "\n",
      "('26 338000', {'entities': []}) \n",
      "\n",
      "('me to buy new book tomorrow at 6:00 PM .', {'entities': []}) \n",
      "\n",
      "('me to clean up room 10 for our colleagues and do the photos for the research wiki .', {'entities': []}) \n",
      "\n",
      "('the size exclusion pull fractions B11 to C6 .', {'entities': []}) \n",
      "\n",
      "('ml water were added to the white suspension .', {'entities': []}) \n",
      "\n",
      "('me to check on sample # 3 .', {'entities': []}) \n",
      "\n",
      "('ID 18S006290 . Value 3.705 ng/µl . Pineapple .', {'entities': []}) \n",
      "\n",
      "('reason why the caff in the leak experiment did not really work  is because the tube was disconnected from the profusion itself . so there was no flow thereby not allowing the caffeine to have it effect on itself . So therefore you have to use the caffeine from previous experiments to determine the SR calcium leak .', {'entities': []}) \n",
      "\n",
      "('initiating Cas9 experiment .', {'entities': []}) \n",
      "\n",
      "('me after 5 min .', {'entities': []}) \n",
      "\n",
      "('me to take the bacillus out today at 3 PM .', {'entities': []}) \n",
      "\n",
      "('current plate map CDK1 samples are in row B and CDK2 samples are in row Followed by 3 wells of substrate controls . Bumblebee .', {'entities': []}) \n",
      "\n",
      "('pericardium . 10:46 .', {'entities': []}) \n",
      "\n",
      "('up syringes with 20 % ethanol .', {'entities': []}) \n",
      "\n",
      "('cell plates out of incubator now . Set timer for 1 min .', {'entities': []}) \n",
      "\n",
      "('46 1.550 E5 .', {'entities': []}) \n",
      "\n",
      "('ID 19 S 000460 sample value 2.200 ng/µl .', {'entities': []}) \n",
      "\n",
      "('of 19 and 20 tubes was more difficult than the other tubes . Might indicate aggregation .', {'entities': []}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('with concentration with fraction C3  it starts to go slower doing runs of ten min now .', {'entities': []}) \n",
      "\n",
      "('28  2019 .', {'entities': []}) \n",
      "\n",
      "('F3 until F11  Uh  for the co - expressed purification SEC to get the unprocessed form . Repeat F3 to F11 .', {'entities': []}) \n",
      "\n",
      "(\"I started looking at the left side  then I skipped right one  and right 2 . I 'm now taking an image of our 3 .\", {'entities': []}) \n",
      "\n",
      "('to the previous note . Add only 20 µl of 1 molar TCEP so that final concentration will be 100 mM TCEP .', {'entities': []}) \n",
      "\n",
      "('equals 1.948E5 .', {'entities': []}) \n",
      "\n",
      "('5 ml of DMA substance .', {'entities': []}) \n",
      "\n",
      "('3 . Cm = 81 . Very nice recording with nice current and transient . Isoprenaline was attempted  although the effect of CAFF / ISO was delayed because of a bubble that stopped the perfusion but after multiple attempts CAFF / ISO was done . However  because of a bubble  the response was very slow so I had to repeat it with caffeine without Isoprenaline .', {'entities': []}) \n",
      "\n",
      "('cells will stimulated for 3 h with 0.5 percent FCS under flow conditions for 6 h.', {'entities': []}) \n",
      "\n",
      "('millliter Marcain .', {'entities': []}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"add 3 µl 3.1 µl reducing agents from new - page lot number 1941681 . I 'll heat all the SDS PAGE samples 5 min at 65 ° in the heating block .\", {'entities': []}) \n",
      "\n",
      "('. I have to make sure that', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('test number 2 with having the microphone closer to the mouth . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath to at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('32 1.709 E5 .', {'entities': []}) \n",
      "\n",
      "('appointment for Thursday at 12 PM . I mean am f * * *', {'entities': []}) \n",
      "\n",
      "('me to take the acinetobacter out today at 12:30 .', {'entities': []}) \n",
      "\n",
      "('up protocol 3 for organoids .', {'entities': []}) \n",
      "\n",
      "('buffer was made with 50 µl of HEPES  10 µl of pluronic  100 µl of magnesium chloride and 20 per µl of TCEP .', {'entities': []}) \n",
      "\n",
      "('remind me to clean the [ ? ] tomorrow at 10:00 AM .', {'entities': []}) \n",
      "\n",
      "('next week  meet Evgeni at 12:15 in front of Amrite Indian Restaurant in [ inaudible ] to talk about open science .', {'entities': []}) \n",
      "\n",
      "('1 3.527x10 to the 5 .', {'entities': []}) \n",
      "\n",
      "('is at 1 mg / ml . I will use .', {'entities': []}) \n",
      "\n",
      "(\"'s try another note . What happens if I tell you I used 100 mm NAOH ? Also  can you do citric acid ?\", {'entities': []}) \n",
      "\n",
      "('polylysine 6 well plates .', {'entities': []}) \n",
      "\n",
      "('1 to 10 .', {'entities': []}) \n",
      "\n",
      "(\"'m filling up the organ baths now with 25 ml of Tyrode .\", {'entities': []}) \n",
      "\n",
      "('19  1.727x10 to the 5', {'entities': []}) \n",
      "\n",
      "('with difficult words in the writing zone  no noise . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('aspirate options from Z from 55 to 50 .', {'entities': []}) \n",
      "\n",
      "('description . Homologation for the customer . Please perform test according TL260 on parts processed on the customer .', {'entities': []}) \n",
      "\n",
      "('buffer 20 mM Tris pH 7.5  200 mM sodium chloride  0.5 mM TCEP  10 % Glycerol and protease inhibitor .', {'entities': []}) \n",
      "\n",
      "('forgot to pipette the volume 5 µl of sample one .', {'entities': []}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('it works . Plan for today evening . No  plan for today evening . First  do the calibration experiment  then do another test experiment with a raised sampling rate up to Hertz .', {'entities': []}) \n",
      "\n",
      "('28 2 . 0308 E5 .', {'entities': []}) \n",
      "\n",
      "('for example  the time points will be 0 h  6 h  15 h  24 h  36 and 75 h.', {'entities': []}) \n",
      "\n",
      "('and shake samples to mix  leave on the bench for 3 min and then set up a gel .', {'entities': []}) \n",
      "\n",
      "(\"the ' aim for ' immobilization method : Flow cell 2 : 4000 RU Flow cell 3 : 8000 RU Flow Cell 4 : 16000 RU\", {'entities': []}) \n",
      "\n",
      "('amount in samples in 79  80  78 and 77 and 76 .', {'entities': []}) \n",
      "\n",
      "('if I have any CHLA 79 in the liquid nitrogen .', {'entities': []}) \n",
      "\n",
      "('27  2.337 E5 .', {'entities': []}) \n",
      "\n",
      "('the BMP 4 box and the Activin A box .', {'entities': []}) \n",
      "\n",
      "('me on the centrifuge in 10 min . Avocado . The centrifuge .', {'entities': []}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('urine samples are now transferred from the HBS freezer to room temperature and the samples will be thawed at room temperature for about 30 min . After that 50 µl of water will be added to each sample containing 50 µl of and each sample will be mixed . Meanwhile  the samples are thawing I will re - arrange the sample list and make sure that the identity of our tubes are matched with the sample lists and unique lab ID number .', {'entities': []}) \n",
      "\n",
      "('632 because the de - lamination and he and adhesion of superior at specification requirements . After 5 cycles no surface changes  including cut edges in wells . ° of rusting DIN dash EN dash O60 dash 4628 dash 3 dash RI0 . After 30 cycles .', {'entities': []}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('2 G has a concentration of 187.1 ng/µl . The fraction of 260 by 280 is 1.98 . The fraction of 260 by 230 is 2.33 .', {'entities': []}) \n",
      "\n",
      "('22 89000 .', {'entities': []}) \n",
      "\n",
      "('test difficult words in writing zone  no noise  microphone close to the mouth . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('tomorrow . Go there early . Redo the calibration  and the test experiment with the high [ frequency 00:00:13 ] test patch cables . Clean up a little bit . Prepare for come out to me  and then recording . Take a break . No  I can not do recording because I have the course  and sports . So  just come out to me  and then the recording either in the night  or in Wednesday  Wednesday morning is meeting . Also  I have to prepare the slides .', {'entities': []}) \n",
      "\n",
      "(\"is wrong with the dilutions of the Q standards . The sample volume in Q2 do n't add up to what it 's supposed to be .\", {'entities': []}) \n",
      "\n",
      "('19 208000 .', {'entities': []}) \n",
      "\n",
      "('µl of Metamizol to 1 ml of NaCl .', {'entities': []}) \n",
      "\n",
      "('sure plate 11 is a Max plate . Use only plates 13  14  15 for um the assay treatment for the drug treatment with compounds .', {'entities': []}) \n",
      "\n",
      "('50 ml TBME were added to the white suspension and then filtered off .', {'entities': []}) \n",
      "\n",
      "('tube 3A3 into equal portions of 27 ml . To 1 portion add Tris pH 8 to 20 mM.', {'entities': []}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('109537345 has been dropped discard that from the analysis .', {'entities': []}) \n",
      "\n",
      "('with Chabra . English background talking very loud . New extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('count from 3 flasks is 8.8 times 10 to the 6 per ml  giving 88 times 10 to the 6 total .', {'entities': []}) \n",
      "\n",
      "('is Thursday  September 6 .', {'entities': []}) \n",
      "\n",
      "('have policies in place where we encourage people to start that write up within 1 week of the start of their experiment . As soon as the analysis is complete  they need to have the write up done within 2 weeks .', {'entities': []}) \n",
      "\n",
      "('2 . Cm = R=3.4 . L - type Ca current and transient as well as SR calcium content was recorded . The current and transient were very good  but the cause of a bubble isoprenaline effect was not done .', {'entities': []}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('just took 10 samples of HFF cells passage number 20 + 1 from box 3 of the -152 ° freezer .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('QPCR Primers for F1 . Check integrase', {'entities': []}) \n",
      "\n",
      "('am adding NBS . To solution Of imidazole . The reaction is stirred at 0 ° .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "('330  neural - D1 . Two more pancreas . IgG slide section is completely clean  no staining whatsoever .', {'entities': []}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('add 10 µl of template to the appropriate wells of the reaction plate . Store plate at room temperature or on ice .', {'entities': []}) \n",
      "\n",
      "(\"did n't understand that . EP0023 Started DNA measurement .\", {'entities': []}) \n",
      "\n",
      "('weight is 352', {'entities': []}) \n",
      "\n",
      "('10 ml of media for the acoustic mass spec of stock solution . Sorry . Correction .', {'entities': []}) \n",
      "\n",
      "('cas9 standard prepared in assay diluent is added to row 1 and row 2 .', {'entities': []}) \n",
      "\n",
      "('321 is recorded at 23 ° at 09:58 AM .', {'entities': []}) \n",
      "\n",
      "('wash superose columns with a program with water subsequently with .', {'entities': []}) \n",
      "\n",
      "('lidocaine start 14:15 .', {'entities': []}) \n",
      "\n",
      "('me at 8 PM to check antibodies for tomorrow .', {'entities': []}) \n",
      "\n",
      "('new strep avidin bio core chip onto the bio core 8 K .', {'entities': []}) \n",
      "\n",
      "('whether for the lowering down the aspirate to 40 would improve the pattern .', {'entities': []}) \n",
      "\n",
      "('is 12.3 ng/µl .', {'entities': []}) \n",
      "\n",
      "('40 1.145 E5 .', {'entities': []}) \n",
      "\n",
      "('start  15:35 .', {'entities': []}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('that I have book a hood for tomorrow 14th at 10:30 .', {'entities': []}) \n",
      "\n",
      "('opens at 8:00 AM in the congress .', {'entities': []}) \n",
      "\n",
      "(\"bioRAD TGX stainfree gel . 350 folds for 11 min in 1 time tris glycine SDS buffer . Forgot to remove the strip at the bottom . Uh Sample . It 's marker in lane 1  sample 2 is input  sample 3 is the flow through  and then it 's the fractions from the gradient . The last 1 did n't fit on the gel  so F2 is not included in this run .\", {'entities': []}) \n",
      "\n",
      "('10:15 material was loaded . Switch to washing . Washing at 5ML / min  initially at 0 %', {'entities': []}) \n",
      "\n",
      "('reaction mixture to Lynn Slash Water 121 was directly used in knew experiment . The biphasic system was cooled down to 50 ℃ and sodium 32 % was added in one shot . Big orange junk crashed out . To dissolve the chunk additional methanol was added  and stirred the mixture at 6 h for fifth at 50 ℃ .', {'entities': []}) \n",
      "\n",
      "(' a 50 % leads to 79 decibel sound pressure level .', {'entities': []}) \n",
      "\n",
      "('me to sign up for the flow cytometer . Sunday 24th of March of 2019 .', {'entities': []}) \n",
      "\n",
      "('1 control media added .', {'entities': []}) \n",
      "\n",
      "('weight is 9.2 ng .', {'entities': []}) \n",
      "\n",
      "('assay started at 15:21 PM .', {'entities': []}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('. Slide 98 . Heart . Good section of . Is sending a order ? Within normal limits .', {'entities': []}) \n",
      "\n",
      "('gel is 4 to 15 % a 12 well comb with 20 µl / gel . Category of a catalog number 456 - 8085 .', {'entities': []}) \n",
      "\n",
      "('Culture . Yesterday I expanded rhesus fibroblasts to 40 times T175 flasks plus one T75 flask .', {'entities': []}) \n",
      "\n",
      "('is a correction of the last sentence . The water was stored in the freezer for 10 min to make sure samples mixed with water were still ice cold to avoid elevated temperatures during sample handling .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely .', {'entities': []}) \n",
      "\n",
      "('round 55 full length 37  mosquitoes .', {'entities': []}) \n",
      "\n",
      "('is set in place . A quarter past 9 .', {'entities': []}) \n",
      "\n",
      "('need to buy more 24-well plate for cell culture .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 21 to 22 ℃ .', {'entities': []}) \n",
      "\n",
      "('24-well table .', {'entities': []}) \n",
      "\n",
      "('have to suck 3 mice and process 4 mice today . Let me step out . Sometimes  it may help if you  if you wait like  if you say Avocado than wait for two seconds and than say Order', {'entities': []}) \n",
      "\n",
      "(': Petri dishes  LB medium 2 boxes  20 mL syringes 3 boxes  20 mL syringes 10 boxes .', {'entities': []}) \n",
      "\n",
      "('me to call Sarah at 5 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('TOP agar add a final concentration of 5 mM of calcium chloride and 1 mM of magnesium sulfate .', {'entities': []}) \n",
      "\n",
      "('30000 G for 30 min . The supernatant is incubated with 1 ml amylose resin at 4 ° overnight .', {'entities': []}) \n",
      "\n",
      "('have to order new single use filter units 0.45 µm .', {'entities': []}) \n",
      "\n",
      "(' the pin connection is as follows . Printer TXD 2 is connected to pin 3  printer RXD 2 . Is connected to ? Pin number 8 .', {'entities': []}) \n",
      "\n",
      "(\"54  position H12 . There 's a small crystal .\", {'entities': []}) \n",
      "\n",
      "('is 8.783 e4 .', {'entities': []}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('3.96 µl of denatured DNA to hyb buffer .', {'entities': []}) \n",
      "\n",
      "(\"one . Value it does n't like it 94.4  but it does n't now do something .\", {'entities': []}) \n",
      "\n",
      "('ml of water were added to the right suspension .', {'entities': []}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('new plate type has shown promising results . Both CDK1 and CDK2 shows viability for longer up to 8000 seconds .', {'entities': []}) \n",
      "\n",
      "('. Column 1 .', {'entities': []}) \n",
      "\n",
      "('me to check the cells  tomorrow morning at 9:00 .', {'entities': []}) \n",
      "\n",
      "('HFFs are in the growing medium with includes . I have to change it too .', {'entities': []}) \n",
      "\n",
      "('measurement is buffer only . 160 150 seconds . After 10 seconds flash . After 10 % .', {'entities': []}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "(\"'s experiment HU 1650 .\", {'entities': []}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "(': 20000 cells per ml . Three controls . Five patients . Prepare RNA post 24 hours incubation .', {'entities': []}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('speed to 300 rpm .', {'entities': []}) \n",
      "\n",
      "('D 360 is for published data so this is new and also retested assays .', {'entities': []}) \n",
      "\n",
      "(\"have split neuroblastoma cells . I got around 10000000 cells in total . I prepared 2 24 well plates with 100000 cells per well . Tomorrow  I will start differentiation so I 'm going add retinoic acid at 10 µM.\", {'entities': []}) \n",
      "\n",
      "('me to order 5 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('massspec samples for the massspec samples for the non processed form or the 1 that is not co - expressed using 70 µl of buffer A  then 50 µl of IMAC eluate  and subsequently for oxidized 20 µl water and for reduced 20 µl of 1 molar TCEP .', {'entities': []}) \n",
      "\n",
      "('20 µl of Cas9 solution at 0.1 ug/µl I will add 1 micolitre of the trypsin solution to give a 1 to 20 ratio .', {'entities': []}) \n",
      "\n",
      "('100 % melting point 193 ° centigrade .', {'entities': []}) \n",
      "\n",
      "('aliquotation are the tubes flash frozen in liquid nitrogen . There is a 24th tube with about half of the volume available that will be used next week for testing freeze point and will not be registered in lab guru .', {'entities': []}) \n",
      "\n",
      "('exp . day -2', {'entities': []}) \n",
      "\n",
      "('volume is 23.2', {'entities': []}) \n",
      "\n",
      "('502 row A column 1 .', {'entities': []}) \n",
      "\n",
      "('the channel integration number 2 .', {'entities': []}) \n",
      "\n",
      "('130 µl into lane 3 .', {'entities': []}) \n",
      "\n",
      "('occlusion 10:57 . Alone placed distal to the first diagonal .', {'entities': []}) \n",
      "\n",
      "('if noise in the lab is too loud to detect my voice . Dishwasher  Cryo other noise sources like hoods are running right now . The phone is being hold close to the mouth . Test 3 . Put the phone in my lab coat . See if detection is still good enough . Test 4 . Test 5 . pH is still calibrated And stopped manually now .', {'entities': []}) \n",
      "\n",
      "('the concentration of row 2 and 4 .', {'entities': []}) \n",
      "\n",
      "('further to 35 the aspirate Z offset .', {'entities': []}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('down 2 day 24 h samples for AZ1768 .', {'entities': []}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('27 258000', {'entities': []}) \n",
      "\n",
      "(' it takes 20 min for instrument linked PCs to warm up . Having a facial recognition as a login would be really useful and would avoid contamination .', {'entities': []}) \n",
      "\n",
      "('assays that we do each week vary depending on a few different factors . Number 1 is the number of the cells of the compound submitted and assays .', {'entities': []}) \n",
      "\n",
      "('10 µl water to microplate .', {'entities': []}) \n",
      "\n",
      "('needs to prepare needles for injection . Program 2 . um  [ inaudible ] it is [ inaudible ]', {'entities': []}) \n",
      "\n",
      "('. Slide 99 . Sciatic nerve . Within normal limits .', {'entities': []}) \n",
      "\n",
      "('number 1 is 17.7', {'entities': []}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('the Santa Cruz antibody at a concentration of 1 to 1000 mg', {'entities': []}) \n",
      "\n",
      "('buffer is made up of 47 mL of water .', {'entities': []}) \n",
      "\n",
      "(\"assay has a folder and experiment write up notes . These are dated and 1 can change manually depending on the Assay . They have an Excel that 's embedded with the cell count on volumes as well .\", {'entities': []}) \n",
      "\n",
      "(': Cell count is 4.4 times 10 to the 6 per ml .', {'entities': []}) \n",
      "\n",
      "('of edited cells and immunogenicity of CRISPR - Cas9 .', {'entities': []}) \n",
      "\n",
      "(\"did the cloning for the sample . I 'm scarred to leave and any starlet life act . I used 100 ng vector and 37.59 ng insert . Fixed libation mix . I kept at room temperature for 15 minutes and transformed in stable 3 cells .\", {'entities': []}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('42 221000', {'entities': []}) \n",
      "\n",
      "(\"now at approximately whatever time this notice taken at . Almost 4 o'clock  we gon na try with tlPC in pure toluene without any chloroform .\", {'entities': []}) \n",
      "\n",
      "('a 0\\\\.5 molar sodium hydroxide by eluting 300 ml 5 molar to 3', {'entities': []}) \n",
      "\n",
      "('12 5.540 E4 .', {'entities': []}) \n",
      "\n",
      "('36 151000', {'entities': []}) \n",
      "\n",
      "('1 immobilization . 3 µg / ml protein batch 19 dash 0884 .', {'entities': []}) \n",
      "\n",
      "('7 2.220 E5 .', {'entities': []}) \n",
      "\n",
      "('eluted with 3 CV then Nanochip was checked of high', {'entities': []}) \n",
      "\n",
      "('perform tests according TL260 on parts processed at VW .', {'entities': []}) \n",
      "\n",
      "('me to check on sample # 3 .', {'entities': []}) \n",
      "\n",
      "('boxes P30 tips . Next . 9 boxes P10 tips . Next . 8 boxes 96 tips . Next . 1 Box 96 tips . Next . 2 boxes 384 well white plates .', {'entities': []}) \n",
      "\n",
      "('white suspension was filtered off and dried in the vacco oven for 2 h at 50 ° and 2 mbar .', {'entities': []}) \n",
      "\n",
      "('test 9  no headphones  very loud lab associate in the lab . extraction . The biphasic solution was stirred at room temp for 20 min and then transferred in a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the roto bath 50 ℃ and 10 mbar to give- elution - Test end .', {'entities': []}) \n",
      "\n",
      "('would like to use 2 µl of culture medium .', {'entities': []}) \n",
      "\n",
      "('37 214000', {'entities': []}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('µl X AMP to lane 1 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('opens at 8:00 AM .', {'entities': []}) \n",
      "\n",
      "('me to take the samples out of this incubator tomorrow at 9 AM .', {'entities': []}) \n",
      "\n",
      "('25 . No Wait . Sample 26 1.350 E5 .', {'entities': []}) \n",
      "\n",
      "('a meeting with Deepma on Wednesday at 2:00 PM .', {'entities': []}) \n",
      "\n",
      "('1.16 g of sodium chloride to 1 l to get a 20 mM sodium chloride solution . The same same batch of sodium chloride is used for the B buffer .', {'entities': []}) \n",
      "\n",
      "('phase of experiments will be started today . As I attempt to use Interleukin 1 beta . For acute stimulation of human atRA myocytes . Bad perfusion system .', {'entities': []}) \n",
      "\n",
      "('will do the mini prep of the bacmids mNeon orf65 orf39 mScarlet and mScarlet orf65 orf39 mNeon . And I will digest the DNA over the day and will do the pulse field in the evening .', {'entities': []}) \n",
      "\n",
      "('BioRad running buffer Tris Glycine SDD buffer prepared 191022 by AS .', {'entities': []}) \n",
      "\n",
      "('you spell 5F URD ? OK  that worked pretty well how about 2 CLADO . OK  um .', {'entities': []}) \n",
      "\n",
      "('out 14.2 g of ammonium formate .', {'entities': []}) \n",
      "\n",
      "('the trypsin solution will actually be diluted to 0.1 ug/µl instead of 1 ug/µl .', {'entities': []}) \n",
      "\n",
      "('that in the Word document for the method the 1 mag and 2 mag have .', {'entities': []}) \n",
      "\n",
      "('1 9.553 E4 .', {'entities': []}) \n",
      "\n",
      "('a list of the following 3 donors .', {'entities': []}) \n",
      "\n",
      "('me to check on sample # 3 .', {'entities': []}) \n",
      "\n",
      "('official run with plate seeded on the 27th of August . Starting with plate 1 . Change the cell wash speed to 1 and the aspirate offset is at 35 . Will be checking for cell loss .', {'entities': []}) \n",
      "\n",
      "('auto centrifugation of 1 h and analyze the 2nd IMAC on massspec .', {'entities': []}) \n",
      "\n",
      "('chest 11:15 .', {'entities': []}) \n",
      "\n",
      "('phase of experiments will be started today . With the Interleukin 1 beta . Have a concentration of 5 ng / ml  will be used to stimulate HL myocytes isolated today . The sequence of perfusion solutions are Tyrode  followed by 4AP  caffeine  4AP 5 mM calcium  4AP with Interleukin  then zero Sodium zero calcium solution  Tetracaine and zero Sodium zero calcium caffeine .', {'entities': []}) \n",
      "\n",
      "('257 ml of media .', {'entities': []}) \n",
      "\n",
      "('3 has error rate of 0.5 % . Bumblebee .', {'entities': []}) \n",
      "\n",
      "('empty . Empty .', {'entities': []}) \n",
      "\n",
      "('then I take the mean of both . It is appropriately 255 pixels per millimeter . Yes  and measure one millimeter and the program will give you the pixels .', {'entities': []}) \n",
      "\n",
      "('images ending on 093424 chip D4- 2 .', {'entities': []}) \n",
      "\n",
      "('of November 2019  Study number BSOO2541 - 19  Experiment BE002541 - 28  Pig purple # 4 . MI surgery .', {'entities': []}) \n",
      "\n",
      "('300µL of Splash lipidomix heavy label mix . Remind me to buy octanoyl - L - carnitine and new batch of SPLASH Lipidomix mix', {'entities': []}) \n",
      "\n",
      "('me tomorrow to buy 20 plates for experiment 7717 .', {'entities': []}) \n",
      "\n",
      "('fractions were run on SDS PAGE but the protein concentration was too low so it was quite clean . Then it is decided to collect fractions B1 to C5 .', {'entities': []}) \n",
      "\n",
      "('albumin samples are flash frozen in liquid nitrogen . And I store them in SC343 Minus 80 freezer rack 2 B4 .', {'entities': []}) \n",
      "\n",
      "('reverse transcription pipet 5.2 μls master mix to know reverse transkriptase control tube .', {'entities': []}) \n",
      "\n",
      "('thawed my competent cells for 20 seconds .', {'entities': []}) \n",
      "\n",
      "('sample 1 is added to well E3 and E4 instead of A5 and A6 and then these samples series continued as usual .', {'entities': []}) \n",
      "\n",
      "('the well spinfection 2 h in the plate . Without polybrene I have round about 30 % green cells .', {'entities': []}) \n",
      "\n",
      "('24 and 25  unable to count . Tried 3 times for each .', {'entities': []}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('need to raise the pH on experiment 321 .', {'entities': []}) \n",
      "\n",
      "('round 55 single chain fab 32 mosquitoes .', {'entities': []}) \n",
      "\n",
      "('harvest the pseudo types and the lentivirus  spin it down and filtrate it with 0.45 filter sterile and freeze it .', {'entities': []}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('started with 100 ml uh bacterial suspension . And uh I need to compare the concentration of DNA I will get at the end . With 50 ml bacterial suspension .', {'entities': []}) \n",
      "\n",
      "('rigid 51 µl .', {'entities': []}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('me to take the plate out at 1:00 PM . Blueberry', {'entities': []}) \n",
      "\n",
      "('these notes to experiment 33 .', {'entities': []}) \n",
      "\n",
      "('am starting cell preparation for Facs . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': []}) \n",
      "\n",
      "('a note to get dinner at 7:00 .', {'entities': []}) \n",
      "\n",
      "('D2 - 1 fluid flow from bottom to top of picture . Minor cell wrapping around Potes . Minor cell wrapping around entry channels .', {'entities': []}) \n",
      "\n",
      "('µl pour 12 into lane 1 .', {'entities': []}) \n",
      "\n",
      "('the well spinfection for 6 h as an pellet with polybrene I can see some green cells . I would say round about 1 % and a lot of cells are dead . I would say half of the well is dead . In the well without polybrene I can not see green cells .', {'entities': []}) \n",
      "\n",
      "(\"number protocol number for phospho - RAD is # 146 . These samples were sectioned yesterday . This includes the control  therefore on this run I 'll be doing 29 phospho - RAD samples and then one control sample .\", {'entities': []}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('timer for 5 min . Pineapple .', {'entities': []}) \n",
      "\n",
      "('2 has cyclin B1 at 1 in 100 . Tube 3 has cycline B1 at 1 in 500 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "('white suspension was filtered off and dried in the vacuum oven at 40 ℃ .', {'entities': []}) \n",
      "\n",
      "('24 7.66x10 to the 4 . Sample repeated blocks .', {'entities': []}) \n",
      "\n",
      "('tell me in 10 min after 10 min to wash the dishes . Timer 10 min .', {'entities': []}) \n",
      "\n",
      "('1 . Capacitance 108  series resistance was be low  was 25 and percentage compensation was done . Very nice current with very nice transient initially .', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('in 30 min to 100 % Uh  timer set for 35 min  so that I have 25 ml for 5 min of 100 % buffer', {'entities': []}) \n",
      "\n",
      "('occlusion 10:24 .', {'entities': []}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('continue with the notes . Most of the cells from A1 to A12 looked OK . The 2 first  more living cells  the rest are like 50/50 in dead and living cells .', {'entities': []}) \n",
      "\n",
      "(\"switch to experiment DNA research D 237 . Sorry I did n't .\", {'entities': []}) \n",
      "\n",
      "('you invert the centrifugation tubes 3 times  homogenizing the solution and start concentrating again up to now about 1.5 ml has been concentrated in 5 min  so increased concentration length to about 10 min .', {'entities': []}) \n",
      "\n",
      "('gel running at program 2  350-folds  11 min will be developed with the stain free system with a long exposure for long activation time .', {'entities': []}) \n",
      "\n",
      "(\"10:00 o'clock added lysis buffer to all the pellets to a final volume of 40 ml . Leave the pellets on rotation at room temperature until being used .\", {'entities': []}) \n",
      "\n",
      "('nuclear staining with 696  although not as clean as 695 . Add 1 to 100  696 is add 1 to 100 .', {'entities': []}) \n",
      "\n",
      "('is less than 7 .', {'entities': []}) \n",
      "\n",
      "('test number 2 but having the microphone closer to the mouth . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give[?]. Test .', {'entities': []}) \n",
      "\n",
      "('aliquoted 60 µl into the Eppendorf tube . Bumble - bee .', {'entities': []}) \n",
      "\n",
      "('is my first donor .', {'entities': []}) \n",
      "\n",
      "('added 2 teaspoons of sugar to 8 ounces of coffee and stirred for 15 seconds .', {'entities': []}) \n",
      "\n",
      "(' no  I am  but I needed to say it 3 times until it works because we have some background noise here .', {'entities': []}) \n",
      "\n",
      "('unsupported language ] Now we got it . Set a timer for 10 seconds .', {'entities': []}) \n",
      "\n",
      "('of bacteria inactivity by different names experiments for antibiotic treatment . Replicates for children  replicate for colon  123 with colon .', {'entities': []}) \n",
      "\n",
      "('me to enter the imaging data at 4:00 PM . OK . It works  yeah .', {'entities': []}) \n",
      "\n",
      "('sample plate is incubated in room temperature at 600 RPM .', {'entities': []}) \n",
      "\n",
      "('for 30 min .', {'entities': []}) \n",
      "\n",
      "('is a ubiquitously expressed ubiquitin E3 ligase .', {'entities': []}) \n",
      "\n",
      "('samples from the non co - expressed relaxing in water for subsequent loading on the desalting column . These are samples non processed fractions F1 to G6  and processed fractions G7 to A6 as well as from the first purification for the co - expressed 1  a fraction called non processed . F3F4 .', {'entities': []}) \n",
      "\n",
      "('28th of September atropine experiment proof of principle .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('plates out of centrifuge . Transfer 5 µl cell supernatants to the blue reagent using the plate mate beginning now .', {'entities': []}) \n",
      "\n",
      "(\"did a mistake in making initial buffer  I did n't add glycerol . So while making an LAC7 enzyme I realized this mistake and for this enzyme . I have added glycerol .\", {'entities': []}) \n",
      "\n",
      "('was presumably an air bubble in the detector during the application of the sample  which gave high background of UV  so washed at high speed 20 ml / min the detector with buffer B and then subsequently buffer A  then apply gradient to 50 % buffer B in 8 min at 4 ml / min . Collect fractions in 1 ml fractions .', {'entities': []}) \n",
      "\n",
      "('have already very fried . Sorry  I have already verified channel 2 of CERTUS . As the next step I am making assay buffer . Assay buffer has HEPES  bridge 35  BSA  Magnesium chloride  and 5 % Glycerol .', {'entities': []}) \n",
      "\n",
      "(\"UHFF medium . I 'm adding 88 ml of FCS to my DMEM medium .\", {'entities': []}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('is standard . Next ', {'entities': []}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "(' need to dilute to 31.5 total ml / ml  will add 6.5 ml to bring uh solution to 800000 cells per ml .', {'entities': []}) \n",
      "\n",
      "('45 193000', {'entities': []}) \n",
      "\n",
      "('696 with human ovary 1 to 100 in TBST .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ethanol fixation fixed in 70 % ethanol  for PFH fixation fixed in 4 % paraformaldehyde  for heat killing killed at 80 ° centigrade for one and a half h. For P free ethanol fixation was done also for one and a half hours .', {'entities': []}) \n",
      "\n",
      "('me to wake up tomorrow at 8:00 PM 8:00 AM .', {'entities': []}) \n",
      "\n",
      "(\"human material looks yellowish . I pipet 20 µl rat serum albumin in a tube labeled with R  also HPLC tube 20 µl . Also the sample look slightly yellowish . And finally I pipet 20 µl of Cyno serum albumin . For the HPLC I will need to spin it because it 's slightly turbid or filter it . I will filter it through a 0.22 micro diameter tube . For the SDS page samples I add 7.7 µl LDS .\", {'entities': []}) \n",
      "\n",
      "('me to order 20 Mil Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('ID 62 also added to E9 .', {'entities': []}) \n",
      "\n",
      "(' um  D1 to D9 1 ml .', {'entities': []}) \n",
      "\n",
      "('August 9th  10:00 AM organize scope room .', {'entities': []}) \n",
      "\n",
      "('10 µl of all samples .', {'entities': []}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('me to speak to Patrick about using native notifications in combination with our custom notifications for timer alerts tomorrow at 2:00 PM .', {'entities': []}) \n",
      "\n",
      "('test with headphones holding the microphone close to the mouth . Vacuum pump is running . Hoods are running  radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('PCR program for colony PCR Denaturation at 95C. Annealing at 53C for 30 seconds and elongation at 72C for one min and 30 .', {'entities': []}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('the old internal standard 1X in column 23 of a plate . Rows M  O  N  I put new internal 1X internal standard in column 24 rows M  O  N  P again . As previous the 4X internal standard new mix is in column 22 in rows M  O  N ', {'entities': []}) \n",
      "\n",
      "(\"half of the sample 343  got sucked up in the filter . On the second transfer so I think I have around 8 µl on yield or something . That 's just really a pity .\", {'entities': []}) \n",
      "\n",
      "('need to raise the pH on experiment 321 .', {'entities': []}) \n",
      "\n",
      "('25 1.235 E5 .', {'entities': []}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('A280 performance of the different pools tube 15  PL-19 - 1088 .', {'entities': []}) \n",
      "\n",
      "('with Chabra . English talking in the backgrounds Marcus is close to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ° and 10 mbars to give . Test end .', {'entities': []}) \n",
      "\n",
      "('sample 5-fold with buffer That should give me around 4 . 20 * 4 is 80 + 130 is 210 / 5 is 20 mM. Can reduce to 123  which would give me 2 times 20 is 40 + 140 about 40 mM. 1 to 2 dilution with buffer A for the sample of the non processed form .', {'entities': []}) \n",
      "\n",
      "('# 14 is 23', {'entities': []}) \n",
      "\n",
      "('13.5 mg of heavy labeled aspartate in Milli - Q water .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('resuspended cells were incubated for 15 min at RT  600 RPM in an Eppendorf Shaker  and centrifuge at 13000 RPM for 15 min at 4 ℃', {'entities': []}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('the settings of the washer for washing with cells taken out a plate seeded on the 27th of August . Uh with Barcodes Plate Number 5 with bar code 1090951005 .', {'entities': []}) \n",
      "\n",
      "('I use 1 drive for SOP Storage . This is so I can access them on my Phone . I have linked it to the AstraZeneca Network .', {'entities': []}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('ml of TBME were added to the white suspension .', {'entities': []}) \n",
      "\n",
      "('run 15 micro there is liquid between the chip in the spacer .', {'entities': []}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('remind me in 1 min to eat my dinner .', {'entities': []}) \n",
      "\n",
      "('samples . Sample 1 .', {'entities': []}) \n",
      "\n",
      "('to reorganize the menace 80 box .', {'entities': []}) \n",
      "\n",
      "('30 µl EPX2 to X AMP .', {'entities': []}) \n",
      "\n",
      "('26 3.166 E5 .', {'entities': []}) \n",
      "\n",
      "('is latency recording for the Android number 2 . I am in N0 .', {'entities': []}) \n",
      "\n",
      "('D2- 3 has one piece of debris in the entry channels very very very minor cell wrapping around the entry channels . The entry post entry channel Interface is clean . There are two instances of cell debris in the second post array from second  third  fourth  fifth and 6 post arrays there increasing cell wrapping and spider web like cell debris . There is evidence of dimples in the chip and the flow direction is from bottom to top of image .', {'entities': []}) \n",
      "\n",
      "('antibody 10 ML TBST 1 µl antibody . Mouse antibody 15 ML TBST 1.5 µl antibody .', {'entities': []}) \n",
      "\n",
      "('and 0.4456 mg were added into the buffer and the electrophoresis was run at 60', {'entities': []}) \n",
      "\n",
      "(\"11 33 GMP DIF287 has really do n't nanog  6 % nanog . Were as titin DIF288 has dot .\", {'entities': []}) \n",
      "\n",
      "('fade standard prepared from the stock solution in assay diluent is added to column 7 and 8 .', {'entities': []}) \n",
      "\n",
      "('the EZ doc gel system with 5 min activation and detection for faint bands .', {'entities': []}) \n",
      "\n",
      "(\"to 240 volts 50 60 Hertz 0.6 A maximum and output is 12 V and 1500 mA. The plug itself appears to be for the Chinese market only  so we need an adapter  however  it should work  although I 'm not even sure if I 'm allowed to plug this in in Germany . Now considering the disgusting smell  I 'm really not sure whether or not you can buy this sell this actually to a customer .\", {'entities': []}) \n",
      "\n",
      "('with jabra difficult words  normal noise in the lab 4 m distance to phone . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('DNase has been active for 45 min before stopped .', {'entities': []}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('experiment 66 AZ 2250 . The 8 to 80 is 0.14 . And finally .', {'entities': []}) \n",
      "\n",
      "(\" and  yeah  dissection of RKG samples Plasmodium infected . Um  replicate 3 began to 3830 at 30.30 time point  instead of 2078 . Um  besides that  um  gut dissection went ok  shape with 6 guts inside  because for one I was n't sure if I collected all  what even  what ever  5 mosquitoes aspirator for defend the body .\", {'entities': []}) \n",
      "\n",
      "('me to check the iSLK bac 16 wild type and ELAAN mutant', {'entities': []}) \n",
      "\n",
      "('pool is 31 ml . Also pool the remainder of the peak with the processed protein and save for later analysis .', {'entities': []}) \n",
      "\n",
      "('to experiment 08026 .', {'entities': []}) \n",
      "\n",
      "('is it possible to test LPS through biochemical assays such as Massspec talk to Erica Forsberg .', {'entities': []}) \n",
      "\n",
      "('coffee step 1 . Add 2 sachets .', {'entities': []}) \n",
      "\n",
      "('amount of cell lysate in the Q standard curve is 1 to 20 rather than 1 to 10 .', {'entities': []}) \n",
      "\n",
      "('me  remind me to call Sandra on the number 432 .', {'entities': []}) \n",
      "\n",
      "('sample 14  15  13  20  21  22 and 23 for flow  so that as we can fix them .', {'entities': []}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is co - expressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when the IMAC  next IMAC is finished .', {'entities': []}) \n",
      "\n",
      "('really difficult to track what our most up to date version is as well . For some studies for example  in the necropsy  it can take 2 days . 1 study can take almost a month . As the animals need to be here for at least a week before we can do start on anything .', {'entities': []}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"back to original values for the protocol on aspirate at 55 steps and on the dispense that 's 45 steps . I 'm going to check now if reducing the dispense improves the pattern in any way .\", {'entities': []}) \n",
      "\n",
      "('me . Often meeting with Dennis at 1313 .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 21 ° to 22 ° .', {'entities': []}) \n",
      "\n",
      "('[ ? ] I need to investigate the bioreactor and by 5 ml of liquid .', {'entities': []}) \n",
      "\n",
      "('protocol step 5 . Step 5 .', {'entities': []}) \n",
      "\n",
      "('will harvest from each cell line the dilutions 1 in 25 and 1 in 125 .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be con contaminated .', {'entities': []}) \n",
      "\n",
      "('Marker 1 control flexible  0.3 flexible  1.0 flexible . Marker 2 control rigid  0.3 rigid  1.0 rigid .', {'entities': []}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('me to check this Phone at 4:00 PM .', {'entities': []}) \n",
      "\n",
      "(\"order to create a single harmonize assay it 's essential to understand the need of stakeholders . This information was captured in user requirement specification document a summary of this can be seen in table 1 . Ongoing evaluations of automated something handling submission and data optimisation are depicted below .\", {'entities': []}) \n",
      "\n",
      "('15 -1 has lots of residual cells inside of it indicating that the surface of the chip might be sticky  causing the cells to stick . There is two very large pieces of debris in this chip . The entry channels are have major cell wrapping around them . There is lots of cell debris build up there . Channels are filled with cells as we come to the first post array no large buildup and post array 2  3  4  5 there is lots of cell wrapping and cell clogging in between the posts . And array 6 to the exit channels there is again lots of cell wrapping around cell clogging . And it looks like there are still cells in the exit channels leading out into the outlet . The outlet design is not the same as the active chips . There is a slight edge angle coming outwards in the inlet channel there seems to be a coupled divots where the uh shape is imperfect and the image is has a fluid flow going from the bottom to the top of the image .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 25 to 28 ° .', {'entities': []}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('chabra normal noise in the lab . Radio in the background difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg . 17 µl diluted . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('43 1.0.7 E5 .', {'entities': []}) \n",
      "\n",
      "('ID 19S0078 sample value 3.3.75 ng/µl . Sample ID 19S0058 sample value 2.78 ng/µl . Sample value 3.5 ng/µl sample ID 18S0054 .', {'entities': []}) \n",
      "\n",
      "(\"first assay I found later it was done with the wrong excitation settings . So I stop assay after about 800 seconds and now I 'm repeating assay with correct settings .\", {'entities': []}) \n",
      "\n",
      "('is 5 plus 5 ?', {'entities': []}) \n",
      "\n",
      "('added 3 µl of protein to 300 µl of buffer diluted 1 into 2 times .', {'entities': []}) \n",
      "\n",
      "('tube with protein solution is 1.208 [ ? ]', {'entities': []}) \n",
      "\n",
      "('is 1.270 E 5 .', {'entities': []}) \n",
      "\n",
      "('for 30 min .', {'entities': []}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('method . PV12103OC requirements VWTL 2600FL dash 663007 dash 2018 after 5 cycles  no surface changes including cut edges and wells .', {'entities': []}) \n",
      "\n",
      "('KP59331 viability 67.1 % concentration 1.02  * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "(': start differentiation of neuroblastoma cells . Dilution of retinoic acid : 1/100 and 1/1000 .', {'entities': []}) \n",
      "\n",
      "('an order for 77 pipettes .', {'entities': []}) \n",
      "\n",
      "('1 column A 500 .', {'entities': []}) \n",
      "\n",
      "('me tomorrow at 10:00 AM that I need to do more Mueller Hinton broth .', {'entities': []}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('Nupage 4 to 12 bis - tris gel 17 lanes and 5 µl of Novex sharp pre - stain standard and apply samples in the order of the fractions from the iMac .', {'entities': []}) \n",
      "\n",
      "('Nupage 4 to 12 bis - tris gel 17 lanes and 5 µl of Novex sharp pre - stain standard and apply samples in the order of the fractions from the iMac .', {'entities': []}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('have taken 20 plates out of the oven .', {'entities': []}) \n",
      "\n",
      "('2016 . Again sample 216 was a mixed culture different cultures and I just took the blue one .', {'entities': []}) \n",
      "\n",
      "('. Samples were all diluted to 5 ng/µl to standardize error rate when doing addition .', {'entities': []}) \n",
      "\n",
      "('error of channel 4 is 2.5 % .', {'entities': []}) \n",
      "\n",
      "('mL of HEPES was added to kinase buffer .', {'entities': []}) \n",
      "\n",
      "('1  1.3 E 5 .', {'entities': []}) \n",
      "\n",
      "('43 8.418 E4 .', {'entities': []}) \n",
      "\n",
      "('NeutrAvidin on CMD 200M.', {'entities': []}) \n",
      "\n",
      "('stimulation voltage around from 20 to 40 and high with 4 msec .', {'entities': []}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('52 µl from the dilute from experiment 67 and analyse it on FSEC . Take the other  the remainder of the material and concentrate that together with all the other samples .', {'entities': []}) \n",
      "\n",
      "('normal headphones microphone close to the mouth extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Dragon fire .', {'entities': []}) \n",
      "\n",
      "('µl .1 N NAOH all 8 lanes .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('of the supernatant  resuspending cells in 100 µl staining solution .', {'entities': []}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "(\"the plates I 've received from compound management I use a barcode reader . There is 1 for the whole lab .\", {'entities': []}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('me to feed 19OLP4 .', {'entities': []}) \n",
      "\n",
      "('3 for organoids .', {'entities': []}) \n",
      "\n",
      "('26.24 Neobi 20.25 g jojoba . 7.45 g coconut oil . 3.6', {'entities': []}) \n",
      "\n",
      "('1 images ending on 09242 .', {'entities': []}) \n",
      "\n",
      "('am running 2 EGF assays together and therefore I am making a common assay buffer .', {'entities': []}) \n",
      "\n",
      "('sample 413 .', {'entities': []}) \n",
      "\n",
      "(\"aim for today is to finish all my sectioning and complete the H and E 's of all 40 samples so that then I can send these to pathologist so he can analyze where the tumor regions are within the brain . This is for the ATM BBB study .\", {'entities': []}) \n",
      "\n",
      "(\"'m measuring sample 2 .\", {'entities': []}) \n",
      "\n",
      "('with workroom pump running in the background  radio running in the background . Hood is running . Extraction . The bay biphasic solution was stirred at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('starter culture 4299 .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('for 2 min .', {'entities': []}) \n",
      "\n",
      "('plates 1 min  1000 RPM  open face .', {'entities': []}) \n",
      "\n",
      "('qPCR was set to run for 40 cycles .', {'entities': []}) \n",
      "\n",
      "('concentration in sample B1 is 5 ng/µl .', {'entities': []}) \n",
      "\n",
      "('for 30 min .', {'entities': []}) \n",
      "\n",
      "('a bioRAD tgx 4 - 15 % gel . Run at program 2  350-folds for 11 min .', {'entities': []}) \n",
      "\n",
      "('D1 is in the oven for 5 min .', {'entities': []}) \n",
      "\n",
      "('µl the Poly EY was added to tube .', {'entities': []}) \n",
      "\n",
      "('cell A2 place', {'entities': []}) \n",
      "\n",
      "('81 and 82 .', {'entities': []}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "(\"today evening  we are looking at the device together . I know  it 's always you have to speak really closely to the microphone . Ok  and then it turns off after 30 seconds .\", {'entities': []}) \n",
      "\n",
      "('with headphones next to the radio  English song is running . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('of the hemolin G3 first mosquito I had to squeeze it a little bit because nothing was coming out .', {'entities': []}) \n",
      "\n",
      "('D2- 3 . uh ? Test .', {'entities': []}) \n",
      "\n",
      "('around 2:49 .', {'entities': []}) \n",
      "\n",
      "('10 ml to vial', {'entities': []}) \n",
      "\n",
      "('me to order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('22 7.635 E 4 .', {'entities': []}) \n",
      "\n",
      "('me to fix my cells in 25 min .', {'entities': []}) \n",
      "\n",
      "('HCL pH 8 Medi [ ? ] 12 - 9 .', {'entities': []}) \n",
      "\n",
      "('with the primary antibody start at 6:30 . And continue overnight .', {'entities': []}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "('. Calculate 12 times 35 .', {'entities': []}) \n",
      "\n",
      "('surgery 9:32 .', {'entities': []}) \n",
      "\n",
      "('1 aliquote of Opsin made in August 2019 with a volume of 500 µl .', {'entities': []}) \n",
      "\n",
      "('to use have aliquot for virus only 0.56 efficiency until thawed .', {'entities': []}) \n",
      "\n",
      "('ID 18S006290 . 3.705 ng/µl .', {'entities': []}) \n",
      "\n",
      "('assay seems to have worked well .', {'entities': []}) \n",
      "\n",
      "('ID 18S00628 sample value 3 . 3.27 ng/µl blueberry .', {'entities': []}) \n",
      "\n",
      "('for MPA supplementation the source plate has failed in well A1 .', {'entities': []}) \n",
      "\n",
      "('multisetup  change the sync pin rate back to .1 Hertz .', {'entities': []}) \n",
      "\n",
      "('[ ? ] Analyzer . Center 490 .', {'entities': []}) \n",
      "\n",
      "('additional sample ID 23 is added as sample # 46 .', {'entities': []}) \n",
      "\n",
      "('35 7.580x10 to the 4 .', {'entities': []}) \n",
      "\n",
      "(\"1 . Capacitance 70 . Series resistance was add  was done  however . Unfortunately  I did n't do percentage serious resistance compensation  which I think might have an effect on the recordings I got . However  only current and transient was recorded with the calcium  L - type calcium current amplitude remaining very consistent  which I do n't really understand why . You might be an artifact . However  the series resistance  I mean  the pipette resistance was 2.5 . Cell appear to be drifting .\", {'entities': []}) \n",
      "\n",
      "('last was PL19127 tube 7 .', {'entities': []}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('BE66AZ2250 the SEC  pool fractions B11 to C5 .', {'entities': []}) \n",
      "\n",
      "('size exclusion to buffer with AZ2250 . Wash the loop with 1 ml of buffer . Then collect the sample and inject it in the loop .', {'entities': []}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('the homogenization buffer  we take 4850 μl of name 2 buffer and add 50 μl of Triton X-100 10 % EP', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to .', {'entities': []}) \n",
      "\n",
      "('after centrifugation the A280 and it was .29 . Filter sample and see if it gives the same OD . Samples look clear after centrifugation .', {'entities': []}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "('200 ml of water to quench reaction .', {'entities': []}) \n",
      "\n",
      "('added Coomassie at 1 in 200 dilution at 1:30 PM .', {'entities': []}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('of the human uncoordinated 5 B E 17 luciferase construct .', {'entities': []}) \n",
      "\n",
      "('pipet 20 µl human serum albumin in tube labelled H and also in a HPLC tube .', {'entities': []}) \n",
      "\n",
      "('number 2199 .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "(\"I 'm using my own stock of HEPES buffer I 'm removing 300 µl from the total buffer to normalize the calculation .\", {'entities': []}) \n",
      "\n",
      "(\"D3- 1 has minor cell wrapping in the later half of the post arrays . The beginning half post arrays really do n't have any cell debris . It is much cleaner than chip D2 - 1 minor cell wrapping around entry channels and two places in the interface of channels in first post array where there is some debris clogging buildup .\", {'entities': []}) \n",
      "\n",
      "('12 . Control media added .', {'entities': []}) \n",
      "\n",
      "('am starting cell preparation for Facs . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': []}) \n",
      "\n",
      "('is very wrong with BV 5 10 .', {'entities': []}) \n",
      "\n",
      "('top Agar add final concentration of', {'entities': []}) \n",
      "\n",
      "('\"', {'entities': []}) \n",
      "\n",
      "(': In a cooled centrifuge spin down the plate for 30 seconds at maximum speed and place immediately on dry ice .', {'entities': []}) \n",
      "\n",
      "('the colony PCR we are picking 10 new colonies .', {'entities': []}) \n",
      "\n",
      "('µl pour 16 into lane 5 .', {'entities': []}) \n",
      "\n",
      "(\" 2  3 and 4 respectively sample 13  15 and 19 . Lane 5 sample 20  but it 's likely that this sample does n't have any protein . The lane 6 sample 21 .\", {'entities': []}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('me to warm cells at 3:00 PM .', {'entities': []}) \n",
      "\n",
      "('is 1.241 E 5 .', {'entities': []}) \n",
      "\n",
      "('2 ml 1 molar TCEP to each of the 2 l of buffers  elution  wash  and SEC . Fill the buffers to 2 l filter the SEC buffer .', {'entities': []}) \n",
      "\n",
      "('length is 2.158 pm .', {'entities': []}) \n",
      "\n",
      "('on for 5 min .', {'entities': []}) \n",
      "\n",
      "('takes me 1 to 3 h to perform an immuno histochemical stain .', {'entities': []}) \n",
      "\n",
      "('5 ml of DMA substance .', {'entities': []}) \n",
      "\n",
      "('me in 2 min to call', {'entities': []}) \n",
      "\n",
      "(\" dilute the processed pool 20 V in small scale 1st to test if the protein does n't precipitate at low salt concentration .\", {'entities': []}) \n",
      "\n",
      "('A  row A 300 . Is this how it works .', {'entities': []}) \n",
      "\n",
      "('photo shows the four plate P1  P2  P3  and P4 . The sample list do not match perfectly the identity on the plates so the plate numbers will be used together with a sample lists when they are delivered .', {'entities': []}) \n",
      "\n",
      "(\"they if the market is large like that way where they will definitely have the solution to like you know . Like when you say Hey Siri  right  it 's just one word . But we wanted to add multiple words because we realized 5  6 people work in the lab . But if we have still have problem we can there are some technologies  they can use . I want to change .\", {'entities': []}) \n",
      "\n",
      "('me to split the cells in 2 minutes .', {'entities': []}) \n",
      "\n",
      "('me to check cells confluence tomorrow at 9:00 AM .', {'entities': []}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('me to lab activity on 23rd of March at 12:00 PM .', {'entities': []}) \n",
      "\n",
      "(\"just go through  click through yourself to all the different windows  cells . Then you will get this output the voltages and what kind of millimeter they are  this is necessary information for all the experiments to do until the next calibration . And you will get the voltages that you should use in order to be approximately around the threshold according to the Vettel paper . And I 'm pretty close . I have to use 1.6 to 9.8 and I 'm just in there in that span .\", {'entities': []}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "(': 25 ml pipettes 8 boxes  pipette tip  Big tips 18 boxes  middle size tips for boxes  small tips 10 boxes .', {'entities': []}) \n",
      "\n",
      "('stopped after 53 min .', {'entities': []}) \n",
      "\n",
      "('a single vial and re - suspending it in 15 mL of media . No centrifugation before the cell count .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(' I remove the PFA from the samples and I will leave the wild type for another 17 min . I also diluted the calcofluor 1 to 1000 to reach 2 ug / mL.', {'entities': []}) \n",
      "\n",
      "('1 3.5  sample 2 3.6  sample 4 3.1 . Done', {'entities': []}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"yesterday  the replacement was n't very successful . But today  I will place another CtO to CtO2 and also watch for the sulfur bond length .\", {'entities': []}) \n",
      "\n",
      "('Cas9 analysis of in vitro samples .', {'entities': []}) \n",
      "\n",
      "('mg pirfenidone', {'entities': []}) \n",
      "\n",
      "(\"now test LabTwin in another sample preparation group where I will use the VX 2500 Multitube vortexer which makes a lot of noise starting now ? How does it work ? When I use this mixer ? It works pretty well still I need to keep pressing I think . You ca n't hear it  but it 's a lot of noise when I talk right now . Pineapple .\", {'entities': []}) \n",
      "\n",
      "('reducing to the aspirate Z offset to 20 steps to check if that eliminates the plate pattern completely .', {'entities': []}) \n",
      "\n",
      "('acinetobacter protocol used today February will be the one from January 31st .', {'entities': []}) \n",
      "\n",
      "('5 9.603 E4 .', {'entities': []}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "(\"of RKG samples Plasmodium infected . Replicate 3 began at 30h time point instead of 27 - 28h . Besides that  gut dissection went ok shape with 6 guts inside because one I was n't sure if I collected all  what even what ever  5 mosquitoes aspirator for defend the body\", {'entities': []}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('Day 3  the cells seem to go into senescence which means that the media conditions are not optimal .', {'entities': []}) \n",
      "\n",
      "('the run pull all the tubes in 20 % ethanol .', {'entities': []}) \n",
      "\n",
      "('infusion Rehydrex  10 ml per kilo .', {'entities': []}) \n",
      "\n",
      "('me to warm cells at 6:00 PM .', {'entities': []}) \n",
      "\n",
      "('µl of BLIMP 1 protein at 1.4 mg / ml was added to 1ml of buffer .', {'entities': []}) \n",
      "\n",
      "('15 - 1 fluid flow going from bottom to top of image . The entry channels are not clogged nor do they have lots of cell wrapping or debris like 15 -1 chip does there is still a prevalence of cells inside the channel . There is a minor uh build up of cells at the beginning of the entry channels . The cell debris at the 1st post interface is very minimal . Rows 2  3  4 and 5 have lots of cell wrapping and have created cell clusters to form a clog in those areas . The exit post exit channel interface has lots of cell debris and clogging going on . There is still a prevalence of cells inside the middle post array and as you go to the exit channel the prevalence of cells goes down but there is still sporadic cells every once in awhile . Again the inlet and outlet is not the same design as the active chip is and there is these weird little divots that come out on one side of the chip when the channels end and the open space begins .', {'entities': []}) \n",
      "\n",
      "('supplementation . Well A1 fails .', {'entities': []}) \n",
      "\n",
      "('do not run [ Fels ? ) 1 . Primers for [ do not work . Blueberry', {'entities': []}) \n",
      "\n",
      "(\"pulse works  but 5V amplitude does not evoke spiking . Also  the histogram tetrode history ... the virtual tetrode did not pick up ... it 's hardly any spikes .\", {'entities': []}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('cells looks really bad after selection with Puro 10', {'entities': []}) \n",
      "\n",
      "('the targeted metabolomics work at Alderley Park  there is 45 µl of plasma in each tube . Not 50 µl as previously estimated and in addition to that I have included a QC pool of approximately 700 µl  of those about 500 µl will make up the 10 replicates of QC samples to run with the batch and the other 200 µl is saved for method developments . So these samples will now be returned to the BioBank . For delivery to Alderley Park and the plate with my samples for the Gothenburg analysis will be stored as well in the local biobank freezer in AC0 and finally analyzed .', {'entities': []}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('4  3.6 E 5 .', {'entities': []}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('take a 1 ml pipette and pipette in 1 ml gently into the bottom of the neck to remove an air bubble .', {'entities': []}) \n",
      "\n",
      "('', {'entities': []}) \n",
      "\n",
      "('0023 . In line . I centrifuge 2 times .', {'entities': []}) \n",
      "\n",
      "('wait for writing the sync file back to 50 milliseconds because  too many occasions  where there was an error . In writing the sync fire .', {'entities': []}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "('timer for 1 h.', {'entities': []}) \n",
      "\n",
      "(\"in cold methanol at 800 rcf for 5 min . It 's very slowly .\", {'entities': []}) \n",
      "\n",
      "('me in 30 min .', {'entities': []}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('7 % ethanol into syringes or pipette with a 1 µl pipette .', {'entities': []}) \n",
      "\n",
      "('make a deviation in Step 3 .', {'entities': []}) \n",
      "\n",
      "('correct for the larger volume add 9 µl of 4 times LDS buffer .', {'entities': []}) \n",
      "\n",
      "('31 214000', {'entities': []}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('the 67 sample the concentration is 0.08  which is very low . This material will anyway be put on the size exclusion . Uh .', {'entities': []}) \n",
      "\n",
      "('ml of MgCl2 was added .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('volume is 3.222 µl .', {'entities': []}) \n",
      "\n",
      "('µl pour 17 into lane 6 .', {'entities': []}) \n",
      "\n",
      "('on gel on gel 1 flowthrough fraction and then 6E5 until 6F2 .', {'entities': []}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('timer 5 min .', {'entities': []}) \n",
      "\n",
      "('50 ml of media . Followed by another addition 50 ml of media . And this will be followed with another 20 .', {'entities': []}) \n",
      "\n",
      "('count the supernatant from the filter centrifugation . It looks a bit thready . There is stuff in it . It is transferred to the tube . Take 50 µl of sample from that for FSEC analysis . Save the centrifuge tube . Then transfer the', {'entities': []}) \n",
      "\n",
      "('30 best in class 3 media added .', {'entities': []}) \n",
      "\n",
      "('10 µl samples starting at E12 and continuing all the way .', {'entities': []}) \n",
      "\n",
      "('solution number 10 50 µl to stock solution number 5 stock solution number 6 79 µl to stock solution number 4 stock solution number 5 78 µl -25 µl were added equally to stock solution number  8 and 9 .', {'entities': []}) \n",
      "\n",
      "('training with Philips  100 trials 80 % correct .', {'entities': []}) \n",
      "\n",
      "('000 . Resistance 3.1 . Capacitance 76 . Cell leaky . Cell leaky . Cell is dying .', {'entities': []}) \n",
      "\n",
      "('summary recordings from 11 March 2019 . Control recordings no pacing . Problems with confusion by 6 . Unless recording 4  no other recordings to be used to verify touch up in or solution changer effect . Only usable as S1 control for pace cells .', {'entities': []}) \n",
      "\n",
      "('3.21 g of TCEP .', {'entities': []}) \n",
      "\n",
      "('1 column is dripping very slowly it the 1 with aggregate at the bottom . Resuspend and aim at removing the aggregate .', {'entities': []}) \n",
      "\n",
      "(\"quotes are . 303 X 1.5 ml . 1 . It 's 1 ml .\", {'entities': []}) \n",
      "\n",
      "('1 is contaminated .', {'entities': []}) \n",
      "\n",
      "('ID 19S000460 sample value 2.600 ng/µl .', {'entities': []}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('tube 383 into equal portions of 27 ml . To 1 portion add Tris pH 8 to 20 mM.', {'entities': []}) \n",
      "\n",
      "('water bath is 37 ° warm and the freezer is 20  -20 ° cold .', {'entities': []}) \n",
      "\n",
      "('sample is missing .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be .', {'entities': []}) \n",
      "\n",
      "('OD in Greiner 96 well plates for the overnight cultures cultures .', {'entities': []}) \n",
      "\n",
      "('3476 cells .', {'entities': []}) \n",
      "\n",
      "('ml water were added to the white suspension .', {'entities': []}) \n",
      "\n",
      "('biorad running buffer Tris Glycine SDS 19 0 412 Blaho who prepared it from 10 times  Tris Glycine SDS buffer bio - rad catalog number 1610772 . Taking marker Precision plus protein unstained standard catalog number 161 - 0363 from bio rad . Applying', {'entities': []}) \n",
      "\n",
      "('3 stopped working at around half past 4 .', {'entities': []}) \n",
      "\n",
      "('30 µl per well using the WellMate dispenser small - bore nozzle .', {'entities': []}) \n",
      "\n",
      "(\"diploidy experiment  we 're going to use 3 cell lines 916 images 2 and 1732 . We gon na incubate them with calcium it for 4 h to arrest them and induce metaphase and then we 're going to instruct him and fix them in 70 % methanol . And  uh  incubating with propidium iodide . We 're going to run them in by fluorescencemetrie\", {'entities': []}) \n",
      "\n",
      "('have cells at a viability of 94.6 percent .', {'entities': []}) \n",
      "\n",
      "('the master mix four 2 tubes of iQ green .', {'entities': []}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('330  neural - D1 . Two more pancreases . IgG section is completely clean  no staining whatsoever .', {'entities': []}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('sodium hydroxide in 5 ml increments until a total volume of 25 ml was reached .', {'entities': []}) \n",
      "\n",
      "('ID . 18 S 006290 . Value 3.705 .', {'entities': []}) \n",
      "\n",
      "('me in 1 min to go to the radioactivity lab .', {'entities': []}) \n",
      "\n",
      "('for 10 min timer for 10 min .', {'entities': []}) \n",
      "\n",
      "('3 . 234 ng/µl ml .', {'entities': []}) \n",
      "\n",
      "('a fluctuation change between the temperature of 24 ° Fahrenheit to 23 ° Fahrenheit .', {'entities': []}) \n",
      "\n",
      "('me to prepare 1 % of the roles jail in 20 min .', {'entities': []}) \n",
      "\n",
      "('µl of pool 20 .', {'entities': []}) \n",
      "\n",
      "('is James Atkinson .', {'entities': []}) \n",
      "\n",
      "(\" I did n't work . It 's registering now by the way . And under the hood you have to somehow approach the hands like this  a lytic here 's the voice you know . Yeah  you 're digesting cardiomyocytes and replating them . In what are you plating them  is it laminin coated  it is matrigel coated ? It 's matrigel . Do you use 1:60 or 1:120 ? I do always 1:60 . 1:60 and you plate them with ROCK inhibitor after digestion ? No . No  you do n't use ROCK inhibitor ? I use it only in the solution when the cells are without  before plating  you know like when you digest the the the cells . When the digestion buffer or after digestion . To block the Accutase . Yeah  to block the Accutase .\", {'entities': []}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "(\"first one we 're trying is the most concentrated at 500 µM and we 're starting with no particles 50 nm and then 200 nm .\", {'entities': []}) \n",
      "\n",
      "('a good preparation . 1 packet brought up in 25 ml water .', {'entities': []}) \n",
      "\n",
      "('I dedicate 1 or 2 h in the afternoon to sort these things out .', {'entities': []}) \n",
      "\n",
      "('105 µl to lane 1 . Add 105 µl of X AMP into lane 1 .', {'entities': []}) \n",
      "\n",
      "('17 pipettes .', {'entities': []}) \n",
      "\n",
      "('me to check on sample number 3 at 4:30 PM .', {'entities': []}) \n",
      "\n",
      "('30 cycles no surface changes including cut edges in wells . Loss of adhesion . Diameter is less than 1.5 mm .', {'entities': []}) \n",
      "\n",
      "('. Slide 98 . Heart . Good section of thoracic aorta . Within normal limits .', {'entities': []}) \n",
      "\n",
      "('serial dilution was made up in a deep well  96 well plate', {'entities': []}) \n",
      "\n",
      "('µl in the tubes 13 to 16 total .', {'entities': []}) \n",
      "\n",
      "('on for 30 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "('cell suspension for the acoustic mass spec assay using the Multidrop Combi with Standard cassette . Dispensing volume 40 µl protocol 384 standard full plate .', {'entities': []}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('concentrating uh using 60 min .', {'entities': []}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('plasma from freezer A1O.', {'entities': []}) \n",
      "\n",
      "('A1 through B6 are my serial dilution of known concentration . Cells .', {'entities': []}) \n",
      "\n",
      "('14  2.603 E5 .', {'entities': []}) \n",
      "\n",
      "('me to check sample 3 .', {'entities': []}) \n",
      "\n",
      "('reperfusion 12:57 .', {'entities': []}) \n",
      "\n",
      "('wash the column with 20 % ethanol . The column is on column valve part number 5 .', {'entities': []}) \n",
      "\n",
      "('treated cells are at 81.8 % viability after fixation which is strange and their concentration is 1.48 . The untreated cells are at concentration of 1.76 .', {'entities': []}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "('HU 1673 .', {'entities': []}) \n",
      "\n",
      "('am adding 300 µl of RL buffer to lyse day 1 cell samples .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('to check the tubes tomorrow at 12:00 AM .', {'entities': []}) \n",
      "\n",
      "('project PMS 6 - 1 QC : No PCR product at all .', {'entities': []}) \n",
      "\n",
      "('50 µl sample for later analysis of each of the two pools . Each of the pools .', {'entities': []}) \n",
      "\n",
      "('jejunum - minimal to mild increase in crypt apoptosis  characterized by 1 to 3 apoptotic bodies within crypts .', {'entities': []}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "('pellet of sample 48 .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "('will digest 40 µl .', {'entities': []}) \n",
      "\n",
      "('have added 778 ml of HEPES buffer .', {'entities': []}) \n",
      "\n",
      "('a 100 . Bye .', {'entities': []}) \n",
      "\n",
      "('analysis  sagA only . Genes collapsing analysis with p less than 0.001 .', {'entities': []}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('D360 is for published data so this is new and also retested assays .', {'entities': []}) \n",
      "\n",
      "('a recent article  Ericsson et al 2018 reported Initial clinical experience with a AZD5718  a novel once daily oral 5 lipoxygenase activating protein inhibitor . They used a method utilizing one ml  purified and pre - concentrated by solid phase extraction  and analyzed by negative ion tandem mass spectrometry with a lower limit of quantification of 2.2 pg / ml LTE4 .', {'entities': []}) \n",
      "\n",
      "('mass spec samples for the mass spec samples for the non processed form or the 1 that is not co - expressed using 70 µl of buffer A  then 50 µl of IMAC eluate  and subsequently for oxidized 20 µl water and for reduced 20 µl of 1 molar TCEP .', {'entities': []}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('? ] spin spinning at 400 for 4 min .', {'entities': []}) \n",
      "\n",
      "(\"4 a very good recording  however  the computer responsible for controlling the TMT switched off allow me to start transient very late and I do n't know if the cell was exhausted . Caffeine was attempted twice . Because I did n't feel the the decay and the amplitude was OK .\", {'entities': []}) \n",
      "\n",
      "('. Starting experiment blah blah 3 .', {'entities': []}) \n",
      "\n",
      "('is 500 next A3 is 800 .', {'entities': []}) \n",
      "\n",
      "('cells at 37 ℃ for 1 h.', {'entities': []}) \n",
      "\n",
      "('temperature 35.5 .', {'entities': []}) \n",
      "\n",
      "('has picked up the FGF antibodies 1 - 12  so he has all the FGF in - house antibodies at his possession .', {'entities': []}) \n",
      "\n",
      "('suggestions to make to . 1 allow adding voice notes to pictures upload it . 2 incorporate a function to plan experiments .', {'entities': []}) \n",
      "\n",
      "('µl Phi X into both lanes .', {'entities': []}) \n",
      "\n",
      "('qPCR protocol used in denaturation 15 seconds instead of 10 and annealing extension 30 seconds instead of 20 . Blueberry', {'entities': []}) \n",
      "\n",
      "('down 2 day 24 h samples for AC1768 .', {'entities': []}) \n",
      "\n",
      "('10 µl of oxidized glutathione', {'entities': []}) \n",
      "\n",
      "('the moment I should get I should get 10 g of NaCl  and add into 2 ml of HCl .', {'entities': []}) \n",
      "\n",
      "('me of a meeting at 6 PM  with Gloth .', {'entities': []}) \n",
      "\n",
      "('12 had the least amount of Protein and you added extra lysis buffer an accident that . So . Watch out for that in the western blot .', {'entities': []}) \n",
      "\n",
      "('acetate buffer 10 mM pH 5.5 .', {'entities': []}) \n",
      "\n",
      "(' it takes 20 min for our instrument linked PCs to warm up . Having a facial recognition as a login would be really useful and would avoid contamination .', {'entities': []}) \n",
      "\n",
      "('25 combination 2 media added .', {'entities': []}) \n",
      "\n",
      "(\"I had 280 μl . It 's far too much in it  and completely wipe a monolayer at minus one . and -2  So what I 'm going to do is I 'm gon na add 28 μl to 28 mL. That 'll be a one in 1000 dilution and so effectively I will be adding -3 -5 -7 . Add to the wells and that should be hopefully should give . Better plaques .\", {'entities': []}) \n",
      "\n",
      "('step 1 . A do gradually 10 min .', {'entities': []}) \n",
      "\n",
      "('me to take my samples out of the fridge tomorrow at 10:00 AM . .', {'entities': []}) \n",
      "\n",
      "(\"works in cell culture and the Biochem Labs . There 's a team of 5 .\", {'entities': []}) \n",
      "\n",
      "('curve is reduced from 10 standards to 8 standards .', {'entities': []}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "('a 100 ml easymax reactor . Add 50 ml of reaction stream A add 2 ml of NN diisopropyl ethylamine . Set temperature to 45 ℃ . Agitate at 600 RPM for 18 h.', {'entities': []}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('me in 5 min incubation finished .', {'entities': []}) \n",
      "\n",
      "('number 1 is 17.7', {'entities': []}) \n",
      "\n",
      "('used 2 µl of buffer solution', {'entities': []}) \n",
      "\n",
      "('used 2 µl of buffer solution', {'entities': []}) \n",
      "\n",
      "('27 g lecithin 90.09 g water .', {'entities': []}) \n",
      "\n",
      "('antibody 1 to 10 ML TBST 1 µl antibody . Goat antibody one to 5 ML 0.5 µl . Rabbit antibody 10 ML 0.25 µl .', {'entities': []}) \n",
      "\n",
      "('took place at 1000 g force .', {'entities': []}) \n",
      "\n",
      "('plates from -80 freezer from the QC dilution experiment .', {'entities': []}) \n",
      "\n",
      "('µl pool 22 into lane 3 .', {'entities': []}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('is 1.2 . Next B1 is cells are contaminated next C1 is protein is as expected next D1 is 10 mol next E1 is 2 nm next R1 is 20 ml next N1 4 CFU next P1 is blue . A2 is 556.23 next . B2 is bacteria are not alive . Next C2 is culture is as expected . Next D2 is mM next E2 is 20 nm next R2 is 1.5 ml next N2 is 3.4 CFU . Next P2 is green . Next A3 is 100 . Next B2 is medium is not fluorescent . Next C3 is tissue is present next D3 is 4 µmol . Next E3 is 0.004 g next R3 is 100 µl . Next N3 is 100 cells per ml .', {'entities': []}) \n",
      "\n",
      "('15  16 and 17 contain only our protein .', {'entities': []}) \n",
      "\n",
      "('for 40 min .', {'entities': []}) \n",
      "\n",
      "('am running EGF profiling assays . Today  we have 5 plates for wild type and 4 plates for axon 20 profiling .', {'entities': []}) \n",
      "\n",
      "('TCEP HCl 1 molar solution from TCEP Melfort . Cas number 51805 - 45-G or 9 . lot number 41196 - 41769 . Molecular weight 286.65 .', {'entities': []}) \n",
      "\n",
      "('- 3 has one fiber in the inlet channel . One fiber going over the left half side of the entry channels . One piece of debris blocking one of the right entry channels . There are two instances of debris at the entry channel post interface . There is three instances of debris in the 2nd post array . There is minor cell wrapping and the 3rd in post array . And the 4th and 5th post array there is minor cell wrapping but lots of spider web like cell debris . And the 6 and exit post channel interface there is minor wrapping but lots of spider web like cell debris . The chip flow goes from bottom to top of image . There are dimples present in the chip . Chip image', {'entities': []}) \n",
      "\n",
      "('me in 10 seconds to do anything .', {'entities': []}) \n",
      "\n",
      "(\"you may new meeting in your calendar . And then to 2  you just type in ingenuity pathway analysis and then you go to scheduling assistant and you see when it 's free so basically it 's free all day tomorrow perfect OK  so then you select the day and the time that you want to book for and then and then just say send Ellen if it 's for you  it will automatically accepted and I do n't have to get it approved by anyone I could just go on and use it . OK  that 's what was messing up last time because I was waiting for someone to approve it or like email me with instructions on how to ? You just go on and access it OK if you booked the time that 's your time and nobody else should be on it . I think that makes sense .\", {'entities': []}) \n",
      "\n",
      "('think I added 3 µl of secondary antibody to my assay arrays yesterday instead of 0.3 µl .', {'entities': []}) \n",
      "\n",
      "('with Multidrop Combi for the phospholipidosis assay . 384 standard  30 µl Standard cassette full full full plate .', {'entities': []}) \n",
      "\n",
      "('10 ml of water .', {'entities': []}) \n",
      "\n",
      "('1 was not responding  and it was not connected to the reader  so I rebooted the system .', {'entities': []}) \n",
      "\n",
      "('pressure step at 1 bar . First sample no bubbles . Second sample  no bubbles . Third sample  1 bubble per two seconds . Last sample  no bubbles .', {'entities': []}) \n",
      "\n",
      "('10 ml to vial', {'entities': []}) \n",
      "\n",
      "(\"here G3 dissection so hemolin perfusion . I perfused 11 mosquitoes because they in general did n't have a lot of hemolin . Only 2 of them has really nice quantity of hemolin . Besides that gut dissection . For the foregut I have included 5 or 6  I 'm not sure if I included the 6 one  because it was stuck to the needle  the foregut got stuck to the needle and for the main hindgut I have 5 hindguts for GT sugar Fat .\", {'entities': []}) \n",
      "\n",
      "('me to culture a chromobacter and all 9 salmonella tomorrow afternoon before the concert at 7 PM . Blueberry .', {'entities': []}) \n",
      "\n",
      "(\"sure where port 1 channel 0 starts  so now I have 0 for optostimulus  1 for center  2 behavior clock . 3 d sensor  4 reach sensor . in case this plug is not 0  but 1 . Then it will all be shifted up one and I will lose the reach sensor which I do n't have connected now . Also  the leg sensor is not connected right now  so nothing lost . eventually I have to look at the names .\", {'entities': []}) \n",
      "\n",
      "(\"is quite too warm in the laboratory . It 's about 30 ℃ .\", {'entities': []}) \n",
      "\n",
      "('2 days  I can see infected  infected cells not yet  maybe the dilution was too high . I will check the cells tomorrow again .', {'entities': []}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('weight is 3.2 mg .', {'entities': []}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(\"northern blot finally works . However  I see too much expression of rear 6 on the cytoplasmic phase  so I do n't know if I should repeat it or not .\", {'entities': []}) \n",
      "\n",
      "('ml tubes 3 boxes  0.5 ml tubes 2 boxes  1.5 ml tubes .', {'entities': []}) \n",
      "\n",
      "('started at 4:00 PM . Cells from 29th of Respond to pacing is good . Density may be too high . 26 V  4 ms  3 Hertz . First control after 2 h. Cells for responding about 80 % .', {'entities': []}) \n",
      "\n",
      "('take this from the order merged list . This takes me about 10 min .', {'entities': []}) \n",
      "\n",
      "('cycles PV 1210 one part 240 h constant humidity CH . Stone chip resistance  chemical resistance .', {'entities': []}) \n",
      "\n",
      "('results for BR 87 and 266 . For T they are 210 and 368 .', {'entities': []}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('to change BV786 .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are .', {'entities': []}) \n",
      "\n",
      "('ELN BE001934_17 . Remember to write that up when you come back on Tuesday .', {'entities': []}) \n",
      "\n",
      "('protein PMS 2 - 52 and PMLC 12 SDS page after purification over heparin column .', {'entities': []}) \n",
      "\n",
      "(\"'s 820  still no spikes  or no good spikes . I 'm changing the activity . Uh the area . Going out again .\", {'entities': []}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('me information about how my experience . You have 5 experiments number one DNA research D 237  # 2 Fish EP002  # 3 PCR extraction  # 4 plant research  # 5 mice research A2 . Please say number of experimental to activate . Number .', {'entities': []}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('cabinet two . Next . About 40', {'entities': []}) \n",
      "\n",
      "('12  13 and 14 TK bleed 14  23 .', {'entities': []}) \n",
      "\n",
      "('me to begin recovery for plate 5 today at 12:45 .', {'entities': []}) \n",
      "\n",
      "('coffee Step 2 . Add 2 sachets of sugar .', {'entities': []}) \n",
      "\n",
      "('stained gel with instant blue that has been used once and allow for staining for 20 to 25 min .', {'entities': []}) \n",
      "\n",
      "(\"got 18S that 's for sure . And we have vim too . GAPDH  have a lot .\", {'entities': []}) \n",
      "\n",
      "('ID 19S00460 . 2.600 ng/µl .', {'entities': []}) \n",
      "\n",
      "('timer for 15 min .', {'entities': []}) \n",
      "\n",
      "('of western blotting is control  treatment  stimulus  stimulus with treatment . Running time was 2 h at 120 Blocking was for 50 min . And then incubation overnight with the primary antibody . Blueberry blueberry .', {'entities': []}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('auto centrifugation of 1 h and analyze the 2nd IMAC on mass spec .', {'entities': []}) \n",
      "\n",
      "('run for 30 min .', {'entities': []}) \n",
      "\n",
      "('cells do um double every 24 hours . The plate looks good .', {'entities': []}) \n",
      "\n",
      "('of cell loss starts to appear at 30 . Increase .', {'entities': []}) \n",
      "\n",
      "('D3- 3 starts on 09403 .', {'entities': []}) \n",
      "\n",
      "('new task today 5 PM lab meeting .', {'entities': []}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('µl of the cleared lysates were mixed with 3 µl reducing agent and 7.7 µl of sample buffer . Blueberry .', {'entities': []}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 24 ° 25 ° on experiment 1 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "(' 70 decibels . Wait . Yes  for mice c57 black . Also  70 is used as a background noise and handled a startle .', {'entities': []}) \n",
      "\n",
      "('amines whole buffer with fuming HCL to 7.6 and having flushed electrode with endotoxin free water .', {'entities': []}) \n",
      "\n",
      "('D3 -2 has one piece of debris blocking the entry channel in the middle . There is two instances of debris at the entry posts channel interface . There is spider web like cell debris in the post arrays getting progressively worse as you go through the chip at the exit pose the cell spider web like debris is the most significant . Um dimples are also present in the chip the fluid flow in the chip is going from bottom to top of image . Otherwise it is a relatively clean chip .', {'entities': []}) \n",
      "\n",
      "('set the water bath at 212 ° Fahrenheit .', {'entities': []}) \n",
      "\n",
      "('25 μl DTT .', {'entities': []}) \n",
      "\n",
      "('fractions were run on SDS PAGE but the protein concentration was too low so it was quite clean . Then it is decided to collect fractions from B1 to C5 .', {'entities': []}) \n",
      "\n",
      "('viscosity is 0.97 .', {'entities': []}) \n",
      "\n",
      "('me to freeze the cells on Friday at 5 PM .', {'entities': []}) \n",
      "\n",
      "(\"cell wash cell loss is seen with wash settings aspirate 25 . Plate patterns emerging does n't appear to be a suitable setting\", {'entities': []}) \n",
      "\n",
      "('note about mouse number 137 .', {'entities': []}) \n",
      "\n",
      "(\"me to check in tonight at 8:00 o'clock .\", {'entities': []}) \n",
      "\n",
      "('ml of water were added to the white suspension and then stirred for 20 min at room temp .', {'entities': []}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('S1  S2  S3  S4  S5  S6  S8 in A1  B1  B4  B6 into the water bottle .', {'entities': []}) \n",
      "\n",
      "(\"'s experiment is to seed 35 12-well plates . Each well will get 1 time 10 to the 5 cells . Therefore  each plate will get 12 times 10 to the 5 cells . Or  1.2 time 10 to the 6 cells .\", {'entities': []}) \n",
      "\n",
      "('36 1.437 E5 .', {'entities': []}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('me to take the cells out of the freezer in 30 min .', {'entities': []}) \n",
      "\n",
      "('HU 1643 . EXP 1 . Cm = 82 Only current on transient was measured because pipette was drifting . Cell was affected by bubbles .', {'entities': []}) \n",
      "\n",
      "('21 . Running prime', {'entities': []}) \n",
      "\n",
      "('A column 1 300 .', {'entities': []}) \n",
      "\n",
      "('the pair 4 and 5 . The mutant 4 and 5 . G2 Seems to be present in less amount for the R side . Blueberry .', {'entities': []}) \n",
      "\n",
      "('4 tyrode  valve 5 2 µM carbachol  valve 6 1 µM atropine  valve 7 2 µM carbachol plus 1 µM atropine  valve 8 1 mM barium chloride .', {'entities': []}) \n",
      "\n",
      "('me to check SFC at 3:15 .', {'entities': []}) \n",
      "\n",
      "('started at 2:20 .', {'entities': []}) \n",
      "\n",
      "('whether we do further reducing the aspirate to Z offset to 25 steps would help with the plate pattern . Setting more aspirate speed 1 CW .', {'entities': []}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "(\"'m currently splitting the [ inaudible ] cells . They are cultured in RPMI medium  I spun down a 10 centimeter dish of . Very dense [ inaudible ] cells that already changed the PH of the media very dramatically so it was nice clear . Yellowish I spun down 200 RPM in our large eppendorf centrifuge and now I will take probably one third over into new 75 centimeter flask . for re - culturing over the weekend protocols .\", {'entities': []}) \n",
      "\n",
      "('perfusion between mosquitoes blood fed  replicate 3 . I perfused 11 mosquitoes in total . Since they did not have much hemolymph coming out . For the guts everything went in order so 5 - 5 .', {'entities': []}) \n",
      "\n",
      "('? ] 1 protocol 331 . Very intense cytoplasmic staining throughout entire tissue .', {'entities': []}) \n",
      "\n",
      "('length is 305.1 µm .', {'entities': []}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('lysis buffer to each tube to 40 ml .', {'entities': []}) \n",
      "\n",
      "('down 28 times .', {'entities': []}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('concentration OCI - cells is 1.78 times 10 to the 6 .', {'entities': []}) \n",
      "\n",
      "('µl of concentrated ammonia to dissolve the cholic acid internal standard .', {'entities': []}) \n",
      "\n",
      "('coffee Step 3 . Add hot water while mixing thoroughly .', {'entities': []}) \n",
      "\n",
      "('added 100 µl of secondary antibody instead of 50 .', {'entities': []}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "('did not pipette in 5  but in 6 .', {'entities': []}) \n",
      "\n",
      "(\" we prime the water line with bleach 3 times to make sure there 's enough volume in there  I can see a solid column in the waste tube . So I guess the bleach must have gone in there are reconnecting it back into any instrument to keep it stored . Actually  keeping the water line off from the instrument probably going to empty the tank there to do not have anything growing up in the water .\", {'entities': []}) \n",
      "\n",
      "('15- 2 chip I am taking pictures within without contrast to demonstrate the cell buildup .', {'entities': []}) \n",
      "\n",
      "('12 1.110 E5 .', {'entities': []}) \n",
      "\n",
      "('am plating 3127 cells .', {'entities': []}) \n",
      "\n",
      "(\"35 seems to be the maximum sort of acceptable . In terms of plate pattern while we 're not sure whether that 's going to be enough .\", {'entities': []}) \n",
      "\n",
      "('me in 10 seconds to stop my incubation . Stop my incubation in 20 seconds .', {'entities': []}) \n",
      "\n",
      "('from the size exclusion fractions . C1  D1  D7  E3', {'entities': []}) \n",
      "\n",
      "('small cell lung cancer 11 protocol 330 Neuro - D1 . Low  very low to no nuclear expression across tissue potential cytoplasmic staining in very few localized areas of tissue . Caterpillar .', {'entities': []}) \n",
      "\n",
      "('sample 18 three times  then stopped . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('25 1.348 E5 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "('15 256000', {'entities': []}) \n",
      "\n",
      "('sample . Time 12:35 .', {'entities': []}) \n",
      "\n",
      "('place pipette on the order .', {'entities': []}) \n",
      "\n",
      "('is true for sample 13  which also has cleared up after centrifugation also has a big visible pellet in the bottom of the well .', {'entities': []}) \n",
      "\n",
      "('infected was M Scarlet  65 and 22 and neon . I can see CPE . I think roundabout 10 % of the cells are infected . I have to change the medium  but it looks very good . 1  1 in 10  sorry 1 in 100 dilution is enough .', {'entities': []}) \n",
      "\n",
      "('5 documents and send to LabTwin .', {'entities': []}) \n",
      "\n",
      "('test with Chabra  English  talking next to me . extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "(\"we do n't have anymore 24-well plates left  I am using plate from another brand called Costar 3524 .\", {'entities': []}) \n",
      "\n",
      "('investigation of the bioreactor needs to happen  the 21st of April .', {'entities': []}) \n",
      "\n",
      "('1 l of endotoxin free Tris pH 8 from Medicago .', {'entities': []}) \n",
      "\n",
      "('says 7 is wrong .', {'entities': []}) \n",
      "\n",
      "('an order for 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('is 1.767 E 5 .', {'entities': []}) \n",
      "\n",
      "('10:30 shake tubes to homogenize stuff sticking to the walls of the tube .', {'entities': []}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('buffer can be stored indefinitely at -20 ° .', {'entities': []}) \n",
      "\n",
      "('two other tools experiment in big Petri dish four mL of water to 25 mL of backed  of four mL bacteria to 25 mL of water . [ inaudible ] 360 µl of food .', {'entities': []}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "(\"'s peak is lying on a bit higher isoflurane dose 2.5 .\", {'entities': []}) \n",
      "\n",
      "('two of the 1st lane is a marker . The second two are input and flow through . I loaded 25 µl of those which is a bit too much it flows over . I cleaned well # 4 and then continue with 20 µl of each sample .', {'entities': []}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('mili ml of HEPES for the 2 l of cells exclusion buffer . 80 ml for elution buffer . And 80 ml .', {'entities': []}) \n",
      "\n",
      "('was at 10 ° when open .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1550 Extra patch .', {'entities': []}) \n",
      "\n",
      "(\"12:00 o'clock . Start opening the chest . Start lidocaine infusion .\", {'entities': []}) \n",
      "\n",
      "(\"'m leaving my competent cells for 20 min in the room temperature .\", {'entities': []}) \n",
      "\n",
      "('supernatant from A9 to 12 and stored in chromatography cabinet .', {'entities': []}) \n",
      "\n",
      "('the samples are placed close to the upper part of the tubes with the small insert . There is no point keeping the tubes on ice . Therefore  1 rack at the time with 24 samples are taken out from the refrigerator and 50 µl of ice cold water is added to each tube now . Mixed and briefly centrifuge .', {'entities': []}) \n",
      "\n",
      "('7 Carboplatin media added .', {'entities': []}) \n",
      "\n",
      "('me to culture all 9 strains of salmonella and a chromobacter tomorrow before the concert at 7 PM .', {'entities': []}) \n",
      "\n",
      "(\"all the results pass the specification . Only the panel 10 and 11 do n't pass the specification TL 227 because the de - lamination .\", {'entities': []}) \n",
      "\n",
      "('milliter Marcain .', {'entities': []}) \n",
      "\n",
      "('for the first row of the replicate 2 samples from 343 till 361 . I resuspend them in 18 µl of water  while the rest are suspended in 20 . Because this row contains many hemocytes  so I would rather concentrate the sample .', {'entities': []}) \n",
      "\n",
      "(\"plates with the samples have now been centrifuged briefly and it 's time to add the internal standard solution containing 1 µM of deuterium labeled Creatinine in acetonitrile : water mixture in 90:10 proportions .\", {'entities': []}) \n",
      "\n",
      "('change the strategy take all samples from E12 till H8', {'entities': []}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('need to check the IC 50 data format for the assays reader run and will analyze those assays as well .', {'entities': []}) \n",
      "\n",
      "(\"need to define 3 different death rates and then I need to make the simulations for each of them and run them on the server and when I am doing that I need . Then I have to set up the right analysis tool . For it also it 's very crucial to determine where It did where It got the control probability number one . And the test one because I do n't remember I do n't have it documented and definitely I need to understand whether them events probability how they relate to the probability of survival .\", {'entities': []}) \n",
      "\n",
      "('- 1 images starting on 09244 .', {'entities': []}) \n",
      "\n",
      "('avidin tube lot number 10273567 .', {'entities': []}) \n",
      "\n",
      "('action potential get after 4 msec and 14 volts stimulation . Amplitude around 50 mV.', {'entities': []}) \n",
      "\n",
      "('chabra difficult words . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('number 7 English talking in the background with headphones . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separate separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotor bath at 50 ℃ and 10 mbar to create Test end', {'entities': []}) \n",
      "\n",
      "('7 ? ] cells were seen in 10 µl of isolation . And also the solutions used a from the previous day .', {'entities': []}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('takes 3 h to run the staining . I then use a USB to pull off the PDF run file from the PC . I attach everything together and add into the Bio ELN .', {'entities': []}) \n",
      "\n",
      "('I23 and I24 are failing in plate wash', {'entities': []}) \n",
      "\n",
      "(\"20 µl of cell suspension . To the RL buffer for today 's Day 0 RNA samples .\", {'entities': []}) \n",
      "\n",
      "('changed 3 V into 5 V in cell one .', {'entities': []}) \n",
      "\n",
      "('49  position H1  2 different phases .', {'entities': []}) \n",
      "\n",
      "('me after 5 min .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "('resuspended cells were incubated for 15 min at RT .', {'entities': []}) \n",
      "\n",
      "('difficult words writing sound no noise . 10 mbar 25 ml 25 µl .', {'entities': []}) \n",
      "\n",
      "('ml of TBME . [ unsupported language ]', {'entities': []}) \n",
      "\n",
      "('is for experiment IG FB 001135 .', {'entities': []}) \n",
      "\n",
      "('252.9 µl of resuspension buffer and to dilution .', {'entities': []}) \n",
      "\n",
      "('D2 -1 image starting 09 .', {'entities': []}) \n",
      "\n",
      "('35 best in class 1 media added .', {'entities': []}) \n",
      "\n",
      "('ran CDK2 and CDK1 enzyme and substrate on FLIPR . I did 3 replicates of positive control and 3 replicates of negative control for each of them using new plate type ultra low attachment offered by Corning .', {'entities': []}) \n",
      "\n",
      "('for 30 min .', {'entities': []}) \n",
      "\n",
      "('1091080537 on top . Plate 10910852 on bottom .', {'entities': []}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"addition  this cell should n't be added to the bulk  the cells looks very weird the cell is actually a combination of 2  should n't be added .\", {'entities': []}) \n",
      "\n",
      "('they weigh 920 mg .', {'entities': []}) \n",
      "\n",
      "(\"'m gon na start working on the plating of different bacteria and the new agar plates . I 'm gon na divide each plate in 6 different slots so that I can have better space .\", {'entities': []}) \n",
      "\n",
      "('me to seed the cells at 3 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('soon as an experiment is closed and the analysis is finished  it needs to be signed off within 2 weeks of closing in the BioELN .', {'entities': []}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "(\"the aspirate Z offset to 35 to check if there 's any cell loss at that setting\", {'entities': []}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('a 100 . Bye .', {'entities': []}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('µl pool 21 into lane 2 .', {'entities': []}) \n",
      "\n",
      "('BRD4 mice # 332 and 333 . Combination mouse 348 and unassigned mouse # 356 .', {'entities': []}) \n",
      "\n",
      "('BRD 4 mice # 332 and 333 . Combination mouse 348 and unassigned mouse # 356', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('44 7.696 E4 .', {'entities': []}) \n",
      "\n",
      "('reaction was performed Asp 33 46 74 . This time it was performed under nitrogen .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('standard 2 is added to column 11 and 12 .', {'entities': []}) \n",
      "\n",
      "(\"me heat up the Aga at 10:00 o'clock . Heat up the Aga\", {'entities': []}) \n",
      "\n",
      "('3 . Single chain fab . 5 D 5 42 .', {'entities': []}) \n",
      "\n",
      "('of distilled water and 20 % ethanol solutions for actor microsystem . Both filter through 0.2 µm filters .', {'entities': []}) \n",
      "\n",
      "('26  2.101 E 5', {'entities': []}) \n",
      "\n",
      "(\"'s experiment is to seed 35 12-well plate . In each well I will be porting 1 times 10 to 5 cells . Therefore  each plate will get 12 times 10 to the 5 cells .\", {'entities': []}) \n",
      "\n",
      "('observations at 14:25 . All animals fine .', {'entities': []}) \n",
      "\n",
      "(\"northern blot finally works . However  I see too much expression of rear 6 on the cytoplasmic phase  so  I do n't know if I should repeat it or not .\", {'entities': []}) \n",
      "\n",
      "('31 1.0.7 E5 .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"seems that the Interleukin 1 beta does n't have a washout experiment . It does n't really change after you apply until you can one Peter because it kind of messes up the membrane of the cell ? So . It 's important to ? Plan your experiments accordingly such that once Interleukin 1 beta is applied experiment is done .\", {'entities': []}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('isolation  in 10 µl 32 cells we have seen . And isolation was done by from Mula', {'entities': []}) \n",
      "\n",
      "('ID 18S006290 sample value 3.705 ng/µl .', {'entities': []}) \n",
      "\n",
      "('min casein block with protocol 695 on human ovary at 1 to 50 . Some very clean areas  but other areas still with some very mild background staining .', {'entities': []}) \n",
      "\n",
      "('23 182000 .', {'entities': []}) \n",
      "\n",
      "('I centrifuged', {'entities': []}) \n",
      "\n",
      "('19327003 basic current too small .', {'entities': []}) \n",
      "\n",
      "('10 mL of water  add 10 mL of ethyl acetate . Set temperature to 25 ℃ . Agitate at 500 rpm for 5 min transfer to a separatory funnel and settle for 30 min . Cut away the lower layer to waste . Wash organic layer with 30 mL of 1 normal hydrochloric acid . Wash organic layer with 30 ml of 500 mM sodium bicarbonate solution . Make a reminder to order a new 50 g bottle of sodium bicarbonate powder greater than 99.5 weight percent purity . Add 62.58 mg of anhydrous sodium sulfate . Sample 0.5 ml of reaction solution and dilute 50X into 30 volume percent ethyl acetate in heptane . Run sample A1 on HPLC Number 23 with method Silicon dioxide column gradient elution 30 to 60 % ethyl acetate in heptane recorded else area percent of product is 43.2 % .', {'entities': []}) \n",
      "\n",
      "('setup preparation at 9:40 .', {'entities': []}) \n",
      "\n",
      "('inoculated yesterday cultures from the construct  of the construct mScarlet of 65 strep kanamycin and mScarlet orf 65 strep orf 22 mNeon kanamycin .', {'entities': []}) \n",
      "\n",
      "('1 l of endotoxin free Tris pH 8 from Medicago .', {'entities': []}) \n",
      "\n",
      "('barcode 1090950954 .', {'entities': []}) \n",
      "\n",
      "('for 30 min . Fixative .', {'entities': []}) \n",
      "\n",
      "('need to order 20 boxes of 24 gauge needles and 5 boxes of 5 ml syringes .', {'entities': []}) \n",
      "\n",
      "('5 radio and English talking besides me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('timer for 5 min .', {'entities': []}) \n",
      "\n",
      "('NT control and the wild type b cells are infected to 90 % or more . But the knockout cells the EphA7 knockouts is also less infected . This is very interesting .', {'entities': []}) \n",
      "\n",
      "('235 ml of Glycerol to the 2 l buffers .', {'entities': []}) \n",
      "\n",
      "('GFP pool E7  E8  E9  E10 and E11 .', {'entities': []}) \n",
      "\n",
      "('1 to 3A were used for first dilution .', {'entities': []}) \n",
      "\n",
      "('timer for 1 min .', {'entities': []}) \n",
      "\n",
      "('me at 5:00 past 10 about making tea .', {'entities': []}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('me to . Run . The experiment . By replicate tomorrow 10:00 AM .', {'entities': []}) \n",
      "\n",
      "(' I said 40 ml Alba HS .', {'entities': []}) \n",
      "\n",
      "('run the LPS assay it was too concentrated . 4 um wild page . It should always they should always be tested on at least 6 10 to the minus one serial dilutions to be within range of the standard curve .', {'entities': []}) \n",
      "\n",
      "('sample 15 has clearly the smallest pellet of sample 12 to 15 after centrifugation .', {'entities': []}) \n",
      "\n",
      "('34 2.118 E5 .', {'entities': []}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('tubes 1 2 down quickly and stored tubes  1 to 16 in the chromatography cabinet .', {'entities': []}) \n",
      "\n",
      "('can see upstates in channel 18 to 16 proximately . But no spikes really . Death . 660 .', {'entities': []}) \n",
      "\n",
      "('for 30 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "('mix up between C10 and B10 in position A4 on plate 2 .', {'entities': []}) \n",
      "\n",
      "('vacing of sample is finished . All liquid is gone no clear pellet is visible . Samples are placed in the minus 20 freezer pending LC - MS analysis .', {'entities': []}) \n",
      "\n",
      "('for 7 min .', {'entities': []}) \n",
      "\n",
      "('µl of high buffer to hybridization reaction .', {'entities': []}) \n",
      "\n",
      "('28 Combination 3 media added .', {'entities': []}) \n",
      "\n",
      "('6 ml of cell suspension . Put the acoustic mass spec assay . And adding 10 ml for the phospholipidosis .', {'entities': []}) \n",
      "\n",
      "('43 163000 . Sample was repeated once .', {'entities': []}) \n",
      "\n",
      "('is 2.985 E 5 .', {'entities': []}) \n",
      "\n",
      "('timer for 10 min .', {'entities': []}) \n",
      "\n",
      "('PTS 20 Charles Rivers endotoxin 3 chip . Lot 7756135 . Expiry date September 2019 . Sensitivity 10 to 0.1 EUper ML .', {'entities': []}) \n",
      "\n",
      "('ml of DTT was added .', {'entities': []}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "(\". We 're looking at something near . Test 123 .\", {'entities': []}) \n",
      "\n",
      "('695 with neural D1 antibody  add 1 to 50 in TBST diluent with a 10 min casein blocking step  looking at human pancreas .', {'entities': []}) \n",
      "\n",
      "('1 cell wash we see bubbles . No visible media .', {'entities': []}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('of Cyclosporin as peroral solution before anesthesia . Rehydrex infusion 125 5 ml per kilo .', {'entities': []}) \n",
      "\n",
      "('2:15 IMAC elution samples are recovered using syringe . They are filtered through acrodisc PF syringe filter 0.8/0.2 µm supor membrane low binding sample . Lots number or catalog number PN4187 lot number 1338843 - 20143 .', {'entities': []}) \n",
      "\n",
      "('5 minutes  I need to stop the reaction .', {'entities': []}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('me tomorrow to order 10 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('cell lung cancer model 10  neural - D1  protocol 330 . No expression whatsoever ever . No areas of nonspecific staining . Negative .', {'entities': []}) \n",
      "\n",
      "('cell lung cancer model 10  neural - D1  protocol 330 . No expression whatsoever . No areas of nonspecific staining . Negative . Caterpillar .', {'entities': []}) \n",
      "\n",
      "(\"'s using it without the headphones  setting 15 min on her watch as well at the same time because he does n't trust it .\", {'entities': []}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('number is HU17 19 . Upon', {'entities': []}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('a reminder to order a new 50 .', {'entities': []}) \n",
      "\n",
      "('from sodium hydroxide sample pump to endotoxin free water and flush . Say 100 ml .', {'entities': []}) \n",
      "\n",
      "('me in 3 min to take out the sample in the freezer .', {'entities': []}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('a note for Friday  August 9th to organize .', {'entities': []}) \n",
      "\n",
      "('3 check the the book for all the necessary parameters . The recording was only current and transient because the bubble came and killed the cell .', {'entities': []}) \n",
      "\n",
      "('opto - tagging stimuli . First  four short stimuli  50 milliseconds from start to start  duration 5 milliseconds  then one second later . One long stimulus  around 29 milliseconds duration . Amplitude 5 .', {'entities': []}) \n",
      "\n",
      "('me to call Sarah 3 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "('BRD 4 mice # 332 and 333 . Combination mouse 348 and unassigned mouse # 356', {'entities': []}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "(\"'m making aliqouts of [ ? ] 25 ml . No no no no no no .\", {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('12  2.0.0 E5 .', {'entities': []}) \n",
      "\n",
      "('sorting cells with 1 mL medium', {'entities': []}) \n",
      "\n",
      "('timer 5 min . Avocado .', {'entities': []}) \n",
      "\n",
      "('unsupported language ] so Labtwin is becoming better and better . Meanwhile  it can already translate mg / ml µl µM centrifuge size exclusion chromatography . And cell expression in insect cells  SF 9 . Done by fabulous . PhD student David .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('day 7 samples are moved to 77 and 78 with PHH NPC order .', {'entities': []}) \n",
      "\n",
      "('experiment DNA research D237 .', {'entities': []}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('27 best in class 2 media added .', {'entities': []}) \n",
      "\n",
      "('21 159000 .', {'entities': []}) \n",
      "\n",
      "(\"I found that all cells and now I 'm adding 35 milliliters of fresh cells . They are not yet ... on 75 milliliters of virus ... cells . So we end up with our final volume of around 100 mill .\", {'entities': []}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('concentration of AmpUri is 24.2 ng/µl . Yes  of movie .', {'entities': []}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('chabra 4 meter distance to phone . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('the dissection of the larvae  I am filling the dishes with 200 μL of cold insect medium .', {'entities': []}) \n",
      "\n",
      "('jabra German radio in the background extraction biphasic solution was steered at room temp for 20 min and then transferred into separation funnel . Face cut is clear and fast new pair . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test and .', {'entities': []}) \n",
      "\n",
      "('me to warm cells at 3:00 PM .', {'entities': []}) \n",
      "\n",
      "('resistance circa 54 .', {'entities': []}) \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('19063 - 4 has a lot of red dots that looks like background .', {'entities': []}) \n",
      "\n",
      "('mixed up a new agarose gel with 1 % agarose in TAE buffer', {'entities': []}) \n",
      "\n",
      "('PCR with new primers and RVG mScarlet plus  Amnion . Number 52 for mScarlet and 61 for mScarlet  and number 42 for gm mScarlet Amnion .', {'entities': []}) \n",
      "\n",
      "('use 5 ml instead of 10 ml for the sample 13 .', {'entities': []}) \n",
      "\n",
      "('T is being prepped for bioanalyzer by being resuspended to a concentration of 200 ng/µl', {'entities': []}) \n",
      "\n",
      "('HM 5264 all 4 mice look good  healthy running around .', {'entities': []}) \n",
      "\n",
      "('to steps 3 .', {'entities': []}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('ID 4 concentration 234 ng / ml .', {'entities': []}) \n",
      "\n",
      "('clone 1  discard clone 226 .', {'entities': []}) \n",
      "\n",
      "('added 500 µl for 6-well plate of trypsin to harvest 2 wells for comparison . To . gel drops .', {'entities': []}) \n",
      "\n",
      "('mg of fluid .', {'entities': []}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('have to order a new kit mini prep classic . 100 preps with the ordering number D4015 .', {'entities': []}) \n",
      "\n",
      "('µl .1 in NAOH all 8 lanes .', {'entities': []}) \n",
      "\n",
      "('Adenosine triphosphate heavy labelled . Adding 13.3 µl .', {'entities': []}) \n",
      "\n",
      "('the 3rd of August tell Lottie with Chris is great .', {'entities': []}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('ng was added to each first strand cDNA synthesis', {'entities': []}) \n",
      "\n",
      "('Massspec for the samples according to what is documented in Word . And started another centrifugation to concentrated samples after 10 min about 2 ml was was concentrated and starting around for 30 min .', {'entities': []}) \n",
      "\n",
      "('will do 5  1 in 5 dilution steps . And .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('10 264000 . Sample was repeated once .', {'entities': []}) \n",
      "\n",
      "('about 3 after 5 ? Nope .', {'entities': []}) \n",
      "\n",
      "('blocking in 10 % BSA add 2 µl of anti - His HPR in TBST .', {'entities': []}) \n",
      "\n",
      "('citrate heavy label . Adding 8 µl', {'entities': []}) \n",
      "\n",
      "('100 μl [ inaudible ] buffer', {'entities': []}) \n",
      "\n",
      "('antibiotic treated bacteria incubated overnight in 50 mg per ml of kanamycine', {'entities': []}) \n",
      "\n",
      "('coffee step 4 . Add whole milk on top of well stirred coffee .', {'entities': []}) \n",
      "\n",
      "('was made up with 940 µl of water .', {'entities': []}) \n",
      "\n",
      "('for 1 min . May require a bit more .', {'entities': []}) \n",
      "\n",
      "(\"app takes a lot of battery so he needs to charge it . He always charges it when he goes back to his desk and he has no problems with the headset . He 's never in the lab for more than 10 h  6 h.\", {'entities': []}) \n",
      "\n",
      "('1 beta was added after 5 suites in the thought file . And after that  and after some suites the 4AP was attempted for recovery .', {'entities': []}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "('D4 - 2 has prevalence of dimples in the chip . It has the right side of the entry channels covered by a long piece of debris and what looks like to be a large piece of debris with maybe cells inside of it . The entry channels have minor cell capping on them . The interface between the entry and entry post has one instance of debris in between one of the posts channels . There is minor cell wrapping or and or capping on all the posts arrays and at the exit channel exit posts interface there again is minor wrapping present . It is relatively a clean chip . The fluid flow it goes from the bottom to the top of the image .', {'entities': []}) \n",
      "\n",
      "('ml HEPES for 250 ml lysis buffer .', {'entities': []}) \n",
      "\n",
      "('column with 10 ml / min buffer A for 150 ml .', {'entities': []}) \n",
      "\n",
      "('39 168000', {'entities': []}) \n",
      "\n",
      "('prefer prepare four times reaction of the T4 DNA polymerase to fill in the gRNA fragment .', {'entities': []}) \n",
      "\n",
      "('. To measurement sample one . Value 90 point  94.4 ng/µl .', {'entities': []}) \n",
      "\n",
      "(\"50 mg of AZ D 1234 . It does n't do very well\", {'entities': []}) \n",
      "\n",
      "('me to plate colonies at 3:45 PM .', {'entities': []}) \n",
      "\n",
      "('sample low copy number plasmid .', {'entities': []}) \n",
      "\n",
      "('is 2.159 E 5 .', {'entities': []}) \n",
      "\n",
      "('in 1.5 ml fractions in deep well block starting with 2A.', {'entities': []}) \n",
      "\n",
      "('all urine samples was added 50 µl of pure Milli - Q water after centrifugation of all the original urine samples to make sure that all liquid liquid was removed from the cap and the walls of the tube into the total volume at the bottom of the tubes .', {'entities': []}) \n",
      "\n",
      "('. For 20 min .', {'entities': []}) \n",
      "\n",
      "('31 . Best  oh  combination 1 added .', {'entities': []}) \n",
      "\n",
      "('drug company for GN-47 drug', {'entities': []}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "('flexible 59 µl .', {'entities': []}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('ml tubes 5 boxes .', {'entities': []}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('need to look up if for some reason myeloid populations like to eat BV510 because it really would explain a lot .', {'entities': []}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('will digest the mini DNA from Tuesday with XhO1 the whole day and in the afternoon  I loaded on the pulse free gel to separate the fragments .', {'entities': []}) \n",
      "\n",
      "('steps are performed either at 4 ° centigrade or on ice unless otherwise mentioned .', {'entities': []}) \n",
      "\n",
      "('11.1 of pool 80 and 81 into dilution .', {'entities': []}) \n",
      "\n",
      "('A2 type 1.11 e5 .', {'entities': []}) \n",
      "\n",
      "('µl 6 shots for Channel 3 . Bumblebee .', {'entities': []}) \n",
      "\n",
      "('me to go to the meeting on October 30th at 10 PM  10 AM . PM meeting will suck .', {'entities': []}) \n",
      "\n",
      "('remove weather .', {'entities': []}) \n",
      "\n",
      "('me to check on sample # 3 .', {'entities': []}) \n",
      "\n",
      "('33  1.432 E5 .', {'entities': []}) \n",
      "\n",
      "(\"now I am staining the cardiomyocytes . Which are all from  um . So they are  um  Gibco 5  Gibco 6  Gibco 7  Gibco 8 and from [ TT ? ] they are [ TT ? ] 17  another [ TT ? ] extra 17  18 and 19 . We will be staining for alpha actinin 1 to 1000 and cardiac troponin Now I 'm adding  so for the first\", {'entities': []}) \n",
      "\n",
      "('1.2 ml of the buffer with 200 µl of the wash membrane preparations .', {'entities': []}) \n",
      "\n",
      "('1.2 ml of the buffer with 200 µl of the wash membrane preparations .', {'entities': []}) \n",
      "\n",
      "('48 191000', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('me to take cells out of incubator tomorrow morning 8:00 AM .', {'entities': []}) \n",
      "\n",
      "('a mini protean TGX stainfree gel 4 to 15 % miniprotean from BioRad . 12 well comb 20 µl per well . Category catalog number 456 - 8085 .', {'entities': []}) \n",
      "\n",
      "('timer 5 min avocado .', {'entities': []}) \n",
      "\n",
      "('ID 718 S 006290 . 3.05 ng/µl .', {'entities': []}) \n",
      "\n",
      "('prepared the Melody today for small sample injection tests . We will run nuclei from [ Ili long ? ] provided by and we will run also in the afternoon ES cells and early mouse embryonic cells coming from the blastula stage . Christian is preparing the small injection pump . The instrument so far is ready . We will start with sorting out the nuclei into 96 well so that we have material for the small sample uptake later on .', {'entities': []}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('11  7.711 E 4', {'entities': []}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('sample 2 . And the value is 468 ng / ml . Measure sample 3 . Value is 195 ng / ml . Sample 4 is 449 ng / ml .', {'entities': []}) \n",
      "\n",
      "('cell number 5 .', {'entities': []}) \n",
      "\n",
      "(\"observation  P10 multisetup  do n't know if when I turned the waiting time before the sync the file is written to 10 milliseconds or to 11 milliseconds . I always get the arrow . It works  however  with 20 milliseconds and it helps to wait a little bit before I start session . After starting the experiment . So .\", {'entities': []}) \n",
      "\n",
      "('transduction test of the HFFs looks better after 48 h post infection . The normal transduction looks very nice with polybrene I have round about 50 % green cells . Without Polybrene I would say round about 10 to 20 % green cells . The cells  which were for 6 h on ice looks also good in the well with polybrene . I would say I have round about 30 to 40 % growth cells and without polybrene just 5 to 10 % . In the wells of the spinfection for 2 h with an cell pellet  I have with polybrene round about 40 % green cells . Without polybrene I have just 5 % and with infection for 6 h with an cell pellet I have 40  no 50 to 60 % green cells . And without polybrene  I would say 40 % . It looks very nice . And with spinfection in plate for 2 h  I have with polybrene I have 70 % green cells  without polybrene I have', {'entities': []}) \n",
      "\n",
      "('me in 10 seconds to check on the batch . To check on the batch .', {'entities': []}) \n",
      "\n",
      "('small gel from the gradient Peak G3 G9 .', {'entities': []}) \n",
      "\n",
      "('10 to the 5 .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('step with qPCR protocol underway . The diluted samples are being stored in a separate bag in the minus 80 .', {'entities': []}) \n",
      "\n",
      "('resistance 1  14  7 mV.', {'entities': []}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('10 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('39  2.267 E5 .', {'entities': []}) \n",
      "\n",
      "('sample 10  1.898 E 5', {'entities': []}) \n",
      "\n",
      "('. I centrifuge the samples in the ultra centrifuge .', {'entities': []}) \n",
      "\n",
      "('me to take the cells out of the incubator tomorrow at 10:00 AM .', {'entities': []}) \n",
      "\n",
      "('me to take the cells out of incubator today at 3:44 PM .', {'entities': []}) \n",
      "\n",
      "('the acquisition of a file called 20190903 Internal standard checks in negative ion mode .', {'entities': []}) \n",
      "\n",
      "('24th of October .', {'entities': []}) \n",
      "\n",
      "('you tell me please in 1 h that I have an appointment ?', {'entities': []}) \n",
      "\n",
      "('20 µl of UDP - N - acetyl - hexosamine', {'entities': []}) \n",
      "\n",
      "('samples are cooled down in the fridge after 37 ° treatment before addition to plates .', {'entities': []}) \n",
      "\n",
      "('me to set up the transformation experiment at 3:00 PM .', {'entities': []}) \n",
      "\n",
      "('anti - rabbit 488 for secondary .', {'entities': []}) \n",
      "\n",
      "('cancel the reminder at 3:00 PM .', {'entities': []}) \n",
      "\n",
      "('17  2.529 E 5', {'entities': []}) \n",
      "\n",
      "('flexible . 43 µl .', {'entities': []}) \n",
      "\n",
      "('a Cas9 Cell Biolabs protocol in Word to use .', {'entities': []}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('to Sprint 2 .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('iPS staining we have nanog PE from BD farm again . The PE mouse anti human nanog clone N 31 - 305 5 . Lot . 8113809 catalog number 560483 . This time is a new trial with 1 : 50 dilution .', {'entities': []}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('are KP58058  KP59331 and 110044394 .', {'entities': []}) \n",
      "\n",
      "('g magnesium chloride .', {'entities': []}) \n",
      "\n",
      "('22  9.922 E4 .', {'entities': []}) \n",
      "\n",
      "('a temperature change of 21 to 22 ° .', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('50 µl in a 96 well plate .', {'entities': []}) \n",
      "\n",
      "('need about 50', {'entities': []}) \n",
      "\n",
      "(\"replicate 3 . Um . Plasmodium falciparum infected G3 mosquitoes  um  dissection of the fat body . And gut  the whole gut started at 27 hours time point in stead of 26 . Um  besides that  um  the first 3 mosquitoes dissection go very well for the guts  besides could not get the entire gut by cutting so for the others I just dissected first like normal . I dissected the gut like normal and then  um  um  I proceeded to dissection of the fat body  which went pretty smooth actually . So that 's an important modification . So basically in the fat body samples I have 5 mosquitoes and in the gut samples I have 7  I think  because two of those I 'm not sure if they have entire gut . So important modification .\", {'entities': []}) \n",
      "\n",
      "(\"idea you said you 're right now  you 're writing the master thesis and this master thesis . It 's order . You have a lab book full of your notes and that 's why you did n't actually even wanna try to switch to electronic lab notebook icon right . But if someone came to you and say OK ? How many lab books do you have with the master thesis you think ? 1 my arms if someone comes to come came to you and said  Give me this book . I will digitize it for you completely completely and would you then switched because all that knowledge that you 've done so far would be digitized and you can search it  you could edit it . If someone does that for me sure I would do it . Yeah  I think so  but it 's a lot and also it 's a mess . That 's that 's a problem like what I mentioned what I have now I did n't fill in for the last 4 weeks because I did n't do a lot  but then I what I did was completely independent from each other . So when I fill it now  it will be completely random because sometimes I do little bit up here and then I do cell culture and then 2 weeks later . I 'm like oh s * * * I need to analyze that and then . And I think that 's the problem probably for lab book . So I work now for 2 groups .\", {'entities': []}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "(\"'m using 2.23 mg of tissue .\", {'entities': []}) \n",
      "\n",
      "('100 µl l into Eppendorf tube .', {'entities': []}) \n",
      "\n",
      "('me on Sunday at 10 PM .', {'entities': []}) \n",
      "\n",
      "('concentration is 5.155', {'entities': []}) \n",
      "\n",
      "('hemolin perfusion RKG mosquitoes  the mosquitoe number 4 barely have any hemolin .', {'entities': []}) \n",
      "\n",
      "('19928001 . Great instability . Um  nearly', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('me to check the cells tomorrow at 9:00 AM .', {'entities': []}) \n",
      "\n",
      "('hepatocyte culture approximately 50 % confluent as of September 2nd .', {'entities': []}) \n",
      "\n",
      "('me to check into flight at 8:00 PM .', {'entities': []}) \n",
      "\n",
      "('rate 50 Hertz .', {'entities': []}) \n",
      "\n",
      "('me in 2 min to get the plate out of the incubator . Get the plates out of the incubator .', {'entities': []}) \n",
      "\n",
      "('me to buy a book today at 21 PM .', {'entities': []}) \n",
      "\n",
      "('the removal removal of the sulfur tag the spin filters are for 3 1/2 min at rcf . This seems to leave a residual of around 40 to 50 µl in the filter .', {'entities': []}) \n",
      "\n",
      "(\"3  column 6 . Is n't good .\", {'entities': []}) \n",
      "\n",
      "('said that you ate at 8:30 .', {'entities': []}) \n",
      "\n",
      "('25 μl reversed transriptase to master mix .', {'entities': []}) \n",
      "\n",
      "('me to autoclave a tab medium on Monday at 10 AM .', {'entities': []}) \n",
      "\n",
      "('weight of ammonium formate 28.32', {'entities': []}) \n",
      "\n",
      "('6  9.377 E 4', {'entities': []}) \n",
      "\n",
      "('6  7  8  9  10  11 contain mostly contaminations .', {'entities': []}) \n",
      "\n",
      "('still have 10 samples in the fridge that I need to analyze .', {'entities': []}) \n",
      "\n",
      "('110044394 viability 42.3 % concentration 5.65 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well .', {'entities': []}) \n",
      "\n",
      "('only had enough cells to load one sample from each one channel  meaning that we are going to get 5000 cells per region .', {'entities': []}) \n",
      "\n",
      "('centrifuge tube  and its 1.186', {'entities': []}) \n",
      "\n",
      "(\"'s 2:45 start eluting again .\", {'entities': []}) \n",
      "\n",
      "('tubes 1 through 12  and tubes', {'entities': []}) \n",
      "\n",
      "('a 3 ms delay to the aspirate to improve pattern .', {'entities': []}) \n",
      "\n",
      "('32 ounces of fluid into the vial .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "(\"and yeah  dissection of RKG samples Plasmonium infected . Um  replicate 3 began to 3830 at 3030 time point instead of 2078 . Um  besides that gut dissection went ok  shake with 6 guts inside  because for one I was n't sure if I collected all  and  what even what ever  5 mosquitoes aspirator for defend the body .\", {'entities': []}) \n",
      "\n",
      "('count 1 : 20 dilution .', {'entities': []}) \n",
      "\n",
      "('695 with human ovary has some sporadic nuclear staining and clusters of strong cytoplasmic', {'entities': []}) \n",
      "\n",
      "('ion tandem mass spectrometry is the method of choice for assessment of LTE4 in biofluids due to its high specificity and sensitivity .', {'entities': []}) \n",
      "\n",
      "(\"diploidy analysis we 're gon na use 916 cells as well as  um  1732 cells and MH\", {'entities': []}) \n",
      "\n",
      "('- 2 chip starting on image 09288 .', {'entities': []}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "(\" let 's do a mark voice note . Both PYNP and PNP excellently retained their activity overnight and still perform reactions to equilibrium with 1 mM substrate on the following morning after 12 h of incubation at pH 6 and 6 ℃ .\", {'entities': []}) \n",
      "\n",
      "('me to check SFC at 3:15 PM .', {'entities': []}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('13  1.0.1 X 10 to the 5 . Sample repeated twice .', {'entities': []}) \n",
      "\n",
      "('I will harvest the B cells infected with RRV after 72 h.', {'entities': []}) \n",
      "\n",
      "('TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': []}) \n",
      "\n",
      "('22 starting from this message .', {'entities': []}) \n",
      "\n",
      "('counting has cells cells . Plate 1.24 million cells .', {'entities': []}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('number 6 with German speaking in the background  headphones in . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give test end', {'entities': []}) \n",
      "\n",
      "(\"injections 12:00 o'clock .\", {'entities': []}) \n",
      "\n",
      "('cells in batch 3 unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are likely to be contaminated .', {'entities': []}) \n",
      "\n",
      "(\"balloon will be placed distal to the first diagonal . Today it 's only 1 branch at the LAD . And we will place that balloon distant to this diagonal . Circumflex is not that big today supplying mainly the left ventricle . LAD we assume supporting mainly septum wall .\", {'entities': []}) \n",
      "\n",
      "('experiment EP 0024 .', {'entities': []}) \n",
      "\n",
      "('me to order 10 cm culture plates .', {'entities': []}) \n",
      "\n",
      "('got 3.29 * 10 to the six mL / cell or cells per mL. We .', {'entities': []}) \n",
      "\n",
      "('HU 1674 .', {'entities': []}) \n",
      "\n",
      "('needs to prepare needles for injection . Program 2 . um  [ inaudible ] it is [ inaudible ]', {'entities': []}) \n",
      "\n",
      "('activate experiment DNA research D237', {'entities': []}) \n",
      "\n",
      "('ID 18S006290 sample value 3.705 .', {'entities': []}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('starts 12:30 .', {'entities': []}) \n",
      "\n",
      "('? ] 1 protocol 331  Medi Small cell lung cancer 4 Model .', {'entities': []}) \n",
      "\n",
      "('1006 packs of sterile filter bottles approximately 250 ml and a boxes tips no longer in use', {'entities': []}) \n",
      "\n",
      "('protein was concentrated at 4000 g for more than an h to roughly 3.5 ml . So it was only concentrated fraction  and 13 .', {'entities': []}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('sit up an ICP for data installation . You need at least 3 nodes with each 16 CPU and 46 gigabytes of Ram .', {'entities': []}) \n",
      "\n",
      "('2169 from Paul Stemmer 7 3 2009 . Thawed and started the high speed centrifuge . Avocado .', {'entities': []}) \n",
      "\n",
      "('16  9.379 E4 . Repeated 3 times . Well twice .', {'entities': []}) \n",
      "\n",
      "('offset reduction to 35 . Does not introduce any cell loss . All cells are there . No visual patterns by microscope examination of individual wells .', {'entities': []}) \n",
      "\n",
      "('3849 3 human CD 45 at 1.38  human CD 19 at 0.76  CD 10 at 1.07  CD 34 at 0.65 and CD 22 at 0.23 . Avocado .', {'entities': []}) \n",
      "\n",
      "('me of a meeting  with Gloth at 6 PM .', {'entities': []}) \n",
      "\n",
      "('change to experiment DNA research D 237 . DNA research .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('9 and 10 are connected together so I also put 4 µl of cyno serum albumin in lane 8 .', {'entities': []}) \n",
      "\n",
      "('42 1.755 E5 .', {'entities': []}) \n",
      "\n",
      "('1 is loaded into well D3 and D4 and then samples are loaded according to protocol .', {'entities': []}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('# 15 is 22 . 2', {'entities': []}) \n",
      "\n",
      "(\"'s map notch one binding sites and N5b gene .\", {'entities': []}) \n",
      "\n",
      "(\"resuspended cell lines 9397 and 9885 ml in 5 ml total total medium . And for the 93 I got around 3000000 cells per ml . What 97 I got around cells per ml and 4988 I got only 8 ? And it took a thousand . And I do n't know why I was expecting higher number .\", {'entities': []}) \n",
      "\n",
      "('number 8  repeats  radio in the background  headphones in Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel point . Phase cut is clear and fast . The clear yellow solution was dried on the roto bath at 50 ℃ and 10 mbar to give . Test end', {'entities': []}) \n",
      "\n",
      "('started 21 min . Has to start reperfusion . So far single ventricular extrasystole .', {'entities': []}) \n",
      "\n",
      "('study 19 experiment 25 .', {'entities': []}) \n",
      "\n",
      "('21  4.764 E 4 . Sample attempted 3 times .', {'entities': []}) \n",
      "\n",
      "('chabra difficult words holding microphone to the mouth . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('me tomorrow to change media at 8:00 AM .', {'entities': []}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('me to order 5 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('recording 19928000 .', {'entities': []}) \n",
      "\n",
      "('an order for 17 plates . Plate Maps .', {'entities': []}) \n",
      "\n",
      "('step 1  I observed some weird stuff .', {'entities': []}) \n",
      "\n",
      "('me in 10 min to go shut the front door .', {'entities': []}) \n",
      "\n",
      "('tighter . On EV you plate with LB soft Agar with 6 g / l of Agar blueberry .', {'entities': []}) \n",
      "\n",
      "('a method run . The method name is SEC 16 60 6 samples .', {'entities': []}) \n",
      "\n",
      "('me to autoclave new TAB medium tomorrow at 10 AM .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are included in the experiment .', {'entities': []}) \n",
      "\n",
      "('50  postion A2  maybe a small crystal .', {'entities': []}) \n",
      "\n",
      "('an order for 12 plates .', {'entities': []}) \n",
      "\n",
      "(\"for the experiment 66 AZ0161 the A280 is 0.30 . The spectrum looks a little bit poor . There 's a slight sloping backgrounds . Interestingly  they all look better than the 67 . I will re - measure that on the NanoDrop .\", {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('close to the mouth . Test with normal headphones . New extraction . The biphasic solution was steered at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give .', {'entities': []}) \n",
      "\n",
      "('months after he said that I was in a corner .', {'entities': []}) \n",
      "\n",
      "('the pH of the ammonium formate buffer after adding 200 µl of concentrated ammonia solution .', {'entities': []}) \n",
      "\n",
      "('ml water were added to the white suspension .', {'entities': []}) \n",
      "\n",
      "(' order RPMI catalogue number 1111 .', {'entities': []}) \n",
      "\n",
      "('is in nitrogen . Looking nitrogen box  rack  Box 7  spot A 4586 .', {'entities': []}) \n",
      "\n",
      "('ml of water added to the white suspension .', {'entities': []}) \n",
      "\n",
      "('ML of water . [ inaudible ]', {'entities': []}) \n",
      "\n",
      "(\"to redo my ATM staining I 've got 6 slides : got my positive control  negative control  my four samples therefore got 6 slides the 6 * 150 add 350 is 950 . I 'm going to round up to 1200  I like that number  then I 'll do the C1V1 = C2V2 calculation . So C2 * V2 will be 0.12 * 1200 divided by C1 which is the 120 which will give me the total volume of 1.2 µl for my anti for my ATM antibody and then I will do 1200 - 1.2  which is 118.8 µl of the diluent which is TBS Tween\", {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('targets for CRISPR Cas9 mediated gene - leptine - titin', {'entities': []}) \n",
      "\n",
      "('Gibco DIF-288p32 fixed on 14 0 619 is  um that the APC 93  94 % . PE so 19  no  PE is 2.4 % and Tra is 77.3 % .', {'entities': []}) \n",
      "\n",
      "('a 2 min timer for my pipette .', {'entities': []}) \n",
      "\n",
      "(\"still do n't understand why there 's a double population in CD19 comma . It makes no sense  but I wonder if it 's because this is two donors mixed together .\", {'entities': []}) \n",
      "\n",
      "('1 ml sample was taken from the culture at harvest for expression analysis .', {'entities': []}) \n",
      "\n",
      "('have standard templates for 36 h and 6 h time points to collect with compound and cell .', {'entities': []}) \n",
      "\n",
      "('µl of pool 23 .', {'entities': []}) \n",
      "\n",
      "('out ammonium formate 28.35', {'entities': []}) \n",
      "\n",
      "('with difficult words in the writing so no noise . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "(' these samples are washed by spinning through 200 µl of 0.5 % TFA in 5 % acetonitrile 3 times . And this is done at room temperature centrifuge at 1500 RCF for 1 min each .', {'entities': []}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('to the hpk 1 folder and add create new study .', {'entities': []}) \n",
      "\n",
      "('10 Carboplatin media added .', {'entities': []}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "(\"the 66 experiments in the Apo shows A280 of 0\\\\.2 mg / ml  which is better . There 's about 250 µl of that .\", {'entities': []}) \n",
      "\n",
      "('36 2.737 E5 .', {'entities': []}) \n",
      "\n",
      "('was around 50 % LTD for this protocol .', {'entities': []}) \n",
      "\n",
      "('cell pellet looks smaller after second spin . Thus we need to slow spin speed and so I included of cells that I put in media and added of media without cells . They are in a T75 flask now .', {'entities': []}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "(\"will be started 11:00 o'clock . Blueberry .\", {'entities': []}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('uh 0.54 ml of 1 molar Tris pH 8 to the 27 ml  and filtered sample through a supor .8.2 mm filter .', {'entities': []}) \n",
      "\n",
      "('on boarding video recorded by Dennis is very bad . This 15 min can be collapsed into a simple animation as a medium scenario and best scenario just some nudging to get the user to create the first note and to learn ask they go .', {'entities': []}) \n",
      "\n",
      "('41 1.210 E5 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "('the most effective method for isolation . The highest producing cell line is 1 of the primary concerns in the biologics drug development process .', {'entities': []}) \n",
      "\n",
      "('me to start [ unsupported language ] at 12:45 . Start recovery at dragonfly . [ unsupported language ]', {'entities': []}) \n",
      "\n",
      "('me to attend the seminar on machine learning tomorrow at 12:00 PM .', {'entities': []}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('32 246000', {'entities': []}) \n",
      "\n",
      "('is 5 . B2 is . 0.16 .', {'entities': []}) \n",
      "\n",
      "('21 temperature change from 24 ° to 28 ° .', {'entities': []}) \n",
      "\n",
      "(\"'m going to plate 1.2 million cells . In one 375 plus . Counting of cells 20.3 million cells from one plus .\", {'entities': []}) \n",
      "\n",
      "('331 . Ashel . Observing cerebral cortex .', {'entities': []}) \n",
      "\n",
      "('me to check SFC at 11 .', {'entities': []}) \n",
      "\n",
      "('of 100 µl restriction solution for 4 samples .', {'entities': []}) \n",
      "\n",
      "('8 250000 .', {'entities': []}) \n",
      "\n",
      "(\"also upload a word Doc and the echo success report and the scanning barcodes . This takes up around 20 % of my write up . I tend to split my time  60 % into the lab and 40 % in the office . But this varies depending if there 's extra troubleshooting to do .\", {'entities': []}) \n",
      "\n",
      "('me tomorrow to the meeting at 12:00 PM .', {'entities': []}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('20 2.419 E5 .', {'entities': []}) \n",
      "\n",
      "('is my standard .', {'entities': []}) \n",
      "\n",
      "('me to feed the cells wednesday at 10 PM .', {'entities': []}) \n",
      "\n",
      "('14 227000', {'entities': []}) \n",
      "\n",
      "('µl of X AMP into lane 2 .', {'entities': []}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "('200 ml of water to quench reaction .', {'entities': []}) \n",
      "\n",
      "('infusion 60 ml / h .', {'entities': []}) \n",
      "\n",
      "('12 weeks starting now .', {'entities': []}) \n",
      "\n",
      "('was added at 50 µg / ml in 400 µl .', {'entities': []}) \n",
      "\n",
      "('run will run 6 sec and clean the column with sodium hydroxide and then leave it in water .', {'entities': []}) \n",
      "\n",
      "('me to call with Jackie at 3:00 PM today .', {'entities': []}) \n",
      "\n",
      "('rpm . Avocado .', {'entities': []}) \n",
      "\n",
      "('gel making process at 10 AM .', {'entities': []}) \n",
      "\n",
      "('plasma sample from the pig was pulled from 2 vials with 2 ml in each vial . Making 4 ml of sample in total . In the freezer remaining 1 vial of approximately 2 ml .', {'entities': []}) \n",
      "\n",
      "('what you just did with the video checking  also with the force sensor . Start ... no  start ... sorry  and just .', {'entities': []}) \n",
      "\n",
      "('17  1 .779 E5 .', {'entities': []}) \n",
      "\n",
      "('storage is tracked in sheets on the deep freezer . For example  the -80 freezers and the -20 freezers . We need to have a system in place that helps search for reagents more quickly .', {'entities': []}) \n",
      "\n",
      "('me 9:45 to go to the lab .', {'entities': []}) \n",
      "\n",
      "('to D9 1 ml . E1 to E9', {'entities': []}) \n",
      "\n",
      "('is 1.147 E 5 .', {'entities': []}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('acid prior to MI ok . Heart rate 76 .', {'entities': []}) \n",
      "\n",
      "(' 13.20 . Time .', {'entities': []}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('36 combination 2 media added .', {'entities': []}) \n",
      "\n",
      "('adding 50 µl water to 50 µl of urine and mixing for about 1 min without centrifugation  35 µl of the diluted samples were transferred to a 96 well plate . Another 35  no another 15 µl from each tube was pulled to create the QC urine pool . The remaining 50 µl of the two times diluted urine were saved and stored at the biobank for transportation to Filippos Michopoulos at Alderly park for targeted metabolomics using negative ion platform .', {'entities': []}) \n",
      "\n",
      "('. Say Step 2 on protocol .', {'entities': []}) \n",
      "\n",
      "('see our conditions are BB [ inaudible ]  BB [ inaudible ] 60 ° temperature .', {'entities': []}) \n",
      "\n",
      "('in 0.75 ml aliquots in sterile endotoxin free tubes .', {'entities': []}) \n",
      "\n",
      "('5 ml ammonium formate .', {'entities': []}) \n",
      "\n",
      "('am testing the Android LabTwin app latency in M2 .', {'entities': []}) \n",
      "\n",
      "('number 4 is a bit difficult to dissociated [ inaudible ] I think [ inaudible ] it faint strokes now . [ inaudible ) flakes .', {'entities': []}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('4 . Top shelf . Set up plates used for BLI no longer in use . Tube top filters no longer in use . FinnTip pipet tips . 10 * 384 pieces  four rack . Next shelf . A set of Sterivex Millipore .22 µm filter units . Set up tubes dark no longer used . A set of plates no longer used except for 1 case of plates  1536 whites for Victoria . Bottom shelf . Set of Thermo Scientific tips used for the PlateMate instrument . A series of HPLC parts used with the G and R mass spec .', {'entities': []}) \n",
      "\n",
      "('that 711 spreads a lot .', {'entities': []}) \n",
      "\n",
      "('et al 2012 . Ink et al 2014 .', {'entities': []}) \n",
      "\n",
      "('concentration is 3.721 µM.', {'entities': []}) \n",
      "\n",
      "('ID 18S006290 sample value 3.705 .', {'entities': []}) \n",
      "\n",
      "('concentration of psPax2 is 848.7 ng/µl  the fraction of 260 by 2 80 is 2.11  the fraction of 260 / 230 is 2.27 .', {'entities': []}) \n",
      "\n",
      "('perfusion of RTG mosquitoes . Mosquitoes are generally smaller than G3 and for many of them the high magnesium was leaking out from the hole or from the injection spot', {'entities': []}) \n",
      "\n",
      "(\"diploidy experiment  we 're going to use 3 cell lines 916 1732 and images 2 cell lines . We 're gon na use .\", {'entities': []}) \n",
      "\n",
      "('PM all control units still working .', {'entities': []}) \n",
      "\n",
      "(': ? 50 ml Falcon tube one box  10 ml pipettes 5 boxes  cell culture dishes 100 mm diameter 2 boxes .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('for 10 min .', {'entities': []}) \n",
      "\n",
      "(\"at 10 and I need to be in the lab quarter to 10 and my time point [ inaudible ] . And my start at finishing on 5  and so I finished on 5 . Hopefully - Looks interesting  well - Let ' see - I will not see -\", {'entities': []}) \n",
      "\n",
      "('. Valves 8 . Fill in is damaged . Remind to replace .', {'entities': []}) \n",
      "\n",
      "('in the batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('with the drug started at 2:50 . Blueberry .', {'entities': []}) \n",
      "\n",
      "('5 mM of calcium chloride and 10 mM of Magnesium Sulfate . Blueberry .', {'entities': []}) \n",
      "\n",
      "(\"me to seed cells this afternoon 2:00 o'clock .\", {'entities': []}) \n",
      "\n",
      "('ID 18S0078 sample value 3.75 ng/µl .', {'entities': []}) \n",
      "\n",
      "('a 200 µl pipette  you can pipette up and down your solution to make sure that you remove any air bubbles .', {'entities': []}) \n",
      "\n",
      "('number 7 is 18.6', {'entities': []}) \n",
      "\n",
      "('pH was adjusted to 5 with one mol [ unsupported language ] hydroxide .', {'entities': []}) \n",
      "\n",
      "('1233 times 85 .', {'entities': []}) \n",
      "\n",
      "('2- 3 is starting an image 09386 .', {'entities': []}) \n",
      "\n",
      "('need to saw my cells tomorrow . And culture them at cell density of cells per ml .', {'entities': []}) \n",
      "\n",
      "('2 times solubilization buffer  which consists of 100 mM HEPES pH 7.5  2 molar sodium chloride  2 mM TCEP  20 % Glycerol and 2 mM calcium chloride . For 200 ml use 20 ml 1 molar HEPES stock  18 ml 5 molar sodium chloride stock  0\\\\.8 mililiters 0\\\\.5 molar TCEP stock . Note that I will be using instead of the 0\\\\.7 stock and I need to adapt the volume . 46 ml 85 % glycerol and 0\\\\.4 mM 1 molar calcium chloride and 4 protease inhibitors . Note protease inhibitors I will add later to the previous when I make a final solutions .', {'entities': []}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('me to put the cells into the incubator today at 12:19 PM .', {'entities': []}) \n",
      "\n",
      "('µl pours 15 into lane 4 .', {'entities': []}) \n",
      "\n",
      "('6 1.544 E5 .', {'entities': []}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('of step 2 . Density is 0.947 kg per square m.', {'entities': []}) \n",
      "\n",
      "('resuspended the beads in the tube in 2 ml wash buffer  and wash the column with those 2 ml .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('1 5 dead flies  tube 2 3 dead flies . Tube 4', {'entities': []}) \n",
      "\n",
      "('110044394  media is much more yellow compared to other donors after 2 days of culture .', {'entities': []}) \n",
      "\n",
      "('used for the EphA 2 mutations 200 ng for the quick ligation . I see on the control plates also colonies  but not so many like on the correct plates . I will inoculate 1 or 2 minis o / n and check to know by sequencing .', {'entities': []}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('A9 sulfur tagged antibodies aliquoted into 5 µl aliquotes and stored in minus 20 .', {'entities': []}) \n",
      "\n",
      "('with buffer A for 5 ml / min', {'entities': []}) \n",
      "\n",
      "('test one . Extraction . The biphasic solution was steered at room temperature for 20 min and 10 transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('cloning of KHVGH shuttle plasmid worked very well . Just on the ampicillin plate . On the kanamycin plate I see just one clone  could be  the reason could be that I used the second elution of P  one PCR for this cloning which I plated on the kanamycin plate . I see a lot of colonies . I will inoculate in the evening 2 or 3 colonies to do a mini prep tomorrow .', {'entities': []}) \n",
      "\n",
      "('on the ATM staining I did last week where I worked super duper duper duper duper duper duper duper duper duper hard on all this staining and then I come to find that some of the stupid blocks  they do n\\'t even have tumor in them  so now I had to go back and I had to contact the scientist who cut the brains before me to say \" and I do not want to get in trouble and be told \" \" . I just got these blocks only cut out like 5 sections  so it could n\\'t have been me . Like shaggy said it was n\\'t me and now I have to go into other blocks of tumors and cut them out and also now out of 40 blocks there are only like 12 had tumors in them  so I have two animals per group  which is n\\'t going to get me a great average . And I \\'m so frustrated about it . I \\'m really  really tired', {'entities': []}) \n",
      "\n",
      "('weight of the GFP pellet is 2', {'entities': []}) \n",
      "\n",
      "('the second IMAC the non co - express Relaxin VOC elute in a gradient of 5 min 200 % B at 10 ml / min fractionation 1.5 ml in a 96 well block .', {'entities': []}) \n",
      "\n",
      "(\"3 . Cm = 101 . It would have been a very good recording if I did n't try to measure shortening  unfortunately leak crept in consequently preventing the measure a very good load . However  the current and transient was very good and representative . Although the second attempt of load could be used so you have to determine that from the analysis . Also  they series resistance was very fine was below 10 and all parameters were measured .\", {'entities': []}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('I did a Gibson Assembly and I did a aqua cloning um assembly for 1 h.', {'entities': []}) \n",
      "\n",
      "(\"flashes after 15 seconds that 's 10 % of the measurement time .\", {'entities': []}) \n",
      "\n",
      "('me to set a timer at 5 PM .', {'entities': []}) \n",
      "\n",
      "(' first plate was read after 15 min .', {'entities': []}) \n",
      "\n",
      "('and adding 8 µl Tris HCL to all lanes  400 mM.', {'entities': []}) \n",
      "\n",
      "('28  2.0.3 X 10 to the 5 . Repeated 3 times .', {'entities': []}) \n",
      "\n",
      "('number 2 .', {'entities': []}) \n",
      "\n",
      "('an order for 69 pipettes .', {'entities': []}) \n",
      "\n",
      "('are added to the plate by first adding 45 µl of cell extraction buffer to the wells and then adding 5 µl of sample to each well  meaning that the final dilution is 1 to 20 .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('20 µl of the unconcentrated elutions from the strep tactin column from the elution tubes and prepare SDS page samples . So 27 and 3 for sample LDS and reducing agent respectively and otherwise use standard conditions .', {'entities': []}) \n",
      "\n",
      "('me to lab activity on 23rd of March at 12:00 PM . OK .', {'entities': []}) \n",
      "\n",
      "(\"the red paper they say rodents typically begin to startle over from 85 to 90 decibels . That 's according to Birdget [ inaudible ] and Susanne Schmidt . in Jay visual experiments 2011 .\", {'entities': []}) \n",
      "\n",
      "('sodium hydroxide 3 pulses EDCNHS 7 min NeutrAvidin .', {'entities': []}) \n",
      "\n",
      "('to buy 5 ml eppendorf tubes and shallow reservoirs .', {'entities': []}) \n",
      "\n",
      "(\"'m doing a stand  a standard curve for IgG 28 .\", {'entities': []}) \n",
      "\n",
      "('5 complete EDTA free tablets .', {'entities': []}) \n",
      "\n",
      "('- 2 might not have gone all the way through because of residue on the Eppendorf Sample 2 .', {'entities': []}) \n",
      "\n",
      "('still have 10 samples .', {'entities': []}) \n",
      "\n",
      "('13:35  surgery finished .', {'entities': []}) \n",
      "\n",
      "('me that the session ends at 2:15 .', {'entities': []}) \n",
      "\n",
      "('me to attend the seminar on machine learning tomorrow at 12:00 PM .', {'entities': []}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('ml of the sample was used passed through SEC S200 660 column .', {'entities': []}) \n",
      "\n",
      "(\"concentration of old protein is 0.62 mg / ml and therefore I had to adjust the calculations from new protein . Finally  I 'm adding 1.913 µl of old stock into 1 ml of total buffer . Bumblebee .\", {'entities': []}) \n",
      "\n",
      "('2 tubes of LMNG  which have been dissolved in about 20 ml of water . Spin them 5 min at 500 g to get a flat meniscus to be able to add on water to exactly 20 ml .', {'entities': []}) \n",
      "\n",
      "('Cas9 in - house assay evaluation .', {'entities': []}) \n",
      "\n",
      "('2 standard is from the 15th of October  and the kit 1 standard is from the first week in November .', {'entities': []}) \n",
      "\n",
      "('µl sodium hydroxide into all lanes .', {'entities': []}) \n",
      "\n",
      "(\"infected B cells with RRV will type and the AELAAN mutant . All cells are infected . So looks like 90 % or more are infected . I did n't expect this because other B cells are not so good to infect but next time I have to use a higher dilution for infection .\", {'entities': []}) \n",
      "\n",
      "('9  1.372 E5 .', {'entities': []}) \n",
      "\n",
      "('me to pick up dried samples at 5:00 PM today . Blueberry .', {'entities': []}) \n",
      "\n",
      "(\"'s doing a big study it a washout study . She incubates on 10 µl removing the comma .\", {'entities': []}) \n",
      "\n",
      "('µl in magnesium chloride was added kinase buffer .', {'entities': []}) \n",
      "\n",
      "('me after 3 min .', {'entities': []}) \n",
      "\n",
      "('approximately of proteins and RNA as well .', {'entities': []}) \n",
      "\n",
      "('dispensing [ ? ] reagent 2 plates . Now .', {'entities': []}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('µl pour 13 into lane 2 .', {'entities': []}) \n",
      "\n",
      "('a Biacore reagent for experiment trial # 4 .', {'entities': []}) \n",
      "\n",
      "('Friday  I did a Western blot from my samples B [ inaudible ] knock out cells IFI7 . Something went wrong  after blotting I had a big smear on the membrane . You can see it if you detected with GAPDH antibody . Is the MP40 buffer too', {'entities': []}) \n",
      "\n",
      "('this is white suspension was stirred for 20 min at room temperature and then filtered off . pH was adjusted to 2 with [ unsupported language - sodium ] carbonate .', {'entities': []}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('free DNA from 4 pig samples were prepared . Pig number 5 ID 232672 . First sample pre - MI 12th of March 2019 . Second sample pre - HVP 15 April 2019 . 3rd sample 1 h post HVP 15 April 2019 and 4th sample post - HVP 17th of June 2019 .', {'entities': []}) \n",
      "\n",
      "('protocol step 7 .', {'entities': []}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('concentration is 124 ng/µl . T concentration is 394 ng/ µl and B concentration is 306 ng/µl .', {'entities': []}) \n",
      "\n",
      "('2  pipet 25 μl ribonuc', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('my protocol of the 15th of October .', {'entities': []}) \n",
      "\n",
      "('- 1 chip images starting on 09261 .', {'entities': []}) \n",
      "\n",
      "('the last purification  pull F9 to G1 uh  which is peak 1 . Pull G2 till G4 which is peak 2 . Pull G7 to G11 which is the correct most interesting peak  peak 3 . And finally Pull H3 to H6 which is peak 4 which will be used for analysis .', {'entities': []}) \n",
      "\n",
      "('samples down for 6 seconds .', {'entities': []}) \n",
      "\n",
      "('normal headphones microphone not close to the mouth . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('. For 5 min  please . Could you play something technical ?', {'entities': []}) \n",
      "\n",
      "('W age to remember to check the sequences of the 2:00 prophage . So that they can be detected in the sequencing .', {'entities': []}) \n",
      "\n",
      "(\"'re doing colony PCR so primer 3 and 4 .\", {'entities': []}) \n",
      "\n",
      "('35 1.745 E5 .', {'entities': []}) \n",
      "\n",
      "('. I use for the PCR 375 and 86 also 0.06', {'entities': []}) \n",
      "\n",
      "('pressure 14.73 .', {'entities': []}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('running buffer SDS page 10 times 2 bottles  Catalog number 3060.2 5L.', {'entities': []}) \n",
      "\n",
      "('electrophoresis buffer needs to be set to 20.2 and I think I need more paper .', {'entities': []}) \n",
      "\n",
      "('is my donor unstimulated .', {'entities': []}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('10 with headphones  very loud lab associate in the lab . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transfered to a separation funnel . New paragraph . Phase cut is clear and fast . The clear yellow solution was dried in the roto bath at 50 ℃ and 10 mbars to give . Test end', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('first result of my calculation is 0.3 µl .', {'entities': []}) \n",
      "\n",
      "('ileum - minimal crypt single cell  epithelial apoptosis .', {'entities': []}) \n",
      "\n",
      "('- 2 chip images ending on 09364 .', {'entities': []}) \n",
      "\n",
      "('accessing legacy data the D360 is more historical .', {'entities': []}) \n",
      "\n",
      "('valve 6 also damaged .', {'entities': []}) \n",
      "\n",
      "('weight of a cholate 7.8 mg . We need to have 1.89 ml of water .', {'entities': []}) \n",
      "\n",
      "('treated with 70 ℃ seem to have granulated .', {'entities': []}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('of actor Micro with superdex TM200 increased 5 150 GL . With a pump wash micro .', {'entities': []}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('me to call Sarah tomorrow at 2 PM .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('solubilization at 8:40 .', {'entities': []}) \n",
      "\n",
      "('ran the centrifuge . I ran the ultra centrifuge for 10 min .', {'entities': []}) \n",
      "\n",
      "(\"'m thawing my competent cells for 20 seconds .\", {'entities': []}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "(\"'m seeding 4000000 cells per T 225 flask .\", {'entities': []}) \n",
      "\n",
      "('for 3 min .', {'entities': []}) \n",
      "\n",
      "('1 mag finished .', {'entities': []}) \n",
      "\n",
      "('3 7.184 E4 .', {'entities': []}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('crystal in plate 53 .', {'entities': []}) \n",
      "\n",
      "('38 9.136 E4 .', {'entities': []}) \n",
      "\n",
      "('48  5.688 E 4', {'entities': []}) \n",
      "\n",
      "('and optimization of FRET particles . So based on the new precipitation  nano precipitation . Nano . Nano . Oh my God . Oh my God  all right 488 nanometers excitation and we normalize at donor initial peak . The best relative concentrations are one percent donor and 0.3 percent acceptor . Uh . All right .', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('6 sample 21 lane 7 GFP all 20 µl loaded except for lane 5 . Program will be run 350 V  11 min using by BioRad Power Pack 3000 .', {'entities': []}) \n",
      "\n",
      "('. Remind me to order 50 ml falcon tubes .', {'entities': []}) \n",
      "\n",
      "(\"prophage needs to be tested . Or ATTR and ATTV . So 4 prophage times 2 that 's 8 and 2 housekeeping gene 's . Prepare 10 master [ ? ] Blueberry .\", {'entities': []}) \n",
      "\n",
      "('1 1.349 E5 .', {'entities': []}) \n",
      "\n",
      "('free DNA was isolating using the MagMAX Cell - Free DNA isolation Kit catalog number 829319 .', {'entities': []}) \n",
      "\n",
      "('record a temperature change from 25 to 29 ° .', {'entities': []}) \n",
      "\n",
      "('in batch 6 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('in the lab with headphones difficult words . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 70 mg 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('injected cell for each mouse . And this is look healthy and 90 % are alive .', {'entities': []}) \n",
      "\n",
      "('the cloning of Rab 4A  Rab 5 and Rab 7 I used 100 ng of vector and 37.5 ng of insert . I used the quick ligase . The master mix . Or the mix . Volume was 20 μl . I incubated for 15 minutes at room temperature . The transformation . For the transformation I used Stable 3 .', {'entities': []}) \n",
      "\n",
      "('added the baculovirus to the Shaker flask for the A 12345 experiment avocado .', {'entities': []}) \n",
      "\n",
      "('freeze epithelial like cells from HSNKX 2 - 5  TBX 5 .', {'entities': []}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('l of acetone from Sigma Aldridge .', {'entities': []}) \n",
      "\n",
      "('1 to 5 were not centrifuged at 14000 G before adding them to the plate . So samples 11 to 15 are centrifuged before addition to plate .', {'entities': []}) \n",
      "\n",
      "('3.7 µl of fluid into the tube .', {'entities': []}) \n",
      "\n",
      "('pericardium 14:08 . Dextran and preclude gore tex membrane .', {'entities': []}) \n",
      "\n",
      "('added to start reaction every 5 min .', {'entities': []}) \n",
      "\n",
      "('123 with the flick button from the background .', {'entities': []}) \n",
      "\n",
      "('out 4.13 mg of internal standard .', {'entities': []}) \n",
      "\n",
      "('. 15 - 56 liver', {'entities': []}) \n",
      "\n",
      "('surgery 10:20 .', {'entities': []}) \n",
      "\n",
      "('the non process from the first purification and desalt it on the size  on the desalting column at 5 ml / min .', {'entities': []}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('concentration is 720 nM.', {'entities': []}) \n",
      "\n",
      "('on a timer for timer for 10 min .', {'entities': []}) \n",
      "\n",
      "('# 2 for the slack channel .', {'entities': []}) \n",
      "\n",
      "('ml . 25 µl 30 bar .', {'entities': []}) \n",
      "\n",
      "('me of a meeting for tomorrow at 2 PM .', {'entities': []}) \n",
      "\n",
      "('me to remove cells from minus 80 tomorrow at 10:00 AM .', {'entities': []}) \n",
      "\n",
      "('number 2 . Hello .', {'entities': []}) \n",
      "\n",
      "('to test stable 3 cell growth in Terrific Broth .', {'entities': []}) \n",
      "\n",
      "('sample 413', {'entities': []}) \n",
      "\n",
      "('60 % increase to 100 % B immediately .', {'entities': []}) \n",
      "\n",
      "(\"'m going to need 425 μL of water .\", {'entities': []}) \n",
      "\n",
      "('week 1 .', {'entities': []}) \n",
      "\n",
      "(\"recordings ECG 9 until 10:00 o'clock . Same BS number in LabChart BS002541 - 19 .\", {'entities': []}) \n",
      "\n",
      "(\"in batch 3 are very likely to be contaminated . Now it 's back\", {'entities': []}) \n",
      "\n",
      "('internal standard did not look good . I am putting it in another column . Column 19 .', {'entities': []}) \n",
      "\n",
      "(\"we 're letting them sit overnight at 28 ° C\", {'entities': []}) \n",
      "\n",
      "('the titers are really low . I can only see clearings in the direct 10 µl spot . However  most mutant wild type pairs show induction in the wild type and not in aspoti mutant . Regardless of the relA mutation . And those inductions can not be corroborated by the OD curves . Blueberry .', {'entities': []}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('on timer for 50 min . Timer 50 min . Timer 50 min .', {'entities': []}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"cells all seem rounded up and that some reason they really did n't like aspirate setting 25 .\", {'entities': []}) \n",
      "\n",
      "('only have primers for beta 2 M in the concentration of 25 micro molar . So instead of 6 μl . For a 6 time master mix  I will use 2.4 μl .', {'entities': []}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('to 50 % buffer B in 8 min at flow rate of 4 ml / min .', {'entities': []}) \n",
      "\n",
      "('against GFP 21 slides .', {'entities': []}) \n",
      "\n",
      "('solution was solved under argon atmosphere at room temperature and added to the mixture and was stirred for 5 min .', {'entities': []}) \n",
      "\n",
      "('do I make a 1 % solution of Tris - HCL ? No  that .', {'entities': []}) \n",
      "\n",
      "('35 . Wait . Yes . Sample 35 failed to read . Attempted 3 times . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('33 1.837 E5 .', {'entities': []}) \n",
      "\n",
      "('of APS and TEMED to 45ml Native buffer .', {'entities': []}) \n",
      "\n",
      "('spin is 300 rcf for 3 min .', {'entities': []}) \n",
      "\n",
      "('continue with fraction G3  G9 .', {'entities': []}) \n",
      "\n",
      "('cells today look fine . I honestly split them on the weekend one more time on Saturday and suspended them again in 15 ml fresh media . I used roughly 40 % of the cell culture  then so less than 1 to 2 split . I will keep the cells in the same media today  may add a few ml on top of it to give enough food to come over the night because tomorrow  we will use the cells for setting up the course experiment .', {'entities': []}) \n",
      "\n",
      "('test number 6 . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transfered into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the roto bath at 50 ℃ and 10 mbar to give . Test end', {'entities': []}) \n",
      "\n",
      "('samples were taken from freezer # 3 and stored in the main fridge .', {'entities': []}) \n",
      "\n",
      "('first 24 RNA samples need to be re - filtrated because of using the wrong washing buffer .', {'entities': []}) \n",
      "\n",
      "('2019 eleventh of November . Start experiment BS002541 dash 19  experiment BE 002541 dash 29 . Pig ID 338679 blue pig 5 . Today we will start the MI surgery . On pig blue . Number . 5 .', {'entities': []}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('these notes to experiment BE0022 .', {'entities': []}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('10 µl Benzonase lysis buffer . Benzonase or DNase PL or PB 170692 date 2017 05 30 .', {'entities': []}) \n",
      "\n",
      "('-1 chips images  ending an 09260 .', {'entities': []}) \n",
      "\n",
      "('during occlusion lasted from 21 min until 30 min . Post start of occlusion .', {'entities': []}) \n",
      "\n",
      "('am splitting cells right now . Number 983  987 and 988 . I will collect the pellets so I can freeze them and extract are in the future', {'entities': []}) \n",
      "\n",
      "('Electrophoresis at Program P4 which is 20 min at 400 Start around at 12:45 .', {'entities': []}) \n",
      "\n",
      "(\"that 1 later . I 'll regenerate it anyway .\", {'entities': []}) \n",
      "\n",
      "('we log into bright lab and navigate to the respective project in the ELN to pick the right experiment  in this case project Tree Top . We opened the experiment and add a new section . We had a name to the section and we selected it to be a formulation . We then hit the end button  which creates new section in the ELN .', {'entities': []}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('is Android latency recording number 3 . This is the 2nd time ', {'entities': []}) \n",
      "\n",
      "(\"will not split the neuroblastoma cells today  they 're still not ready  but they look healthy . In maybe 48 h  they will be ready . Today I will be splitting number 983  987 and 988 .\", {'entities': []}) \n",
      "\n",
      "('2 ml of light aspartate .', {'entities': []}) \n",
      "\n",
      "('29 1.715 E5 .', {'entities': []}) \n",
      "\n",
      "('in batch 3 unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('are dated to the D360 . This reports the data to the chemistry teams .', {'entities': []}) \n",
      "\n",
      "('10 ml of Quant - iT dsDNA HS reagents and 50 µl of the dsDNA HS buffer .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('123456789 ten .', {'entities': []}) \n",
      "\n",
      "(\"[ infected with this place it 's me . I had fresh medium open 5 and over well . All I suppose I said case are dead . I ca n't see I ca n't see the . Pieces are infected . I will harvest them tomorrow in the morning and now I will inspect the beach of Swiss . Only . By type and the Illan Mutant .\", {'entities': []}) \n",
      "\n",
      "('sample completed . I did not find second gonad .', {'entities': []}) \n",
      "\n",
      "('density 2 * 10 to the minus 6 .', {'entities': []}) \n",
      "\n",
      "('50 mL Conicals  one of them is CAGGS Tert and the other 2 are Tert-1 lentivirus . We need to swim the CAGGS turt into 2 Eppendorf tubes and the Tert lent into 5 face down and put in -80 .', {'entities': []}) \n",
      "\n",
      "(\"me to meet Peter at 1:00 o'clock .\", {'entities': []}) \n",
      "\n",
      "('to my city - 4 mices - avocado .', {'entities': []}) \n",
      "\n",
      "('fractions 11  12  13  14', {'entities': []}) \n",
      "\n",
      "('group 3 first .', {'entities': []}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('and RIPK3 were already known as important components of the necroptosis path and were proposed as new targets for ALS therapy .', {'entities': []}) \n",
      "\n",
      "('26 control media added .', {'entities': []}) \n",
      "\n",
      "('strep Tactin Sepharose High performance from GE Healthcare . Lot number 10282530 expiry date 22 08 . Catalog number 28935599 .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('to clean and the bench without antibiotics next 2 weeks .', {'entities': []}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "(' uh  F1 to H4 .', {'entities': []}) \n",
      "\n",
      "('aliquotes of 125 .', {'entities': []}) \n",
      "\n",
      "('plates are incubated at 4 ℃ overnight at 800 RPM .', {'entities': []}) \n",
      "\n",
      "('you remind me I have an appointment at 14 PM ?', {'entities': []}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('are centrifuged at 10000 RCF to try to pellet down the gooey texture of samples .', {'entities': []}) \n",
      "\n",
      "('on top . Plate 1091080520 on bottom .', {'entities': []}) \n",
      "\n",
      "(\"is slightly bending . Why ? What 's that mean that ? Death at 700 . Still no spikes .\", {'entities': []}) \n",
      "\n",
      "('infected 2 days before RFs with RRV wild type  ELAAN  delta plexingDC2 and the double mutant . Now I will check if I can see infected cells .', {'entities': []}) \n",
      "\n",
      "('am going to split neuroblastoma cells and colon carcinoma cells later . I need to remember that I can not split the colon carcinoma cells higher than one to 10 .', {'entities': []}) \n",
      "\n",
      "('4 μl Glycin buffer to each sample tube', {'entities': []}) \n",
      "\n",
      "('cells look healthy after 2 days in culture with 0 arginine .', {'entities': []}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('cell culture dish was has depth of 2 nm and 5 cm in width .', {'entities': []}) \n",
      "\n",
      "('21.64 g preservative 72.05 g pentylene glycol . 14', {'entities': []}) \n",
      "\n",
      "('µl pour 17 into lane 8 .', {'entities': []}) \n",
      "\n",
      "('in the batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('21 . Assay 21 .', {'entities': []}) \n",
      "\n",
      "('T 1.0.0 E5 .', {'entities': []}) \n",
      "\n",
      "(\"for Wednesday  9 o'clock an appointment with webcam mobiles .\", {'entities': []}) \n",
      "\n",
      "('up the syringe with at least 50 % of volume for storage purposes .', {'entities': []}) \n",
      "\n",
      "('added Coomassie at 1:30 PM .', {'entities': []}) \n",
      "\n",
      "('29 273000', {'entities': []}) \n",
      "\n",
      "('timer 20 min .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 24 to 22 ° .', {'entities': []}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "('is 1.712 E 5 . Sample was repeatedly once .', {'entities': []}) \n",
      "\n",
      "('write down the lot number of the 1.5 ml cups .', {'entities': []}) \n",
      "\n",
      "('whether reducing the dispense . Z offset from 45 to 40 improves plate pattern .', {'entities': []}) \n",
      "\n",
      "('non reducing samples for SDS PAGE by adding 6.7 µl of 4 times LDS buffer .', {'entities': []}) \n",
      "\n",
      "('me to meet with Adam at 2 PM .', {'entities': []}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('to higher expression  but still medium intensity for small cell lung cancer 08 study 19154 .', {'entities': []}) \n",
      "\n",
      "('previous decibel reading of 85 . I think decibel was achieved with maximum volume on and 75 % output . According to the behavioral protocol .', {'entities': []}) \n",
      "\n",
      "('weight of the sample is 4.57 mg .', {'entities': []}) \n",
      "\n",
      "('Massspec for the samples according to what is documented in Word . And started another centrifugation to concentrate samples after 10 min about 2 ml was was concentrated and starting around for 30 min .', {'entities': []}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('blocking in 10 % BSA at 2 µl of anti - His HPR in TBST .', {'entities': []}) \n",
      "\n",
      "('5 l of puffer for the 8 K bio report .', {'entities': []}) \n",
      "\n",
      "('the reported concentrations the LLOQ reported was higher  namely 20 pg / ml of LTE 4 .', {'entities': []}) \n",
      "\n",
      "('filtration A280 is 0.24 .', {'entities': []}) \n",
      "\n",
      "('added 50 mM of sodium phosphate into the 250 Mill Beaker .', {'entities': []}) \n",
      "\n",
      "('measured the It was 7.12 . On pH m MRK 100134 .', {'entities': []}) \n",
      "\n",
      "('that next time I need to plate more knockout cells  especially for the work washout studies . So I need to go from 2000 cells per well to 4000 cells per well .', {'entities': []}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('him imaging studies are planned for tomorrow in that car barcode . Also plating of the cells with yellow fluorescent protein is due tomorrow at 12 .', {'entities': []}) \n",
      "\n",
      "('the sample loaded I can now start the program for the size exclusion  which is called 15 0 9 21 6 SEC  320ML position 4  loop 5 .', {'entities': []}) \n",
      "\n",
      "(\"does n't appear to be any significant cell loss when with 45 offset on the aspirate .\", {'entities': []}) \n",
      "\n",
      "('16  1.0.8 E 5', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(\"3rd sample was not included  which means that it 's the flow - through fraction that was not included .\", {'entities': []}) \n",
      "\n",
      "('images for Chip D3 - 2 starting at image number 09344 .', {'entities': []}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('for 1 h. Timer for 1 h. Timer for 1 h. Switch on timer for 1 h.', {'entities': []}) \n",
      "\n",
      "('5 ml of regeneration buffer for the strep tactin column as prepared for BE 002397 - 59 . And let that elute and incubate for over lunch .', {'entities': []}) \n",
      "\n",
      "('detection antibody is added to the plate and left to incubate protected from light on Orbital Shaker at 600 RPM .', {'entities': []}) \n",
      "\n",
      "('reverse transcription  um  first pipet 5.2 μl of reverse transkriptase to controlling tube . No correction', {'entities': []}) \n",
      "\n",
      "('8 combination 3 media added . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('running buffer from 10 times Tris Glycine SDS Buffer BioRad catalog number 1610772 . Prepare 2 l by taking 200 µl of the 10 times concentrated stock and dilute with Milli Q - water .', {'entities': []}) \n",
      "\n",
      "(' this particular chair is black in color  and is set up at an angle of 90 degree with respect to the floor . Out .', {'entities': []}) \n",
      "\n",
      "('full length first round 26 mosquitoes .', {'entities': []}) \n",
      "\n",
      "('record test  123 blah blah .', {'entities': []}) \n",
      "\n",
      "('incubated for 2 min and 35 seconds .', {'entities': []}) \n",
      "\n",
      "('cycles BV 1210 first part .', {'entities': []}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('rigid 49 µl .', {'entities': []}) \n",
      "\n",
      "('CDT solution has to be warmed up to 40 ° to achieve clear color  colorless solution .', {'entities': []}) \n",
      "\n",
      "('test number 3 with German talking in the background . Extraction . The biphasic solution was steered at room temperature for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': []}) \n",
      "\n",
      "('infected one and a half day before RFs with RRV wild type  the AELAAN mutant the Delta Plexin DC 2 mutant and the double mutant . Today I see in all flasks infected cells round about 5 % just the double mutant has too less green cells for changing medium . Means I will change the medium and all other flasks . The Medium and the double mutant I will leave till Monday and then I have a look if I can change the medium or not .', {'entities': []}) \n",
      "\n",
      "('Mass spectrometry did not provide sufficient sensitivity to assess endogenous levels of LTE 4 in plasma or urine at that time .', {'entities': []}) \n",
      "\n",
      "('34  1.740 E5 .', {'entities': []}) \n",
      "\n",
      "('15 ml substrate to large batch .', {'entities': []}) \n",
      "\n",
      "('have to wait longer and maybe I have to repeat the experiment and I have to change the medium earlier after 8 h. So', {'entities': []}) \n",
      "\n",
      "('make titers grow the culture for one colonie overnight in 3 ml . Then in the morning do 2 to 5 ml of 1 in 100 dilution and incubate for 2 h and 15 min . Blueberry . Blueberry', {'entities': []}) \n",
      "\n",
      "('1.2 ml of the buffer with 200 µl of the wash membrane preparations .', {'entities': []}) \n",
      "\n",
      "('20 ng in 4 µl was added to the cDNA first strand', {'entities': []}) \n",
      "\n",
      "('yesterday samples again 10 µl of 4 times SDS were added .', {'entities': []}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('need to order more of 15 ml Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('concentration 5.16 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('dilution of standard from wells A1 to A9 .', {'entities': []}) \n",
      "\n",
      "('me to book to Cuba for tomorrow at 2 PM .', {'entities': []}) \n",
      "\n",
      "('to 6 ratio good .', {'entities': []}) \n",
      "\n",
      "('for 30 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "('timer 5 min .', {'entities': []}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('tissue blocks of 100 cores of different tumours . This is what we take .', {'entities': []}) \n",
      "\n",
      "('8  1.261 E5 .', {'entities': []}) \n",
      "\n",
      "('GFP in the cells was expressed in the extracellular vesicles as expected  I will look at them with the magnification of 40', {'entities': []}) \n",
      "\n",
      "(\"µl of 10  add into agarose gel with 365 ° Most of it the old fashion - that was n't too bad - No  it 's ...\", {'entities': []}) \n",
      "\n",
      "('reaction was stopped by addition of 1 ml of a acetonitrile and extracted with something .', {'entities': []}) \n",
      "\n",
      "('offset Z to 35 causes the reappearance of the plate pattern  which is now visible in the beginning of the plate and towards the end . 35 seems to be the maximum what we should go .', {'entities': []}) \n",
      "\n",
      "('ID 18 S 006163 value 3.705 .', {'entities': []}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('203 . Have the same ATT right side . But . Mutant 2 . Which is the deficient mutant has a higher ATTB . Meaning . Higher . Viral induction . Than the complementary mutant . Blueberry .', {'entities': []}) \n",
      "\n",
      "('[ ? ] create around decrease to -16 mV.', {'entities': []}) \n",
      "\n",
      "('pipette tips 200 µl .', {'entities': []}) \n",
      "\n",
      "('temperature change from 5 ° to 6 ° for experiment 77 .', {'entities': []}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('SCID mice not engrafting as well as NSGs .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated . Are you kidding me ?', {'entities': []}) \n",
      "\n",
      "('1 is unstimulated', {'entities': []}) \n",
      "\n",
      "('the 20 ml of water were added to the white suspension and then steered for 20 min at room temperature .', {'entities': []}) \n",
      "\n",
      "('are frosted on the ice . From 9:40 it will take 10 min to defrost . And then lysis buffer will be added  analysis will be start .', {'entities': []}) \n",
      "\n",
      "('even meeting with at 10 AM . Set a task for that .', {'entities': []}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\"'m thawing my competent cells for 20 seconds .\", {'entities': []}) \n",
      "\n",
      "('ml of water were added to the suspension .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('are KP58058  KP59331 and 110044394 .', {'entities': []}) \n",
      "\n",
      "('Glutathione internal standard  adding 133µl .', {'entities': []}) \n",
      "\n",
      "('28 172000', {'entities': []}) \n",
      "\n",
      "(\"ml to well three - It 's a big well - oh  it did n't go - wait\", {'entities': []}) \n",
      "\n",
      "('reagents QUANTI- blue for 1 plate .', {'entities': []}) \n",
      "\n",
      "('length is 352.12 µm .', {'entities': []}) \n",
      "\n",
      "('1 3.387 E5 .', {'entities': []}) \n",
      "\n",
      "('me in 1 min to get my plate out of the incubator . Take my plate out of the incubator .', {'entities': []}) \n",
      "\n",
      "('me in 1 min incubation of sample one finished .', {'entities': []}) \n",
      "\n",
      "(' [ inaudible ] perfusion  RTG mosquitoes  sugar fat  I perfused 11 mosquitoes in total', {'entities': []}) \n",
      "\n",
      "('it quick widely reported to monitor week antigen responses or detect changing in low responding patients it may be a good idea to add or more target cells per well .', {'entities': []}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('502 row A column 1', {'entities': []}) \n",
      "\n",
      "('will use 7 µl of 290 ng/µl plasmid for the acinetobacter .', {'entities': []}) \n",
      "\n",
      "('- 1 chip images  ending on 09287 .', {'entities': []}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('me to buy 100 pipettes .', {'entities': []}) \n",
      "\n",
      "('QPCR . Check primers for F1 . According to at sight and integrase . Primers for G2 are OK . Primers for G1 . Check . Right hand side . At B should be fine . Primers for Fells 2 . Should be fine ? Double check to integrases to the right . Blueberry .', {'entities': []}) \n",
      "\n",
      "('volume for TCEP to get a 1 molar solution is 11.2 ml at 5 ml  then adjust pH to reach final volume .', {'entities': []}) \n",
      "\n",
      "('reaction was performed Asp 3346 - 74 .', {'entities': []}) \n",
      "\n",
      "('to buffer 3 times onto the biocore 8 K', {'entities': []}) \n",
      "\n",
      "('the last note to experiment 31 .', {'entities': []}) \n",
      "\n",
      "('fluoride stock was created using 3.72 g of potassium fluoride . In 15 ml of Milli Q water .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('block F 00105613  Block 2 F00105722  Block 3 F00106304  Block 4 F0010767  Block 5 F00104997  Block 6 F00105782  Block 7 F00106794  Block 8 F00107977  Block 9 F00107235  Block 10 F00106037 and Block 11 F00108363 . Porcupine . OK .', {'entities': []}) \n",
      "\n",
      "('6 custom class 3 media added .', {'entities': []}) \n",
      "\n",
      "('test  9 no headphones  very loud lab associate in the lab . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into separation funnel . New paragraph . Phase cut is clear and fast . There [ In German : \" . Wrong New test .', {'entities': []}) \n",
      "\n",
      "('me to book a flight on November 3rd .', {'entities': []}) \n",
      "\n",
      "('a temperature change from 5 ° to 6 ° on temperature . Oh f * * *', {'entities': []}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('feed the surface of any kind of searcher area with R 2000 to ensure cleanliness . I will let this incubate for at least 15 minutes and then clean it off with ethanol . Another try .', {'entities': []}) \n",
      "\n",
      "('size L 2 boxes  gloves size M 3 boxes  Correction 4 boxes  gloves size S 2 boxes .', {'entities': []}) \n",
      "\n",
      "('want to record something with the neuralinks file 5 .', {'entities': []}) \n",
      "\n",
      "('past 10 . And I need to go home .', {'entities': []}) \n",
      "\n",
      "('prepare . I produced the cassette for mScarlet with Strep - tag by PCR with Fusion and Dream taq . I used different conditions for the Fusion . I used a gradient from 65 to 72 ° . And for Dream taq I used 68 ° . I did a hot start means I added the second primer at 95 ° in the Fusion mix and in the Dream taq mix I added the taq polymerase at 98 ° .', {'entities': []}) \n",
      "\n",
      "('protein batch PB190843 .', {'entities': []}) \n",
      "\n",
      "('29  2.575 E 5', {'entities': []}) \n",
      "\n",
      "('me to call Sarah at 5 PM today .', {'entities': []}) \n",
      "\n",
      "('a 100 ml easy', {'entities': []}) \n",
      "\n",
      "(\"the scs samples are diluted now with 40 µl LDS buffer . From that I 'm injecting 3 . In Lane  3 . 4 µl of bovine serum albumine In Lane . 5 four µl of the human serum albumine .\", {'entities': []}) \n",
      "\n",
      "('in the 25 setting reveals or confirms cell loss and also cell death . Very few cells remaining looking very unhappy .', {'entities': []}) \n",
      "\n",
      "('23  1.426 E 5', {'entities': []}) \n",
      "\n",
      "('me to order  small 150 ml Media Models on 24th august .', {'entities': []}) \n",
      "\n",
      "('A column 1 is 500 .', {'entities': []}) \n",
      "\n",
      "('3 pipette 10 µl Trition', {'entities': []}) \n",
      "\n",
      "('added at 2:11 .', {'entities': []}) \n",
      "\n",
      "('a temperature change of 21 to 22 ℃ .', {'entities': []}) \n",
      "\n",
      "(\"numbers 694  696  855  856  857 and GFP  which I do n't know the plasma number by heart . Also prepare 30 µl for HPLC . In HPLC tubes make sure that there 's no air bubble at the bottom  and measure UV - VIS as well . Leave the SDS samples on the bench for 5 min do not use cooking heating .\", {'entities': []}) \n",
      "\n",
      "('me to meet at 4 PM .', {'entities': []}) \n",
      "\n",
      "(\"infected with M Scarlet 65 . The cells are infected  but I ca n't change the medium because too much cells are infected so I have to add a little bit more fresh medium .\", {'entities': []}) \n",
      "\n",
      "('BLIMP 1 . BLIMP 1', {'entities': []}) \n",
      "\n",
      "('purification replicate 3  there again samples that are missing beads . And I again did not notice that I collected them in the P - tip [ inaudible ] So my samples number 374  375  382 and 394 are the palest once . Then also sample 389 is not very dark either .', {'entities': []}) \n",
      "\n",
      "('cell loss was noticed at 55 aspirate offset reducing the aspirate offset to 45 . Executing .', {'entities': []}) \n",
      "\n",
      "('pump in MXBI 5 is slower than MXBI 3 .', {'entities': []}) \n",
      "\n",
      "('number 2 has a much bigger palette than sample number 1 .', {'entities': []}) \n",
      "\n",
      "('me to image the blot in 20 min .', {'entities': []}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('46 161000', {'entities': []}) \n",
      "\n",
      "('was then concentrated to 0.87 mg / ml with 500 .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('up resistance 2.3 .', {'entities': []}) \n",
      "\n",
      "(\"transformation should work with genomic DNA . In the selection has to be with kanamycin 10 because once incorporated into the genome it 's a single copy number . Blueberry .\", {'entities': []}) \n",
      "\n",
      "('with Mercedes in GL51 .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('is methotrexate .', {'entities': []}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('the remainder of the lysates on the column . Add 5 ml / min .', {'entities': []}) \n",
      "\n",
      "('27  1.551 E 5', {'entities': []}) \n",
      "\n",
      "('is RH exon 5 .', {'entities': []}) \n",
      "\n",
      "('ID 18s0075 3.75 ng per µl . Sample ID 19s0178 . Sample value 3.57 .', {'entities': []}) \n",
      "\n",
      "('17 1.278 E5 .', {'entities': []}) \n",
      "\n",
      "('a timer for 30 min blueberry .', {'entities': []}) \n",
      "\n",
      "('aspirate to 25 .', {'entities': []}) \n",
      "\n",
      "('to change BV711 CD14 .', {'entities': []}) \n",
      "\n",
      "('low amounts of samples in 74 .', {'entities': []}) \n",
      "\n",
      "('serum albumin Sigma A6272  human serum albumin Sigma A9511', {'entities': []}) \n",
      "\n",
      "('3850 - 1 : human CD 45.20  CD 19 0  CD 10 0  CD 34 0.04  CD 22 .02 .', {'entities': []}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "(\"HU1678 EXP 1 . C = 34 Series resistance was n't really stable . L type current was okia the initial point but because of the force from the perfusion . The seal was n't stable anymore and caffeine was attempted twice  but they decay was not in sync with the decay of the caffeine induced Calcium Transient . However  this should be considered during analysis .\", {'entities': []}) \n",
      "\n",
      "(\"preparing the samples added by accident  7.7 µl of reducing agent instead of 3.1 . Ca n't correct it for the oxidizing samples I 'll add 7.7 µl of water instead then .\", {'entities': []}) \n",
      "\n",
      "('platform on it . Start surgery . 9:50 .', {'entities': []}) \n",
      "\n",
      "(\"'m using Eppendorf 346816 .\", {'entities': []}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('lost about 1/3 of the cell dissociation .', {'entities': []}) \n",
      "\n",
      "('aliquotted 60 µl into the Eppendorf tube and then centrifuged it for 30 seconds .', {'entities': []}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('plate at 10:30 .', {'entities': []}) \n",
      "\n",
      "('example  it could then reach back what you have done  you could have a 100 % accuracy mode where it automatically records it back .', {'entities': []}) \n",
      "\n",
      "(\"'s try another one . Can you tell me the difference between one hundred double M NAOH and 200 double M NAOH ?\", {'entities': []}) \n",
      "\n",
      "('me to go to the stakeholder event on March 27th .', {'entities': []}) \n",
      "\n",
      "('the 2 times solubilization buffer 0.57 ml 0\\\\.7 molar TCEP is added .', {'entities': []}) \n",
      "\n",
      "('a dozen six packs of sterile filter bottles approximately 250 ml and a box of tips no longer in use .', {'entities': []}) \n",
      "\n",
      "('28 2.03 times X to the 5 .', {'entities': []}) \n",
      "\n",
      "(\"'s have a look with the distance from 2 m is it also work perfect .\", {'entities': []}) \n",
      "\n",
      "('purge prime with 50 µl of prime and 50 µl of purge .', {'entities': []}) \n",
      "\n",
      "('22 1.212 E5 .', {'entities': []}) \n",
      "\n",
      "('1.25 ml Triton X-100 to lysis buffer . Triton X-100 Sigma Lot number SLBB0703V.', {'entities': []}) \n",
      "\n",
      "('45 9.314 E4 .', {'entities': []}) \n",
      "\n",
      "('pump still is inconsistent dropping stops once in awhile . Previous experience is that a flow is about half of the anticipated flow . Check up after 10 min .', {'entities': []}) \n",
      "\n",
      "('16 min re - perfusion some arrhythmias with bundle branch block .', {'entities': []}) \n",
      "\n",
      "('remind me to do a retrafo from  of AX 137 .', {'entities': []}) \n",
      "\n",
      "('28  2.0.8 E5 .', {'entities': []}) \n",
      "\n",
      "('the slack channel integration number 3 .', {'entities': []}) \n",
      "\n",
      "('chair is black in color  and is set up at an angle of 90 degrees with respect to the floor .', {'entities': []}) \n",
      "\n",
      "('g of the crude were put in a round bottom flask and diluted with 20 ml water .', {'entities': []}) \n",
      "\n",
      "('me to buy new book tomorrow at 3:00 PM .', {'entities': []}) \n",
      "\n",
      "('put those lenti viral aliquots in my lenti aliquots box in the -80 near the bottom of the box . Near 7 tubes .', {'entities': []}) \n",
      "\n",
      "('27th . New mini preps .', {'entities': []}) \n",
      "\n",
      "(\"approximately 3 o'clock we began the first experiments .\", {'entities': []}) \n",
      "\n",
      "('1 . L2 characterization', {'entities': []}) \n",
      "\n",
      "('will harvest the B cells after 48 h.', {'entities': []}) \n",
      "\n",
      "('PCR of KSHV GB Delta RGD number 3 and number 12 .', {'entities': []}) \n",
      "\n",
      "('18  1.223 E 5', {'entities': []}) \n",
      "\n",
      "('123 . This is a test recording . Remind me to take out the samples out of the fridge . In 2 weeks . At exactly 2 PM .', {'entities': []}) \n",
      "\n",
      "('was 500 µl corrected', {'entities': []}) \n",
      "\n",
      "('hepatocyte culture approximately 50 % confluent as of September 2nd .', {'entities': []}) \n",
      "\n",
      "('AmMag beads from 20 % ethanol  25 % slurry  2 times in Milli Q water using the Magic rack maxi from GE Healthcare .', {'entities': []}) \n",
      "\n",
      "('if cells are 3 rows down .', {'entities': []}) \n",
      "\n",
      "('Lidocaine bolus at 12:45 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('me at 6 PM to stain my antibodies .', {'entities': []}) \n",
      "\n",
      "(\"opening the chest at 12 O'clock .\", {'entities': []}) \n",
      "\n",
      "('need to be able to follow grant rules data storage for about 2 or 3 years .', {'entities': []}) \n",
      "\n",
      "('name  um the plates on the envision plate reader click the three dots under output settings and select plate barcode  then write dot CSV . And it should be recorded exported in plate format 1 without any information and in each plate to a separate file . Then I \\'m going to add this \"  but we \\'ll see later on whether that would work . Press RUN', {'entities': []}) \n",
      "\n",
      "('me in 1 min to get the plate out of the cell incubator . Get the plates out of the cell incubator .', {'entities': []}) \n",
      "\n",
      "('injections in infarcted area .', {'entities': []}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('ID 18S00064 . 3.75 ng/µl . Sample ID A19S00064 3.75 µl .', {'entities': []}) \n",
      "\n",
      "('such project or the phenotypic screen in collaboration with the University of Lisbon to find novel targets and compounds which inhibit necroptosis .', {'entities': []}) \n",
      "\n",
      "('plates for - 80 freezer from the QC dilution experiment .', {'entities': []}) \n",
      "\n",
      "('sodium hydroxide for beads cleaning', {'entities': []}) \n",
      "\n",
      "('D5 is in the oven for 20 min .', {'entities': []}) \n",
      "\n",
      "('normal headphones normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('5 ml of DMA substance .', {'entities': []}) \n",
      "\n",
      "('6 plate on the cooling platform and turn on the cooling platform .', {'entities': []}) \n",
      "\n",
      "('5 min incubation finished .', {'entities': []}) \n",
      "\n",
      "('250 µl of Phosphate Buffered Saline to tubes 1 through 12', {'entities': []}) \n",
      "\n",
      "('me to check the E.coli cells tonight at 9:00 PM . Avocado .', {'entities': []}) \n",
      "\n",
      "('µl each library made all 8 lanes .', {'entities': []}) \n",
      "\n",
      "('was measured  calibrated to with solutions 4 and 1 .', {'entities': []}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('PB 1900088 : no inflection temperature detected  for PB 1900089 : 61.1  PB1900090 : 54.4  PB1900091 : 59.2  PB1900092 GFP : 57.4 and 85.4 and for PB1994 : 58 ° .', {'entities': []}) \n",
      "\n",
      "('50 mg of AZ D1234 .', {'entities': []}) \n",
      "\n",
      "('D2 - 2 images ending on 09383 also that larger dimple might not be as simple itself . It actually might be some liquid .', {'entities': []}) \n",
      "\n",
      "('crystals in plate 51 and 52 .', {'entities': []}) \n",
      "\n",
      "('appear to have been diluted to 20 µM.', {'entities': []}) \n",
      "\n",
      "('elution from the Strep - Tactin column for experiment 67 with an additional 2 ml of Apo elution buffer .', {'entities': []}) \n",
      "\n",
      "('for 30 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "('sample for Troponin 11:20  1 h post MI .', {'entities': []}) \n",
      "\n",
      "('25 8.509x10 to the 4 or 85000 .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('were started at 7:03PM .', {'entities': []}) \n",
      "\n",
      "('0.54 ml of 1 molar Tris pH 8 to the 27 ml  and filtered sample through a supor .8.2 ml filter .', {'entities': []}) \n",
      "\n",
      "('put 6 250 ml bottles in the fridge for cooling .', {'entities': []}) \n",
      "\n",
      "('. Dilute all overnight salmonella cultures one in a hundred and dispense in 96 4-well  but only 150 µl per well for aeration .', {'entities': []}) \n",
      "\n",
      "('to test number 3  microphone was hold close to the mouth during talking .', {'entities': []}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('pipette tips .', {'entities': []}) \n",
      "\n",
      "('5 µl of IPA to 25 µl of water .', {'entities': []}) \n",
      "\n",
      "('of the following section 1 10  11  12  13  14 and 15 .', {'entities': []}) \n",
      "\n",
      "('the 1 h observation  she used LabTwin three times to note results of the microscopy check checking timers this is for incubating the cells . This is for 15 min and also setting another time when she split the cells after another 10 min . She lost the 2nd timer .', {'entities': []}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('E9 to E12 and F12 and F11 .', {'entities': []}) \n",
      "\n",
      "('qPCR . Dilute DNA samples to run a standard curve . Other DNA gradient . Starting at 50 ng at 4 different DNA concentrations .', {'entities': []}) \n",
      "\n",
      "('me to pipet 30 µl .', {'entities': []}) \n",
      "\n",
      "(\"also upload a word Doc and the echo success report and the scanning barcodes . This takes up around 20 % of my write up . I tend to split my time 60 % into the lab and 40 % in the office  but this varies depending if there 's extra troubleshooting to do .\", {'entities': []}) \n",
      "\n",
      "('me to call tomorrow at 2 PM .', {'entities': []}) \n",
      "\n",
      "('the cloning of rab 4a  rab 5 and rab 7 I used 100 ng of vector and 37.5 ng of insert . I used the quick ligase . The master mix . Or the mix . Volume was 20 μl . I incubated for 15 minutes at room temperature . The transformation . For the transformation I used Stable 3 .', {'entities': []}) \n",
      "\n",
      "('14B is also added to wells A11 and A12 .', {'entities': []}) \n",
      "\n",
      "('35 the pattern actually increases . So maybe we should try increasing that . We should try and increasing the dispense .', {'entities': []}) \n",
      "\n",
      "('P4 tube number 20 pool E10  E11  E12  F12  E9 .', {'entities': []}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('the Q Standards are added to the end of the plates starting with Q1 for sample 35  ending at Q5 for sample 39 and then the sample 40 will be the Q equivalent of the blank .', {'entities': []}) \n",
      "\n",
      "('started the experiment EP 0023 and the Qubit assay .', {'entities': []}) \n",
      "\n",
      "('# 3 for the Slack channel .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('activation after 3 min and 30 seconds .', {'entities': []}) \n",
      "\n",
      "('order 20 tablets .', {'entities': []}) \n",
      "\n",
      "('ml tubes one box .', {'entities': []}) \n",
      "\n",
      "('recording some notes 123 .', {'entities': []}) \n",
      "\n",
      "('takes me 3 h to prep the assay .', {'entities': []}) \n",
      "\n",
      "(\"here the IMAC medium for the dissections . It carries  um  6 times  6 parts of Schneider 's medium  3 parts of citrate buffer and one part of FBS . So basically 6 ml of Schneider 's  3 ml of citrate and 1 ml of FBS . Later everything is mixed and filtered through the filter 22 µm filter  so it 's sterile . So I made that and I 'm filling the tubes that I prepare yesterday with 30 μl of this IMAC medium . So once I dissect tissues and the cells they have something to  um  stay  stay alive .\", {'entities': []}) \n",
      "\n",
      "('start . Timer until 2:30 .', {'entities': []}) \n",
      "\n",
      "('thawed my competent cells for 20 seconds .', {'entities': []}) \n",
      "\n",
      "(\"'re doing colony PCR again with 6 colonies . So colony with aqua cloning fragment and 5 random chose colony of the Gibson Assembly . And we are putting in the master mix for the colony PCR 16 µl buffer  3.2 µl dNTPs  3.2 µl of the primer 3 and 3.2 µl of the primer 4 and 270 7.2 µl water .\", {'entities': []}) \n",
      "\n",
      "('plasma sample from the pig was pulled from 2 vials with 2 ml in each vial . Making 4 milliliters of sample in total . In the freezer remaining 1 vial of approximately 2 ml .', {'entities': []}) \n",
      "\n",
      "('yellow suspension was steered for 30 min at room temperature .', {'entities': []}) \n",
      "\n",
      "('imaging at 2:00 PM .', {'entities': []}) \n",
      "\n",
      "(\"April 4 o'clock reminder .\", {'entities': []}) \n",
      "\n",
      "('315 µl of dpx1 .', {'entities': []}) \n",
      "\n",
      "('me to call Sarah at 2 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('run 2 12 % SDS gel for the samples 1 for staining 1 for western blotting .', {'entities': []}) \n",
      "\n",
      "('started the experiment EP 002 .', {'entities': []}) \n",
      "\n",
      "(\"think it 's it cut off  too early for that 1 .\", {'entities': []}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('in 7 minute countdown ?', {'entities': []}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('less than 2 min .', {'entities': []}) \n",
      "\n",
      "('note down a change of time and temperature 24 ° to 27 ° across a range of different medicines testing across 1 to 4 . Para leases of a seismic cuboid .', {'entities': []}) \n",
      "\n",
      "('15- 2 images ending on 09315 and chip D4 - 2 is starting on number 09316 .', {'entities': []}) \n",
      "\n",
      "('a note to organize the scope room on Friday the 9th of August .', {'entities': []}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test 5 . Radio running English talking besides me  microphone close to the mouth . New paragraph extraction . The biphasic solution was steered at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast point the clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('30 pattern starts to appear . Uh . But is much better than what it used to be at the previous settings .', {'entities': []}) \n",
      "\n",
      "('4  close to the radio with English song running  phone in the lab coat pockets . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbars to give . Test end .', {'entities': []}) \n",
      "\n",
      "('24  1.905 E5 .', {'entities': []}) \n",
      "\n",
      "('round 5T4 full length 22 mosquitoes .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('ml syringes 2 boxes  Green towels 2 boxes .', {'entities': []}) \n",
      "\n",
      "('protocol step 2 .', {'entities': []}) \n",
      "\n",
      "('Strep - Tactin with 1 ml of homogenization buffer .', {'entities': []}) \n",
      "\n",
      "('mNeon of 65 Delta 21 to 27 mutant 0.75 DNA negative and A4 the same sample 1.5 . µg DNA negative also .', {'entities': []}) \n",
      "\n",
      "(\"'s a massive need to centralize all information . At the min I 'm using 1 note .\", {'entities': []}) \n",
      "\n",
      "('mixed TEA buffer with 1 % agarose . Avocado .', {'entities': []}) \n",
      "\n",
      "('to check in Sample 216 already has been DNA extracted .', {'entities': []}) \n",
      "\n",
      "('30 1.496 E5 .', {'entities': []}) \n",
      "\n",
      "('wild type strain for T4 phage is called G5OG1250 . Blueberry', {'entities': []}) \n",
      "\n",
      "('resuspended in 20 ml cell media .', {'entities': []}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('49  positon G1  2 small droplets .', {'entities': []}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('was 500 µl .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('is Android latency recording test 1 . I am in M2 .', {'entities': []}) \n",
      "\n",
      "('a temperature change of 23 ° to 27 ° .', {'entities': []}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('ID 18S006290 sample value 3.705 ng/µl . Sample ID 19S00646 sample value 2.00 ng/µl . sample ID 18S006286 sample value 16.80 ng/µl .', {'entities': []}) \n",
      "\n",
      "('I will work up blood creatinine provided by Vandana Veenit . The project code in my time is pre - TSID and the bio ELN Project Code is RTIDN . This is blood 10 µl in each well in 96 well plates  and there are 4 plates altogether for the study . I will follow the protocol as attached in bio ELN and my bio ELN entry is BE002667 - 20 .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('load the reagent in the cassette in the proper order . Then place the samples as described on the menu depending upon the number of samples and then we will select the program with their desired final volume for RNA which can be modified up - to 50 µl . And then select the RNA Easy Plus and start the program by pressing next .... The critical thing is to load the reagents  samples  DNA  and RNA isolation columns  and the collection 2 in the same order as a sample and then start the robot for the 12th sample . It will take about an hour for 12 samples .', {'entities': []}) \n",
      "\n",
      "(\"flow seems to be OK now . There 's no pressure variation and it looks like the actual volume being delivered by the sample pump is correct . This is being for last  I guess 200 ml  so samples being pumped at 10 ml / min onto the column .\", {'entities': []}) \n",
      "\n",
      "('an order for 17 tube samples .', {'entities': []}) \n",
      "\n",
      "('concentration is 3.2 molar .', {'entities': []}) \n",
      "\n",
      "('- 3 images start on 09424 .', {'entities': []}) \n",
      "\n",
      "('point letter . 2 point PTX .', {'entities': []}) \n",
      "\n",
      "('2 is done for incubation overnights .', {'entities': []}) \n",
      "\n",
      "('µl marker . Lane  2  empty . Lane 3 . 2 µl Bovine Serum albumin .', {'entities': []}) \n",
      "\n",
      "('remind me that there is a lab meeting today at 5 PM  please . Thank you .', {'entities': []}) \n",
      "\n",
      "('timer for 10 min on centrifuge .', {'entities': []}) \n",
      "\n",
      "('with buffer aA for 5 ml / min . Uh .', {'entities': []}) \n",
      "\n",
      "('using the geldoc EZ imager from biogas using the stain free tray . With medium bands . And 2  1/2 min of activation . Saved under protein geldoc . Relaxin .', {'entities': []}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('105 µl X AMP into lane 2 .', {'entities': []}) \n",
      "\n",
      "('is 5 . B2 is 0.16 .', {'entities': []}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('D3 - 1 images starting on 09227 .', {'entities': []}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('with normal headphones difficult words . 10 mbar  25 ml  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': []}) \n",
      "\n",
      "('to buy some pipette tips 200 µl .', {'entities': []}) \n",
      "\n",
      "('culture yesterday  I expand RFs to T75 175 Flasks 40 plus .', {'entities': []}) \n",
      "\n",
      "('bar codes 1090950954  0947093009230916 .', {'entities': []}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('of master mix pipette 12.5 μl [', {'entities': []}) \n",
      "\n",
      "('- 17 - slide 85 . Sciatic nerve . Within normal limits .', {'entities': []}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('2 Apo samples together and concentrate a bit more to be able to have 1 size exclusion .', {'entities': []}) \n",
      "\n",
      "('just pelleted the cells . And as usual  the pellet from 983 is a little bit bigger . I suppose I have more cells there .', {'entities': []}) \n",
      "\n",
      "('30 1.307 E5 .', {'entities': []}) \n",
      "\n",
      "(\"me to order filter tips . Looks like we 're low on 1000 μl tips and 200 μl tips . The catalog numbers for both are as follows . So the 1000 μl tips catalouge number is 02- 707 - 40400 and the catalogue number for the 200 is 02 - 707 - 430 . And these are Fisher brand sure one . Aerosol barrier pipette tips .\", {'entities': []}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('for 36 min .', {'entities': []}) \n",
      "\n",
      "('12 with chatting in the background  not too loud  no headphones in . Extraction . The biphasic solution was stirred at room temp for 20 min and transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the rotor bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('was aliquotted with a dilution of 1 in a 100 with 100 µl aliquots .', {'entities': []}) \n",
      "\n",
      "(\"I 'm creating a 3rd note .\", {'entities': []}) \n",
      "\n",
      "('13  7.431 E4 .', {'entities': []}) \n",
      "\n",
      "('1  used 100 ng per sample  job 2 use about 300 . Scratch that 3 to 400 ng/µl . [ ? ]', {'entities': []}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('have been placed in 37 ° to incubate overnight .', {'entities': []}) \n",
      "\n",
      "('timer 2 min .', {'entities': []}) \n",
      "\n",
      "(' lower amount of samples in 69 and 71 and 72 .', {'entities': []}) \n",
      "\n",
      "('1068248169  349.31 g / mole . 2.2 mig .', {'entities': []}) \n",
      "\n",
      "('is 2.570 E 5 .', {'entities': []}) \n",
      "\n",
      "('sample for blood gases 10  12 .', {'entities': []}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('20 143000 .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are very likely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('me in 10 seconds to complete my ELN findings . To complete my ELN findings .', {'entities': []}) \n",
      "\n",
      "(\"'m going to image the calbindin slices from 0305 mill .\", {'entities': []}) \n",
      "\n",
      "('pooling an concentration of the 2 Apo samples the Nanodrop shows an absorption at 280 nm of 0.35 . A 15 µl sample .', {'entities': []}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "(\"'s doing a big study is a washout study  she incubates on . 10 µl removing the comb\", {'entities': []}) \n",
      "\n",
      "('between 20 and 30 mV.', {'entities': []}) \n",
      "\n",
      "('and 6 210 µl water .', {'entities': []}) \n",
      "\n",
      "('3.5 mg of heavy labeled citrate and add that to the heavy labeled standard solutions .', {'entities': []}) \n",
      "\n",
      "('that for the 15 nm and 200 nm particles Jose first has to spin code them to add them .', {'entities': []}) \n",
      "\n",
      "('DNA is 000456 ng / ml and the RNA is about 1478.6 . pg / ml .', {'entities': []}) \n",
      "\n",
      "('# 11 is 23', {'entities': []}) \n",
      "\n",
      "('second sound oil wars 0.045 .', {'entities': []}) \n",
      "\n",
      "('volume of dispense solution is 13 ml of the stock solution .', {'entities': []}) \n",
      "\n",
      "('me to check LC MS at 1:00 PM .', {'entities': []}) \n",
      "\n",
      "('number 245322458 is [ Cree ? ] samples . Commandos .', {'entities': []}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('µl into lane 2 .', {'entities': []}) \n",
      "\n",
      "('something from 1 bottle to another .', {'entities': []}) \n",
      "\n",
      "('165 µl dpx3 .', {'entities': []}) \n",
      "\n",
      "('15  1.964 E 5', {'entities': []}) \n",
      "\n",
      "(\"1 . Cm = R = 2.2 . The recording was was OK with god current and transient . However it appeared as if the potassium current was n't i.e 4AP did not working properly . I also noticed same effect in the isoprenaline measurement . All parameters were measured including Isoprenaline effect as well as caffeine in the presence and absence of Isoprenaline .\", {'entities': []}) \n",
      "\n",
      "('10:30 started centrifugation at RPM for 30 min in TI-45 rotor at 4 ° .', {'entities': []}) \n",
      "\n",
      "('5.7 μls of master mix [ inaudible ] to each sample .', {'entities': []}) \n",
      "\n",
      "('with headphones in the lab radio in the background . Difficult words . 10 mbar 25 ml 25 µl 50 bar 50 mg 15 mg 12 mbar 70 mbar 17 mg 17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "(\"SG is connected to pin number  number 5 . The next pin is printer PG . And PG is connected to . Nothing apparently . Nothing apparently um OK  the bias is . 12 V and 12 V is connected to . Here we have it 's pin 4 . And Com is connected to . Bias com . That is this connected to a pin number one . OK and now we have the ground shielding . And the ground shielding . Is weirdly not connected to the metal connector . That 's odd . Yea  pin PG is not connected . Pin printer PG is not connected .\", {'entities': []}) \n",
      "\n",
      "('ID 77 also added to H9 .', {'entities': []}) \n",
      "\n",
      "('the dispense seems to leave some dry wells and some pretty wet wells . Exacerbating a pattern and also potentially leading to cell loss . Still increasing it to 35 to check with that was just too of an extreme of a value .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I HFFs one in three and plated 8 6 wells with 300000 cells per well for transduction . I SLKs one in ten T75 flask and plated one 48 well plate with 50000 cells per well . I 293Ts one in ten T75 flasks', {'entities': []}) \n",
      "\n",
      "('me tomorrow that I need to order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('flexible . 55 µl .', {'entities': []}) \n",
      "\n",
      "('330 neural - D1 model LU5223 . Low no expression level .', {'entities': []}) \n",
      "\n",
      "('27 not counted . Three failed attempts .', {'entities': []}) \n",
      "\n",
      "('35 364000', {'entities': []}) \n",
      "\n",
      "('120 ml 5 molar sodium chloride .', {'entities': []}) \n",
      "\n",
      "('me to make the test for the product 2001 tomorrow at 9:00 AM .', {'entities': []}) \n",
      "\n",
      "('reducing the dispense setup of Z offset does not appear to reduce the run um . Increasing it back to original value of 45 .', {'entities': []}) \n",
      "\n",
      "('HisTrap . Fast flow crude 5 ml  which has been stripped and re loaded IM 190716 . Ah .', {'entities': []}) \n",
      "\n",
      "('the Cas 9 solution will be 0.1 ug/µl and trypsin will be added in a ratio of 1 to 20 to the samples . And the trypsin is resuspended in the trypsin resuspension buffer to 1 ug/µl before addition .', {'entities': []}) \n",
      "\n",
      "('15 ml 20 mm 20 µm . Part number 120 - 1036/ Biotage control number 19290001 XSP .', {'entities': []}) \n",
      "\n",
      "('a beaker an add 50 mL of methanol .', {'entities': []}) \n",
      "\n",
      "('test jabrar next to the radio German . Extraction . The biphasic solution will still at room temp for 20 min and then transfer it into a separation funnel the face cut is clear and fast . The clear yellow solution was dried on the road to balk at 50 ℃ and 10 mbar to give . Test ends dragonfly .', {'entities': []}) \n",
      "\n",
      "('me to order AP 2 .', {'entities': []}) \n",
      "\n",
      "('is standard . A2 is sample .', {'entities': []}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('starting volume of the bacteria 24 mL.', {'entities': []}) \n",
      "\n",
      "('10 µl 1 molar TCEP pH 7 to the solubilization buffer . Add the 3 ml homogenate to solubilization buffer . Add 1', {'entities': []}) \n",
      "\n",
      "('accessing legacy data the D360 is more historical .', {'entities': []}) \n",
      "\n",
      "('timer for 3 min .', {'entities': []}) \n",
      "\n",
      "('is bending 350 . Finding a different spot .', {'entities': []}) \n",
      "\n",
      "('sample ID is 18vS006290 and the sample value is 3.705 .', {'entities': []}) \n",
      "\n",
      "('the 27th do mini preps .', {'entities': []}) \n",
      "\n",
      "('of centrifuged at g for 15 min at 4 ° .', {'entities': []}) \n",
      "\n",
      "('4 7.0.7 E4 .', {'entities': []}) \n",
      "\n",
      "('need to repeat sample 3 . So good', {'entities': []}) \n",
      "\n",
      "('me tomorrow to summarize the poster presentation talk before 10 AM .', {'entities': []}) \n",
      "\n",
      "(\"Wednesday  the 27th at 9 o'clock . I have a meeting with Paul Charlton .\", {'entities': []}) \n",
      "\n",
      "('600 ml imidazole to the elution buffer to reach 300 million molar [ ? ]', {'entities': []}) \n",
      "\n",
      "('31  6.927 E 4', {'entities': []}) \n",
      "\n",
      "('a BioRad TGX Stain free gel with BioRad sample . The precision plus protein unstained standards . Applying 5 µl of that and then applying a as much sample as possible like 25 µl .', {'entities': []}) \n",
      "\n",
      "('an order for 5 cell counts .', {'entities': []}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('reperfusion 12:53 from 9 atmospheres to 0  and withdrawing the balloon .', {'entities': []}) \n",
      "\n",
      "(\"this is the MI surgery team . It 's  Ida  and Anna . We prepared the surgery . Introducer in a.carotis and it went without any remarks . Then we put it in the balloon with a guide and it went very well . the vessels in the heart were very thin and we could n't see any branches from the LAD except the first 1  that was rather big . The circumflex was very thin  but very much branches . But we could place the balloon in a good way and we think that we got the second branch . We are not sure because we could n't see it in the X - Ray . But we we think we noticed it when we went down with the guide . And now we have had the occlusion for 1 h and 50 min and we had some arrhythmias around 23 to33 min after occlusion and after that  it has been . Rather . OK  OK and not so much noise from the ECG .\", {'entities': []}) \n",
      "\n",
      "(\"1 . Yeah  there is a voice flow . I 'm not 100 % sure how it 's going to be like .\", {'entities': []}) \n",
      "\n",
      "('of new gel filtration column . Wiz gel filtration market for protein of molecular weights of to  Dalton from Sigma Aldrich .', {'entities': []}) \n",
      "\n",
      "(\" she runs 6 assays per week or it 's usually around 4 on average .\", {'entities': []}) \n",
      "\n",
      "('me to order WENT 3E for stimulation experiments .', {'entities': []}) \n",
      "\n",
      "('of trypsin is 2027191 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "('upon a time  there was a little Princess . She was only 10 years old . But she was really tall .', {'entities': []}) \n",
      "\n",
      "('48  6.427 E4', {'entities': []}) \n",
      "\n",
      "('me to call Sarah at 2 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('me of a meeting at 6 PM this evening .', {'entities': []}) \n",
      "\n",
      "('free DNA was isolating using the MagMAX Cell - Free DNA isolation Kit catalog number A29319 .', {'entities': []}) \n",
      "\n",
      "('have 3 50 Mil Conicals CAGGS tert and 2 that are tert1 lenti . I am going to split into one CAGGS tert actually 2 CAGGS tert lenti Eppendorf and 5 of the Tert1 lenti Eppendorfs .', {'entities': []}) \n",
      "\n",
      "('was added at 2:35 .', {'entities': []}) \n",
      "\n",
      "('in batch 6 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('in batch 3 are unlikely to be contaminated .', {'entities': []}) \n",
      "\n",
      "('ml . 25 µl . 30 bar .', {'entities': []}) \n",
      "\n",
      "('me to warm cells at 4:00 PM 10 min .', {'entities': []}) \n",
      "\n",
      "('initial denaturation time to 4 min . Change small denaturation time to 22 seconds . Change extension annealing time to 45 seconds . Blueberry', {'entities': []}) \n",
      "\n",
      "(\". A1 is 200 µg . I could n't say .\", {'entities': []}) \n",
      "\n",
      "('12  6.650 E 4', {'entities': []}) \n",
      "\n",
      "('8:30 take out the strep tactin incubation  centrifuge 5 min at 500 Gs to pellet the resin and then D count the supernatant .', {'entities': []}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('to 12 ratio .', {'entities': []}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('me to get ready for lab meeting at 8:30 AM tomorrow .', {'entities': []}) \n",
      "\n",
      "('timer for 1 min .', {'entities': []}) \n",
      "\n",
      "('can you please write down the following ? Letters one below another in the line . So first line G 1  second line G 2  third line D 3  forth line G 4 .', {'entities': []}) \n",
      "\n",
      "('number 13 is 21.5', {'entities': []}) \n",
      "\n",
      "(' take 4895 μl of name one buffer . Add 1 5 μl of one millimolar DTT plus 100 μl of 50 ex protease inhibitor .', {'entities': []}) \n",
      "\n",
      "('number 1 was run .', {'entities': []}) \n",
      "\n",
      "('the temperature change of 5 ° .', {'entities': []}) \n",
      "\n",
      "('you get over these start waiting product so you have to sign up 2 hands free ?', {'entities': []}) \n",
      "\n",
      "('1 . Cm R = 2.2 . Was a good recording initially . However leak crept in making me not do isoprenaline experiments  but normal 4 AP and Ca transient was done  caffeine was also done and leak experiment was attempted as well . In the SR leak experiment use the wash out as the baseline ( diastolic ) .', {'entities': []}) \n",
      "\n",
      "(\"for analyzing the calibration from the behavior set up . Copy file CAL BIN something from the other set up . From the computer set up to the [ inaudible ] file . Also  measure X and Y axis in pixels . Then open the two part notebook  piezo calibration . In the folder P10 calibration  enter the name of the experiment . Keep the other names as they are . Do n't change them .\", {'entities': []}) \n",
      "\n",
      "('poured 200 ml from A to', {'entities': []}) \n",
      "\n",
      "('12.5 mg of heavy labeled aspartate in', {'entities': []}) \n",
      "\n",
      "('have added 50 µl of the 4x internal standard mix to wells N22  O22 and P22 . I will now check if all peaks are present .', {'entities': []}) \n",
      "\n",
      "('me to check this Phone today at 4:00 PM .', {'entities': []}) \n",
      "\n",
      "('ml . [ unsupported language ]', {'entities': []}) \n",
      "\n",
      "('temperature reached 45 ℃ .', {'entities': []}) \n",
      "\n",
      "('in a 15 ml Wheaton glass homogenizer . In 15 ml homogenization buffer pre - cooled . Homogenisation on ice with 20 strokes and a bit of rotating .', {'entities': []}) \n",
      "\n",
      "(\"may be some variability with the cells but by high there 's no significant cell losses at 45 . Steps on the Z offset for the aspirate . All right .\", {'entities': []}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "('rest a little say rest at 37 ° for . Half . Hour', {'entities': []}) \n",
      "\n",
      "('cable I just measured was number 5 from Box 797 in case I forgot to say', {'entities': []}) \n",
      "\n",
      "(\"mScarlet of 65 0.75 ug DNA I ca n't see red cells . C4 the same sample 1.5 µg DNA I ca n't see red cells .\", {'entities': []}) \n",
      "\n",
      "('used is poli  32 channels . E35B. Second usage . Painted with die . On the back .', {'entities': []}) \n",
      "\n",
      "('for 30 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('me to treat the cells in 5 min .', {'entities': []}) \n",
      "\n",
      "('number 4 close to the in to the radio with an English song running . Phone is in the pocket of the lab coat . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel point phase cut is clear and fast point the clear yellow solution was dried on the rotavap at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': []}) \n",
      "\n",
      "('qPCR for phage Fels-2 perform qPCR with 45 cycles instead of 40 . Because primers have low efficiency between 70 % and 65 % . Blueberry', {'entities': []}) \n",
      "\n",
      "('resisted resistance in all 3 experiments  yeah 2.7  2.9 and 3.4 respectively .', {'entities': []}) \n",
      "\n",
      "('to 50 % buffer B in 8 min at flow rate of 4 ml / min .', {'entities': []}) \n",
      "\n",
      "('weight is 3.25 µg .', {'entities': []}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1 738.5 µg/µl .', {'entities': []}) \n",
      "\n",
      "('me to lab activity on 23rd of March at 12:00 PM . OK .', {'entities': []}) \n",
      "\n",
      "('me to inoculate top 10 and stable 3 bacteria in the evening .', {'entities': []}) \n",
      "\n",
      "('1 estimated protein concentration is 3.8 mg / mil .', {'entities': []}) \n",
      "\n",
      "('for 10 min .', {'entities': []}) \n",
      "\n",
      "('down 6 cell lines and extending NHK 2 to cell line .', {'entities': []}) \n",
      "\n",
      "('sample is missing .', {'entities': []}) \n",
      "\n",
      "('the protocol so it says 7 µl of RNA and 13 µl of Multiscribe Mix .', {'entities': []}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('5 ml of DMA substance .', {'entities': []}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('phenylphosphine 32', {'entities': []}) \n",
      "\n",
      "('32 best in class 1 media added .', {'entities': []}) \n",
      "\n",
      "('a reminder for today at 1:30 to take out the samples out the fridge .', {'entities': []}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"am running flipper assay today . The purpose of this assay is to generate data for a paper . I 'm testing 3 conditions . Number 1 spike effect  number 2 art effect and number 3 signal saturation . The plate map and calculations are as mentioned on the notebook . With the date of today  28.8 . 2019 .\", {'entities': []}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('for 1 h. Timer for 1 h. Blueberry .', {'entities': []}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('12:05 secondary western blot is done .', {'entities': []}) \n",
      "\n",
      "('a temperature change of 21 to 22 ° .', {'entities': []}) \n",
      "\n",
      "('out 20 mg of lucifer yellow . Caterpillar .', {'entities': []}) \n",
      "\n",
      "('a control I used 0.75 µg of Midiprep DNA . The samples are Gao B and Gao off M Scarlet .', {'entities': []}) \n",
      "\n",
      "('the ECG we can see a small St deviation  during the start of the Reperfusion . Now 3 min reperfusion .', {'entities': []}) \n",
      "\n",
      "('for 10 min .', {'entities': []}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('ml Falcon tubes half a box  50 ml Falcon tubes one box .', {'entities': []}) \n",
      "\n",
      "('1 till 10 . PCR product digested using restriction enzymes . Second half of the gel . Slot 1  2  6 . [ In German : \" Wrong RFP .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('spend have I tend to have a 50/50 split between the lab on the office .', {'entities': []}) \n",
      "\n",
      "('4 pipet 50 μl on your booty .', {'entities': []}) \n",
      "\n",
      "('701 with the U145 cells  same conditions as 700 in TBST  but this is with no retrieval and 8 min block .', {'entities': []}) \n",
      "\n",
      "('culture  meaning the buffer or media and cells weigh 2 mg .', {'entities': []}) \n",
      "\n",
      "('a in Tris Glycine SDS buffer using a program of 33 350 V for 11 min program 2 .', {'entities': []}) \n",
      "\n",
      "('table . I will ask for your input to fill table 1 . Please say next when you are ready to go to the next cell you can always save back to correct a mistake . A1 230 .', {'entities': []}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"'s experiment patient number is 1730 .\", {'entities': []}) \n",
      "\n",
      "('performs a bespoke assay once a week . The plate map is always different . It tends to be 308 plates .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm=68 Same problem of leak after getting a gigaseal and starting recording . I do n't know if it 's because of the drifting pipette or probably because of the debri under cell  but only current and transient can be used . Use the initial traces of the transient because the leak influence the transient .\", {'entities': []}) \n",
      "\n",
      "('cell loss was seen on plate 1 1090950961 .', {'entities': []}) \n",
      "\n",
      "('cooling Ambrose at 10 or 7 .', {'entities': []}) \n",
      "\n",
      "('catalog number 553211 from Calbiochem InSolution Rapamycin .', {'entities': []}) \n",
      "\n",
      "('need to raise the experiment pH on the experiment 321 .', {'entities': []}) \n",
      "\n",
      "('group 3 first . OK .', {'entities': []}) \n",
      "\n",
      "('Glutamax 5 ml in 500 ml .', {'entities': []}) \n",
      "\n",
      "('Jackson plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('50 ml of media . Adding second dose of 50 ml . Adding', {'entities': []}) \n",
      "\n",
      "('have one flask per T25 for the patient cell .', {'entities': []}) \n",
      "\n",
      "('116 cells are ready to start the assay tomorrow .', {'entities': []}) \n",
      "\n",
      "('sample one shows varied behavior to patient sample 2', {'entities': []}) \n",
      "\n",
      "('me to drink a glass of water at 7:00 PM today .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('of MTRCAB and PSCB and test A we put uh Q5 and test B we put Fusion .', {'entities': []}) \n",
      "\n",
      "('cell free DNA samples were eluted in 50 µl of elution buffer .', {'entities': []}) \n",
      "\n",
      "('1 chip has minor cell wrapping around the later half of the post arrays . Cell capping on the first half of the post arrays two instances at the interface between entry channels and post where there is debris . Fluid flow is going from bottom to top of image and dimples in the chip are very prevalent in this specific chip . There is also some minor cell  capping/ wrapping around the entry channels .', {'entities': []}) \n",
      "\n",
      "('process improvements in formulation optimization in vivo expression levels have increased by 10 to 100 fold  compared to the first generation mRNA materials .', {'entities': []}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('bacterial suspension for 30 min at room temperature .', {'entities': []}) \n",
      "\n",
      "('cells look healthy after 2 days in culture with 0 R gene .', {'entities': []}) \n",
      "\n",
      "('cells look healthy after 2 days in culture with 0 arginine .', {'entities': []}) \n",
      "\n",
      "('buffer 40 mM  HEPES pH 7.4  300 mM sodium chloride  10 % glycerol  1 mM TCEP  5 mM imidazole', {'entities': []}) \n",
      "\n",
      "('buffer was made with 50 µl HEPES  10 µl pluronic  100 µl . magnesium chloride  20 µl TCEP and made up with 4.820 ml of water .', {'entities': []}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('with microphone close to the mouth  normal noise in the lab difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('330  neural - D1 Medi small cell lung cancer 12 model . Low level of cytoplasmic staining .', {'entities': []}) \n",
      "\n",
      "('the volume to 40 µl to dispense with the acoustic mass spec assay .', {'entities': []}) \n",
      "\n",
      "('me to prepare cells . In 15 min .', {'entities': []}) \n",
      "\n",
      "('is unstimulated', {'entities': []}) \n",
      "\n",
      "(\"'m running two assays together  IT 24325 and IT 24424 . For making buffer I have used my own stock of buffer 500 mM made on 12.08 . 2019\", {'entities': []}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('38 193000', {'entities': []}) \n",
      "\n",
      "('me to feed the cells at 4 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('that all non kit standards are prepared from aliquotes except 1 of the fade curves that is prepared from the stock solution .', {'entities': []}) \n",
      "\n",
      "('infusion of Lidocaine 30 min after start re - perfusion .', {'entities': []}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('105 µl X AMP into lane 1 .', {'entities': []}) \n",
      "\n",
      "('2A7 until 2D6 total volume is about 55 ml is collected in two 50 ml Falcon tubes . It will be concentrated later on .', {'entities': []}) \n",
      "\n",
      "('1 µl of diluted DNA into denaturation .', {'entities': []}) \n",
      "\n",
      "('41 70000 .', {'entities': []}) \n",
      "\n",
      "('we took cyanobacteria with an OD of 0.23 and the inoculated them with 1 ug of PSCBYFP .', {'entities': []}) \n",
      "\n",
      "('a note . 19 ml .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('I need to dilute virus and I need to . Infect cells  I have 35 plates to infect . And I have a virus titer of 10 to 7 per mL  which is 10 to 4 . Per μl . I want to make 6 dilutions of that -1 to -6 .', {'entities': []}) \n",
      "\n",
      "('. 363 . 61 . 31', {'entities': []}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('Bolus Heparin and Lidocaine 10:45 .', {'entities': []}) \n",
      "\n",
      "('the samples in the upper row with reducing agent and SDS buffer respectively 7.7 reducing agent and 19.2 SDS and in the lower row use oxidizing condition . So instead of the SDS sorry instead of the reducing agent use simply water .', {'entities': []}) \n",
      "\n",
      "('I found a way to adapt the glass fittings into an adapter that I can put directly to the water  arium water system so I can clean them out with roughly 6 l of water directly should be good  alright .', {'entities': []}) \n",
      "\n",
      "('batch in C3 in the table .', {'entities': []}) \n",
      "\n",
      "('infection of the B cells  the results are round about 30 h post infection .', {'entities': []}) \n",
      "\n",
      "('the remaining tubes which are 12345610  11 tubes through a .45 µm PES filter from reference number 514 - 0333 . Catalog number 10040 - 438 .', {'entities': []}) \n",
      "\n",
      "('- 001  stored in my draw for TEAD .', {'entities': []}) \n",
      "\n",
      "('80 % viability  1.32000000 cells per ml .', {'entities': []}) \n",
      "\n",
      "('14.5 μl primer . Correction . 14.5 μl nuclease pre [ inaudible ]', {'entities': []}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('me in 10 seconds .', {'entities': []}) \n",
      "\n",
      "('µl Swift 102819 .', {'entities': []}) \n",
      "\n",
      "('time course 100 . I aliquoted ', {'entities': []}) \n",
      "\n",
      "('need to buy more 24 well plates .', {'entities': []}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('AZD 8601 project focuses on Vegfam RNA delivery to the heart by local injections of this drug substrate in buffered solutions .', {'entities': []}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "('number 1  bullet  35 mL. Flask number 2  25 mL. Flask number 3  20 mL. Flask number 4  20 mL. Flask number 5  20 mL. Flask number 6  20 mL.', {'entities': []}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('20  unable to count . Attempted 3 times .', {'entities': []}) \n",
      "\n",
      "('19  2.356 E 5 .', {'entities': []}) \n",
      "\n",
      "('pipet 10 µl BSA in the first tube labeled with Correct 20 µl .', {'entities': []}) \n",
      "\n",
      "('from A1 to B4 via label thing that is called not yet done .', {'entities': []}) \n",
      "\n",
      "('recording number 2 . I am in N0 .', {'entities': []}) \n",
      "\n",
      "('digest the mini prep DNA with XHOI for round about 4 to 6 hour . I added 17.5 µl of DNA  2 µl of Cutsmart buffer and 0.5 µl of XHOI enzyme . As a control', {'entities': []}) \n",
      "\n",
      "('in appointment with Evgeny . We are meeting 31st of August next to Amrit  Friday  12:15 . I just said that but in German .', {'entities': []}) \n",
      "\n",
      "('21  1.692 E5 .', {'entities': []}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('were incubated at 4 ° overnight .', {'entities': []}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "('for 1 h. Timer for 1 h. Timer for 1 h.', {'entities': []}) \n",
      "\n",
      "('16 8.585 E4 .', {'entities': []}) \n",
      "\n",
      "('the 4X heavy labeled internal standard solution . Adding aspartate internal standard into 400 ml of water . Adding 1.6 ml .', {'entities': []}) \n",
      "\n",
      "(\"'ve tried 3 wake words Pineapple Dragonfly Grasshopper  I found Pineapple is the best .\", {'entities': []}) \n",
      "\n",
      "('sample is missing in box 1', {'entities': []}) \n",
      "\n",
      "('is a sample sample 334 . There is multicultural .', {'entities': []}) \n",
      "\n",
      "('number 8u 1642 .', {'entities': []}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "('single chain fab 5T5 38 mosquitoes .', {'entities': []}) \n",
      "\n",
      "('labeled 333 1.5 ml tubes for the compliment pool .', {'entities': []}) \n",
      "\n",
      "('to clarify that is 7.35 mL cells into a total volume of of media and cells at a concentration of 0.2 .', {'entities': []}) \n",
      "\n",
      "('me to check sample 4 .', {'entities': []}) \n",
      "\n",
      "('to a new  um  move to experiment DNA research D237 .', {'entities': []}) \n",
      "\n",
      "('HM 5211 overall looks good slightly scruffy .', {'entities': []}) \n",
      "\n",
      "('usually support 3 projects at once .', {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('. Equal . .25 . 4.2 . Equal . 0.268 . 4.3 . Equal . 0.246 . 4.4 . Equal 0.25 . 4.5 equal . 0.26 . 0.26 .', {'entities': []}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('whole Sigma 56749 dash 250 G lot BCBX6824 .', {'entities': []}) \n",
      "\n",
      "('183.16 µl into lane 7 .', {'entities': []}) \n",
      "\n",
      "('from GL17 with Mercedes .', {'entities': []}) \n",
      "\n",
      "('fly 5 µl of antibody .', {'entities': []}) \n",
      "\n",
      "('the last time I replaced these was not successful . So the next trial will be to change 2 atoms to see if the parameters will change or not . So really complicated system and I never expected that in this case  some and also have prepare some kinetic stuff and some measurements to see . I want to change or if something would change or if not  it will be the task for today .', {'entities': []}) \n",
      "\n",
      "('Room 200 order new test tubes . Avocado .', {'entities': []}) \n",
      "\n",
      "('15 by and divided by 50 .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "(\"me to read the paper at today 12 o'clock .\", {'entities': []}) \n",
      "\n",
      "('with headphones an and iPad as visual feedback . new paragraph extraction . The biphasic solution was steered at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('the purification wash with lysis buffer for 10 .', {'entities': []}) \n",
      "\n",
      "('of sample wild type 2 . Using primers for G1 left  Correct . G1 right G1 B G2 Right G2B. [ Fells ? ] right [ fells ? ] Housekeeping 1  housekeeping 2 with 4 dilutions starting with 5 ng and diluting 1 in 10 . Blueberry', {'entities': []}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('me to check if we have a enough filter tips 1000 µl .', {'entities': []}) \n",
      "\n",
      "('the samples in the upper row with reducing agent and LDS buffer respectively 7.7 reducing agent and 19.2 SDS and in the lower row use oxidizing condition  so instead of the SDS sorry instead of the reducing agent use simply water .', {'entities': []}) \n",
      "\n",
      "('at 100 nM followed by .', {'entities': []}) \n",
      "\n",
      "('I continued with the inoculated strains of listeria monocytogenes . However  I only inoculated 5 ml with one ml during the day so there was too little culture for the experiments . In the future make sure to have at least 10 ml of each strain growing in lock culture before starting the experiment full stop . Today I will inoculate a new overnight culture of each strain .', {'entities': []}) \n",
      "\n",
      "('2 8.827 E4 .', {'entities': []}) \n",
      "\n",
      "('µl pour 14 into lane 3 .', {'entities': []}) \n",
      "\n",
      "(\"are 3 stages to this protocol . First and foremost  I would dissect the larvae  then I 'll fix and stain them and in the last step  I will mount them .\", {'entities': []}) \n",
      "\n",
      "('ID 51 also added to D9 .', {'entities': []}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('37 1.904 E5 .', {'entities': []}) \n",
      "\n",
      "('small cell lung cancer 02 model . Protocol 330  neural D1 very high level of nuclear expression across the entire tumor tissue .', {'entities': []}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('template for the cassette I diluted AX 222  1 in 10 .', {'entities': []}) \n",
      "\n",
      "('have 3 l of baculovirus . Avocado .', {'entities': []}) \n",
      "\n",
      "('was repeated for a third time . End concentrations were around 300 for B and 350 for', {'entities': []}) \n",
      "\n",
      "('this method a dose dependent reduction of LTE 4 in urine of up to 90 % was observed with the treatment of healthy volunteers .', {'entities': []}) \n",
      "\n",
      "(\"-1 chip also the post arrays do n't actually look very circular . They almost looked a little bit squashed .\", {'entities': []}) \n",
      "\n",
      "('ID 18 S 00629 . Sample value 3.705 .', {'entities': []}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('for 7 min timer for 7 min .', {'entities': []}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "('background to protocols 700 . Little', {'entities': []}) \n",
      "\n",
      "('of flexible 0.3  20 µl .', {'entities': []}) \n",
      "\n",
      "('order CD 163 and 68', {'entities': []}) \n",
      "\n",
      "('standard 1 is added to column 5 and 6 .', {'entities': []}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('turn pump off on MXBI 5 only after completely reset of the whole device unplugging also', {'entities': []}) \n",
      "\n",
      "('a timer for 5 seconds for my suspension . [ unsupported language ]', {'entities': []}) \n",
      "\n",
      "('on timer for 10 min .', {'entities': []}) \n",
      "\n",
      "('buffer is in 20 mM.', {'entities': []}) \n",
      "\n",
      "('the M2 name 2 buffer we add 100 μl of 50 exprotease inhibitor and 5 μl of one millimolar DTT .', {'entities': []}) \n",
      "\n",
      "(\"'m using my own stock of HEPES buffer 500 mM.\", {'entities': []}) \n",
      "\n",
      "('. Mouse OG 1 has no antrum tissue .', {'entities': []}) \n",
      "\n",
      "('that all standards are prepared in assay diluent except for the standard of Cas9 prepared in the BSA solution .', {'entities': []}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('test difficult words in writing zone  no noise microphone close to the mouth . 10 mbar . 25 ml . 25 µl . 50 bar . 50 mg . 15 mg . 12 mbar . 70 mbar . 17 mg . 17 µl . New paragraph . Diluted . Test end .', {'entities': []}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('can see states on all channels now at the 730 .', {'entities': []}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('the 500 µl of Opsin to two times 250 µl and add 1500 µl of 20 mM BTP pH 7.5 with 1.25 % OG to each  then vortex all 3 samples for at least 30 seconds .', {'entities': []}) \n",
      "\n",
      "('of the human luciferase construct E 17 . 5 out of 5 sequencing results were in general positive .', {'entities': []}) \n",
      "\n",
      "('buffer was added 500 µl into 1 ml .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min . Blueberry .', {'entities': []}) \n",
      "\n",
      "('cell losses seen at 35 . We will try to reduce the 30 .', {'entities': []}) \n",
      "\n",
      "('with FSEC started 2 blank injections followed by BSA  human serum albumin  rat serum albumin and cyno serum albumin  which was filtered . Each case injection of 2 µl at a speed of 0.35 ml / min .', {'entities': []}) \n",
      "\n",
      "('experiment number 25  film CAL 5 . Amplitudes 1.5  2.5  3.5  4.5  5.5  6.5  7.5  8.5 volts  in a cosine .', {'entities': []}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "('put AZ D1234 into the NMR', {'entities': []}) \n",
      "\n",
      "('the LSRFortessa set up at 9:23 AM .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('the reaction up to 50 ° .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "(\"1 . Cm = 127 . Was a very good recording . All parameters were measured . The calcium current  calcium transient  the caffeine  isoprenaline and isoprenaline load . However  the leak did n't really work . There was a continuous increase in the diastolic value . I do n't know the exact reason why that happened but leak did n't work  but the nice recording and every parameter was measured .\", {'entities': []}) \n",
      "\n",
      "('ID 19S0075 sample value 3.75 ng/µl .', {'entities': []}) \n",
      "\n",
      "('6 226000 .', {'entities': []}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('used the protocol function about 10 times during the observation .', {'entities': []}) \n",
      "\n",
      "(' uh  tubes 13  which is PL190694 MF 19128l11.7 Took out tube 15  which is PL190696 MF19130 11.7 Tube 19 is PL190855 MF19134 10.4 Tube 20 PL190856 MF135 9.8 g and finally tube 21 which PL190856 MF19136 10.1', {'entities': []}) \n",
      "\n",
      "('ml of elution buffer left .', {'entities': []}) \n",
      "\n",
      "('me that I have a meeting with Michael at 2 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "(\"'m adding EGF wild type to the 5 plates first .\", {'entities': []}) \n",
      "\n",
      "('temperature change from 22 ° to 24 ° .', {'entities': []}) \n",
      "\n",
      "('the experiment EP 0024 .', {'entities': []}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('the samples in the centrifuge at 4000 rpm for 15 min .', {'entities': []}) \n",
      "\n",
      "('ml tubes 3 boxes .', {'entities': []}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('might be that this pig requires a little bit higher isofluorane than usual . 1.4 no 2.4 percent compared to 2.2 in average .', {'entities': []}) \n",
      "\n",
      "('I formulated more ACD5153 by adding 15.8 ml of vehicle .', {'entities': []}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "('34 7.916x10 to the 4', {'entities': []}) \n",
      "\n",
      "('ml of TBME were added .', {'entities': []}) \n",
      "\n",
      "('13 9.318 E4 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('off with a gradient in 1 min of 4 ml 100 % B', {'entities': []}) \n",
      "\n",
      "('2 3.5 E 5 .', {'entities': []}) \n",
      "\n",
      "('23 2.304 E5 .', {'entities': []}) \n",
      "\n",
      "('got or Rplp0 too .', {'entities': []}) \n",
      "\n",
      "('20 µl of 5 times sample buffer to all the samples .', {'entities': []}) \n",
      "\n",
      "(\"second pool is 20 ml . It 's fractions G7 to H6 .\", {'entities': []}) \n",
      "\n",
      "(\"n't complain to me about voice notes when you 're getting 72 M BPS down 11 up .\", {'entities': []}) \n",
      "\n",
      "('me of a meeting with Laura tomorrow at 2 PM .', {'entities': []}) \n",
      "\n",
      "('buffers are at 37 ° . I have to take them out later today .', {'entities': []}) \n",
      "\n",
      "('Re - Perfusion 11:24 .', {'entities': []}) \n",
      "\n",
      "('full gradient  force elution for 15 min .', {'entities': []}) \n",
      "\n",
      "('1.21 e to the 6th viable cells per ml and 1.43 e to the 6 total cells per ml . 85 % viability .', {'entities': []}) \n",
      "\n",
      "('me to book the qubit for 2 PM tomorrow .', {'entities': []}) \n",
      "\n",
      "('sample for troponin 10  45 .', {'entities': []}) \n",
      "\n",
      "('you set a timer for 10 min ?', {'entities': []}) \n",
      "\n",
      "(' mNeon of 65 RE delta 21 to 27 mutant  0.75 µg DNA  I can see some green cells . B2 the same sample 1.5 ug DNA  I can see green cells also . B3 and B4 are negative controls .', {'entities': []}) \n",
      "\n",
      "('5 120000 .', {'entities': []}) \n",
      "\n",
      "('to change CD4 .', {'entities': []}) \n",
      "\n",
      "('have just extracted RNA from samples 983  987  and 988 . And I got very small RNA pellets . How though  I started with 2000000 cells for each sample', {'entities': []}) \n",
      "\n",
      "('stock solution 1 mM.', {'entities': []}) \n",
      "\n",
      "('timer for 2 min .', {'entities': []}) \n",
      "\n",
      "('culture yesterday I 293 T AX123 to produce monoclonal cells . I plated 1 96 well plate with one cell per well . I used different dilutions .', {'entities': []}) \n",
      "\n",
      "('up protocol sprint 6', {'entities': []}) \n",
      "\n",
      "('5  6  7 and 8 . Cells with medium . Injection 7 and 8  in border zone . 3 and 4 cm from the LAD septum wall . Injection 5 and 6 in the infarcted area  below injection 7 and 8 . Vehicle injection close to the upper marker . Matrigel vehicle injections above marker . Medium injections below markers .', {'entities': []}) \n",
      "\n",
      "('77 µl of 0.2N sodium hydroxide into dilution .', {'entities': []}) \n",
      "\n",
      "('4 letters to the protocol .', {'entities': []}) \n",
      "\n",
      "('a resource S 1 ml column  number 640486 . Wash the column with 20 mM sodium hydroxide .', {'entities': []}) \n",
      "\n",
      "('lot number of the 1.5 ml cups is 18282 . Expiring date 1st of September 2023 .', {'entities': []}) \n",
      "\n",
      "('3 mM EDTA to the buffer . Reduce the binding artifact .', {'entities': []}) \n",
      "\n",
      "('1 1.985 E5 .', {'entities': []}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "('on the number of samples I can do 2 compounds per plate  usually or 23 or 28  depending on the assay and uh the number that are used to test . We prepare ml accordingly .', {'entities': []}) \n",
      "\n",
      "('of EGF wild type assay ready plates started at 1:47 PM .', {'entities': []}) \n",
      "\n",
      "('gel free stain imaging with 2.5 min staining or activation .', {'entities': []}) \n",
      "\n",
      "('HU1679 .', {'entities': []}) \n",
      "\n",
      "('temperature 38  so high ?', {'entities': []}) \n",
      "\n",
      "('TCEP with 10 mM sodium hydroxide to a pH of 7 . Label tube and store at minus 20 for later use .', {'entities': []}) \n",
      "\n",
      "(\"just checked my cells under the microscope number 93 and 97 are 100 % confluent . Number 988 it 's probably around 90 % confluent  so I will I will split them in a different number . Afterwards  I have to calculate .\", {'entities': []}) \n",
      "\n",
      "('F/58/liver . No abnormalities detected . .', {'entities': []}) \n",
      "\n",
      "(\"'m verifying channel 3 next after Channel 4 .\", {'entities': []}) \n",
      "\n",
      "('incubate the Rhodopsin sample at in the cold room for 2 h at 15 rounds per min . On a', {'entities': []}) \n",
      "\n",
      "('incubate the rhodopsin sample at in the cold room for 2 h at 15 rounds per min .', {'entities': []}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('of growth is seen after 14 days incubation in both containers .', {'entities': []}) \n",
      "\n",
      "('this is the evening of the 25 Okay  almost . And then how do I stop ? This recording ? Do you know ?', {'entities': []}) \n",
      "\n",
      "('plate barcode 115 - 115 - 118 . Strong blue color in wells A3 and A3 .', {'entities': []}) \n",
      "\n",
      "('plate barcode 115 - 115 - 118 . Strong blue color in wells A2 and A3 .', {'entities': []}) \n",
      "\n",
      "('solution number 2 25 µl to number 5 stock solution number 7 25 µl to number 8 Stock solution number 10 246 µl to stock solution number 1 stock solution 3 78 µl to stock solution 12 test end .', {'entities': []}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('10 to 20 % of my whole analysis is setting up this run .', {'entities': []}) \n",
      "\n",
      "(\" the calcium transient are planned for today . With the calcium free medium using Tyrod 's solution  medium is serum free and the calcium influx studies will be performed for 5 mins .\", {'entities': []}) \n",
      "\n",
      "('15 - 3 has one piece of debris in the entry channel space . One piece of debris blocking one entry channel and cell clumping between some of the entry channels . There is three instances of debris clogging the entry channel post interface . There is spider web like cell debris increasingly getting worse throughout the chip through post array 2 through 6 and there is minor cell capping at the entry channels . Again this inlet outlet shape is not similar to active delivery and there is these little grooves on the side of the chips also there is seems to be a 3D printing malfunction on the side of the chip it is not going to impact delivery .', {'entities': []}) \n",
      "\n",
      "('number is actually 10 .', {'entities': []}) \n",
      "\n",
      "('pressure 135.81 .', {'entities': []}) \n",
      "\n",
      "('block P5 tube 21 pool E10  E11  E12  F12 .', {'entities': []}) \n",
      "\n",
      "('the gel  skip B4 to get enough space with the other samples .', {'entities': []}) \n",
      "\n",
      "(\"so very ascorbic acid 100 mg and one g of glucose in 10 ml of sterile Tyrode 's solution .\", {'entities': []}) \n",
      "\n",
      "('chemocides wild type 2 .', {'entities': []}) \n",
      "\n",
      "('neuroblastoma cells are still not ready for being split so I will just change the medium . The plates I have prepared on Friday  they look quite good . I have added retonoic acid on Saturday . But now will change the medium with fresh 10 µM retonoic acid .', {'entities': []}) \n",
      "\n",
      "('. Pour 19 into lane 8 .', {'entities': []}) \n",
      "\n",
      "('12 271000 .', {'entities': []}) \n",
      "\n",
      "('50 ml to vial .', {'entities': []}) \n",
      "\n",
      "('amplitude readout  approximately 9.5 volts .', {'entities': []}) \n",
      "\n",
      "('47 48000', {'entities': []}) \n",
      "\n",
      "(\"80 % of my time is also image analysis as I sit on an AI in data science side of the team . For studies that I 've run and we want answers to specific questions we typically run an AI program based on the slide 's images that we obtain .\", {'entities': []}) \n",
      "\n",
      "('sample ID is 18 S 006290 . And the value is 3.705 .', {'entities': []}) \n",
      "\n",
      "('order new hard drives  5 TB or so . No  please order new hard drives . Yes .', {'entities': []}) \n",
      "\n",
      "('of sepharose beads at 800 g  for 2 min at 4 ℃ .', {'entities': []}) \n",
      "\n",
      "('name is tube Preparation 6 samples .', {'entities': []}) \n",
      "\n",
      "('protocol . Step 1 .', {'entities': []}) \n",
      "\n",
      "('123 123 .', {'entities': []}) \n",
      "\n",
      "('trypsinization 10 min .', {'entities': []}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "('Cas9 standard prepared in 0.1 % BSA is added to column 9 and 10 .', {'entities': []}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "('already prepared the name one buffer . Um  I only have to prepare name 2 . Homogenisation buffer and storage buffer .', {'entities': []}) \n",
      "\n",
      "('mixed TEA buffer with 1 % Agar [ unsupported language ]', {'entities': []}) \n",
      "\n",
      "('me to change the DNA concentration of mutant # 3 .', {'entities': []}) \n",
      "\n",
      "('13 101000 . Sample repeated once .', {'entities': []}) \n",
      "\n",
      "('of the buffer is 7.1 .', {'entities': []}) \n",
      "\n",
      "('plates with barcode primers can be prepared ahead of time and stored at minus 20 ℃ .', {'entities': []}) \n",
      "\n",
      "('the high turnover  FADE Cas9 preserved its', {'entities': []}) \n",
      "\n",
      "('2 times solubilization buffer 25 ml with 2.5 ml 1 molar HEPES  10 ml 5 molar sodium chloride  15 µl 1 molar TCEP from experiment BD 002397.65 . 5.9 ml glycerol at 85 %  50 µM  1 molar calcium chloride and 1 protease inhibitor tablet .', {'entities': []}) \n",
      "\n",
      "('me tomorrow at 9:00 AM that I need to do more Mueller Hinton broth .', {'entities': []}) \n",
      "\n",
      "('joke this is best number 3 .', {'entities': []}) \n",
      "\n",
      "('me to order 20 Falcon tubes . 20 Falcon tubes 50 ml .', {'entities': []}) \n",
      "\n",
      "('- 3 images and end 09423 .', {'entities': []}) \n",
      "\n",
      "('Novex Sharp pre - stain standard 5 µl in lane 1 of the first gel and lane 1 and 2 of the second gel .', {'entities': []}) \n",
      "\n",
      "('30 min remind me to go get coffee .', {'entities': []}) \n",
      "\n",
      "('cell plates  4000 rpm for 5 min . Flamingo .', {'entities': []}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('need order 5 boxes of hundred µl pipette tips .', {'entities': []}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('Day 28 of 2019 - 5 .', {'entities': []}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('a temperature change from 27 to 28 ° .', {'entities': []}) \n",
      "\n",
      "('M knockout cells showed no surface H HLA - A expression and showed increased resistance to allogenic T cell killing .', {'entities': []}) \n",
      "\n",
      "('one sample and diluted by 5 ml .', {'entities': []}) \n",
      "\n",
      "('6  3.128 E5 .', {'entities': []}) \n",
      "\n",
      "('EP 0024 .', {'entities': []}) \n",
      "\n",
      "('LSRFortessa set up at 9:23 AM .', {'entities': []}) \n",
      "\n",
      "('with headphones normal noise in the lab  radio in background  hoods are running . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the rotavap 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('we should start a list of possible targets for CRISPR Cas9 mediated gene knockout . Leptine Titin Otoferlin Other genes uh diabetes', {'entities': []}) \n",
      "\n",
      "('33  1.262 E 5 .', {'entities': []}) \n",
      "\n",
      "('with POT legend loud English talking in the lab . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': []}) \n",
      "\n",
      "('13 and 14 might have been added to well to column 8 and 9 rather than column 7 and 8 .', {'entities': []}) \n",
      "\n",
      "(\"have repeated my northern blot from yesterday . Unfortunately  I got the same wrong results . But now I can see 3 bands that could be the bands that I 'm looking for . I 'm thinking about repeating this experiment  increasing the concentration of the probe .\", {'entities': []}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('will digest the mini DNA from Tuesday with Xho1 the whole day and in the afternoon  I loaded on a pulse field gel to separate the fragments .', {'entities': []}) \n",
      "\n",
      "('did the cloning for pLenti Rab Rab 4  5 and 7 . I used', {'entities': []}) \n",
      "\n",
      "('M232 antibody is aliquoted into 5 µl Aliquots .', {'entities': []}) \n",
      "\n",
      "('HU1672 .', {'entities': []}) \n",
      "\n",
      "(\"for analyzing the calibration . From the behavior set - up copy  file  CAL BIN something from the other set - up from the  computer set - up  to pick the tracking file . Also measure X and Y axis in pixels . Then open the to apply the notebook piezo calibration in the folder P10 calibration . Enter the name of the experiment . Keep the other names as they are  do n't change them .\", {'entities': []}) \n",
      "\n",
      "('34 277000', {'entities': []}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('me to check on tomorrow . 8 .', {'entities': []}) \n",
      "\n",
      "('control virus IV looks good  um  AX172 is infected and AX 173  174  175 are not infect .', {'entities': []}) \n",
      "\n",
      "('is a slight colour gradient in sample 12 in wells G5 and G6 with G5 is darker than G6 .', {'entities': []}) \n",
      "\n",
      "('330  neural - D1 small cell lung cancer 14 model . Areas of non - specific staining  no true nuclear staining .', {'entities': []}) \n",
      "\n",
      "('weight is 3.2 ng .', {'entities': []}) \n",
      "\n",
      "('do start my write up in the lab sometimes . If the BioELN was available this would help me become more compliant . We do get 5 or 10 min here and there  which where we could be using that dead time .', {'entities': []}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('cell line should be located in Box 7 of the liquid nitrogen row 9 .', {'entities': []}) \n",
      "\n",
      "('all of the 5 sequencing results look looks looking promising now .', {'entities': []}) \n",
      "\n",
      "('me in 10 min incubation finished .', {'entities': []}) \n",
      "\n",
      "('µl kinase buffer was added to substrate tubes .', {'entities': []}) \n",
      "\n",
      "('8 7.659 E4 .', {'entities': []}) \n",
      "\n",
      "('me to change the DNA concentration of mutant # 3 . Tomorrow  in the morning at 9:30 AM .', {'entities': []}) \n",
      "\n",
      "('recording 19327005 shape of current nice and no reaction to carbachol stable recording solution changing does not change current . Good control recording for perfusion .', {'entities': []}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('me to go to a seminar at 5 PM .', {'entities': []}) \n",
      "\n",
      "('- mice IgG was used for the ELISA . Used the same antibody for 20 rows .', {'entities': []}) \n",
      "\n",
      "('take . Set a timer for 9:00 AM .', {'entities': []}) \n",
      "\n",
      "('15 -3 ends on 09463 .', {'entities': []}) \n",
      "\n",
      "('1200 or not sorry . Renew 1200 ml', {'entities': []}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 . 20 µl 2 N NaOH into dilution .', {'entities': [(30, 38, 'science_word'), (96, 100, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('µl 2 N NaOH into dilution .', {'entities': [(7, 11, 'reagent')]}) \n",
      "\n",
      "('decided to put 2000000 [ inaudible ] cells into 10µl of RPMI media and they will be cultured for the next 2 to 3 days . I will monitor the growth probably on Sunday quickly and see if I need to exchange the media . Jokerit cells are highly productive cells that consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(215, 222, 'cell_line')]}) \n",
      "\n",
      "('mix lucifer yellow added to 10 % FBS plus 90 % PBS .', {'entities': [(47, 50, 'reagent')]}) \n",
      "\n",
      "('dehydrogenase 5 mg / ml was diluted in PBS to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase was added to 40 µl of PBS buffer .', {'entities': [(39, 42, 'reagent'), (122, 125, 'reagent')]}) \n",
      "\n",
      "('started with 12 samples . With 3 different concentrations for the dilution profile till and total volume is 10 µl with 0.5 . µl of the sample and 0.5 . µl of the primers is done for the 4 primers CTGF  collagen 1 alpha 1  periostin and PBGD .', {'entities': [(236, 240, 'gene')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 to the 5th cells per ml .', {'entities': []}) \n",
      "\n",
      "('and washing the cells with PBS  centrifugation 300 G 10 min .', {'entities': [(27, 30, 'reagent')]}) \n",
      "\n",
      "('added 3 µl of PBS to my cells .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('. Purification .', {'entities': []}) \n",
      "\n",
      "('67 the second Nanodrop measurement looks more promising  better spectrum as a absorbance of 0.11 .', {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('streptavidin Strep - Tactin buffers for GSK compound AZ0161 . Use strep - A buffer BE002397 strep 60 . Take 10 ml and add 7 mg D Biotin .', {'entities': [(0, 12, 'gene'), (40, 43, 'gene')]}) \n",
      "\n",
      "(\"is experiment FEK 109 . I 'm taking samples at time . Points 0  1  2  5  10  15  20 min .\", {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('cells with 3 µl of TCB .', {'entities': [(19, 22, 'gene')]}) \n",
      "\n",
      "('µg pcr successful .', {'entities': [(3, 6, 'science_word')]}) \n",
      "\n",
      "('µl RSB in the lanes 1 and 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"make high virus I need also fresh cells for today . So Nico already prepared 133 milliliers of one million cells per milliliter yesterday . So we also have checked the cell density and healthyness in seconds . I 'm checking adequate 100 microliter to cell density and viability .\", {'entities': [(55, 59, 'gene')]}) \n",
      "\n",
      "(\"'s test it with a replacement of CT2 to CT02 and check if it works  or not .\", {'entities': [(33, 36, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of the assay . 38 spheroids', {'entities': [(0, 9, 'science_word'), (33, 42, 'science_word')]}) \n",
      "\n",
      "('temperature in cell HepG2 to 28 ° Fahrenheit .', {'entities': [(20, 25, 'cell_line')]}) \n",
      "\n",
      "('with Olga with 2 different compounds is possible . You need to add both compounds on one plate and then add the right compound column to the right assay plate column', {'entities': [(5, 9, 'cell_line')]}) \n",
      "\n",
      "(' so FACS is done  we have very low numbers so we will have to do only in 40 μl . Count once and load up . Hopefully we will have enough for 10000 .', {'entities': [(4, 8, 'gene')]}) \n",
      "\n",
      "('hydroxide dilution 180 µl PW1 .', {'entities': [(26, 29, 'gene')]}) \n",
      "\n",
      "('round 5D5 full length 26 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "('cell viability is 65.3 % and concentration is 4.4 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"I 'm preparing samples for our needs it . And first I need to prepare the dilutions for the RNA . So the RNA does n't stand too long on ice to prevent degradation . And um . Then I need to . So first prepare the water  then prepare the master mixes so with oligo DTs  Dntps and RNase Inhibitors and the second mastermix that contains RT reaction . Without the reverse transcriptase that will come the last . Once I have prepared that  then I pick up 30 samples make dilutions . And start typing basically to do the binding with oligos . Factors that Add RT reaction and run the pcr .\", {'entities': [(257, 262, 'science_word'), (528, 534, 'science_word'), (578, 581, 'science_word')]}) \n",
      "\n",
      "('homogenised sample number 2 is very yellow compared to the others that are dark red .', {'entities': [(0, 11, 'science_word')]}) \n",
      "\n",
      "(\"some reason  it looks like the HL ADR is showing up very oddly in the CD ADP S 7 channel and I do n't think there 's anything I can do about it .\", {'entities': [(34, 37, 'gene'), (73, 76, 'gene')]}) \n",
      "\n",
      "('at 96 well plate . Third 3- GFP UOC meg N going from the plate left to right . Starting at A1  A2 .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('value is 1524 parts per billion .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I put the Falcon tubes into the water bath for 10 min . All the samples were immersed into the water bath except the last one as the water bath was too small .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('SDS samples pipetting 20 µl of sample with 7.7 µl of nupage LDS sample buffer 4 times and 3.1 µl new paste sample reducing agents . Samples are pipetted in order 13  15  19  20  21  and GFP . Respective .', {'entities': [(12, 21, 'science_word'), (144, 152, 'science_word')]}) \n",
      "\n",
      "('robot test . Stock solution water . Stock solution number 2 methanol . Stock solution number 3 Ethanol . Stock solution number 5 DMSO . Stock solution number 7 acetone . Stock solution number 8 ethyl acetate . Stock solution number 9 . Water at pH 10 . Stock solution number 10 water at pH 2 .', {'entities': [(129, 133, 'reagent')]}) \n",
      "\n",
      "('cells looked contaminated this morning  so I treated them with 2 rounds of 5 X pen strep . Wash for 15 total min  10 min for one wash in 5 min for the second .', {'entities': []}) \n",
      "\n",
      "('switchavidin 4 µl MPR 3 bio human 1139 2.5 µl . Uh .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('more MPR 3 .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('in deteriate cells from 5th of November  wild type . Looking good . Culturing with a endothelia cell growth factor . Comparing with list . Looking good .', {'entities': [(68, 77, 'science_word')]}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer 04 model . High nuclear expression across tissue medium intensity level . Few areas that are not stained .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('me to add an appointment for TEM on Wednesday at 9 .', {'entities': [(29, 32, 'cell_line')]}) \n",
      "\n",
      "('1 spr wells B11  B12  G12  J11  J12  K12  O12', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('decided to split the Jurkat cells one more time . I did a full split of the culture  so not wasting any cell at all . And split them into 2  so I have a backup flask in case something goes wrong today during the set up of the drug dilution  so kill curve test .', {'entities': [(21, 27, 'cell_line')]}) \n",
      "\n",
      "('μl of Fusion master makes 1.2 μl of DMSO 0.4 μl of forward primer and reverse primer and 17.6 μl of water . To the pcr reaction for chop and LISA stuff .', {'entities': [(36, 40, 'reagent'), (115, 118, 'science_word')]}) \n",
      "\n",
      "('are 18 mil of Jurkat cells and 23 mil of Raji cells .', {'entities': [(14, 20, 'cell_line'), (41, 45, 'cell_line')]}) \n",
      "\n",
      "('1 spr buffer 20 mM HEPES  150 mM sodium chloride  1 mM TCEP 0.05 % Tween20 pH 7.5 .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('one ml of MACS buffer . Centrifugation 300 g 5 min .', {'entities': [(10, 14, 'gene')]}) \n",
      "\n",
      "('TOC value is 286 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('µl RSB in the lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('72 h time point was washed 1 x with PBS . The PBS was then removed and the plate was placed at - 80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('MCF-7 cells appear to be coming off the plate .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('CAS 9 LCMS experiment to identify peptides for a quantitative assay .', {'entities': [(6, 10, 'cell_line'), (34, 42, 'science_word')]}) \n",
      "\n",
      "('has a live cell count of 7 * 10 to the 5th .', {'entities': []}) \n",
      "\n",
      "('5 µl of AZ 225021 of the solubilization buffers . Into the second add AZ0161 GSK compound . Final concentration of both is 1 µM', {'entities': [(77, 80, 'gene')]}) \n",
      "\n",
      "('50 µl of each solubilise solubilisate and FSEC analysis . Add the 1 ml washed strep tactin to the remainder of the solubilisate .', {'entities': [(14, 24, 'science_word'), (25, 37, 'science_word'), (115, 127, 'science_word')]}) \n",
      "\n",
      "('bands Project PMS 6 . 1 . No bands PMS6 2  no bands PMS6 3 .', {'entities': [(35, 39, 'gene'), (52, 56, 'gene')]}) \n",
      "\n",
      "('x PBS into well A12 of third plate .', {'entities': [(2, 5, 'reagent')]}) \n",
      "\n",
      "('today we are going to isolate RNA for the complete set of human adult fibroblasts . We have two sets . Set one  set two . The protocol is step 1 : wash all 6-well plates with ice cold PBS three times for TRIzol reagent in 6-well plate for 10 minutes  600 microliters each . Pipette the lysate into ice cold eppys ..', {'entities': [(184, 187, 'reagent')]}) \n",
      "\n",
      "('project PMS 6 - 1-UC : Digest with DPN1 restriction enzyme .', {'entities': [(35, 39, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"me introduce this person . He 's from Discovery Sciences Department with Discovery Biology team . He is an associate principal scientist working in SC3  SC4  SC1 Lab .\", {'entities': [(148, 151, 'cell_line'), (153, 156, 'cell_line'), (158, 161, 'cell_line')]}) \n",
      "\n",
      "('21.5 ml of cold water to the solubilisation mixture . Then add 5 ml of 10 percent filtered LMNG . And finally mix in the membrane suspension .', {'entities': [(29, 43, 'science_word')]}) \n",
      "\n",
      "('2 . Cm . 170 . R = 3.4 . A nice recording with good current and transient . NCX and caffeine was very beautiful and stable . Also  isoprenaline was attempted with nice current  although the series resistance was increasing probably due to the suction being too high and the volume of the liquid in the bathmaking the seal unstable . However  isoprenaline and Isoprenaline load was also seen or done in this experiment .', {'entities': [(76, 79, 'gene')]}) \n",
      "\n",
      "('inspection  prepare the DNA by mixing 0.1 µg of super coiled plasma DNA into the prepared PBS solution .', {'entities': [(90, 93, 'reagent')]}) \n",
      "\n",
      "('fourth TOC value is 571 parts per billion .', {'entities': [(7, 10, 'gene')]}) \n",
      "\n",
      "('ISF cells passage 13 measured at 11 AM . Cells in total 3.8 * 10 ^6 cells per ml  viable cells 3.77 . Resulting in a viability of 99 %  the new cell concentration will set to 1000000 cells per ml with 30 ml of cell suspension and 37 ml of medium .', {'entities': []}) \n",
      "\n",
      "('nonreducing samples by pipetting 20 µl of sample and 6.7 µl 4 times LDS buffer .', {'entities': [(23, 32, 'science_word')]}) \n",
      "\n",
      "('MPR 3 immobilization 5.5 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('thinks that the 500 µl is too dilute he could not see a film optically with his eye . He believes the threshold should be approximately so we I have prepared new stocks with POBC .', {'entities': [(174, 178, 'cell_line')]}) \n",
      "\n",
      "('µl RSB all 8 lanes .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"change in CO2 . Let 's start occlusion\", {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('reminder in 20 min . Give peptides to [ ? ] .', {'entities': [(26, 34, 'science_word')]}) \n",
      "\n",
      "('have pipetted 5µl into Falcon tube . I want to .', {'entities': [(5, 13, 'science_word')]}) \n",
      "\n",
      "('1st lane of the gel is filled with the selection plate and the 2nd lane is filled with L1A  L1B  L1C  RFP and after that comes the selection plate again .', {'entities': [(87, 90, 'cell_line')]}) \n",
      "\n",
      "('695 neuro D1 with human ovary TBST 1 to 50 .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "('a quality control sample for acoustic mass spectrometry . Mass spectrometer from HepG2 C3A cells . Uh doing a wash with 150 mM ammonium formate buffer . Wash number 1 .', {'entities': [(81, 86, 'cell_line'), (87, 90, 'cell_line')]}) \n",
      "\n",
      "('number 15 . Concentration 170 ng/µl . Fraction A 1.87 . Fraction B 2.39 .', {'entities': []}) \n",
      "\n",
      "('number 8 . Concentration 209 .', {'entities': []}) \n",
      "\n",
      "('PB17 0692 with date 2017 0530 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells X 127 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('for the LPS Assay the number of reactions for the master mix has to be exactly that of the one is stated on the protocol because that one already has extra reactions for pipetting error .', {'entities': [(170, 179, 'science_word')]}) \n",
      "\n",
      "(\"pipette my volume  I use the PAL pipette pro and a 5 ml stripette to pipette my 5000 volume . To pipette the I 've pipetted 4000 µl of 4 milliters and to pipette the remaining amount I did the P1000C blue  serial number R59658C calculation number DOB9URYCEZ .\", {'entities': [(115, 123, 'science_word')]}) \n",
      "\n",
      "('to self figure out what to do about like yielding TIA1  TIA1 exogenous overexpression', {'entities': [(50, 54, 'gene'), (56, 60, 'gene'), (71, 85, 'science_word')]}) \n",
      "\n",
      "('the samples from the HPLC  which have 10 µl left for Mass - spec dilute with 90 µl of LCMS buffer  which contains 3 % acetonitrile and 0.1 % formic acid .', {'entities': [(86, 90, 'cell_line')]}) \n",
      "\n",
      "('transduced 48 h before BJab cells with different lentiviruses . I will harvest I will select them after 48 h normally I select after 24 h. But the control with GSP looked after 24 h not so good so after 48 h all cells are green means they are transduce .', {'entities': [(0, 10, 'science_word'), (23, 27, 'cell_line'), (160, 163, 'gene'), (243, 252, 'science_word')]}) \n",
      "\n",
      "('response does seem to have worked . I was too hasty in my assessment assay seems to be best 0\\\\.5 % DMSO  but seems to be tolerant up to 5 % DMSO screening at 2 % DMSO should be fine for the fragment set .', {'entities': [(99, 103, 'reagent'), (140, 144, 'reagent'), (162, 166, 'reagent')]}) \n",
      "\n",
      "('cells wild type cells needs to be a split into a T75 and the label with heavy SILAC media .', {'entities': []}) \n",
      "\n",
      "(\"tubes are labeled 5B3 and 3A3 and it 's the 2nd A3 . Pour these in water and add Tris buffer to get pH 8.3 .\", {'entities': [(18, 21, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % . Live', {'entities': []}) \n",
      "\n",
      "('with difficult words and PLT legend . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(25, 28, 'gene')]}) \n",
      "\n",
      "('washing the FSEC on SC3 with Milli Q Water at 0.35 ml / min . The column is Sepac SN2A13625 5 µm  500 angstroms  4.6 * 300 mm .', {'entities': [(20, 23, 'cell_line')]}) \n",
      "\n",
      "(\"puri for the purification for the samples that I made a mistake previously in during the . Um preamplifications step that I have now increase the volume of the beads I start at 25 I put 30 . I 'm just in case because it 's much higher volume that I have not . And the samples needed to the mistake .\", {'entities': [(0, 4, 'gene'), (94, 111, 'science_word')]}) \n",
      "\n",
      "('through checked . Expression of aBA First ID 1A Ecolize train BL21 D free  chaperone plasmid PJKJE8  ID1B  BL21 BE free  PGKJE8  ID 4A  BL21 D free  chaperone plasmid PGTF2  ID4B  equalize train BL21 D free chaperone plasmid PGTF2   BL21 D free and chaperone plasmid G PGTF2 78 BL21 D free P Grow 7 ID7B BL21 D free P Grow 7 ID 10A BL21 D free and chaperone plasmid PTF16 ID 10B strain BL21 D free PTFC 16 .', {'entities': [(62, 66, 'cell_line'), (107, 111, 'cell_line'), (136, 140, 'cell_line'), (195, 199, 'cell_line'), (233, 237, 'cell_line'), (278, 282, 'cell_line'), (304, 308, 'cell_line'), (328, 331, 'cell_line'), (332, 336, 'cell_line'), (375, 378, 'cell_line'), (386, 390, 'cell_line')]}) \n",
      "\n",
      "('me to order AP2 for iPSCs .', {'entities': [(12, 15, 'cell_line')]}) \n",
      "\n",
      "('4 . Cm = 96 . R = 2.9 . Very good recording the L type calcium current and transient very stable initially . Caffeine and NCX was done with very nice representative traces . The effect of Isoprenaline was also done with a nice increase in the current . Also  isoprenaline  caffeine / ncx and caffeine induced calcium transient was also very good and representative . After the ISO experiment leak was also done . A nice representative Leak Trace was seen . Still observed the continuous increasing series resistance . However  this only happened after I applied Caffeine  which disrupted the position of the cell and could influence the stability of the seal .', {'entities': [(122, 125, 'gene'), (578, 587, 'gene')]}) \n",
      "\n",
      "('sample has now been concentrated to approximately 10 ml and this is the Relaxin  which is coexpressed so it is processed . Pull from the two concentration tubes for subsequent filtering and application to the size exclusion column when you make the next IMAC is finished .', {'entities': [(90, 101, 'science_word')]}) \n",
      "\n",
      "('two eluted peptides from the column 30 µl of 70 % acetonitrile is added to the spin filters and centrifuged at 1500 RCF for 1 min at room temperature and this step is repeated twice .', {'entities': [(11, 19, 'science_word')]}) \n",
      "\n",
      "('66 µl of the ATP and 20 µl of deoxycholic acid .', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into well a1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells viability 62.2 % . Live cells 8.31 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "('round 5D5 full length 25 .', {'entities': [(6, 9, 'cell_line')]}) \n",
      "\n",
      "(\" process midi for S1 construct number 6 . Second  prepare RNA and protein LiSET 's for human fibroblasts seated on Monday  20th of August . Third  analyze adipocyte differentiation that are being performed on mouse and browning pepper blasts . Process LN9SV in modulised fibroblasts saline for transfection experiments . Set up PCR for new patients .\", {'entities': [(294, 306, 'science_word')]}) \n",
      "\n",
      "(\"have decided to use channel 3 for substrate . Channel 4 . Sorry . Repeat . I 'm No  no  I 've decided to use channel 3 for ATP  Channel 4 for substrate and again channel 3 for both enzymes .\", {'entities': [(123, 126, 'gene')]}) \n",
      "\n",
      "('antibody is buffer exchange using 50 kDa molecular weight cut off filters by washing the solution 3 times with PBS . After that  the antibody solution 100 µl is mixed with 2.67 µl of sulfur tag and leave to react for 2 h in room temperature protected from light .', {'entities': [(111, 114, 'reagent')]}) \n",
      "\n",
      "('110044394 . NK cells . Make media much more yellow .', {'entities': [(12, 14, 'gene')]}) \n",
      "\n",
      "('spheroids in Corning 3830 ultra low attachment plates 384 well .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('Sf9 cells in order to start expression of test expressions of MC4 construct 2 . Therefore 200 mL cells are stored in 4 flasks as 4 mL and they will now be collected in one 500 mL flasks . Then the cells are counted and further to 8 flasks at 1 million cells per flasks .', {'entities': [(0, 3, 'cell_line'), (62, 65, 'cell_line')]}) \n",
      "\n",
      "(\"pipette I 'm using is P100 at a 0.73 µg / ml dilution .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('tubes A5 to 12 for', {'entities': []}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 5 and 10 .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('count for HepG2 C3A cells its 2.09 * 10 to the 6th . Viable cells 1.85 * 10 to the 6th . Viability is 88.6 % .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "(' order 20 Falcon tubes .', {'entities': []}) \n",
      "\n",
      "('homogenised rat liver is sonicated in the sonication bath in AC1 west using these sweep function for 3 min .', {'entities': [(0, 11, 'science_word'), (25, 34, 'science_word'), (42, 52, 'science_word'), (61, 64, 'cell_line')]}) \n",
      "\n",
      "('delivered IgfR 1Rtide . Catalog number . BML - P257 - 001 . Store at -20 ° in freezer .', {'entities': [(47, 51, 'cell_line')]}) \n",
      "\n",
      "('TOC value is 1673 and 1603 that was the value before .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('experiment . Choose day 5th of November . I will today take all the 48 well plates with a ULK1 knockout . Divide in half of the cells in new 48 well plate and the rest will continue to make genomic DNA on to see if knock out still in the pool .', {'entities': [(90, 94, 'gene')]}) \n",
      "\n",
      "('PBS plus FBS to column 3 and 9 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('viability 83.7 % . Live cells 1.02 * 10 ^ 6 cells per ml .', {'entities': []}) \n",
      "\n",
      "('culture . Yesterday I expanded LEC cells AX 205 and AX 218 to T25 flasks .', {'entities': [(31, 34, 'gene')]}) \n",
      "\n",
      "('µl NaOH 2 nanomol .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "(\"something . Hello my name is Julie . I 'm 28 . I 'm French . See it does n't understand me . Is just be tre cute ? But he 's not . Sometimes they are mean with me .\", {'entities': [(104, 107, 'gene')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('concentration of L1a is 36.6 ng/µl .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('spheroids left of fatty acid treated .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "(\"B12 and B11 do n't fit on the gel  but that 's okay . Run the gel 30 min at 200 V in MES buffer using standard conditions .\", {'entities': [(85, 88, 'reagent')]}) \n",
      "\n",
      "('pipettes and 3 centrifuges and 1 FACS set .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('make the 1 mg / ml stock of the IGF . Need to add 2 µl of the 5 M', {'entities': [(32, 35, 'gene')]}) \n",
      "\n",
      "('culture . Yesterday I A549 transduced with IFITM . I split one in ten and harvest a cell pellet for western blot . Yesterday I SLKs transduced  SLK EphA2 knock - out cells with AX 245 . I to one in ten and harvested a cell pellet for western blot .', {'entities': [(22, 26, 'cell_line'), (27, 37, 'science_word'), (132, 142, 'science_word'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('[ ? ] stock bottle  measured at 11 A.M. Cells 4.49 in total . 4.47 viable . Viability 100 % . Split in final concentration with 11 ml of cells and 39 ml of medium .', {'entities': []}) \n",
      "\n",
      "('TOC value is 324 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('2 molar NaOH .', {'entities': [(8, 12, 'reagent')]}) \n",
      "\n",
      "(\"splitting of HepG2 C3A cells . We 've got 2.8 million cells per mill in a volume of 25 ml . Viability is 96.1 % . I 'll be seeding two flasks  one with 5 million and one with 15 million cells .\", {'entities': [(13, 18, 'cell_line'), (19, 22, 'cell_line')]}) \n",
      "\n",
      "('96 h time point was washed 1 x with PBS . The PBS was removed and the plate was placed at -80 ° .', {'entities': [(36, 39, 'reagent'), (46, 49, 'reagent')]}) \n",
      "\n",
      "('cells for the IT 25834 HepG2 C3a spheroid assay following a single vial and .', {'entities': [(23, 28, 'cell_line'), (29, 32, 'cell_line'), (33, 41, 'science_word')]}) \n",
      "\n",
      "('1 spr assay using', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "(\"make my virus dilutions  I will take 280 μl of virus stock . And add this to . μl . Of PBS . That gives me - 2 dilution . I will then take 280 μl of that and add it to μl of PBS to give them -4 and then the same together -6 dilution full stop . That will give me my -2 -4 -6 donation and I live there and remove . ? cell and add 200 μl to each well . In the corresponding -2 -4 -6 wells . So should be adding virus to all of them and then I will put the ? cell back on top . I want to make sure that I add extra pen - strep and gentamicin . To the ? cell before add it back on to make sure it 's killing all the bacteria that are in there . OK .\", {'entities': [(87, 90, 'reagent'), (174, 177, 'reagent')]}) \n",
      "\n",
      "('pipetted 60 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('ml of PBS . 2 µl of anti rabbit .', {'entities': [(6, 9, 'reagent')]}) \n",
      "\n",
      "('ADP - Glo reagent was added using Certus dispenser and read on a FlexStation . 10 mM magnesium chloride solution was used .', {'entities': [(0, 3, 'gene'), (6, 9, 'gene')]}) \n",
      "\n",
      "('count for MCF 7 cells is 3.5 million per ml .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('ending this small scale centrifusion  pipette 50 µl of the supernatant into a vial for FSEC analysis . And run FSEC to understand the solubilisation', {'entities': [(24, 36, 'science_word'), (134, 148, 'science_word')]}) \n",
      "\n",
      "('this is a test to see if LabTwin can say Leukotriene E4  or LTE4  or oxylipids .', {'entities': [(69, 78, 'science_word')]}) \n",
      "\n",
      "('first round . Single chain 5D5 25 mosquitoes .', {'entities': [(27, 30, 'cell_line')]}) \n",
      "\n",
      "('microchip runs faster than the other ones .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('a gel . Taking out a 17 lane gel from the Invitrogen . Nupage 4 to 12 % . Bis Tris gel . Lot number 19040470 . Catalog number NP0329 box . Running it in 1 times MES buffer standard 200V 30 min . Loading 10 µl in each well starting with reducing their non - reducing with a SeeBlue Plus2 standard 5 µl . All samples 10 µl .', {'entities': [(161, 164, 'reagent')]}) \n",
      "\n",
      "('cells for the HepG2 C3A assay using a multidrop Standard cassette on low speed . Full plate 384 standard protocol .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line')]}) \n",
      "\n",
      "('supernatant into tubes A1 through 12 .', {'entities': []}) \n",
      "\n",
      "('330 neuro - D1 Medi small cell lung cancer model 7  wide range of nuclear expression across tumor tissue  but at a medium intensity level .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"first let 's check the binding mode  um  here we have the hydroxy function and we will check the file to see if we can replace the CR4 with the CT2 .\", {'entities': [(131, 134, 'cell_line'), (144, 147, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('40 µl of BRD for protein stock to 4 ml buffer without DMSO .', {'entities': [(54, 58, 'reagent')]}) \n",
      "\n",
      "('number 3 . Concentration 286.5 ng/µl . Fraction number 1 1.86  fraction number 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('infection after 48 hours', {'entities': []}) \n",
      "\n",
      "(\"prepare the system by flowing through 2 times PBS  and clean the loop just to make sure that it 's cleaned .\", {'entities': [(46, 49, 'reagent')]}) \n",
      "\n",
      "('Mtag [ ? ] knock out . Clone 1 . Split on T75 .', {'entities': []}) \n",
      "\n",
      "('with PLT legend normal noise in the lab . 2 m no 4 m distance to the phone  difficult words . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test ends . Dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('absorbance of this sample at wavelength 280 .', {'entities': [(0, 10, 'science_word'), (29, 39, 'science_word')]}) \n",
      "\n",
      "('test normal noise in the labs difficult word . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': []}) \n",
      "\n",
      "('of HUK cells 20 ml of N2A medium .', {'entities': [(22, 25, 'cell_line')]}) \n",
      "\n",
      "('LT2 . Run qPCR with RPE housekeeping gene for all mutants . Together with Fels-2 gene prophage . Blueberry', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('ATP concentration curve row 4 with row 6 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('mg were dissolved in DMSO in order to make an animal sample . The NMR is run on the 500 Megaherz machine .', {'entities': [(21, 25, 'reagent')]}) \n",
      "\n",
      "('relaxin which is co - expressed pool G4 to H11 the size exclusion peak .', {'entities': [(0, 7, 'gene')]}) \n",
      "\n",
      "('. 16 min .', {'entities': []}) \n",
      "\n",
      "('I start recording with you optogenetic stimulus search puls . At death -860  2.5  temperature 36.7 and isoflurane is on 1.4 .', {'entities': [(27, 38, 'science_word')]}) \n",
      "\n",
      "('project PMS6 1 Real - time PCR .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "(\"10 µl of 100 µl of lipid and 1 ml of LMNG detergent 10 % . It 's in homogeneous and I 'm going to sonicate it with Branson sonifier 250 with a special microtip . At output control 2  duty cycle initially 40 to homogenize and make a clear solution .\", {'entities': [(19, 24, 'science_word'), (98, 106, 'science_word')]}) \n",
      "\n",
      "('Cas9 assay development a piece of blank rat heart is homogenised in 5 times with volume of RIPA buffer where after it will be sonicated . And after that  Uh  sodium chloride solution will be added to a final concentration of 0\\\\.5 molar .', {'entities': [(53, 64, 'science_word'), (126, 135, 'science_word')]}) \n",
      "\n",
      "('293 T cell plates for NPR to experiment .', {'entities': [(22, 25, 'gene')]}) \n",
      "\n",
      "('and 12 cells with Matrigel / PSC . Injection 11 and 12 in borderzone . Close to the septum wall . 1 cm below the lower marker . Injection 8 and 9 along LAD in the infarcted area  below injection 12 .', {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('PBS to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('HSD 1012 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath of up at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('substrate has been added to the wells and the plate is put on a orbital Shaker at 450 RPM .', {'entities': []}) \n",
      "\n",
      "('pipetted 5 µl into 5ml Falcon tube and then I transfer the media into 69 well plate .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . Buffer is 20 mM HEPES  150 mM sodium chloride  1 mM TCEP  0.05 % Tween-20  1 % DMSO pH 7.5', {'entities': [(8, 11, 'cell_line'), (85, 93, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('the purification after restriction of L1RFP amp ori with the general selection cassette .', {'entities': [(48, 51, 'gene')]}) \n",
      "\n",
      "('BJAB infection experiment . Um . I induce the iSLKs just with Dox for 24 h and it looks much better than the last time  because all iSLKs are living and I can compare the B cells and the iSLKs . I can see that the B cells are infected . JAB', {'entities': [(0, 4, 'cell_line'), (62, 65, 'gene')]}) \n",
      "\n",
      "('pipetted 25 µl into the Eppendorf tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('staining solution CD102 FITC 1:200 . CV140a 1:200 .', {'entities': [(18, 23, 'gene')]}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('1 µl HPA1 protein to 1.945 ml of buffer to make 2 µg / mill of HPA 1 protein .', {'entities': [(5, 9, 'gene'), (63, 66, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3 % . Live cells 3.08 * 10 ^ 5 cells per ml .', {'entities': []}) \n",
      "\n",
      "(\"5 . Cm = R = 2.7 . The recording would have been a very good recording if not because the cells became unstable . L type calcium current and transient were very good . Caffeine induced calcium transient was good but NCX had a and they did n't really get to baseline before having some noise . An unstable noise of the current so you should try to analyze and see whether the NCX can be used if not only L type calcium current and transient should be used .\", {'entities': [(216, 219, 'gene'), (375, 378, 'gene')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  3  9 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('with Andrew robots was done . Pipetting 25 µl to a 2 ml stock solution .', {'entities': [(30, 39, 'science_word')]}) \n",
      "\n",
      "(\"in addition to experiment 4 the current might be very small  which is an observation when you do n't really have good access to this cells or the NCX could still be used as it was very nice with a nice decay . Flamingo .\", {'entities': [(146, 149, 'gene')]}) \n",
      "\n",
      "('was removed from the 16 h timepoint plate . cells were washed 1 time with PBS and placed the plate was placed at minus 80 ° .', {'entities': [(74, 77, 'reagent')]}) \n",
      "\n",
      "('today  we did the nuclei experiments on doner 3 . Left ventricle  right ventricle and left atrium . Um  the homogenisation was a little bit of a struggle . So the left ventricle went perfectly  the first sample worked well  with the second one it was rather hard to homogenize with the second pestle . And with the third one there was a lot of dirty material . And that was shown as a film on top of the centrifuge tube .', {'entities': [(108, 122, 'science_word')]}) \n",
      "\n",
      "(\"description . Homologation for the customer BMW . Please perform test according to TL 260 on parts processed on the customer . 60 cycles PV 1210 one part  240 constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles PV 12101 part paragraph 60 cycles PV 12101 part 240  Constant humidity  CH-1 part . Stone chip resistance panels to be sent . Chemical resistance panels to be sent . 30 cycles  PV 1210 one part Conclusion  all the results pass the specification VW and only the past panel 10 and 11 do n't pass this specification . VW TL227 OFL Dash X3 2 because the delamination and adhesions of superior add specification requirements . After 5 cycles . No service changes  including cut edges in wells . ° of rusting DIN EN ISO 468 dash 3 RI 0 . After 36 30 cycles no service changes including cut edges and well lost of adhesion . The diameter is less than or equal to 1.5 mm\", {'entities': [(178, 182, 'cell_line'), (354, 358, 'cell_line')]}) \n",
      "\n",
      "('samples were heated at 95 ° for 5 min and then loaded on a 12 well  4 to 12 % Tris HI gel and run in MES buffer at 200 V for 35 min .', {'entities': [(101, 104, 'reagent')]}) \n",
      "\n",
      "('is robotics lab . This is shared by MBP  Cell Assay Development  Safety Sciences  and functional genomics .', {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('immune . The next TOC value is 1604 .', {'entities': [(18, 21, 'gene')]}) \n",
      "\n",
      "('chip lot number 10271301 .', {'entities': []}) \n",
      "\n",
      "('TOC value is 812 parts per billion .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "(\"1 . Cm = 130 . Started as a very good recording with a little spontaneous events . However  the pipette was drifting which started reflecting in the increase in the series resistance . L - type calcium current was good . NCX with caffeine transient was good as well . However  upon moving to isoprenaline the transient effect was not the same was n't really good . It 's not the first time this is happening  it might be due to perfusion problems or cell problem . Regardless current transient and NCX can be used .\", {'entities': [(221, 224, 'gene'), (498, 501, 'gene')]}) \n",
      "\n",
      "(\"experiment 66 is AZ2250 about 300 µl is left after the second centrifugation and this is also same for the info from the same experiments and two others started out with slightly more material . So there 's a little bit more left as well . Respectively  500 µl for experiments 66 AZ0161  so that 's already good enough for size exclusion . The experiment 67 requires some more centifugation and also addition of the remainder of the solution .\", {'entities': [(377, 390, 'science_word')]}) \n",
      "\n",
      "(\"dehydrogenase was 5 mg / ml was diluted in PBS buffer to 3 mg / ml . Therefore  60 µl of Alcohol dehydrogenase were mixed with 40 µl of PBS buffer . Better I 'm realizar with 4 mg / ml was diluted to 3 mg / ml in PBS . Therefore  75 µl of beta Amylase was mixed with 25 µl of PBS . All samples were added to a centrifugal Google Filter Unit Ultrafree MC GV 2 22 µm 0.22 µl and centrifuge for 10 min at 10000 rpm .\", {'entities': [(43, 46, 'reagent'), (136, 139, 'reagent'), (213, 216, 'reagent'), (276, 279, 'reagent')]}) \n",
      "\n",
      "('digested the H3K27 M and G34A vectors that came from Millers glycerol stocks . That are now in my box in the -80 and I digested them with the SpeI and NotI enzymes along with an CAG EEF2 K puro backbone . Which is also digested with the NotI and SpeI enzymes . I plan to run them on a gel . Today .', {'entities': [(30, 37, 'reagent')]}) \n",
      "\n",
      "('HSE RNA sample 2.3 µl were added to 12.7 µl . Expanded HSE samples : 5.5 µl were added to 9.49 µl . Fresh erythroid samples : 2.73 µl were added to 147.3 µl . Expanded erythroid : 1.94 µl were added to 15.1 µl', {'entities': [(0, 3, 'gene'), (55, 58, 'gene')]}) \n",
      "\n",
      "('M232 antibody is biotinylated 200 µl of 1 mg / ml solution . Is a big process . To that 2.33 no 2.66 µl of biotinylated reaction solution is added .', {'entities': [(17, 29, 'science_word'), (107, 119, 'science_word')]}) \n",
      "\n",
      "('spheroids into Corning 3830 ultra low attachment plates with Multidrop Combi Standard cassette 384 . 40 µl of seeding volume full plate .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('round 20 mosquitoes PBS .', {'entities': [(20, 23, 'reagent')]}) \n",
      "\n",
      "(\"able to access information is our biggest pain points . We need a smooth experiment trail where we can see PDFs that I 've run that day . I have 3 Excels . When the run was organized for one experiments the calculations .\", {'entities': [(107, 111, 'cell_line')]}) \n",
      "\n",
      "('200 µl .', {'entities': []}) \n",
      "\n",
      "('1 sample group for low concentration of Cas9 is also added called 11A until 15B in the same conditions as 1 A to', {'entities': [(66, 69, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('6.5 µl of RSB into denaturation .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('- 46  slide 88 . Liver . Diffuse minimal to mild centrilobular hepatocyte glycogen accumulation . Rare single cell hepatocyte apoptosis .', {'entities': [(49, 62, 'science_word')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('a rotator SB3 [ ? ] Stuart .', {'entities': [(10, 13, 'cell_line')]}) \n",
      "\n",
      "('the sample from yesterday . So the co - expression IMAC purified relaxin concentrating that in a 1 KD filter at 4000 Gs 10 min at 14 ° . Preparing Massspec samples . Taken 5 50 µl of the IMAC pool to 1 at 50 µl of 1 molar TCEP to the other 50 µl of water and then top it up with hundred µl of Massspec buffer Run the samples at a Masspec on SC1 .', {'entities': [(65, 72, 'gene'), (99, 101, 'gene'), (341, 344, 'cell_line')]}) \n",
      "\n",
      "('cells viability 39.3', {'entities': []}) \n",
      "\n",
      "('test PLT legends English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transfered to a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('PBS lot number RNBH5735 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('I prep my tissue slides I use an autostainer which performs an immunohistochemical analysis . This takes 1 to 3 h.', {'entities': [(33, 44, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"'m doing a restriction digestion now with a total volume of 16 µl of which I will take 45.4 µl water 3 µl of Mot1 and 8 CC1 enzyme . And 6 µl of 10 times buffer . And 2.6 µl of the plasmid DNA for a total of 16 µl .\", {'entities': [(120, 123, 'cell_line')]}) \n",
      "\n",
      "('10 µl from tubes A1 to 8 in the tubes 13 through 16 20 µl .', {'entities': []}) \n",
      "\n",
      "(' no not pool  but analyse fractions 3A3  5A1  5B1  5B3 and continue later .', {'entities': [(41, 44, 'cell_line'), (46, 49, 'cell_line'), (51, 54, 'cell_line')]}) \n",
      "\n",
      "('500 µl of SEC buffer in a 100 KD . Centrifuge 0\\\\.5 ml and run it 2 min at 9000 RPM . For preparing it for concentration .', {'entities': [(30, 32, 'gene')]}) \n",
      "\n",
      "('2 . 6 min .', {'entities': []}) \n",
      "\n",
      "(\"'m taking out plasmid PL19 121  124  125  126  128 . These are tubes number 1  4  5  6  8 respectively .\", {'entities': [(22, 26, 'cell_line')]}) \n",
      "\n",
      "('the first wash steps and fixation step most of the MCF-7 cells came off the plate .', {'entities': [(51, 56, 'cell_line')]}) \n",
      "\n",
      "(' 5 micro 4 µl of NaOH into both lanes .', {'entities': [(17, 21, 'reagent')]}) \n",
      "\n",
      "('one PBS 35 mosquitoes', {'entities': [(4, 7, 'reagent')]}) \n",
      "\n",
      "('of September 2019 . BLIMP 1 spr assay for the Biacore 8K. Made up 5 l of buffer containing 20 milimolar HEPES  150 milimolar sodium chloride  1 milimolar TCEP  1 % DMSO and 0.05 % Tween-20 with pH 7.5 .', {'entities': [(28, 31, 'cell_line'), (164, 168, 'reagent'), (180, 188, 'reagent')]}) \n",
      "\n",
      "(\"infected yesterday the BJAB monoclonal cells  which were reconstituted with EphA2 and EphA7 with KSHV wild type . In the empty vector reconstituted cells  I ca n't see infected infection . But in the EphA2 and EphA7 reconstituted cells I can see an infection  but it looks not like normal infected cells . Not the whole cell is green just the surface  I would say  and the control cells  means non infected cells  are not green also .\", {'entities': [(23, 27, 'cell_line')]}) \n",
      "\n",
      "('BLIMP 1 spr assay . 50 µg / mil BLIMP 1 immobilized for 10 min to fluorated 5 µl / min .', {'entities': [(8, 11, 'cell_line'), (66, 75, 'science_word')]}) \n",
      "\n",
      "('of neuro 2a cells 20 ml of N28 plus medium .', {'entities': [(3, 8, 'gene')]}) \n",
      "\n",
      "('16A 97.1 1.95  gnome 16B 97.0 2.53  gnome 16C 94.8 2.55  1732 A 96 2.31  3032 B 94.2 6.6 .', {'entities': [(21, 24, 'cell_line'), (42, 45, 'cell_line')]}) \n",
      "\n",
      "(' I am removing 4 % PFA from the LP1 mutant and to step fixation I will wash 3 times 15  3 times with PBS for 15 min .', {'entities': [(32, 35, 'cell_line'), (101, 104, 'reagent')]}) \n",
      "\n",
      "('will use 40 µl of PI in 400 µl of FACS buffer and 40 µl', {'entities': [(34, 38, 'gene')]}) \n",
      "\n",
      "('a buffer . Prepare buffer of 0.1 times PBS by diluting 50 ml 2 times PBS in 950 ml water .', {'entities': [(39, 42, 'reagent'), (69, 72, 'reagent')]}) \n",
      "\n",
      "('HepG2 cell count is cells per mL.', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('pipetted the DNA into well A1  A2  A3  A4 and A6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('number 6 . Concentration 259.8 ng/µl . Fraction 1 1.86  fraction 2 2.36 .', {'entities': []}) \n",
      "\n",
      "('out a vial of HepG2 C3A cells . Cell bank ID 82067 to make spheroids .', {'entities': [(14, 19, 'cell_line'), (20, 23, 'cell_line'), (59, 68, 'science_word')]}) \n",
      "\n",
      "('me to pick up the plates when I pass by the AC3 lab .', {'entities': [(44, 47, 'gene')]}) \n",
      "\n",
      "('PLT legend with radio in the lab in the background . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl  diluted  test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('sonicated the samples for 35 seconds .', {'entities': [(0, 9, 'science_word')]}) \n",
      "\n",
      "('µl RSB into lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('noted that the signal for creatinine was almost 0 despite normal sample and everything looked okay . But after cleaning all the cones  the signal increase to 1 million peak height  meaning 10 thousand times more sensitive system after cleaning all the cones . My theory is that the cone gets covered with lipids and they give no sign of contamination by visual inspection  but they isolate the cone probably . So it is critically important to make sure that no lipids are wasted into the Mass Spec unless necessary and unless they already analysed of course .', {'entities': [(305, 311, 'science_word'), (461, 467, 'science_word')]}) \n",
      "\n",
      "('dilution 9 µl of RSB .', {'entities': [(17, 20, 'cell_line')]}) \n",
      "\n",
      "('count for MCF7 cells is 3.5 million per ml .', {'entities': [(10, 14, 'cell_line')]}) \n",
      "\n",
      "('DAPI at 1 in 200 dilution in PBS . For a final 5 µM concentration .', {'entities': [(29, 32, 'reagent')]}) \n",
      "\n",
      "(\"a third centriguation of 30 min to concentrate the samples . It 's now almost around 13 ml and I want to go down to 10 ml to load onto the size exclusion chromatography graphic column .\", {'entities': [(8, 21, 'science_word')]}) \n",
      "\n",
      "(\"was a ... a mistake in my measurements . I forgot to square the radius . Now I also get 100 mW / mm2 . I do n't know if I ever really used this calculation . I hope not .\", {'entities': [(97, 100, 'cell_line')]}) \n",
      "\n",
      "('supernatant cells . Washing with PBS . Trypsinization : 500 µl trypsin / EDTA', {'entities': [(33, 36, 'reagent')]}) \n",
      "\n",
      "('40 ml from HS-5', {'entities': [(11, 15, 'cell_line')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('culture 293 T AX123 expanded to T25 flask . SLK AX123 plus AX245 expanded to 6 well .', {'entities': [(44, 47, 'cell_line')]}) \n",
      "\n",
      "('number 4 . Concentration 195.4 ng/µl . Fraction 1 1.87  fraction 2 2.35 . Stop .', {'entities': []}) \n",
      "\n",
      "('OCI - AML3 cells MHH cloud 2 cells . Check Tax cells and MB4 cells .', {'entities': [(6, 10, 'gene')]}) \n",
      "\n",
      "('have now added 10 µl of water to 10 µl of blood . Next step is a to add 10 microliter of calibration sample and 10 µl water in their wells . centifugation after mixing is the final step prior to adding internal standard', {'entities': [(141, 154, 'science_word')]}) \n",
      "\n",
      "('3 project . Eleven in - house peptides 3 plus 19 [ ? ] .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n",
      "('note  project PMS6 1  annealing temperature 62 ° to 72 ° .', {'entities': [(14, 18, 'gene')]}) \n",
      "\n",
      "(\"make the SB stock . There 's a 10 mg powder add 2.6 ml of DMSO for the new 50 mg powder need 13 ml DMSO . Make 500 µl aliquots .\", {'entities': [(58, 62, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "(\"tomorrow do n't forget to run the gel . To check the AAV9 plasmids from .\", {'entities': [(58, 66, 'science_word')]}) \n",
      "\n",
      "('supernatant resuspending cells with 500 µl MACS buffer . Going to sorting sorting on CD 102 High ; CD 140a low population .', {'entities': [(43, 47, 'gene')]}) \n",
      "\n",
      "('30 µl of each in HPLC vial . PL-19 - 1088 : 7.5 ml final volume . PB 19 1089 : almost 10 ml . PB 19 1090 7.5 ml .', {'entities': []}) \n",
      "\n",
      "('me to select my infected BJAB cells . With hygromycin 200 µg / ml . On Friday evening at 4:00 PM .', {'entities': [(25, 29, 'cell_line')]}) \n",
      "\n",
      "('buffer . 20 mM HEPES  150 mM sodium chloride  1 Milimolar TCEP .', {'entities': []}) \n",
      "\n",
      "('PBS used as for injection number . Falciparum infection from Friday 2nd of November .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('pipetted the DNA into well a1 a2 a3 a4 and a 6 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('PBS plus DMSO plus FBS to column 4 and 10 . 200 µl in each .', {'entities': [(0, 3, 'reagent'), (9, 13, 'reagent')]}) \n",
      "\n",
      "('µl RSB to  into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('1 spr using EDTA after immobilization .', {'entities': [(2, 5, 'cell_line')]}) \n",
      "\n",
      "('injection PBS 167.1 μL.', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PBS plus FBS to columns 3 and 10 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "(\"let 's try this . I used 100 Milli molar NaOH and for other substrates  I used 200 Milli molar and NaOH .\", {'entities': [(41, 45, 'reagent'), (99, 103, 'reagent')]}) \n",
      "\n",
      "('cells 4x with PBS .', {'entities': [(14, 17, 'reagent')]}) \n",
      "\n",
      "('another purification with 5B3 and the remainder of 32A3  and adapt the gradient to be slightly more shallow to separate the peaks better .', {'entities': [(26, 29, 'cell_line')]}) \n",
      "\n",
      "('supernatant from tubes A1 to 12 and put into B1 to 12 . Added 500 µl of PBS to A5 to 12 . Vortex . Removed 10 µl .', {'entities': [(72, 75, 'reagent')]}) \n",
      "\n",
      "('M1 cells cultured in 4 ml media .', {'entities': []}) \n",
      "\n",
      "(\"at 9:15 sonication was finished so the whole procedure for 6 tubes takes about 1 h  but it 's also a lot of incubation time with just resuspending the pellets .\", {'entities': [(8, 18, 'science_word')]}) \n",
      "\n",
      "(\"cells were at 0.68 and 86 % . I 'm going to split them so that 7 . of 10 mL of Jurkat cells goes into 25 mL of new cells . Actually that might be wrong . Let me check .\", {'entities': [(79, 85, 'cell_line')]}) \n",
      "\n",
      "('696  no block neuro D1  1 to 100 in TBST  looking at human pancreas .', {'entities': [(14, 19, 'gene')]}) \n",
      "\n",
      "('number of viable cells of HepG2 is . I would add of cell suspension and 70 mL of media to get of total cell suspension to seed into 5 plates .', {'entities': [(26, 31, 'cell_line')]}) \n",
      "\n",
      "('BJAB cells infected with the RRV AELAAN mutant  the wild type and NT control are round about to 50 % infected . And the knockout cells much less round about 10 % . I will harvest for FACS after 72 h.', {'entities': [(0, 4, 'cell_line'), (183, 187, 'gene')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . The phase cut is clear and fast . The clear yellow solution was dried on the water bath up at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('HeLa cells at the 96 h time point had a lot of debris .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into denaturation .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\". I 'm ready for 5 PGF assays . Bumblebee .\", {'entities': [(19, 22, 'cell_line')]}) \n",
      "\n",
      "('and MCC 950 . ATP', {'entities': [(14, 17, 'gene')]}) \n",
      "\n",
      "(\"more pulses of sonication did n't give a clear solution so heating at 65 ° for 10 min . Set timer for heating 10 min .\", {'entities': [(15, 25, 'science_word')]}) \n",
      "\n",
      "(\"the CRISPR I experiment we have cells none 16 cells that we have a stable transduced with the Tet dCas KRAB plasmid . And once we have a stable expressing the plasmid we induce them with doxycycline to express and cherry . Once we have the cells growing we transduced them with the guide RNA plasmid and that contains BFB as a reporter . And that contains the guide RNA for the Bcl 2 gene . I 'm being we run experiment on flowJo to select the cells that are expressing  um  Tet dCas plus BFB so that indicates that they have both inducible system as well as the guide RNA .\", {'entities': [(74, 84, 'science_word'), (94, 97, 'gene'), (257, 267, 'science_word'), (475, 478, 'gene')]}) \n",
      "\n",
      "('330 neuro D1 tumor pancreas  low level of nuclear expression .', {'entities': [(4, 9, 'gene')]}) \n",
      "\n",
      "(\"4 . Cm = 66 . Not a very good recording  because there are lots of debris on the cell and there 's also leak in the cell . Although there was a nice NCX and nice Current that could be used . eventually lost the seal of the recording and had to end the experiment .\", {'entities': [(149, 152, 'gene')]}) \n",
      "\n",
      "('me ... reading 3.12 mW and the area is 10 W / cm2', {'entities': [(46, 49, 'cell_line')]}) \n",
      "\n",
      "('the 4 different transfections the I 5 P Deletion Mutant showed 10 positive colonies and we picked 4 colonies  which are now in mini culture and about to be mini - prepped tomorrow and sequenced subsequently ( punkt ) .', {'entities': [(16, 29, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('for the HEK cells in 12 well plate and plan through perform the calcium imaging studies tomorrow .', {'entities': [(8, 11, 'cell_line')]}) \n",
      "\n",
      "('culture . Transduced LEC cells looks like all cells are green or red means 100 per cent transduction or the selection was nice . The culture medium is without selection antibiotic .', {'entities': [(21, 24, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl  no 50 µl .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22 RV 1 cells are maybe 50 % confluent', {'entities': []}) \n",
      "\n",
      "('08095 is 60 % conversion by TLC at 5:30 PM .', {'entities': [(28, 31, 'gene')]}) \n",
      "\n",
      "('10 times neuro  Basil on my order list .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 8 . Concentration 299.1 ng/µl . Fraction 1 .', {'entities': []}) \n",
      "\n",
      "('split 293 T in a T-75 flask twice . SLK in a T-75 flask and HFFs 1 in 3 in 3 times 175 flask .', {'entities': [(36, 39, 'cell_line')]}) \n",
      "\n",
      "('HepG2 cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line')]}) \n",
      "\n",
      "('this 2 µl of sample was pipetted in triplicates in the plate according to plate layout .', {'entities': [(24, 32, 'science_word')]}) \n",
      "\n",
      "('me to seed Axin1 cells for the Axin1 project', {'entities': [(11, 16, 'gene'), (31, 36, 'gene')]}) \n",
      "\n",
      "('then it starts recording whatever I say for example . I just pipetted 2 mL of PBS into my cells .', {'entities': [(61, 69, 'science_word'), (78, 81, 'reagent')]}) \n",
      "\n",
      "('pipette my 71.43 µl of my phospho - RAD antibody number 4223 I use the P200 pipette Blue C  serial number U545796 calculation number DOB 9 URYCXE .', {'entities': [(71, 75, 'cell_line')]}) \n",
      "\n",
      "('Sf9 splitting .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "(' we starting to pipette the experiment at 1:30 . We hope to be done with premixing all the reagents by 3:00 in the afternoon and pipet on the cells  so that we have at least 22 h incubation over night .', {'entities': [(73, 82, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube . I then put the tubes into the water bath . The results show that the water bath was not hot enough .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('1 . Cm = 99 . Series resistance was a little high . The current and transient transient were good even though there was leak in the L - type Calcium current . Caffeine was also applied with nice NCX and caffeine induced transient .', {'entities': [(195, 198, 'gene')]}) \n",
      "\n",
      "('protein is at 5 mg / mill .', {'entities': []}) \n",
      "\n",
      "('2 . Cm= 85 . The recording was very good one  the best of the day . The L - type calcium current and calcium transient being OK . The current had little leak making membrane potential a little unstable . The first caffeine NCX had some artifact due to the electrical noise from the perfusion . But the second NCX is OK and manageable to be analysed with good caffeine induced calcium transient . Leak was attempted also and PMCA can be analyzed . Also  leak was measured using tetracaine .', {'entities': [(223, 226, 'gene'), (309, 312, 'gene'), (424, 428, 'gene')]}) \n",
      "\n",
      "('recently radioimmunoassays have been replaced by enzyme linked immuno assays for urine LTE4 .', {'entities': [(9, 26, 'science_word')]}) \n",
      "\n",
      "('need to check viability of the OCI - AML 3  the MHH call-2 cells  the Jurkart cells  the NB4 cells .', {'entities': [(89, 92, 'cell_line')]}) \n",
      "\n",
      "('viability 83.7 % .', {'entities': []}) \n",
      "\n",
      "('300 ml SEC buffer without compound without detergents  which will be filtered and then split up in afterwards detergent  lipids  and compounds are added . Competition 20 ml mM Tris HCL 150 mM sodium chloride  1 mM calcium chloride  1 mM T - CEP . Eventually also 0.01 % LMNG  0.01 mg / l lipids  and 0\\\\.1 sorry 0\\\\.2 µM compound .', {'entities': [(121, 127, 'science_word'), (288, 294, 'science_word')]}) \n",
      "\n",
      "('1 is Phosphate Buffered Saline . Tube 2 is Phosphate Buffered Saline plus 5 % DMSO . Tube 3 is Phosphate Buffered Saline plus fetal bovine serum . Tube 4 is PBS plus DMSO plus FBS 10 % . End .', {'entities': [(78, 82, 'reagent'), (157, 160, 'reagent'), (166, 170, 'reagent')]}) \n",
      "\n",
      "('µl RSB lane 1 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"something went wrong with one sample  with magnetic purification  so in the second row where I actually put 30 µl of the beads  which is more 5 µl more  in wonder when now when i suspend them the sample beads with water to take the DNA out . I notice that one of the sample has less beads  so it 's paler in color . I do n't know why . So I did n't observe any loss of beads during the pipetting  so I really do not understand what has happened .\", {'entities': [(386, 395, 'science_word')]}) \n",
      "\n",
      "(\"4 . Cm = 36 . Only current and transient should be used . Because the NCX did n't really recover upon application of caffeine . Current and transient had Leak  but it could still be added to the bulk .\", {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('4 . Cm = R = 2.2 . L - type calcium current was measured . Caffeine NCX was measured . Also isoprenaline effect on both the L type calcium current and the caffeine effect was also measured . Leak was attempted as well but failed because the bubble stop at the tip of the pipette preventing the solutions from getting to the cell .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('. Mouse NPR 3 . Ratio 12 to 7 µl .', {'entities': [(8, 11, 'gene')]}) \n",
      "\n",
      "(\"3 . Cm= 42 . The current was initially stable  although Leak crept in . NCX which was very good upon caffeine application . Series resistance was n't so high so it was actually a good recording . Be careful when analyzing the maximum or minimum because . I forgot to stop the protocol before changing the perfusion back to 4AP .\", {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 3 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('cell culture . I expanded rhesus fibroblast transduced with AX107 and AX 143 plus NT control cells . I expanded 2 T25 flasks .', {'entities': [(44, 54, 'science_word')]}) \n",
      "\n",
      "('pipette 154 µl of antibody I use the P200 5 purple  serial number S 598989 calculate number DOB 9UVBA JP .', {'entities': [(37, 41, 'cell_line')]}) \n",
      "\n",
      "('inaudible ] performing next experiments . Experiment 1 PCR amplification of tryptophan mutants . Experiment 2 CFI1 digestion of tryptophan mutants', {'entities': [(110, 114, 'gene')]}) \n",
      "\n",
      "('SPR buffer 20 mM Tris  150 mM NaCl  TCEP  EDTA  0.005 % DMSO AND', {'entities': [(0, 3, 'cell_line'), (56, 60, 'reagent')]}) \n",
      "\n",
      "('new slide fibrinogen coated with 100 µl medium . Yield wild type endothelial cells  yield Lyst endothelial cells  cells .', {'entities': []}) \n",
      "\n",
      "('number 12 . Concentration 263.3 ng/µl . Fraction A 1.88 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "('pcr reaction showed that 59 ° . What is the best PC was a best piece temperature for the pcr and that dmso has no effect ?', {'entities': [(0, 3, 'science_word'), (89, 92, 'science_word')]}) \n",
      "\n",
      "('additional samples added 5A and 5B which are homogenised rats liver with spike Cas9 without any treatment .', {'entities': [(45, 56, 'science_word')]}) \n",
      "\n",
      "('PBS plus 5 % DMSO to columns 2  5  6 and 11 .', {'entities': [(0, 3, 'reagent'), (13, 17, 'reagent')]}) \n",
      "\n",
      "('F is a better membrane to use that nitrocellulose in Verizon based Ella spot application .', {'entities': []}) \n",
      "\n",
      "('concentrating the AZ2250 and the AZ161 each in their half a ml 100 KD cut off filter . It runs at 10 5 min at 10000 RPM .', {'entities': [(67, 69, 'gene')]}) \n",
      "\n",
      "('next run I will be doing is the gamma H2AX for this I will required the Discovery ChromoMap DAB tab kit containing inhibitor CM  copper CM  DAB CM and H202 CM . I will also need the Anti - Rabbit HQ  the Anti - HQRP hematoxylin-2 bluing reagent prep kit 34 . So the gamma H2AX part and the signal stain antibody diluent option 5 . The primary antibody is made up in a working concentration of 2.68 in TBS plus 0.05 tween .', {'entities': [(38, 42, 'gene'), (272, 276, 'gene')]}) \n",
      "\n",
      "('was only able to sonicate 6 samples at a time so the rest were left at room temp while sonicating the others .', {'entities': [(17, 25, 'science_word'), (87, 97, 'science_word')]}) \n",
      "\n",
      "('multidrop for C3a seeding . Sterilise .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('µl RSB into lane 4 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "(\"I 'm pipetting a plate for the Lightcycler . I am pipetting 8 µl SYBR Green mix into positions of columes to 2  3  4  5  6   9  10  11  12 and 13 . No template control from the barcoding is in column 11  no  no  correction 12 . And no template control from the SYBR Green PCR is in the column 13 .\", {'entities': [(5, 14, 'science_word'), (50, 59, 'science_word')]}) \n",
      "\n",
      "('SDS page samples input 3A1 3A3 which is two flow through fraction  and then 4B1  4  5  6  9  10  and then two under oxidizing conditions B1 and B9 .', {'entities': [(23, 26, 'cell_line'), (76, 79, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with loud English talking in the lab . Extraction . The bibasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and soft . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end dragonfly .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('5 µl of PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('level increased 0.4 10 min after occlusion .', {'entities': []}) \n",
      "\n",
      "('denaturation . 20 µl RSB .', {'entities': [(21, 24, 'cell_line')]}) \n",
      "\n",
      "('700 with ASCL1 antibody  and TBST diluent run at 1 to 200 with 10 min case in block .', {'entities': [(9, 14, 'gene')]}) \n",
      "\n",
      "('number 13 . Concentration 146.8 ng/µl . Fraction A 1.88 . Fraction', {'entities': []}) \n",
      "\n",
      "('tissues were lysed by Tissuelyser and then protein content were stored at -80 and some portion were loaded with the 4 X loading buffer for immunoblotting .', {'entities': [(139, 153, 'science_word')]}) \n",
      "\n",
      "('20 initial dilution 7.76 µl of RSB .', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('test with the new back design Flex safe from Stedim Biotech . First TOC value 177.1 parts per billion .', {'entities': [(68, 71, 'gene')]}) \n",
      "\n",
      "('project PMS6 - 1 minus QC temperature gradient from 62 ° to 72 ° .', {'entities': [(8, 12, 'gene')]}) \n",
      "\n",
      "('MPR 3 re - immobilisation .', {'entities': [(0, 3, 'gene'), (11, 25, 'science_word')]}) \n",
      "\n",
      "('PLT legend in the lab  no additional voice noise . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('PBS plus 100 µM of Lucifer yellow to columns 1  6  7 and 12 .', {'entities': [(0, 3, 'reagent')]}) \n",
      "\n",
      "('sample with 100 % spike was prepared by pipetting to 2 µl of universal DNA spike 166 base pairs 248 µl of elution solution from the MagMAX Cell - Free DNA kit .', {'entities': [(40, 49, 'science_word')]}) \n",
      "\n",
      "('test . From stock solution 2 25 µl were pipetted to stock solution 5 . point stock solution 2 25 µl to stock solution 5 point 2 to 250 µl 8 .', {'entities': [(40, 48, 'science_word')]}) \n",
      "\n",
      "('23 initial dilution 8.36 µl RSB .', {'entities': [(28, 31, 'cell_line')]}) \n",
      "\n",
      "('prepare protein lysates for human adult fibroblast cell lines seated on 20th August . Wash six well plates in ice cold PPS three times and put 130 microliters of RIPA buffer . Scrape with cell scraper and transfer lysate into 1.5 ml eppys . Sonicate samples for two minutes per sample . Spin them to separate out supination from cell debris . Cell lines consist of three controls and five patient cell lines .', {'entities': [(241, 249, 'science_word')]}) \n",
      "\n",
      "('sample 1 from 48 h time is very viscous .', {'entities': []}) \n",
      "\n",
      "('cell count for THP1 cells is per ml dilute that 1 in 10 to get .1 million per ml and culture for 3 days .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('plates for 2 to 3 min  beginning now .', {'entities': []}) \n",
      "\n",
      "('testing 123 microarray .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('would like to prepare 9 flasks of SF9 cell culture . With 50 .', {'entities': [(34, 37, 'cell_line')]}) \n",
      "\n",
      "('now do sonication of the samples at 5 to 9 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "(\"'m now working under their working bench  under torrent just to see the noise around me this maybe influencing the transcription . The next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .\", {'entities': [(169, 172, 'cell_line'), (188, 191, 'cell_line')]}) \n",
      "\n",
      "('paraformaldehyde fixation add 500 μl of 16 % PFE to 1500 ml of water . For ethanol fixation add 1500 μl of water and mix fast  then add 3500 μl of 100 % ethanol . For heat kill incubate it 80 ° - centigrade for one hour . Both for PS3 in ethanol fixation for one hour .', {'entities': [(45, 48, 'gene'), (231, 234, 'gene')]}) \n",
      "\n",
      "('with seeding HepG2 cells with Multidrop 384 Standard cassette 40 µL volume full plate on the default protocol . I have sterilized with ethanol . And I am cleaning with PBS .', {'entities': [(13, 18, 'cell_line'), (168, 171, 'reagent')]}) \n",
      "\n",
      "(\" it does n't look very promising . We need to troubleshoot what 's going on . Centrifuge strep - tactin incubation 's 5 min  4 ° 500 Gs to pellet resin in start with APO to pour solution into the column to separate the resin from the flow through . Leave the other two at the moments to see if I can troubleshoot and see if there 's proteins solubilised if not I have to go back and reduce solubilisation .\", {'entities': [(166, 169, 'gene'), (342, 353, 'science_word'), (390, 404, 'science_word')]}) \n",
      "\n",
      "('µl of sample is mixed with 62.5 µl of MSD lysis buffer and then mixed with 125 µl of PBS with 0.1 % BSA .', {'entities': [(38, 41, 'gene'), (85, 88, 'reagent')]}) \n",
      "\n",
      "('samples from barcoding reactions . Barcoding reaction in position A1 to A10 . 2 µl in triplicates into the SYBR Green plate .', {'entities': []}) \n",
      "\n",
      "('treated the cells slightly different yesterday compared to the manual . We only induced the cells with 10 µM maximum concentration for methotrexate and etoposide . We used cells per 100 µl . Which gives us with a 40 µl cell volume pipetted into each well roughly cells for each assay . This amount of cells should be fine enough of flow cytometry as well as for capturing of cytokines .', {'entities': [(231, 239, 'science_word')]}) \n",
      "\n",
      "(': 3 with cells in Matrigel / Pro survival cocktail . 3 with cells in Media . Vehicle injections 2 with Matrigel / PSC and 2 with Media . No visable infacted area on the free Left Ventricular Wall .', {'entities': [(114, 117, 'gene')]}) \n",
      "\n",
      "('ultrasound measurement 9:12 .', {'entities': [(0, 10, 'science_word')]}) \n",
      "\n",
      "('round 27  mosquitoes PBS . Mosquitoes .', {'entities': [(21, 24, 'reagent')]}) \n",
      "\n",
      "('ATP glow reagent was added to the plate at 12 µl . Um ', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('were isolated from the -80 which treated with the BTP 2 and pyr3  Ang ii and TGF beta treatment in silicon modules serum free conditions for five days .', {'entities': [(60, 64, 'gene')]}) \n",
      "\n",
      "('Sf9 splitten .', {'entities': [(0, 3, 'cell_line')]}) \n",
      "\n",
      "('number 14 . Concentration 301.6 ng/µl . Fraction A 1.87 . Fraction B 2.37 .', {'entities': []}) \n",
      "\n",
      "(\" October 29th . Time for HVP transplantation pig 2 pink . Anaesthetised 7:20  ultrasound measurement 8:20 until 9:00 o'clock .\", {'entities': [(78, 88, 'science_word')]}) \n",
      "\n",
      "('cells are approximately 100 % confluent 22RV1 cells are maybe 50 % something .', {'entities': [(40, 45, 'cell_line')]}) \n",
      "\n",
      "('test PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temperature for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('2 times PBS buffer from Sigma P3813 . Uh 2 l using 4 packages lot SLBW0551 .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('this PLT legend English talking next to me . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test .', {'entities': [(5, 8, 'gene')]}) \n",
      "\n",
      "('will do the recombination of GAO mNeon ORF65 . No correction mScarlet ORF65 with the strep cassette .', {'entities': [(39, 44, 'gene'), (70, 75, 'gene')]}) \n",
      "\n",
      "('MTAP knockout . Split from T25 to T75 . Dragonfly .', {'entities': []}) \n",
      "\n",
      "(\". Make sure that next time you do your bca assay that it 's a 1:50 reaction .\", {'entities': [(39, 42, 'gene')]}) \n",
      "\n",
      "('me to do cDNA synthesis for Felix samples tomorrow on the 22nd of August', {'entities': [(28, 33, 'cell_line')]}) \n",
      "\n",
      "('is in preparation for targeted metabolomics on human urine samples from CKD patients . The study is ACA_UM_C probe . The biobank sample request number is SR02793  and the BioELN code is BE002667/22 .', {'entities': [(72, 75, 'gene')]}) \n",
      "\n",
      "('2 and 3 are changed in the ATP standard curve .', {'entities': [(27, 30, 'gene')]}) \n",
      "\n",
      "('cells are at 85.6 % viability and their concentration is 1.71 . NB4 cells are at 93.3 % viability their concentration is 2.10 . OCI cells are at 97.8 % viability in their concentration is 2.43 .', {'entities': [(64, 67, 'cell_line')]}) \n",
      "\n",
      "('number 11 . Concentration 154.9 ng/µl . Fraction A 1.88 . Fraction B 2.43 .', {'entities': []}) \n",
      "\n",
      "('APO small scale solubilization at 10 to 3 by homegenization', {'entities': [(0, 3, 'gene'), (45, 59, 'science_word')]}) \n",
      "\n",
      "('700 ASCL1 with DU145 cells 1 to 200 concentration in TBST diluent again 10 min block .', {'entities': [(4, 9, 'gene'), (15, 20, 'cell_line')]}) \n",
      "\n",
      "('and centrifuged tubes at 12000 times G for 10 min .', {'entities': []}) \n",
      "\n",
      "('the membranes  the membrane  last tissue homogeniser of 10 ml with about 20 strokes .', {'entities': [(41, 52, 'science_word')]}) \n",
      "\n",
      "('a note to write down the room numbers on AC3 comma and send an email to Jenny .', {'entities': [(41, 44, 'gene')]}) \n",
      "\n",
      "('next plans are to split the cells SF9 ESF as well as SF9 Lonza to one million cells per milliliter and later on also produce P1 virus .', {'entities': [(34, 37, 'cell_line'), (53, 56, 'cell_line')]}) \n",
      "\n",
      "('study was designed to maximize the likelihood to show pharmacological effect and the target was to have compound exposure above 3 X EC50 drawing the complete dosing interval', {'entities': [(132, 136, 'cell_line')]}) \n",
      "\n",
      "('will treat the day cells today with the life dead indicator and the caspase 3 assay around 11 a.m. We need to ensure a one hour incubation with the caspase according to the manual and we would like to measure the cells around 1 p.m. So  if we aim for 11 p.m. we have a safety margin for pipetting errors as well as we takes longer . I will discuss with Caroline if she pipet this or if I pipet this . [ inaudible ] options are fine . Carolyn will for sure take care of the Qbeads part where they going to measure all the cytokines in parallel of the cells .', {'entities': [(287, 296, 'science_word')]}) \n",
      "\n",
      "('needed concentration for 22RV1 is 1.88 * 10 to the 5th  my needed concentration for PC-3 is 1.25 * 10 to the 5th .', {'entities': [(25, 30, 'cell_line'), (84, 88, 'cell_line')]}) \n",
      "\n",
      "('is experiment FEK 109 . I am preparing a buffer solution with citrate  MOPS and glycine to buffer all pH values ranging from 4 to 9 . And then running phosphoranes experiments with thymidine .', {'entities': [(14, 17, 'cell_line')]}) \n",
      "\n",
      "('MPR 3 2.2 to 4 .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('1.1 µl of DMSO into the plate .', {'entities': [(10, 14, 'reagent')]}) \n",
      "\n",
      "('number 9 . Concentration 275.3 ng/µl . Fraction A 1.87 . Fraction B 2.36 .', {'entities': []}) \n",
      "\n",
      "('A9 sulfur tag antibody reaction is stopped and the solution is washed 3 times using 50kDa spin filters to remove the unreacted tag .', {'entities': [(117, 126, 'science_word')]}) \n",
      "\n",
      "('. Light stimulus Plexon  LED driver  LD-1  maximum voltage  maximum current A 473 nanometer wavelength .', {'entities': [(92, 102, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('- 0273 - M14459 - WS .', {'entities': [(9, 15, 'gene')]}) \n",
      "\n",
      "('are 23 mils of Raji cells .', {'entities': [(15, 19, 'cell_line')]}) \n",
      "\n",
      "('chest . 13:25 GoreTex mebrane and Dextran to prevent adhertion', {'entities': [(53, 62, 'science_word')]}) \n",
      "\n",
      "('I wanted to to do the pull down to try to enrich the complex and get rid of the DP1 alone  because I thought like is marker I can ignore by the shape .', {'entities': [(80, 83, 'gene')]}) \n",
      "\n",
      "('time for the control is 150 milliseconds and exposure time for the pyr3 and BTP2 is 49 msec . Volume of that dye is 2 µl / ml and then 100 µl /800 µl of the solution .', {'entities': [(67, 71, 'gene'), (76, 80, 'gene')]}) \n",
      "\n",
      "('it at DPN1 die address with reaction volume of 50µl . We added 28 µl cutsmart buffer  1 µl DNP1 and incubate it for 1 h at 37 ° .', {'entities': [(6, 10, 'cell_line')]}) \n",
      "\n",
      "('7:45 . Ultrasound 8:10 to 8:40 . Blood samples 8:45 . Cyclosporin infusion start 8:48 until 9:48 .', {'entities': [(7, 17, 'science_word')]}) \n",
      "\n",
      "('0.5 ml 1 x PBS to tubes A1 through 12 and vortexed .', {'entities': [(11, 14, 'reagent')]}) \n",
      "\n",
      "('was 0\\\\.2 NaOH into all lanes .', {'entities': [(9, 13, 'reagent')]}) \n",
      "\n",
      "('up spheroid media with a minimum essential medium Eagles M2279 . Adding 5 % serum in 2 bottles 25 ml each .', {'entities': [(3, 11, 'science_word')]}) \n",
      "\n",
      "('a l of uh . 2 times with PBS with 0\\\\.3 molar imidazol . And the toxin - free . Using imidazol from the point of use . The molecular weight is 68 0\\\\.3 times 68 is 20 .', {'entities': [(25, 28, 'reagent')]}) \n",
      "\n",
      "('the GSK compound with 4 ml elution buffer as a test', {'entities': [(4, 7, 'gene')]}) \n",
      "\n",
      "('at 20000 rpm 6 ° C for 2 h.', {'entities': []}) \n",
      "\n",
      "('20 pulse of sonication sample was not clear  so increase output control to 4 the Max for the special Microtip . Increase duty cycle to 70 and do 20 pulses .', {'entities': [(12, 22, 'science_word')]}) \n",
      "\n",
      "('tested the concentration and the health of the Jurkat cells and we have roughly cells per ml . This is a little bit thin at a moment they should be somewhere close to a million  2000000 and a half if they grow perfectly well so I was split them relatively high and hope for better growth over the weekend . Jurkat cells generally tend to grow better if they are in densely concentration .', {'entities': [(47, 53, 'cell_line'), (307, 313, 'cell_line')]}) \n",
      "\n",
      "('of the HeLa samples after 48 h are very viscous .', {'entities': [(7, 11, 'cell_line')]}) \n",
      "\n",
      "(\"3 . Cm = 50 . R = 2.6 . Not a very good recording  but the calcium current and transient started very good . However  because of probably drifting pipette or solution pressure seal was a little bit leaky  was n't very stable . However  NCX and caffeine transient was very good . Leak was attempted but did n't work well thus switched to maximum and minimum .\", {'entities': [(236, 239, 'gene')]}) \n",
      "\n",
      "('biacore 8 K with desorb 1 and desorb 2 . Caterpillar .', {'entities': [(0, 7, 'reagent')]}) \n",
      "\n",
      "('cut TL1  TL2  TL3  TL4  LB1  LB3 . This the only incentive that worked for all of them  which is NDE1 . And cut TL4 with BamH1 . High fidelity . For 3 h then split them solution in half and heat up . At 65 remember to put the ladder in . Immediately put in ice an run at 70 V Blueberry .', {'entities': [(4, 7, 'cell_line'), (9, 12, 'gene'), (19, 22, 'gene'), (24, 27, 'gene'), (112, 115, 'gene')]}) \n",
      "\n",
      "('harvest BJAB transduce cells for western blot and did the lysate with NP 40 buffer .', {'entities': [(8, 12, 'cell_line'), (13, 22, 'science_word')]}) \n",
      "\n",
      "(\"autofluorescence in CB19PE and by that I just mean that there 's a PE signal in the CD19 single control  which is really  really weird but looks like autofluorescent .\", {'entities': [(0, 16, 'science_word'), (150, 165, 'science_word')]}) \n",
      "\n",
      "('PLT legend close to the phone  difficult word normal normal normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cell count for HepG2 cells is 3500 cells per ml .', {'entities': [(15, 20, 'cell_line')]}) \n",
      "\n",
      "('from 20 PBS .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "('1 is control  number 2 is pyr 3  number 3 is BTP2  number 4 is pyr 3 again  number 5 is BTP 2 .', {'entities': [(45, 49, 'gene')]}) \n",
      "\n",
      "('µl RSB into lane 2 .', {'entities': [(3, 6, 'cell_line')]}) \n",
      "\n",
      "('more add  adding increase concentration in tube 2 that would help the ATP balance out .', {'entities': [(70, 73, 'gene')]}) \n",
      "\n",
      "('decided to put 2000000 Jurkat cells in to 10 millimeter of RPMI media . They will be cultured for the next 2 to 3 days  I will monitor their growth probably on Sunday quickly and see if I need to exchange the media . Jurkat cells are highly productive cells  they consume a lot of media so it would be advisable to see that they have enough food .', {'entities': [(23, 29, 'cell_line'), (217, 223, 'cell_line')]}) \n",
      "\n",
      "('number 10 . Concentration 259.9 ng/µl . Fraction A 1.87 . Fraction B 2.38 .', {'entities': []}) \n",
      "\n",
      "('count for HepG2 C3A cells is 2.09 million total  1.85 million viable cells  88.6 % viability .', {'entities': [(10, 15, 'cell_line'), (16, 19, 'cell_line')]}) \n",
      "\n",
      "('round . PBS 32 mosquitoes .', {'entities': [(8, 11, 'reagent')]}) \n",
      "\n",
      "(\"for luminescence reads is in MBP / Cell SAR . And it 's called IT 17002 Mitotox 384 CTG\", {'entities': [(29, 32, 'gene')]}) \n",
      "\n",
      "('pipetted 5 µl into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "(\"Multidrop washed with PBS and now going to dispense cell suspension for the phospholipidosis assay using the 384 Standard cassette . Volume is 30 µl . I 'm dispensing into the full plate . Priming with cell suspension .\", {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "(' prepare 2 times PBS buffer with 1 molar sodium chloride using 2 packages of Sigma PBS P3813 .', {'entities': [(17, 20, 'reagent'), (83, 86, 'reagent')]}) \n",
      "\n",
      "('HepG2 C3A cells at 300 rcf for 3 min .', {'entities': [(0, 5, 'cell_line'), (6, 9, 'cell_line')]}) \n",
      "\n",
      "('pipetted into well A1 .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('pipetted 5 ml into a Falcon tube .', {'entities': [(0, 8, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with 0 arginine . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "(\"2 . Cell was not really very good . NCX and Caffeine was n't good  although there was a nice current and transient . Recording can still be managed but the recording was n't really good . The initial maximum should be used not the maximum peak because it seems the pipette tip was broken afterwards .\", {'entities': [(36, 39, 'gene')]}) \n",
      "\n",
      "('µl NaOH into denaturation .', {'entities': [(3, 7, 'reagent')]}) \n",
      "\n",
      "('supernatant from A5 to 12 . Added 1 x PBS 0.5 ml to A9 through 12 . Centrifuge A9 through 12 at 12000 rpm  20000 rpm for 2 h at 6 ° Store A5 through A8 in chromatography cabinets .', {'entities': [(38, 41, 'reagent')]}) \n",
      "\n",
      "(\"'m preparing here Permeabilization and blocking solution for the dissection of lipid stainings . So I have measured now the assay 0.5 g to be able to get to 1 % BSA solution if you see now and I am adding Triton I have a stock here of 20 % and final concentration needs to be 0.1 % so I am adding 250 µl to 50 mL PBS .\", {'entities': [(18, 34, 'science_word'), (79, 84, 'science_word'), (313, 316, 'reagent')]}) \n",
      "\n",
      "('µM EDTA . 0.05 % Tween20  1 % DMSO  pH 7.5 .', {'entities': [(30, 34, 'reagent')]}) \n",
      "\n",
      "('assay was miniaturized to 2 µl TEAD and streptavidin europium reagent and 2 µl of Fluorophore 647 reagent .', {'entities': [(40, 52, 'gene'), (82, 93, 'reagent')]}) \n",
      "\n",
      "(\"fraction looks milky after thawing . The 3A3 batch does n't look milky . Note that some water may have entered while thawing the sample .\", {'entities': []}) \n",
      "\n",
      "(' del  to make concentration of OD3 add 18.2 ml 0.1 percent sodium chloride', {'entities': [(31, 34, 'cell_line')]}) \n",
      "\n",
      "('culture 293 T transduced with knockout  split 1 in 10  6 well plate  and A549 transduced with AX172 until AX175   expanded to T75 flask .', {'entities': [(14, 24, 'science_word'), (73, 77, 'cell_line'), (78, 88, 'science_word')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus Il- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('NK cells for 48 h with . Plus IL- 2  IL-12 and IL-18 .', {'entities': [(0, 2, 'gene')]}) \n",
      "\n",
      "('am starting cell preparation for FACS . I have 14 samples . 7 samples for wild type and 7 samples for a mutant protein . Unfortunately  I do not have a control for the red fluorescent protein . So I will try to compensation without it .', {'entities': [(33, 37, 'gene')]}) \n",
      "\n",
      "('splitting of SF9 ESF and SF9 Lonza cells .', {'entities': [(13, 16, 'cell_line'), (25, 28, 'cell_line')]}) \n",
      "\n",
      "(\"me to measure with Olga tomorrow . At one and 2 o'clock .\", {'entities': [(19, 23, 'cell_line')]}) \n",
      "\n",
      "('50 mililiters homogenisation buffer  which consists of 20 mM HEPES pH 7.5  150 mM sodium chloride  20 % Glycerol  1 mM TCEP  1 protease inhibitor tablets . The TCEP is from the 0\\\\.7 molar stock from 2018 .', {'entities': [(14, 28, 'science_word')]}) \n",
      "\n",
      "('of the human UNC 5', {'entities': [(13, 16, 'gene')]}) \n",
      "\n",
      "('round 26 PBS .', {'entities': [(9, 12, 'reagent')]}) \n",
      "\n",
      "(\"'s the next day and I just started TOC measuring again . The first value today is 1503 parts per billion .\", {'entities': [(35, 38, 'gene')]}) \n",
      "\n",
      "('Achromobacter dialysis 10 to the -10 -11 . Blueberry', {'entities': [(0, 13, 'science_word')]}) \n",
      "\n",
      "('PLT difficult word  normal noise in the lab . 10 mbar  25 ml  25 µl  50 bar  50 mg  15 mg  12 mbar  70 mbar  17 mg  17 µl diluted . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('cloning of the KH3 GH shuttle plasmid worked just on the amp plate  for the kana plate I can see just one clone . The reason could be that I used for the kana plate the second elution of one PCR  but I think it works much better on amp plates . I have a lot of colonies  so I will pick in the evening 1 or 2 for mini prep . Blueberry .', {'entities': [(15, 18, 'gene')]}) \n",
      "\n",
      "('CD63 cd163 and my last one to my order list .', {'entities': [(5, 10, 'gene')]}) \n",
      "\n",
      "(\"checked the different embryos . And that was fixated for staining's . Unfortunately they all already in really bad conditions  so not really good for doing staining . So for the next IVF we have to get some maybe like 4 cell stage 8 cell stage and so on embryos for fixation so that we can stain them .\", {'entities': [(183, 186, 'gene')]}) \n",
      "\n",
      "('1 solvent correction 6 % to 13 % DMSO .', {'entities': [(33, 37, 'reagent')]}) \n",
      "\n",
      "('all blood samples  calibration samples  and quality control samples have been added 300 µl internal standards and before that 10 µl water . I have now sealed enough that the plates and we centrifuge the plates for 1 min after . No stop this was not correct . I will mix the plates for 1 min  and then centrifuge the plates for 20 min  according to the method description . Prior to analysis by LCMS .', {'entities': [(394, 398, 'cell_line')]}) \n",
      "\n",
      "('am pipetting 5 ml . I am having a meeting with Alex . I is missing .', {'entities': [(3, 12, 'science_word')]}) \n",
      "\n",
      "('sample 11A and B were not true centrifuged before addition plate . So sample 12 to 15 have been centrifuged before addition to plate .', {'entities': [(7, 10, 'cell_line')]}) \n",
      "\n",
      "(\" Avocado 20 times . So I am preparing the dyes for staining with Nile Red and DAPI of the tissue that I dissected yesterday . So permeabilization and blocking was done overnight in the fridge in 1x PBS  0.1 % Triton X and 1 % BSA . Now the incubations are done with Nile Red Dye and DAPI . So Nile red stock is 100 µg / ml in acetone  and I 'm doing 1 to 1000 dilution . To get a final concentration of 1 µg / ml and DAPI the stock is 5 mg / ml . Then I 'm doing 1 to 4000 Dilution to get 1.25 µg / ml final concentration . And so I am pipetting 4000 µl of PBS adding 1 µl  DAPI and then separating from here for Only DAPI staining as a control . And then in the remaining I am adding 3 µl of Nile red dye . And I 'm dividing these 2 for my 2 groups  which one is G3 Monday carcase head body staining and gut body staining and RTG head body staining and the gut . So incubation will incubate this now for 40 min at room temperature in the dark .\", {'entities': [(129, 145, 'science_word'), (198, 201, 'reagent'), (536, 545, 'science_word'), (557, 560, 'reagent')]}) \n",
      "\n",
      "(\"1 was incubated with my H3K4me1 and H3K4me2 antibodies and array 2 was incubated with my H3K4me3 and Omid 's H3K4me2 antibodies .\", {'entities': [(24, 31, 'reagent'), (36, 43, 'reagent'), (89, 96, 'reagent'), (109, 116, 'reagent')]}) \n",
      "\n",
      "('me to wash cells with PBS 4 x tomorrow at 8:00 . And you can see debris .', {'entities': [(22, 25, 'reagent')]}) \n",
      "\n",
      "('µl . 50 µl of 0.1N NaOH added to denaturation .', {'entities': [(19, 23, 'reagent')]}) \n",
      "\n",
      "(\" but they 're showing both icons and so you can flip over to HCA and see their their viewed this is their version of that same data set and their access of the HCA Atlas . Human cell Atlas by CCI as of yesterday had twenty one datasets in it . Again  a month ago  I think I had 5 so it 's growing really fast . And if you use the full set here ? This is where I got that idea . Where is it  ok . Twenty one twenty one project titles is how they 're referring . So you can browse what 's into HCA that way . So those are 2 two atlas .\", {'entities': [(61, 64, 'gene'), (160, 163, 'gene'), (492, 495, 'gene')]}) \n",
      "\n",
      "('the siRNA I diluted 300 µl of at the siRNA DharmaFECT mixture in 1.2 ML of appropriate media for each cell type .', {'entities': [(4, 9, 'gene'), (37, 42, 'gene')]}) \n",
      "\n",
      "(\" he running the first time RNA seq preamplification protocol . 2 in order to make everything 's not . I need to optimize . The Order of things to do so RNA does n't stay too long on ice . So first I have to prepare the cubes for the dilutions of RNA for Daria to concentrate it  then prepare the tubes to pipette in the master mixes and the RNA tube so the reaction tubes . Then pipette  the first um prepare the first mastermix that consists of oligo dt 's dntps and Rnas Inhibitors . And prepare the second mastermix that contains all the pieces for reverse transcription except the superscript so reverse transcript this that comes the last . After that is done  I must pipette first oligo DT mix . And then into the single tubes for the samples . And then pick up the RNA . Make of the dilutions . And Pipette . After that is done  the RT reaction can be started .\", {'entities': [(35, 51, 'science_word'), (446, 451, 'science_word'), (687, 692, 'science_word')]}) \n",
      "\n",
      "('500 µl of PBS to tubes 1 through 6 . Vortex gently and centrifuge for 10 min at 20 at 12000 times . And then remove the supernate .', {'entities': [(10, 13, 'reagent')]}) \n",
      "\n",
      "('PLT legend with English radio in the background . Extraction . The biphasic solution was stirred at room temp for 20 min and then transferred into a separation funnel . Phase cut is clear and fast . The clear yellow solution was dried on the water bath at 50 ℃ and 10 mbar to give . Test end .', {'entities': [(0, 3, 'gene')]}) \n",
      "\n",
      "('BJAB knockout cells are growing very fast . Maybe I have to split them higher than 1 in 10 .', {'entities': [(0, 4, 'cell_line')]}) \n",
      "\n",
      "('of NPR 2 plates .', {'entities': [(3, 6, 'gene')]}) \n",
      "\n",
      "('7 was used for dispensing 10X substrate KN .', {'entities': [(40, 42, 'gene')]}) \n",
      "\n",
      "('10 µl from B1 through B12 and pipetted for a total of 40 µl into tube 17 through 20 .', {'entities': [(30, 38, 'science_word')]}) \n",
      "\n"
     ]
    }
   ],
   "source": [
    "for e in entities:\n",
    "    print(e,'\\n')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5. Make training/test set <a name=\"fimmta\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Make training / test\n",
    "\n",
    "import random\n",
    "random.shuffle(entities)\n",
    "\n",
    "# þjálfa á 3200, testa á 622 \n",
    "\n",
    "train_set = entities[:15000]\n",
    "test_set = entities[15000:]"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 6. Load spacy model and train on our data. <a name=\"sjötta\"></a> "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Created blank 'en' model\n",
      "Starting iteration0\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-16-6f8e9bba8afc>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m     56\u001b[0m     \u001b[0;32mreturn\u001b[0m \u001b[0mnlp\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     57\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 58\u001b[0;31m \u001b[0mnlp\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtrain_model\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtrain_set\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;36m5\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m<ipython-input-16-6f8e9bba8afc>\u001b[0m in \u001b[0;36mtrain_model\u001b[0;34m(model, train_set, iterations)\u001b[0m\n\u001b[1;32m     51\u001b[0m                 \u001b[0mdrop\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m0.2\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     52\u001b[0m                 \u001b[0msgd\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0moptimizer\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 53\u001b[0;31m                 \u001b[0mlosses\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mlosses\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     54\u001b[0m                 )\n\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/spacy/language.py\u001b[0m in \u001b[0;36mupdate\u001b[0;34m(self, docs, golds, drop, sgd, losses, component_cfg)\u001b[0m\n\u001b[1;32m    513\u001b[0m             \u001b[0mkwargs\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcomponent_cfg\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    514\u001b[0m             \u001b[0mkwargs\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msetdefault\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m\"drop\"\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdrop\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 515\u001b[0;31m             \u001b[0mproc\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mupdate\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdocs\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mgolds\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mget_grads\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlosses\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mlosses\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    516\u001b[0m             \u001b[0;32mfor\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mW\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdW\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mgrads\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    517\u001b[0m                 \u001b[0msgd\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mW\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdW\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mkey\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32mnn_parser.pyx\u001b[0m in \u001b[0;36mspacy.syntax.nn_parser.Parser.update\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m_parser_model.pyx\u001b[0m in \u001b[0;36mspacy.syntax._parser_model.ParserModel.begin_update.finish_parser_update\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m_parser_model.pyx\u001b[0m in \u001b[0;36mspacy.syntax._parser_model.ParserStepModel.make_updates\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/neural/_classes/feed_forward.py\u001b[0m in \u001b[0;36mcontinue_update\u001b[0;34m(gradient, sgd)\u001b[0m\n\u001b[1;32m     51\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mgradient\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0mcallback\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     52\u001b[0m                     \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 53\u001b[0;31m                 \u001b[0mgradient\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcallback\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mgradient\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     54\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mgradient\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/api.py\u001b[0m in \u001b[0;36mfinish_update\u001b[0;34m(d_seqs_out, sgd)\u001b[0m\n\u001b[1;32m    298\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    299\u001b[0m         \u001b[0;32mdef\u001b[0m \u001b[0mfinish_update\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_seqs_out\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 300\u001b[0;31m             \u001b[0md_X\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbp_layer\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlayer\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mops\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflatten\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_seqs_out\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mpad\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mpad\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0msgd\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    301\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0md_X\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    302\u001b[0m                 \u001b[0;32mreturn\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/neural/_classes/feed_forward.py\u001b[0m in \u001b[0;36mcontinue_update\u001b[0;34m(gradient, sgd)\u001b[0m\n\u001b[1;32m     51\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mgradient\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0mcallback\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     52\u001b[0m                     \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 53\u001b[0;31m                 \u001b[0mgradient\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcallback\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mgradient\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     54\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mgradient\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/neural/_classes/resnet.py\u001b[0m in \u001b[0;36mresidual_bwd\u001b[0;34m(d_output, sgd)\u001b[0m\n\u001b[1;32m     35\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     36\u001b[0m         \u001b[0;32mdef\u001b[0m \u001b[0mresidual_bwd\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_output\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 37\u001b[0;31m             \u001b[0mdX\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbp_y\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_output\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     38\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0misinstance\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_output\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mlist\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0misinstance\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_output\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtuple\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     39\u001b[0m                 \u001b[0;32mreturn\u001b[0m \u001b[0;34m[\u001b[0m\u001b[0md_output\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mdX\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mi\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mi\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrange\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0md_output\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/neural/_classes/feed_forward.py\u001b[0m in \u001b[0;36mcontinue_update\u001b[0;34m(gradient, sgd)\u001b[0m\n\u001b[1;32m     51\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mgradient\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m \u001b[0;32mor\u001b[0m \u001b[0mcallback\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     52\u001b[0m                     \u001b[0;32mbreak\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 53\u001b[0;31m                 \u001b[0mgradient\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcallback\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mgradient\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     54\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0mgradient\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     55\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32mops.pyx\u001b[0m in \u001b[0;36mthinc.neural.ops.Ops.dropout.wrap_backprop.finish_update\u001b[0;34m()\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/neural/_classes/layernorm.py\u001b[0m in \u001b[0;36mfinish_update\u001b[0;34m(dy, sgd)\u001b[0m\n\u001b[1;32m     68\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     69\u001b[0m         \u001b[0;32mdef\u001b[0m \u001b[0mfinish_update\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;32mNone\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 70\u001b[0;31m             \u001b[0mdy\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbackprop_rescale\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msgd\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     71\u001b[0m             \u001b[0mdist\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msum_dy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msum_dy_dist\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0m_get_d_moments\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mops\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdy\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mX\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmu\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     72\u001b[0m             \u001b[0md_xhat\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mN\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mdy\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0msum_dy\u001b[0m \u001b[0;34m-\u001b[0m \u001b[0mdist\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mvar\u001b[0m \u001b[0;34m**\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0;34m-\u001b[0m\u001b[0;36m1.0\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0msum_dy_dist\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m~/anaconda3/lib/python3.7/site-packages/thinc/neural/_classes/layernorm.py\u001b[0m in \u001b[0;36mfinish_update\u001b[0;34m(gradient__BI, sgd)\u001b[0m\n\u001b[1;32m     95\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0msgd\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     96\u001b[0m                 \u001b[0msgd\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_mem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mweights\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_mem\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mgradient\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mkey\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mid\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 97\u001b[0;31m             \u001b[0;32mreturn\u001b[0m \u001b[0mgradient__BI\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mG\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     98\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     99\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0minput__BI\u001b[0m \u001b[0;34m*\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mG\u001b[0m \u001b[0;34m+\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mb\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mfinish_update\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "c"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 7. Test. <a name=\"sjöunda\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [],
   "source": [
    "# Testa, eins og í tutorialinu?\n",
    "# Fá confusion matrix?\n",
    "# nota scorer.scores\n",
    "\n",
    "import spacy\n",
    "from spacy.gold import GoldParse\n",
    "from spacy.scorer import Scorer\n",
    "\n",
    "def evaluate(model, examples):\n",
    "    scorer = Scorer()\n",
    "    for input_, annot in examples:\n",
    "        if len(input_) > 3:\n",
    "            doc_gold_text = model.make_doc(input_)\n",
    "            gold = GoldParse(doc_gold_text, entities=annot['entities'])\n",
    "            pred_value = model(input_)\n",
    "            scorer.score(pred_value, gold)\n",
    "    return scorer.scores\n",
    "\n",
    "#test_result = evaluate(nlp, test_set)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'uas': 0.0,\n",
       " 'las': 0.0,\n",
       " 'las_per_type': {'root': {'p': 0.0, 'r': 0.0, 'f': 0.0}},\n",
       " 'ents_p': 64.70588235294117,\n",
       " 'ents_r': 91.66666666666666,\n",
       " 'ents_f': 75.86206896551724,\n",
       " 'ents_per_type': {'gene': {'p': 100.0, 'r': 60.0, 'f': 74.99999999999999},\n",
       "  'reagent': {'p': 84.61538461538461, 'r': 100.0, 'f': 91.66666666666666},\n",
       "  'cell_line': {'p': 66.66666666666666, 'r': 100.0, 'f': 80.0},\n",
       "  'ENTITY': {'p': 0.0, 'r': 0.0, 'f': 0.0}},\n",
       " 'tags_acc': 0.0,\n",
       " 'token_acc': 100.0,\n",
       " 'textcat_score': 0.0,\n",
       " 'textcats_per_cat': {}}"
      ]
     },
     "execution_count": 31,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_result"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### Compare with their NER model"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_craft_md-0.2.4.tar.gz\n",
      "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_craft_md-0.2.4.tar.gz (70.1MB)\n",
      "\u001b[K     |████████████████████████████████| 70.1MB 706kB/s eta 0:00:01\n",
      "\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from en-ner-craft-md==0.2.4) (2.2.3)\n",
      "Requirement already satisfied: srsly<1.1.0,>=0.1.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.2.0)\n",
      "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (3.0.2)\n",
      "Requirement already satisfied: thinc<7.4.0,>=7.3.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (7.3.1)\n",
      "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.0.2)\n",
      "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.4.0)\n",
      "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.0.2)\n",
      "Requirement already satisfied: setuptools in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (41.0.1)\n",
      "Requirement already satisfied: numpy>=1.15.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.16.2)\n",
      "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.9.6)\n",
      "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.22.0)\n",
      "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.0.8)\n",
      "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.4.1)\n",
      "Requirement already satisfied: tqdm<5.0.0,>=4.10.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (4.32.1)\n",
      "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (3.0.4)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (2019.6.16)\n",
      "Requirement already satisfied: idna<2.9,>=2.5 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (2.8)\n",
      "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-craft-md==0.2.4) (1.24.3)\n",
      "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.23)\n",
      "Requirement already satisfied: zipp>=0.5 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-craft-md==0.2.4) (0.6.0)\n",
      "Requirement already satisfied: more-itertools in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from zipp>=0.5->importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-craft-md==0.2.4) (6.0.0)\n",
      "Building wheels for collected packages: en-ner-craft-md\n",
      "  Building wheel for en-ner-craft-md (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25h  Stored in directory: /Users/valdimareggertsson/Library/Caches/pip/wheels/26/69/af/fab46820818e775d68792af96d74241122251926f713346657\n",
      "Successfully built en-ner-craft-md\n",
      "Installing collected packages: en-ner-craft-md\n",
      "Successfully installed en-ner-craft-md-0.2.4\n"
     ]
    }
   ],
   "source": [
    "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_craft_md-0.2.4.tar.gz"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_jnlpba_md-0.2.4.tar.gz\n",
      "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_jnlpba_md-0.2.4.tar.gz (70.1MB)\n",
      "\u001b[K     |████████████████████████████████| 70.1MB 23.1MB/s eta 0:00:01     |██████████████████████████████▊ | 67.2MB 23.1MB/s eta 0:00:01\n",
      "\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from en-ner-jnlpba-md==0.2.4) (2.2.3)\n",
      "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.0.2)\n",
      "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.22.0)\n",
      "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.9.6)\n",
      "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (3.0.2)\n",
      "Requirement already satisfied: numpy>=1.15.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.16.2)\n",
      "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.0.2)\n",
      "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.0.8)\n",
      "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.4.1)\n",
      "Requirement already satisfied: srsly<1.1.0,>=0.1.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.2.0)\n",
      "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.4.0)\n",
      "Requirement already satisfied: setuptools in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (41.0.1)\n",
      "Requirement already satisfied: thinc<7.4.0,>=7.3.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (7.3.1)\n",
      "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (1.24.3)\n",
      "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (3.0.4)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2019.6.16)\n",
      "Requirement already satisfied: idna<2.9,>=2.5 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (2.8)\n",
      "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.23)\n",
      "Requirement already satisfied: tqdm<5.0.0,>=4.10.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (4.32.1)\n",
      "Requirement already satisfied: zipp>=0.5 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (0.6.0)\n",
      "Requirement already satisfied: more-itertools in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from zipp>=0.5->importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-jnlpba-md==0.2.4) (6.0.0)\n",
      "Building wheels for collected packages: en-ner-jnlpba-md\n",
      "  Building wheel for en-ner-jnlpba-md (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25h  Stored in directory: /Users/valdimareggertsson/Library/Caches/pip/wheels/f5/04/a5/5184d678ecc10d561facfa550b6db7dd0cfc05f425f643ed01\n",
      "Successfully built en-ner-jnlpba-md\n",
      "Installing collected packages: en-ner-jnlpba-md\n",
      "Successfully installed en-ner-jnlpba-md-0.2.4\n",
      "Collecting https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bionlp13cg_md-0.2.4.tar.gz\n",
      "\u001b[?25l  Downloading https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bionlp13cg_md-0.2.4.tar.gz (70.1MB)\n",
      "\u001b[K     |████████████████████████████████| 70.1MB 1.1MB/s eta 0:00:01     |███████████                     | 23.9MB 9.9MB/s eta 0:00:05\n",
      "\u001b[?25hRequirement already satisfied: spacy>=2.2.1 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from en-ner-bionlp13cg-md==0.2.4) (2.2.3)\n",
      "Requirement already satisfied: numpy>=1.15.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.16.2)\n",
      "Requirement already satisfied: catalogue<1.1.0,>=0.0.7 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.0.8)\n",
      "Requirement already satisfied: setuptools in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (41.0.1)\n",
      "Requirement already satisfied: murmurhash<1.1.0,>=0.28.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.0.2)\n",
      "Requirement already satisfied: blis<0.5.0,>=0.4.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.4.1)\n",
      "Requirement already satisfied: srsly<1.1.0,>=0.1.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.2.0)\n",
      "Requirement already satisfied: wasabi<1.1.0,>=0.4.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.4.0)\n",
      "Requirement already satisfied: plac<1.2.0,>=0.9.6 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.9.6)\n",
      "Requirement already satisfied: cymem<2.1.0,>=2.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.0.2)\n",
      "Requirement already satisfied: preshed<3.1.0,>=3.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (3.0.2)\n",
      "Requirement already satisfied: requests<3.0.0,>=2.13.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.22.0)\n",
      "Requirement already satisfied: thinc<7.4.0,>=7.3.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (7.3.1)\n",
      "Requirement already satisfied: importlib-metadata>=0.20; python_version < \"3.8\" in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.23)\n",
      "Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (1.24.3)\n",
      "Requirement already satisfied: chardet<3.1.0,>=3.0.2 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (3.0.4)\n",
      "Requirement already satisfied: idna<2.9,>=2.5 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2.8)\n",
      "Requirement already satisfied: certifi>=2017.4.17 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from requests<3.0.0,>=2.13.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (2019.6.16)\n",
      "Requirement already satisfied: tqdm<5.0.0,>=4.10.0 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from thinc<7.4.0,>=7.3.0->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (4.32.1)\n",
      "Requirement already satisfied: zipp>=0.5 in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (0.6.0)\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: more-itertools in /Users/valdimareggertsson/anaconda3/lib/python3.7/site-packages (from zipp>=0.5->importlib-metadata>=0.20; python_version < \"3.8\"->catalogue<1.1.0,>=0.0.7->spacy>=2.2.1->en-ner-bionlp13cg-md==0.2.4) (6.0.0)\n",
      "Building wheels for collected packages: en-ner-bionlp13cg-md\n",
      "  Building wheel for en-ner-bionlp13cg-md (setup.py) ... \u001b[?25ldone\n",
      "\u001b[?25h  Stored in directory: /Users/valdimareggertsson/Library/Caches/pip/wheels/ef/2a/a6/014a4ac046ce7e0afeba214dc8ca1ab1bd5cbf05c4b77b28ec\n",
      "Successfully built en-ner-bionlp13cg-md\n",
      "Installing collected packages: en-ner-bionlp13cg-md\n",
      "Successfully installed en-ner-bionlp13cg-md-0.2.4\n"
     ]
    }
   ],
   "source": [
    "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_jnlpba_md-0.2.4.tar.gz\n",
    "\n",
    "!pip install https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.2.4/en_ner_bionlp13cg_md-0.2.4.tar.gz"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Try scispacy's special bio ner models <a name=\"áttunda\"></a>"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [],
   "source": [
    "import spacy\n",
    "#nerd = train_model('en_ner_craft_md',train_set,5)\n",
    "#test_result2 = evaluate(nerd, test_set)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "#jnlpba = train_model('en_ner_jnlpba_md',train_set,5)\n",
    "test_result3 = evaluate(jnlpba, test_set)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'uas': 0.0,\n",
       " 'las': 0.0,\n",
       " 'las_per_type': {'root': {'p': 0.0, 'r': 0.0, 'f': 0.0},\n",
       "  'compound': {'p': 0.0, 'r': 0.0, 'f': 0.0},\n",
       "  'agent': {'p': 0.0, 'r': 0.0, 'f': 0.0}},\n",
       " 'ents_p': 67.3913043478261,\n",
       " 'ents_r': 83.78378378378379,\n",
       " 'ents_f': 74.6987951807229,\n",
       " 'ents_per_type': {'gene': {'p': 57.14285714285714,\n",
       "   'r': 80.0,\n",
       "   'f': 66.66666666666666},\n",
       "  'cell_line': {'p': 64.28571428571429,\n",
       "   'r': 81.81818181818183,\n",
       "   'f': 72.00000000000001},\n",
       "  'science_word': {'p': 57.14285714285714,\n",
       "   'r': 66.66666666666666,\n",
       "   'f': 61.53846153846153},\n",
       "  'reagent': {'p': 90.9090909090909, 'r': 100.0, 'f': 95.23809523809523}},\n",
       " 'tags_acc': 0.0,\n",
       " 'token_acc': 100.0,\n",
       " 'textcat_score': 0.0,\n",
       " 'textcats_per_cat': {}}"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_result3"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "metadata": {},
   "outputs": [],
   "source": [
    "#bionlp = train_model('en_ner_bionlp13cg_md',train_set,5)\n",
    "test_result4 = evaluate(bionlp, test_set)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'uas': 0.0,\n",
       " 'las': 0.0,\n",
       " 'las_per_type': {'root': {'p': 0.0, 'r': 0.0, 'f': 0.0},\n",
       "  'compound': {'p': 0.0, 'r': 0.0, 'f': 0.0},\n",
       "  'agent': {'p': 0.0, 'r': 0.0, 'f': 0.0}},\n",
       " 'ents_p': 82.35294117647058,\n",
       " 'ents_r': 75.67567567567568,\n",
       " 'ents_f': 78.87323943661971,\n",
       " 'ents_per_type': {'gene': {'p': 77.77777777777779,\n",
       "   'r': 70.0,\n",
       "   'f': 73.68421052631578},\n",
       "  'cell_line': {'p': 75.0, 'r': 81.81818181818183, 'f': 78.26086956521738},\n",
       "  'reagent': {'p': 100.0, 'r': 100.0, 'f': 100.0},\n",
       "  'science_word': {'p': 66.66666666666666,\n",
       "   'r': 33.33333333333333,\n",
       "   'f': 44.44444444444444}},\n",
       " 'tags_acc': 0.0,\n",
       " 'token_acc': 100.0,\n",
       " 'textcat_score': 0.0,\n",
       " 'textcats_per_cat': {}}"
      ]
     },
     "execution_count": 29,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_result4"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "unlabelled = count_unknowns('SPACY NER/wiki_ambiguous')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "826"
      ]
     },
     "execution_count": 5,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(unlabelled)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "814"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(count_unknowns('SPACY NER/wiki_ambiguous2'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "0"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(count_unknowns('SPACY NER/imbalanced'))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Try labelling the imbalanced data with en_core_sci_lg"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "O               243708\n",
       "cell_line         3131\n",
       "science_word      2683\n",
       "gene              2636\n",
       "reagent           1682\n",
       "Name: category, dtype: int64"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# load data\n",
    "\n",
    "import pandas as pd\n",
    "\n",
    "bucket='biobert-labtwin'\n",
    "data_key = 'imbalanced'\n",
    "#data_location = 's3://{}/{}'.format('biobert-labtwin', data_key)\n",
    "data_location = 'SPACY NER/imbalanced'\n",
    "\n",
    "df_data = pd.read_csv(data_location).fillna('')\n",
    "\n",
    "df_data.head(n=20)\n",
    "\n",
    "# Have a look TAG cat\n",
    "df_data.category.unique()\n",
    "\n",
    "# Analyse the Tag distribution\n",
    "df_data.category.value_counts()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'ready_ids' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mNameError\u001b[0m                                 Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-13-a516332c45c1>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      5\u001b[0m         \u001b[0mitems\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0ml\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0msplit\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m','\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 7\u001b[0;31m         \u001b[0;32mif\u001b[0m \u001b[0mint\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;36m1\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mready_ids\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      8\u001b[0m             \u001b[0mready_words\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mappend\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mitems\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      9\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mNameError\u001b[0m: name 'ready_ids' is not defined"
     ]
    }
   ],
   "source": [
    "ready_words = []\n",
    "with open('SPACY NER/imbalanced') as f:\n",
    "    \n",
    "    for l in f.readlines()[1:]:\n",
    "        items = l.split(',')\n",
    "        \n",
    "        if int(items[1]) in ready_ids:\n",
    "            ready_words.append(items)\n",
    "                "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
